## TRANSCRIPT     
0:01     
hello hello how are you much better oh     
0:09     
good that's good okay Bradley question     
0:15     
okay uh that is use GitHub for depositing software yeah yeah we do that     
0:23     
okay are there some idiot proof instructions uh yeah I could we could do     
0:29     
that I have uh some tutorials but it really have to be like I probably have     
0:35     
to walk you through setting things up what were you interested in doing uh I'm     
0:40     
working with a kid who's using Python and uh I'd like to see him document his     
0:46     
code and uh put it in GitHub okay yeah that would be good uh I     
0:52     
can I can help with that if uh I don't know how the best way to do that would be but I I'll let him know but if you     
0:59     
have a document you can start with that okay yeah yeah yeah I I can set them up     
1:05     
with some things yeah okay great I talked to him at noon today oh     
1:14     
okay well that's great okay yeah we're doing something     
1:20     
very strange and uh you might want a presentation on it when we're ready uh     
1:25     
in a week or two I guess uh and that is we're trying to reconstruct pictures     
1:32     
from a subset of lines across them oh okay okay and apparently I don't know     
1:39     
any I don't think it's been done     
1:45     
right okay so uh I've come up with a very naive in painting algorithm and uh it     
1:52     
seems to be working oh good this is the one for uh the breast     
1:58     
cancer uh yeah it's leading up to it we're we're not doing the breast cancer case yet just just a standard picture     
2:05     
gray scale all right yeah that sounds great     
2:12     
yeah I'd like to see it when it's done yeah we're about half done     
2:20     
okay yeah     
2:25     
great organ how are you     
2:36     
usually oh yeah usually he's uh running errands or something in the morning good     
2:45     
morning yeah morning School run so that's     
2:50     
great uh yeah so that's I don't know if Susan's going to make it today     
2:56     
but sort of our nucleus no pun intended uh yeah     
3:04     
so I've got some things today uh I know Morgan wanted to go over this paper this     
3:09     
hypergraphs paper that I have out um probably also talk about some     
3:16     
things okay yeah and then Morgan says been meaning to follow up with Susan and     
3:21     
Moro oh the moro program yeah     
3:26     
okay uh so let me share my screen     
3:40     
all right so I'll start with hypergraph paper I guess uh so recently this is put     
3:47     
on the archive this is something I've working on for a while this is this paper hypergraphs demonstrate     
3:53     
anastamosis during Divergent integration and we we talked about this in a meeting oh several meetings last     
4:01     
year I think and so it's just been kind of going along and I kind of uh finish     
4:06     
put some finishing touches on it and put it out so um basically what this paper does     
4:14     
is it uses hyper graphs or graphs basically graphs within graphs to uh     
4:21     
model uh Mor morphogenesis embryogenesis Etc and so you know     
4:29     
there's a work that has been done uh earlier this is on Spa on different cell     
4:37     
differentiation processes of spatial networks so these are embryo networks that you know uh where you take the     
4:44     
embryo and you put it into a network representation but that's what we call a     
4:49     
first order Network so A first order network is where you have this point cloud of cells this is of course you     
4:57     
know if we uh take mic croscopy images and we extract the uh centroid out of     
5:04     
each one and then we plot them in this 3D spatial coordinate system and maybe     
5:10     
even a 4D coordinate system if you go over time and then you put together you know     
5:16     
you find the connections uh via a number of Criterion so it could be the distance     
5:21     
between the cells it could be signaling molecules it could be other things so     
5:27     
this point Cloud here as you can see this point cloud is growing as we go through time and we get cells dividing     
5:34     
and even differentiating in space but then also in terms of fate and you can build all     
5:40     
sorts of networks so you get these Point clouds that grow in in density and size     
5:46     
over time then you get you know you can like investigate what they look like sort of     
5:52     
two-dimensional uh slices and then you can actually build these circular networks so these are these networks are     
6:00     
rather disembodied these these plots because what they do is they just take like they put things in a circle and     
6:06     
they show the connections between cells these are kind of ordered by uh you know the anterior end and the     
6:13     
posterior end but you see this connectivity in the middle all these lines and those lines could represent     
6:20     
connections from uh you know the anterior to the posterior side or the anterior to     
6:26     
another cell in the anterior portion of the embryo and so forth so you can see     
6:33     
that in this case you know you have uh some con you know some patterns of     
6:38     
connectivity you actually have to run graph statistics Network statistics to     
6:43     
really kind of get at some of the properties of these networks uh the other problem of course     
6:49     
then is that those descriptive parameters oftentimes don't describe very much about the network they just     
6:55     
describe the global parameters so for example you know there their different uh parameters like     
7:01     
centrality and other types of parameters that uh you know give you basic     
7:07     
information about the network as as a global entity but not local entities and     
7:14     
so there are a number of network uh approaches like Community finding and things like that which we actually do I     
7:19     
think later on this paper look at for communities in these networks uh but uh     
7:26     
that's that's a those are other exercises you can do and it helps you describe the network a     
7:31     
little bit more in detail in terms of the local uh components of these     
7:37     
networks so you can see that this kind of connectivity is different than this one you use different uh connectivity     
7:44     
thresholds so you take uh two cells pairwise and you use your Criterion and     
7:50     
you set a threshold for connectivity so you can have a very permissive threshold     
7:55     
for example and it can be you know uh you can have more connectivity you can     
8:02     
have a more restrictive threshold and you have less connectivity and so as you drop that connectiv or you you uh bring     
8:09     
that connectivity threshold up you start to get these more these sparer networks     
8:14     
but you also start to reveal some uh structure in the network so you see here     
8:21     
this network here has basically this is uh uh kind of it's It's not a random     
8:27     
network uh I've done test with random networks and those are different than     
8:34     
this uh so you know there are these uh types of network topologies that are     
8:40     
non-random uh there scale-free topologies small world topologies this is more of a scale-free topology it's     
8:46     
not random but this one is more uh more scale free than this one I guess is the     
8:52     
lesson and and you end up with these clusters which reveal uh patterns within     
8:57     
like locally in the embryo so you can see that there are certain parts of the     
9:04     
uh this is actually where you get uh down the lineage tree a bit and you     
9:10     
start to get places uh in the P1 uh hemisphere the posterior end of the worm     
9:18     
embryo that you know have these sort of clusters of cells so that's that's what we were     
9:25     
doing in this paper and you know they're different you know tests you can do on this you can get degree Network diameter     
9:34     
graph density modularity you can actually look at the number of     
9:39     
communities uh and then the clustering coefficients so those are all globally descriptive     
9:45     
parameters um so yeah this is and then you know you can look at that Global     
9:51     
Network and you can say things about the global structure and you can get these summary statistics but they don't really     
9:58     
tell you much much about the you know the network structure other than just     
10:04     
this snapshot Global description so you can look at these n uh Network Statistics over time from the 16 cell to     
10:11     
the 128 cell and you can get some sort of you know maybe like a Time series     
10:17     
plot over the number of as the number of cells increases their division and their     
10:23     
differentiation events going on this uh structure of the network changes these     
10:29     
statistics will change as well so the degree will change the network diameter will change you can see that with as you     
10:35     
increase the threshold you get these large gains in uh Network diameter so as     
10:42     
the uh threshold increases and you get this more restrictive Criterion you saw     
10:48     
in let me zoom in you saw in the figures sort of     
10:54     
intuitively that you start to get these clusters and that represents a larger Network diameter so that means is that     
11:01     
if I started any one edge of the embryo and I want to go to another edge of the embryo it takes more hops across     
11:07     
different cells to get to the other side and so as you increase the threshold in     
11:13     
this case you know as it as it goes up and you're more being more restrictive     
11:19     
then you get a larger Network diameter there's this big jump at certain points uh you can see that in the 16 uh cell     
11:27     
embryo for example the network d Dieter doesn't exceed eight and then that would be obvious because they're fewer cells     
11:34     
but when you get to 64 or 128 you know when you get to the like 0.95 threshold you really start to get     
11:42     
large Network diameters and so these are good you know descriptive statistics but     
11:48     
they don't really get at uh the sort of the properties of the lineage tree directly or some of the local     
11:56     
features and so you know there are some other experiments in here like the cleck analysis which is where you know you     
12:02     
look for areas that are densely interconnected so you know a lot of     
12:08     
people look for clicks and you know they'll do other things like Community finding to find these local     
12:15     
substructured groups in the network so this is a click analysis you know for     
12:20     
like I think the 128 cell embryo uh and you know it kind of tells     
12:26     
you a little bit about you know these sublineages in the lineage tree so when     
12:32     
we have our ab and our P1 which is the anterior and the posterior end you start     
12:38     
to get these sublineages which are based on divisions of ab divisions of P1 and     
12:44     
you end up with eight founder cells what they call founder cells and those are eight sublineages that continue to the     
12:53     
hatch and the post embrionic period what we can do is we can take those sub     
12:58     
Networks works and then analyze them and see if they match ceks and you know that's what this was     
13:05     
supposed to be so this this paper was in 2018 and I bring it up because it kind     
13:12     
of motivates what I'm doing in this other paper so I've been working on this idea     
13:18     
of hypergraphs uh for a while and the idea behind hypergraphs for this type of     
13:24     
network is uh kind of to take uh different you     
13:30     
know take the lineage tree and represent the cells in these sublineages as not     
13:36     
only the individual cells and their connectivity but larger categories that are connected not just in space like we     
13:43     
did at this network but also in time so if you go to the YouTube page we have a     
13:51     
a p a channel a playlist they call it for Network science and so we have a lot     
13:57     
of interesting things on the network Science Channel or the playlist uh we have a couple of     
14:04     
presentations that have been given at uh networks the networks conference     
14:09     
the NSI conference uh a couple of other presentations but these are the two that     
14:16     
I wanted to draw attention to for this recent work and that is most relevant to this recent work so this is the first     
14:24     
one hypergraphs demonstrated asmosis during Divergent integration     
14:30     
and embodied hyper networks graphs of spatial anatomical embeddings and so     
14:35     
these are why kind of you know kind of go over this work but you know in different uh in different ways and     
14:43     
there's also this other video embryo networks computational development and neuros simulation this one includes some     
14:50     
of this work as well but it's not um you know it's it's about more broadly what     
14:56     
the network vision is uh you know that includes all this kind of work     
15:03     
so this is the pre-print uh so this is hypergraphs demonstrated an asmosis during Divergent     
15:11     
integration so an anastomosis is like a a pathway or a     
15:16     
place where two things are joined together and that's an anatomical term and I think dick mentioned it once in a     
15:22     
meeting and I used it in the title to kind of get at like what we're you know     
15:27     
what some of these graph structures are actually doing in terms of describing the biology so it's like you have the     
15:34     
spatial network but you also have this temporal component you have the lineage     
15:39     
tree but you also have the anatomical spatial relationship that is preserved     
15:46     
and then you can describe things like an asmosis or other types of division     
15:51     
events you'll see what it looks like so this paper kind of goes through it's     
15:57     
very theoretical in the sense that it doesn't use uh like it doesn't use data directly     
16:06     
it does a lot of you know hypothetical modeling of some of these things but then there also there's also some data     
16:12     
in here so uh they're two so basically developed uh hyper graphs for     
16:20     
two uh or modeled on two organisms so it wasn't actually taking the data set from     
16:26     
the organisms but it's modeling it on on sort of uh an Exemplar organism and then     
16:32     
building a model from that so the first one of course was celegans and this is something called a twoy tree and then     
16:40     
the nosto punk uh punct fori which is the uh cyanobacteria that we talked     
16:46     
about in the group this is constitutes multiple discontinuous two arey trees so     
16:52     
the idea is that these hypergraph models models model development and the lineage     
16:58     
trees is an niry tree which can model two or more descending categories per division event so if you have uh if you     
17:06     
think about a lineage tree is a binary tree that binary tree then can be you     
17:12     
know you can represent the tree and each node has things in it so let me draw this on the on a board so you can see     
17:20     
what I'm talking about this works better if I do it this     
17:25     
way so you have a a binary     
17:30     
tree that has a root that has nodes coming     
17:36     
down and two area just means like it has two descendants so the term n area is     
17:43     
just how many descendant does it have and so this is a term from mathematics     
17:49     
but basically it's where you know you have a single cell divides into two     
17:54     
cells but that's not all we're interested in of course we're interested in the relationship The ancestral     
18:02     
descendant relationship so it's ordered and then if we open up one of     
18:08     
these uh nodes we see it's not just a node it's a     
18:13     
category and inside the category there are things inside of it and in fact     
18:18     
there's a network inside of it if we would like we could have a network in there things that are connected or they     
18:25     
don't have to be connected the point is is that each one of these nodes you have things so you could actually model this     
18:31     
as single cells you could model this as categories of sub uh lineages whatever     
18:39     
there's their constituents within each of these nodes they could be cells they     
18:44     
could be molecular events they could be uh things in inside the cell itself so     
18:51     
there are a lot of properties that you can model within each of these nodes so it's a hyper hyper graph so it means     
18:57     
that there there's a graph here there's a graph structure but there's also this structure that exists within each node     
19:05     
and those are connected so a lot of times with Hyper graphs you'll have like a graph within a graph and you'll have     
19:12     
the the you'll have nodes within a single node and it passes things on to a     
19:20     
descendant so we can actually take the constituents of this node here and move it you know pass it down     
19:27     
to this nodee here there's some of inheritance Rule now not everything in this node is going to be inherited by     
19:34     
this node we can have division events so we can have a doubling of the things inside this node we can have things you     
19:42     
know we can lose constituents we can you know do whatever now in the case what I'm going     
19:50     
to show you it is it just uh you know a direct uh what they call a dag or     
19:56     
directed A CYCC graph so a dag is a direct directed aycc graph and that     
20:02     
represents a lineage tree uh often times they'll use this terminology just to say     
20:07     
that everything goes from the root downward and that there are no Crossings     
20:14     
or intersections between nodes that kind of divide out so you're always going     
20:19     
outward in branching uh you know doing subsequent branching phases but you     
20:25     
never actually join back together there's no Ram ification but in this case we're going     
20:30     
to ramify things and we're going to do that by having go to the next slide here we're     
20:39     
going to do that by having like different categories that come down like     
20:44     
this and then we'll kind of I want to do this kind of just to show you'll have Crossings into     
20:51     
different categories so if you think about this as like a stem cell this is a neuronal lineage     
20:59     
AG this is like a I don't know     
21:05     
muscle the muscle cells will descend from this category to this category you     
21:11     
might have a doubling event and the neurons you might have a doubling event     
21:17     
and you know the idea is that you have roughly twice as many muscle cells in this node as you do in this node and     
21:24     
twice as many neurons in this node that you do in this node so you see you have     
21:29     
this division event or this maybe this differentiation event here and you have this proliferation     
21:36     
event here which is division and then you know that's one round of cell division but it doesn't mean that all     
21:43     
the cells maintain their identity so one of the reasons why we want to use hypergraphs is to uh you know kind of     
21:51     
allow for cells to switch maybe from like you know a neural precursor cell to     
21:58     
a muscle cell or muscle precursor to a neuron cell or what's more likely in     
22:03     
some of these graphs is that you have some early you know some cell that's like basically a developmental cell or a     
22:10     
stem cell that switches uh into a new fate so you have like you know cells     
22:16     
that this this node the stem cell node May double so you might have something like     
22:23     
this where you have a stem cell category and then you have stem times     
22:30     
two and then you have like     
22:36     
muscle and then muscle times     
22:43     
two and then this is like you know uh stem times [Music]     
22:50     
4 and then you'll have like maybe something from this category move down into the muscle category and so when we     
22:57     
sum up all the cells within this node it will be all of the muscle cells here so     
23:02     
say we have 16 this will be 40 we take the 32 we take this 16 we double it and     
23:09     
then we have eight cells that migrate from the stem category into the muscle category and then this stem category     
23:16     
will say be uh here eight 16 and then     
23:21     
this is uh not 32 it's a subtraction of that so would be 20     
23:28     
four so this would actually not result in a doubling uh or well it would this would     
23:35     
be times two I'm sorry so it now result in a doubling but it would be um you     
23:41     
know doubling minus eight because eight cells have moved into this category so that's the way these hyper graphs are     
23:47     
set up they're set up to allow for these kind of changes in category and that's     
23:53     
the way that we kind of represent changes in in differentiation     
23:58     
and changes in fate or changes in the cell State and so you can actually go     
24:03     
down be beneath or below sort of uh cells within cells you can have uh     
24:10     
cells uh like the nodes can represent cells and then inside it could represent molecular components so it would be a     
24:16     
little bit different setup there the molecular components maybe like you know transcript counts or something like that     
24:23     
would change uh in a similar way but it would be a little bit different setup but it allows you it's very flexible and     
24:30     
allows you to represent these different events and things like that so one of the concepts that I I     
24:36     
kind of started out with and this was during uh n the networks conference in     
24:42     
2021 I gave a talk on Divergent integration and Divergent integration is     
24:47     
the idea that you take an embryo that's differentiated and it starts to develop     
24:53     
you know you start to have these differentiation events and new tissues and those tissues diverge in terms of     
24:59     
their connectivity but they retain these integral sort of connections so for     
25:05     
example if you have like uh in a seigan embryo you know at some point you begin     
25:11     
to have a ner a nervous system or a connectome and so in     
25:16     
the in the connectome you have like it looks like you know has you're kind of     
25:22     
in this stage of the embryo still you start to get neur neuronal cells up here     
25:27     
here some neuronal cells down here and then you get the rest of the cells in the background so they can be of     
25:33     
different Fates but they're kind of disconnected functionally from these neuro precursor     
25:39     
cells as they start to differentiate into you know a ner an into a connectone     
25:45     
and so these connectone cells are connected in in different ways um and     
25:50     
you know they're distinct from the rest of the cells in the embryo so they start to diverge in terms of their function     
25:57     
and there's you know and and what they're kind of related to so they start to Cluster together but they still have     
26:02     
these connections to other cells you know the the uh connecto and the head     
26:07     
and the connecto and the tail are are weakly connected they're going to be St more strongly connected later in     
26:13     
development but you also have these other cells in the embryo that are still     
26:19     
connected to the connecto obviously because they're going to form this unified organism but they're going to     
26:25     
have weaker connections than the cells its own category which might be muscle     
26:30     
cells or other types of cells okay so back to this paper so     
26:37     
that's Divergent integration and it's going to play a role in these networks it's going to play a role in sort of     
26:43     
this ramification in the network so I'm going to get right to an example of a hyper Network here um or     
26:52     
hypergraph so this is the category so I've used category Theory to represent the different categories ories you can     
26:58     
have two categories here that come from an initial category you have X and Y and     
27:04     
you can have an interchanges between X and Y you can have interchanges between X and Y in the Next Generation and so     
27:12     
forth uh this is an example of a hypergraph now this is modeled on not on     
27:18     
C Elegance proper but on sort of an idealized form of C elegans so this is     
27:23     
the hypergraph uh uh model here and so you can see that starts with the one     
27:29     
cell and you start to get these doubling events throughout the lineage tree and it's doesn't look like a conventional     
27:35     
lineage tree because it's tracking these categories and so this gray category is     
27:42     
uh what we call developmental cells these are like the stem cells that you see in seans that are     
27:49     
undifferentiated you start to get the germ line that diverges early so that's     
27:54     
Divergence The Germ line is less integrated than the other types of cells will be but it's still part of that     
27:59     
organism and so it's still part of this tree you're not going to see any ramifications from The Germ line and so     
28:06     
you know that's that's something to keep in mind when we see some of these other categories so we have the germ line     
28:12     
diverge early then we have our start to get some other things that diverge we     
28:18     
have this connecto and these White net nodes that have you know you get these a     
28:23     
small number of neurons that initially diverge and then you get more of these Embry uh like these these developmental     
28:30     
cells that form this embryo Network that start to sort of add to the connectone     
28:36     
as cell as cells and the developmental cells differentiate into connectome     
28:42     
cells they move from one category to another so now they're part of this uh Network within a node and so they're     
28:49     
forming this connecto you start to get in time you get these categories that     
28:55     
grow in size and they exch change with other categories in time and other     
29:00     
categories in function so you can see that there it gets pretty messy towards the top of this where you have arrows     
29:07     
going in all directions but that's just simply because you have more categories where things are being interchanged and     
29:13     
things are you know moving forward in time so that's a very different representation of the lineage tree than     
29:20     
what we may be see in the uh solsten lineage tree for example and this is not     
29:26     
cant but it's a model based on celegans and the reason I did that was because I didn't want to have to uh you know I     
29:33     
just wanted to kind of show a a theoretical model where it's very clean     
29:38     
um so this you know you can see for example you can Define things like the eight cell embryo or the 32 cell embryo     
29:45     
in this case in the 32 cell embryo you have 28 developmental cells and four germline cells in the E cell emry you     
29:52     
have seven uh uh like the founder cells the developmental cells and then the one     
29:58     
germline cell or germline founder cell and so you can go up and you start to get like where you know you have to     
30:06     
Define like different stages of the embryo and you have more and more categories to consider so it gets a     
30:12     
little Messier as you go move upward in time so that's uh for C     
30:19     
Elegance uh now I go through a little bit more about this about describing the     
30:25     
hypergraph spatial Hy hper graphs and so this is a spatial hyper graph so this is     
30:31     
actually a little bit different formulation this is where developmental time is now going downward uh reasons     
30:38     
that we want to be able to show these things kind of unfolding in time so uh     
30:45     
this this is the 128 cell embryo and this is the 256 cell embryo again it's     
30:52     
not seans it's an analog of seans but you know this basically shows how these     
30:59     
kind of categories that we saw in the last figure translate into spatial uh     
31:04     
distributions so this is the anterior posterior axis but the top and this is     
31:10     
the left right Axis here and then we have our 128 cell uh diagram here and     
31:15     
the 256 down here so we have these connections between the the you know the     
31:21     
doubling event so we go from our large category developmental cells at the 100     
31:27     
28 cell stage and we move down to the 256 cell stage where we retained some of     
31:34     
our developmental cells we also get more get different categories of germ cell we     
31:42     
get uh these other differentiated tissues and so we get it's the 256 cell     
31:48     
embryo in this diagram is much more diverse in terms of the categories so we     
31:53     
have different categories that are smaller or larger than what we see up here so these you know these there's an     
31:59     
accounting here where you say basically if there's a doubling event you have doubling but you also have contributions     
32:07     
to new categories contributions to different categories and the like and So     
32:12     
within each of these nodes you get a a network and that Network can be different cells that are interacting and     
32:19     
so you can structure the network in that way but you can also have a representation where you have these     
32:25     
categories that are also interacting in a network and this can be done spatially     
32:31     
as well as just simply temporally in terms of that lineage tree     
32:37     
okay um this is actually a different example this is the     
32:43     
cyanobacteria nosto p uh pfor punct fory and so this is where you have a bunch of     
32:50     
cells in a chain and this was another paper we discussed in the group where you know they have this they basically     
32:57     
have this very uh defined set of rules for morphogenesis where you have cells     
33:04     
in the middle that divide and you know they have different factors in them and they form different functional     
33:10     
categories so here you go in time you have this chain that's growing and so     
33:15     
they there's this phenomenon called restricted heteros cell differentiation where these you know     
33:21     
these cells will this chain will grow by cells in the middle of the chain to     
33:27     
dividing and increasing the length of the chain so here we have one cell two     
33:32     
cells four cells and eight cell categories so as this cell starts to divide it starts to you know uh     
33:40     
contribute to this category and so you have these networks that are discontinuous you have these     
33:46     
discontinuous trees and so they kind of go through the chain and you can visualize how this chain grows using     
33:53     
this kind of an approach you have the hypergraph the cells with within that hypergraph and the you know the cells     
33:59     
within there they have molecular components and you can model those as a network so that's you know why you might     
34:06     
want to do something like this it isn't in the same way that we see in the seans example but I think it provides a     
34:12     
different perspective on this this kind of methodology um so it's basically the     
34:18     
same type of thing it's just a little bit different topology these are uh some graphs to show like the sort of the     
34:25     
expected output that we might expect from one of these hypergraphs so if we have a number of cells or a number of     
34:31     
molecular components within the hyper nodes those hyper noes will have a distribution so you know all the     
34:38     
components of those hyper nodes you can take a spectrum and read it out so each     
34:44     
uh hyper node which were these colored nodes will have some sort of output it's     
34:49     
not just the network that interacts and changes its shape and its uh size over     
34:55     
time it actually can output different types of uh things that are in it and so     
35:01     
you can get a spectrum of cell types you can get a spectrum of uh you know cell     
35:08     
types are different uh in different conditions and then you can you know so     
35:13     
this is a a spectrum where you have different types of cells a very small     
35:19     
Spectrum but you can get all sorts of Spectra that have different distributions and this is counted in the     
35:25     
number of cells so it's basically a histogram that tells you some it's a description of that Network or that wiow     
35:32     
sub Network and so this is another example     
35:38     
where you have transcriptional Regulators of protein types you have different types of proteins and this is     
35:44     
more of a spectrum where this is based on some pseudo data that was generated so this is more of the distributional     
35:50     
character of it where you get these uh uneven distributions of different     
35:55     
protein counts in one of those hyper nodes and so it's descriptive of the     
36:01     
structure of the network but it's also descriptive of what's going on inside the hyper node what's the     
36:06     
process and so finally you can actually do this for like a uh like an a     
36:12     
connectome where you have this output Behavior so you can have these connectomes that link from the sensor to     
36:19     
the affector and you you as you have like your sensor which are your sensory organs and your factors which are like     
36:26     
your hands or your legs uh you can think of like an insect and this is a good     
36:31     
example because in insects you have a very simple a relatively simple mapping between the sensory organs and the     
36:39     
affectors and you think about the things in between the connectome and that     
36:44     
connecto will grow in a way that's similar to what we're trying to do here so we have uh in in development we have     
36:52     
this uh connectum that grows in size in terms of the cells in it different categories of cell and so you     
36:59     
can actually account for this uh you know this sort of Developmental growth     
37:05     
in in the connectome that's embodied in an organism uh in this way as well so I     
37:11     
showed a spatial hypergraph I showed a temporal hypergraph which was analogous     
37:17     
to a lineage tree and then now this which is embodied in an organism going     
37:23     
from like the anterior to the posterior end or the sensor to the affector and this just models the nervous system in     
37:31     
terms of its detail and so you know the different Cate categories of cell the     
37:37     
different numbers of cell that are there and so forth and so that's yeah there are a lot     
37:43     
of nice uh references in here on hypergraphs on systems of biology on     
37:50     
other types of things a lot of some category theory in here as well um and     
37:56     
yeah it's it's really you know this is something I've been working on for a couple years so     
38:02     
um that's all I have on that um if you want to go to the YouTube channel I have     
38:08     
some talks where they're kind of more condensed and it kind of gives some examples so we have some oh yeah thanks     
38:16     
so we have some questions in the chat here uh Morgan I guess a long time ago     
38:22     
said I've been in touch with the principles behind the recent rsaf     
38:28     
well go ahead sorry this was before You' even started oh     
38:34     
yeah but uh super super interesting and I I I just dropped a link in to a paper     
38:42     
a very Network science paper on the dropa uh     
38:48     
connecto that that's you know again potentially of interest just in terms of     
38:54     
I'm uh uh yeah you know C curious about the latest     
39:00     
in network science to you know identify circuits um and yeah like even     
39:09     
and then going one step past that like you know potential logic     
39:15     
circuits um um and you know like not     
39:21     
sure how much this is relevant or how much of this is is um embryogen     
39:27     
you know because this this certainly this particular Network science um paper     
39:32     
is is focusing on an adult brain right so um you know definitely differen     
39:40     
is there in terms of of looking at development on grass um and     
39:47     
um one last thing it's if I haven't already I think I've posted somewhere     
39:53     
but um the Greta um the Greta seminars um this is like the you know it's graph     
40:02     
oh I can't remember now the acronym but it's basically trans transforms Transformations on     
40:08     
graphs uh seminars and you know like again like     
40:14     
how that could be useful um not only in terms of how networks um can have     
40:23     
multifunctionality but also just embryo like you know how how networks are     
40:29     
changing over time as as the as an organism grows but um yeah I i' I'd     
40:36     
mention this before you got started just because U the um the Cog laab in Paris     
40:44     
uh has some people interested in in this dropa uh connectum Flywire that is you     
40:52     
know kind of crowdsourced crowdsourced it's all academic la     
41:00     
but this um multicenter uh repository     
41:08     
Flywire that is got um you know something like     
41:14     
120 120 uh thousand um proof red     
41:21     
neurons at this point something like that um and and you know what they've     
41:29     
been doing with it in terms of um some some pipelines they've got using     
41:37     
you know imagej fij to um like skeletonize that and then turn turn it     
41:44     
into basically more like connectivity weight matrices and then use like spiking girl     
41:51     
Network simulators like Brian 2 in this particular case Berkeley to do um you     
41:58     
know to then look at uh um at least some aspects of of neuronal modeling with it     
42:07     
right yeah and uh you know so that don't     
42:12     
want to get you you know like don't want to um get off track the presentation but     
42:17     
that that was that like basically all my comments are around that oh yeah yeah     
42:23     
that's that's great uh yeah the bioarchive paper here is that the paper     
42:28     
you're were talking about yeah that that's just the one that that was very much like you know     
42:34     
Network um um Network statistics I think it was called yeah yeah Network     
42:39     
statistics right and um yeah you know so I you know more     
42:47     
summary measures but um but uh uh again     
42:52     
like how certainly a lot of a lot of Network science is focused on you know Finding     
42:59     
clicks finding finding sub networks right yeah and and um uh some of the uh     
43:09     
I've been there's actually a lot of so talking with um Shu and uh well the F     
43:17     
first author of The brine 2 simulation um there's a lot of great     
43:24     
videos from from the Flywire Princeton group that um just kind of     
43:29     
present you know what's there as well as the tools that they're using this     
43:34     
Flywire brain um sorry F fly brain lab um so the     
43:42     
from the fruit fly Observatory this is all new to me so for     
43:48     
me if this is this is old hat but um uh um some some really nice resources and     
43:55     
what's great is just that that like it's a a large complex     
44:00     
Network that that's kind of been curated and you can do stuff with um and you     
44:07     
know I'd love to I'd love to learn more about how um C elegans and openworm is     
44:15     
you know matches that or is doing things you know similar to that yeah yeah me     
44:23     
you know we we haven't really gotten too much into neural modeling for them but you know I think the the idea you know     
44:30     
is to kind of come up with different representations for you know networks so     
44:36     
I was looking at this paper here the network statistics and you know they're doing like a lot of different statistics     
44:42     
but so they're trying to find these two and three Network motifs it's like kind of yeah if you look at like a two net     
44:49     
nodes that are connected or like a triangle which basically a three node Network Motif will form a triangle that     
44:56     
will give you some information about like local things but this other thing which they talk about a population of     
45:03     
Highly connected neurons known as rich rich clubs those are those have been well characterized in seans and uh     
45:11     
getting into like what the development of those Rich clubs looks like that's that's a really interesting     
45:17     
question um you know it's not something that we we've done or I don't know I we could do it but maybe with simulations     
45:24     
like why do you have Club neurons what's the point or what what does it actually     
45:30     
do for the network yeah yeah yeah     
45:37     
and now go ahead okay so yeah then dick said please define hypergraph which was     
45:44     
um and then actually Morgan has a description of hypergraph which is hypergraph is a generalization of a     
45:51     
graph in which an edge can join any number of vertices so yeah basically     
45:57     
what happens in a hyper graph and I didn't go over this uh formally but I'll     
46:03     
I'll do this um so like I'll compare it to a regular graph and then I'll show     
46:08     
what a hyper graph looks like so we have in a regular graph we'll have um like a     
46:16     
node here single node and then we'll have maybe a connection to another node     
46:21     
and a connection to another node and a connection to another node this is one of these triangles that I mentioned     
46:27     
where you have three nodes that connect singularly to its n nearest neighbor and     
46:33     
you have this kind of a motif now in a hyper graph you have like     
46:39     
a node here what they call hyper node and Hyper node just means that it has things within it so you might have this     
46:47     
Motif within a hyper node this would be a single node but there'd be three nodes     
46:53     
within it so now when you get move to another hyper node and you have the same Motif and     
47:01     
let's say in this case we're not thinking about like desc you know ancestor descendant relationships we're     
47:08     
just thinking about this kind of connectivity here so here in this with     
47:13     
this Edge you have two nodes that connect it each node has a single single     
47:19     
thing so there's an exchange of things between these two nodes A and B and then     
47:25     
these no here are A and B but you also have a prime B Prime and C Prime and so     
47:33     
the idea here is that you can transfer things between nodes um these connections like a prime     
47:42     
will be connected to a prime prime down here B and it can also be connected to B     
47:49     
prime prime down here so you have these connections across uh different networks     
47:54     
so it would be like if you had another network over here which is like a prime B Prime C     
48:01     
Prime and you have not only these nodes but then connections between these     
48:07     
nodes so you have a sort of second layer     
48:13     
Network and it's like you know you're connecting not only within your own cque or this three this triangle but you're     
48:20     
connecting with another triangle and those those connections are isomorphic meaning that they can be connected in     
48:26     
different ways so you can connect from a to a prime B to a prime and so forth so     
48:34     
it's almost like you have this these two networks that are sort of in parallel     
48:39     
and they're connecting together so like basically in this case you have one node     
48:45     
connecting to another node in the hyper Network's case you have one node connecting to another node but there are     
48:51     
many connections that move along this route and they're connecting to     
48:57     
different things so the connectivity isn't just from like one node to another     
49:02     
it's to various constituents within the node and so that's the difference     
49:07     
between the two things um you you can have connectivity that's very uh very     
49:14     
complex even just having two nodes there that are connected and so it's it's it's     
49:19     
kind of hard to get your mind around because it's like well you know I just thought that they were two nodes but you     
49:25     
know you're nesting this stuff and then you're saying well any one of these no uh constituents of node a can connect     
49:33     
any other constituent of node B and that creates a very complex system of     
49:39     
accounting to see which goes which thing goes to what and and it also allows for     
49:46     
a lot of these kind of processes in biology where you have you know things nested inside of other things to you     
49:52     
know be represented in a in a way that can really kind of maybe understand and     
49:58     
so that's that's what a hyper graph is compared to a regular graph so any Edge can join in any number of vertices and     
50:06     
so in an ordinary graph an edge connects to exactly two vertices so that's the     
50:11     
difference there and then yeah check out the YouTube playlist and so that's     
50:18     
that's all I wanted to mention about uh that paper and some of that     
50:23     
work um any questions at this     
50:31     
point so yeah okay uh well yeah so let me know if     
50:39     
you have any questions or comments um so I last week was the     
50:46     
learning on graphs conference which is a conference we we did last year you know     
50:51     
we had never we never got around to submitting a paper to it we were working working on the graph neural network     
50:57     
stuff uh last summer and this summer and learning on graphs is basically a graph     
51:02     
neural networks conference uh but they also kind of venture into this world of     
51:07     
uh graph Theory and networks and so we weren't able to submit a paper this year     
51:14     
last last year I wrote a blog post on the event and I used it for this year's     
51:21     
gck project um this year I was able to kind of catch up on sort of the broad     
51:28     
outlines of learning on graphs but I haven't delved into any of it and I want to do a recap of that conference and say     
51:36     
you know say something about what's going on there the major themes but I haven't had a chance to do it so that's     
51:43     
that's something I need to do um I yeah I mean it was like I have all the     
51:48     
materials bookmarked and I'm ready to go but I didn't get a chance to do it so we'll see I'll try to give a recap on it     
51:56     
maybe I'll do you know a video or maybe I'll do something like that too um but     
52:03     
it was a really interesting conference from what I saw a lot of things especially with respect to topological     
52:11     
data analysis and there was some work on asymmetries in networks in this case     
52:17     
neural networks that you can use to process data so graph neural networks     
52:22     
are basically taking features out of you know your your data uh finding the     
52:29     
connections between them building these graph embeddings and then you know using those to     
52:36     
process you know what what you what you've analyzed you know use it as your model and some people were using they've     
52:43     
been using a host of very Innovative techniques to try to get at some of try to improve performance one of those     
52:50     
being asymmetries so where the network that you're using has some sort of of     
52:56     
asymmetrical properties of information processing um you know people are using     
53:01     
topological techniques yeah to represent geometric graph neural networks to     
53:06     
represent the data input things like that so it's really interesting stuff     
53:12     
it's you know something we can kind of get into I'll find some good examples maybe and bring them to the group as a     
53:19     
whole so I don't want to like overwhelm people with because it's like there's so much there now     
53:25     
was like several days of talks so you know that's kind of like okay any other     
53:31     
questions um     
53:38     
comments just on the the full week of those talks     
53:46     
yeah H so yeah Morgan put a couple things in the chat thisa     
53:53     
package well that's the seminar oh this is okay yeah and um you know so my my     
54:03     
scroll through there at some point and just have a look see if anything's anything's interesting um uh the um and     
54:12     
then the icts um like U was it winter school on     
54:19     
quantitative uh systems biology has has a number of interesting talks um yeah I     
54:26     
think I I posted one which was like the energetics of embryogenesis or     
54:32     
embryology okay um yeah yeah this is bioenergetics yeah yeah yeah um but     
54:42     
um do they have the list if you go     
54:47     
down the schedule they have speakers and topics     
54:52     
here okay yeah they they they um yeah     
54:58     
um and great great speakers yeah yeah for for covering covering this kind of     
55:04     
material oh yeah cell cell metabolis cell metabolism is an economy um and then self-organization of     
55:12     
metabolic networks um yeah lots of lots of     
55:18     
interesting stuff yeah yes I mean a measurement uh uh     
55:25     
calorimetry that's yeah interesting stuff I mean we did a paper last year where I read through a paper where we     
55:31     
talked about like people were doing this in embryos where they were measuring like the energetics of an embryo and the     
55:39     
embryogenic process basically looking at the I think they were looking at metabolomics as well but uh yeah it's     
55:46     
interesting people have done this but not it's not in the mainstream really of yeah yeah yeah but the other the other     
55:53     
thing I just wanted to say was that um uh because I didn't didn't I forgot this at the Saturday meeting but um there's a     
56:01     
new um biolab space in San Francisco that's opened up okay like a community     
56:07     
community biolab and um uh anyway it was it was uh someone who's     
56:15     
done this before in other cities you know so he's got some experience um     
56:21     
hopefully he's got you know enough connections to you know bring a bring a     
56:26     
community around to you know keep keep the funding going but um they're already getting some support from the city that     
56:33     
uh is nice to see um because they took over a I guess a reclaim you know deric     
56:41     
building um but uh uh I was wondering whether you know like like they're gonna     
56:48     
have some some cold rooms and things like that I I didn't you know I was just like how about cell culture     
56:57     
and uh and you know I I knew we had we had some shared uh connections between     
57:05     
um the hacker space noisebridge and and this space so i' um I pointed them to     
57:11     
some some particular like I thought easy projects um again not sure whether     
57:18     
they're gonna have the facilities but um but I was thinking like like hopefully they would get a microscope or at least     
57:24     
they'd have a a space for a good microscope and I was wondering about um     
57:30     
at least uh doing some um what's the um the geometric     
57:36     
structures that form     
57:41     
um oh I forget the uh the name of those dick what's the um the     
57:48     
uh it's the single celled organism that makes a a     
57:53     
silicon shell oh di yeah Dion yeah yeah yeah sorry sorry yeah yeah I you know     
58:01     
like um again I wasn't sure you know if there's anything of interest there but     
58:07     
um uh um you know I I certainly would love to to have a good space for a a     
58:14     
microscope to set up and and you know not be disturbed and uh or at least you     
58:20     
know have some have some uh have some functionality and some     
58:26     
some video recording you know I'm like always jealous watching these Journey to     
58:31     
the microcosmos videos oh yeah yeah by by the way there's a     
58:40     
there's a technique which needs to be developed before Dions that shell is made of silic     
58:48     
yeah and uh it's hard to look through because it's refractive index     
58:54     
from water is quite different so the idea is to add something to water which     
59:00     
is non-toxic and matches the refractive     
59:05     
index of the I see I see I call it cloaking of     
59:12     
diom okay okay that that that's cool that's cool I mean uh as a hint I     
59:19     
suspect the material that one could use is methyl cellulose okay okay     
59:25     
okay um I'm GNA follow up with you on that because uh I I used to have a um     
59:31     
grad students working you know doing doing small jobs at the lab     
59:38     
and um he was in the uh Optical like photonics group at Berkeley and they     
59:46     
were doing this kind of like negative refractive index stuff you know this is just matching no no no I I     
59:54     
but he would know he would know good materials and he would know how to test it and things like that like I I yeah he     
1:00:02     
would be able to point me to people at Berkeley it might be able to help refractive index is around 1.4 for the     
1:00:08     
so it's not I don't think it's GNA be that difficult to match yeah yeah yeah yeah you don't you don't need full full     
1:00:14     
klling on cloaking yeah yeah yeah the point is that there there is almost no     
1:00:20     
work uh which shows what's going on inside of Dion interesting     
1:00:26     
and okay the uh Jeremy picket heaps you should look     
1:00:33     
at his movie okay he he doesn't do cloaking but he's he's done some     
1:00:39     
spectacular Imaging okay the uh the whole video that he produced is online     
1:00:47     
now uh with the Russian Journal okay okay uh if I can look it up for you you     
1:00:54     
want I'll I'll follow up with you I've gotta I gotta get in the car to take this girl's school okay but I I'll     
1:01:01     
follow up with you dick thank you so much all right okay sure thanks Morgan put my     
1:01:08     
uh uh my email in the oh I've got it I've got it you got it oh okay yeah yeah     
1:01:15     
yeah thank you all right     
1:01:22     
yeah all right uh so yeah that's it for today I think     
1:01:28     
okay all right thanks take care take care bye bye all right now I'd like to     
1:01:33     
talk about a few papers and a category I'm calling new directions and     
1:01:39     
morphogens and so of course morphogens are these theoretical constructs that we often talk about when     
1:01:46     
we talk about morphogenesis and the theory of morphogenesis so let's get started so     
1:01:53     
this first paper is is on this is by Amy Shire and Alan     
1:01:58     
Rodriguez and co-authors and this is a research article summary of a paper we'll get into after I go through the     
1:02:05     
graphical abstract and some of the uh details outlining this I'm going to read the introduction and then go through     
1:02:11     
some of the figure during vertebrate organ morphogenesis large collectives of cells     
1:02:18     
robustly self-organized to F architectural units which are bone VII     
1:02:23     
or fol these forms persist into adulthood so in the hypergraphs presentation I gave     
1:02:30     
earlier you know we had these categories of of cell types and those persist as     
1:02:36     
you know they grow and change but they persist as categories from some point in development to     
1:02:42     
adulthood over the past few decades mechanisms of organ morphogenesis have been developed predominantly through     
1:02:48     
molecular genetic and cellular Frameworks so this is where people look for genes that are expressed and people     
1:02:55     
look at single genes uh in in cells and they characterize this in different ways     
1:03:02     
but they have not linked it very effectively to morphogenesis itself so     
1:03:08     
this is what a lot of the Empirical research has been about it you know but     
1:03:13     
of course we have all this theory that we need to link to more recently there has been a     
1:03:18     
Resurgence of interest in Collective cell and tissue mechanics during organ formation so this is where we have this     
1:03:25     
uh this literature I think we've been reviewing it over the past several months where you have mechanics and     
1:03:31     
people are looking at different mechanical aspects of tissues but again we're not linking all this together     
1:03:37     
theoretically uh this approach has Amplified the need to clarify unambiguously linked events across     
1:03:43     
biological length scales so different biological length scales are important for not only for understanding how     
1:03:51     
molecular events link to morph um morphology but also in terms of     
1:03:58     
complexity Theory so for example we could look at the length scale of an entire embryo the length scale of a     
1:04:05     
tissue or the length scale of a couple of neighboring cells interacting with each other those are all important to     
1:04:12     
look at sort of the complexity of a of an embryo or some other Collective uh     
1:04:19     
structure doing so may require reassessing canonical models that continue to guide the field the most     
1:04:26     
recognized model for organ formation centers around morphogens as determinants of gene expression and     
1:04:32     
morphological patterns so of course in in tring morphogenesis we have this     
1:04:37     
concept of morphogens it's a the gratical construct that uh that Turing     
1:04:42     
defined as this thing that guides morphogenesis the question is what are morphogens and how do they behave if you     
1:04:50     
take a any one system and you find particular MO molecules and you find how     
1:04:55     
they their mode of action and things like that so the classical view of a morphogen is a morphogen gradient     
1:05:01     
specify differential gene expression in a distinct spatial order so this is where we have these gradients and     
1:05:07     
morphogens they specify differential gene expression so we have a gradient to morphogens that are expressed perhaps in     
1:05:14     
a certain region of the embryo and then that gene expression is uh differential     
1:05:20     
meaning that as those Radiance overlap you get stripes or you get boundaries     
1:05:26     
when you have differential gene expression in one cell type versus another cell type you get different     
1:05:32     
gradients those gradients interact and ultimately you have different gradients     
1:05:37     
set up in different parts of the embryo and this is how you get pattern     
1:05:42     
formation because morphogen expression colocalizes with emerging feather and hair follicles and this is where some of     
1:05:49     
their previous work has led to looking at the formation of feather and hair f icles in uh in the     
1:05:59     
uh in the uh epidermal layers of skin this is served as a paradigmatic     
1:06:07     
example of such morphogen prepatterning mechanisms and so this is an example here of what they're trying to do so     
1:06:13     
they have this this is from a previous paper this is sort of their their model system where they have organ     
1:06:21     
morphogenesis they have this budding they have the margin in the core of this     
1:06:27     
hair follicle they have a layer of epithelial cells and then you have the epithelial cells that bend upward as     
1:06:35     
this follicle grows and you get this uh coordinated morphogenesis that     
1:06:41     
occurs so this is what we're looking at in section D here now in section A we're     
1:06:48     
looking at the gene expression space of a AV and skin dermis so we look at like     
1:06:54     
where if we look at these different spatial regions we can identify the genes that are involved in these     
1:06:59     
different spatial regions and we can use a umap uh representation to represent     
1:07:06     
next gen sequencing data that describe this gene expression space so we have     
1:07:11     
this um map analysis where there's this point cloud and within this point Cloud you have different um molecules that     
1:07:19     
belong to these different spatial regions so for example here here are the     
1:07:24     
genes that are involved in margin here are the genes that are involved in core and if you look down here you see that     
1:07:30     
the margin and the core are distinct they have distinct functions the margin serves to sort of as the base of this     
1:07:37     
follicle the core is the expanding part of the inner follicle and so you can see     
1:07:42     
that they have distinct gene expression profiles meaning that they have distinct     
1:07:48     
gene expression so if you look at the core on the margin there's a boundary the margin margin cells set up     
1:07:54     
a certain gradient of gene expression the core cells set up a certain gradient of gene expression when they intersect     
1:08:01     
you get a boundary between the margin and the core and so you can see that even though     
1:08:06     
they are distinct in gene expression space they interact in anatomical     
1:08:13     
space b is where we have supracellular biophysical effects of morphogens so these are different     
1:08:20     
aspects of function and then you get different types of biophysical effects so we can see here that elasticity it's     
1:08:28     
fgf expression and BMP expression and so you have these different properties here     
1:08:33     
this is a micropipet aspiration so they're kind of pulling on this tissue     
1:08:39     
so with when fgf is expressed there's less aspiration so that means elasticity     
1:08:45     
is limited so the idea is you're aspirating against the surface and you're pulling it into the pipet and so     
1:08:52     
if the material is more elastic you get more of it in this pipe pet tip and so you can see for BMP you get more of it     
1:09:00     
in the pipe pet tip suggesting that uh BMP uh expressing tissues are more     
1:09:07     
elastic fgf expressing tissues are less you get viscosity which is where you     
1:09:13     
have these spheroids merging together and when you get a lot of BMP expression there's merged and when     
1:09:20     
there's fgf expression there's unmerged and then activity which is collagen disc     
1:09:26     
contraction you get more disc contraction for BMP versus fgf so you     
1:09:31     
can see the size of the dis changes now the color coding represents the core and the margin so BMP is expressed in the     
1:09:39     
margin which is the blue and the blue or the margin cells have these different properties so you can not only look at     
1:09:46     
the uh at these type of cells as having a different spatial collocation and     
1:09:52     
expressing different genes but you know if you look at a single factor which is one of the main factors     
1:09:59     
that are expressed in morphogenesis fgf or BMP they have different property they     
1:10:05     
provide different properties to these tissues different physical properties and functional     
1:10:11     
properties then C we have our parameter space of super cellular phases so we     
1:10:16     
have activity versus elasticity versus viscosity so we're looking at activity     
1:10:22     
as this collagen disc contraction versus viscosity versus elasticity so that's what we're plotting     
1:10:28     
in this three-dimensional space these three properties and then we see that if     
1:10:34     
we take a control tissue it's uh very low in activity it's you know moderate     
1:10:41     
elasticity and viscosity this fgf core which is the core here which is solid like it has     
1:10:49     
very little elasticity and viscosity and very little low activity     
1:10:54     
um or I guess yeah so and then in the uh BMP     
1:11:02     
expressing margin which is this blue area it has very high     
1:11:08     
viscosity mid middling elasticity and high activity so you can see that the     
1:11:13     
BMP margin is fluid like and contractile so the physical properties uh you know     
1:11:19     
move this around in parameter space these two tissue are different there's     
1:11:24     
this gradient that's formed for each one there's this boundary and they have different gene expression profiles so     
1:11:31     
there's this different this tissue differentiation which then has to sort of play a role in uh TI Morag     
1:11:40     
Genesis so let me read the conclusion and then we'll move to the paper um so I'll start up actually up     
1:11:48     
here where they talk about their hypothesis we hypothesize that the emergence of these two materially     
1:11:54     
distinct adjacent tissue phases produces an instability whereby the active     
1:12:00     
contracti fluid margin propels the inner solidified core into the epidermis to induce budding so there's this     
1:12:07     
instability that forms between the margin and the core this margin because     
1:12:12     
of the differential behavior of these two uh tissue types the margin propels     
1:12:18     
the core up into this layer of epidermal cells it bulges up upward and then that     
1:12:25     
forms this structure so uh these are two material distinct     
1:12:32     
adjacent tissue phases uh the inner solidified core is uh shot     
1:12:40     
up into the epidermis and it induces budding indeed by coupling quantitative phase field modeling and experimental     
1:12:47     
data we showed that these two phases create an unstable complex resolve through buding so this is uh you know     
1:12:54     
when these two things are in sort of first of all they're different types of tissue they're expressing different     
1:13:00     
types of genes you get these gradients that you know form a boundary then     
1:13:06     
they're unstable sitting next to one another so they need to sort of resolve the tension and you end up with this     
1:13:13     
buding so this is an interesting way to kind of look at this process so our work     
1:13:19     
shows that understanding the role of morphogens and morphogenesis requ is characterizing emergent material and     
1:13:25     
mechanical properties at the super cellular scale so this is where we're looking beyond the cellular scale to     
1:13:31     
these tissues and to these uh tissue wide length scales this stands in contrast to the     
1:13:38     
prevailing view that functional effects of morphogens are on properties that could be characterized at the scale of     
1:13:43     
individual cells such as concentration sensing proliferation and chemot texis     
1:13:50     
so there's this idea that you know um we can really only understand the     
1:13:57     
functional effects of morphogens on single cells instead of tissues and this is showing that cells acting     
1:14:03     
collectively in these tissue types actually can do this as well so you know we're not really worried here about uh     
1:14:11     
chemotaxis or some sort of cellular sensing we're thinking about the collective behavior of     
1:14:18     
cells this Paradigm highlights the need to distinguish between the proximal effects of morph     
1:14:24     
which include modulating gene expression of individual cells their ultimate functional effects which enable the     
1:14:30     
formation of distinct supercellular phases that are capable of morphological     
1:14:35     
transformation so again there's this supercellular phase which is sort of the     
1:14:42     
tissue level of organization and this is kind of what we're dealing with here so     
1:14:47     
it's not just individual cells that are doing all the work here in terms of responding to orphagen and producing     
1:14:54     
it's we think of collective Behavior we think of single cells doing things collectively and then we kind of wave     
1:15:00     
our hands and say you know this is a emergence or then a miracle occurs or     
1:15:05     
something like this what we're seeing here is that cells are in this tissue the properties of the cells are just     
1:15:12     
fundamentally different and there's interactions between these two cell types not just in terms of gradient     
1:15:18     
boundaries but in terms of like resolving the distinct modes of behavior     
1:15:24     
between these two tissues so the full paper is here this is a science paper morphogens enable     
1:15:31     
interacting supracellular phases that generate Oran architecture and so this     
1:15:37     
is where we get into the uh you know sort of generating these bigger length     
1:15:43     
scales of organ architecture and the idea that morphogens in the form of     
1:15:49     
BMP and uh and FG GF in the form of BMP and fgf can     
1:15:58     
regulate those so uh the editor summary     
1:16:03     
morphogens are secreted molecules that have critical roles in the generation of tissue     
1:16:08     
morphology but how individual cells effects scale up to create supercellular structures composed of hundreds to     
1:16:15     
thousands of cells remains elusive focusing on Aven skin Yang at all     
1:16:20     
discover that the two morphogens enable material mechanical properties that manifest at the suell level so this is     
1:16:28     
another perspective article bavar Nelson which uh you know can look into if     
1:16:34     
you're interested functional effects of morphogens and large tissues can thus be understood by consideration of emergent     
1:16:41     
supracellular properties and again you know this goes back to the hyper networks because we     
1:16:47     
can actually Define in our hyper networks these categories that are bigger than the individual cells and we     
1:16:53     
kind of I in the examples I gave they are kind of like very simple you know where you have a tissue type like     
1:17:00     
neuronal tissue and I had several categories for neuronal tissue they were not really clear in the in the example     
1:17:08     
but if you think about this in terms of coming up with a a framework for this     
1:17:13     
you know those kind of different categories could be different uh parts     
1:17:18     
of that organ or different parts of you know different constituents cells that     
1:17:23     
have different properties it may Express different types of mole have different types of molecular signatures as I said     
1:17:30     
the components of a hyper node can be molecular components forming a network     
1:17:36     
or it could be different cells that have different you know varying properties if     
1:17:41     
if you think about these different categories as having different properties of     
1:17:46     
um different different molecular profiles and different functional properties then we can begin to see how     
1:17:53     
we can represent something like this so we can get to that supracellular level um and that's an important part of     
1:17:59     
development as we see in this paper so this kind of goes through some     
1:18:05     
of what we saw in the preview this is the figure that we went over um and     
1:18:11     
don't have the full article up but this is just showing the full article sort of     
1:18:17     
the the name of it in the citation okay     
1:18:24     
the next paper I'd like to talk about is this uh optimal control of Gene regulatory networks for morphogen driven     
1:18:32     
tissue patterning and this is by Albert pizota and James     
1:18:37     
Brisco and so again we'll go over the graphical abstract in the inbrief so the     
1:18:43     
inbrief reads an approach based on optical control theory using reinforcement learning to identify     
1:18:49     
signaling dynamics that guides cells to specific Fates and in a timely precise and reproducible manner offers insights     
1:18:57     
into how morphen control Gene regulatory networks pattern tissues and establish     
1:19:02     
an alternate framework to the French flag model so they'll talk about the French     
1:19:08     
flag model which is where you have these Stripes that look like the French flag     
1:19:13     
talk about in the paper a little bit uh so this is using a lot of mathematical     
1:19:19     
tools um as opposed to experimental tools although they do use some experimentation     
1:19:25     
here um they're using optimal control theory reinforcement learning and     
1:19:30     
they're looking at signaling Dynamics so they're actually looking at cells here but they're also looking at these kind     
1:19:36     
of uh tissue wide patterns so this shows the regulatory     
1:19:43     
Landscapes and the morphen driven patterning so here we have regulatory     
1:19:48     
Landscapes we have a landscape which is usually one of these dynamical Networks Landscapes where you have by stable by     
1:19:55     
stability so this is the by stable point and a cell on this point can either     
1:20:00     
going One Direction or another these valleys represent distinct states that     
1:20:06     
cells have you know if they can change their state if they can climb out of this energy well and come up to the by     
1:20:14     
stable point at which point they can transition into another state but you know they Ed these Landscapes to     
1:20:21     
represent uh sort of this the way that states are sort of uh evolve over time     
1:20:27     
so we have a cell that's you know usually stem cells are     
1:20:32     
poised uh in at one of these bable states to go into a number of states um     
1:20:38     
ultimately they'll probably go into a single state and stay there but sometimes they'll pop out and go into     
1:20:44     
another state like for example cellos through hosis or a cancer phenotype or     
1:20:49     
something like that so these energy l landcapes are you know we use the parameter V which is you     
1:20:56     
know usually associated with some sort of physics U you know representation or physical parameter uh but it's really     
1:21:03     
just a conceptual model to show that you know you have these distinct states that they're sort of uh valleys of Potentials     
1:21:12     
in other words once you're in a state it's hard to climb back out you have to put a lot of effort in getting out of     
1:21:17     
that state and moving to another state whereas if you're poised at a multi-stable point     
1:21:23     
it's easy to go into one state or another different uh things going on in     
1:21:28     
that cell so we have this representation here of these two cells where there's some     
1:21:35     
sort of you know relationship between this uh one cell state in another and     
1:21:41     
there's either a break or an accelerator put on that cell um and this says optimize speed     
1:21:48     
accuracy and energy tradeoffs so there are these different tradeoffs and in     
1:21:54     
differentiation so there's this uncontrolled mechanism here versus a closed loop mechanism so basically in     
1:22:01     
the uncontrolled mechanism we have this feed forward that drives it from one state to another you have this feed     
1:22:07     
forward of a break or an accelerator in the closed loop case however you can     
1:22:13     
optimize the speed accuracy and energy tradeoff a set of trade-offs that need to be made by introducing this parameter     
1:22:20     
fee and so this is the control this is where they're pulling in reinforcement learning and optimal control theory so     
1:22:28     
you can actually shape this process with some external parameter and there's this Clos Loop feedback that forms between     
1:22:35     
the initial State and the break accelerator and then the final state so     
1:22:40     
you can actually shape where that goes so you see here it's no longer a by     
1:22:45     
stable State you're just moving towards a final State and there's this control mechanism that makes sure it gets there     
1:22:52     
and so then this is the equation that we use for to Define V which is negative uh     
1:23:00     
uh change in V control there's this gradient um that's defined as a negative     
1:23:07     
gradient going downward that's your V control and then uh VF which is     
1:23:12     
effective which is V plus v control so basically you're adding the control component to this process V which is you     
1:23:20     
see here is the landscape so landscape V going negative and you have this Control     
1:23:25     
process that guides it down into the well and so you can see from your initial state to your target State you     
1:23:32     
gradually get to your target state so that's a regulatory landscape     
1:23:37     
now we talk about morphen driven patterning which is where we have this closed loop control we take this fee     
1:23:44     
parameter we plug it into this process which so the regulatory landscape is a     
1:23:49     
single cell these single cells individually go through this process of finding its state it can either be     
1:23:56     
uncontrolled which is an open loop uh feedforward system where you have this by stable State and then you go into one     
1:24:04     
Lan one one stable state or another or you going have this closed loop control where you have this fee parameter which     
1:24:11     
basically guides it down to a state but a morphogenetic driven     
1:24:16     
patterning you do this in multiple cells at once or you do this on a gene regulatory Network so there's this Clos     
1:24:22     
Loop control parameter fee it controls this process at every step so the     
1:24:27     
morphogen is actually send a signal to both transduction and this closed loop     
1:24:33     
control system transduction is also sending a signal to the closed loop     
1:24:39     
controller these two processes morphogen and transduction lead to an affector and a     
1:24:45     
Target genetic regulatory Network so this is where we have more     
1:24:52     
orens acting on a cell things are being transduced into the cell then there's an     
1:24:57     
affector which is the behavior of the cell and then there's this target or     
1:25:02     
into the uh gene expression mechan uh mechanism and then you have this target     
1:25:08     
genetic regulatory Network where you have these genes operating or cells     
1:25:14     
sometimes operating in a network and so Clos Loop control then takes a signal     
1:25:19     
from the grn as well and then sends a signal to transduction and affectors to     
1:25:25     
control this process and so you can see that morphogen Dynamics change you get     
1:25:30     
this gradient and you get you go from morphen Dynamics and a gradient to patterning Dynamics where you get this     
1:25:37     
gradient over developmental time so that's what they're talking this is very different from the French flag     
1:25:44     
model um so the highlights of this is morphogen morphogen signaling controls     
1:25:50     
the pattern of gene expression and develing tissues optimal control theory I identified signaling mechanisms for     
1:25:57     
morphogen patterning by incorporating feedback between signaling and Gene regulation it     
1:26:03     
explains Dynamics it being the model provides an     
1:26:08     
alternate framework to the French flag model for morphen morphogen patterning so this French flag model is the stripe     
1:26:15     
model where you have this these cells that are interacting in these gradients and the boundaries between gradients     
1:26:21     
Define the edges of the stripes and you get these distinct stripes and so this     
1:26:26     
is a different model which uses this sort of Clos Loop     
1:26:34     
control so the summary reads the generation of distinct cell types and developing tissues depend on     
1:26:41     
establishing spatial patterns of gene expression so this is the French flag model where we have these Stripes that     
1:26:47     
are organized in space you know one after the other after the other and they're in parallel     
1:26:53     
often this is directed by spatially graded chemical signals known as morphogens so there's a gradient that     
1:26:59     
happens and then their boundaries between the gradients you get boundaries and then that's how you define your and     
1:27:06     
the cells within those regions are defined by whatever those gradients look     
1:27:11     
like and the boundaries look like in the French flag model morphogen concentration construct cells to acquire     
1:27:17     
specific Fates so depending on what part of the gradient you're in which gradient     
1:27:22     
you're in this tells the cell this is your fate so if you're in the leftand     
1:27:28     
part of the French flag model you'll acquire a certain fate if you're in the right hand part of the French flag model     
1:27:34     
you'll acquire another fate it's really depends on you know what the you know     
1:27:39     
what what kinds of things chemical the chemical milu is guiding this process of     
1:27:44     
differentiation but it's all spatial and so you know it kind of almost seems like a topology you say well okay there's     
1:27:51     
this gradient there these boundaries the cells Within These uh parts of the     
1:27:57     
gradient are reinforced and you get this pattern formation out of nothing then the question is well how does it begin     
1:28:04     
you know how do you is it just that the cells start expressing these uh factors     
1:28:09     
autonomously or are they responding to signals in the environment and if they are the signals patterned in the same     
1:28:16     
way that this French flag will will form uh we don't you know that that's stuff that people kind of it work kind of     
1:28:23     
tried to work out but really I think there's some assumptions there that are um maybe total logical um so this is     
1:28:31     
this is kind of where they're coming from this how this mechanism produces timely and organized sufate decisions     
1:28:37     
despite the presence of changing morphogen levels molecular noise and individual variability is unclear so we     
1:28:44     
don't like I said we don't necessarily know the origins of this in addition there are a lot of things like there's     
1:28:50     
FL there fluctuation and morphen levels there's molecular noise there's individual variability which means that     
1:28:58     
cells respond differently to different types of signals and so those should not     
1:29:03     
necessarily result in a flag with you know solid colors from left to right     
1:29:10     
with stripes but it should lead to some sort of Mosaic nature so you could have stripes but the stripes would be Mosaic     
1:29:17     
or there might be you know cells kind of drifting across the boundaries um we don't really know you know we have to     
1:29:24     
account for all those things so this is something that we really need to think about in terms of     
1:29:29     
feedback we kind of think of sell uh differentiation as this feed forward     
1:29:35     
process as I said but you also have feedback sometimes those that feedback     
1:29:40     
can be kind of like you know adding more of something to something but sometimes it's control and so that's what they're     
1:29:47     
trying with their fee parameter here they're adding a control mechanism that monitor monitors what's going on in     
1:29:53     
different processes and then using that to feed to can sort of guide the process     
1:29:58     
of differentiation so this accounts for noise you can plug noise into the fee term you can plug individual variability     
1:30:06     
into the Fe term and so forth so moreover feedback is present at     
1:30:11     
various levels in developing tissues breaking the link between morphogen concentration signaling activity and     
1:30:18     
position so that Fe this feedback and if you think about this at various levels of tissues so if we go back to our     
1:30:25     
figure regulatory Landscapes of individual cells in the morphogen driven patterning we see that we're dealing     
1:30:31     
with two different scales of organization we're dealing with single cells and often times the single cells     
1:30:36     
are doing this whole process of finding things on the landscape on their own sort of like     
1:30:44     
independently uh but that's of course not the way biology Works cells are in you know uh proximity to each other     
1:30:51     
cells are not in proximity to each other and so that has to play a role you would think in some of these regulatory     
1:30:58     
Landscapes that they may look different from one end of the French flag to the other furthermore we have this morphogen     
1:31:05     
driven patterning which involves multiple cells we have grns which are     
1:31:10     
different in each cell we have but we have to account for that so we need this sort of Clos Loop control that provides     
1:31:17     
feedback to the different cells and the grn is impacted in different ways depending depend on its structure     
1:31:22     
depending on its activity level and so forth so this kind of model if they're     
1:31:29     
introducing here could deal with these things and kind of getting this breaking this explicit link but also sort of you     
1:31:37     
know providing us with a means to have some sort of coherent pattern formation     
1:31:44     
at the super cellular level so here we develop an alternate framework using optimal control theory     
1:31:50     
to tackle the problem of morphen driven patterning so to recap intracellular     
1:31:56     
signaling is derived as the control strategy so intracellular signaling which is within the cell you have     
1:32:03     
signals that you know are kind of feedback from the environment feedback from what this the cell's current state     
1:32:09     
is this is derived as the control strategy that guides cells to the correct fate while minimizing a     
1:32:15     
combination of signaling levels and time so the you know signaling levels are     
1:32:21     
kind of you know sort of averaged over or you know they're they're given a sort of a     
1:32:27     
summary by the feedback mechanism and again we don't know exactly what the feedback mechanism is in terms of     
1:32:33     
biology but we know that cells do you know people have proposed things in systems biology such as you know     
1:32:40     
cellular signal processing where signals are you know summed where there you know     
1:32:46     
there's some sort of signal processing going on with respect to smoothing or     
1:32:52     
Peak clipping or things like that so you know and these are all these like morphogens are all analogies so you know     
1:33:00     
we have to take that with a grain of salt and so that this is basically the     
1:33:07     
control strategy intracellular signaling and of course intracellular signaling is incredibly complex so it's easy to throw     
1:33:14     
your fade in with a very complex system and say well you know whatever comes out the other end will be the answer so just     
1:33:22     
that's another aren of uh this approach recovers experimentally observed properties and patterning strategies and     
1:33:29     
offers insight and design principles that produce timely precise and reproducible morphen patterning so     
1:33:36     
that's you know you know they they've actually tested this out and you know it's something that I think you know is     
1:33:43     
probably you know it's it's a candidate it looks like it it's it's a workable model I don't have any real problems     
1:33:49     
with it um so one of the key things here are these Gene regulatory networks or grns in the     
1:33:57     
responding cells so these grns comprise of intracellular signaling Pathways and     
1:34:04     
these are where we have you know interactions between morphogens um you know things that     
1:34:10     
control the expression of morphogens in single cells so you're actually getting down to the grn level which is where the     
1:34:16     
morphogens are actually being produced um and the downstream transcriptional responses which are     
1:34:23     
Central to transforming The Continuous spatio temporal input am morphogen signaling into discrete     
1:34:30     
sulfates again with grns you're defining a network of genes you're saying this is     
1:34:36     
you know the way this is this is what the cell was doing but you know again     
1:34:42     
you have to pick the things that you're interested in for your grn so that's a representation of this intracellular     
1:34:49     
signal it's a standin for things that may or may not be sort of the primary     
1:34:54     
drivers of this process we can say that you know Morag Gom morphogens is the     
1:35:00     
answer but you know they you you know I don't really have a problem with doing it that way but it's     
1:35:07     
a it's a way it's a choice regulatory interactions between components of these     
1:35:13     
networks appear to perform the equivalent of an analog to digital conversion this is what I was talking about with uh cellular signal processing     
1:35:21     
where these regulatory interactions uh perform this analog to digital conversion in other words these     
1:35:28     
analog signals you have morphogens that occur at a certain concentration that     
1:35:35     
concentration is analog in nature because it goes up and down with the production of the you know the activity     
1:35:42     
of the genix regulatory Network and the uh production of     
1:35:48     
morphagene and so that is you know analog that fluctuates in an analog manner but     
1:35:54     
then it needs to be converted to some digital uh representation or digital     
1:36:00     
thing that the gene regulatory Network can then act on so whether that be a threshold whether it's above or below a     
1:36:07     
threshold or something else like that where it's a m number of discrete States     
1:36:12     
that's the kind of thing that they're proposing that these regulatory interactions are     
1:36:19     
doing so uh moreover nonlinearities and feedback within the grn can confir     
1:36:24     
multi-stability memory and hysteresis to so decision making and again we have that M example     
1:36:32     
of multi-stability where you know a cell can sit in a multistable state and go     
1:36:37     
into one fade or another with ease uh you know so you know you could have a     
1:36:42     
cell that could be poised to go into two or three states depending on other things going on in the environment and     
1:36:50     
it wouldn't be that much of a a task you know it's just kind of like where you're poised to you know go into     
1:36:57     
one or another state so you know a lot of stem cells or uh precursor cells are     
1:37:04     
like that where you have like if you're a neural precursor cell you can go into a number of neural States but you're not     
1:37:10     
going to become like a bone cell or a muscle cell um other cells are kind of     
1:37:16     
could become either a bone or a muscle cell it just really depends on the sort     
1:37:21     
of the overall state of the cell and so this is something that's an important property of um this this kind of a model     
1:37:30     
of morphogenesis then of course memory and hysteresis hysteresis being the uh     
1:37:36     
inability to return to a previous state so you know if you want to lock a cell into its state you know you want     
1:37:44     
hysteresis where it can't just move back to being a stem cell or move to another     
1:37:50     
state easily so these are things this is this whole literature on cellular decision making     
1:37:56     
again this is an analogy to human decision making or animal decision making which is not really the same     
1:38:02     
thing it's just kind of referring to how changes happen in a cell and how they're     
1:38:08     
sort of initiated so this is a different kind of they use the term decision making but it's not at all like what we     
1:38:15     
would see in say the brain a consequence is that is that Sate     
1:38:21     
depends not only on the levels of signals and effectors but also on their temporal features so again as they     
1:38:28     
evolve over time through development as they make these decisions as they change     
1:38:34     
their fate the temporal features are uh also kind of feeding back on this so     
1:38:39     
feedback allows you to take those into account as well taking together the complexity of interactions within the grn can produce Rich Dynamics in the     
1:38:47     
signaling and gene expression developing tissues understanding the origin and function of these Dynamics offers     
1:38:53     
insights into patterning moreover the interplay between morphogen gradient and grn     
1:39:00     
allows cells to actively contribute to morphogen signaling rather than simply being instructed by the gradient so it's     
1:39:06     
not a topown process it's an emerging process but it's emergent through     
1:39:11     
individual cells doing their thing and then there being this feedback mechanism and then interacting with other cells so     
1:39:18     
this is where we get this this gradi into sort of a topown sort of a template     
1:39:24     
that we assume because we see this pattern but that's of course not necessarily the way it works we need     
1:39:30     
this bottom up emergent phenomena that we you know kind of that's kind of what     
1:39:36     
we had initially started out with and then we impose something from the top down some kind of cheat using the French     
1:39:42     
flag model this highlights a need for alternative par times to the French flag model in which the gun plays a con a     
1:39:48     
complimentary and equally important role to the the morphogen to frame questions about morphogen activity so we can look     
1:39:55     
at the dorsal vental patterning of the developmental verteb neural tube this is     
1:40:00     
a well established example of a morphogen pattern tissue and so this     
1:40:05     
involves Sonic Hedgehog being uh produced from the notic cord and the floor plate which are located at the     
1:40:12     
vental pole which is at the top forms a ventral the dorsal gradient so you get less Sonic Hedgehog As you move     
1:40:19     
down and so this involves a number of other molecules so you see that these     
1:40:24     
morphogens can be you know pretty complex and so if you have a network of them then it makes it easier to account     
1:40:31     
for these these interactions so the neural tube gr onic     
1:40:37     
you know combines uh Sonic Hedgehog with the Glee family of proteins so this is     
1:40:43     
includes members of the nkx Ole PX and KS families so you can include all those     
1:40:49     
in your grm then dynamical systems theory provides a framework to describe the activities of morphogens and new     
1:40:55     
year enss as we saw we had our model of our potential model with the wells and     
1:41:02     
we had v as the parameter that described the sort of the energy uh you know     
1:41:09     
negative energy Minima so like a negative V is downward a positive V is upward and that's that's from dynamical     
1:41:16     
systems theory and that's a good way to describe it from a physics perspective but then have to map that V to Something     
1:41:23     
in the cell something biological and that's kind of where it gets a little tricky but um you know basically people     
1:41:30     
think of this as like potential or cell state or something like that so Behavior produced by such models     
1:41:38     
can often be represented geometrically as a dynamical landscape this provides an intuitive description of self fate     
1:41:44     
decisions that correspond to the idea of an epigenetic landscape proposed by wadington so this is actually very CL     
1:41:51     
similar to the epigenetic landscape but it's not identical to the epigenetic landscape so this is a landscape that is     
1:41:58     
sort of describing cells they're using a model from physics and you know     
1:42:03     
waddington's model involved these channels or canalization where these Wells are actually parts of channels     
1:42:09     
that move in time so they're moving you know this this uh sort of B this bable     
1:42:16     
or multi-stable state is really the top of like the channel or the canal and then     
1:42:25     
the canal branches into two and you can take one route or another down so you're always minimizing V in     
1:42:33     
the epigenetic landscape this doesn't really form an epigenetic landscape however in in the     
1:42:39     
conventional sense this forms kind of like more of a you know something that     
1:42:45     
always leads down to one single channel so you know this is kind of a different     
1:42:50     
view of this I don't know how this would map to an epigenetic landscape versus the     
1:42:55     
uncontrolled but they do mention that this kind of ties into     
1:43:05     
uh this kind of ties into a gentic landscape sinu wadington     
1:43:11     
so in this view the developmental trajectory of a cell is analogous to a particle rolling on an un undulating     
1:43:18     
landscape where we have these valleys or these we and water sheds representing Fates and     
1:43:23     
decision points respectively warens can be thought of as tilting the landscape in such a way that The Valleys can be     
1:43:30     
made deeper shallower disappear all together so those those Wells that we saw represented by negative V they can     
1:43:37     
get shallower or deeper so that negative V can be larger or smaller or the     
1:43:43     
maximum negative V can be larger or smaller depending on the morphogen activity so this is an interesting kind     
1:43:49     
of Insight because you know morphogens are really controlling this     
1:43:55     
process uh in this way the morphagene controls the terrain and hence a valley     
1:44:00     
a cell the valley a cell enters so again if we map this back to these energy Wells those energy Wells are entered as     
1:44:08     
we get some sort of decision making Happening by the     
1:44:14     
cell um so nevertheless it remains a challenge to construct landscape models     
1:44:19     
and incorporate the knowledge of signals and grns how is the landscape Modified by an     
1:44:25     
external signal and how are feedback mechanisms Incorporated how can experimental invert Landscapes give     
1:44:30     
insights into the signaling Dynamics so these are things we don't you know these are things that they try to answer in     
1:44:36     
this paper this actually talks about uh Mars levels this framework made explicit for     
1:44:43     
the visual system has become a widely accepted Paradigm for understanding General cognitive processes and     
1:44:49     
information process processing and biology more generally the distinct levels pertain to     
1:44:55     
the tasks performed problem solved by the system under consideration which is the computational level rules and     
1:45:01     
representations used in solving these tasks which can be described input output mathematical laws or the     
1:45:07     
algorithmic level and the physical realization of these laws the hardware and physical processes used to solve the     
1:45:14     
computational problem or the implementation ofel so they got to actually break this problem down into     
1:45:19     
Mars level which I find interesting so this is the reinforcement learning solution for the optimal control of     
1:45:25     
vental neural tube so they're using ventral neural tube and the gr unassociated with that to do the     
1:45:31     
simulation so this is where they're doing this reinforcement learning solution so schematic representation of     
1:45:38     
the Clos Loop control the system the activator and repressor forms of GLE     
1:45:43     
protein are given by the function of the neuroprogenitor marker is pack 6 Li 2 NK     
1:45:50     
x 2.2 and Ur 3 which evolve according according to the stochastic Dynamics     
1:45:55     
defined in Star methods which are in the paper uh ventral neuroprogenitor grn     
1:46:00     
model which is p i Network so this is ex you know kind of an example the     
1:46:06     
expression which is in arbitrary units and time which is in arbitrary units and     
1:46:12     
so we have these diff this is the G uh the grn here where we have uh glea glear     
1:46:19     
pack six X3 LG 2 and KX 2.2 and their     
1:46:25     
respective colors and then you see the differential equation here and then this other equation with     
1:46:31     
v now this is the expression here so this is a profile for different genes in     
1:46:37     
the network uh this is GLE a this is BL 2 PEX six works     
1:46:45     
three and then this is uh AA versus R in     
1:46:51     
this figure here and then this is o 2 xks     
1:46:57     
3 and pack six in the bottom so you can see the Dynamics as they fluctuate so     
1:47:04     
basically as these in these gr ends you get these fluctuations of activity over     
1:47:10     
time uh some of them settle down and some of them rise in terms of expression so nkx 2.2 Rises over time and then it's     
1:47:18     
a plateau we two has a big pulse at the be near the beginning of the process and     
1:47:24     
then dies off uh the a sort of establishes things so does XS three they     
1:47:30     
kind of have this High activity initial High activity in them follows with with     
1:47:35     
Lea exibiting a small Peak uh early on and so you can see this you know as you     
1:47:42     
run these models and the interactions between them you see different patterns here so this is a reinforcement learning     
1:47:49     
solution the optimal control problem is solved separately for two different targets A and C the nkx 2.2 plus Target     
1:47:57     
d& so these different uh figures here show so okay let's see so if you go up     
1:48:05     
to a this shows the network b c d and e we'll focus on B and D here uh     
1:48:14     
so uh D and E are the only two Target and and C is the     
1:48:21     
nkx2.2 target um and so this kind of goes through the     
1:48:28     
B is a sample of the control Dynamics for the nkx2.2 plus Target solid lines     
1:48:33     
are the medians and shaded areas are the 10th to 90th percentiles so you see that it's you know number of runs and you see     
1:48:41     
the Shaded area which is like the variation in the runs the control ustar comprising     
1:48:47     
activator and repressor Glee top and and the gene expression Dynamics bottom the     
1:48:52     
solution although the repressor Glee remains always low the activator Glee exhibits an initial transient     
1:48:58     
characterized by high values and then to then reduce drastically at steady state     
1:49:04     
reminiscing of the Adaptive Dynamics observed in experiments so they're actually replicating what they see in     
1:49:10     
experiments here you get these uh these different phases these transient vers of     
1:49:15     
activity and then these plateaus where things are stable so this is basic basically their simulation results and     
1:49:22     
so you can also apply reinforcement learning to morphen driven patterning tasks you have a number of results here     
1:49:29     
and then we have our our methods which are available online here so these are detailed methods which we'll go through     
1:49:35     
right now but there're you know different types of things for the reinforcement learning Solutions     
1:49:42     
multi-agent control Dynamics environment Dynamics optimal control the potential     
1:49:48     
and their optimal stochastic control and solution method so this is pretty informative if you're interested in this     
1:49:54     
kind of modeling um I'll read the last paragraph to finish here um the optimal control     
1:50:02     
approach with its focus on linking mechanisms with control is ideally suited for the analysis of inv vitro and     
1:50:09     
synthetic systems this could be used to design and refine signaling regimes from the directed differentiation of stem     
1:50:15     
cells in vitro and the production of specific sets of cell types in Define     
1:50:20     
proportions an understanding of the control principles operating in biological systems May     
1:50:26     
provide insites and inspiration from the construction of artificial systems as well as support the use of stem cells     
1:50:33     
and disease modeling and your generative medicine and with that thank you for paying attention and I hope you learn     
1:50:39     
something
