## Transcript     
0:03     
hello hi Hi how are you um pretty good um I just want     
0:11     
to tell you I probably won't make the next meeting     
0:17     
okay going on a holiday oh yeah that's good     
0:27     
yeah suric and an     
0:33     
hello how are you hello     
0:41     
everyone there's Morgan uh so I don't know if people     
0:46     
anyone I thought maybe uh one of the aspiring G so     
0:52     
students wanted to present today     
1:00     
so yeah I'm for I'm an aspiring G parp okay yeah but I mean I had someone in     
1:08     
mind and I don't know if they want to present this week I don't know they're here I can't remember who it was but uh     
1:16     
if they if they want to present that's great uh well I'll I'll do some stuff     
1:21     
first and then we'll Circle back around with that so um but I'm going to present     
1:27     
some uh something on Sen tium seans today and     
1:33     
here's and then uh I'm gonna talk about some other things as     
1:41     
well actually let me ask m m did you want to present     
1:49     
something uh hi Bradley uh yeah so I really I don't have     
1:55     
anything to present but I was talking to Joy he wanted to present uh this     
2:01     
transaction graph uh so but he told me that today he's not     
2:06     
well like from yesterday he's not well and he won't be able to join so next week he has a presentation ready and     
2:13     
yeah so that was his plan okay yeah all right that's good um thanks for that yeah I'm looking forward     
2:19     
to it transaction graphs you say uh I think he called about this I     
2:26     
don't remember exactly uh uh it was temporal graph not not transaction gra I'm     
2:31     
sorry yeah oh yeah so he has good literature review over it he has one published paper in that I think all     
2:39     
right yeah yeah okay I look forward to that so that may be next week uh so yeah     
2:47     
I don't know yeah we'll see yeah so that's that's good um yeah we'll see about that     
2:55     
looks like we have a new person Pierre galapo so hello I don't know if you     
3:01     
wanted to introduce yourself in the chat we have a chat function at the bottom where you can type in an introduction if     
3:09     
you Jesse's here no Jessie Jessie has been here I have a     
3:15     
couple things today um the first thing is to we talked last week about graphs     
3:21     
and about layouts and there are all sorts of layouts you can use for graphs you can use Force layouts Force directed     
3:28     
layouts uh circular layouts uh different types of sort of     
3:35     
topological layouts where you know you have like space that's taken into     
3:40     
account this is uh from a package called geffy I don't know how many of you are familiar with it um this is something     
3:47     
that is made kind of for     
3:53     
graphs this is made for graphs um for plotting them you put them in in the     
3:58     
form of your nodes in your edges and then you plot them out you know in some     
4:04     
sort of layout so you can gather statistics and all this and it's in a graphical user interface and this is     
4:11     
from the documentation for geffy so this is where they talk this they guide you     
4:16     
through tutorial on layouts so this is an introductory tutorial about layouts     
4:21     
and geffy it will guide you to the basic and advanced layout settings in geffy     
4:27     
you learn how to use various layouts according ACC to the feature you want to emphasize in the topology and size of     
4:33     
the network so this is where they have built in topologies for networks they     
4:39     
have a built-in circular layout they have a Geo layout which means you can plot them in uh I think Geographic space     
4:47     
or geometric some sort of geometric space there's some Metric space they use     
4:53     
uh you can also have a geometric transformation and there other ones as well they force layout I know I've use     
5:00     
those where it's like they use uh sort of like they just put the nodes in the     
5:06     
optimal locations where they're not overlapping and you don't see a bunch of overlap in     
5:12     
in the layout so it's like it might be like you know laying it out on an Axis or laying it out in clusters or whatever     
5:20     
and those don't mean anything other than that's where it's physically optimal to lay those nodes     
5:25     
out so they have uh a number of different so they have     
5:30     
Ferman reinold uh Force Atlas 2 they have the circular layout radial access     
5:38     
layout geographic map uh and you know various other ones     
5:44     
and I think you can write your own but one of the reasons I bring this up is I think you can write your own uh plugins     
5:50     
for this so geffy of course runs um it's an open source tool and it runs from a     
5:57     
guy but you can write extensions and plugins and you can do interesting things with that um I've generally used     
6:05     
Guffy for a lot of the stuff I've done it's maybe not optimal there's also um     
6:11     
you know tools JavaScript tools there also python tools I believe but for     
6:17     
visualizing things geffy Works fairly well uh so that's just that     
6:24     
tool okay now I want to get into this sentian development so I did a I think     
6:30     
last week I mentioned that I had done I had collected some papers on and this     
6:36     
was an original question from dick Gordon about uh centium and C Elegance in     
6:44     
development so centium are cells that have multiple nuclei and they're usually the product     
6:51     
of Cell fusion and the question he asked was do these you know do these show up     
6:58     
in the lineage tree and if so can you put them in a lineage tree and so the answer that I found the     
7:05     
short answer I go through some slides but the short answer is I don't think that people have really put that into     
7:11     
lineage trees um you could do that um but there are a lot of in actually as you get into     
7:17     
the literature it's both unclear as to how to represent them and then like what     
7:26     
um you know what you know basically visualizing their larger significance     
7:32     
and development there's a lot of things that sential cells do and where they are in the anatomy that are really     
7:40     
interesting so I have a chat message here okay yeah welcome Pierre yeah so     
7:46     
this is Pierre's introduction Pierre gallipo um good afternoon morning     
7:52     
everyone sorry I of another talk I promised to attend so I'll try to follow both I am a postto specialized in     
7:58     
morphogenesis and I've been in contact with Dick for a few days concerning a project to structur at the     
8:05     
morphogenesis welcome here yes this is our group um we've been doing this group     
8:10     
for about 10 years a little over 10 years and you know we we talk about developmental biology and biophysics and     
8:18     
computational biology so we have a lot of uh fingers and a lot of things and so     
8:24     
we'll talk about like machine learning stuff we'll talk about developmental biology stuff we'll talk about a lot of     
8:31     
other things so welcome and um you know uh we'll see we'll see if you what     
8:38     
you find interesting and please let us know if you find something     
8:43     
interesting okay so I'm going to start this presentation and this is uh centium in     
8:51     
the sea alans lineage tree so sential cells are these cells that have multiple     
8:57     
nuclei we'll see some mic croscopy of what they look like I was trying to     
9:02     
figure out how to draw them out and I I don't know if I I got I would get it     
9:07     
right if I drew it out I mean I know what the microscopy looks like but I was thinking of drawing some sort of time     
9:15     
laps or time series pictures and I couldn't do it so maybe I'll do that later but uh this is just kind of an     
9:22     
overview of the a few papers in the literature so that we can see what were you know what they have just in C     
9:28     
Elegance so this of course is our seans lineage trees you see we're at an advantage when     
9:35     
we work with seans because we have uh you know cells     
9:40     
that divide and then we have a nomenclature for each sublineage and     
9:46     
those cells are invariant meaning that they don't change their identity uh in     
9:52     
development so when you have say for example a zygote you have it divides into two cells an AB cell and a p one     
9:59     
cell that P1 cell has a number of deterministic Fates and the ab cell has     
10:07     
a number of deterministic Fates and if you remove some of these cells in development if you ablate them out of     
10:13     
the embryo um then the uh worm will have be dysfunctional in some way you have uh     
10:21     
mutant phenotypes that have missing cells and mut mutation acts upon the     
10:27     
lineage tree as the cells are going dividing it just doesn't either it divides in a asymmetric way or it     
10:33     
doesn't divide and you end up with these you there's no compensation like you might see in mammals all the cells are     
10:41     
have a known role deterministic and and the organism is what we call ellic which     
10:46     
means it has a a finite number of adult cells that it ends up with so that's the     
10:52     
context for uh sential cells and celegans uh you have to have a certain     
10:58     
number you always have a certain number of cells in an adult either if it's a a hermaphrodite or a male and you know     
11:08     
the mutants have a certain number of cells usually so you if you have a mutant that's at a certain wheel that     
11:15     
mutation then has a results in a certain loss of cells so everything is very     
11:21     
regular so it's a good system to study uh the sential cell process formation     
11:27     
process so this is the L tree again um you know we're organized from anterior     
11:33     
to posterior so AB that side is anterior the p is posterior and all these other     
11:43     
uh sublineages are different specialized sublineages with you know that end up     
11:48     
with different types of tissu so uh this this figure here on the top you see the     
11:54     
blue which are endodermal cells so sublineage e results in endodermal cells d results in mesodermal cells P4 is the     
12:02     
germ line C is a mix of neurons endodermal or neurons mesodermal and     
12:09     
hypodermal cells or epithelial cells and then Ms is a mix of mesodermal cells um     
12:16     
and neurons and uh I guess Endo um I guess     
12:21     
it's misal and endodermal cells or I mean isoderm and neurons so     
12:29     
yeah the point is is that there's this uh we know what every cell is and so     
12:35     
down here at the bottom right or the bottom left you see that there's this axial organization of the lineage tree     
12:42     
anterior to posterior and then also right to left if you notice our nomenclature the A's are anterior the uh     
12:50     
the P's are posterior the L's are left and the RS are right so you can you can pretty much know where a cell will end     
12:57     
up by its nure and you also see in the lineage tree that if it's drawn in a     
13:02     
certain way this is from worm book you get these cell migrations that happen     
13:07     
across the anatomy so the cells will move around their Origins are in a     
13:13     
certain part and they they'll migrate to certain places and you get these this is a special interesting part of the ab     
13:19     
sublineage where you get these repeated sublineages that kind of migrate and so     
13:25     
these lineage tree crosses location here but that's actually kind     
13:31     
of important for sential cells so this is um Cal cells in seans this is just in     
13:38     
seans there are sential cells in all sorts of organisms in dropa in uh     
13:45     
vertebrates of course and all sorts of tissues form these uh uh sential cells     
13:52     
sential cells of a common cytoplasm and so you might have a bunch of different nuclei with a common     
13:59     
cytoplasm they usually have these intracellular Bridges so this is an example on the right of what they call     
14:06     
the ratches this is in seans and this is your shared cytoplasm so you know we     
14:12     
have here um this is from this paper U Goldstein uh where they talk about     
14:21     
this uh this result it's basically showing the ratches um where right here     
14:28     
where you have nuclei in a common cytoplasm and so this isn't just the     
14:35     
only type of Cal cell muscle cells are also or muscle tissues are also Cal     
14:41     
cells where you have multiple nuclei in a in a common cytoplasm so this is of     
14:46     
course you know results from Cell fusion of course you might have guessed and other types of processes that bring     
14:54     
these cells together and form these these uh cell wheer Bridges and other     
14:59     
types of ways to share cytoplasm so sential cells have a common cytoplasm     
15:06     
they have a sing they're single large cells in multiple nuclei as you see with the ratches and then efficient distribution     
15:12     
of cellular components so the cellular components are distributed within the     
15:17     
ential structure okay so now this is of course I said that self fusions were important     
15:24     
this is a paper from developmental biology self fusions in developing epithelia     
15:30     
epithelium of cgans uh this is where they talk about epithelial Fusion so we had some     
15:36     
epithelial cells in our um in our lineage tree and those epithelial cells     
15:42     
will fuse and they'll form different uh structures they'll have different     
15:47     
functions so in this case you know they kind of talk about how you see this on the hypodermis in hypodermis where you     
15:54     
see it in vulva uterus expiratory gland uh germ cell formation and Fingal     
16:01     
muscles so all these different types of tissue uh usually it's in the germline     
16:07     
OR in some of the other areas uh in in the worm and then of     
16:14     
course the frenal muscles the the fium is a structure towards the front of the     
16:20     
celegans where there's a ring that's formed and that ring has these fusions that result in these uh in these CTI     
16:29     
okay so then there's also something called epitheli aderin which are cell cell fusions and embryonic and     
16:35     
postembryonic development so these are adher EP epithelial adherin where you     
16:40     
get these cell cell fusions and this is of course in the embryo but also post     
16:46     
embon and then we have this uh so cells to do this they'll migrate around and     
16:53     
they will migrate from embryonic development to post embrionic development to the adult Phoenix type     
16:59     
which we know from you know when we see a lineage tree it's usually the     
17:04     
developmental sort of lineage and then you see the adult cell as the output but     
17:10     
actually there's a spec you know there's a significant amount of lineage tree     
17:16     
that comes out of postm branic development and you get a lot of cell migration and plasticity before you     
17:22     
reach adulthood so these cells will migrate they'll fuse in post embrionic development so the     
17:29     
final positions in the adult phenotype the boundaries and the sizes of these uh centium are all similar across organisms     
17:37     
so as you might have guessed in cgans we have this common um you know we kind of know where a cell will be in development     
17:44     
we kind of know where cells are in adulthood and there's no difference in     
17:51     
the centium they end up in these sort of the same positions as we might expect there those     
17:58     
also this issue of variation in the order of fusion events in lineage trees so if we look at a lineage tree you know     
18:08     
cells will migrate they'll start to fuse but there isn't like this deterministic     
18:13     
fusing order so one cell doesn't fuse to another cell and then another cell fuse to there's no recipe for the cells will     
18:21     
fuse but they'll fuse maybe in their own order or they'll they'll migrate around and they'll kind of fuse on their own     
18:27     
time there's no there's variation across seans in that regard um it's what's     
18:34     
interesting is that this process is initiated in the anterior portion of the embrya so towards the left hand side of     
18:41     
the lineage tree and then proceeds posteriorly or towards the right hand side of that lineage tree so that's an     
18:47     
interesting point and it may have to do with the tissues where the centium form     
18:53     
so it could be that you see it in the tissues that result from the ab sublineage first and then later on we     
18:59     
talked about how the germ line is at the um at the far right end of the lineage tree or the posterior end of the lineage     
19:05     
tree so that may have something to do with there also you know if we start to     
19:11     
think about modeling Cal SS we have these Dynamics we have to think about and this is a figure from this paper uh     
19:18     
where they talk about this membrane protein That's essential for developmental Cell fusion but they show     
19:24     
in this paper you know they show the adult here which is laying in egg the     
19:29     
egg forms an embryo and you see these cells which are differentiated     
19:34     
developmental cells so you have this developmental cell lineage it produces these differentiated cells so this is a     
19:40     
hypodermal cell six these are hypodermal cell s and then those cells will go on     
19:47     
to form these vulval rings and those cells will either fuse or they're     
19:53     
committed to they're unfused so some cells are committed to a fusion fate and     
19:58     
some cells are unfused or have that unfused fate so you can see in this structure as it forms in the vvo Rings     
20:04     
you go from the embryo and you have these canidate cells and then they form     
20:10     
these Rings these are toroidal structures and you can see that they stretch out like this and sometimes they     
20:16     
fuse and sometimes they don't and as we go through to The L4 stage of     
20:22     
um uh the the AL or laral stage we usually get this uh gradual fusion into     
20:30     
these vulvo rings and then we end up with this fic structure towards the anterior end of the animal so you see     
20:37     
this fering structure these are these Volvo rings in the fering structure they form the ring they not only go through     
20:44     
this differentiation process but they migrate and fuse to form this ring     
20:50     
that's how it works and so there are two important things here there's timing so we have to model the timing which is a     
20:56     
sequence of migration events see sequence of fusion events and then the fusion itself which is in the cell and     
21:04     
I'm just showing you what cartoon to the cell but in the cells you have this rearrangement of adherence Junctions and     
21:10     
its location in the anatomy in this case in the Volvo ring which one up in the anterior portion of the organism and you     
21:17     
can see that in the embryo where you get these Hy hypodermal 7 cells that need to     
21:24     
move over to near hyperthermal Six and form this ring     
21:29     
this is another paper from Bower the initial expansion of the sea elegance and tial germ line it's coupled to     
21:36     
incomplete primordial germ cell cyto canis so this is where the sential germ     
21:42     
line so in the germ line we have these sential structures that um basically have to do     
21:49     
I think with cyto canis so this is where we have this diagram we have this two cell centium at the left we have this     
21:57     
cyto kinetic ring you have the cell division event and then we have where they're kind of fused     
22:04     
together so you have two cell centium cyto Kinesis cyto kinetic ring     
22:10     
stabilization so the ring forms it stabilizes you get these uh sential     
22:16     
cells as a product of cell division incomplete cell division I guess um cyto     
22:22     
kinetic ring displacement and then cyto kinetic integration where you have a three cellon tissue so you get these     
22:28     
centium that grow in terms of their uh nuclear material or their nucleus so you     
22:34     
go from having two nuclei to three you get this incomplete cell division and     
22:39     
you maintain this centium throughout so this is an interesting structure I'm     
22:45     
never really familiar with this sort of uh they talk about the ger the germ cell     
22:51     
uh centium have been studied a lot so that's a model system within CL for     
22:57     
that uh uh then this is uh the same paper and this is just showing     
23:02     
difference you know we have the embryo here we have the L1 worm and the adult this this shows that there're these     
23:09     
different morphologies for each of these stages of life history and this shows kind of where these are through showing     
23:17     
this uh microscopy data here where you have these structures you can see it at a and c and then uh the row below it's     
23:26     
highlighted so it's color with false color uh you can see the structures     
23:31     
where the cells divide but they retain a a common cytoplasm but they have two uh     
23:39     
two nuclei and then they have one nucleus here so you can see like their differences in the number nuclei that     
23:46     
they all share this common cytoplasm okay um this paper is born and     
23:54     
Weiss this is an interesting paper because what they do in this paper is they take an embryo and they constrain     
24:01     
it artificially mechanically and they want to see if they can form sential     
24:06     
cells where it in a normal cigan embryo you wouldn't see sential cells so this     
24:13     
is uh born in was robust spatio tempor organization mitotic events in     
24:19     
mechanically perturbed seans embryos assist biophysical Journal last year this is the figure up here where they     
24:26     
show like these cell division events and they show the nuclei so this at the top is sort of the normal process of     
24:33     
development where you get these uh se you know get these membranes that separate out the cytoplasm between each     
24:40     
nucleus so you can see that this has four cells four nuclei actually this say five nuclei and you get basically one     
24:47     
nucleus per cell uh roughly in the experiments that they do they constrain     
24:53     
these embryos in terms of their size they introduce uh physical stresses and     
24:59     
they end up with these sential structures where you get multiple nuclei basically the nuclei divide but you end     
25:07     
up with a single cell or just a few cells where these nuclei share cytopl     
25:12     
and so that's basically what they're doing here so they do this volume compression of embryos that's their     
25:18     
perturbation that they introduced to sort of constrain the embryo so they do is they suppress cyto canis and it forms     
25:26     
the centium which is pregastric relation so gastation is where you get this first     
25:32     
uh these first pattern formation uh you know they get the first     
25:37     
pangs of pattern formation you say and so you know INE I guess in gastrulation     
25:42     
is when a lot of the centium start to appear in The Germ line so we're trying to induce this before we get any natural     
25:50     
sential so the sential state is still feature still features mitotic events     
25:55     
and multiple rounds of nucleus duplications as we saw in here so the nuclei divide it's just that the cell     
26:01     
membranes aren't dividing into individual compartments uh and then we have if we     
26:07     
look at the unperturbed embryos we can look at the lifetimes of sential nuclei     
26:12     
between successive anaphases which are these stages of cell division and these lifetimes increase with every generation     
26:20     
so we see that like you know there's this pattern that we can observe in the emperature of embryos that are a little     
26:27     
bit different than the ones we get in the perturbed embryos but we get this these nuclei that divide but not the     
26:36     
membranes so this is where they show some data for the artificial centium     
26:41     
which are the squeezed embryos versus somatic and then versus germ line cells so I mentioned that in the germ line     
26:47     
cells we see these centium of course this is pregastrulation but that's I mean that     
26:54     
would be the place where you would see them first so we have the centium here the sematic cells and the P1 which is     
27:00     
the origin so you can see here in the lineage tree you have P1 P2 P3 and then you get P4 and then that's where you get     
27:07     
like your Z cells which is this germ line and then all the other     
27:13     
U sematic cells are out here a EMS C and D so these are all kind of you know     
27:20     
these are all the sort of the cells that we're interested in outside of the germline this is the sort of the germline precursors and so you can see     
27:28     
that like this is in uh sort of this towel parameter in minutes and then for     
27:34     
each generation for each cell division basically we're looking at these three categories and we see that centium have     
27:41     
this increasing tow in terms of time and you know in P P1 P2 P3 P4 it     
27:49     
increases and and also increases in the centium and in somatic cells it kind of     
27:55     
increases but mostly the variance increases so yeah that's uh not you know without     
28:01     
getting into the paper too much that's some data that they show they have the uh they kind of work out some of this     
28:08     
these data and they just show kind of the trends between their artificial uh they artificially induced     
28:16     
stici and the sort of the precursors of normal CCI okay so then finally there's this     
28:24     
paper line of hand lesses on the Force form function connection cell biology for modeling a cal germ so     
28:32     
this is where they talk about this Force form function connection and so this Force form function connection is     
28:38     
basically where you have forces acting it results in form and then that results     
28:44     
in function and so there's this connection between those three things and so that's something that they're looking     
28:51     
at here it's kind of an extension of what people you know the same kind of similar technique they're using in the     
28:57     
last uh so their first point here is that CTI creates exceptionally large cells     
29:04     
dramatic enlargement of cell compartment to support morphogenesis so when you go to centium it's this very large cell     
29:11     
there's this dramatic enlargement of the cell compartment it's supporting morphogenesis because you have these     
29:18     
structures that kind of can result and you know you can have different I don't     
29:23     
know I guess what they're getting at here is that it can serve a function like a muscle     
29:28     
or something in the germ line it can also result in you know sort of the     
29:34     
precursors for Morag Genesis this is why we're looking prast relations of this     
29:40     
stuff um then we have intricate multi- compartment architectures where you have     
29:45     
this cortical contact cytoplasmic flows and germine extrinsic forces so you get     
29:51     
this sort of in in the centium you get this multicompartment architecture even though you have a single cell with     
29:59     
multiple nuclear and then you have this uh where the germ line which is one     
30:05     
example of stici is because they're modeling sential germal there's this deform deformation and displacement and     
30:12     
so this res this is sort of analogous to local and Global Force production so the     
30:18     
force production from the formation of the centium have local and Global     
30:23     
effects     
30:30     
okay so that's all I have for that let's take a look at our chat so we     
30:39     
have let's see um okay it's also spelled     
30:45     
centium also called Mosaic and then     
30:50     
uh mahul that was about like the idea of     
30:55     
sential cells the whole says this looks quite useful for the project um and he'll go through the worm     
31:03     
book and meeting recording thank you for attending the whole then dick says I     
31:08     
recently learned that some people regard all tissues as sential via Gap Junctions     
31:14     
despite each cell in a tissue having just one nucleus yeah I think that's an interesting point something I noticed     
31:19     
and I was reading through the uh the papers they talked about these uh sort     
31:27     
of con connections that form between cells and you saw that sort of in The Germ line where you kind of had this     
31:34     
connection there was like incomplete cell division and so you had these kind of blobs that had like a maybe a ring     
31:42     
Remnant you know and then a cytoplasmic ring and then you had like the a     
31:47     
division of the nuclei so the nuclei were kind of in their own semi compartment so I mean let me see if I     
31:53     
can draw this out you'll have a cell here let start on the cell     
31:59     
nucleus here's the cell then you'll get like this ring that forms you get these     
32:06     
nuclei that divide the ring is here and then the ring stops so you get this sort     
32:13     
of peanut-shaped centium with two nuclei and then maybe two more as that cell     
32:20     
divides you know now what's interesting is that you could also have a situation     
32:26     
where you have multiple cells like I said we can have multiple     
32:31     
developmental cells this okay they kind of migrate towards     
32:38     
one another then they form these connections so of course Gap Junctions     
32:44     
are these physical connections between cells and so you end up with these     
32:51     
intercellular Bridges which are kind of like you know I guess it could be Gap directions could     
32:58     
be other things too which you would end up with a centium technically it's all sharing the same cytoplasm but it's much     
33:04     
more compartmentalized with multiple nuclei and then these cells say it could     
33:09     
divide up like you know we saw in the last example where you would have something like     
33:15     
this this cell would would kind of incompletely divide with the ring here and then you have this shared     
33:24     
cytoplasm right like that and you have your your uh complement of nuclei and of     
33:30     
course we haven't talked about the organal which are in here so you have mitochondrian you have other things     
33:36     
mitochondri are interesting because look in muscle cells those are quite important right because you're trying to     
33:43     
produce Force you need to have ATP and all that so that's kind of my what I'm     
33:50     
thinking about for how how it was how this was     
33:55     
represented um in the literature and so at least explained to me I that's     
34:02     
what I was trying to struggle with was how to draw that out okay so it says each Gap Junction     
34:08     
only lets molecules through up to a maximum size yeah not all cytoplasms so yeah Gap Junctions are usually like     
34:16     
electrical connections in the sense that you know you have     
34:21     
like I guess maybe ions or something going through so you could have     
34:27     
something yeah you could have something like Gap Junctions as well yeah also D     
34:34     
molecule okay um yeah so that's good um any questions any other questions about     
34:40     
that so U yeah that I hope that was useful I might try to work out some of that more uh like I said with respect to     
34:47     
the lineage tree which was kind of the original question um I guess you could like pick     
34:54     
something and like look at the lineage tree CU we have inter the interactive lineage tree and see element as well so     
35:00     
you can go through that and look at like the data that they have on um I actually     
35:05     
had going into like some of the muscle CS would be kind of interesting um and what those kind of look like I suspect     
35:13     
that you know the way you would model it on the lineage tree or maybe like in a     
35:18     
differentiation tree is to kind of model the cell divisions and then model where they kind of come together just kind of     
35:25     
estimate where they come together and develop um I don't know if we have the timing     
35:30     
down in the embryo so or even in the post embrionic organism but yeah it' be     
35:36     
interesting to maybe even you know create some simulations where you say these cells fuse into a centium here     
35:45     
what does that look like well anyways that's um that's CTI I'll talk about     
35:51     
this paper um to finish off uh this is     
35:56     
uh from the Journal of physics a this is not uh developmental biology     
36:02     
so much as it is this idea of lassy and thermostats so this is kind of like     
36:08     
thinking about Collective behavior and some Physics so this is going to be a little bit difficult to     
36:15     
get through I think but we'll see how we do um so this is a paper discrete lassy     
36:23     
and therat for flocks and swarms the fully conserved inertial spin     
36:28     
so this is by group physicists and they're going to talk about uh swarms in the sense of bird     
36:36     
flocks and other types of swarms we see in insects and other organisms that     
36:42     
exhibit Collective Behavior so the abstract reads experiments on bird flocks and midg     
36:48     
swarms reveal that these natural systems are well described by an active theory     
36:55     
in which conservation laws play a crucial role and so we you know we've talked about like soft soft active materials and the     
37:02     
role in embryogenesis but also the role of those kind of theories in describing     
37:08     
Collective behaviors and uh emergence so this is kind of you know it has its it's     
37:15     
kind of a physics paper but it'll be relevant to all those areas by building     
37:20     
a simplec structure that couples the particles velocities to the generator of their inertial rotations or spin the     
37:28     
inertial spin model or ISM reinstates a second order temporal dynamics that     
37:33     
captures many phenomenological traits of flocks and swamps so this is a spin     
37:38     
model um so it's like spin glass we talked about spin glasses uh and this is     
37:44     
kind of similar to that so they're basically looking at uh particle velocities and then the     
37:52     
spin of these uh these elements and so     
37:57     
they're trying to do is they're trying to capture the properties of flocks and swarms and their behavior and model them     
38:04     
with this this inertial spin the reversible structure of the ism     
38:09     
predicts that the total spin is a constant of motion the central conservation La responsible for all the     
38:16     
novel dynamical features of the model so I think um in our Saturday meeting which     
38:22     
is um it's a separate meeting but we kind of talk about things that overlap with this meeting um we I think it was     
38:31     
Morgan and myself we were talking about emergence and how in emergence you know     
38:37     
a lot of times you'll like say take a swarm of birds where they're flocking     
38:42     
and they exhibit Collective behaviors they exhib you know the flock has     
38:47     
different patterns and the individual birds you know don't engineer that from sort of uh a master plan they kind of     
38:55     
just interact and the the pattern forms from the interactions and that's emergence and one of the properties of     
39:02     
emergence is where you can't reverse engineer an emergent flock if a flock     
39:08     
forms a pattern a coherent pattern that pattern is a p product of their interactions and you can't reverse     
39:15     
engineer that to any one set of birds doing any one set of things at any specific time those patterns happen sort     
39:22     
of as a consequence of gross interactions so there are a lot of different ways to get to that pattern     
39:28     
and so this is kind of interesting that there's this reversible structure of this internal spin model so this is um     
39:36     
nice because you can have this model and then on Saturday we talked about another way that people were doing this to to     
39:43     
get the sort of reversibility um however fluctuations and dissipation     
39:49     
introduced in the original model to make it relax violate the spin conservation     
39:55     
so that the ism aligns at the physical phenomenology only within finite size     
40:01     
regimes Beyond which the overdamped characteristics of the V model takes     
40:07     
over here we introduce a novel version of the ism in which the irreversible terms     
40:15     
needed to relax the Dynamics strictly respect conservation ofis we perform a     
40:21     
numerical investigation of the fully conservative model exploring both the fixed Network case with which belongs to     
40:27     
the equilibrium class of model G in the active case characterized by self-propulsion of the agents and an out     
40:35     
of equilibri reshuffling of the underlying interaction our simulations uh not only     
40:41     
capture the correct spin wave phenomenology of the ordered phase but they also yield dynamical critical     
40:48     
exponents in the near ordering phase that agree very well with the theoretical predictions we make so this     
40:55     
is where they talk about this little and thermostat so I'm going to get into that in a minute I don't know there might be     
41:02     
some items in chat here so this is where do Words Spin while     
41:08     
flying that might be a good question I don't know     
41:17     
um so this is the introduction here um so approximately three decades ago     
41:25     
this paper was from 2024 so probably in the 90s bit at all     
41:32     
and listed a sizable number of unwilling statistical physicist into the fast gring field of biophysics by introducing     
41:39     
a simple Widow model that elegantly captured the key traits of collective animal behav this model partly rooted in     
41:46     
the physics of ferromagnets and critical phenomenon partly drawing from inspiration from previous efforts in the     
41:52     
field of computer Graphics so they're referring to spin glasses and then also     
41:58     
uh Boyds which was of course the paper by Reynolds this is I think that's what     
42:04     
this citation is yeah Reynolds Boyds and then this     
42:09     
paper by vit I think this is the one of the original spin glass paper so an early iteration     
42:20     
this okay um these two bringing these two things together demonstrated that     
42:27     
local alignment could induce Collective motion and Long Bridge correlations even     
42:32     
in active systems so we have this long range correlation which is where you     
42:37     
know you have correlations between something say in the posterior part of the embryo and the anterior part of the     
42:45     
or something in the front of the flock and the back of the flock usually you know if you have something that isn't     
42:51     
sort of this emergent Collective Behavior you might have like every bird     
42:57     
doing a random walk or every bird in it for themselves and you can imagine how chaotic that would look from the outside     
43:04     
they' just be kind of like it would kind of be like static on a television set or     
43:09     
like this just this sort of uh gurgling percolating mass of particles and so     
43:17     
this long that's that's where you have no correlation or maybe very short range correlations long range correlations are     
43:24     
where the things that are happening at the back of the flock affect things that are happening in the front of the flock     
43:31     
or things that are happening anteriorly also happen posteriorly and there's a     
43:37     
correlation so in equilibrium physics it really prevents long range ordering     
43:43     
under more than one respect but not all respects this was the start of active matter physics of course we've talked     
43:48     
about the stuff that de started in the 60s and then moving forward to soft     
43:55     
active materials and some of these other areas active matter physics where you     
44:01     
know people are doing things with development biology but also with Material Science and even using their     
44:08     
models in complexity V's models are truly remarkable for their Simplicity and     
44:15     
great generality so much so that they can be cons they are considered the iing model of active matter so the vit model     
44:22     
is the uh sort of the izing spin glass model of active matter or the iing spin     
44:27     
glass model of course is where you have ferromagnets that spin and you measure the spin and then there's this you know     
44:33     
you can build networks that are have this long exhibit this long range correlation but in active matter you     
44:39     
need to treat it a little bit differently hence the vid however as soon as experimental data     
44:45     
on actual biological groups started to flow in it was clear that the quantitative description of certain     
44:50     
biological features required models to incorporate some additional elements so     
44:57     
among the many variants and evolutions of the vit model aimed at better describing     
45:02     
biological one particular interest uh one one area of particular     
45:07     
interest for this paper is this a nural spin model or ISM so if you look at Starling flocks     
45:15     
you see that the collective behavior of Starling flocks especially during turns     
45:20     
if you've ever watch bird flocks you know they're they're kind of coordinated in their movement they're all kind of     
45:26     
moving in the same Direction and then they make these really complex turns and that's really where the coordination     
45:32     
comes becomes apparent because they make these turns and they're kind of these really complex waves almost that they     
45:40     
form so as they follow each other around these turns they form these waves and     
45:45     
but they maintain their coherence so that's really the thing that's really interesting to think about staring     
45:51     
flocks for a lot of biophysicists is that you get this long range correlation you get this sort of stability even     
45:59     
through this sort of perturbation and you don't see a lot of damping so you don't see a lot of decay     
46:06     
of these flock formations even when they're making these turns not much later experiments on natural swarms of     
46:13     
midges show that the dynamical correlation functions display unmistakable traits of inertial     
46:20     
relaxation uh so this is where they're talking about these natural swarms and midges which is a different system um     
46:28     
and they have this inertial relaxation so you can see the physical parameter in these natural systems um you can measure     
46:35     
it you can model it at the same time swarms have a dynamical critical exponent lower than any other reference     
46:42     
pherom magnetic model so these animal swarms these swarms of birds and insects     
46:48     
they have these you know you can actually compare them to materials in ferromagnets and so this is a lot lower     
46:55     
than the ferromagnetic model and even the viit model which is actually for soft active materials so     
47:02     
this is interesting because you have these natural systems that you can model with this with this uh you know you can     
47:09     
approximate it with this model you can approximate the behaviors quite well but they behave much differently than say     
47:16     
something like a material something that's sometimes inorganic sometimes you're organic but they don't have the     
47:23     
sort of uh maybe like cognitive compon individual cognitive components um in     
47:30     
any case all these trades prompt the development of a new model in which the inertial features of the Dynamics which     
47:37     
in the vit check model are washed out and and averaged out or reinstated given rise giving rise to     
47:44     
conservation laws able to account for several new biological pieces of phenomenology this model was the     
47:51     
ISM developed in further study and for references here the overdamped limit of     
47:57     
which notably is the V the ism model this uh model that     
48:03     
they're using successfully reproduces several traits of natural flocks and swarms including a the winar     
48:10     
propagational orientational waves and bird flocks so this is where you have these bird flocks and move in a certain     
48:17     
direction follow curves and turns and they can maintain their stability even in density homogeneous     
48:24     
systems which are you know uh different types of bird flocks uh the quantitative     
48:31     
dependence so two the quantitative dependence of the speed of propagation of these waves on     
48:36     
polarization three the precise inertial shape of the dynamical correlation functions in mid swarms so you have this     
48:44     
shape and then the shape gets maintained as they kind of move around may we move     
48:50     
around barriers or change direction so you would expect some inertia there     
48:55     
because of the physics but you don't NE necessarily see that so that means that they're doing they're making a lot of     
49:00     
Corrections sort of maybe before they even go into the turn or encounter these perations they can really predict maybe     
49:08     
what's happening at the front of the flock so it doesn't affect the back of the flock you also have Dynamic scaling laws     
49:16     
and then finally the precise prediction of the dynamical critical exponent in natural swarms to our knowledge the     
49:22     
first case of agreement between Theory experiments and new numerics on a critical exponent of an actual     
49:29     
biological system so this is um you know they have a lot of of course     
49:36     
they have a lot of equations here this is the inertial spin model rule so the     
49:41     
key idea of the ism compared to the vit model is that the social alignment Force     
49:47     
which is the force that aligns individual birds and insects or whatever     
49:52     
instead of directly rotating the velocity of the agent X on its spin     
49:58     
which is the generator of the rotations of velocity in turn spin rotates the     
50:04     
velocity according to these equs so there's this relationship between velocity and spin which is     
50:11     
considered to be the same as between position and angular momentum and a ferromagnet so they don't actually     
50:18     
rotate they're just like approximating the velocity and sort of the     
50:24     
orientation of the of the bird and then this corresponds to these     
50:29     
different parameters position and angular momentum when rotations in the external space are     
50:35     
considered so the birds will actually change kind of their orientation as they're moving like they're pitch and     
50:41     
yaw you know as or insects will do this as well sometimes from this analogy we clearly see that the     
50:47     
parameter this parameter here act as a generalized inertia social or     
50:54     
physiological resistance of the agents to rotate their direction of motion so     
50:59     
usually we assume in a lot of swarm or flocking cases that you know birds will     
51:06     
operate on Simple Rules so the boids model of Reynolds for example it operates on these simple rules you don't     
51:12     
get too close to your neighbor you don't get too far away you follow their you know directional change and those those     
51:20     
are very simple social cues that organisms but and this could work with     
51:25     
cell migration as well but it you know it works they're very simple rules so there's this     
51:32     
resistance it's social it could be physiological as well where they rotate in the direction their direction of     
51:37     
motion so this is where as long as the interaction Matrix Matrix is symmetric     
51:44     
the equation of motion for the velocity can also be derived from a Time dependent lran so we can maintain this     
51:50     
sort of we can sort of have this control over the inertial aspects we have this     
51:57     
uh so this is sort of the inertial which is uh set by these social and     
52:02     
physiological resistance factors and then of course we have because of this     
52:09     
and if the interaction Matrix is symmetric we can use a Loni a Time dependent liy for the crucial point is     
52:17     
if the interaction is symmetric under a given transformation Global     
52:22     
conservational this is the case of the alignment interaction the social Force f subi is invariant     
52:29     
under the rotation of velocity so that once a simplec structure velocity and spin is restored the generator of     
52:36     
rotations namely the total spin of the system absence of dissipative forces     
52:42     
constant emotion so this is where you have this conservation law the ordered     
52:48     
spas of the ism it is the cause of the emergence of linear spin which is observed or spin waves observed in     
52:55     
natural biological so they get go through some more of the     
53:00     
math they kind of model the stochastic components this conservative fluctuation dissipation as as you can see there is a     
53:07     
lot of physics involved and these alignments with a biological system and     
53:13     
so we end up with this time dependent lran which is the discrete lassan so we have this discrete lassan operator which     
53:20     
is here and this is where remodel system and then we have this     
53:27     
from noise on the SES to noise on the lnks so kind of talks     
53:32     
about making this creating this network of a bird set of bird uh interactions     
53:38     
which result in flocks you have three birds here I J and K the connections A and B so they form this network and the     
53:46     
network is between individual birds that are may be observing each other or     
53:51     
responding to each other in this inertial way you have this adjacency Matrix of the s Network so we have for I     
53:59     
and J or I and J interacting if see yes one if so no we have a zero so obviously     
54:07     
I doesn't interact with I and so forth so we can model these networks of interaction as adjacency matrices then     
54:14     
we have the discrete lassan which kind of makes us into another you know transforms us into a different set of     
54:20     
values and then this incident Matrix so you can see that you have the adjacency     
54:25     
Matrix the incident Matrix and the discrete lassan which form this very simple Network and this then of course     
54:34     
can be plugged into this discrete laian operator here for the discrete     
54:39     
laosan okay I think that's it for this paper so um any questions about that paper we have a thing in the chat here     
54:48     
so okay dick asked another question similar similarity to eising models from     
54:53     
Atomic spins yeah I think I talked about the we talked about that the ising model     
54:59     
this is a CA a specific case of the eing model where they account for some of the     
55:05     
inertial Dynamics looks like we have dantu and Sham or soam and Leith so we     
55:13     
have some anat so we have like total turnover meeting we had one group of people on     
55:20     
the other group people okay so um I don't know if anath or Devon or soam or     
55:25     
waya that anything sir uh I have a small vague uh idea of     
55:32     
uh certain things I would like to discuss about okay so so uh I would like to share my screen     
55:41     
and just give a short presentation     
55:47     
of yeah     
55:57     
[Music] so yeah so I'm not pretty much sure     
56:02     
about anything here in detail but uh I just wanted to let know like what I'm     
56:09     
thinking of currently does it like really is a good fit for the you know this gck or not     
56:17     
so yeah so so first I've been studying this uh there is a work in the nature     
56:25     
Communications I got the data that is related to embryogenesis of calans so it     
56:33     
was for like 17 subjects and for each subject There Was You Know start of this     
56:39     
video I mean like uh the movie from like uh you know the start of the like how     
56:46     
the egg was uh you know started and from there how it you know uh developed into     
56:52     
a a fully uh adult warm and during this     
56:58     
process it they analyzed using this principle where the cell membrane     
57:05     
delineation they have given this red with uh you know certain uh type of you     
57:11     
know uh I mean like know this thing and uh the nucleus was like uh know coded     
57:17     
into green so I think this is uh uh I mean like generally the dying process     
57:24     
doing we add Dy and based on that this florense mapping is provided so they     
57:29     
have given uh a proper data the the data for like all these files and uh what I     
57:38     
was thinking is so based on this data so so first uh we can build a basic     
57:45     
lineage tree so when we do not have uh I mean like when we do not have where exactly the cell locations are so the     
57:52     
lineage tree has no where it evolved I mean like exactly the pattern the     
57:58     
pattern it the way the seans the each and every I mean like nucleus have been     
58:03     
evolved into a fully grown uh you know the I mean like adult we we cannot know     
58:11     
but whereas with the data of the exact location 3D locations then we might know like what it has to do and then we can I     
58:19     
mean like if we have like realtime nucleus data where exactly it is located we can update this lineage accordingly     
58:25     
so I wanted to show so suppose this is a lineage tree where the zygote is here     
58:31     
here and from there it was you know I mean like uh developed slowly and it has     
58:39     
been you know each and mutated and later so we have proper for like four you know     
58:47     
the four nuclei division this looks like this so this is an example where like four nuclear division so we can map you     
58:53     
can find the distance between the each nuclei to nuclei and uh the images that     
58:59     
were previously used uh cannot give the accurate decision of the cell boundary     
59:05     
because cell boundaries are also important because the way the membrane is structured based on that we can     
59:10     
determine certain characteristics uh such as uh mean uh the Fate     
59:17     
determination and the growth and the division patterns so so the I mean like     
59:24     
so we can simply localize this nucleus based on you know uh certain standard     
59:29     
deep learning methodologies and based on that we can map back to the the lineage     
59:34     
trees and yeah then later on we can uh I mean like build by as good visualization     
59:41     
tool determining the you know this application such as self F determination growth division patterns and probably     
59:49     
spal temporal uh lineage tracking so we can do that I mean like I thought     
59:56     
thought like you know this could be a good start off I mean like I mean this     
1:00:01     
looks good to me I mean like I like I mean like I've been seeing the data data is like rich and uh and well uh I mean     
1:00:10     
like uh uh it is well structured so I am     
1:00:16     
thinking that you know we can apply different models into this space suppose we wanted to see how the lineage evolves     
1:00:22     
we can use uh I mean like hypergraphs exactly into the space where we can fit the coordinates and uh design     
1:00:29     
accordingly I mean so I believe uh this data could be like useful in this in     
1:00:36     
this I mean like uh I mean like exact space so I mean like so this is what     
1:00:43     
currently I am thinking sir I mean that's it I mean like I wanted to make it a more detailed chis presentation but     
1:00:50     
uh I I probably need little more time for that I think so yes     
1:00:57     
s that'ss good uh thank you for going over that a little bit Yeah I think that's good I did talk     
1:01:06     
about the wi earlier in the meeting but yes we basically have that so I think     
1:01:13     
with a lot of data sets that you would find you'd have you know with with like microscopy data set you have to annotate     
1:01:20     
the cell or find they can be annotated but once you have that then you can     
1:01:25     
follow those El through through their process you know division so yeah I mean that would work     
1:01:33     
um and I think hypergraphs of course you can use that um I don't know if you read     
1:01:38     
the preprint I don't know if it's posted in the the slack Channel but there's a a     
1:01:46     
preprint that u i developed uh originally and then uh we're working on     
1:01:52     
it last year but this is where I kind of lay out a conceptual model of how to think about     
1:02:00     
hper graphs and lineage trees so I'll send you a copy of that or make sure     
1:02:05     
it's opt in the slack yes sir I'll probably look into it uh in a detail and uh I'll see like how     
1:02:12     
this can be connected into that space yeah yeah because I mean you know that's     
1:02:18     
like kind of a conceptual model we can like use uh you know different data sets to     
1:02:24     
kind of figure the idea basically the idea is you have     
1:02:29     
like as we saw with the lineage Tre we have sub lineages where we have maybe     
1:02:34     
like uh different types of tissues that form and those would form the basis for     
1:02:40     
the hyper hyper noodes so hypernodes would basically be categories so like if     
1:02:45     
you had cells that would become neurons we know this is the we could put all     
1:02:51     
those cells in the neuron category which is hyper and then those cells sometimes     
1:02:56     
cells move to another fate sometimes they die sometimes you get you know cell     
1:03:02     
doubling which means you get more cells their descendant cells are still in that     
1:03:08     
category or not so you have certain number of cells in a category then you move to other     
1:03:14     
categories uh from you know from developmental cells say which are a different category and we can build a     
1:03:20     
network like that where it's like in time it evolves in time but it also has a fun fun aspect where you have     
1:03:28     
different cell types or tissue types that are sort of defined so a cell     
1:03:33     
belongs to this category the category is a hyper node     
1:03:38     
the cells are nodes and then they connect other cells and other categories     
1:03:43     
of hyper nodes so there's a hyper Edge those hyper edges then are analogous the     
1:03:51     
hyper edges we find in a hyper graph we can do all sorts things of that because     
1:03:56     
that's basically how a lineage tree is organized it's organized by cell division but then there's also this     
1:04:02     
functional aspect what do the cells going to become or doing right now or     
1:04:07     
you know where what are they like a lot of times in mammals you have this these multi cells where they can become or     
1:04:14     
what they call U Flur poent where they can become different typ types of cells     
1:04:21     
sometimes they can become a range of muscle cells or range of neurons we don't know until they actually     
1:04:27     
did but putting these categories is useful because it can tell us from point     
1:04:33     
what they're going what their way to be so it's kind of an interesting way to do the     
1:04:38     
representation anyways I'll send you the I'll make sure I re up the paper through and yes sir yeah see how that matches     
1:04:47     
your ideas     
1:04:52     
yes okay sir     
1:04:58     
yeah hi Bradley am I AUD yeah yeah hey um I'm sorry I didn't know     
1:05:04     
the meet time was shifted so I joined a bit late um I wanted to share my idea regarding the project I uh I was     
1:05:11     
thinking on so I've divided into three parts um the first one it takes up the     
1:05:17     
growing neural network work that we have done and I was thinking if we could extend it uh to a hyper uh like we have     
1:05:26     
a graph uh Network which is growing if you could extend it to a hyper graph um     
1:05:32     
with the same analogies maybe then we could capture some more relations between the cells um the second part was     
1:05:40     
um the work that is done till now has been specifically focused on C Elegance     
1:05:45     
so if we could turn uh that work to a generalized Library which could cater to     
1:05:51     
any kind of data um that will be a more uh reproducible work book and third um     
1:05:57     
I'm not sure if I will be having enough time but if time permits I was thinking to um tap into the visualization part of     
1:06:06     
the graphs if that's something I could improve yeah yeah I think we talked     
1:06:13     
about this in the slack yeah the first part yeah I think     
1:06:18     
that's growing ear that works that you know that was something that was sped foral development of programs     
1:06:26     
and that's you know it's definitely needs more work in terms of making sure that that can recapitulate our lineage     
1:06:33     
tree right like so we want to make sure that we have like every time s     
1:06:38     
divides two new locations that graph ising nodes     
1:06:45     
and and then you know that's something of course we can extend to a hyper graph     
1:06:50     
where we have these functional categories or we could have you know you     
1:06:55     
don't know things within the cell that there there are different ways you can represent it they probably try like     
1:07:03     
maybe like in the lineage tree we talked about this where you have you have     
1:07:09     
subline so like in C you have1 you have a bunch     
1:07:18     
of just using those as categories as subar would help us with like in terms     
1:07:24     
of visualizing the growing gra so you'd have like a bunch of suine     
1:07:32     
form so you have new hyper nodes form when you get a sub then you have nodes     
1:07:39     
within the Hy your hyper node complement increases with division you have these     
1:07:46     
hyper form and would stabilize then the hyper edges     
1:07:52     
would increase you can see it like that um which would be good in celegans     
1:07:57     
because it's just a new way to visualize and then we can do other things from that um yeah I think     
1:08:06     
expanding to beyonc elegans is always useful so we don't you know the reason     
1:08:11     
why we don't do that as often as we should is because it's extremely hard     
1:08:17     
all you know even if you go to see Allan sister species um you know like     
1:08:27     
that's another that's a very different animal literally they're you know     
1:08:34     
lineage tree is different there a different feat so it's hard to kind of build a universal model but um we could     
1:08:44     
do that and I think it's maybe Beyond uh what we could do in Gs maybe     
1:08:49     
we could build like a more General doing so we don't restrict it just a c into like a system where we     
1:08:57     
have like a growing network but something where you can assign the hyper node to some identity and then the you     
1:09:06     
know the nodes are clearly cells but then the hyper nodes are may be categories that you can just assign     
1:09:12     
anything and then you know assign the nodes to some cells or cell     
1:09:17     
division and then that would be like because every organism seems to have     
1:09:23     
those so that that would be     
1:09:29     
then finally visualization so yeah expanding the seans you know we want to be careful we don't get in our heads we     
1:09:36     
want to make sure that we have the sort of General model we definitely then visualization of course is probably the     
1:09:42     
better option in terms of a g project of the sum like you know we have the     
1:09:49     
topological data analysis stuff and that's you know kind of vague terms of     
1:09:55     
how to view it you know we have like you can view pretty pictures like you know everyone puts     
1:10:02     
like a umap in their paper but it's hard to interpret it so we have um this     
1:10:08     
opportunity to make these visualizations that can give you insight into what you're doing so that's where     
1:10:14     
you know again like visualizing the hyper nodes     
1:10:20     
comp and then how they and then we have these layouts and like I don't know what     
1:10:26     
the layout options are for hypergraphs because those are a little bit different than the ones I was talking about     
1:10:31     
earlier in the meeting for the geffy plugins because those are just for regular graphs but that's something you     
1:10:38     
might explore some of the geffy plugins some of the stuff for like Java's Network Tools or     
1:10:47     
Python's Network Tools might explore in that     
1:10:52     
as um so should I try to do all three of them are like prioritize on two maybe oh     
1:10:59     
I'd probably prioritize the growing neural networks and then extending that hyper graphs and then the visualization     
1:11:06     
Stu then the expanding Beyond Sean might be something for later you work out your     
1:11:12     
methodology from one two will     
1:11:17     
easier okay so put that in the if time permits section     
1:11:22     
yes okay okay yeah so this is uh so this is yeah     
1:11:29     
this is the preprint this is on the archive this is the one that U this is the     
1:11:35     
version kind of the older version there's a newer version we haven't put it on the archive yet um this is with     
1:11:41     
mul and I'm co-authoring those but basically what I was talking about were     
1:11:47     
these categories and then this is the lineage tree here so this is a hypergraph of a     
1:11:54     
early lineage tree it's not seans but it's very similar to seans it just works     
1:12:00     
out the idea so what we have here is we have the one cell and then we get cell division and we get this uh basically we     
1:12:07     
have the single that are uh in this case these Bray ones are developmental     
1:12:14     
somatic cells and so there's a cell division event you go from one to two     
1:12:19     
two4 then 74 28 because then you get this new category which is the germ line     
1:12:25     
which starts at one and goes to 24 6 12 so you get these um hyper noodes that     
1:12:31     
grow in terms of their size how many nodes are within it and which each with each doubling event we either can have a     
1:12:40     
new category be born some of the cells assigned to that category or we grow our     
1:12:46     
original category by a certain number so that's what that's why these numbers increase and then eventually we get to     
1:12:53     
these differentiated cells like the connectome which is this white these white categories and again we have some     
1:13:01     
cell when this 28 here uh this gray Circle 28 in it you have a doubling     
1:13:10     
event which is doubling from 28 to 51     
1:13:15     
well it's not all doubling but it's basically going from 28 to 51 48 of those are still one     
1:13:23     
differentiated and three different differentiate in neurons so you get this like you get this doubling event and     
1:13:29     
then you assign all the cells from this 28 times roughly two you can have I     
1:13:36     
think the idea is you're going to have asymmetric divisions and then you assign those cells so if you're looking at an     
1:13:43     
ember or you're looking at a lineage tree you get these events where the     
1:13:48     
cells divide and then you have to assign them to either the same uh continuity of     
1:13:54     
the same um hyper node category or a different hyper node category and then of course     
1:14:01     
cells can move from one category to another one hyper node to another uh over time and then of course you have     
1:14:09     
connections between them so if you are looking at like their proximity or some     
1:14:14     
signaling you know you're going to have hyper edges which go between two hyper     
1:14:19     
noes and you're going to also have edges within that node so that's kind of how     
1:14:25     
this is supposed to be and then there's another figure in here which is where we add a spatial component so we add in the     
1:14:32     
spatial information of the lineage tree as well and we look at that over time so     
1:14:38     
we have from 128 C to 256 embrio these     
1:14:43     
we have more of these hyper nodes because they're more categories and it's organized in terms     
1:14:51     
of two spatial Dimensions but that's something and again we could do that with C Elegance     
1:14:57     
because we have the spatial data but um you know I would keep it     
1:15:03     
simple to start with and just try a lineage tree I think that probably working that out would be like a whole     
1:15:12     
sumar sir I have a small question so the given uh I mean like uh     
1:15:20     
the way the lineage trade develops so we are trying to model it exactly I mean     
1:15:27     
like using hyper graphs and hypernodes so so I mean like uh when uh suppose     
1:15:34     
like there is in during the developmental process if uh I mean like a parent from the parent it split into     
1:15:40     
like multiple Childs and these all do belong to a certain hyper Edge so and     
1:15:46     
from that we are I mean like are we modeling it in a similar approach like the hypergraph does I mean like it has     
1:15:52     
multiple I mean like we have hyper Edge based on the given multiple nodes and this Edge is associated with a similar     
1:15:58     
weight that is associated with certain number of nodes and it and the lineage evolves based on the the I mean like the     
1:16:05     
hyper graph structure so I mean like so     
1:16:10     
so I mean like while it's evolving I mean like based on the lineage tree so each and every node that is associated     
1:16:17     
with individual cell and uh and the spatial location of that cell so we     
1:16:24     
model the similar I mean like uh suppose we have initi in first it divides into     
1:16:30     
two cells and the two divides into like two I mean like four or eight or certain     
1:16:35     
pattern so these all do belongs to a hyperedge So based on like the way it's growing so is it like that or I mean     
1:16:43     
yeah yeah so a single cell will divide into two cells those two cells will have     
1:16:49     
their own divisions of four cells and so the edges are um you know you have edges     
1:16:55     
that are retained from the old case because the two new cells will maybe be     
1:17:01     
in similar location or they all right so let's say we have a cell here gives rise     
1:17:08     
to two daughter cells and then this gives rise to two four     
1:17:15     
cells now we have another cell over here so in a hyper in a a graph relationship     
1:17:22     
we have two cell connected those cells divide daughter cells so it's a growing     
1:17:30     
Network we had two cells here now we have four cells these might retain their     
1:17:36     
connectivity this and then if we think about it as a     
1:17:41     
hyper graph you know both of these might be in the same     
1:17:47     
hyper node one hyper node that's unconnected to anything     
1:17:53     
else and then here here we have maybe two hypernodes     
1:17:59     
now which have the daughter cell so these two cells are in this hyper these     
1:18:06     
would be a these two nodes would be in this hypernode     
1:18:12     
b b and so there we have now a new hyper     
1:18:17     
node with nodes within them and you're connected through these hyper     
1:18:22     
edges here right of course these could be connected     
1:18:28     
as well yes yes yeah so we just grow in that manner you don't always have to     
1:18:33     
have a new hypernode but right yeah do you see what I'm saying like     
1:18:40     
each of these cells is assigned to a hyper this is a     
1:18:46     
a12 this is A1 A2 B1 B2 so each node will change its     
1:18:55     
each daughter will change his assignment it might ret its assignment or having an     
1:19:00     
assignment from the mother yes     
1:19:06     
yes yes sir thank     
1:19:13     
you right so any other questions before we go or yeah okay so I had like two main     
1:19:22     
directions for this year's gck ideas so the uh one idea was regarding the     
1:19:29     
growing graft new network implementation that we have right now the one thing that I noticed was it was more leaning     
1:19:36     
towards the uh visualization like essentially optimizing how the uh     
1:19:42     
overall graph is structured with the data that we have so it's not essentially like predicting     
1:19:48     
anything new or doing anything new so I was thinking of like the and coming to     
1:19:56     
that I was thinking of coming up with something similar to an ensemble of models to like given a     
1:20:03     
starting uh number of parameters maybe let the network grow and figure out how     
1:20:10     
it actually evolves so for that there were actually I was actually going through the uh note features that I     
1:20:15     
might be considering right now the data set that is there in the uh repository it has spatial temp plal information and     
1:20:25     
so that's already there and the other things what I was thinking was maybe like cell tracking cell cell interaction     
1:20:31     
and the uh the Tre as a whole but like we were we did talk about this I believe     
1:20:38     
last week and you didn't mention that there aren't uh many data sets that are     
1:20:43     
good for this uh which has all the data so we might have to end up stitching     
1:20:48     
together data sets to get a complete uh idea so and for that I     
1:20:55     
actually went looking into the riken data set uh like from Japan let me just     
1:21:03     
share yeah so in that in that set of data sets as well there were like a     
1:21:08     
couple of issues one being that most of the data sets is just one single uh uh     
1:21:14     
entity or one single sample that's being studied so you you can't essentially trade a new model on it so that's one     
1:21:23     
thing I was thinking was I'm not like I'm not completely sure of how feasible     
1:21:28     
this is because again I'll have to be coming through a lot of data sets if we like go through the stitching together     
1:21:34     
data idea the uh other idea that I had was leaning more towards the actual     
1:21:42     
visualization of the graphs and to try and create uh a UI or front end using     
1:21:48     
electronjs or something similar uh so that you have a bit more control over     
1:21:53     
how the graph is also for example you could like let the graph or let the     
1:21:58     
model optimize how the graph is being displayed for a bit then you can pause     
1:22:04     
the model and then make changes in the UI itself give new uh like maybe add new     
1:22:11     
cells or give new uh positions to the cells and see how that evolves so I     
1:22:17     
believe that would actually be uh interesting for like the visualization aspect so those were the two ideas I had     
1:22:24     
had yeah any comments on it yeah so the     
1:22:30     
first one about like uh having a visualization of the lineage tree and     
1:22:36     
some of that last week I talked about the DVA learn repository and some of that was like     
1:22:43     
some of those some of that work has visualizations of the lineage Tree in it     
1:22:50     
so I'm I have uh some of the work that was done in previous GX was pushed to     
1:22:56     
dvo learn and some of it is in other repositories which I can point you to I'll have to go look at it but I     
1:23:03     
remember in years past people have generated a lot of either benchmarks or     
1:23:09     
uh output workout put where they've like taken the uh sublineages of the lineage     
1:23:16     
tree and they plotted them out in certain ways and they like extrapolated the data and those are available um I'll     
1:23:23     
have to find find them and show them but that might be useful for the first part     
1:23:29     
um we could build like a you know the the thing about like training a modelto     
1:23:35     
on SE allans and you mentioned the uh the work from the data sets of riken     
1:23:41     
which are actually from published papers but they have a repository um that most of the you go     
1:23:49     
across seans the lineage tree is mostly stable I mean there isn't a lot of of     
1:23:54     
variation it's not really interest biologically so really if I take a     
1:24:00     
single celegans and I and those data that you mentioned also aren't just a single Worm but like it's basically you     
1:24:07     
get you know the tree so the lineage tree is you know has these certain     
1:24:14     
properties it's actually a pretty simple lineage tree comparatively speaking compared to other types of organisms but     
1:24:21     
it does have the structure we can understand and we do have the data that can build like a model of the lineage     
1:24:27     
tree so one of the things we can do is we can vary that lineage Tree in different ways we can build a continuous     
1:24:34     
model of a lineage tree and I think we have some modeling exercises in some of     
1:24:39     
the previous here G repositories which are not in DVA learn but I'll have to     
1:24:45     
bring them up and and you know kind of maybe I'll present bring them up next week so you can see them or I can send     
1:24:52     
them to you you can look them over but I think we can do that and it you know that that's sort of these things     
1:24:58     
where they normalize their data they produce these cool graphs but then it     
1:25:03     
doesn't really do anything beyond so that would be a cool you know kind of a nice thing to kind of take that and run     
1:25:11     
with it and see what you can do uh maybe we can because we've talked before about     
1:25:16     
simulating lineage trees and we haven't done that in a way that was like     
1:25:22     
something we could publish or something we could like see anything so then that     
1:25:28     
would be an interesting thing and then I think the the visualization of graphs is always good like I was saying I was     
1:25:35     
talking I think to anak or soam about it um you know this is something we can uh     
1:25:42     
work out uh with different tools I think electronjs might be useful for having an     
1:25:48     
interactive form of it like say I want to plot it in this way or that way     
1:25:54     
visualize it but you know you can do a lot of that in geffy too and that's     
1:26:01     
that's all the you know geffy is kind of like the gooey and then you can actually     
1:26:06     
you know build extensions into that so you know you'd focus on kind of the     
1:26:12     
wayouts um but yeah you'd have to kind of we'd have to talk a little bit more about that in terms of what might be     
1:26:19     
Reinventing the wheel and what might be really useful     
1:26:25     
yeah okay all right well thanks to everyone     
1:26:31     
for meeting today and uh hope everyone has a good week um yeah see you next     
1:26:40     
week okay bye okay bye bye bye
