     
Transcript     
0:02     
okay uh welcome um so we'll start off here I don't know     
0:08     
if anyone has any updates they want to share or I don't know Susan how are you fairing with your modeling well I'm     
0:16     
just I'm just modeling I'm just putting in force numbers and getting numbers out     
0:23     
and it's a slow process because I can't seem to automate     
0:28     
it maybe if I had a different computer someone was suggesting that I get a new computer for $3,000 that that would     
0:36     
help just sure well just go do that     
0:42     
[Music] yeah so but at least I'm getting numbers     
0:51     
um yeah I would like this process finished sooner rather than     
0:58     
later it's good at least that you're getting numbers yeah so I guess uh neither of our gck     
1:08     
people will be here today they have other things that they're doing um other than that uh we have a     
1:18     
new paper it's come out uh they they put out the uh I don't know what they call it     
1:25     
like the pre proof and then they're going to update it with the actual proof later but this is the molecular basis of     
1:33     
differentiation wave activity and embryogenesis you can see they put my name in here somehow in the title but     
1:40     
that'll get fixed later uh but anyways they've got a stub up now for it and     
1:45     
then they update it so it has an address here or a DOI and uh the paper is you know it's     
1:55     
now on uh biosystem so that's where it's being published in     
2:01     
biosystems and uh so it's a nice paper it's got a lot in in it um you know we     
2:07     
have I don't know how many figures um but there's a lot of analysis     
2:13     
in here this is something that uh the analysis I worked on with uh Lucas or     
2:19     
SOS and U you know he's also doing this openworm um student ship where he's     
2:27     
working on building some pipe lines for proteomic data and genomic data so he's     
2:33     
working on that but but for the paper we worked on an analysis of secondary     
2:39     
proteomic data as well as genomic data and methylone data so we were looking at     
2:47     
genes associated with proteins and then sort of the methylation sites on those     
2:53     
genes and how they might affect the expression of proteins or how they might     
2:59     
be related to develop so yeah we have a lot of analyses here of the secondary     
3:06     
data we've used four species so the main uh focus of this is seans so seans was     
3:14     
you know where we have the genes the proteins and the methylation sites uh     
3:19     
then we also had uh protein data for uh     
3:25     
dropa uh yeast and mouse and so we had     
3:30     
those and we have like a corresponding list of proteins and again you know when you go     
3:36     
across Evolution like that you have to look for paralogs because the same list of     
3:42     
proteins isn't necessarily available in every species you have to uh you know     
3:48     
find out what the paralog is for that species and then look at it so um yeah     
3:54     
there there's a lot of analysis and this this is a figure of sort of the protein prot     
4:00     
analysis basically it's where we took counts of different proteins uh present for each uh uh ID in     
4:08     
each species and looked at the differences between the species and the types of development so uh we had Mosaic     
4:16     
development represented through uh seans and gropa regulative development uh     
4:23     
represented by Mouse and then uh there uh yeast are I wouldn't say no     
4:29     
development vment they have these basic changes in polarity and adhesion they're sort of like a developmental stage but     
4:36     
they don't express the same type of development as other     
4:41     
metazol so that's an interesting um set of relationships there and as you might     
4:48     
expect you know the celegans and groph kind of share proteins and protein     
4:55     
abundances whereas the mouse shares less of it and then the uh ye shares even     
5:01     
less but it's an interesting analysis and then I I uh was working on these uh     
5:07     
types of uh relationship uh networks uh which uh you know let's see     
5:17     
these type of relationship networks which are can't remember the name right now     
5:24     
models of interactive contributions so these are uh there two different types of the fully connected     
5:30     
models of interactive contribution in ordered models of interactive contribution basically looking at all     
5:37     
proteins or all genes related to a certain protein as a connected Network     
5:43     
and then all genes associated with a certain protein in terms of their order on the chromosome and how they're you     
5:49     
know maybe regulated so that that stuff is in the paper     
5:54     
too and so there was a lot of conceptual um stuff in the paper uh we had uh you     
6:02     
know a nice review of differentiation waves and some of the mechanisms for     
6:08     
them and you know kind of going through the sort of the cell biology so you know     
6:14     
just kind of relating it to some of the mechanics but most importantly relating it down to the genomic     
6:20     
level uh and so I guess the the takeaway actually is this diagram     
6:27     
13 so this is diagram or figure 13 and this has sort of a nice diagram of what     
6:35     
we have so we have all these uh uh proteins and or these genes related to     
6:42     
proteins here but you you have the labels uh the gene labels and then the     
6:47     
proteins to the right in parentheses you have the expression level genes by order of expression and     
6:54     
their presence in these different data sets we took a number of data sets and     
7:00     
found different data points so there was like an early you know uh early     
7:05     
development mid-development late development and we had all those data sets and and those were sort of our uh     
7:12     
so for this it was mid to early development so it went from mid to early development just showing the     
7:19     
change uh showing across genes so you had the genes that are ordered by their     
7:24     
expression then their expression level which goes up to here so you can see the most high highly reg uh the most highly     
7:32     
uh expressed genes are SEC one associated with mapk protein the second     
7:38     
highest is gsk3 related to GSK protein and then some of these down here like     
7:44     
pkc 2 and pkc protein these are all cgans so you have like the gene     
7:50     
associated with protein and seans the level of expression rank ordered and     
7:56     
then these different two different time points there wasn't a lot of VAR ation between early and     
8:02     
mid-development but that's sort of creates that 3D surface and so this is     
8:07     
the 3D surface without the labels kind of from a different perspective just showing the sort of the nature of that     
8:14     
surface and then you know mapping that to morphology you have then this isn't in     
8:21     
terms of this landscape but this is in terms of one of these images that we     
8:26     
worked on for another paper this is the uh this uh camera Luci drawing of the     
8:36     
dropa imaginal disc and you know from this image we have a nice sort of     
8:43     
character uh something that characterizes the differentiation we of going across um the imaginal dis so it here in     
8:52     
this part of the dis you have differentiated cells these are Atia so     
8:57     
this is in the the eye of the dropel and the imaginal disc is sort of the     
9:02     
precursor to the eye and it can also be a precursor to a wing and this case it's the precursor to the eye so you have     
9:10     
these different omaia that form they're basically little tiny you know uh I     
9:16     
don't know what I call them eyes but they're um cell complexes that sensibly     
9:22     
and so this is the differentiated State on this side this is the furrow and then this is these are the undifferentiated     
9:28     
cells here the idea is that this moves across the surface at a certain rate and you know a     
9:36     
lot of these proteins are expressed as a consequence of that um once the wave     
9:42     
moves across you get differentiation behind it and you get this and these proteins expressed and so the only     
9:49     
problem with our data set it was it wasn't well I guess it was explicitly developmental for the genes but the     
9:55     
proteins of course were you know kind of more General so there were a lot of     
10:01     
challenges in this paper with working with secondary data sets um but I think we did a pretty good     
10:07     
job of of characterizing a lot of this stuff um you know it's a very long paper     
10:12     
I think it's very good uh you know paper that kind of goes over the concept     
10:18     
relates it to a lot of secondary data and then gives this cross species     
10:24     
analysis and some insights uh final bibliography     
10:30     
information so uh yeah     
10:35     
we um what do you mean by final bibliography information     
10:46     
like oh pages okay so number of pages I open up the     
10:56     
PDF so 63 pages I think but this is let me see what's included in     
11:04     
this yeah it's 63 Pages uh with everything in it so we have our sub we     
11:10     
have uh you know our main text or bibliography we have some supplemental     
11:16     
tables let me get the URL I'll post the URL in the chat I     
11:23     
don't know if this is going to change when they have the final version of it but uh     
11:30     
there we go so yeah I think it's um like I said     
11:36     
uh this is the pre yeah the pre-proof version and the I guess there's going to     
11:43     
be like a final version soon but that I think that URL will be the same not     
11:49     
really sure how they do that once the final version comes out     
11:55     
I'll send everyone who's an author on it I'll send you know all the stuff that I get     
12:01     
as corresponding author     
12:06     
so okay so that's uh that yeah congratulations to everyone on that that     
12:13     
was good work um yeah yeah so actually the authors here are myself uh SOS     
12:20     
bastani who is Lucas he's been in the meetings Natalie Gordon of course uh     
12:26     
Richard Gordon that's dick and then Susan rord young so     
12:37     
congratulations right um so hello Pocky how are     
12:43     
you I I'm good how are you good yeah hi so I won't be able to share     
12:52     
my screen today I don't have my laptop with me but I'll just give a quick update so uh for the last three days     
13:00     
I've been working on like uh creating the NDP file which will like uh it uh     
13:06     
I'll give like a brief overview so I'm using two MLPs one will actually add     
13:13     
like the children notes of like the parent which will expect into the children war and then there is another     
13:18     
MLP which will like compare the ukan distance between what the model is     
13:25     
generating and what we actually want it to be and then based on the difference it was like back propagating the losses     
13:31     
and stuff so I uh had a doubts right now I have like data set for just one warm     
13:38     
growth like the XYZ coordinates also by the way right now I'm training it to     
13:43     
learn the spatial coordinates for the new celles like that is the learnable     
13:49     
parameter I'm training it on right now so for that I have the data set for just like uh one worm so I wanted to ask if I     
13:57     
could get like data sets from a few more maybe so that it can like learn it     
14:03     
better uh like what kind of data sets like uh like uh you had shared that     
14:11     
lineage tree with the XYZ coordinates but it has just the coordinates for one worm so similarly it has like for     
14:19     
different types of worms and all preferably with like different cells in     
14:24     
them some with maybe mutations and all if that's not available that's also okay but like some uh like a variety of data     
14:31     
set yeah for the model to turn on yeah uh we do have uh one I think we     
14:39     
have a zebra fish data set that's kind of set up the same way where it has like the cell positions and that's something     
14:46     
it's like larel it's laral zebra fish so you know it has basically the shape of     
14:54     
the embryo and that's differentiating into this uh you know t poll I I don't     
15:00     
remember the details but we that's the same type of data as what you have for the     
15:06     
celegans uh I've also been working on this uh larvel celegans data set I     
15:12     
showed it in the meeting a couple weeks ago that might be useful as well for training data um this is something that     
15:18     
from a Pap recent paper and I'll have to pull it out but um yeah I can get you     
15:24     
those data sets I think those would be useful because we you know we have the the seans this is like a data set that     
15:33     
is you know they they' basically taken a bunch of embryos and they've uh gotten     
15:38     
the uh microscopy data but it's not really that useful unless you kind of     
15:45     
average it out so we only have like kind of one uh celegans I guess you know uh we     
15:52     
could probably go back and use the raw uh the raw form of that data as well but     
15:59     
but um yeah I mean we can think about that but it's not I don't think it's     
16:04     
going to be as clean I mean you're aren't going to have like the representative cells it'll be kind of     
16:09     
like uh you'll have a lot of incomplete data um so yeah we might try like     
16:15     
different like you know an embryo larvel SE alans a larel laral zebra fish to see     
16:22     
what that looks like um but yeah we don't really have like M     
16:28     
mutant phenotypes and stuff like that uh all right all right I possible could     
16:36     
you please share the link I'll go through the data set as well     
16:46     
yeah all right do you uh that that your entire update     
16:51     
or yeah yeah uh I will also like once I reach home I'll push the code that I've     
16:57     
done till now uh I actually created a separate repository I will like add those to the L worm repository in a few     
17:05     
days I'm just testing right now if the model is performing well I'll share the     
17:10     
link of that other repository with you as well okay it sounds     
17:15     
good okay yeah yeah that was all thank you okay thank you yeah that's great     
17:21     
thank you okay so I think that's it for     
17:29     
updates um anything else before we move     
17:42     
on so um interesting stuff uh come across     
17:49     
my reading cue here so this is an interesting paper that I ran across     
17:56     
here um this is uh paper on it's developmental biology extending the limb     
18:02     
and body with vectors and scalers and it's a developmental biology paper but it's kind of talks about this     
18:10     
sort of how you know uh different things in the body sort of     
18:16     
differentiate and form these kind of uh well they're not really data structures     
18:21     
but I guess that's where they're going with this at least in terms of like you know um you know maybe even a metaphor     
18:29     
but it's it's kind of it might be interesting to go over in light of kind of the computational view of the sort of     
18:37     
stuff so the abstract here is outgrowth of the embryonic limb and vertebrates is     
18:44     
driven by a proximo distal gradient of cell movement with wind and fgf     
18:50     
activities controlling Direction and velocity respectively so what they've     
18:56     
done is you know they're looking at uh these gradients of gene expression     
19:03     
one of them being wi which is a factor in early development and the other is fgf which is a growth factor and so the     
19:11     
idea is that this uh this sort of outgrowth of embryonic limbs which are     
19:17     
like the limbs uh like for Limbs and the hind limbs invertebrates is driven by this kind of     
19:24     
gradient of cell movement so as wi and fgf are expressed in these gradients     
19:30     
there's this proximo distal gradient of cell movement that moves outward from the central axis of the organism out to     
19:38     
these Limbs and so wi then is associated with uh Direction and fgf is associated with     
19:47     
velocity so direction would be like position and maybe like the change in     
19:53     
position and then velocity would be like the speed at which that changes     
19:58     
Direction occurs so this this gives us kind of this these vectors and scalers I guess     
20:05     
of what they're talking uh the direction would be the vector and the velocity would be the     
20:12     
scal um a similar gradient though without a directional bias where codal     
20:17     
body axis extension so codal body axis extension is in the kind of in the kind     
20:25     
of other direction so so proximal p is sort of uh going from sort of the     
20:34     
midline outward and so that's the gradient and cells move outward     
20:39     
according to these signals okay so this is just kind of what this paper is is a summary of some     
20:47     
papers in current biology in this issue of current biology and you know they     
20:52     
kind of these articles usually just highlight what those papers are about so     
20:57     
they're useful from that persp perspective um so this paper you know it     
21:03     
starts out the outgrowth of the posterior axis of vertebrate embryos has often been speculatively compared to     
21:10     
outgrowth of the appendages um indeed there is considerable overlap in terms of the     
21:16     
genetic regulation of both processes but how how the genetic networks instruct     
21:22     
the cellular behaviors that drive access and Lim extension is remained unclear so     
21:28     
we don't know how genetic networks instruct cells to sort of form these you     
21:36     
know uh extensions of the Lim     
21:44     
book what was the title of that paper the one you're talking     
21:56     
about us so the title is developmental biology     
22:02     
extending the limb and body with vectors and scalers this is from current biology     
22:09     
yeah so they ask this question and they say three papers one through three one     
22:14     
of them in a recent issue of current biology now address this issue pointing to conserve mechanisms of cell     
22:22     
movement um although two of these papers make some use of established techniques     
22:28     
such as labeling specific line edges of vital dieses the core data of all three papers are generated by tracking cell     
22:35     
Behavior with sophisticated time-lapse video microscopy so across development they're     
22:41     
tracking cell Behavior so this is cell tracking uh whereas you can also use     
22:46     
labeling with dieses but it's hard to get a sense of you know that movement without the cell tracking data of course     
22:54     
now the cell tracking data is where you have you basically end up with set of numbers in XYZ coordinates and so that's     
23:02     
kind of what we want for these kinds of you know statements about um Direction     
23:08     
and velocity this approach is possible due to the toolkit of genetically encoded     
23:15     
fluorescent probes now available to wbel cells within a tissue or a subset of     
23:20     
structures within individual cells enabling the determination of cular     
23:26     
orientation division plane and veloc okay so basically uh they talk about     
23:34     
vertebrates the Lim starts to sort of a local swelling in the lateral plate mism     
23:40     
so you get the swelling where the limbs will be and that's the limb bud and so     
23:46     
this Bud formation occurs and after this Bud formation     
23:51     
occurs a thick column or structure arises along its distal margin called     
23:57     
the apical e dermal Ridge and so that's where we get these fgf uh genes these     
24:04     
fiberblast growth factors expressed so it's a family of uh genes fgfs and you     
24:11     
get a number of them there are a number of them invertebrates uh but this one FG     
24:17     
fgf8 is uh important particularly important so these fiberblast growth     
24:23     
factors are important in growing cell type called fiber blasts which are sort     
24:28     
of the Grist for you know tissue um invertebrates you get you know different     
24:35     
uh you know they proliferate and they form sort of the basis for a lot of the     
24:41     
tissue um in that in that segment so uh     
24:46     
fgf8 is is particularly important um and then this signals to     
24:52     
the underly mezen Kim and is essential for extending this Lim but outward so     
24:57     
you're extending limt outward from an origin and then you're it's it's being     
25:03     
patterned in different ways so you can think of the limb butt as being like kind of like your arm with your hands at     
25:08     
the end and you get patterning along the arm you get patterning at the end of the     
25:14     
hand um and you know in a lot of vertebrates this is a fin where you get     
25:20     
some fin you know uh some rays at the end of the fin so you can think of it     
25:25     
that way too uh so basically you need to have the     
25:30     
signal gradient set up as the limb is extending it's not just the cells aren't     
25:35     
just migrating out but they're differentiating and forming patterns uh so this is what they call     
25:42     
outgrowth then during this outgrowth the limb bug first forms as a hemispheric     
25:47     
swelling that then becomes a flat and markedly elongated structure so it goes     
25:52     
from that limb bud it's kind of like a bump to elong an elongated structure     
25:59     
and so let me see if I can draw this out to get a sense of what we're talking about     
26:06     
here so you know we have this sort of the     
26:13     
body we have the body here and we     
26:22     
have so we have the body here we have Lind Bloods that come out like this     
26:28     
they start out like this and then they grow in this direction     
26:34     
outward and this limb Bud goes from this bump out to kind of a flat appendage     
26:40     
that comes out like this and then you start to get differentiation along that uh appendage     
26:47     
so it might be you know a joint here or it might be raay at the end or something and so that's and then     
26:56     
this is fgf which is acting along this     
27:03     
radient okay okay so that's kind of where     
27:08     
they're going with this um this process has been traditionally perceived as driven by a isotropic growth that is     
27:16     
spatially graded proliferation at a high rate in the dis distal BL nzen kind driven by mitogenic     
27:23     
activity of uh derived fgf okay so this you know this is kind     
27:29     
of the idea that there's this an isotropic growth it means it's it's growing in One Direction where it's not     
27:36     
growing like um you know uh an isotropic Direction so not in     
27:42     
all directions at the same time um     
27:50     
so so that's one idea you know this is kind of this idea that spatially graded     
27:56     
and that this is all driven by fgf on the other hand AER specific Gene     
28:01     
inactivation studies in the mouse have revealed that although the loss of fgf signaling results in a much smaller Bud     
28:09     
it causes no clear proliferative or cell survival defect so what that means is     
28:14     
that if you lose fgf signaling if you knock it     
28:20     
out you might end up with say a smaller     
28:25     
uh limb bud so he might end up with something like     
28:31     
this and Lim might be smaller and as a result you get a     
28:37     
smaller with maybe some defects here you don't actually get defects this process is driven then by     
28:45     
no fgf so we take fgf out and that's not happening and we get     
28:51     
the smaller bud and what they're saying here is that you uh     
28:59     
basically can have a much smaller bud but it causes no clear proliferative or     
29:04     
cell survival defect so you should just get a smaller version then of what you     
29:09     
have in the previous diagram so that's the idea um on the basis of this     
29:16     
observation the Martin Leb speculated that the lack of fgf signaling might impair cell movement in Li but     
29:23     
initiation so it might mean that the cells are not moving as far out as maybe     
29:29     
as fast but really just impair cell movement so it's not moving as far out     
29:34     
to form these limbs uh bone atal on five so Martin lab     
29:39     
paper was seven bone was five uh bone metal recently addressed     
29:45     
the validity of the growth driven morphogenesis model which is this model where you get this you get growth and     
29:51     
you get morphogenesis from the growth uh from this gradient using a combination     
29:57     
of sophistic measurements of the Dynamics of limb butd shape uh using     
30:02     
optical Optical projection tomography and the spatial distribution of cell cycle Times by double labeling of newly     
30:09     
made DNA so they're thinking about uh measuring the Dynamics of lid Bud shape     
30:16     
so this limb bud has a shape if you look at our limb Bud again um it has a shape     
30:23     
that's you know determined by you know maybe something like this right so you     
30:31     
can have the limb Bud here and then the shape of the final thing and then the     
30:36     
limb bud can have different shapes here so you're looking at characterizing these     
30:43     
shapes um and then you're combining that with the spatial distribution of cell cycle times     
30:50     
and so you can do this by labeling DNA uh but also using mathematical     
30:56     
modeling so you're thinking about this shape you're thinking about the cells that are migrating out and you're     
31:03     
thinking about how those cells sort of double or how you know how they double     
31:09     
differentially in certain places so if we go back to our diagram um this in     
31:15     
this example here um you know if we have like an um sort of an an     
31:22     
isotropic setup like where the butt is looks something like this this part of     
31:27     
the butt will have a different rate of uh cell doubling than this part of the butt so     
31:34     
this part of the butt will sort of proliferate faster and this one will proliferate slower and then that     
31:40     
translates as you know as the as the uh limb grows and differentiates it it     
31:47     
contributes to that asymmetry so uh the pro they they do     
31:56     
this sort of study to PR eration rates they measured in three dimensions indicate that graded proliferation is     
32:02     
not sufficiently high or rapid enough distally to generate elongated limo growth as predicted in silico so this     
32:10     
was an in silico study that they did in five and they're not able to uh you know     
32:16     
get the same results as you get in the laboratory hence there must be other     
32:21     
directional processes such as oriented cell division or migration that drive     
32:27     
morphogen so it means that for these lims as they grow they have this sort of oriented     
32:34     
cell division and migration and so if you look at um like you know cells that are     
32:42     
dividing they might divide like this along an axis it might divide like this     
32:47     
along an axis you can see there's a bias in each direction as those cells divide     
32:54     
and so if you have like a mass of cells where division occurs more in this direction     
33:00     
than this direction then you get more mass up here     
33:06     
down here and so it just becomes like this you know it it contributes to these     
33:11     
asymmetric shapes this a NI at tropy uh this clarification sets the     
33:19     
stage for two papers from the hen and tabin labs and that's two and three that     
33:25     
examine exactly such directional processes during develop a third paper from the POR lab     
33:31     
number one illuminates how with the proper constraints random cell movement     
33:36     
leads to codly directed Body axis extension so they they do these     
33:42     
experiments uh they talk a little bit about uh some of the results that they     
33:48     
get and then this this part here is interesting having established that oriented cell behaviors control both     
33:55     
normal limb but initiation and subsequent outgrowth The Next Step was to identify factors that may control     
34:02     
such behaviors as wi planner cell polarity or PCP signaling is known to     
34:08     
regulate oriented cell movements and division both groups examine mice     
34:14     
lacking win 5A which is a win Factor wi Gene in which both the limb and codal     
34:21     
body axes are severely short so this is a a mutation of wind 5A where there it's     
34:27     
knocked out and this uh this knockout represents     
34:33     
lemon codal body axes being severely shortened so again we have this again uh     
34:39     
the body axis is shortened and the limb is shortened so you basically have the signal that can't be uh you know     
34:47     
propagated down it doesn't contribute to     
34:53     
grow okay so the rostal ceral movements in the lateral uh plate neoderm at very     
34:58     
early lmud stages were absent in these mutants uh as outgrowth progresses     
35:04     
mutant cells near the AER migrated with a reduced velocity and efficiency     
35:09     
whereas cells near the dorsal and ventral ectoderm displayed a reduction in velocity so this is where you have uh     
35:17     
these mutant cells as outgrowth progresses they migrate with reduced velocity so they slow down and     
35:25     
efficiency whereas cells near the dorsal on ventral eor display reduction and     
35:30     
velocity efficiency and coherence so this is where we have cell these mutant     
35:36     
cells in two different places near the aeer and in the dorsal and vectoral ENT     
35:42     
ectoderm and in both cases they exhibit this reduction in velocity but in these     
35:48     
uh this latter case you also have a well I guess in the AER migration case you     
35:54     
have reduced velocity and efficiency and then the latter case case you have a reduction in velocity efficiency and     
36:00     
coherence whatever that means but anyways the point is is that the velocity is reduced relative to the wild     
36:07     
type cells and so that shows you how this velocity can be controlled by wi if     
36:13     
you have wi in or this wind 5 a gene if it's normally functioning you get this     
36:19     
uh you know a velocity of the Wale Tye and if it's knocked out then you get     
36:24     
this reduced velocity um and I my computer's acting up um so I thought     
36:32     
I would add to this that I um and an embryology course I took thanks today     
36:40     
um if you don't develop a lung you don't develop Li limbs oh okay and it fits in     
36:49     
with developing um kidneys so you develop uh kidneys and then the lungs     
36:57     
are a ble to breathe in and out with the amniotic fluid that's produced by the     
37:03     
tiny kidneys yeah um and when you get the lung development then you get proper     
37:11     
limb development yeah so anyways and I may     
37:18     
come in and out or maybe I should just give up I'm taking my computer in this week okay to have it looked at yeah it     
37:27     
just freezes yeah this is     
37:32     
just all right okay so um yeah so these     
37:42     
disorganized behaviors May account for the somewhat expanded dorsal ventral axis and wind 5A mutants so both groups     
37:49     
found that implanting a small source of win 5A protein so this isn't the gene it's not The Express Gene but it's the     
37:56     
protein in normal buds so you take the win 5A protein you put it in normal buds     
38:03     
resulted nearby labeled cells orienting are moving towards the wind 5A     
38:08     
Source consistent with the idea that the normal gradient of wind 5A in the limb Bud may act as a     
38:15     
chemoattract and instruct both directional cell behaviors so this is interesting this wi     
38:22     
5A protein wi 5A Gene is expressed it contributes to the BL velocity of the     
38:28     
cells but then it produces a protein which then acts as a chemoattractant and can instruct     
38:35     
directional cell Behavior so it kind of reinforces what's going on with the cells and moving towards the end of the     
38:41     
LM B both proximal distal extension and dorsal vental compression appear to be     
38:47     
governed by this wind PCP activity contributing to the distinctive paddle shape of the outgrown limb butd so the     
38:55     
outgrown limb butd is going to have this paddle shape going to go from this sort of rounded     
39:04     
Bud like this sort of a paddle shape like this     
39:12     
and this can be done by you know you can have cells that are moving along     
39:17     
according this constant velocity and then you can also have the source this     
39:23     
chemoattracted source     
39:29     
and that acts to like further uh regulate this     
39:35     
system okay so that's uh kind of their insight into that     
39:43     
um and so you know they've done a number of other experiments     
39:48     
here and so let me get to the sort of the conclusion     
39:54     
here um in cical simulations demonstr that graded random motility leads to     
39:59     
coddle extension and generates an opposing gradient of cell density which indeed the authors observed in the     
40:06     
posterior embryo with the added constraint of high cell density in the midline and lateral plate to impede     
40:17     
broadening of the prematic Semitic me mesoderm the random motility gradient     
40:22     
doesn't require other Vector instructions or vector in quotes but you     
40:27     
know we need these kind of instructions to drive velocity to you know Target a     
40:32     
certain location and so forth uh doesn't require other Vector instructions to drive easy location this leaves the open     
40:40     
question of what role went 5A plays in codal axis outgrowth wi 5A is clearly     
40:47     
required from normal body axis extension of course we saw that but does it play a     
40:53     
similar Vector role in polarizing cell behavior in the prestic neoderm as it     
40:59     
does in the limb which would be the most parsimonious or does it carry some other     
41:05     
function uh so that's that's the question they have so they have these Vector instructions here which are just     
41:11     
basically telling the cell how fast to move where to move to and then you have     
41:17     
this other set of instructions that involves uh the sort of the     
41:23     
shape um so this parallels between these parallels between limb outgrowth and     
41:29     
body AIS extension provided by these three studies 1 through three support     
41:34     
ideas that mechanisms controlling axis and tail extension might have been co-opted over the course of evolution to     
41:41     
control the outgrowth of multiple appendages so this is interesting that there's this sort of idea that um you     
41:49     
know these axis and tail extension mechanisms are sort of you know there     
41:55     
were independent mechanisms maybe for the growth of very simple axial systems     
42:02     
and then they were sort of uh coopted and then coordinated across all limbs in     
42:10     
uh invertebrates to you know to sort of coordinate each of you know when you     
42:16     
have in a tetrapod you have five limbs you have the arms the legs and the tail     
42:23     
and they all grow kind of in a coordinated fashion and you know maybe     
42:28     
your arms and legs need to be the same length or maybe they need to be scaled to a certain extent or you know the tail     
42:35     
this can't be too long relative to the arms and legs all that needs to be coordinated um in development so maybe     
42:43     
that's what's going on here that there's this co-option of this simple mechanism for multiple     
42:49     
appendages and that's reference 16 so we'll go through the references in a minute it will be interesting to see to     
42:56     
what degree dire cell behaviors Drive the morphogenesis of other vertebrate structures that extend from the body     
43:03     
such as genitalia fgf and wind signaling control cell migration in other developmental     
43:10     
contexts across diverse species including flies and worms as well as vertebrates so flies and worms you see     
43:17     
fgv and wi so in our list of genes that we do that we had in our paper uh we we     
43:23     
looked at wi uh we didn't I don't know if we did fgf but we definitely had wi represented there and so wind has its     
43:31     
role in uh flies and worms but it also has its Ro in vertebrates and so you     
43:37     
know when we're making these comparisons across species sometimes we have to look     
43:43     
at the paralogues but also sometimes we have to think about what exactly the function is sometimes the function is a     
43:49     
bit different in these different species and so you have to uh you know maybe that's something we didn't really take     
43:56     
into account in our data analysis but might be important so you can also ask the     
44:04     
question to what extent are Downstream signaling components conserved and by that we mean you know if we have these     
44:11     
simple mechanisms and they're co-opted for coordinated extension of     
44:17     
Limbs you know they had to start from somewhere and those basic components     
44:22     
have to be conserved in evolution that means they can't change they can't be mutated they can't be recombined they     
44:29     
can't be lost they have to be conserved and so if those things are conserved highly conserved then these things you     
44:37     
know you probably have a better chance of having these kind of components to     
44:42     
work well but we don't know uh the answer to that uh conversely now that     
44:48     
common cell mechanisms appear to operate between extension of the vertebrate axis and limb what distinctions in these     
44:55     
morphogenetic processes dictate the final differences in shape achieved so     
45:01     
you know we have some you know differences in how uh this process works in different     
45:08     
contexts um you know how you know what are kind of the details required or what are the changes     
45:15     
required what is the anisotropy required to get these final differences in shape     
45:21     
so obviously exciting work lies ahead and so this is something that     
45:27     
they're looking forward to I think it's interesting um but let's look at these     
45:32     
references so references one through three we have reference one which is     
45:37     
this paper a random cell motility gradient Downstream of fgf controls     
45:42     
elongation of an amnio embryo so this is where they're looking at a     
45:48     
vertebrate uh and they're looking at this fgf and they're looking at this     
45:53     
random cell motility gradient and this is a this was from     
45:58     
2010 uh number two is this paper gross also in 2010 so I guess this paper I'm     
46:05     
reading from is from 2010 that's according to the DOI information but all these other papers     
46:11     
are from 2010 as well uh the second paper then being went 5A junk jnk and     
46:18     
fgf mapk Pathways and we looked at all these in our paper so that's interesting     
46:24     
that those pathways are you know involved in orogenesis they're involved in growth     
46:31     
and so they're looking at these but these are invertebrates not in in celegans or in veropa um so wind 5A junk     
46:40     
and fgf mapk Pathways regulate the cellular event shaping the vertebrate     
46:45     
limb body so these are these Pathways then regulate different cellular     
46:52     
events uh paper three is uh oriented cell motility and     
47:01     
division underly early bud morphogenesis or limb bod morphogenesis so this is where we're     
47:08     
looking at lmod morphogenesis and that kind of you know things I was talking about with grow but in terms of     
47:17     
cation okay so there's some other interesting papers in here as well uh     
47:22     
this one five the role of spatially controlled self proliferation and     
47:27     
morphogenesis it's kind of related to number three um there's of course functions of     
47:34     
fgf signaling from the apical ectodermal rigin limb     
47:39     
development um there's noncanonical wind PCP signaling during vertebrate     
47:48     
gastrulation there's a wi five pathway underlying outgrowth of multiple structures in the vertebrate embryo so     
47:55     
that's specific to Wi that wi F pathway and you know the mutants and all     
48:02     
that um there's also spatial and temporal patterning of irk signaling     
48:07     
during most embryogenesis so this is again looking at these spatial and temporal     
48:13     
patterns um this one number 13 fgf8 mRNA Decay     
48:20     
establishes a gradient that couples axial elongation to patterning in the vertebrate embria so this is again     
48:27     
fgf8 and this is looking at mRNA Decay so where you have mRNA Decay usually the     
48:34     
gene will Express a certain amount of this whatever Factor it is it'll produce     
48:39     
a certain amount of mRNA and then as that Gene shuts off the the sort of the supply of mRNA that     
48:47     
tissue or in that cell will will will Decay with a certain halflife and so you     
48:53     
can do these mRNA Decay experiments where you actually turn off the cell replication machinery     
49:00     
and look at how long it takes for each uh mRNA species to Decay and it's     
49:06     
interesting because it takes like quite a while for that to happen and of course this is what happens in in the cell     
49:12     
cycle where you have something that's uh genes that are upregulated and then they     
49:17     
stop upregulating and then those those mrnas have to Decay to a certain extent     
49:23     
to trigger us into the next part of cell cycle so in this case fgf8 if you produce a certain amount of     
49:30     
it and you stop producing it then the MRNA decays and you lose uh you know     
49:36     
your the stochiometry decreases of that mRNA and so you have to keep you either     
49:43     
turn it back on turn the gene back on produce more of it or it loses its     
49:48     
effect and so that's what we're doing here we're looking at mRNA to K establishing a gradient that couples     
49:55     
axial elongation pattern then this paper 19 fgf signaling     
50:03     
and flies and worms more and more relevant to vertebrate biology so this is where the citation of what we're     
50:09     
talking about with the relationship between flies and worms which is like celegans and groph to vertebrates so     
50:18     
this is uh another connection here I mean people made these connections people used their soft and celegans sort     
50:25     
of as Medical model organisms but this is again we can learn a lot more from     
50:31     
this um finally this paper 20 genetic control of cell intercalation during     
50:37     
tracheal morphogenesis indrosophila so this is where we're looking indrosophila     
50:43     
at the genetic control of uh you cell     
50:48     
shape changes okay so that's all for that     
50:54     
paper any questions or comments     
51:00     
so if there other questions or comments I did want to mention that artificial life 2024 is occurring this week and so     
51:09     
uh this is of course the artificial life conference is where people do artificial     
51:14     
life and that's the main conference so we talked last time about Gecko and what they're doing there but     
51:21     
this is artificial life this is the more sort of theoretical Conference of but     
51:27     
they do genetic algorithms but they also do a lot of solo automa uh and other types of models as well so this is the     
51:35     
website for aive 2024 it's happening in Copenhagen Denmark this week um as you     
51:41     
can see July 22nd 26 they have a virtual component as well I didn't attend but um     
51:48     
you know this is just to make it more convenient for people to attend so the theme this year     
51:54     
is exploring New Frontiers and because it's being held in Copenhagen they have this Viking theme I     
52:01     
don't know exactly what that means but um in any case uh     
52:09     
so we can look through the program maybe a little bit and see what they have going on at the conference so it's     
52:15     
usually pretty diverse there are a lot of things going on um you know a lot of like they do things with vesicles and     
52:22     
they do things in silico and they do things uh they even do art and philosophy     
52:29     
so let's look at some of the Keynotes we have Katie Bentley who does a lot of stuff with uh developmental     
52:38     
biology and um computational models uh and her talk is a active     
52:45     
perception and cellular or adaptive Behavior so this is where you know there     
52:50     
she says that humans and many other organisms use feedback between movement and sensing known is sensory motor     
52:57     
coordination or active perception to inform better decision um and so we move to improve     
53:04     
the quality of what is sensed and so this is taking this principle of sensory motor feedback or sensory motor     
53:11     
coordination and applying it to cells and some of kind of what cells are doing     
53:17     
as these Dynamic and protrusive shaped objects so you know this is very much     
53:24     
sort of the embodiment of cells and how they coordinate their movement uh     
53:29     
through these sort of sensory motor feedback loops so this is really interesting where local movement cell     
53:37     
shape changes improve the quality of sensed information via signaling or mechanical receptors it improves     
53:44     
cellular decision making so in other words when cells make a fate decision or     
53:49     
they make a decision to move somewhere collectively this is based on the quality of the sensed information so     
53:57     
this could be through um chemical signaling this could be through mechanical signaling if you get a     
54:03     
mechanical signal and you can sense it really well then you produce this     
54:08     
behavior that is sort of uh coordinated if you can't get that information then     
54:14     
you can't make the movement then you basically go into apotosis or necrosis uh I guess that's the logic     
54:22     
behind this any case it's uh sort of what she's talking about here     
54:27     
uh and sort of building this sort of embodied model of Cell     
54:32     
Biology there's kadoma and she does synthetic biology so     
54:39     
her talk is life but not alive synthetic biological cells and of course we've talked about synthetic     
54:45     
biology uh where you know you want to create sort of these artificial life in     
54:52     
biological cells and so creating artificial life would allow us to diversify the chassis of biological     
54:59     
studies and bioengineering and so I think this is just kind of showing off what we can do     
55:05     
with synthetic biology and how that kind of relates to artificial life so there's     
55:10     
some papers here uh this is this operating system called Trumpet which is     
55:17     
an operating system for simple and robust self-free Computing that's a nature     
55:23     
Communications there's the present future of synthetic cell development ment nature reviews molecular cell     
55:29     
biology and controlled exchange of protein and nucleic acid signals from between synthetic minimal cells and     
55:36     
that's in cell systems so uh it's Juan Perez     
55:41     
mercader um and he's interested in uh making biochemistry free artificial life     
55:48     
in a test tube so this is where you know you don't you're not using biochemistry     
55:53     
you're using this artificial life and a test tube um so starting with a     
55:59     
biochemistry free aquous and homogeneous blend of several less than 10 including     
56:04     
water small non- amphilic and non-biochemical molecules some of them which function as     
56:11     
fuel and using biochemistry free polymerization and do self assembly or     
56:16     
Pisa and that's a method that they've developed we demonstrate experimentally     
56:21     
the exnovo autonomous photochemistry controlled one pot synthesis of amplifi     
56:28     
and so this is like kind of building very basic life um from these     
56:35     
aquous Solutions and so um it's really interesting you can uh     
56:44     
build these my cells that grow and evolve into functional vesicles protocells or artificial cells that     
56:51     
exhibit the characteristics that are defined above here so there are these certain Criterion that he plays out and     
56:59     
up here and they exhibit these so yeah     
57:04     
this is interesting stuff on Proto cells and early life and sort of the biochemistry of     
57:11     
that uh oh Nakajima is     
57:16     
um presenting un physicalizing deep learning so this is where you know we     
57:21     
look at Deep learning and we have these sort of disembodied netw works that can     
57:27     
do a lot of interesting stuff but what he's talking about with physicalizing is that he wants to again turn to     
57:33     
embodiment to uh build better models and so uh the concept of embodiment has     
57:40     
always been an important topic to consider when form meets the physical world physical deep learning or PDL     
57:47     
which is a framework for implementing deep learning into physical substrates is an area where embodiement has     
57:53     
recently taken a critical role and this discussion we review the concept of PDL     
57:59     
with a number of examples and clarify its necessity in today's computational systems then we show that PDL is     
58:06     
essentially connected to the concept of physical Reservoir Computing so we have Reservoir Computing that is embodied as     
58:14     
well and introduce a learning scheme based on gradient for approaches to PDL finally we report our recent     
58:21     
approach as to how this learning process can be physicalized into neur neuromorphic devices and soft robots and     
58:28     
discuss their impacts so there are a lot of interesting citations here on physical deep learning that's something     
58:35     
that I know my other group is interested in uh with respect to embod so that's     
58:41     
really good so those are the um the Keynotes and then of course we have     
58:48     
workshops and tutorials and you know we go over looking at some of the titles of the     
58:53     
workshops we have a lot of diversity we have for example this one which is Agent     
59:00     
based modeling of human behavior this is uh where you know take agent based models and model different human     
59:07     
behavioral things Collective behaviors uh the the evolution of     
59:14     
things which is basically about evolving and learning machines and populations and machines     
59:21     
that develop themselves so you know this can include a bunch of different things     
59:27     
it can involve a number of technical issues including the joint evolution of morphology and control the integration     
59:34     
of evolution in lifetime learning the physical incarnation of evolving     
59:40     
entities so this is a lot of the evolution of things apparently is very     
59:45     
specific um we also have Evolution criticality     
59:50     
and creativity and collective intelligence and of course this relates to um sort of these emergent intelligent     
59:59     
behaviors that arise from the cooperation collaboration another Collective efforts of any individuals in     
1:00:05     
a system so I guess these include like you know biological systems that are collectives social systems and so     
1:00:13     
forth um then there's this Workshop from a imited im imit     
1:00:20     
imitative imitative to biological pla biologically plausible synthetic models     
1:00:26     
of natural cognition and so this is again kind of going through like basically a synthetic     
1:00:33     
cognition type uh approach biologically plausible and all that so that that's     
1:00:41     
that'd be kind of an interesting one to catch up on uh we have molecular communication approaches for wet wear     
1:00:46     
and artificial life we have a hackathon AI music and visual arts emerging     
1:00:53     
researchers in artificial life which is always good because you get like a lot of upand cominging topics and people um     
1:01:01     
I actually attended that one year a life and organizations which is organizational psychology and     
1:01:08     
organizational complexity and all that uh making time tempor uh temporality and     
1:01:14     
natural and artificial systems this is talking about time in biological and     
1:01:19     
social systems and any other type of system you can think of the Ali ency encyclopedia hackathon     
1:01:28     
so this is uh the organization that sort of sponsors the aife conference isow     
1:01:35     
they have their own online encyclopedia and so they run a hackathon to update that     
1:01:42     
encyclopedia um gold directed behavior in life and non-life so this is been     
1:01:48     
having to do with gold directed behavior and a lot of things related to     
1:01:53     
thology and you know uh uh chimala forms for the fourth     
1:02:00     
workshop on chemistry and artificial life forms uh and then finally nature     
1:02:07     
inspired simulation and visualization of complex networks so that's a lot of     
1:02:13     
interesting workshops as well     
1:02:19     
okay so yeah dick has a comment here from the first paper we did or the I     
1:02:26     
guess the the limb development paper I looked into limb development long ago but decided it was 3D and too much for     
1:02:33     
getting images and computer simulations then yeah might be a lot easier to do     
1:02:39     
now I don't know um to do those kind of simulations certainly maybe in compy     
1:02:45     
cell 3D we could do something like that I in fact I'm not really sure they may have some things some interesting things     
1:02:52     
having to do with like chick embryos as I know they do a lot of modeling of chicken Mars I'm not sure if they've got     
1:02:58     
you know like really kind of deal with limb butting or anything like that but     
1:03:04     
you know it's a thought okay so there any other things we need to talk about before we go     
1:03:15     
okay all right thanks have a good week
