     
Transcript     
0:00     
how is everyone morning so is everyone today     
0:06     
okay uh yeah Pocky hello uh so Pocky would you like to give     
0:11     
an update yeah hi hello am I audible yeah     
0:18     
yeah uh yeah so over the last week I uh     
0:24     
like last week I had given my update where I was generating the nodes for the graph so then I was also able to like uh     
0:32     
incorporate the position like the coordinate position of the different     
0:37     
cells there was one another uh CSV file and then now I'm using the UK distance     
0:45     
between two nodes to generate like the weight for the edge between those two     
0:52     
nodes all right yeah so I have also said like a     
1:00     
threshold for that uh like if the cells are too far away they will not necessarily have an edge between them     
1:05     
because then uh like that's not required so uh I     
1:11     
mean uh this is like a rough idea for now like when I actually feed it into     
1:17     
the data model when I feed it as a data set then I will see if I need to like     
1:22     
get just some parameters over there and then in the NDP network uh like I had     
1:28     
told you like a couple of meetings ago I was facing some issues uh errors and gunning I was able to fix them as well     
1:35     
and now like the model is uh training but is it is training uh taking forever     
1:41     
to train by the way I was able to train it like uh few days ago and now I     
1:46     
started it like an hour ago it still hasn't rained so yeah I will see how     
1:52     
that goes right yeah and yeah that was about my     
1:58     
update so I'm just right now I'm working on like the NDP code so I was able to     
2:04     
like finish the function for the multi-layer percepton and now I'm just like working on the training. pii like     
2:11     
which will actually train the model all right yeah I think that's uh so how are     
2:18     
you running how are you training the model the NDP model how was     
2:25     
that uh okay so they had provided some code like for training but like we don't     
2:31     
need uh all of that so like I just took the necessary uh functions from it and     
2:37     
then I like modified them a little so uh one     
2:45     
second I will actually share my screen okay     
3:06     
is it visible yes yeah uh so here like they have uh     
3:13     
bunch of functions uh in the NDP code itself so out of these like uh we didn't     
3:20     
need all of them so I uh just took the necessary ones and like I've changed the     
3:26     
code also in a few functions I I have pushed this code to a repository uh like     
3:33     
not the latest changes but like the this NDP file is there in this and in the     
3:39     
train. file uh so they're using this function called Fitness where they're basically fitting the model with the     
3:46     
right parameters that's why the name and uh yeah so this function is     
3:53     
basically like uh for the growing graph this is for that and this is for like uh     
3:59     
fitting that graph with the multi-layer perceptron which was uh we were using for like uh determining where the nodes     
4:06     
will be added and the weights of the edges and all so this is that function and uh this is another function     
4:15     
that I was like experimenting with so I will give updates about this this is     
4:21     
also for like uh the we basically using uh two three MLPs one MLP will tell     
4:27     
where the node will be added other one will tell what what will be like the weight of the edge because we have like     
4:33     
a weighted graph here and it is weighted and directed as well so this will uh     
4:38     
manage all that so I'm still working on this one as of now okay looks like they     
4:46     
have an interesting comment in here um under worm in the worm growth function     
4:51     
High Fitness value yeah go back to the where you were yeah yeah uh so yeah it looks like they     
4:58     
have an interesting a comment in the worm growth function High Fitness value means the graph has small worldness     
5:06     
returns a scalar measure of small worldness of a graph G uh Omega ought to be close to zero and or Sigma is greater     
5:13     
than one so this is where uh you're basically getting a measure of what they     
5:18     
call small worldness which is where you have a lot of centralized hubs you have     
5:23     
a very small uh diameter of the graph so if you go from one end of the graph to the other it's a small number of hops     
5:31     
and so what you're basically what that means is that if the graph is more fit     
5:37     
you should sort of get a I guess a more a small world graph which I guess you     
5:42     
know make kind of makes sense um that you know the graph gets Consolidated and     
5:48     
the distance between like the path length what they call which is if you go     
5:53     
from one end of the graph to the other that you and you just make random hops across it that's the path line length     
6:00     
that path length then is minimized uh and then you get the small world uh     
6:06     
architecture which has a lot of these hubs a lot of these centralized nodes     
6:11     
it's not you know it doesn't necessarily follow that you know uh the best graph     
6:17     
is small world or you know certainly an optimized graph can be small world but you can optimize it for other types of     
6:24     
topologies and that's that's maybe kind of where you know what they're doing     
6:29     
here is maybe uh I don't know if it's as biological as it could be let's just put     
6:34     
it that way so you could have like you could make the fitness or you could make     
6:40     
it so that the optimized graph is actually what they call scale free which     
6:45     
means that there are many different uh you know type numbers of connections to     
6:51     
each node so yeah it's it's definitely something that I wouldn't worry about now but that would be interesting to     
6:57     
explore with the code and then these are of course the sigma and Omega values which I guess are related to Fitness so     
7:05     
yeah this is interesting I'm glad you're getting into the code and kind of looking at how to run it and train it so     
7:11     
you actually train on a data set or do you train on like what do you what is it     
7:16     
that they have that they're actually training on is it just kind of like idealized graphs in their     
7:24     
depository they have like a num AR for like they uh experimented with uh     
7:29     
different kind of models like they have an exor gate and then a loner model all those things so for that they have like     
7:35     
a n AR I uh like this over     
7:42     
here uh yeah so they're using like these arrays for that but like in our model uh     
7:50     
I was thinking of like that day had shown one second like yeah so I'm     
7:57     
planning on passing these uh nodes along with their like weighted     
8:03     
edges so uh for the edges also I was planning on uh like having Vector     
8:08     
embeddings at each node which will have like the lineage name in the form of a string and like then along with the     
8:15     
cordinates so like we can get both the edge weights and like which node it should connect to based on the naming of     
8:21     
the node so I'm planning on using this as the data set to train the     
8:27     
model that's good yeah that's very good I think that's having that kind of abstraction and then training it on     
8:33     
those weights that should help with celegans uh since we're kind of focusing     
8:39     
on seans and we can actually get these kind of graphs for other embryos but then having that framework we can then     
8:46     
like plug those matrices in and use them as uh you know ways to train the model     
8:52     
for different types of embryogenesis but yeah this is good this is good     
8:59     
thank you yeah was there something else you wanted     
9:05     
to show us or uh no no this was like this is about it that I had done in the last week I     
9:13     
was busy with a lot of evaluations so wasn't really able to put much time into it but like I could uh get somewhere     
9:21     
like yeah yeah so what are your plans for this coming     
9:27     
week uh so for this coming week TR and planning on like finishing up the function for like the training the model     
9:35     
and like getting it running and then uh once it's stained then only I will like uh know how to like adjust the     
9:41     
parameters and all according to our like for more optimized model so I want to     
9:48     
like get it done by this week okay yeah that would be good yeah yeah this is this is good     
9:55     
stuff yeah I'm glad you're able to get this working and we can into it a little     
10:00     
bit um thank you yeah so any comments or questions from Hussein or     
10:14     
Jesse okay um so uh Jesse pulled up a link here small     
10:21     
world network uh yeah this are more along the lines of what we talked about some time ago in the cybernetics group so this is     
10:28     
actually it would be this Wikipedia article     
10:36     
um see so it'll be this Wikipedia article     
10:41     
where they talk about small world networks this is kind of an example of what they're getting at here so uh a     
10:48     
small world network is a graph characterized by a high clustering coefficient so they use in graph in     
10:55     
graph Theory and in network science they use these parameters to character ize a graph usually they'll characterize it by     
11:02     
a couple of uh parameters such as degree clustering     
11:08     
coefficient uh and other types of distance metrics and then you know you you can actually measure out sort of     
11:15     
some of the properties of these topologies so they kind of look kind of strange because everything's connected     
11:20     
to everything else but the fact of the matter is you can actually characterize these with different types of statistics     
11:27     
and so path link is one so you might say if I want to go from this point up here     
11:33     
at the top to this point down here at the bottom left that are kind of in circled and purple you might say okay     
11:40     
what is my least least uh path length or how many what's the shortest path I can     
11:47     
take from one of each of these two nodes and so you know the shortest path length     
11:52     
here appears to be one because you have a connection here you have many ways to make two     
11:59     
connections between these two nodes and some three you know some ways to make three and four hops to this node so you     
12:07     
can have like a minimal distance which is one you can have like a mean distance     
12:13     
which is in this graph maybe 2.5 or maybe two which means that on     
12:19     
average if you take all of the paths possible you get about two hops per you     
12:24     
know graph and then you have you know three Hops and four hops     
12:29     
so you can actually take I think here you can take five hops around the edge but you know that's not something     
12:37     
that's an outlier that but that's added into that average and then you can also optimize the graph for the longest path     
12:43     
length as well it's and sometimes that's that's important because if you have a growing Network these edges might     
12:51     
disappear over time or they might reconfigure themselves over time so you     
12:57     
know there are a lot of ways to measure path length a lot of ways to measure a centrality meaning if you take a single     
13:03     
node you know how Central is it to the graph so how many other things is it connected to and so if you take these     
13:10     
kind of measures um and clustering coefficient is one of those where you basically say     
13:16     
you know how like how how much is sort of concentrated on this node if I got     
13:22     
rid of this node how many things would be affected and so that's you know that's going to be for this node node     
13:29     
it's uh at least six nodes are going to be affected so that's that's what we     
13:35     
might call a hub or Highway connected node and so that's where we come to Small World networks because small world     
13:42     
networks have these hubs they have these hubs of large effect meaning that     
13:48     
they're connected to a lot of other things and as a consequence they affect a lot of other things when they     
13:54     
change and so we can count this out we can get these statistics and Des describe the topology of these networks     
14:01     
you know we can have random graphs like this where we have no hubs at all so in this case we have things that are     
14:07     
connected maybe at most by four things this node is kind of maybe huike but not     
14:13     
really but it's connected randomly so we don't expect too much structure and so     
14:19     
you know there are a lot of ways we can characterize these graphs and so small world networks is one way to describe     
14:26     
this kind of graph um these were just discovered in social networks so you     
14:31     
know Watson strogatz in 1998 they kind of described or they kind of coined the     
14:36     
term small world Network they coin this from uh aspects of the clustering     
14:41     
coefficient this no to node distance or the shortest path length and you know     
14:47     
and so they were able to characterize this in a in a social network or uh like     
14:52     
I guess it was an an instance of the internet in fact but uh you know there     
14:58     
are all sorts of of things you know having to do with the network structure and small world networks have a certain     
15:04     
aspect to them where they have certain types of ceks that are local highly     
15:09     
connected groups and other types of sub networks and so you know we have things     
15:14     
like small world networks we have scale free networks which I mentioned we have random networks and we have other kinds     
15:20     
of networks where you can uh where the community structure varies and the sub     
15:26     
the subgraphs vary in ways that are inter interesting so you know this is relevant to Sean's connectome because     
15:32     
Sean's connectome is considered a small world Network um there there have been     
15:38     
Papers written on what they call the rich Club of uh of neurons in the     
15:45     
connectome and those are neurons that are connected to a lot of other neurons and so those neurons um are key in sort     
15:53     
of the operation of some of the function functional aspects of the connectum so     
15:58     
it could be like you know um some motor pathway or some sensory pathway and     
16:03     
there's certain neurons that sit at those in those at those Junctions and it     
16:09     
governs a lot of functions so that's uh it's not really related to cybernetic so     
16:15     
much uh there's some connect interesting connections between like what we've been doing in network Theory or network     
16:23     
science for like 30 years and cybernetics and that might be an interesting foray because uh you know     
16:30     
you look at these structures and you say oh that kind of looks cybernetic like and it could be but no one's really ever     
16:36     
explored that so that's the answer to that question um and then Jesse says I     
16:42     
can't remember the guy's name or a small world inness was associated with computation and sort of a finite or     
16:48     
limited possibility space um yeah I don't know I'll have to     
16:54     
look that up but I think that that's that's an interesting thread as well uh yeah it's it's really about you know     
17:01     
like these networks they give you this possibility space when you have a randomly connected graph you know there     
17:07     
are a lot of different combinations or a lot of different Pathways and when you have these kind of networks that are     
17:13     
selected for maybe in a small world direction or some other direction like a regularized network that closes down the     
17:20     
number of possible Pathways but sometimes it opens them up it really depends on you know what your what what     
17:26     
part of the graph you're in and where you want to go and and and you know so there are a lot of things you can do     
17:31     
with networks that are really interesting in that regard so thank you Pocky for your uh     
17:38     
update and yeah so husin hello hey hi yeah so um yeah thanks for     
17:47     
the meeting thanks um so I mean um you know I don't have anything to say regarding Diva War updates but um I     
17:54     
could talk a bit for the uh the best cansas CD project um I think dick might be online right now I should just     
18:00     
message him he was here earlier wait is he here oh he's not the room he's not the room right     
18:06     
now okay oh so he was here he was he was here okay cool yeah thanks thanks so     
18:13     
much thank you for the discussions um small world small small worldness and um     
18:21     
like it's um like the small world Network I think um uh there there was a     
18:27     
uh there there there was one paper I came across     
18:33     
um I think it was yeah um okay I mean speaking of like     
18:40     
like the excuse me like like okay what one one thing I wanted to quickly add was     
18:46     
basically on the you know like on on the idea of groups and networks on how small how large they are     
18:53     
basically exploring the properties of whether they um in what ways they can when they expand um from small to larger     
19:00     
ones um How likely are they to to share edges     
19:07     
and in what ways are how and how likely and how highly connected are they how highly connected are they would     
19:14     
they share Loops yeah there's there's there there theories being put theorems     
19:19     
there are theorems and proofs being put forward that in small high small highly collected networks there have to be     
19:26     
Loops there must be a loop and even being able to to Define that in all     
19:31     
circumstances basically would reduce uh what I mean okay being able to prove     
19:37     
that and being able to show that would not only just be able to to make a lot of proofs in theory work in mathematics     
19:43     
and logic a lot simpler but prove very fundamental for a lot of uh structur     
19:50     
tools from a Computing and Quantum Computing point of view so it's all very very interesting yeah yeah of course we     
19:55     
know in small world networks you might know it as sort of the Six Degrees of Kevin Bacon where it's like you have     
20:02     
like a bunch of actors and that you know or people who all kind of know Kevin Bacon through six     
20:08     
associations at maximum so that speaks to these uh shortcuts that go across the     
20:14     
network you know you don't have to rely on you know um and this was something     
20:20     
actually done by um Stanley uh Stanley mgrm back in I think the 60s he didn't     
20:26     
call it small world networks but he sent a bunch of letters out to people in the mail and he said you know forward this     
20:31     
to people who you know and try to get to this you know if you think you can get to this destination back in Boston try     
20:38     
to send it to the person who you know and see if we can get the letter back and and they uh and they just looked at     
20:44     
the number of postmarks on each letter that ended up back at the destination and they ended up figuring out that it     
20:50     
took sort of on average six hops it could take more it could take less but     
20:56     
that suggests that people's social connection you know uh that relay was around     
21:02     
six uh you know stops on average which is actually for just sending out letters     
21:07     
at random actually quite impressive you know you you distribute things out on the nodes and you say how long does it     
21:14     
take to come back to the central node and it doesn't take as long as you might think even for a network of millions and     
21:21     
millions of nodes so that's that's kind of the power of small world networks now     
21:26     
there are other types of network topologies that are interesting as well and you know that's that's something we     
21:31     
can talk about later but yeah     
21:38     
absolutely okay thanks thank um so yeah     
21:45     
uh so dck and I have been looking into the loo CT Grand     
21:51     
Challenge okay so excuse me just just just taking a step back our uh     
21:59     
our algorithm R rbr RCT excuse     
22:04     
me r r algorithms multipli multiplicative algebraic     
22:09     
reconstruction two different versions of them being used optimizing the efficiency of the effectiveness of the     
22:15     
breast cancer detection uh screening techniques and protocols as um as we use     
22:21     
uh the seable JIS sphere technology between x-rays um and poap Optics     
22:29     
we want to see how effective our algorithms and our approaches are and uh in particular we've been comparing them     
22:34     
to the standards of what we found is the low do CT challenge let me just share my     
22:40     
screen all right show this low C there we go like dick was     
22:48     
asking me are we lowest do     
22:53     
um yeah getting dividing together on this so basically the the L the Lo Grand     
22:58     
Challenge would be us to see the diagnostic perance of the of different D noising and reconstruction techniques     
23:04     
for low patient CT data sets and um this was uh several years ago back in 2016 I     
23:11     
think uh we just wanted to compare our results in our data against what was shown there or to see how well we're     
23:17     
doing in comparison to the others so that's very very very important     
23:22     
um so this is could you bring that back up no problem no problem yeah     
23:29     
so I just wanted to look through some of this um yeah yeah no problem yeah so you     
23:35     
have patient data which is sort of what you're training the models on and then you have uh contrast enhanced abdominal     
23:43     
CT examinations uh all data were obtained on similar     
23:48     
models uh and then these data were acquired using a flying focal spot that's just basically describing the the     
23:54     
training data and then go down a little bit I mean this is just kind of describing how this was done not     
24:01     
necessarily you can do this with other kind of methods but then you're no inserting noise to reduce dose levels so     
24:09     
you're kind of the dosage is like giving a signal that's higher and then the noises is taking that away correct yeah     
24:17     
absolutely absolutely like um so so uh what does it talk about that yeah     
24:26     
dtive reconstruction in particular okay so uh like uh B     
24:34     
basically when when forming the image upon the screen uh we want to get rid of like aberations blur other parts of what     
24:41     
could be produced that are detracting deating from what um we need to be looking for as we uh form the screening     
24:48     
screening processes and procedures so D noising or excuse me     
24:55     
do d d noising has done a long the dec convolution so basically what for any sort of Imaging like uh I think     
25:03     
um well I mean in particular for screening for breast SC like for for FM     
25:09     
CD scans but for any sort of image image image processing and image production typically the set of steps     
25:16     
would be like calibration then removing gradients deconvolution and um depending on what     
25:22     
your workflow is D noising would be uh D noising would be somewhere there too so     
25:29     
we just wanted to compare our techniques and our approaches to what had already been done from this because this is     
25:34     
already this is long agone yeah when was this uh 2016 I think I     
25:42     
think it was 2016 okay 2017     
25:48     
yeah oh they have it right out there the Mayo Grand Challenge     
25:54     
yeah yeah 2016 so many things things to keep up with yeah yeah looks like they published     
26:01     
it though in 2017 so they actually have a paper associated with this so yeah who     
26:07     
won the Grand Challenge I wonder I think there's a CIT there something     
26:15     
I um maybe they put it on Cav yeah they did they did put on okay good     
26:23     
cool 20 GES cool cool yeah yeah EXC exciting stuff exciting stuff Happ to     
26:28     
keep you guys bu in all this um yeah yeah great yeah that looks really     
26:34     
interesting uh so on kle you have uh you can download I guess this is the     
26:40     
training data you can download or is this the yeah yeah the TR the training     
26:45     
data can be downloaded and the um yeah yeah okay excuse me um yeah it     
26:54     
can be down it can all be justed one so yeah that looks     
27:00     
great thank you okay thanks     
27:05     
thanks all right so uh that's that looks good um and so we have I don't know if     
27:12     
we have okay Pocky had to leave thank you Pocky yeah yeah in in other news like     
27:18     
the uh the work with alter learning is going very well we're being very thorough with uh plans and designs and     
27:26     
initiatives so I'm the AI team leads and uh goodness I wish um like this this is     
27:33     
for the company for alter learning for the Ed Tech startup that I'm working for so yeah are going things are going WR     
27:39     
that so well that's good morgan isn't here today but I'm gonna do something that he's interested     
27:45     
he's talked about this a number of times in our Saturday meetings and in the slack this is the     
27:52     
build to cell uh I guess project not really sure they have a lot of things they have a seminar series they have     
27:59     
workshops they have a work set of working groups and what they're doing is they're doing it's this open     
28:05     
collaboration supporting the science and engineer engineering of building synthetic cells so this is all synthetic     
28:12     
biology type stuff this is where they're trying to build cells and you can see that they're do it using their fluorescent proteins to sort of uh you     
28:21     
know verify their cells that they're building so uh so what is build a cell     
28:26     
cells are the fundamental building blocks that make up living organisms yeah we don't know exactly how cells     
28:32     
were formed in the first place we also don't know what all mo all the molecules that make up any natural     
28:38     
cell do so you know we have a lot of genes being expressed and proteins being     
28:43     
produced and things in the cytoplasm you know organel and uh structures in the     
28:51     
cell but we don't necessarily we know what they do and how they sort of interact I mean we kind of have some     
28:56     
idea but you know this is something that of course is is very hard especially when you move from cell type to cell     
29:03     
type or organism to organism uh finally we can't yet put molecules together ourselves to make new     
29:10     
synthetic cells and that's really the thing we we haven't really been able to master kind of building a cell and what     
29:16     
that means is basically you know building a cell from its components um     
29:21     
not just taking a cell and modifying it by building a cell on its own so we've had things we've done things like taken     
29:28     
bacterial cells and derived sort of a minimal genome and then basically     
29:35     
stripped out all the genes except for that minimal genome and we've had those cells uh you know we've been able to     
29:40     
culture those cells successfully we've also been able to     
29:46     
um you know do things like have a a you know a sort of a vesicle that has     
29:52     
components in it and what they call a minimal cell and you know what allow     
29:57     
that vesicle which is like an open space with a membrane allow that to have some     
30:03     
sort of metabolism and study those sorts of systems but those aren't really cells     
30:08     
in the in the sense of like a cell that you would find in your body so these are     
30:15     
you know very minimal systems that are very limited functionally so this is what they're trying to do is they're     
30:21     
trying to build something that is you know really something that we would find in the body we can use it for studying     
30:27     
disease and other things like that so um so they want of the future of     
30:32     
biotechnology is in realizing fully understood lineage agnostic organisms     
30:38     
beginning with a single cell so they want to build things that are not tied to lineage or lineage trees like what we     
30:45     
have in uh we have in seans is very lineage dependent cells have to be of a     
30:52     
certain lineage and that defines their function in mamalian cells you know     
30:57     
usually there's some uh there's some flexibility in terms of what a cell can     
31:02     
be so we can take cells and reprogram them to different Fates but uh by and     
31:07     
large you know their cells are usually restricted to some sort of plur poent set of Fates uh we do have cells that     
31:14     
are toyy potent but those are harder to work with in terms of building very     
31:19     
specialized cells so building cells is hard but if we could better work together then we should be able to build     
31:25     
many cells for many purposes so so this is an open- Source Community basically around this goal and trying to achieve     
31:33     
this goal so this this is a current but brief overview of the synthetic cell field it     
31:39     
gives you some uh information about synthetic biology and what people are doing to build synthetic cells there's     
31:46     
this paper this comprehensive road map summarizing recent progress and defining the most urgent needs of our field this     
31:53     
paper is from ACS synthetic biology um this kind of goes over um Lyn     
32:00     
rosch child is one of the authors on it she's actually a pretty big name in um astrobiology and um some of this sort of     
32:07     
stuff uh I attended a workshop last year on um was     
32:14     
simbio uh which was held at the University of Minnesota and they they     
32:19     
kind of talked about a lot of the stuff so I think I reviewed that in one of the meetings from last year so this is kind     
32:26     
of in this in this area um so this this paper kind of talks about the different     
32:32     
challenges that are sort of at the Forefront of this field the denovo construction of a living organism is a     
32:38     
compelling Vision despite the astonishing Technologies developed to modify living cells building a     
32:44     
functioning cell from scratch has yet to be accomplished so we haven't been able to build a cell from scratch and we've     
32:50     
just basically been able to take parts and manipulate them but not build something from the sort of the proteins     
32:57     
or the raw uh DNA or whatever and build a cell and so that's kind of the goal uh     
33:05     
it's worth asking whether that's going that's achievable and so that's maybe one of the challenges here the pursuit     
33:12     
of this goal alone has and will yield scientific insights affecting Fields as diverse as cell biology biotechnology     
33:20     
medicine and astrobiology multiple approaches have aimed to create biochemical systems     
33:25     
manifesting common characteristics of life such as compartmentalization metabolism     
33:32     
replication and the defined derived features Evolution responsiveness to stimuli and directed movement so we have     
33:39     
these sort of structural aspects of cells compartmentalization which means     
33:44     
that you have organel you have you know like a you have a an empty sort of     
33:51     
vesicle you have a membrane with organel you have a nucleus set forms you have um     
33:58     
you know processes that happen in the cell like DNA replication transcription     
34:05     
translation protein assembly and so forth and those sorts of things are all sort of the structural characteristics     
34:13     
of the cell you have those things in service of metabolism replication and     
34:18     
then you have things like the evolution of a cell the responsive the stimuli and     
34:23     
directed movement which we know from a lot of the stuff that we've talked about with respect to um uh epithelial sheets     
34:32     
is very important in development significant achievements in synthesizing each of these Criterion     
34:39     
have been made individually and in limited combinations uh so they review a lot of     
34:45     
this stuff and they identify the bottlenecks in the current research and they produce a road map so I don't think     
34:51     
I have access to this right now but this is something that you know if you're interested in this area would be good to     
34:58     
kind of go over and get familiar with um so they have a lot of they have a lot of     
35:04     
nice resources here this video and Engineering synthetic cells which I won't play u and I think what um Morgan     
35:12     
was highlighting is they have a seminar Series so they have the seminar series here which is uh I don't know if they     
35:19     
have the sort of the play a playlist or anything but they have these uh meeting     
35:25     
they have the workshops they have the seminar series and oh there is a YouTube channel okay so that's good and they have an ins     
35:32     
silica cell working group and a build a cell chemical reaction Network set of modeling tutorials for chemical reaction     
35:39     
networks so the Inc silica cell working group is working on these chemical reaction Network     
35:46     
tutorials the build to cell Workshop is uh so there's one coming up in Boston     
35:53     
Workshop number 13 and these workshops I guess go you know back to I don't know     
35:59     
when this might have been the one actually that I attended this is may be associated with the sin bio     
36:06     
conference um this is let's see if yeah they have I think a     
36:12     
little bit of a oh they don't really tell you what these are uh but they have breakout     
36:19     
rooms here so okay so they yeah they do a lot of interesting things at these workshops um     
36:26     
we talk about different topical areas let's see if they have anything on with working groups that may be     
36:34     
useful so they have this uh they have these working groups one is life OS     
36:40     
which is building an operating system for Life uh this is where you know     
36:45     
you're basically trying to Define this you know how to kind of have an operating system for a cell if we know     
36:52     
we remember what an operating system does for a computer it allows programs to run on a computer computer so would     
36:58     
basically interface with like maybe the genome and some of the Met uh metabolic     
37:03     
processes to sort of allow you to program things in in the     
37:09     
cell and you know this is again a metaphor so we don't know exactly how     
37:14     
what that would look like you know if there's going to be like a graphical user interface for that but that's life     
37:21     
OS then there's the in cical cell group so they're doing these uh types of things they're working on models of     
37:28     
cellular subsystems Benchmark data sets and touring tests to measure the viability of models across physical     
37:35     
scales so basically if you have like uh you know you talk about maybe how you     
37:41     
can do some sort of set of computations in a cell and you know have tests that     
37:48     
you know show the viability of some sort of computational approach uh and then     
37:53     
doing this at different physical scales so from the atomic scale to the whole cell so you know the cell ultimately is     
38:00     
composed of a bunch of atomic processes but you have to link that to things that are going on at this at the cell     
38:08     
level uh then they have visualization methods and games and competitions to     
38:14     
enable decentralized modeling I think that's more along the lines of organizing the     
38:19     
community uh interlab groups here there's ethics and bios     
38:24     
safety and there's this uh trans a group that seeks to translate these findings     
38:30     
out to sort of the medical community making sure that they get it applied the     
38:36     
essential genes group is where they look for these minimal Gene lists so basically you take a list of genes and     
38:43     
you test each gene to see if it results in a viable cell if you knock out a gene     
38:48     
and there's no viable cell then that cell that that Gene is essential if you knock out that Gene and the cell lives     
38:55     
and that's not an essential Gene and you can get rid of that and so you end up with this minimal Gene list that allows     
39:01     
you to boot up a cell in sort of its ground state and you know it isn't like     
39:07     
uh you know it isn't a very robust cell it just allows a cell to be able to be alive and maybe replicate so then once     
39:15     
you have a minimal Gene list you can then add genes to boot up different functions so if you add in like you know     
39:23     
you have a minimal cell and you add in a pathway for some sort of metabolic um enhancement that would be     
39:31     
you know taking that minimal Gene list adding to it to produce something that     
39:36     
can be you know like you can think of that as maybe a program so this is how they're kind of approaching this and     
39:42     
then there of course synthetic cell Gene repositories where they give like the recipe for this they have them sort of     
39:49     
in a repository here and so that's uh those are the working groups for build cells so that's     
39:56     
I think useful in kind of thinking about how people are approaching this and you     
40:02     
know the kinds of challenges that exist in this field so again this is the build a cell     
40:10     
group and uh you know well there's Susan and uh Morgan is here so Morgan yeah we     
40:16     
were just talking about build the cell um I don't know if you had anything to add with respect to     
40:24     
that seems like there's a lot more than just the seminar series     
40:34     
series okay um so hello Susan I don't know if you're available to talk     
40:43     
but wonder if you had any updates you wanted to     
40:48     
give uh sorry I'm just um taking a note on build a cell okay so um     
40:57     
yeah sorry I'm late it's a holiday here so we're um I've stepped in     
41:06     
actually um updates I don't know C Soul's still     
41:12     
being it's not giving me results I watch YouTube videos and it says it works this     
41:18     
way but I think it's for an older um C Soul um and I my help desk has decided     
41:27     
to quit entirely it says 404 error     
41:32     
no so like I I really need to do chat GPT     
41:39     
today yeah it'll give me that one last little     
41:46     
hint yeah I just wish wish I could talk to someone who used has used the program     
41:55     
yeah it's always hard yeah anyway nothing     
42:01     
much doing a lot of gardening instead of of homework     
42:10     
yeah anyways yeah yeah and Morgan said that yeah it's     
42:15     
really nice seminar series build the cell and yeah so yeah go     
42:23     
ahead um yeah do you have the URL for it     
42:28     
[Music] yeah build as sell.or let me put it in     
42:33     
the chat there you     
42:40     
go all right so the last thing I want to talk about today was um this paper on     
42:46     
spectral decomposition of embryos it's really interesting work um so this is     
42:54     
actually an eife uh the way they do eif now is they have like they they allow you to pre-print     
43:00     
things and then they have they review them and then eventually if you get published it becomes like a published     
43:07     
paper so it's it's it's a like kind of a a transparent peerreview model so I     
43:13     
think they're at the stage of like being reviewed pre-print but not hasn't been revised so but I think this is a really     
43:19     
interesting paper it kind of goes through some interesting things to do that you can do in embryos to look at     
43:25     
like you know space variation and some of the information that you might     
43:31     
have you know break down what what's going on in a embryo into this sort of     
43:37     
frequency spectrum and then decomposing that and understanding you know different types of     
43:43     
morphogenesis so this paper is spectral decomposition unlocks a cidian     
43:49     
morphogenesis and aidans are otherwise known as C Squirts and aside from cgans and another     
43:56     
organism that exhibit a lot of similar traits especially in early morphogenesis or early     
44:03     
embryogenesis is the SE squirt or the aidian and fortunately we have a lot of     
44:09     
data on this early stage of Aidan morphogenesis and it's something that we've explored in uh papers in the group     
44:17     
here uh we had a paper from 2016 where we compared a cidian morphogenesis and     
44:22     
seans morphogenesis so there's this like celegans you have these cells that uh     
44:30     
start with a single cell and they differentiate or they divide into these U sort of clonal uh cell types there's a     
44:39     
very specific uh nomenclature for the cells and it's predictable highly     
44:45     
predictable from an embryo to the ne so you can look at this early morphogenesis that results in this large     
44:51     
ball of cells eventually though aidans are going to uh you know the Aidan embryo is going to differentiate into     
44:58     
this long tubular uh tadpole that actually uh     
45:05     
doesn't look anything like its early morphogenesis but the part of morphogenesis that's most interesting at     
45:12     
least from the standpoint of this spectral decomposition technique is this very early stage so keep that in mind     
45:18     
when I show you the images because it's you know just G gives a little bit more     
45:24     
context so um this group here Patrick lamir and some of these edman R they     
45:30     
work on this uh a cityan morphogenesis and this community like the seans community they have a lot of good open     
45:37     
data sets for what they're doing so that's good too uh so the abstract reads describing     
45:44     
morphogenesis generally consists in aggregating the multiple high resolution space your temporal proc processes     
45:52     
involved in reproducible low dimensional morphological processes consistent Ross     
45:57     
individuals of the same species or group so they're basically taking uh different     
46:03     
spatio temporal processes and then you're trying to reproduce some sort of statement about     
46:11     
morphogenesis and you know you can't really do maybe try to do it across species but usually it's species     
46:18     
specific uh that's what they're getting off this sentence uh in order to achieve this goal biologists often have to submit     
46:25     
movies issued from live Imaging of developing embryos so this is kind of the standard that people use you know we     
46:33     
take uh microscopy movies a lot and those are usually the most informative ways to look at uh morphogenesis we     
46:40     
certainly do that with celegans and what we do when we want to do machine learning is we take those movies take it     
46:46     
frame by frame and train the model on those movements or whatever that we're looking at so we're looking at division     
46:53     
events we're looking at movements within the embryo we can take from this time series of images and like describe the     
47:00     
movement from frame to frame so how much is something displaced from frame to frame that gives us a good indicator of     
47:08     
like movement or of Dynamics so what they're saying is that that's sort of the standard in the field     
47:15     
uh these approaching however present noticeable drawbacks as they can be timec consuming hence unfit for scale     
47:22     
and often like standardization in a firm foundation so these are all challenges we've seen the seigan data     
47:28     
sets in this work we Leverage The Power of a Continuum mechanics approach and     
47:34     
flex flexibility of spectral decompositions to propose a standardized framework for automatic detection and     
47:40     
timing and morphological processes so they're looking at a Continuum mechanics approach which means taking the uh     
47:48     
embryo and the changes in the embryo and putting it into this lran model uh which     
47:53     
is a Continuum mechanics model so you write a Gran out and what Gran describes     
47:59     
movement in space and then you can use that type of a model to drive you know     
48:05     
you can derive things like like we talked about last week you can derive uh calculus and you can derive other types     
48:11     
of continuous uh mathematics to analyze these so in this case they're looking at     
48:18     
spectral decompositions and how to take that time series or that that those those Dynamics     
48:25     
and describe it as as like a spectral uh like a power Spectrum or something and     
48:31     
then decompose the information there to get you know some to say something about     
48:39     
morphogenesis first we quantify whole embryo scale shape changes in developing Aidan embryos by statistically     
48:46     
estimating the strainer tensor field so this is something that we know we're interested in uh and it's of its time     
48:53     
evolving surface without the requirement of cellular segmentation so you know a lot of times in the     
48:58     
celegans stuff we're doing with the machine learning is we're taking apart these movies we're kind of using     
49:05     
segmentation algorithms to find features those features and we link those features across the frames and time we     
49:13     
maybe count the centroid and we say what's the displacement of a centroid from frame to frame and then we get this     
49:20     
we so we use these cell wheel segmentation uh you know techniques as a guide to how to track things in in the     
49:28     
movie how to track things over time what they're saying in their approach is that you can use a strain ret tensor field to     
49:35     
build this time evolving surface that gives you similar information or maybe different     
49:41     
information uh and so you you know this is something that you can use as well it's more mathematically uh complex but     
49:48     
it gives you a good uh it's a good approach for looking at these things we     
49:53     
then apply to this data spectral decomposition in space using spherical harmonics and time and     
50:00     
using Wave transforms so they use a combination of spherical harmonics and wavelet transforms to get the     
50:06     
information out so wavelet transforms are you know where you take a Time series and you decompose it into its     
50:14     
high frequency and low frequency components you do this with different types of mathematical uh     
50:21     
transforms um and you do this at different scales so you can take like a very long time series and decompose it     
50:29     
using decompose it using these high and low frequency filters that give you that     
50:35     
sort of emphasize the high frequency and the low frequency components of the time series and it gives you sort of this     
50:41     
characterization of maybe you know like in in a very small uh time series uh you know it     
50:50     
summarizes every the basic changes so for high frequency events it emphasizes     
50:55     
those High frequen queny events it shows if there are key high frequency events occurring in the time series that'll     
51:02     
kind of identify the sort of the uh part of the time series where that's happening and the same with the     
51:08     
low frequency components so that's you know that's a that's a nice approach but     
51:14     
it's very hard to implement and that's kind of what where they're going to go with this you know you have to use kind     
51:20     
of a system like a cidian or celegans to do this otherwise I think it would probably not work very well     
51:26     
these Transformations result in the identification of principal dynamical     
51:32     
modes of a cidian embryogenesis and the automatic unveiling of its blueprint in     
51:37     
the form of spectrograms that tell the story of a cityan embryos uh so that's that's what they're     
51:44     
doing they're building these spectrograms from these principal dynamical modes so this is a lot of uh     
51:51     
sort of dynamical systems analysis and looking at the Dynamics of the embryos     
51:56     
and instead of using the kind of features that we're used to using the feature the feature-based approach is     
52:02     
easier to implement but you know it doesn't give us the same uh     
52:08     
information so this is uh so yeah they they talk about this approach     
52:13     
embryogenesis can be described as a series of stages delineated in space and time by the identified landmarks and     
52:19     
that's sort of the standard in the field and so you know this is sort of     
52:25     
congruent with our a feature-based approach because you're basically finding these landmarks in the embryos     
52:31     
uh morphology using those landmarks then to connect between the landmarks and you     
52:37     
know look at how the embryo changes and that's something that you know can give     
52:42     
you some information but not the full story so um yeah they go through some of the     
52:49     
standardized methods the the sort of contemporary methods and then they kind of talk about some of the challenges of     
52:56     
that because a lot of the processes in morphogenesis are inherently multiscale     
53:02     
and this is true both in spatial and temporal domains which you know these multi-kill     
53:08     
processes also interact and overlap and as in the case of several     
53:13     
species acid and gastrulation is a playground featuring important displays of sub divisions and tissue mechanics     
53:20     
most notably endoderm and vagination so there are these processes that occur these Collective     
53:27     
processes that are collective Behavior across cells that were interested in these kind of large scale movements that     
53:33     
are hard to capture using this sort of landmark or feature-based approach     
53:41     
so uh you know they're basically trying to get local trying to differentiate     
53:46     
between local and Global sort of behaviors and and make distinctions between     
53:52     
them um yeah spec then they talk they talk up spectral Decon composition and     
53:57     
these are the references here Romeo at l21 and delmo at l21 spectral     
54:03     
decomposition where a signal was broken down into its canonical components as well studed the study of systems that     
54:09     
exhibit multimodal behaviors the benefits are at least twofold the first is that individual     
54:16     
constituents may represent distinct dynamical processes thereby enabling the     
54:21     
decoupling of physical processes entangled in the data and two only a handful of     
54:27     
components May significantly contribute to the original function resulting in a compressed lower dimensional     
54:33     
representation that capture the main features of the study process so you have this thing where you     
54:40     
have parts and they can contribute to the hole on their own but you also have     
54:47     
these situations where you have a handful of components that contribute more than other components to a global     
54:55     
process so you can actually extract those the the effects of those     
55:00     
components or you can look at sort of this you know the individual components kind of     
55:06     
additively so there are a lot of things you can do here to sort of get information out of the embryo so without     
55:13     
further Ado I'm going to go down here and I'm going to so they use these little Gran markers on the surface of     
55:19     
the embryo so they build this lran model with the embryo and this is a Continuum description so it allows us to have The     
55:26     
Strain rate field and and you know uh characterize the embryo as a strain rate     
55:31     
field so things are happening in this field across different cells and you're getting this uh model that is continuous     
55:39     
rather than discrete so I want to go down to the images this is the acian embryo so you     
55:47     
can see it's this clump of cells and this clump of cells actually has uh many     
55:55     
fold uh symmetry so you have I think it's four-fold symmetry where you have     
56:00     
each quarter of this sphere has sort of the same cells the same structure the     
56:08     
you know uh the divisions happen uh by they're binary divisions but they end up     
56:13     
giving you this four-fold structure um so this is an interesting feature of this embryo so just to keep     
56:20     
that in mind uh so the first stage here is the level set scheme for evolving surface generation     
56:26     
they build this strain field here they have a a a dynamic they they model the     
56:33     
Dynamics of this equation here and this is the map then they have this raw they     
56:38     
have the raw data where you get a resulting mesh so they're trying to render an embryo surface from the raw     
56:44     
data you have the scheme for evolving surface generation so now once you have the data and you have the surface     
56:50     
generation model then you can validate this by tracking cellular Junctions so     
56:56     
this is something that is relevant what Susan's doing you have the cellular Junctions in red where you have the     
57:04     
where the cells meet together so remember you have the raw     
57:09     
data from the cells you have this mesh that you build so you're not just relying on features and they centroids     
57:16     
and then you have this model of sort of a strain field so things are being transmitted across the different parts     
57:23     
of the embryo not necessarily the centroids but things like at cell Junctions and so here they have the cell     
57:30     
Junctions from raw data they plot this on the mesh so they're able to measure using the strain Fields the cell     
57:37     
Junctions and they're able to put you know uh apply this to the mesh um then     
57:43     
you have these simulated cell Junctions so you can both take things from raw data and simulate cell Junctions and     
57:50     
then you can find the little BR and error between these two components so     
57:55     
you have the cell Junctions on Raw data the simulated cell Junctions and then you have this lran error so the L Gran     
58:03     
will build you this time series where you have the cell Junctions and sated cell Junctions and they diverge and so     
58:10     
there's this error that you get and this shows the error between the simulation and the     
58:16     
observation so you see the two components the red component here and the green component here in the     
58:21     
equation and so that's that's what it looks like and then you can do some     
58:27     
interesting things so you can plot out relative displacement between cellular Junctions and their markers over time so     
58:34     
you have the time component here this is a Time series where you see the average displacement you have the cell count     
58:40     
that goes up so this is one of these graphs where the average displacement is on the left and the number of cells is     
58:47     
on the right so as the number of cells increases in this model it goes from zero to about 256 cells roughly and this     
58:55     
is just just in time this is um you know from four to 5.75 so it's like a     
59:02     
fraction of this time series and so you get like capture most of the division     
59:08     
events in this time series but you can see the average relative displacement fluctuates quite a bit during this     
59:14     
process and in fact it looks like there's a Aver sort of a peak in the average relative     
59:20     
displacement right before there's a burst of division activity so you can see well at least for here here you get     
59:27     
this burst of division activity whereas here you don't see that as well but you see it here and here so you get these     
59:34     
interesting you know signatures that are maybe you know something to follow up on     
59:39     
ler so you get some really interesting findings of these kind of models um so these are what they call     
59:46     
level set inspired lran markers um that's what they're doing up at the top and then they're applying a     
59:54     
mesh and then they're able to both observe things from the data and simulate these things and then they're     
1:00:00     
able to findian error which gives us these kind of graphs uh the second uh thing here is     
1:00:09     
where they talk about the strain rate field and it describes a city in morphogenesis a correct parameterization     
1:00:15     
of the Velocity field at every position requires three coordinates and this is in VX you have VXX vyx and vzx so you     
1:00:25     
have have this uh this basically this three-dimensional model The Strain field     
1:00:32     
is derived as a symmetric part of the discrete gradient of the Velocity field as described in Manelli     
1:00:38     
2018 intuitively The Strain rate field uh The Strain rate evaluated on a given mesh vertex measures how the velocity     
1:00:46     
Vector varies in the neighborhood this point so this is uh the point here and     
1:00:52     
this is I guess uh how the uh velocity Vector varies depending on     
1:00:59     
the Symmetry and what's going on locally in in the embryo so this is you know     
1:01:04     
allows us to look at these uh local variations in the embryo um and then they talk about the     
1:01:12     
mathematical construction of these different axes a diagonal elements of the Matrix capture the linear strain     
1:01:18     
rate in the XYZ axis depend depicting the change in length per unit length uh     
1:01:24     
the nine diagonal element that stand for the Shar strain rates in the XY XZ and YZ directions because D is symmetric up     
1:01:32     
in this equation here uh there exists an alternative representation which holds     
1:01:37     
stronger local geometric meaning so you have this geometric local geometric     
1:01:43     
aspect that's really interesting um and so this is another image here where you have the strain R     
1:01:48     
tensor field or the the tensor strain rate field and that's here for these     
1:01:54     
different uh components so you can see that this is an example of this one here so this is an example     
1:02:00     
of the string rate tensor field across the entire embryo then you have a scalar strain     
1:02:06     
rate field which is this here so these are two different measurements the tensor strain rate field given by this     
1:02:12     
equation and the scaler strain rate field given by this equation and you can see that it gives you these different     
1:02:18     
pieces of information and this is all you know this is a sort of a phase portrait of     
1:02:24     
the embryo so you can get uh you know different information from this uh for different points in     
1:02:31     
time uh so the strain rate tensor field measures the rate at which morphological     
1:02:36     
changes occur and the embryo is a function of time the strain rate tensor field is locally represented as a 3X3     
1:02:43     
symmetric Matrix and is completely determined by its igen Vector Fields so     
1:02:48     
this is a heat map in a and then in B we have a heat map of     
1:02:54     
the scaler string range field so these are heat Maps but you can use them to determine dynamical     
1:03:01     
modes As You Wish um and so yeah they they give more information about their     
1:03:06     
methods here and they talk about spherical harmonics revealing the M mov     
1:03:11     
of a cityan morphogenesis so in order to capture the relevant features of the string rate     
1:03:17     
field in the spatial domain that can conduct a spectral analysis of the scaler strin rate field so this is uh     
1:03:25     
down here we have sort of this decomposition so I think these are the     
1:03:31     
different um let's     
1:03:37     
see yeah these are the different     
1:03:43     
um these are the different spherical harmonics here so you have these     
1:03:48     
different modes where you see these different uh sets of values so you you     
1:03:53     
know you kind of average it out I think basically down to L equals z l equals 2 and you     
1:03:59     
can get different you can see what they look like at different scales um you're decomposing the     
1:04:04     
spherical harmonics and then you have this dominant modes of morphogenesis where you have the variance ratio is is     
1:04:11     
a function of percent and the number of cells so we have y z0 and y10 and we can     
1:04:17     
see what those behave like given the division the number of division events the     
1:04:23     
time then we have this time zies of F0 0 and F10 which has they're they're kind     
1:04:28     
of consistent but you know they're uh they're value here in this in this y-     
1:04:35     
axis is a little bit different so it's it's interesting the kind of information you get out of     
1:04:42     
this so I think in summary we can look at the wavelet transform so this is figure four of the wavelet transform     
1:04:49     
where we have F z00 which is this this for gastation and neurulation period so     
1:04:56     
these are different periods of of uh of embryogenesis where you get these     
1:05:01     
different types of uh folding and so neurulation is the formation of the neural tube gastation is a type of     
1:05:09     
invagination where you're getting changing the shape of the embryo so this is the wavelet transform and again     
1:05:16     
you're you're doing this wavelet transform and you're taking these components and you're plotting them out     
1:05:21     
in this kind of map where you can actually identify uh these different dynamical modes so     
1:05:29     
you get the duration in hours here so you go from the top which is the one cell stage down to four hours and then     
1:05:37     
you get this uh these hpf values which are from four to 8.4 so you get this invagination here in     
1:05:45     
this region you get zippering you get synchronized divisions up here because you get this at the early stages of of     
1:05:52     
cell division you get these synchronized divisions that you can actually where we see here and then you get this change in     
1:05:59     
shape so you get these changes in in Mass which are you know symmetrical and then you get these     
1:06:06     
asymmetries like invagination and zippering that occur and you can see the modes here the same thing here where you     
1:06:12     
get these different movements you get synchronized divisions up at the top at the beginning of embryogenesis and then     
1:06:18     
you get things like endoderm apical apical basil shortening and mism cell     
1:06:23     
movements you can see those sort of these phases you have the synchronized     
1:06:29     
divisions again and you get endoderm plus mism morphogenesis down here so you start to get     
1:06:34     
differentiation as a function you know as you go longer into the process and then this is for evagination     
1:06:42     
period here for Ft uh 20 which is these are I think     
1:06:47     
decompositions of the uh spectral decompositions uh and and then putting     
1:06:52     
that into the wavelet transform so you're doing two t of decompositions you're doing this type of     
1:06:58     
uh spect spherical harmonic decomposition and you're doing the wavelet uh transform that which also     
1:07:05     
decomposes the time series and so you end up with this apical endoderm apical constriction and AP endoderm apical     
1:07:12     
basil shortening which you can identify in the same manner that you can identify these other kinds of global     
1:07:21     
processes and so this is a really interesting paper I think uh kind of describing a different way to do this     
1:07:27     
kind of analysis I said it's a pretty mathematically involved way but I think     
1:07:32     
you know if you're you think about the physics of the embryo and thinking about like modeling things from single cells     
1:07:38     
to these large scale movements and other things it's a valuable approach okay and do they have pictures     
1:07:45     
of the inside of the embryo uh I think they're just modeling the outside I mean the inside is I think basically     
1:07:52     
symmetrical I'm not sure how there you know when you look at these data sets they kind of describe the cells as these     
1:07:59     
uh at the stage that we're looking at as these uh fourfold symmetric structures     
1:08:05     
so each quarter of the embryo has certain cells in place and then they have you know their their four of those     
1:08:12     
and the outside is the thing that's going to change so as you get these movements it's the outside of the the     
1:08:18     
cell mass that changes and so that's their modeling what their they need to look at the inside too um and the     
1:08:26     
technique for doing that has been around for five years what is that Optical coherence     
1:08:33     
tomography oh yeah yeah yeah and as the     
1:08:39     
inside um becomes flu fi um that affects the cells on the     
1:08:45     
outside um it's it's a very important part of this yeah yeah so they're     
1:08:51     
analyzing something but they haven't looked at the density of the embryo or     
1:08:57     
the um pressured on the inside of the embryo um just think it would be good to     
1:09:04     
put with that data set yeah it would be and they they don't talk about the sort of the material aspects too much they     
1:09:12     
just kind of talk about the I'm glad they have cell Js those are important yeah I thought you'd like that yeah yeah     
1:09:20     
yeah I like that but there's still like it's sort of like     
1:09:26     
oops they missed [Laughter]     
1:09:31     
something oh well yeah okay uh well I think that's all for     
1:09:38     
today uh do we have any other comments or     
1:09:45     
questions okay well look at the recording because I     
1:09:52     
missed the first part oh yeah okay well thanks for attending and uh     
1:09:59     
have a good week thank you bye thanks bye bye     
