     
Transcript     
0:00     
hello hello hello how are you hi I might be eating breakfast while I watch     
0:08     
fine just don't get too much on your shirt that's fine     
0:15     
okay Morgan hello     
0:20     
Morgan hello hi right I will also be eating breakfast     
0:26     
oh yeah same to you same advice I mean it's yeah all right so uh yeah welcome uh to     
0:35     
the meeting Susan hi Susan     
0:40     
hi uh my camera is not working today okay so how is how's everything going     
0:46     
with your projects and well I've um my data made it some     
0:52     
interesting graphs so I'm just going to go with     
0:57     
that so I have a he basically a mechanical     
1:03     
model of um very stable tissue because it's all um     
1:10     
hexagons and that particular model makes uh some     
1:17     
graphs that look like what happens in reality so that's nice     
1:24     
yeah always good to see your data conform to reality or your simulations     
1:34     
maybe it's the opposite reality conforming to simulations     
1:42     
yeah anyways um yeah I'm going to check with     
1:50     
them this week to see if I need to um add more     
1:56     
tests like all I've got is uh Force and then um the mount it     
2:03     
moves and um yeah so I don't know if that's enough like do I need um instead     
2:10     
of a force in the Z direction do I need to pull it sideways or something but I     
2:17     
don't know yeah he might need visco L isn't     
2:24     
he do I need visco elasticity okay maybe oh it's um it's a     
2:33     
tensity so it automatically has some of that built     
2:39     
in yeah but the uh time course of visal     
2:44     
lasticity uh in uh outed cytoplasm might be quite     
2:50     
significant yeah well it would probably stabilize the other models that I have     
2:55     
that are not quite stable if I can put them in a fluid     
3:01     
um calmo is a bit difficult they want you to convert everything to an actual     
3:08     
real solid model um instead of a     
3:14     
truss before it'll interact with other things like     
3:20     
fluid so I'm not sure if I can just take the trust model and and add a fluid to     
3:27     
it it probably won't interact they probably have to change the dress to     
3:32     
steel and aluminum or     
3:37     
whatever okay yeah who they actually give you non-biological materials huh     
3:45     
well what are you going to do they the biological materials don't have a     
3:52     
proper stress drain curve they go with them like proper Young's modulus and     
3:58     
what is it like there's you can they've done um actin for instance and and I've     
4:06     
also done micro two wheels and there's a nice stress string curve of that but     
4:12     
that's not what it is because it is a uh there are linkages between the act     
4:18     
and they form a a belt so yeah but is the stress train curve of the actum belt     
4:25     
actamin belt for that matter     
4:30     
nobody knows nobody knows     
4:37     
okays like maybe someone's done some studies on it I don't I mean I guess     
4:43     
it's like thiser a phenoma yeah I have I have a     
4:49     
recent paper about um yeah well what they call it uh n networks yeah n yeah     
5:02     
um oh well um I this is a Windows 11 machine     
5:10     
now and uh so it just did something stupid     
5:15     
so um yeah I just had it     
5:24     
um very great um sorry about the that     
5:29     
okay quantifying the not notin of Tangled     
5:35     
networks okay that sounds pretty good yeah that's     
5:42     
uh that's one thing okay that's not that was not it     
5:51     
um all right uh let's     
5:56     
see chapter one introduction to biopolymers and their     
6:03     
networks okay something like that they're biopol Network there we     
6:10     
go okay so they're different than just the additive effects of like act hermas     
6:18     
and well they're they've got what is it Cal modulin in     
6:23     
between um and other things in between yeah molecules     
6:31     
you got lovely um images of them people have gone to some work to do that     
6:39     
but the physical properties of it who     
6:47     
knows look this up yeah     
6:53     
so let's pretend it's like sort of like a steel cable right right yeah     
7:03     
so I'm not too sure what to do with it     
7:09     
um that out in in my write up yeah yeah I just mentioned it in passing     
7:17     
I guess and say that maybe an area for future research yes is what is missing     
7:25     
from uh the literature so uh I can't go any further with in that direction     
7:35     
yeah let yeah it sounds good you're getting to know the     
7:45     
problem all right looks like mahul and Jesse have joined     
7:50     
us little Mah Jessie bradle yeah hi badley I     
7:58     
every uh uh so last time we were actually talking about uh node and layer Duality and like     
8:05     
somewhat uh in in in terms of hypergraph so uh just for the context for everyone     
8:11     
uh Bradley has shared me a paper so I was thinking that if you can if I can present uh few of my learnings I don't     
8:17     
have much updates on the coding part but I read that paper and it was very interesting but quite high level view     
8:24     
let me share my screen all right ahead     
8:34     
yeah I hope my screen is visible yes yeah so this is the paper which     
8:40     
bradle shared me uh from nature communication node Duality Network system so it's a recent paper somewhat     
8:46     
very recent paper I mean July so uh uh what it was so this is basically not a     
8:53     
hypergraph paper but a graph paper and which is talking about uh so for the context     
9:00     
uh uh let's say we have a graph of something like this where connectivity are like this and then we have another     
9:08     
graph of maybe the same uh of a same object but a different data or from a     
9:14     
different angle uh and it is somewhat like this like in the second part and     
9:22     
there can be some interactions between the two kind of imagings so if like we     
9:27     
can consider elephant uh in the above graph someone is looking at the elephant from a different context from a top view     
9:34     
and in the second image they looking from a side view and then there can be some interconnections between them so     
9:39     
this is not uh how this paper described it but that is what how I was like trying to understand um so this look     
9:47     
very interesting I made some nodes on this I'll just go through them once uh     
9:53     
yeah so now what is like node layer similarity node duality in so uh so     
10:00     
traditional network analysis like focuses only on the nodes right like when we have go nodes in the graph and     
10:06     
their interaction so this new approach which is node layer similarity propos considering different layers also like     
10:12     
different graphs representing the same object but different types of interactions or     
10:18     
relationships uh and what what they're proposing is uh instead of like this uh     
10:24     
they are going to Interchange this node uh interactions with the layer interactions     
10:29     
I'll show how it looks like and but what this says is this Duality allows for a dual perspective of the same network     
10:36     
providing like richer understanding of the structure and dynamics of what's going on in the     
10:41     
scene so uh there can be many uh examples so one of like in social     
10:47     
networks and all and brain Etc so one example which this paper also focuses on     
10:52     
is uh uh understanding these networks in of the Alzheimer we come to that     
11:00     
so to represent this type of graph they they said something called super ad     
11:06     
symatrix which they did not explain very much but uh I'll try to explain what I understand um so uh for the first graph     
11:14     
which I described we will have an adjacency Matrix right so there are three nodes so three nodes will have adjacency Matrix of 3 three and uh this     
11:22     
uh the second perspective will also have like three cross three which we can call a second layer uh so the information     
11:30     
representing these connections and these connection also the interconnections between     
11:36     
them yeah so the interactions between uh these two layers uh including these     
11:43     
graph interactions will be represented by this supery Matrix uh which is quite     
11:48     
complex like I was seeing a paper for that uh but we don't need to go into it     
11:53     
that much mathematically we will just get an idea so now by Duality what they're doing is uh so you can see the     
12:00     
second graph in which they are interchanging this type of no so now for every node they have made it like layer     
12:06     
so we had one to three nod right so now they have made three layers and they have uh uh made these layers as the node     
12:14     
so we had one and two layers in the primary image and in the transformed image they have made it as nodes instead     
12:20     
so like mathematically what we can do is we can just transpose ad Matrix very     
12:26     
simply yeah so yeah and then uh this is their concept and they have explained     
12:32     
through examples I think couple of them but there was an example of Alzheimer's disease and that is what I understand like little bit brain Imaging so I read     
12:39     
that example uh uh so basically what they are going to do is this no layer     
12:46     
Duality framework uh for that the data which they're using is M signals uh which is capturing the interaction     
12:52     
between different rois and these interactions are captured for different frequencies of the brain signals uh or     
12:58     
we can say as uh frequencies can be set as layers in this particular instance and then uh     
13:05     
they are using some like method called bispectral coherence uh uh to get the to     
13:11     
get the weighted interactions among the rois but also the interactions between these frequencies I'm not really sure     
13:17     
how they are obtaining it um but uh that is what they said they are     
13:23     
doing yeah then there were few outcomes from uh like what they propos that was     
13:29     
very interesting so what they proposed was uh from the layerwise perspective uh the frequency band they give a frequency     
13:36     
band of 8 to 13 Herz uh uh so these frequency bands had lower connectivity     
13:42     
in ad patients compared to the healthy uh control uh patient so that was very interesting study instead of focusing     
13:48     
more on like uh Roi connection which is Regions connections instead the frequency connections were actually very     
13:55     
prominent in deciphering the difference between AD patients and     
14:00     
and second point was yeah so that is what mentioned differences in frequency connectivity are more pronounced and     
14:06     
they can better discriminate ad patients and healthy controls uh compared to the region wise connectivity differences     
14:13     
otherwise uh both node and layer centrality correlate like significantly with how severely uh with the severity     
14:21     
of a cognitive m in ad patient this was measured by this mm so this is like a     
14:27     
popular method ad of FR so uh this paper     
14:32     
actually gave a very high level high level overview I did not check the code I think they had a code code repository     
14:37     
also uh but uh for me it was very high level view because maybe because I'm     
14:43     
like not very aware of the literature but it looked like very interesting idea so I was thinking how we can uh put it     
14:49     
in our case like of C Elegance so in C Elegance we have one hyper graph is of     
14:55     
like spatial distance like where we are catching the interactions BAS based on the distance if the distance is lesser     
15:00     
then we have stronger interactions between those two cells uh and the second type is uh is like based on     
15:07     
hierarchy so if like a mother has like a daughter and those daughters have like     
15:12     
further daughters so that mother or grandmother will have like some connections and based on that we are     
15:17     
making hyper graph so uh now we have two layers of hypergraph made right now I'm     
15:23     
not sure how to make let me switch my camera on     
15:30     
I'm like I'm I have I'm very visible so now we have two layers now we     
15:37     
have like two uh layers of hypergraph created one through the facial and one through the uh this mother daughter     
15:43     
relationship and I don't know how to how how will these two hyper graph these two layers of hyper graph will have     
15:49     
interrelations between so but otherwise this was uh     
15:54     
interesting paper and probably we can utilize uh some in other     
16:00     
ways yeah I like this uh paper thank you for going over it m uh so yeah I guess uh yeah I had a     
16:10     
couple of questions could you go down through the uh the figures let's look at     
16:15     
so that we talked about one and then about two what does that look like     
16:20     
yeah so that kind of shows like the nodewise distance and layer-wise distance so you have those kind of     
16:27     
measures so the the nodewise distance is kind of like the normal graph like if     
16:32     
you had a spatial set of spatial relationships the layerwise would be like a lineage in our case and     
16:40     
c and you could plot those out kind of as a like a comparison uh between them     
16:46     
so you could say like this if there's a lineage distance of two uh you know     
16:53     
daughters or two generations then the nodewise distance does a change     
16:59     
in other words do you get this distance do you get migration or do you get a difference in position relative to one     
17:07     
another and then you could use that for other things you could look at like spatial distance you could look at like     
17:12     
transcriptional distance too in that way uh or you could have like you know maybe     
17:18     
three layers where you have transcriptional distance on the third axis but that's that's you know that's     
17:25     
something that' be a little bit harder to wrap your head around what that means     
17:30     
um yeah so they're these complimentary properties so basically what they're doing is they're building this set of     
17:37     
layers you know people do this with multi-level graphs there's this area of network called multi-layer     
17:45     
Network um and you know people will do this for certain problems where they'll say I have a network maybe that evolves     
17:51     
over time where I have this you know um this network is embedded in like a     
17:57     
discrete process where where we need to have you know uh we don't want to necessarily look at the evolution of the     
18:04     
graph by adding and removing nodes we want to take snapshots of the graph and     
18:09     
then look at the differences between those discrete points so you know some discrete s uh discrete system you can     
18:16     
think of that would be an example of that um so that's that's kind of where they're actually where they're coming     
18:22     
from on this in terms of like hyper graphs I don't I'm not sure how that would you know uh how that would all     
18:29     
plug in I did a a bit of uh I did like about 20 minutes at the end of last     
18:34     
week's meeting on uh different uh graph measurements and different types of ways     
18:40     
to look at graphs uh connectivity and so you know there's like connectivity in     
18:47     
network in in graphs and then connectivity in hypergraphs and so the     
18:52     
difference being of course is that you have these nodes within a hyper node and there's sort of contivity within that     
18:59     
but then there's this connectivity between hyper noes and it actually is     
19:05     
very similar to say like you know a network where you have high modularity     
19:12     
where you have modules that are inter tightly interconnected then you have these very weak connections between them     
19:20     
and so if you cut them you end up you can end up like disrupting the network     
19:25     
you can disrupt Network flows of information of of whatever is in the network and then you know that of course     
19:33     
that that's you know that's uh relevant to hyper networks or hypergraphs because     
19:38     
you have this sort of uh you know you have this Duality where you have     
19:43     
processes going on inside the hyper noodes and then processes going on between     
19:48     
hypernodes so that's that's you know so there are a lot of sort of dualities here when you start getting into these     
19:55     
kind of layered networks or hyper graphs it's it gets a little bit tricky to kind     
20:00     
of keep everything straight that's why people have developed these methods because you know otherwise we can't     
20:06     
really describe systems that are say hierarchical or have complexity at     
20:12     
multiple spatial scales or whatever I have a doubt actually uh you     
20:20     
mentioned the transcriptional data uh what do you mean exactly about that well if you you know there's of course uh     
20:27     
transcriptional data which is of course gene expression so there are all sorts of     
20:32     
databases uh where you can look at the expression of a certain Gene and in seans they've done this for you know     
20:39     
single cells although they don't have a very adequate uh single cell Atlas in     
20:45     
development like say we have a gene and it's in maybe two different cells and we     
20:53     
want to look at its expression and we want to say like something like if I have a neuronal cell     
20:59     
the transcription is going to be higher than in like a um like an intestinal     
21:05     
cell and so what we can do is we can take those two cells and we can look at their transcriptional distance which is     
21:10     
the distance between the expression value so if I have something that's say     
21:16     
in the neuron four-fold oated ver is the twofold in the um intestinal cell then     
21:22     
we have a transcriptional distance there so we have spatial distance transcriptional distance but we can     
21:28     
basic you know we can't necessarily correlate them but we can have this sort     
21:33     
of you know maybe there's a pattern there because you're going to have differentiated cells they tend to be     
21:39     
maybe clustered together in a tissue and so that might be that you have like a hyper node of neural cells a hyper node     
21:48     
of intestinal cells or epithelial cells and that those connect you know uh     
21:53     
because they're similarly in space they have like a similar location in space space and then the transcriptional     
22:00     
distance maybe will maybe correlate a little bit with that because those cells     
22:05     
tend to have the same transcriptional profile that's what I mean by that     
22:12     
um that's PR interesting yeah yeah so but anyways yeah that's that's figure     
22:17     
two why don't we go through the rest of the figures here uh figure three Is this     
22:22     
different size effects your multi-layer multilex configurations     
22:28     
so this is a multi-layer versus n minus M uh and this is uh n is the number of     
22:36     
nodes n equal to the number of layers M so you have nodes minus     
22:42     
layers and then you have multi-layer and multilex multilex of course is uh     
22:50     
different than multi-layer I think multilex is where there's interactions between the different     
22:55     
layers um and multi-layers is simply where the maybe connections that are not like you     
23:02     
know there's not a dependency there I'm never really sure what they mean by Multiplex in this case but interest an     
23:08     
interesting set of relationships yeah yeah maybe Multiplex is like the     
23:14     
second uh in the first image the second figure okay yeah yeah like     
23:21     
that okay I'm not sure though uh multi layer looks like this where layers are like     
23:29     
maybe not really sure anyways yeah so let me go to figure four um that is uh     
23:38     
dual characterization of world real world Multiplex networks so this is interesting because it gives us oh it     
23:44     
gives us seans in here but it also gives us some networks in uh uh social     
23:51     
media uh archive genetic I don't know what exactly is involved in that human     
23:56     
human microbiome uh social networks and Airline networks and it     
24:03     
looks like they've got so these different symbols at the at the left could you increase the size there we go     
24:11     
uh so C alans are the x's and so uh here's seans down here uh     
24:19     
yeah that X and so that is compared to say like an airline Network which is uh     
24:25     
down below so it's like the airline workor has lesser values on the Y AIS     
24:32     
and and more scattered values on uh x-axis uh you have social media which is     
24:40     
up at the top in the top right it looks like C SE or no it's not seans it's     
24:47     
genetic data which is also up in that green     
24:53     
regime yeah genetic     
24:58     
yeah that and then archive which is the upward facing triangle right below     
25:06     
genetic yeah the one that's like so that's archive and that's that's that's the     
25:12     
compare you know people often compare different types of networks using their methodology so you can actually look at     
25:18     
these different types of networks I don't really know what the lesson is here um so they kind of get like maybe     
25:25     
in the legend they talk about it so so uh you know they're     
25:31     
comparing okay uh so they say that uh the way representation yeah the way     
25:39     
representation enables a better classification of networks that would otherwise be indistinguishable so you     
25:46     
have archive versus a transportation Network so Archive of course is preprints uh Transportation Network you     
25:53     
think those are very different but they actually uh you know maybe if you're     
25:59     
using a conventional method you can't classify them as as as different things     
26:04     
but when you have this layered representation you can do that more easily uh networks optimally group into     
26:10     
two clusters almost perfectly matching the spatial and non-spatial nature of the     
26:15     
systems uh for German transport and archive uh in the nodewise markers     
26:21     
correspond to nodes and gray lines to layers that's up in this right hand side differently from archive     
26:29     
the markers in the German Network tend to accumulate on few main lines meaning that both nodes and layers tend to     
26:36     
contribute preferentially to few components so this is where you know you     
26:41     
have this sort of maybe greater modularity or greater uh well I don't     
26:46     
want to say small world inness because we don't know that but it's definitely greater modularity because you get this     
26:53     
uh preferential attachment to few components um and so that's that's     
26:58     
something also the values in the layer-wise tend to span larger intervals in comparison with nodewise indicating     
27:06     
the presence of more heterogeneous multi-degree centrality distributions so this is where you get this centr     
27:13     
relevance the centrality measure um which means that you look at     
27:19     
a node and you say how Central is it to the network and you can find like you know places that are Central if you     
27:25     
knock out that node you have a major failure of the network if you don't knock out if you knock out the node in     
27:32     
the low centrality Network then it doesn't really matter you can find another route to get through it but High     
27:38     
centrality networks you have if you knock it out I mean I imagine a transportation network if you knock out     
27:43     
the Transportation Hub you know the the network suffers quite a bit um but if     
27:49     
you don't you know if you have a low centrality Network you can any one node gets knocked out and it's you know you     
27:56     
can reroute things as needed that's what they're getting at there in this figure so I think that's enough for     
28:02     
the figures actually uh I just wanted to go through that a little bit to kind of think about how this relates to     
28:09     
different systems and I don't know what they mean by seans like what what seans     
28:15     
represents I guess maybe the connectome which is what it usually means so in the case of the connecto you     
28:21     
get this sort of you know you have this uh centrality that's you know uh well     
28:28     
characterized we know there are these small world networks um that exist in the yeah it     
28:35     
says connect over I saw it yeah I see all those connect okay so     
28:43     
yeah that's good     
28:48     
um and like what you mentioned if you knock out a certain node in like uh like     
28:54     
if you have like low centrality let in a transport Network that can some be correlated to maybe in C can connect     
29:01     
them also like if uh like in hypernode we have like many nodes of like same types if you knock out like a few should     
29:07     
it like matter or not that can be observed centrality maybe just like given a thought at least not like     
29:14     
guaranteed well I think in the case of like uh like a high centrality network     
29:20     
if you were to build a hyper network uh depends on how you classify the nodes first of all so if you say     
29:26     
that like I want to classify celegans connectone by its cell types um you know or like okay never     
29:34     
mind the in the connectone you classify it by its uh sort of you could classify     
29:40     
by connectivity so you'd have a very Central uh single you know hyper node     
29:47     
that basically is the the main node and then it connects to all the other hyper nodes you classify Maybe by like     
29:54     
location you know you might have like some highly connected nodes in one hyper     
30:00     
node versus another and then that hyper node would also have uh connections with     
30:06     
very you know cells with very few connections so depending on how you slice the data depending on how you     
30:13     
classify the nodes into the hyper nodes it's going to have different amounts of     
30:18     
central different degrees of centrality different types of um sensitivities of their connections of     
30:25     
the hyper edges so you know if you cut a hyper Edge or if you cut within a hyper     
30:31     
Edge and again I covered a paper last week in the in the second part of the     
30:36     
meeting that kind of goes over how people think about cutting these hyperedges because a hyper Edge is a     
30:42     
bundle of connections between different nodes and different hyper nodes so if     
30:48     
you cut it you know you can cut some of them and still there's a differential sensitivity there so if you cut the     
30:55     
whole hyper edge of course you you uh unwire the network quite     
31:01     
significantly but if you cut certain edges in the hyper Edge um then you know     
31:07     
it depends on how sensitive how what the centrality of the network as a whole is     
31:13     
so that I know it's kind of hard to wrap your head around with that but I I did I     
31:18     
did Cover a paper where they' have a methodology for that so um yeah I think yeah so I mean what     
31:27     
what is the plan I guess I want to ask a question about gso uh so what is the plan to um you     
31:34     
know kind of from the last couple weeks as we've already got I think the regular deadline is now today so what what do     
31:41     
you want to do in the last couple weeks uh so like uh first uh is like     
31:49     
since right now we have plans of two uh ways of making the hypergraph uh so first is like finding     
31:56     
the centrality measures for those uh like uh I mentioned and also like I     
32:01     
presented uh which you mentioned last week uh cutting of the extra edges and after we make that like finding     
32:07     
centrality measures on like complete graph also but also on like this um this     
32:14     
this more cleared graph and uh uh after that like uh yeah for now     
32:23     
like this is what I have planned after that I have to discuss with you what exactly we can do like probably I would     
32:30     
like to discuss this discuss that in next week okay yeah yeah so what yeah let's     
32:39     
uh you know it's only a couple more weeks I think what I've recommended to other people is to find a good stopping     
32:45     
point and say this is where we can stop and and you know uh because you know the     
32:50     
the project doesn't need to be over but you need to get to a point where you can write up a read me file and say this is     
32:57     
going to to be like what I'm going to submit to uh Google and they'll say you know you'll pass of course but like you     
33:06     
know you need to get like a just put it in a repo and then you know have a link     
33:11     
to submit to them so that's you need but you know we I know we want to work on it after uh the deadline so that's that     
33:19     
that will be of course we'll do probably a manuscript at least one I know we have     
33:24     
talked about a lot of ideas and that's where we can do that we can lay out some of that more um but yeah I want to make     
33:32     
sure that we get we kind of know where we wna you know kind of for the gso part     
33:38     
conclude things right up like the documentation and then have a link for     
33:43     
submission and then you know we want to also make some issues in GitHub and I     
33:48     
know in the dvo learn uh actually in the dvo graph repository uh we have uh a board and we     
33:57     
want to make issues on that board and just things that we want to follow up on so I know there's a lot of stuff for     
34:04     
like maybe getting a manuscript together what are these things that you want to explore for the     
34:10     
manuscript um so you know we'll have the code base we'll have some of the other measures that you showed last week what     
34:17     
do we how do we want to put that translate that into a manuscript and then of course if people     
34:23     
do pick up this project next year uh what would you say would be the most most uh Salient followup issues and just     
34:32     
think about that like what are the and that doesn't need to be done by the gso deadline but what are the things that     
34:37     
need to be maybe worked on more or improved I mean you know there's always     
34:42     
that case of like you know this maybe needs to be maintained or improved I didn't get to this     
34:49     
thing maybe this is something that people could just develop their own project on just just one issue and so     
34:56     
just think about those things and I'm sure from a technical standpoint there are things that we could put on the     
35:04     
board right right yeah that provides good good insights like what should be done yeah yeah     
35:10     
yeah okay well thanks thanks yeah it was a good presentation I enjoyed that and I     
35:16     
think that's like something we can apply to not only to uh some of the things we're doing with um you know uh     
35:24     
hypergraphs and network Theory but also some of the things that Susan was talking about with uh you know act and     
35:32     
amas and networks and things like that because you have that same structure uh where you have things that     
35:39     
are interconnected you have you know uh it's     
35:44     
not quite the same as what we're talking well what we're talking about is quite theoretical so it would have to be you'd     
35:50     
have to apply it in a way that's like consistent with the system you're looking at so but anyways I think we     
35:56     
could probably G some insights from thinking about you know in that in that case it     
36:02     
would be more there would be more physical inputs couldn't just take it off the sh couldn't take an algorithm off the shelf     
36:08     
or something like this and apply it to to a network like that so last week we     
36:13     
talked a little about about interactions uh in interact interactive things in in cells and so dick sent me     
36:22     
this paper I don't remember when it was maybe it was recently maybe not but     
36:28     
uh this is a paper from pnas and this is a very heavily     
36:33     
biophysics paper so this is not like you know it's not Network Theory it's     
36:40     
grounded in interacting cells during Mouse embryotic so this is neighbor specific     
36:47     
gene expression so this is a little bit like what I was talking about with respect to uh you know looking at that     
36:54     
transcriptional distance between cells so you know you have like uh     
36:59     
neighboring cells you have uh neighbor like I we talked about last week you     
37:05     
have cells that communicate with one another sometimes there're these paracrine signals which are just kind of     
37:12     
chemical signals or you know uh different uh types of macromolecules     
37:18     
that bind to the neighboring cell maybe they're secreted from one cell and and     
37:24     
bound to another and they have some effect on their neighbors or nearby cells and then that of course affects     
37:30     
gene expression which is the idea in this paper so this is neighbor specific gene expression revealed from physically     
37:37     
interacting cells during Mouse embryonic development and I think we can go     
37:42     
through the significance and the abstract and that'll be probably enough to get a sense of what they're     
37:47     
doing so um the significance of this paper is in     
37:53     
this box at the right uh physical contact between neighboring cells is known to induce     
38:00     
transcriptional changes in the interacting Partners uh accurate measures of these     
38:05     
cell cell-based contact based influences on the trans crypto is a very difficult     
38:11     
experimental task like we said you know there's this aspect of looking at uh you     
38:17     
know in seans we know the sufate it's pretty well you know it's pretty easy to     
38:22     
do these kind of single self studies but even there it's hard to kind of get the data for everything soell um in Mouse     
38:29     
it's it's probably harder because the Fate can is is much more flexible in     
38:35     
terms of you know what to expect in terms of what's being expressed but also the Fate itself so classifying the cell     
38:42     
and um you know we'll see how they approach that um accurate measurements     
38:48     
of these cell cell contact based influences is very difficult however determining such transcriptional changes     
38:55     
will highly enhance your understanding for the developmental process uh so they     
39:00     
use uh scna a seek for this they isolate the tissue into individual cells so what     
39:05     
they'll do is they'll take a sample of like a tissue they'll uh you know sort     
39:12     
the cells or they'll get this they'll isolate the different cells out of the tissue uh they just basically take it     
39:19     
out of its extra cellular matrix and isolate the cell um and so the problem of course     
39:26     
there is that you can get single cell measurements using NextGen sequencing but then of course you don't have you     
39:32     
remove the context of the tissue so you get these neighboring cells you're measuring their gen expression but is     
39:39     
that due to them actually being in proximity and communicating or is it just you know kind of happen stance and     
39:46     
you know of course there's a context in the tissue between Not Just Between the cells but there's extracellular Matrix     
39:54     
there you know there's function that's Miss say when you isolate the the single     
40:00     
SI all right so here we combine pixek which is a technique that they talk     
40:06     
about in and computational algorithms so they're using this technique pixek and     
40:11     
algorithmic uh enhancement of you know to analyze a data to identify cell type     
40:17     
contact dependent transcriptional profiles focusing on endoderm development so they're looking at     
40:23     
endoderm development they're looking at these contact dependent Prof profiles they want to identify this sort of cell     
40:31     
type contact dependent transcriptional profile we have computationally     
40:36     
identified and experimentally validated specific gene expression patterns depending on the presence of specific     
40:42     
neighboring cell types our study suggests a new approach to disentangle     
40:47     
the role of cell cell interactions during embryogenesis so that's the sort     
40:52     
of the pro promise here uh they talk a little bit about cell cell contact in the     
40:58     
introduction they talk about this as being important for suell fate specification during development so when     
41:04     
cells come into contact in in mouse in mamalian embryos you know there are a     
41:09     
lot of signals that shape the fate of the cells so if you take a a stem cell     
41:15     
and you put it into heart tissue you can get a heart likee cell you can get a     
41:21     
heart muscle leg cell you can get something cardiac muscle or something like that so depending on where the cell     
41:27     
is is its proximity to neighbors of a certain type it can change its fate or     
41:32     
you know or it can modify its gene expression certain so the cell cell     
41:38     
contacts are extremely important for sulfate specification in mammals you see     
41:43     
in seans we don't really have the same sort of system but what's important there is that you have uh cells in     
41:50     
proximity you know then there are a lot of paracrine effects and and there a whole set of new papers on this     
41:58     
um uh with respect to things called the interactome so people are looking at the     
42:03     
interactome of different cells and saying you know what is the uh relationship between you know cells and     
42:11     
what are they doing in terms of signaling and how is it affecting each other um so that that's what they're     
42:17     
talking about here uh removing specific tissue replacing ectopic explants     
42:24     
meaning that they're just basically taking cells out of the tissue putting them into a     
42:29     
dish uh in various embryonic regions can dynamically change the fate of cells adjacent to the manipulated area for     
42:37     
instance manipulation with the crucial signaling Center anterior visceral endoderm so it's a part of the endoderm     
42:43     
causes defects in the specification of forbrain identity later in development     
42:48     
so if you uh so they're not talking about taking things out and putting them in culture what they're talking about     
42:54     
here is they're taking talking about if you move the cells around if the cells don't make the contacts that they should     
43:02     
in develop there's this crucial signaling Center in the endoderm that where the cells you know need to be kind     
43:09     
of in in a certain have certain connections to each other and if you have some a defect in the shape of that     
43:17     
area or the number of cells in that area or they're not making contact they're not communicating with one another it     
43:23     
can cause defects in the specification of horor so it has these long-term     
43:28     
effects in develop it is well recognized that ectopic grafts of signaling centers such     
43:35     
as the node n or spaman organizer so this is uh these are two different types of organizers in develop a node     
43:41     
organizer and spaman organizer and can induce a secondary interal axis so if you take like um you     
43:49     
know these signaling centers and you put them in uh you know you can graph them uh you     
43:57     
get the secondary neural axis so you can actually uh you can supplant sort of the     
44:04     
uh initial signaling Center and you can move it around and you can get these different effects in     
44:10     
development even with the importance of various cell type interactions for development it is still challenging to     
44:16     
study gene expression changes in association with neighboring tissue so it sounds like you know when you do     
44:23     
sometimes you can do experiments and development and you can get these interesting effects like you know getting a a second set of     
44:29     
limbs or you can get like this sort of secondary neural Axis or you can get other kinds of changes and you think     
44:36     
well you know that means that the gene expression must be pretty neat and tidy but that's actually not the case and in     
44:42     
fact gene expression is notorious for being very hard to sort of correlate with uh you know phenotypic effects so     
44:52     
while the grafting experiments have significantly enhanced her understanding of the inductive ability of various     
44:57     
neighboring cell types and that's citation 11 these experiments are technically demanding are not always     
45:04     
feasible so that's why they're doing this uh single cell RNA SE uh they're     
45:10     
looking at that but then they're using this other approach um that they're getting at here so um I talk about RNA     
45:19     
sequencing a multiple interacting cells have been used to measure the transcript of physically interacting cells or picks     
45:27     
without relying on lean receptor coexpression so this is where they kind of talk about how people are applying     
45:34     
this this uh methodology of physically interacting cells with pigs and like I said the RNA seek SE seek is a technique     
45:42     
but it's not enough to sort of give us the full picture so they've developed     
45:48     
this methodology called pixek which is using something very similar to SC seek but it's relying on     
45:56     
these physical interacting cells they're not relying on this leing receptor coexpression they're relying on the     
46:02     
coexpression aspect they're looking at this physically interacting cell     
46:07     
methodology so if you go up to the abstract here dick has made a few notes     
46:13     
um so uh the first note is in the first sentence here first and second sentence     
46:20     
uh development of multicellular organisms is orchestrated by persistent cell cell communication     
46:27     
between neighbor and partners so it's not just that they sort of are next to one another they have this persistent     
46:34     
communication between them and that's kind of the idea behind these grafting     
46:39     
experiments that you put cells in in contact with one another and you can get     
46:45     
different effects if you take cells apart you get these deficits so you get     
46:50     
you know either you get two of something if you put two organizers together or if     
46:55     
you can lose something uh if you take cells away from so that cell cell     
47:01     
communication is necessary uh in in the in the EMB direct interaction between     
47:07     
different cell types can induce molecular signals that dictate lineage specification and self fate decisions so     
47:14     
this is the reason why they're using pixie they're looking for these physical interactions they're not really worried     
47:20     
so much about like just having the cells in proximity and just measuring gene expression and say well are they     
47:25     
coexpressed uh they want to have these direct interactions and measure     
47:31     
those um yeah so uh currently single cell RNA seek or scna seek cannot     
47:39     
adequately analyze cell cell contact dependent gene expression so they're at a specific type of gene expression not     
47:46     
this coexpression they're looking at this cell cell contact dependent gene     
47:51     
expression so this is the the key and you can't do this necessarily uh where you take the cell out and you measure     
47:57     
them independently because you lose that signal essentially so to overcome these     
48:03     
problems uh and resolve cell cell context specific gene expression during embryogenesis We performed RNA RNA     
48:11     
sequencing of physically interacting cells and assess them alongside similar single cell transcriptomes derived from     
48:18     
developing Mouse embryos between embryonic day 7.5 and embryonic day 9.5     
48:25     
so this is when they're a LED changes in the mouse embryo analysis of the pixie data     
48:31     
identify gene expression signatures that were dependent on the presence of specific neighboring cell types so this     
48:38     
is where we have this Pixi data identifying gene expression signatures that exist when you have specific     
48:45     
neighboring cell types but don't exist when you don't have those neighboring cell types and they have to make     
48:51     
connections physical connections with the cell so this is where you can identify the things that are added and     
48:58     
subtracted when those interactions are present or absent our computational predictions     
49:05     
validated experimentally demonstrated that the neuroprogenitor cells which are     
49:10     
NP cells upregulate lhx5 and NX k21 those are two     
49:18     
genes that are upregulated uh when exclusively interacting with definitive endoderm     
49:24     
cells so you have neural pregenerator cells and endoderm cells when they make connections uh the neuroprogenitor cells     
49:31     
help regulate ox5 and nkx2 so that's a specific finding that     
49:38     
they've noticed in when you remove the definitive endoderm from the neurop proog from the proximity of the     
49:43     
neuroprogenitor they don't over or upregulate these two genes and these two genes are involved     
49:50     
in uh morphogenesis moreover there was a reciprocal impact on the transcriptor of     
49:56     
definitive endoderm cells is they tend to up regulate racks and GSC when in     
50:03     
contact with NP cells so again you can look at this two ways you can look at when the neuro progenitor and definitive     
50:09     
endoderm cells are in physical contact the neural progenitors reg upregulate     
50:14     
certain genes the uh definitive endoderm cells upregulate specific genes and they     
50:21     
have to be in contact for those to happen so if you think about like if those two cell types are in contact     
50:27     
you get these different types of uh genes expressed in these different cell types which reinforce their Fates and     
50:34     
and drive uh you know morphogenesis especially when you have a whole bunch of them in a     
50:40     
tissue uh moreover there was a reciprocal impact on the transcri okay we talked about that using individual     
50:46     
cell transcriptome data we formulated a means of comp computationally predicting     
50:51     
the impact of one cell type on the transcriptome of its neighboring cell types so this is where we have one cell     
50:57     
type that's predicting we have those cell types in physical proximity we can uh you know do     
51:05     
a computational study where we just basically say okay if these two cell types are in uh contact then this is     
51:12     
what we should see uppr and so that uh those computational predictions were able to Bear themselves     
51:19     
out uh they used individual cell transcriptome data for doing these     
51:24     
predictions um and so they were able to do that we have further developed a distinctive spatial T distributed     
51:30     
stochastic neighboring embedding which is I guess you know this     
51:35     
sort of uh dimensional embedding to display this pseudos spatial distribution of cells in a two     
51:42     
dimensional space so that means that they have this two-dimensional sheet that they've been able to you know put     
51:48     
different cell types in they have the distribution of cell types and their connections their their contacts I     
51:55     
should say and then they are able to sort of build this embedding that     
52:01     
basically predicts the different genes that are upregulated in this two-dimensional spatial context so you     
52:08     
should be able to look at a single cell and know that some Gene is being upregulated in a certain location in     
52:14     
cell sheet and so you know depending on the cells that are nearby if there's a     
52:19     
boundary say between neural progenitors and endoderm cells there should be a strip where certain genes are     
52:25     
upregulated and not in other parts of that two-dimensional sheet so this is a very uh interesting     
52:34     
approach we're looking at contact specific Gene regulation during     
52:39     
embryogenesis and so this is kind of showing the tissue here this is actually the mouse embryo at day     
52:46     
7.5 day 9.5 so you can see there's a significant amount of     
52:51     
morphogenesis uh you have the obligatory U map which shows some some of these uh     
52:57     
cells so the U map you know is basically where you have different cell types and     
53:03     
they're uh you know you have gene expression data so you have these single cell sorted by gene expression data and     
53:10     
you can see that and this doesn't have a spatial relevance these are just the umap dimensions but you have neural uh     
53:18     
so you have neural progenitors uh here in red and you have definitive endoderm     
53:23     
in this uh green so you can see that they're not necessarily overlapping     
53:30     
there's some sort of continuity between them there's some so basically the idea     
53:35     
is that they express different types of genes you know they they're because they're next to one another they may     
53:42     
coexpress genes but generally coexpression would not allow you to look at this interaction because the you know     
53:49     
NP cells and D cells are separated than the umap space especially along     
53:54     
Dimension one that's much in dimension too but the point being is that you can't necessarily rely on coexpression     
54:01     
between different cells to get at this problem and so this is a heat map here     
54:07     
this is the single cells these are the physically interacting cells so you can see that there's this C value of     
54:14     
expression so you can see that you have NP plus NP uh and then you have let's see if we     
54:21     
can find NP plus de I don't know if they have that here I is okay NP plus d is     
54:28     
here so you see this uh I don't know how many columns it is over six so it's the     
54:33     
First Column versus the sixth column and you can see if you look at the First Column versus the sixth column you get     
54:39     
these differences so you get like high levels of expression up here uh you get some similarities     
54:47     
actually as you might expect because you're looking at the interactions between you know NP versus NP NP versus     
54:55     
d uh but there are differences so you're going to have this is using this physically interacting cell methodology     
55:03     
then if you look at this single cell methodology you have you know NP cells     
55:08     
and de cells and there's a difference in terms of what they're expressing they're coexpressing things but they're also     
55:15     
differences so we want to get at these differences more explicitly with this pix     
55:23     
methodology uh I think this is actually the D is the sheet of cells right oh     
55:28     
this is goek okay so I want to get to the sheet of cells part     
55:36     
here if I can find it I don't think they have a figure     
55:42     
but so this is a visualizing spatio structure of tissue using spatial tne     
55:49     
for scna SE data you have this uh again this is kind of like the umap that we     
55:54     
saw before this is where we have NP cells D cells and then we have this     
56:00     
boundary and this is I was talking about the sheet of cells example I couldn't find a figure for that but basically     
56:07     
this describes what I was talking about so you have these different cell types where you have you know definitive     
56:14     
endoderm over here R progenitors over here and then there's this boundary this     
56:19     
boundary because it's spatial tne it's like finding the variation in spatial u     
56:25     
in space so it's spatially relevant here so this is the boundary here between uh     
56:32     
neural progenitors and definitive endoderm you can see that it forms this kind of stripe I don't want to say     
56:38     
stripe because it's kind of confusing this is not a real embryo it's this tme space and then uh neuroprogenitors and     
56:46     
these NC's uh which are not sure what they but yeah uh so     
56:54     
there's another stripe here and again that's the the boundary between NC's and NPS so that's that's what you're getting     
57:01     
you're getting these kind of difference you're getting kind of these intersections they're in different parts of expression space but they do kind of     
57:09     
form these boundaries and that's where you would have these physically interacting cells along these uh     
57:23     
boundaries okay so I think that's um I think that's all I wanted to talk     
57:28     
about with that paper I don't know if people had any questions or comments it's like they had to leave actually but     
57:34     
I don't know if people had questions or comments on that like M maybe or     
57:41     
organ call people out but see what people had to say um sounds     
57:48     
interesting I missed half of it so and this is recorded right I can     
57:55     
watch it on YouTube yeah okay yeah I can go back over     
58:03     
okay now glad actually me too I in the beginning missed a few parts uh because     
58:08     
of Po connectivity I'm going to watch recording and then we'll come back to this no problem yeah um yeah but the T     
58:16     
PLS uh like when you mentioned the strains yeah that makes sense like we cannot compare it to T the strains     
58:23     
cannot be compared to T PLS definitely because yeah the metrics are changed in     
58:29     
a way when we are plotting like T yeah yeah I was hoping for a figure     
58:35     
where they had like the two-dimensional sheet and they had like the cells and there physical interactions where we could see the like boundary it would be     
58:43     
much like you know you would conventional morphogenesis where you have like gradients then you have the intersection     
58:49     
of gradience they form like a a stripe or something you know seeing that would be like kind of powerful but I think you     
58:56     
saw it from the spatial tne uh probably pretty decently uh what's going on there     
59:03     
so yeah the physical interactions part is interesting because you're not talk you know you're talking about like gene     
59:08     
expression you're not talking about co-expression you're talking about like what's interesting are the physical interactions versus absence of physical     
59:15     
interactions so if I were building a network I would want to have like you know nodes and edges but the edges would     
59:23     
be you know different quality so you'd have like an edge that's physically in in physical proximity maybe like where     
59:31     
you know we have like Gap Junction networks of the connectto we also have synaptic uh connection connectomes that     
59:38     
are different because they're the cells are making physical connections or they're kind of these longer range     
59:45     
connections and so or maybe they're not connected at all and so we have to take     
59:51     
that into account when we build our Network we have to say the edges physically connected or not physically Al connected and then we can say it's a     
59:58     
spatial Network that has a certain distance and we can't just say from distance whether they're connected     
1:00:04     
because cells aren't just these little spheres they kind of move you know they kind of have processes and um so this is     
1:00:11     
something that uh even in progenitor cells you'll get this where you have     
1:00:16     
where they're not just met spheres they have this complex shape and it you know     
1:00:22     
in space that makes a difference in terms of physical maintaining physical connection so that that's something for     
1:00:29     
Network Theory or network building that we have to consider and of course that relates to the physical networks like     
1:00:36     
active myosin networks as well because you have these physical properties of a     
1:00:41     
bunch of nodes and edges it's not just this abstract set of drawings or or     
1:00:49     
graphs ahead that's interesting thanks for the explanation yeah so I actually I missed that part     
1:00:56     
out in which there was um Maps also uh so when um Maps I think they are like both similar in a way like both does the     
1:01:03     
same thing so is there a comparison between um map and tne do I actually miss that so really sorry for that yeah     
1:01:09     
so I guess the like umap and tne are just competing multi-dimensional techniques um you know um maps are very     
1:01:16     
common in like a lot of these nextg sequencing papers they always have to     
1:01:22     
have a umap uh graph like that where they show the umap dimensions but like the spatial tne is different     
1:01:29     
because it has a I guess a spatial component to it although tne natively     
1:01:34     
does not a umap definitely does not so when you look at a umap you see this this kind of pretty picture of you know     
1:01:41     
these Blobs of different categories and there maybe look like they're connected     
1:01:48     
right but they're not necessarily connected there's no like it's not like clustering where there's maybe some     
1:01:54     
inherent spatiality it's more like just they're in this embedded in this space and they're kind     
1:02:02     
of have a similarity but we don't know what that similarity means so that's why     
1:02:07     
we don't say that like there's a spatial relationship in a um graph it's just not     
1:02:13     
the case it may be the case but it's not inherent we can take like something     
1:02:18     
that's spatially like maybe spatial tne there are also different spatial uh PCA     
1:02:24     
techniques we can use that it'll take into account basically the the spatial     
1:02:29     
dependence of two data points so you know usually our tne or umap will be you     
1:02:36     
know rely on spatially Independent data so it's just independently drawn data and you put it in get like these uh you     
1:02:44     
know these these patterns but in the spatial you know in a spatially explicit     
1:02:50     
model you basically assume that there's a spatial dependence of points so     
1:02:56     
obviously two cells that are next to one another are going to be related in some ways they could have the same fate they     
1:03:03     
could have this you know similar gene expression profiles because they're nearby they're communicating but they're     
1:03:10     
you know they may coexpress things they may be in the same tissue but then you know cells that are further apart we     
1:03:16     
assume that maybe they have less of an effect and so that's that's where the     
1:03:21     
spatial dependence comes into play so it violates an assumption of the test so     
1:03:27     
you have to make sure that your method is spatially explicit that you're dealing with that dependence versus     
1:03:32     
Independence and then you have the graph and the graph doesn't necessarily mean again that there's any that it's     
1:03:38     
uncovering any explicit spatial relationship it's just that you're taking the space into account so spatial     
1:03:45     
tne should show maybe some of the effects of spatial proximity but it     
1:03:51     
doesn't really explain it very well it's just kind of showing so don't forget the tri     
1:04:00     
Junctions yeah yeah that's actually interesting because like you have they     
1:04:05     
gave these examples of just two cells being in proximity but you do have a lot of in sheets of cells you have these Tri     
1:04:11     
Junctions and multi-junctions we talked about those and that's where you have a     
1:04:17     
could test three cells or more cells and they should have these same effects Me Maybe I don't know um but it makes it     
1:04:24     
interesting once you get past two because then you have like you know then you're coordinating different cells now     
1:04:31     
they're in spatial proximity they're upregulating things and it's really getting you know much greater than like     
1:04:38     
these simple pair wise interactions So yeah thank you Mah for     
1:04:46     
attending oops     
1:04:51     
um of I my model that I'm working on um     
1:04:57     
there's an effect from the attachments of the cells like the attachments to a     
1:05:04     
subsrate for sure but I think there's also an effect for the attachment to to     
1:05:11     
each other like the the tri Junction attachments     
1:05:19     
yeah anyways um I guess I I I'll just keep working on     
1:05:25     
my mod and eventually I'll get be able to present it here yeah that would be great     
1:05:31     
I'd love to see this yeah looks like it's coming along pretty     
1:05:37     
decently yeah well there's supposed to be a J curve I don't get a J curve but I     
1:05:42     
get wobbles at the beginning and if you had a whole bunch of wobbles that added     
1:05:48     
up you get a jcurve because it's     
1:05:54     
um it's in effect some um a bunch of um the network being     
1:06:03     
stretched to start with and then once it's stretched then it then you get a     
1:06:08     
constant straight Young's modulus but before that you're stretching all the     
1:06:15     
fibers at different at different points so you don't get as high a a Young's     
1:06:23     
modulus it's because it's none of it is stretched so     
1:06:29     
it's sort of shows     
1:06:34     
that wob the beginning and then it becomes than you so that's my result so     
1:06:42     
far yeah great yeah I look forward to seeing that     
1:06:47     
the finished product there something approximating finished yeah and it's just the mechanic     
1:06:58     
um it's just a mechanical structure nothing     
1:07:05     
else okay well thanks for attending I think that's all for today um I'll see you next week good     
1:07:15     
luck bye bye all right now I want to take a look     
1:07:20     
at three seelan studies talking about neighbor specific interactions     
1:07:26     
so the first paper is uh this is from 2004 so this is an older paper um this     
1:07:32     
is a map of the interactome network of the metazone Cs so this paper has been     
1:07:38     
around for like 20 years now uh a lot of authors on it uh this     
1:07:43     
gives us a baseline for what we're talking about CL interactome networks     
1:07:48     
and sort of the stateof the art of 2004 um but these are you know when we     
1:07:54     
talk about interaction the way people have studied the the interactions between cells in this case     
1:08:00     
what they're thinking about are protein protein interaction networks so this is a specific type of network where you     
1:08:07     
have you know you look at proteins uh and how they kind of allow     
1:08:13     
different cells to communicate uh so that's you know ppis are a common technique in uh network uh     
1:08:23     
bioinformatics and it allows people to get they characterize things in a way that they can't with     
1:08:29     
transcriptional so this paper uh the abstract reads to initiate studies on     
1:08:35     
how protein protein interaction or interactome network so not we don't     
1:08:40     
always have to worry about protein protein interactions when we talk about the interactome but this is traditionally how this um has been     
1:08:48     
characterized uh how these relate to multicellular functions we have mapped a large fraction of the cgans in     
1:08:55     
interactome Network so this is in seans and so of course like I said we know all     
1:09:01     
the developmental cells in celegans we actually in this case it's the adult and     
1:09:06     
so we can look at uh you know different cells looking at proteins in them and     
1:09:13     
how those proteins interact with each other so this is something that you know is going to the indirect measure of     
1:09:21     
interactions within the CL the adult elegant um but the idea here is that     
1:09:28     
proteins interact and they form things that will build tissues and uh have a role in     
1:09:37     
differentiation in morphogenesis so this is where um     
1:09:42     
they've sort of established this interactome network starting with a subset of me metazone specific proteins     
1:09:49     
more than 4,000 interactions were identified from High throughput east2 Hybrid screens so these are the hybrid     
1:09:56     
screens here uh independent co-affinity purification essays experimentally     
1:10:01     
validated the overall quality of the Y2 data sets this is something that's     
1:10:06     
available um you know I think it's available data set but this is something     
1:10:12     
this so basically they have to uh purify the proteins get them U you know to a     
1:10:18     
point where they can analyze them uh and then they actually have these uh this     
1:10:24     
data set protein protein interactions together with already described Y2     
1:10:30     
interactions in interal logs predicted in silico the current version of the     
1:10:35     
worm interactome wi5 so this is something to um if you go to gen Bank     
1:10:42     
you'll find that they have you know these data sets are used to validate future studies so you'll see wi5 often     
1:10:49     
this map referenced in some of the protein D proteomic data that they have     
1:10:54     
on so this current version of Li acto this     
1:11:00     
is in 2004 remember so this is the wi5 map contains around 5,500 interactions     
1:11:07     
so remember these are just proteins that are interacting with one another there's this um you know interaction that they     
1:11:14     
can Char quantify and characterize and that's how they so they have like the nodes or the proteins and the edges are     
1:11:21     
the interactions between proteins so often times protein will interact or in     
1:11:27     
uh you know building a tissue or being in a tissue or something like that so that's what we're interested in it     
1:11:34     
doesn't necessitate the interactions between cells or the interactions between anything else other than     
1:11:40     
proteins so sometimes people will take protein protein interaction networks and we'll try to compare them with gene     
1:11:47     
expression networks which are of course uh common commonly uper with genes and     
1:11:53     
of course in general genes and proteins in terms of their expression and their     
1:11:59     
presence in a biological sample very poorly correlated for a number of reasons uh transcription the the space     
1:12:07     
between transcription and the building of a protein of many steps and so there are a lot of things that are not you     
1:12:14     
know they're not going to give you a strong correlation uh topological and     
1:12:20     
biological features of this interactome network as well as its integration with uh phenome transcriptome data sets so     
1:12:26     
this is phenome is the phenotype transcriptome is the transcriptional data set so they can integrate these     
1:12:33     
data sets um in different ways lead to numerous bi biological     
1:12:39     
hypothesis so uh this talks a little bit about how this is done um so to     
1:12:45     
understand biological processes it is important to consider protein functions in the context of complex molecular     
1:12:52     
networks the study of such networks requires the availability of proteom protein protein interaction or intera     
1:13:00     
Maps the yeac ryes has been used to develop a eukariotic unicellular     
1:13:06     
interative Maps this is where they're talking about that yeast assay that they're talking about where you're just     
1:13:12     
basically building this interact o map and this is something that in 2004 was     
1:13:18     
well characterized this references 1 through six celegans is ideal model for studying     
1:13:24     
how protein networks relate to multicellularity so this is where you know in a in a beast you have a single     
1:13:30     
cell it's a it's a metazoan cell so it has a lot of the same proteins uh and     
1:13:36     
you know we're getting a sense of how they maybe interact in the unicellular context now in seans you can look at     
1:13:43     
protein networks in relating to multicellular organisms where you have cells in relation to one another and not     
1:13:51     
in relation to but you can see the variation across different cells and different functions as well because East     
1:13:58     
has a limited set of biological functions because it's the single cell organism sometimes it exists in clumps     
1:14:05     
or colonies but you know that's a different sort of uh context than cagans     
1:14:12     
where you have tissues in formal developmental uh     
1:14:17     
period these have sort of a quasi developmental period but that's not     
1:14:22     
something we need to wor about in any case um here we investigate its     
1:14:29     
interact own network with HT y2h so this is this uh methodology they're using     
1:14:36     
based on yeast uh as Y2 baits we selected a set of     
1:14:41     
3,241 predicted proteins that relate directly or indirectly to multicellular     
1:14:46     
functions this is something that U they're using ye to sort of predict these proteins that are involved in     
1:14:53     
multicellular function that's reference seven uh Gateway cloned open reading     
1:14:58     
frames were available in the seelan orome which is just the way that the uh     
1:15:04     
Gene is read in terms of the amino acid uh composition codin and     
1:15:11     
form amino acids and then those proteins so you're interested in the open reading frames because they tell you sort of     
1:15:18     
what the composition of amino acids would be and so that will give you an     
1:15:23     
idea and of course open reading frames they're just kind of like threers that you isolate out of a gene sequence and     
1:15:30     
you can predict what kinds of proteins would y yielded from those orfs of these 81 Auto autoactivated the     
1:15:40     
W2 G one H3 reporter Gene is G4 DNA binding domain fusions and 24 others     
1:15:48     
conferred toxicity to East cells this is just kind of going through how they're doing this in East and they're kind of     
1:15:54     
predicting kinds of things we should find and see up okay so having applied these     
1:16:01     
Criterion we subdivided the interactions into three competence classes those that could be found at least three times     
1:16:07     
independently and which the ad y Junction is in frame those in frame found fewer than three times and that     
1:16:13     
pass the retest these are basically just making sure that the open reading frames are you know predictive and so they're     
1:16:21     
just validating here the core data sets 2011 156 high confidence interactions     
1:16:27     
between 502 dates and 1,39 praise this is just the method the jargon theuse for     
1:16:35     
this methodology it's not that important so um estimate the coverage of the ht2     
1:16:42     
data sets we manually search the bait screened here for the known interactions of our PD this is the database     
1:16:49     
10 the search gave rise 108 interactions referred to as the literate data set the     
1:16:56     
core and noncore data sets recapitulated eight and two interactions in this Benchmark data set respectively or     
1:17:03     
overall rate of coverage for the first past data set is about 10% so this just kind of goes through how they're     
1:17:08     
constructing this data set they perform these experimental screens but then they also performed in cico searches for     
1:17:15     
potentially conserved interactions or in terog so these are these potentially conserved interactions that you would     
1:17:22     
find across species see you know you have things that are specific to seans     
1:17:27     
you have things that are across species so you can do the experiment and see where the interactions by in specific     
1:17:34     
species You can predict this Sean but then you can also do things that are more     
1:17:40     
conserved um so you're looking over orthologous pairs of genes and proteins     
1:17:45     
that are known to interact in more or more other species so uh this is     
1:17:50     
something that of course we want to know as well     
1:17:56     
uh in addition the Y2 interact maps that have been previously generated for individual biological processes     
1:18:03     
including vulvo development protein degradation DNA response and germine formation so these interactive maps at     
1:18:10     
that time had developed specific functions specific tissues and so these     
1:18:15     
were pulled um to form this new map this H2 y2h so the interactive maps they use     
1:18:24     
for these specific specific uh biological functions uh done using the same yeast methodology but then you also     
1:18:32     
have this new data set HT y2h which includes all this these     
1:18:37     
interologs uh the scaffold data set literature search they have things in     
1:18:43     
the literature that we published previously and those were all pulled into this uh worm interactive version 5     
1:18:50     
W5 so this contain 5534 interactions and connected 15% of     
1:18:56     
the cigan proteum this is where this W5 data set gives rise to a giant Network component     
1:19:03     
which is where you have a talked about the giant or compon network components     
1:19:08     
last week a giant component is basically this a large interconnected graph that     
1:19:15     
is you know basically consistent uh hard really     
1:19:20     
to Define it uh in this context but basically they're looking for the things     
1:19:25     
that are all connected because you know just because you have a protein doesn't mean it interacts with     
1:19:31     
anything so this is where they're trying to find the sort of biggest connected     
1:19:37     
component proteins and so this has a there's a giant Network component of 289     
1:19:43     
nodes connected by 5,460 edges uh so then you know we can look at     
1:19:49     
other biological networks it's in reference 15 the worm interactome exhibits small world and scale through     
1:19:55     
properties as we talked about with some of the other networks like connecto     
1:20:02     
in this is a very different type of network it's prot interactions rather     
1:20:07     
than connected cells but functionally connected cells but it does exhibit     
1:20:13     
these same network features small worldness scale-free     
1:20:18     
properties uh in in light of that you have this tenden to preferentially     
1:20:25     
interact with each other so the proteins in this network the nodes     
1:20:31     
preferentially interact with each other rather than with ag proteins so the or     
1:20:41     
actually so in light of the small worldness and scale properties we can     
1:20:46     
see you know differences in some of the different features of the proteins to see how these these connected components     
1:20:54     
are manag managed how you get skill freeness and small prots so this data set also allows us to analyze whether or     
1:21:01     
not evolutionarily recent proteins and to preferentially interact with each other rather than nature proteins and     
1:21:08     
you might think that that's the case but you might also think that Evol recent     
1:21:13     
proteins interact with the proteins because they share it might be like an     
1:21:18     
additional uh part of our function so you might have ancient proteins that maybe enable function at some base level     
1:21:26     
and then have evolutionarily recent proteins that augment that function or you could have evolutionarily recent     
1:21:32     
proteins that form their own module that don't really interact with ancient proteins so that's that's what they're     
1:21:39     
asking we subdivided the nodes of the network into three classes 748 proteins     
1:21:45     
with a orthol in yeast so the orthol in East would be an ancient protein three     
1:21:52     
1,314 proteins of the ortholog and ropa arabidopsis are humans but not in E this     
1:21:58     
is where you're looking at all multicellular uh metazol or actually     
1:22:04     
rapid opsis it's it's any multicellular organism so it's Ropo Rapid opsis or     
1:22:09     
humans but not in E so if it's single cell we don't you know we want to     
1:22:15     
compare the any multicell organism whether it's an uh metazone or a plant     
1:22:22     
and then 836 proteins with no detect what worth along outside of Clans so     
1:22:27     
this is where we just compare it with other with other clans or others so this     
1:22:32     
is where we have this breakdown 748 proteins that are of ancient     
1:22:38     
origin 1314 proteins with orthologs and other uh multicellular organisms only     
1:22:46     
which means they're probably evolutionarily more recent than U maybe like the Divergence of yeast and other     
1:22:54     
single cellular organisms and 836 proteins that no detectable orthol     
1:23:00     
outside of celegans so these are very Evolution early recently they just exist within seel so you can see that you have     
1:23:07     
a lot of them that are like specific to multicellularity some that are ancient     
1:23:12     
and of course about the same number they're very recent the lesson here is that these     
1:23:18     
three groups seem to connect equally well with each other so this is in figure 2C and so you know there's no     
1:23:25     
difference in terms of ancient versus Evolution early recent which suggests that they're functionally integrated I     
1:23:32     
guess what they find here is that these three groups seem to connect equally well with each other which suggests that     
1:23:39     
a new cellular function relies on a combination of evolutionarily new and ancient elements and this is consonant     
1:23:46     
with the classic proposal of evolutionist Tinker that modifies and adds to pre-existing structures to     
1:23:52     
create so you know we build that we can build this uh transcript or we can build this     
1:24:00     
uh this protein protein interaction Network we can use methodologies to get     
1:24:06     
like a unified Network we get our data from many different places we can use experimental probes we can use uh     
1:24:13     
previous literature for specific tissues uh we can look at different you     
1:24:19     
know other types of De that generated generate this master map and then look     
1:24:25     
at like answer biological questions like as we talked about and so this is you know this is a     
1:24:33     
lot of this is protein protein interaction work this is all kind of like looking at the sort of The     
1:24:40     
Coincidence of two proteins and it's interesting but there are other ways we can look at the     
1:24:47     
interactive so the second paper is also about protein protein interactions and     
1:24:52     
but this is from 2009 this is a little bit later and this work focuses on empirically controlled mapping so this     
1:24:59     
looks at protein protein interactions in a little bit different way so to provide accurate biological     
1:25:07     
hypotheses and elucidate Global properties of cellular networks systematic identification protein     
1:25:13     
protein interactions must meet high quality standards we present an expanded     
1:25:18     
sealance protein protein interaction Network or interact map Der from testing     
1:25:23     
a matrix of 10,000 by 10,000 proteins using a highly specific highr east2     
1:25:30     
hybrid system so the same thing that we using in the last paper uh that     
1:25:36     
Y2 model and so this is uh where you know you're looking at this kind of     
1:25:42     
comparison with yeast and getting these protein interactions through a new empirical quality control framework we     
1:25:49     
show that the resulting data set verm interactive 2007 or wi2 2007 this is a     
1:25:54     
different data set that from the other paper was similar in qual uh quality to     
1:26:00     
low through data cared from the literature so this is high quality or     
1:26:06     
higher quality data than previously existed this is specifically comparing it with the literature existing     
1:26:12     
literature and finding that loow group of data from the literature which means that people did experiments where they     
1:26:17     
just looked at one tissue or one function and they actually mapped the interactions in a different way uh that     
1:26:24     
doing this in in a high through Point manner is you know although the quality     
1:26:30     
is better um well I said Quality quality is similar using this highr you know the     
1:26:37     
reasons why maybe we should expect it be better or even worse but it's similar in     
1:26:42     
quality so it means that it's you know you think you you you do a focus study     
1:26:48     
and you do these high throughput studies and you get basically the same quality we filtered previous inter action data     
1:26:54     
sets and integrated them with wi 2007 they generated a high confidence Consolidated map so this is now     
1:27:01     
W8 so this is now this new map W8 so this is a new higher confidence     
1:27:08     
map this work allowed us to estimate the size of the worm interative at about     
1:27:13     
116,000 interactions comparison with other types of functional genomic data shows the     
1:27:19     
complementarity of distri distinct experimental approaches and predicting different functional relationships     
1:27:25     
between genes or proteins so they make a couple of points     
1:27:31     
in the introduction here the first is that the interact do of an organism in the network formed by a complete set of     
1:27:38     
binary physical interactions that can occur between all proteins so this means     
1:27:44     
that the this interactome or this is protein protein interactome map is a     
1:27:49     
network that is this sort of it records these binary physic phical     
1:27:55     
interactions so it's the physical interactions between the proteins and the cell so the paper we talked about     
1:28:00     
earlier in the meeting talked about the physical interactions between the cells and there's a specific way to look at     
1:28:05     
them in this case we're looking at the physical interactions between proteins and using up binary uh technique so     
1:28:12     
we're either saying there's physical interaction there's and so this is something to look     
1:28:18     
at here uh the low throughput interaction experiments are of considerable value in understanding     
1:28:24     
cellular processes of the molecular level so low throughput interaction experiments really kind of elucidate a     
1:28:31     
lot of these interactions in in detail but of course High through approaches are kind of you know     
1:28:38     
maybe have a lot of false positives so they basically found that that's not the     
1:28:44     
case in this using this particular methodology initial steps towards binary     
1:28:49     
interative mapping metaz zon undertak the resulting partial Interac maps that     
1:28:55     
provided insights into the organization of biological networks and assisted in the determining functions of many     
1:29:02     
proteins and complexes along with identifying hundreds of connections to proteins     
1:29:07     
associated with human diseases so we're interested in organization of biological networks this     
1:29:14     
sort of topology what's connected to what we want to know something about     
1:29:19     
like the different uh functions and how those you know how aren't just kind of like you know one     
1:29:26     
protein for one function but many proteins for a function and it involves complexes and     
1:29:33     
interactions and then we want to identify hundreds of connections to proteins associated with human diseases     
1:29:39     
so we want to have some season period so as they said highr interact     
1:29:45     
interact mapping is particularly needed for c el     
1:29:59     
okay so the first proteom and scale version of the warm interact over W5 which we just talked about and by     
1:30:06     
several sources of protein protein interaction this includes literature curated interactions these E2 hybrid     
1:30:14     
interactions module Maps each devoted to a specific biological process that's     
1:30:19     
where we did the E2 hybrid approach so you have these module Maps which have     
1:30:24     
the modules these sort of subgraphs that uh then you can assemble     
1:30:30     
and you can you know assemble the biological processes in a larger man in     
1:30:36     
talog interactions we predicting pairs of interactors these respective     
1:30:41     
orthologs interacting other organisms this is where you have these interactors where you have that's not just specific     
1:30:49     
to celegans but has this sort of Lev of conservation so you kind of know that     
1:30:54     
it's not spurious and you can kind of validate it with other organisms in their in their     
1:31:01     
interactives and lastly Y2 interactions derived from a high throughput screen     
1:31:06     
performed with around 2,000 metazone proteins as be so this kind of goes through last people talking     
1:31:13     
about uh this is but but this is a little bit later so now we know how to use these Maps we have one now we're     
1:31:20     
going to improve so we're updated worm interact map W8 implements several techniques and     
1:31:27     
strategies that are critical for generating high quality protein prot interaction data on a proteomic scale     
1:31:34     
first we expanded the worm interacton m by screening a matrix one 10,000 by 10,000     
1:31:40     
proteins second we develop new standards uh and then you know because     
1:31:46     
worm genome annotations are improved frequently we update previous protein protein interaction data according to     
1:31:53     
the and Gene models so we use the latest data for Gene models to sort of figure     
1:31:59     
out what the functions are uh finally we empirically estimated     
1:32:04     
the full size of the seon's interactome through the implementation of a new interactome mapping from based     
1:32:11     
exclusively on protein protein interaction so this is just kind of kind of coming up with better     
1:32:18     
methods um we can actually use tools like mum     
1:32:23     
protein C interaction trap technique or map it to measure how a random sample of     
1:32:29     
wi 2007 performed in an independent protein interaction detection asset     
1:32:34     
compared to a positive reference set so there are different ways you can look at uh the data to make sure that you're not     
1:32:41     
hitting false positive and so this is you know kind of what they're doing     
1:32:47     
here we have these high quality binary interactions and results so we have to sort of uh figure figure out what you     
1:32:54     
know we want we don't want to just have any set of ler interactions we want to be able to have a stringent Criterion so     
1:33:01     
that we can sort of get you know just the sort of the essential     
1:33:08     
interactions so this actually shows kind of out of the 20,000 genes and     
1:33:13     
celegans uh that you have wi2 w27 there's this overlap in terms of the     
1:33:21     
open reading frame so wi2 2004 at 15,000 open reading     
1:33:27     
frames uh wi2 2007 actually wi2 15,000     
1:33:34     
by 2000 wi2 2007 uses 10,000 by 10,000 open     
1:33:40     
reading frames and there's this amount of overlap here 10,000 by 2,000 open reading frames between the two data sets     
1:33:47     
so they're actually finding there's this like a core interact and then they have     
1:33:52     
this new set of relationships they validated uh that are different from     
1:33:58     
2004 so I guess you know the idea is that you're depending on the methodologies you're using you could     
1:34:04     
establish different types you know different data sets for connectiv or inter interactives and so these     
1:34:10     
interactomes then U you know this is this is sort of when we talk about these     
1:34:16     
networks lessen heriz that networks aren't always inherently like ideal you know they're     
1:34:22     
idealized for your Tech so you know there's no one I guess there could be a     
1:34:27     
consensus uh interactive but really you know when when you publish an set of interactions     
1:34:34     
it's really based on the assumptions you're making and that's the best you can't really say I mean there may be     
1:34:41     
a consensus uh of of these two data sets but this is just basically the most     
1:34:46     
conservative map you can make it doesn't describe all the connections but neither do any of the other techniques any Tech     
1:34:53     
technique you use is going to miss certain interactions and stress interactions maybe aren't that important     
1:35:00     
so another thing this paper brings up is this idea of biophysical truth and so that's like kind of what is you know     
1:35:06     
what is actually biophysically plausible and true so these confirmed O2     
1:35:11     
interactions may be biophysically true interactions that do not actually incur     
1:35:16     
in Vivo if the involved proteins are not present at the same time in place with a     
1:35:22     
multi within a multicell organ or are not present with the proper post transation     
1:35:27     
modifications so we can look at things that are found in the yeast probes and     
1:35:33     
we can make predictions of things that should be interacting but just simply aren't in our sample or if it's you know     
1:35:40     
there's some post transational modification that hasn't happened those are of course Very across uh context in     
1:35:47     
organism and so forth so we can have these kinds of things um where we can     
1:35:53     
actually say is something biophysically true we can make predictions on things that we     
1:35:58     
actually see in the in the biology but also things that we can predict should be     
1:36:05     
there and so uh here they talk about the sort of their methodology they use     
1:36:10     
geneontology to sort of sort of predict the function um and so they look at uh     
1:36:17     
different sub uh component subsets of W8 we compared the degree of functional     
1:36:23     
linkage with that of minor interactions DED from the literature so they use some literature mining to build these uh     
1:36:31     
annotations and they kind of get a sense of what the proteins are doing um all     
1:36:37     
data subsets show a high enrichment of both Broad and specific functional linkage suggesting High biological     
1:36:44     
relevance so this is where you know we have uh this degree of functional     
1:36:49     
linkage among the w207 constituent was similar to or exceeded the     
1:36:56     
literature enrichment at each functional specificity limit tested so basically     
1:37:02     
you know looking at how this data set was assembled looking at the different     
1:37:08     
sources variation and saying that this is actually pretty consistent with what     
1:37:14     
we might say is actual fun biological function so we see interactions that map     
1:37:20     
to specific types of function like micr tual Dynamics axon growth sex myoblast     
1:37:28     
migration Inon like protein building accent     
1:37:33     
outgrowth cell miction and proteins that are involved in micr Tu outg or     
1:37:41     
centes so this is something I focus on is integrated functional networks and     
1:37:46     
kind of looking at what these are uh so integration of diverse large scale data sets were previously used to demonstrate     
1:37:53     
the coordination when are connected yet distinct molecular machines involved in more early AOG Genesis so there's     
1:38:01     
another recent publication as of 2009 that describes basian integration of     
1:38:06     
functional linkages under a single Network waiting each type of evidence according to reference data sets so     
1:38:13     
taking a benchmark of making predictions based in Basi in integration so you can     
1:38:19     
look at these things as very well validated binary interactions things that you can get from the literature you     
1:38:24     
can get annotations you can connect them to function you can also use predictive     
1:38:29     
methods like a and integration and those should all kind of contribute to this     
1:38:35     
mod so the last two papers focused on protein protein interactions that's pretty far away from like the cell cell     
1:38:42     
interactions we talked about in the first paper so this is you know kind of going over the methodology of building     
1:38:48     
an interative map and having this sort of model of biological function     
1:38:54     
and so you know there are a lot of ways you can look at interactions within the organism I focused on protein protein     
1:39:00     
interaction networks that's called her phenotype say like gene expression networks but gene expression networks     
1:39:06     
also give you this sort of you know set of potential interactions so this is something     
1:39:12     
relates to function as well but it's a little bit further REM moov from function of course we have connectone     
1:39:19     
papers these are you know classic uh look at those as well look at the     
1:39:25     
methodologies we have our embryo networks this is actually a very specific type of network set of     
1:39:34     
interactions and this has this is a paper a new paper genetics I believe     
1:39:40     
this is from 2024 so this is actually recent Pap uh this is endroid     
1:39:46     
morphogenesis and C elegans this is Hyman and buo and you know they're interested in     
1:39:53     
is sort of dendr morphogenesis and looking at this across the eleg across the     
1:39:59     
connector so the Abra read since the days of ronal the vast diversity of     
1:40:04     
neuronal and particularly dender morphology so we're looking at again this morphology of cells we're looking     
1:40:12     
at the morphology of neurons of course this is important for physical interactions between cells as we know     
1:40:19     
that like you know instead of just looking at protein protein intera action     
1:40:24     
uh you know if neurons are inv physical contact that has implications for its     
1:40:30     
connectivity but also if they're you know if they have they processes those     
1:40:35     
processes are also important in knowing which cells are connected or which cell will be connected in you know we're     
1:40:43     
we're in embryonic development which will be connected in the adult so uh this vast diversity of     
1:40:52     
dender ology has been used to catalog neurons into different classes so this is something that Roman Cal did sort of     
1:40:59     
describing different types of neurons it's the way neurons the motor neurons and they are kind of defined as much by     
1:41:05     
their dendrites and their processes as they are by their sort of function so we     
1:41:11     
can look at proteins and genes we can look at like you know what the cell does     
1:41:16     
we can look at these morphologies and they kind of give you a sense of what they're doing but also kind of what uh     
1:41:25     
you know we classify itology just as much as like if it's in normal cell or if it's a epithelial cell     
1:41:33     
or so forth endite morphology varies greatly and reflects the different functions     
1:41:38     
performed by different types of neurons so our different types of neurons we classified by cender     
1:41:44     
morphology significant progress has been made in our understanding how dendrites form and the molecular factors and     
1:41:50     
forces that shape these often elaborately scoped conu structures so this case in this case we're looking at     
1:41:57     
yans and we're looking at the developmental mechanisms that mey Denver morphogenesis and there's a focus here     
1:42:04     
on studies investigating solated Sensory neurons and highly elaborated dritic     
1:42:09     
trees of somata sensory neur so we're looking at different sensory inputs in these dendritic trees we have solated     
1:42:16     
Sensory neurons and some medison     
1:42:24     
these studies which combine timelapse Imaging genetics and biochemistry reveal an intricate network of factors that     
1:42:30     
function both intrinsically endites and extrinsically from surrounding tissues     
1:42:35     
so there's this input from these uh you know from the biochemistry from the     
1:42:41     
genetics that also you know there's this uh this affects the function of     
1:42:47     
dendrites but also there's this effect of surrounding tissues as you might imagine the dend     
1:42:53     
and of inate the surrounding extracellular Matrix and the other supporting cells and you know supporting     
1:43:01     
structures play a role in how these dendroides are functioning and even where they projected so there's Stu     
1:43:08     
projecting the dendrites sort of their morphogenesis uh therefore dendrite     
1:43:14     
morphogenesis that result in multiple tissue interactions which ultimately determine the shape of dender Barbers so     
1:43:21     
this is basically what we're talking about with these tissue interactions so you know how would we describe these     
1:43:26     
tissue interactions in the protein protein interactions papers we said they     
1:43:32     
basically took these physical interactions and treated them as binary interactions and then said those binary     
1:43:37     
interactions form a an edge in a so if we have a physical interaction it's     
1:43:44     
either present or absent that's an edge in this case we have multiple tissue interactions so we can have con     
1:43:51     
between neurons given that they have a dendritic structure and there's an input     
1:43:56     
to the dendritic structure and there's coming you know it's coming in and we have that as an edge and that can be     
1:44:04     
weighted or that can be binary but we also have these multiple tissue interactions which determine the     
1:44:10     
shape of den vers so we can also consider these tissue interactions as     
1:44:16     
playing a role in shaping that or weighting that edge so when we have a weighted Edge it just doesn't come out     
1:44:22     
of nowhere it comes out of these tissue interactions and in fact we can do this we can uh have you know multiple factors     
1:44:30     
play a role in how these dritic garbers form what their shape is and ultimately     
1:44:37     
what their function is so I think in something like this it's a lot different than a protein protein interaction in     
1:44:42     
fact I think protein protein interactions are much richer than these binary uh physical interactions and that     
1:44:50     
certainly helps us filter out some of the the noise of spous interactions but     
1:44:56     
in this case we do have to consider this complexity and we have to you know characteriz it quite a bit to get a good     
1:45:03     
uh representation of the problem so they're focused on Sensory     
1:45:08     
neurons the dendroides play a very important role in how we perceive the world how information gets into the     
1:45:15     
neurons and how it's processed Sensory neurons exhibit     
1:45:20     
strikingly distinct Sol of architecture is intimately tied to the function so we have to take these not only the nites     
1:45:27     
into account but how they are formed and shaped and says Ken it goes     
1:45:35     
through talking about the ciliated sensory neurons in celegans as well as     
1:45:41     
samat Sensory neurons both of these are common in that they have a bipolar morphology the two processes     
1:45:47     
corresponding to a dendroid in an axon so the dendroid information coming in the aent     
1:45:53     
out of the neuron is there there you know information is being integrated     
1:45:59     
coming into the neuron and information is going out to other neurons uh so unlike most other seigan     
1:46:07     
neurons that are unipolar possessing a single process that runs along other processes and form synapsis and bant     
1:46:15     
other bipolar neurons and seans include touch receptors and motor and so this was reviewed to Chisel     
1:46:23     
2016 in this chapter the anatomy development and genetics of dender     
1:46:29     
morphogenesis so dorate morphogenesis can be modeled as its own network or we can use like a gene Gene interaction     
1:46:36     
Network or protein protein interaction Network to describe the morphogenesis of these dendroides and then link that to a     
1:46:43     
model of the dendroid network as BL or in its own so this is where we have this     
1:46:50     
uh they talk about this important theme that they own emphasizes that dendroid development is fundamentally a product     
1:46:55     
of cell cell inters and between the neuron and surrounding cells Lea skin muscle and     
1:47:03     
other neurons so this is interesting because in protein protein interaction networks we talked about building this     
1:47:09     
network building these edges that are these physical connections between cells or physical interactions but then of     
1:47:16     
course we talked about building a network of modules that are connected that describe functions so these modules     
1:47:23     
describe functions then the functions are connected and you know different protein modules are connected to     
1:47:29     
describe sort of the biology in this case what we have are these you know     
1:47:34     
protein networks describing different functions so you have a function of the neuron a function of the glea a function     
1:47:40     
of the skin those are connected but of course the glea is the function of the     
1:47:46     
glea is affecting the function of the dendroid formation and the function of the neuron the same thing with skin and     
1:47:52     
muscle and other neurons so you have for example on a sensory neuron you have     
1:47:57     
things happening in the skin you have sensory projections of the skin they     
1:48:02     
come down they get integrated into the dendor they come down into the uh into the sensory neuron so all these things     
1:48:09     
are interdependent underlying all these tissues are of course protein protein     
1:48:15     
interaction so it gives you you know it may it gives you pause to think about     
1:48:20     
like you what kind of a network structure would we use to integrate that would we use one of these Multiplex     
1:48:28     
networks that we talked about earlier in the meeting we use a hyper graph I think     
1:48:33     
we could use any model we want but we have to justify how we model you know what we include in the model and how we     
1:48:40     
can like fit data model and make it consistent so this is uh again you know     
1:48:48     
the cellular molecular steps that give rise to these structures do provide a model of how developmental cell cell     
1:48:55     
interactions are used to coordinate the morphogenesis of diverse cellular structures with     
1:49:01     
tissue so this is a you know kind of the takeaway message what's interesting     
1:49:06     
about this paper is they give us some data here they have uh solated dendroides they have a small sort of     
1:49:13     
table here where they talk about some of the so dendroides that we have so we have for each neuron here and these are     
1:49:21     
like the these neurons have pairs so there's a left and a right uh you have the Symmetry measure     
1:49:28     
the seated ending which is e ex or em or CU you have this description of like     
1:49:36     
what these uh cells are so these are ampid neurons 12 neurons and two Gia     
1:49:42     
this list we have a symmetry value of two solated endings uh em ex and EM then the gal     
1:49:50     
attachment which is amsh and then the cell specific promoter     
1:49:56     
which of course is a gene that's involved in this uh in the cell so this     
1:50:01     
is interesting it's an interesting data set I'm not sure uh you know how this would fit into say what the connecto but     
1:50:09     
be interesting if someone would explore that um and so this kind of talks about     
1:50:14     
the role of soled dendrites and sheath by glea so there's an interaction between the dendrites of neurons and     
1:50:21     
glea the wheel and Sheet dendroid are topologically part of the skin     
1:50:27     
epithelium so there's the skin epithelium inter actually neurons we talked about interactions and mice     
1:50:33     
between uh epithelial cells and neuronal cells so that's interesting an     
1:50:38     
interesting parallel there as well and this kind of shows the morphology so you have again the cuticle which is the     
1:50:46     
other uh surface of the worm skin which is underneath the cuticle there's this     
1:50:52     
projection that comes up above to the cuticle which brings information down to the neurons     
1:50:59     
and it's this is amphid type cell this is the O and C C type cell so this is a     
1:51:08     
different configuration then the I cells are these are in sheath bya and the I     
1:51:14     
cell case is not in sheath bya or actually the red are in sheath bya so part of the I is in sheath bya but the I     
1:51:21     
cells have a non-en uh non-ch by Leo     
1:51:29     
aspect okay so these are just talking about different types of cated dendrites     
1:51:35     
and their morphologies that kind of shows that those set of interactions so how would you model this as a I guess is     
1:51:41     
the question so it's a very complex system in of itself but then we think about sort of the protein for commun     
1:51:48     
interaction networks in by this it's it's quite you know it's quite the problem so that's all I'll talk about     
1:51:55     
for that paper so I hope that people were able to understand what was going on here in terms of you know Network     
1:52:03     
interactions and not just like last week we talked about connectivity and this week we're talking more about     
1:52:08     
interactions and so hopefully that was helpful to you and I hope you something
