     
Transcript     
0:00     
I could maybe cut out the iteration I'm going to try anyway because I need um Step function     
0:07     
force and then I need uh one that's that slopes a     
0:12     
ramp and uh I've seen somewhere uh someone tried um a triangular function and then     
0:21     
a sinusoidal function um I could try that but really I'm     
0:27     
interested in the slope yeah in having a mild slope a steeper slope and then a     
0:34     
step function got make control over the step     
0:39     
size control the step size yeah um I just want it to be as St     
0:48     
functional one way of is to go for more Computing time than smaller     
0:54     
STS oh well it already does that if you     
1:00     
make it between this value and that value if they're close together it'll do small     
1:07     
steps and if they're far apart then it'll do bigger steps to get there uh something like     
1:15     
that um I don't want it to do steps I just want it to do one step so I can get     
1:21     
the rate of change like I'm looking for rate of     
1:27     
applied force because that's very important in tissues so I need to do an     
1:33     
experiment that way okay can you visualize after a     
1:39     
single step even know it goes on does more     
1:44     
steps um I've got it so that it shows the process um like it yeah I've got it     
1:52     
so I can see it happening I don't know how to record that yet take a     
2:00     
it's it's really strange it it goes down and then the middle part that I've got     
2:06     
the tri Junction pancakes and then it flips all over the place and     
2:13     
then finally finally settles there you go you should take movies from some of     
2:20     
these art structures when it's     
2:25     
windy seg the art we should take pictures of it it's windy oh yeah well     
2:33     
this yeah I should build one then put it out in the wind     
2:39     
yeah and also they if I put a water um     
2:45     
into the model it it slows the oscillations or at least somebody's     
2:51     
already done an experiment like that and it slows the oscillations if you have     
2:57     
water mixed in okay so why don't we get started uh so     
3:03     
it looks like yeah no I can't find anything it's     
3:08     
okay okay any anybody who's on using     
3:13     
ACT uh I mean besides what I'm using right now you trying to do I'm trying I     
3:20     
get a message from jitsy that I don't have permission to use by camera oh yes     
3:26     
yes but yes and and it's not giving you a dialogue to to n     
3:34     
accept no yeah     
3:40     
um sorry my wife's laptop isn't here so um I yeah I'm on     
3:49     
theist sure sure sure I I would just need that just to remind myself of what     
3:56     
to yeah particular path to get to where you need to except those yeah     
4:05     
[Music]     
4:11     
yeah it would be in     
4:20     
settings there I'm on my phone     
4:28     
too hello     
4:34     
I had the same problem last     
4:39     
[Music]     
4:46     
week why don't we get started and okay go ahead all right so hello everyone uh     
4:53     
hello Pocky and Susan and Morgan dick Jess uh yeah we didn't have a meeting     
4:59     
last last week um I was actually at a confer or a workshop and I'll talk about     
5:04     
that in a little bit also talk about our uh Community period for Google summer of     
5:10     
code and um maybe some other things as well so I don't know if anyone had any     
5:16     
updates before we get started in terms of projects or anything else that they're     
5:23     
doing it was a um it was an interesting Journal Club um Saturday afternoon at     
5:31     
least my time um on the organoid intelligence paper that     
5:37     
was a multiple lab review paper really uh well was kind     
5:46     
of a position paper with with a lot of review     
5:53     
references um uh covering you know very     
5:58     
much the the direction of um uh well as they said the one the one     
6:06     
reference that they felt was the closis to what they were kind of proposing in     
6:11     
this organoid intelligence framework is um was the Kagan at all paper that um is     
6:19     
cloaky known as dish brain so that was um you know a um     
6:26     
yeah you know I I I it's interesting like I think we're     
6:32     
going back and forth in terms of you know whether that's a a 2d organoid um I I would I would     
6:43     
disagree with that or you know anyway but they um that was dish     
6:50     
braining yeah and um uh paper where you know they had a     
6:57     
a stem cell that um that were definitely grown um uh to play and and then then     
7:07     
you know together with the microelectric array     
7:12     
um given kind of bir directional uh communication for playing     
7:20     
pong and and um yeah     
7:26     
so the idea being that this is where biology is used as     
7:34     
a as a as an actual processor or you know control     
7:40     
system and uh it was nice there was two undergrad students from um from UCSC did     
7:50     
the presentation uh or you presented this review paper and     
7:56     
um and uh they are also um in David     
8:02     
hosler's class the mathematics of the mind that you know I've talked about and     
8:09     
I'm hoping to to get a they they said he he plans to release the the notebooks     
8:16     
and as part of like an interactive textbook um for this and so it was nice     
8:23     
to see that you know get a get a um a bit of a perspective from these     
8:29     
undergrad ad s that were you know actually you know I mean they they they     
8:34     
felt a little bad about officially calling themselves bra Engineers um because I think that was     
8:40     
the the more the grad school um students but uh but you know uh definitely     
8:46     
involved in the kind of braer group that is there     
8:52     
ecsc and this was hosted by cortical Labs so um like an actual you know     
8:59     
employee of cortical Labs um uh um so     
9:06     
this was on Discord so his his Discord handle is some Galactic Galactic     
9:15     
arachnid and I only know that his his uh first name is Peter uh and he identified     
9:23     
as the cortical labs in but but seemed you know was was quite um adamant that     
9:31     
you know he's not a part of the research group and didn't have any particular insights um although again it was     
9:37     
interesting to get his perspective on you know the the commercial pressures     
9:44     
that they are under and how that diff you know and and     
9:49     
how that is somewhat limiting in terms of like what approaches that they're taking and um in terms of kind of     
9:58     
shortterm uh demonstration um anyway it was great it     
10:03     
was it was nice to be involved and it was you know because uh these students     
10:10     
being in Santa Cruz are quite close to San Francisco so that was nice and this is coming right on the back of sbio beta     
10:17     
where we got to talk about some of the some of the potential um yeah just     
10:24     
just the the intersection there as well as some of the companies that were were there that do this kind of intersection     
10:31     
of synthetic bio and and um and stem cell research um because yeah all all     
10:39     
the groups that are engaging in this and you know there was again there was probably you know a good 12 groups     
10:48     
represented on this organoid intelligence paper including you know UCSD um Thomas Harding is the the last     
10:58     
author um so representing Hopkins but you know again lots lots of other groups     
11:04     
represented um um check the well yeah just say that but that was really it was     
11:10     
yeah really interesting yeah it's great um we talked a little bit about that on     
11:16     
Saturday but it's nice to have that update um talking about what they're doing so what are they doing with     
11:22     
organoids in terms of people say you know organoid intelligence um right you know would you     
11:29     
tell the group kind of what the that imp the implications of that are yeah you know so um I mean I I I     
11:39     
said that you know despite all the uh I mean one of one of     
11:45     
the undergrads had said um you know that he felt like some of the numbers that     
11:50     
they threw out in terms of Energy Efficiency and like total storage     
11:56     
capacity of you know kind of like one millimeter cubic of of neural tissue you     
12:02     
know it seems seems somewhat arbitrary and perhaps even a little exaggerated and I and I I     
12:10     
did say you know it um it struck me that for an     
12:16     
organoid intelligence paper they had so few like what I would call neuromorphic     
12:22     
Computing expertise represented you know like like     
12:27     
and all of these institutions have very active meromorphic Computing research     
12:34     
groups so it would have been you know for such a a still you know     
12:42     
multidisciplinary uh set of people and and you know research Labs like that     
12:49     
that that aspect was kind of missing and I felt like some of the um uh some of     
12:56     
the needed kind of you know Bridge uh e expertise just wasn't wasn't     
13:03     
there to kind of Def even talk about it sincerely you know um certainly the     
13:12     
the what they the arguments that they gave are     
13:18     
the kind of standard neuromorphic arguments right that that that um you know chip energy use um and seral um     
13:29     
it's going of you know serial execution or are     
13:34     
um you know the the current Paradigm and     
13:40     
and it's it's you know unsustainable limitations at scale     
13:46     
right um and yeah just just just that     
13:51     
and that's you know that organoid I mean that was the funny thing is that they were kind of     
13:57     
using you know neuromorphics uh um arguments and and     
14:03     
numbers but with no clear you know like like we we could we     
14:10     
could barely you know it's one thing for Intel's group to throw around these     
14:15     
numbers because they legitimately have like neural processing units     
14:22     
that you know like yeah you can you can actually get qu quantitative an     
14:29     
from um and that do have you know remarkably     
14:34     
low power requirements in comparison etc etc right so um yeah there there was a     
14:42     
lot of of um wish fulfillment or you know wish     
14:48     
wishing um that we could achieve neuromorphic U numbers yeah uh but but     
14:55     
very little in terms of you know kind of you know proper breakdown of what did     
15:01     
Pong achieve and you know and and I felt that that even their their use of the     
15:07     
dish Brin paper was very um you know this is our this is our best example of     
15:15     
what we're talking about and yet no real breakdown of you know     
15:20     
how how easy it was to achieve that with with just neurons in a dish you know     
15:27     
like even even this the dish brain part of it seemed almost unnecessary as as     
15:33     
like Max hodak and others have have stated privately you know like lot lots     
15:39     
apparently lots of grad students have trained neurons to play Pong oh yeah but that's a pretty easy control problem     
15:46     
yeah yeah yeah I mean to the to the to the point yeah yeah which is like like     
15:51     
is this and it it um it kind of reminded me of the criticism     
15:57     
of the Indiana group that was doing um     
16:02     
um what was it like voice voice identification I think with organoid     
16:07     
yeah like you know the paper itself was nice     
16:13     
from a biology electrical engineering standpoint     
16:19     
um and you know that was the other issue where it was just like why didn't they     
16:24     
have anybody who whose expertise is kind of kind of brain impl plants in terms of     
16:30     
dealing with the the um uh Material     
16:35     
Science interactions with the cellular you know yeah cell culture yeah um uh in     
16:44     
terms of you know what's what's feasible especially for longterm you know like it's one thing to say you're growing     
16:51     
these again if you're going to make a supercomputer out of them right like like you're you're using you know I I     
16:59     
again I would assume you're using the same organoids over time you know like like what kind of what's going to happen     
17:06     
with a lot of long-term electrical stimulation of those organoids you know and     
17:12     
um yeah anyway lots lots of holes in it it certainly wasn't fair to put these     
17:19     
undergrads under the gun in terms of trying to explain them um but um but I     
17:25     
thought it was really interesting in terms of U you know getting a group together and um their interest like like     
17:34     
I could tell from the background that there was a lot of um actual little sin bio people there okay um that was     
17:41     
interesting and that were kind of looking at it as a um is this something     
17:49     
that my lab would be well positioned to to get involved with that that was kind     
17:55     
of encouraging too yeah and um yeah a a lot of work still to be done on that     
18:02     
in this in this front uh as the neuromorphic oh and the other thing     
18:07     
saying that um you know I pointed out the fact that open neuromorphics kind of principal is a professor there at     
18:15     
UCSC and you know like may maybe we could use this as an opportunity to kind     
18:22     
of have as the next Journal Club do something wither more so that was that     
18:28     
was cool yeah sounds great a lot of interesting stuff going on in that space uh and you     
18:35     
know it's kind of interesting how like you know we talk about you know sort of biology and how     
18:44     
things are sort of organized and regulated and then you have Computing     
18:49     
and some of the earliest Computing work was on kind of taking biological analoges you know certainly uh some of     
18:57     
the earliest like neural networks papers were about biological analoges and and then now people are kind of coming back     
19:04     
to that um as it means to see what it can do so yeah yeah yeah and     
19:11     
and I know this is D.A Worman not biocomputing yeah but but I I I'd love     
19:17     
to follow up on the the biocomputing paper that you     
19:22     
um that you covered I think a couple weeks ago in um at the end of Saturday's     
19:30     
meeting and just talk a little bit about that and um uh     
19:37     
um what I was what I talked about on slack which is like um art artificial     
19:44     
molecular machines being a generalization of neuronet like you're saying like like artificial neuron Nets     
19:51     
came from this you know is simplified mathematical representation they they simplified     
19:58     
mathem representations of of cellular Machinery that can also do computation     
20:04     
and learning and that that's that's like you know as as valid and     
20:12     
potentially more so for for kind of cellular Computing uh with organoids     
20:19     
that um that was another interesting track from just a few groups at the in     
20:24     
bya me last week you know and um yeah so Al what was it called     
20:33     
it's like Al Rus bio um was a former MIT grad student     
20:40     
that that gave me some good references on on work that that his Labs there were     
20:46     
doing before he started doing this synthetic bio company um     
20:51     
anyway I'd love to follow up on that that sounds good um yeah so yeah     
20:58     
thanks for that I don't remember what that paper was uh certainly I'll have to maybe look it up but um yeah these kind     
21:04     
of mathematical representations of biology are kind of also very interesting because yeah yeah it was it     
21:12     
was like maybe but with like like certain logic components that um you know kind of Bio     
21:22     
Computing it seemed very similar to me as as like a void like bios like logic     
21:30     
circuits but anyway yeah thank you oh yeah no problem yeah so that's good uh     
21:36     
so it said cocky is here uh and then mul was here but he had to leave so um yeah     
21:43     
and then dick said he found his permissions on on Mac OS uh system settings privacy yeah it's     
21:50     
always a security issue sometimes it blocks the camera um so if you're using ma go asking your problems in jitsy it's     
21:58     
a security thing on Mac or so I don't know um you have to apparently have to     
22:04     
give permissions for every app that you have so um so yeah     
22:12     
yeah the browser oh okay yeah not oh yeah you have to get     
22:20     
permissions in the browser but yeah um yeah so that's uh good um so     
22:28     
yeah it looks like may had to leave but I I'll go over some things for Google summer of code our community period uh     
22:36     
so we're continuing with our community period and uh we started at the beginning of the month so hopefully the     
22:43     
whole and Pocky had been going through some of the resources I provided uh you know at the be when they     
22:49     
were accepted to the program but I wanted to talk about today some open data sources that we have in the group     
22:57     
and how the could be uh utilized perhaps during the course of your uh experience     
23:04     
as sum so um let's let's take a look let me share my     
23:10     
screen and so you know your problems you'll be working on programming problems but you always need some sort     
23:16     
of Benchmark data and I think some of you you know during the application     
23:21     
period as for uh data and you know we have different types of data that we can     
23:28     
go to I think uh during the application period we talked about the cell tracking challenge data sets so there's a website     
23:36     
for the sell tracking Challenge and those are nice validated data sets that     
23:41     
you know give you some benchmarks for different algorithms they've been already you know they've already been     
23:47     
benchmarked by different research groups the problem with those data or the advantage of those data is that they're     
23:53     
Benchmark the problem is they're not necessarily specific to what we're trying to do     
23:59     
let's say like C Elegance or development so there's a range of data sets there     
24:04     
that have you know they're good for doing machine learning and other things like that but they have uh some     
24:11     
limitations as well so we have some resources fortunately you know on the     
24:17     
types of data sets we're interested in and I'll go through those now so one of the longstanding resources     
24:24     
that we have is dvo zo uh this is uh you know we haven't we updated this in a     
24:30     
while I'd like to come back to this and build this out a little bit more but basically what it does is it offers uh     
24:37     
different types of data uh that we have in the group their pointers different data sets that we have access to so some     
24:44     
of these are numeric data sets that we have in XLS files or CSV files you know     
24:50     
things that are tabular uh and they're not Imaging they're not image data but     
24:56     
they've already been extracted from image data and they provide numbers and labels and so we have things for the     
25:03     
seans lineage tree we have things for cell birth and death timing we have     
25:09     
things for the differentiation tree we have data for cell position and then     
25:15     
some data for gene expression although this is very limited and I'd like to be able to add more on uh gene expression     
25:23     
unfortunately there isn't a really good data set that gives gene expression for un aell cell byell basis for C elegans     
25:31     
but that's something that you know we we always have access to different atlases and things like that so the next thing     
25:37     
is we've had these collections of microscopy movies that we've collected over the years we have like four basic     
25:45     
uh data sets that we've worked from in the past we have this uh multi view spim     
25:51     
data set of cigan development where you know you have this uh embryo that's     
25:57     
expanding in terms of dividing cells so you go from a two cell state to like a     
26:04     
256 cell State and you can track the cells dividing and it gives you pretty     
26:10     
good resolution so you can take that video unfold it some of these already have labels or they have uh markers     
26:18     
molecular markers that you can track uh there's also uh some other data     
26:24     
sets that have that are maybe a little bit less uh uh uh good in terms of     
26:30     
resolution but they offer good training sets for you to to use so you can look     
26:35     
at the time series of the embryo uh we also have uh data from SE     
26:40     
squirt or aidans this is embryo data this is early ciona intestinalis     
26:46     
embryogenesis this is actually numeric data and then we have microscopy movies     
26:52     
um that are you know track this type of embryo Genesis so this is the C squirt     
26:59     
which is a different organism than sea elegans and this might be useful for benchmarking uh you know different uh     
27:07     
you know we just usually focus on SE elan's uh embryogenesis in these     
27:12     
projects but we can also use other organisms to get a sense of some of the things going on during erogenesis some     
27:20     
of the cell division events and how those can be kind of detected as features and then we have the fruit fly     
27:27     
uh Data Center again these are um I think these are like various so the IM imaginal disc is     
27:35     
a drawing that you can look at and you can use it as a way to train your models     
27:41     
we also have some data on fuse cell embryo Genesis and I can't remember exactly what those data entail but we     
27:49     
have data from gropa again this is just looking at these processes of cell division and differentiation and it     
27:56     
gives you some IDE you know it would allow you to train a model on these kinds of you know processes as they're     
28:03     
unfolding uh we have zebra fish uh we have mouse spider and then we have these     
28:10     
basaria which are single cell morphogenesis models so you know we're trying to get like we have all these     
28:17     
different uh data sets and the idea is to get a good sense of you know how to     
28:24     
characterize these different processes how to extract features from the process     
28:29     
and then put together some sort of U you know you know put it into an analysis or     
28:36     
put it into a model and things like that so that's dvo zo and I need to clean that up some uh but you know it's a     
28:43     
constantly evolving thing uh we get access to data sets and you know those     
28:48     
are you know you can use those as you want put those together in different ways uh we have this longstanding     
28:56     
repository on GitHub for dorm this is where we have uh different data sets     
29:02     
again this is a lot of the data sets here are linked your dvo zo but we have     
29:08     
additional data here on uh L lineage tree     
29:13     
information um there some other things that we've done with differentiation     
29:18     
trees uh cell birth and death timing and celegans there's raw nuclei data for     
29:24     
cgans and so forth so we have a combination of numeric dat data and     
29:29     
microscopy data where you can take the microscopy image apart and have an image     
29:34     
for each time point and then you can you know stack those together and say you know this is a process that unfolds over     
29:41     
time we have an image here where we have a single nucleus or a single centroid     
29:46     
and then that divides into two centroids and you can track things in that way and     
29:52     
the cell tracking data sets that exist and I'll get to that next uh those are     
29:57     
where you have uh you know you they'll take high resolution microscopy images and then     
30:04     
they'll track the nuclei or the centroids that they can extract from those data they do this uh     
30:12     
computationally and they actually have cell tracking data sets where they can track those in terms of their 3D     
30:18     
position so you have the stack of microscopy images you have this time     
30:23     
series of of those stacks and you can put together three-dimensional volumes that have these uh centroids in them and     
30:32     
you can track their evolution so if they divide if they create more cells you can     
30:38     
get all that information so when we do things with development we're trying to do we're trying to get to that point     
30:45     
where we have some volume which is the embryo we have these centroids in the volume which represent the cells and     
30:52     
those Cades are going to divide and migrate around the embryo volume and and     
30:58     
so being able to track those is important but also being able to detect different types of patterns in them and     
31:05     
so you know whatever the features are there they can be extracted by the algori now Bradley question about that     
31:15     
uh does that uh when you get the 3D coordinates of the uh     
31:22     
cells can you identify uh cell divisions indidual cell uh you can uh     
31:31     
you because those the cells and seans are labeled with names of the cells so     
31:36     
we know they're General location and so a lot of the cell tracking data that exists for seans will have labels on the     
31:43     
cells and you can easily label them by knowing that they're a to position and so you know we can track like divisions     
31:50     
and we know what the cells will become you can also track divisions through like looking at like a centroid and then     
31:57     
having two daughters come off of it okay in in     
32:03     
dropa uh the the work uh my Virginia oh     
32:08     
what was her name uh I forgot her off here okay any rate     
32:14     
uh she discovered that mitosis occurred in waves is there any software to detect     
32:21     
waves from the 3D data uh well we don't necessar well that's so some one of the     
32:28     
things we've been trying to do with uh Diva learn is to to detect higher level features so the higher level features     
32:35     
are going to be things that you know you can detect if we also have membrane data     
32:40     
so sometimes if we have the right data set people have stain the membran with markers those markers could be like     
32:47     
fluorescence or something else and so you can the algorithm can pick those uh     
32:52     
Contours up and then you can actually ask questions like how does this cell     
32:58     
change shape how does the Cent you know does the Cent split so it means that's     
33:03     
representative of a cell division and then how do the cells act collectively and so you can get you can kind of     
33:10     
extract maybe higher level features like waves or other types of migration     
33:16     
patterns or you know there other kinds of movements Collective movements and     
33:22     
development that we might want to be able to characterize the um I'm not sure people     
33:28     
have done that people have tracked cells quite well but I don't think people have     
33:34     
extracted higher level features like that's something might project in itself     
33:42     
yeah to see what conord Behavior occurs between cells     
33:49     
yeah especially long range the     
33:54     
whole yeah and you know espe especially for the graph neural networks projects     
34:01     
you know we're trying to build these graphs and U you know the graphs of course are not just static they're often     
34:09     
times they're adding nodes they're changing their topology so if you have a     
34:14     
graph that's random uh you know at some point it might not be random so like a random     
34:20     
graph is just where things are connected uh you know in no particular order or or     
34:26     
in any sort of like like hierarchy or anything like that uh eventually you     
34:32     
will will end up maybe with some sort of structure like a small world Network or a scale-free network where the number of     
34:39     
connections per node is variable um in in a in a predictable way     
34:46     
um but that that's what happens in the connectone for example in C so you know     
34:51     
those are things we might want to detect and those would be signatures of things like you know waves moving across the     
34:57     
network or organization of of phenotype where you're getting differentiation has anyone made a movie     
35:04     
of The Changing network uh not in terms of a network uh     
35:10     
people have there some of the movies you know they show kind of like you know     
35:15     
cells kind of dividing and the embryo changing shape and it's hard to detect     
35:21     
any patterns within that because you know I mean you can look at it and see and say oh yeah that's a wave or     
35:26     
something but it's hard to to actually put like a mask on it and say these are the cells that are     
35:32     
involved that's kind of the thing we're trying to do there is what are these networks of uh well you know usually an     
35:39     
embryo Network say is like cells that are adjacent to one another use some     
35:44     
Criterion to measure like the okay so the cells would be and the edges would     
35:50     
be distance between them uh they could be the distance or they could be like some signaling mechanism like uh sell     
35:58     
toell communication or something like that okay now if you did that versus     
36:04     
time you see anything um well I mean we didn't really run that analysis uh on you know these     
36:12     
networks at snapshots and time and then comparing so that's yeah I can only     
36:20     
say what was that might be worthwhile doing it yeah yeah yeah that I mean I     
36:27     
can only say it from my kind of like intuition I haven't actually produced the graph where we know what kind of     
36:34     
what the they're different network measures um last summer hansi worked on     
36:39     
some things with topological data analysis and that was kind of where that was going in in a way because you can     
36:46     
take like a graph or you can take like a a topology of a graph and you can look     
36:51     
at how you can summarize it with a single parameter and you can look at it how over time it changes and so this is     
36:59     
like we didn't really get very far on that but like that's the idea you'd have like one or two parameters representing     
37:07     
shape and you would say you know there's some shape that we're interested in in the data in the network and is it how is     
37:13     
it changing over time you know is it changing is it like you have like a ring     
37:18     
say is it narrowing down is it expanding out is it like becoming oblong things     
37:25     
like that and then you could measure that over time so you can measure different aspects of the     
37:31     
topology so that that's that's kind of one way maybe to do that but again don't     
37:37     
we don't have that graph that actually shows that very well it's kind of a a     
37:43     
reach goal at this point well in a way you do because if you have the     
37:49     
uh if you have the 3D coordinates you know which cells were derived from which     
37:55     
ones we can make a graph of that and uh in a sense the     
38:03     
graph is threedimensional and CH changing with time it would show     
38:09     
you where the cells are versus time and the where the so to speak where the     
38:15     
action is yeah so it might be worth while making a     
38:22     
graph of the 3D positions and just making a movie of that see what you     
38:29     
get yeah yeah yeah there are a lot of things     
38:34     
we can do here it I know like uh I know we'll probably get some real interesting     
38:41     
data from the summer so we'll see what we can get there too but yeah just kind of coming up with some of these um time     
38:48     
series analysis that be interesting yeah so that's yeah that's basically     
38:55     
what we're trying to do um with respect a lot of the data sets but then there's     
39:00     
also uh this uh resource and this is the systems uh the     
39:05     
ssbd database system science of biological Dynamics database and this has access to a lot of     
39:12     
different developmental specific data sets so this is you know sea Alan's data     
39:18     
is here but they also have data from zebra fish from ecoi from dropa and so     
39:26     
you know these are good qu data sets these are published data sets with uh metadata so you have this you know     
39:34     
process of development where someone's taken embryos that put them under a high     
39:39     
density micro or high resolution microscope they've gotten you know information for very short time periods     
39:48     
so you know you have this sort of almost continuous uh flow of of movement of     
39:54     
cells divisions uh migrations whatever and then you have tracking data so it     
40:00     
has some labels and some like you know spatial information that you you know     
40:07     
you don't have to sort of add of course you can add spatial information you could say for example add a coordinate     
40:13     
system to an image like this but a lot of the cell tracking software will add     
40:18     
in a coordinate and then you can normalize this coordinates so it makes it a lot easier to kind of track cells     
40:25     
relative to one another so this this is a database that gives a lot of these different     
40:31     
resources um you know we have quantitative data of different types so this is a stub for zebra fish this has     
40:39     
the nuclear positions of zebra fish this is uh this is from Philip     
40:46     
Keller uh so you know this is a from a paper science paper in 2008 they     
40:52     
published their data set now some of these data sets are quite old uh you know they're over 10 years old but     
40:58     
they're still useful they're very useful for getting you know the information and that's what we're interested in um zebra     
41:05     
fish you know uh embryogenesis and SE elans embryogenesis don't change that     
41:11     
much over like 10 years the Technologies get better but the technology I think here is is is fine for what we want to     
41:19     
do so you know we have access to this in a zip uh ZIP file or XML and so you can     
41:26     
take these and and it's an exercise in building a small pipeline so you pick     
41:31     
the data that you want you open it up you have a program that opens it up you have a program that validates the data     
41:38     
to make sure it's there in the right order that you know what the variables are what they mean and then you put that     
41:46     
in you analyze you maybe produce some graphs or you can train a model and use     
41:52     
that okay has this has this been done for the set shorting problem     
41:58     
uh or uh or organoid organoids uh what do you mean like uh cell sorting is     
42:05     
where you separate the sizes of different tissue types uh and then you put them back together they're initially     
42:11     
random and they sort out make onion structures okay okay this is the old uh     
42:19     
trink house I think started it and uh nothing Steinberg continued this     
42:25     
work okay okay uh so you know     
42:34     
it's there there uncertainties about what was happening so you could observe it in 3D     
42:41     
and uh track the individual cells probably get some interesting     
42:46     
information about what they're doing yeah yeah I mean that would be yeah     
42:52     
another option to to with this yeah there are a lot of things you could do with it I guess     
42:58     
and there's all sorts of organiz organized around so     
43:03     
uh it's should be possible do with organoids too and see what's going on oh     
43:08     
yeah yeah yeah there are now I didn't even mention organoid data sets but there are open organite data sets I know     
43:14     
Mor probably know there are good yeah well yeah there are some I mean I don't know like some of them are uh     
43:22     
electrophysiological because a lot of people are interested in like what they're doing that way but they're also     
43:29     
I think they've done some cell tracking on some organoids um but they they definitely have like organoids that have been     
43:35     
stained so they have like microscopy images of them things like that     
43:43     
so uh yeah so I mean there are a lot of things we can do with these data aside from like training models we can you     
43:49     
know look at different tissue types we can look at different uh uh you know higher level features we can do all     
43:55     
sorts of things like that and and so this the point of this was     
44:02     
just to go over kind of some of the resources that we have on these things and U you know if if you need something     
44:10     
uh some sort of data set for your gso project uh we have those available and     
44:16     
and we have other types of data that we can find this isn't the limits of data     
44:21     
that's available so there are all sorts of data sets that are out there um some     
44:27     
are you know kind of we want to make sure it's decently validated in other words we want to make sure maybe we have     
44:33     
labels or we can label the data if we need to or that it has you know that it's uh if we want to take multiple     
44:40     
microscopy movies that we can align them properly we don't want to just you know     
44:46     
kind of grab the data um sometimes you know we have to do that like when we were doing the uh digital Bassel area     
44:53     
project we had all these videos from YouTube people have taken videos of Basel area colonies and you know you     
45:01     
just extract them and you know you kind of take the data and you build a little data set and you know it's a pretty     
45:08     
simple process that you want to isolate so it's not a big deal uh but it's you     
45:13     
know it's still you want to be able to align the data properly and so being able to making sure you can do that is     
45:20     
an important thing so there's a pre-processing step and I wouldn't like spend a lot of time     
45:28     
on that but I would be mindful of it and then there's the analysis step getting it you know analyzed to a point where we     
45:35     
can then make some insights or graphs some other     
45:42     
things okay so that I think that's enough on that um now I wanted to talk about something what I was doing last     
45:48     
week we didn't have a meeting last week and that was that I was at a workshop okay I'll send I'll share a link in     
45:54     
slack SS Ed data so I was uh at a conference where we were talking about     
46:00     
building minimal cells and this is interesting because I was uh this Workshop was put on by a group that have     
46:07     
been modeling uh minimal what they call minimal cells so they're these cells     
46:12     
that are um the if you're familiar with the J cig Venture Institute they've been     
46:19     
building what they call the minimal bacterial cell and that bacterial cell is like the minimal cell you can have     
46:25     
that's still alive so it's like got just the genes you need for like uh energetics and maintenance and that's it     
46:32     
then you can boot up with new jeans you know for specialized things but you know     
46:37     
you're working from minimal cell so the upshot of that is that you have a model that you something that you can model     
46:44     
relatively easily you don't have to worry about a lot of things like you know uh you know different organel or     
46:51     
things like that so I'm going to talk about that I'm going to go through a couple slides and then some resources on     
46:59     
this so um so this is building minimal cells and this was this Workshop here     
47:06     
this was put on at the University of Illinois this is uh called the advanced     
47:11     
computational workshop simulations and visual visualization of a minimal     
47:17     
bacterial cell so these are bacterial cells not animal cells I think they've tried to do some work in East so far so     
47:24     
they're moving up from bacteria to yeast but they're not there yet so their focus of this Workshop was on bacterial cells     
47:32     
um so they they covered a number of topics here uh you know it's like all     
47:37     
week so they talked about the minimal cell which is the jcvi sin 3A which is     
47:44     
the standard uh Workhorse of of that um and they they they had to figure out how     
47:50     
to model that so they have the sort of the uh biological specimen then they     
47:56     
have to figure out how to model that cell so they have to figure out how to model the genome and how to model the uh     
48:04     
proteins and we're talking about basically a vess so it's not a huge     
48:09     
hugely complex thing although you'll see that it's actually looks pretty complex     
48:14     
uh you know we use a number of computational models and put them together in a pipeline to simulate the     
48:21     
cell so this is like kind of what we're doing in open world with some of the different tools so we're modeling     
48:27     
movement we're modeling um the neurons we're modeling the connectome we're     
48:32     
modeling the muscle and all that and so they're doing this with a bacterial cell by putting together different tools and     
48:39     
and producing a visualized output so if I go back to the uh slideshow here this     
48:46     
is jcvi sin3a this is a micoplasma and so this is actually a species they call     
48:53     
micoplasma laboratorium this comes from from micoplasma genitalium which is a a a     
49:01     
micoplasma with a bigger genome and what they've done is knocked out the genes that are un non-essential so they do     
49:08     
these knockdown experiments combinatorial knockdown experiments where they would test each cell     
49:14     
basically test each gene they remove the Gene and if the cell died that it was     
49:19     
essential the cell didn't die then it was nonessential and it was removed from that genome so in doing so they could     
49:26     
generate a minimal set of genes that are required to sustain life from The genome     
49:31     
and so you know they may had they did they've done some additional uh genetic     
49:37     
engineering to enhance some of those genes to make sure that the line is is uh you know can be propagated but they     
49:44     
end up with 482 genes and 582,000     
49:57     
with a A lipoprotein membrane with soluable proteins inside these purple     
50:03     
circles these purple balls um ribosome which is this green ball bigger green     
50:09     
ball you have these black uh things that are uh twisted around here this is the     
50:14     
DNA so it's a circular genome but it's kind of packed in there and a looks like     
50:20     
it's twisted around like a a coil or a cord of some type um and you have     
50:27     
soluble RNA which are these uh yellow like squiggles and all that is packed     
50:34     
inside of this uh vesicle so you have a lot of stuff in there and you know but     
50:40     
it's not like hugely complex you can model this using molecular Dynamic simulations and that's what they were     
50:46     
talking about in this session so if you look at the model that they build this is the computational model here as     
50:53     
opposed to the microplasma here this is where where they've modeled all these components and they're packing in this     
51:01     
uh inside this vesicle they have the outside of the vesicle uh with with receptors on the edge and then inside     
51:08     
you have all these things that are packed inside you have this sphere of a certain volume you have water you have the     
51:16     
physics of water and you have all these components like DNA RNA proteins and so     
51:23     
we're at a very minimal State not only in terms of the genome but in terms of what's inside this vesicle and so the     
51:30     
idea is that you want to model this as a single cell you can do this in two ways     
51:36     
you can do this using a lattice micro methodology which is where you basically have this whole inside which is the     
51:43     
cytoplasm and you sample the cytoplasm for the different components you look at     
51:48     
the metabolism and you use a chemical Master equation to estimate what the the     
51:54     
inside of the cell is at any time and so you build these models from the sampling you know kind of what the constituents     
52:01     
are of DNA and RNA protein and you can do these different uh estimates uh you     
52:07     
can also use something called Martini or gromax which produces beads so you can     
52:12     
either have this lattice which is like a bunch of points and you know like this     
52:18     
this lattice it's kind of like a triangular mesh or some sort of uh     
52:24     
rectangular mesh that covers this covers the uh entirety of the of the vesicle     
52:31     
and you can actually put things inside those inside that mesh so you can have like a triangular Motif and there are     
52:37     
things inside of that and you model that as a unit of the cell and you can do that with lattice microbes or you can do     
52:44     
this uh use this gromax package where you produce beads which are these like little volumes of of cor springed     
52:51     
volumes of this space and inside the volume you it's almost like a voxel in your Imaging where you have things     
52:58     
inside that volume different chemical reactions different atomistic reactions and that's how you would represent this     
53:05     
what's going on in the cell so you know they are able to look at DNA replication     
53:10     
trans transcription translation so you can go from DNA to proteins you can generate     
53:17     
trajectories uh so what they do is they basically simulate this in these different ways and then instead of     
53:23     
predicting every interaction over like maybe 50 to 100 micros seconds they generate trajectories so they say what     
53:31     
is the mean behavior of this model and what does it look like over time what do     
53:36     
the metabolism look like uh from thir our initial condition to maybe like cell     
53:41     
division and so that's how they get a sense of what the minimal cell is doing over time and then the visualization     
53:48     
step involves something called bmd which is uh uh visual molecular Dynamics or     
53:54     
they actually are using Minecraft which is a program in uh that creates these Virtual Worlds um that you know you can     
54:02     
interact with in a VR headset so the lattice microbes methodology is where you have these     
54:10     
lates and you know things that represented in these lce like membranes and ribosomes uh you know you can do you can     
54:17     
simulate these on high performance Computing nodes so this is something that requires a lot more than like a     
54:22     
laptop you use like many cores to do this and you can set up a job and you     
54:28     
know there's you need a lot of computational power um what you do is you sample the     
54:34     
cytoplasm within the minimal cell you feed it into a chemical Master equation     
54:39     
in this case it's a reaction diff Fusion type equation and you can make predictions about what's going on in the     
54:44     
Cell at any time and then of course they have different assumptions so sometimes this you can make the assumption that     
54:51     
the cell is well stirred meaning that the components are just kind of drifting in there r     
54:57     
or sometimes there's spatial segregation in the minimal cell so sometimes things are clumped in different places this is     
55:04     
maybe you know that you can use both assumptions the homogeneous assumption works most of the time but when we start     
55:11     
getting into cells that exhibit differentiation we're going to have this problem of heterogeneity so that adds     
55:18     
computation we need to have better computational resources devoted to that     
55:23     
and this Robert at all paper yeah yeah one curious thing about this work was that the homogene the well homogeneous     
55:32     
is good enough in most cases yeah uh which means     
55:37     
that the actual three-dimensional structure inside the cell is not     
55:44     
important Yeah well yeah they're just the concentrations right substances     
55:52     
right yeah I mean that's that's one of the things about like these models is like you're making tradeoffs so you're     
55:58     
saying that you know you're making assumptions about sort of the you know     
56:03     
the energetics inside the cell the metabolism it's not often an equilibrium     
56:09     
so but you still make an assumption about the behavior you you you know you don't try to model every interaction you     
56:15     
try to model sort of the averages and then again you know the 3D resolution you can model it with these beads but     
56:22     
really you're making the assumption that there's no particular structure that you need to replicate so you can just take like things at     
56:28     
random and say well everything is basically well mixed it's fine and it saves you     
56:34     
computational effort because otherwise what you would have to do is you'd have to model     
56:39     
subcomponents and they'd have to be modeled differently and and that sort     
56:45     
of so this Robert's paper kind of goes over what lattice microbes are it's a     
56:50     
stochastic simulation method for modeling this reaction to fusure master equation it's very much chemistry and     
56:58     
and and physics uh based modeling so there's gromax Martini gromax is a     
57:04     
package that allows us to do this modeling uh where we're producing a trajectory in a computationally cheapest     
57:10     
manner possible so if you have a system you want to be able to produce a trajectory to say what is the state of     
57:17     
the cell what is the state of these different uh components of the cell and what you know how can we estimate that     
57:24     
we can't really again measure every interaction not least of which because they're not necessarily important to get     
57:31     
those precise but because we can't like you know we don't have the computational     
57:36     
power to do that so you know we try to approximate things in this uh we try to     
57:42     
approximate things like hexagons and bounding boxes and they were talking about these different methods like     
57:47     
triclinic and rectangular methods which reminded me of how we think about like cells you know we think about Tri     
57:54     
Junctions and quad Junctions and cells and how those are important and that     
57:59     
that's one of the things that they're doing with modeling they're trying to model at that level of like what's     
58:05     
interacting with what we have these like kind of volumes that we can you know the sub volumes that we can take and you     
58:12     
know how do they interact with each other um and then this this paper here has ji andl 2020 backmapping     
58:19     
triangulated surfaces course brain membrane models this is where you have     
58:25     
these surfaces that are in this triangular mesh and you're mapping that to sort of a coar brain model where you     
58:31     
have these volume smaller volumes that tell you kind of you know what what are the atomistic interactions in this     
58:37     
volume and you know it's just we're not trying to model every atomistic interaction not trying to model like     
58:43     
water molecules we just want to know what's going on in the little volume um     
58:49     
so you know we use force fields physical force fields to simulate like things     
58:54     
like water things like actions of proteins Etc you'll use repeated motifs     
59:00     
so that cuts down on your computational time so if you have a bunch of triangles that are doing the same types of things     
59:06     
you might use that as a single unit in your computation uh and then we have this corar graining method which is where we     
59:14     
choose a higher level of analysis so this is important when we do things like     
59:19     
try to abstract things to the phenotype and this is especially true in like     
59:24     
organism modeling where you know we say what do we want to model we want to model every molecular interaction do we     
59:31     
want to model every atomistic interaction is it just good enough to just Model A Cell as a cell and maybe     
59:37     
have some features as a cell or maybe we don't even care about cells maybe we care about different higher level     
59:44     
features in the phenotype whatever those do so this this is a term they use in this area called core screening which     
59:50     
means you take the simplest interactions and you combine those into maybe volumes or     
59:56     
you know motifs and you kind of model those things and you you get the sense     
1:00:01     
of what they're doing and so you know it's a choice you can um model things     
1:00:07     
with high Precision every interaction sometimes it doesn't matter sometimes it does but you know sometimes you just say     
1:00:15     
well I want to core screen things because I don't really don't care about these lower LMS and so this is this is a     
1:00:21     
you know this is all about modeling and choices you make during modeling uh um you know so they do a     
1:00:27     
number of things when they do this they build a vesicle build a membrane introduce proteins polymers and DNA and     
1:00:34     
then we also introduce some water so we have we build a simulation by introducing these different things and     
1:00:41     
then putting them all together we'll use statistical mechanics ensembles to model     
1:00:46     
the Dynamics of chemical species reactions and particles so all those things in that in that model in the     
1:00:52     
volume will do things uh one of the things do is they build this poniz mesh     
1:00:58     
so we talked about pontalis in the meetings uh and this is something that     
1:01:03     
dick is interested in with respect to uh Medical Imaging but they actually use     
1:01:08     
pism in this uh modeling framework as well and that's interesting because um     
1:01:14     
and I'll show some stuff on it real quick at the end of this it really allows you to deal with uh things like     
1:01:20     
coordinate systems and curvatures um and there's been a lot of interesting work in computer vision on     
1:01:26     
quantizing things in modeling so it's that's an interesting point I picked up     
1:01:31     
uh maybe to look into later there are a lot of open data sets we can use metabolomics data sets proteomic data     
1:01:39     
sets and genomics data sets so you know these are these are things that are uh open to people to use you you take some     
1:01:46     
data you use it to sort of inform your model you add all your components in you     
1:01:51     
estimate things from the data and then you you know have your simulation so your simulations you know it depends     
1:01:59     
on what you want to do so in these minimal cells they simulate things at T tens to hundreds of microseconds and     
1:02:06     
that doesn't seem like a lot but actually in terms of like the molecular reactions the scales of those are much     
1:02:13     
smaller so when you have your time step in your simulation you know what are you representing that time step is is the     
1:02:21     
reaction of Interest or the phenomena of Interest sort of do you do you core scin     
1:02:27     
your model on time well enough to pick those up so if I'm interested in embryogenesis you know I might want to     
1:02:34     
uh course grain or fine grain or Co grain my model to some time period time     
1:02:40     
step that allows me to see the relevant Dynamics so if I'm interested just in     
1:02:45     
like protein folding I want to have a very short set of Windows if I'm interested in maybe like cell division I     
1:02:52     
might have a longer set of Windows that sort of or not you know cell division is     
1:02:57     
interesting because there are a lot of things happening at very short time scales and then things happening at longer time scales then this gets all     
1:03:05     
gets visualized into this kind of a model and then so one example problem     
1:03:10     
they gave was of cell division so in their models here you know these are static models in the sense that they're     
1:03:16     
just this model of a cell with stuff in it but you can actually have models that     
1:03:22     
where the cell divides and so you that this outer membr needs to stretch out     
1:03:29     
elongate and then all this stuff needs to double in number so all the DNA the     
1:03:34     
protein well the the components need to double at least the DNA needs to double and so the idea here in the simulation     
1:03:41     
is that all of these components need to double in number but they don't you know they need to do this as the the uh     
1:03:48     
vesicle membrane is stretching out and so as it's expanding you're doubling eventually doubling the number of the     
1:03:55     
components the water is expanding everything out and then eventually you get this divion     
1:04:01     
where the membrane of each daughter cell closes and then you have two celles and     
1:04:07     
so that's what they were looking at modeling um now you know we have a     
1:04:13     
problem where we need to represent this properly so sometimes you know we need to take these contents and we need to     
1:04:18     
segregate them out to each pole of this elongating uh membrane we need to then     
1:04:24     
have something in the middle to pinch off the uh single cell into two cells so     
1:04:30     
the vesicle membrane must grow stretch and pinch off accordingly and this all needs to be done with these lates and     
1:04:36     
Beads and things like that so it's not that we're actually observing this process it's that we need to model this     
1:04:42     
in a in the context of our Digital model and so we can simulate different     
1:04:47     
mechanisms like ftsz which actually accumulates uh as a ring in the center     
1:04:53     
of a cell or a vesicle that allows us to have this sort of cell division where     
1:04:59     
this F ftsz ring like contracts and pinches off this pinches off the     
1:05:05     
membrane so you know you can simulate that process that's a protein driven     
1:05:10     
process you can actually also simulate this segregation of materials this     
1:05:16     
doubling in number of materials and this stretching of the vesicle so all those     
1:05:21     
things we need to model for this process to happen and they showed how this can be done in their models so you know this     
1:05:29     
is microplasma laboratorium this is the Wikipedia page for it if you're     
1:05:35     
interested uh then you know there's some open source software associated with this so this is the software for the     
1:05:41     
workshop this is actually a vesicle that's dividing so this is a vesicle it's stretching out and doubling the     
1:05:47     
amount of DNA in the vesicle so you can see how this thing stretches out and starts to     
1:05:53     
pinch um you know they have code here here for the DNA model for this model as     
1:05:58     
well uh which is the tutorial I mean this is just stuff that you can't really do on your own but you can run it if you     
1:06:06     
have access to a cluster you can run it and then there's some images in here where you can see some examples uh     
1:06:13     
there's also this uh this is the gromax uh open- Source uh repository where they have a     
1:06:20     
number of uh tutorials and it kind of goes through how you would build these     
1:06:25     
simulations the kind of commands youd use but also the kind of processes you need to     
1:06:30     
simulate then there's this vmd which is the visualization tutorial so this is a     
1:06:36     
visualization program you can install you can install these on your home on your own computer or on a cluster and     
1:06:43     
you can do really interesting things with that um so I wanted to get back one     
1:06:49     
last thing before we go which is to talk about um this pontalis aspect     
1:06:57     
so uh I found a number of papers so this is the piki and paper on back neck being     
1:07:04     
triangulated surfaces to course gr membrane models and in this paper they kind of talk about this pontalis process     
1:07:11     
so they talk about back mapping on a lattice by sort of introducing these points so you can see these points on     
1:07:18     
the triangle and so you're trying to refine this mesh in terms of its uh     
1:07:23     
shape and in terms of you know know what's inside of it so you're using these points to do that so these is ponm     
1:07:31     
process uh these are to grow vesicles as well um so they're backmapping what back     
1:07:37     
backmapping means is they're refining everything so they're able to use these beads to refine the mesh and make it     
1:07:44     
more realistic and so this is a screenshot here of a pism this is from uh the     
1:07:51     
method section where they talk about pism extending the upper and lower TS so     
1:07:58     
in this step the number of vertices of each TS upper and lower surfaces here     
1:08:03     
this is the triangulated uh surface will be increased to a larger value than the required number of lipids to fully cover     
1:08:10     
their surfaces to do this we first need to make an approximation of the geodesic     
1:08:15     
which is the curvature rep connecting two neighboring vertices consider two vertices that are     
1:08:22     
connected by a vector link one of size l the projection of one on each vertex     
1:08:27     
plane is denoted here we require that a geod deser connecting the two vertices     
1:08:32     
satisfy three conditions on each vertex uh six conditions in total so they go     
1:08:38     
through this and they're basically using these points to refine the mesh the triangulation mesh and the geodesic so U     
1:08:47     
this is actually something that has been done in uh this is we can build     
1:08:52     
algorithms for pism and we can do pism pain paintings and pythons so this can be implemented actually using py game as     
1:09:00     
a as a toolbox and so we can actually do this and so quism has been well studied     
1:09:05     
in computer science and it's been used in computer science for a number of applications this is an example of pism     
1:09:13     
uh just in general so it's not in a biological simulation but they've been able to do this simulation where you can     
1:09:19     
generate points um in a model um and you know and then of course this paper here     
1:09:27     
which is a 2013 paper um your eurographic Symposium and rendering     
1:09:33     
where they talk about uh generating quantal ISM paintings based on s's color     
1:09:38     
composition where they can get these images that you know they use pism uh sort of they're mimicking different     
1:09:44     
paintings and they're doing this algorithmically so the point is is that you can get these different techniques     
1:09:50     
in quism and put them into an algorithm and generate things uh for different     
1:09:55     
applications so this is you know computer science hardcore computer science uh but it's it's something that     
1:10:01     
can be used not just in Fine Arts but in biological     
1:10:08     
model so that's all I wanted to talk about with respect to that I thought that was a really interesting workshop     
1:10:14     
and they know they have their model of like simulating cells right it's not like that's not what poorm does and it's     
1:10:22     
not what other groups do but you're working from the sort of first principles this     
1:10:27     
abono uh type of approach where you're you kind of know the the biochemistry in     
1:10:33     
the physics and so you're working from that and you're just assuming that you can build a model uh up from that and     
1:10:40     
build a cell okay I've actually used well I call     
1:10:46     
it pointalism tomato tomato thing um but I've used it um for my paintings     
1:10:55     
yeah since that's my hobby um and there's also a compell 3D     
1:11:04     
Workshop this summer um end of July I believe yeah and I found some um a     
1:11:13     
YouTube video about one of their um one     
1:11:18     
of their sessions say 2015 2018 I don't know so um     
1:11:25     
I'll send that to you okay yeah that' be good yes so another thing we can do is compare this approach with copy cell 3D     
1:11:33     
and how they kind of approach problem modeling cells and of course they use uh     
1:11:38     
a different method they use a kind of a you know a different kind of method     
1:11:44     
where you're modeling cells and lattice and you're doing physics on the cell and     
1:11:49     
it's you know you're not worrying so much about what's inside the cells you're modeling the cells themselves and you do you do inform what they do     
1:11:57     
there's you know data about what's you know what kind of processes inside the cell like gene expression or something     
1:12:03     
but it's not like you're not modeling it down to like DNA content proteins and     
1:12:08     
you know getting those things in place and they're not doing T segurity either     
1:12:14     
which is very disappointing yeah I was thinking about t segurity throughout all this I'm like     
1:12:19     
how would you model T secrity how because it seems like it's right there I don't know how it's done yeah well I     
1:12:25     
don't like the way my finite analysis is doing it but I'm going to live with it     
1:12:33     
yeah yeah you know try to get it to stop being so automated     
1:12:39     
yeah he goes oh no it's not settling down maybe it's broken so then it it     
1:12:45     
redo itself and puts another force on it is like you're putting pulses on on the     
1:12:52     
Tegrity and of course it wobbles all over Place     
1:12:58     
yeah yeah not too not too coordinated yeah having fun with     
1:13:07     
that so uh dick says shape of a pixel not important equals an example of     
1:13:12     
pointalism so that what is that could you elaborate on that or uh yes the computer     
1:13:23     
tomography what she did did is in you have to you have to project a pixel onto     
1:13:31     
a line and uh what she did is allowed rotation of the pixel so that it     
1:13:38     
projected as uh the same width as pixel but no matter what the angle was so the     
1:13:46     
pixel was allowed to rotate yeah and this actually improved theion the     
1:13:52     
pictures good yeah so it's kind it was kind of strange you     
1:14:00     
you start ignoring different features of the pixel and it gets     
1:14:05     
better yeah it's sort of like these re retina screens you make your if you make     
1:14:11     
your uh pixels on the retina screen small enough doesn't matter what shape     
1:14:17     
they are right     
1:14:23     
yeah yeah yeah that's the other thing about modeling like we can ignore things but     
1:14:29     
we have to ignore only the right things or else we miss yeah yeah in this case we kept the density of the pixel but     
1:14:36     
not all right well I think that's it for today did anyone have any other things I wanted to talk about before we go or     
1:14:43     
yeah I guess so are you getting anywhere with the uh cing noise uh well a little     
1:14:49     
bit there's still some stumbling blocks there I think but we'll figure that out yeah okay good it's hard to find like     
1:14:57     
existing software though well you know it's kind of like their packages for     
1:15:02     
like uh computer graphics and it's very focused on that then like using it for     
1:15:09     
other purposes is less easy to you know to actually get it to work I don't know     
1:15:15     
if that makes any sense but okay yeah uh there's there very     
1:15:24     
there's python for generating prog was right okay uh which we can pick up but     
1:15:31     
uh I think what were you able to get the parameters out of that paper on uh     
1:15:39     
restrooms uh yeah I'm still working on that okay okay because that's kind of     
1:15:44     
the uh that paper is interesting but poorly     
1:15:51     
written yeah     
1:15:58     
okay all right well thanks for attending today and uh talk everyone next week uh     
1:16:05     
again we're going to be still in our community period for gck so um I don't     
1:16:11     
know we I I'll probably come up with some uh we probably maybe we'll go over some of the papers that we've done and     
1:16:18     
talk a little bit about that so that new people know kind of where we where we're where we've been and where we're going     
1:16:24     
so I also have uh put out a video on the     
1:16:29     
the 10 years of Diva worm so that's the thing I talked about the meeting a couple weeks ago I actually have a video     
1:16:35     
now on the YouTube channel it's about 90 minutes long so it's not uh not trying     
1:16:41     
to torture people but there is a lot there and you know with the YouTube videos you can actually go through a     
1:16:48     
chapter by chapter plan you know interesting things so what what is it called uh 10 years of d one oh     
1:16:58     
okay yeah oh Bradley one one thing about the uh uh the small cells or the mineral     
1:17:07     
cells yeah is that they confirm the concept of autoc catalysis and that is     
1:17:13     
if you throw the right chemicals together the chemicals organize themselves yeah     
1:17:18     
yeah okay so the so aut catalysis why well it has been demonstrated sense it     
1:17:25     
has been demonstrated through the minimal cell simulations yeah uh the     
1:17:31     
other thing is they got about 75% uh of uh success in other words 15%     
1:17:41     
of the cells didn't work right yeah okay but uh that's not bad for a start     
1:17:49     
right yeah like a lot of cells I guess they're saying that their their experiments that they do with Min animal     
1:17:55     
cells there's a significant amount of cell death one of the challenges is to figure out why they die because not     
1:18:01     
every cell dies but when you're at that minimal set you know you're not going to have like you can survive but uh maybe     
1:18:08     
not in every condition so it's like why did they yeah okay thanks for attending uh see     
1:18:14     
you next week bye bye bye bye all right so one additional thing uh related to     
1:18:22     
all the stuff we've talked about about bi modeling of minimal cells this is an     
1:18:27     
article recent article from Steven Wolfram on biological evolution and     
1:18:34     
other adaptive processes and using a minimal model to capture the attributes     
1:18:40     
of that of those systems so this is uh from his blog uh basic as you could     
1:18:46     
guess he's using some sort of cellular autometa so let's go through the model     
1:18:52     
and then some of the other things in this AR we have the model why is biological evolution work and for     
1:18:57     
that matter why does machine learning work both are examples of adaptive processes that Surprises with what they'     
1:19:04     
managed to achieve so this is this idea of computational universality where you can basically     
1:19:10     
take biological systems the machine learning systems and find the fundamental aspects that under     
1:19:16     
viable as as complex systems so he's going to concentrate on     
1:19:22     
biological evolution but then the is also relevant to machine learning and     
1:19:27     
this is going to be the subject of another article so what's the appropriate minimal model for biology so we talked     
1:19:34     
about the minimal models that they're working on with um micoplasma and that's     
1:19:40     
you know having this vesicle with things in it with stuff packed stuff and it has     
1:19:47     
kinetics and it has uh metabolic activity and it divides and you can simulate     
1:19:55     
what's going on inside the cell and also vision of the cell and other things like that um but you know sometimes we want     
1:20:02     
our minimal models be even more minimal than that so this is where we have UC     
1:20:08     
cellular autometa so cellular autom of course is this grid of cells where each     
1:20:14     
grid has certain property each cell on the grid has certain properties and can transition between states and so forth     
1:20:22     
so in this case he builds a Sol autometa an example using cells of three possible     
1:20:27     
colors rules are shown on the left here so we have these transition rules depending on what your neighbors are     
1:20:34     
doing that influences the state that the focal cell takes on and then that is     
1:20:39     
applied in parallel so that you get this these sort of collective     
1:20:45     
behaviors so this is the minimal model here we start with a single red cell and     
1:20:50     
we see from that from this CA structure is grown so we start up at the top with a single SLE red cell and we get this     
1:20:56     
structure of red and purple and white cells coming down and so uh a structure is grown     
1:21:04     
which in this case dies out after 51 steps so step 51 we end up with a single blue cell and then it dies so we end up     
1:21:11     
with with a Genesis of a cellular structure the evolution of a cellular structure and then it's done now we can     
1:21:18     
model this in cellular autometa and this minimal model in this case we're modeling cells interacting with each     
1:21:25     
other as a multicellular Ag glomeration and then dying we could also think about this as     
1:21:33     
a bacterial Colony you know this is very abstract but let's say we start from the     
1:21:39     
trivial or n rule that makes any pattern diode immediately and we end up adaptively evolving to the rule above     
1:21:46     
imagine making a sequence of randomly chosen point mutations so this in this case we have a one-dimensional array     
1:21:53     
where we make point mutation and essentially at any one point in this onedimensional array we change the state     
1:22:00     
so we mutate the state to a new state and then we evolve the system and see what happens and we end up adaptively     
1:22:07     
evolving to the rule above imagine making a sequence of randomly chosen point mutations each changing just one     
1:22:14     
outcome of the rule as in this example here then suppose in each step a minimal     
1:22:19     
analog of natural selection occurs so or minim Al analog     
1:22:25     
of natural selection involves accepting any mutation that makes the lifetime longer without not infinite or at least     
1:22:33     
the same as before and we reject a new mutation that makes the lifetime shorter or infinite so we basically select for     
1:22:41     
longetivity so we accept the mutations that allow for longivity but reject the     
1:22:47     
mutations that limit the lifetime of the structure so you can see that like based     
1:22:53     
on the interactions between cells sometimes you know if you have these one-dimensional arrays that are kind of     
1:23:00     
laid out like this you get this uh you get states that persist if you know you     
1:23:06     
mutate them in different ways you get states that get overwhelmed and this is just simply a product of interactions     
1:23:12     
between neighboring cells in cellular autom sometimes you get convergence to a single state sometimes you get     
1:23:19     
conversions to like all zeros or something like that so this is something that     
1:23:25     
happens in onedimensional arrays as a consequence of its     
1:23:31     
Evolution so we can actually mutate these rules so that you can avoid this     
1:23:36     
convergence and allow the structure to persist so it turns out that with this     
1:23:43     
procedure we can indeed adaptly evolve to the rule above here we're here we're showing only     
1:23:49     
way points of progressiv liberat lifetime so in this in this example here we have have a single red cell we end up     
1:23:56     
with this multi-state Cog glomeration which grows and continues to grow and     
1:24:02     
expand across the grid and so we can see this final step here where we have this     
1:24:08     
very complex structure and you can think of like you know a colony of cells having this property so you know you     
1:24:15     
might have a colony that dies off quickly have a colony that is you know     
1:24:21     
under go a burst of growth and then dies off and you have colonies that persist at very complex levels of comple uh     
1:24:28     
significant levels of complexity over time and so different sequences random     
1:24:35     
mutations in other words different ways of changing these states give different sequences of rules     
1:24:43     
but the remarkable fact is that in almost all cases it's possible to make progress and rely reach rules that give     
1:24:49     
Long Live patterns for lifetimes 107 162 and 723     
1:24:55     
so this is a you know these can persist for quite a while this can persist up to 700 Generations or more with a laborate     
1:25:02     
morphological structure so here are some examples where you get these solomous     
1:25:07     
structures that persist for very long time periods this is in this case over 700     
1:25:14     
Generations so this is an interesting thing I think this is another interesting thing about this is that     
1:25:21     
they talk about making progress and then there's this whole sort of controversial idea in evolution or progress in     
1:25:29     
evolution and it's tempting to think of progress is like you know something that     
1:25:35     
is you know intentional but really it's just the product of different certain     
1:25:41     
conditions so you know in life we tend to see more complex Life as we move get     
1:25:49     
you know forward in geologic time so in you know the early years of Earth's     
1:25:56     
geology we had the early years of Earth history we had very simple     
1:26:01     
organisms and those organisms became progressively more complex and now we have a diverse Tree     
1:26:08     
of Life with incredibly complex life and so there's this uh I think Stephen J gou     
1:26:15     
termed it the wall of complexity where you you know start out at a minimal State and you evolve through these very     
1:26:22     
complex phenotypes indeed you can see that in the data but you know is this really a the imperative     
1:26:29     
of some sort of progress or is it just based on the conditions that exist and     
1:26:35     
so this is arguing sort of that those conditions have to be there to make     
1:26:41     
progress and he puts it in quotes and I would definitely put it in quotes because it's not something that we want to confuse with what's actually going on     
1:26:50     
which is that we give uh sort of this self Collective the right sets of conditions we're modeling     
1:26:57     
this a cellular aoma which in in of itself I think is a little bit abstract for you know what we want to do but it     
1:27:04     
gives us interesting results and that's what's important looking at the sequence of     
1:27:09     
pictures above we see that they're often in effect different mechanisms for producing long lifetimes that emerge in     
1:27:16     
different sequences of pools so we can produce long lifetimes we can have different rules for getting     
1:27:21     
there different ways of getting there so we have different mechanisms for building complexity and in fact that's     
1:27:27     
also what we see in the tree of life is that complexity can happen in a wide variety of branches of the tree of life     
1:27:35     
but let's drill down and look a little bit in a little more detail what's happening in the simple model of Music     
1:27:40     
in the three color nearest neighbor K = 3 Ral 1 C autom there are 26 relevant cases in the     
1:27:49     
rule point mutations affect a single case changing     
1:27:54     
changing it to one up two possible alternative outcomes so that Al together there are 52 possible distinct point     
1:28:01     
mutations that can be made to a given rule so again these are just point mutations a changing a single     
1:28:08     
cell and then when you change a single cell and you keep evolving the system and changing another cell you end up     
1:28:15     
with an explosion combinatorial explosion of possible point mutations     
1:28:21     
and Poss and diversity of behavior outcomes so we see that from this seed     
1:28:29     
we can mutate in different ways at the beginning of it and we can end up with all these different     
1:28:35     
behaviors in quite a few cases the pattern generated is exactly the same as the one for the original rule other     
1:28:41     
cases it dies out more quickly or it doesn't die out at all     
1:28:47     
so we're working on this approach using cellular automa called the emca approach     
1:28:54     
that's using these one-dimensional automa and we're finding some similar results and I find this interesting     
1:29:00     
because not only because he's doing some experiments of minimal s of cell     
1:29:05     
cellular systems but because we've kind of seen similar results uh in some of the working so     
1:29:12     
this is an interesting article um so he goes through a lot of this and then he     
1:29:18     
talks a little bit about sort of this uh Dimension reduced rendering of the 27     
1:29:23     
dimensional rule space so he sets these rules in a rule space and builds this network of rules and you know basically     
1:29:33     
the idea is your phenotype will sojer through these different through this space through these different conditions     
1:29:41     
and there are periods when there's lots of wandering around going on with many mutations needed to make progress there     
1:29:47     
are other periods and things go much faster fewer mutations are needed so we can see from this uh plot     
1:29:55     
here that you know we have this number of I guess Generations we can plot the maximum life     
1:30:01     
type a cheap so far against the total number of mutational steps made so obviously the longer something persists     
1:30:07     
the more mutations it occurs and you know if it incurs mutations and doesn't survive then it's     
1:30:14     
dies and so sometimes mutations can be good and sometimes mutations can be aerious it just depends on if we pick     
1:30:21     
the right ones we see plateaus including an extremely long one which is here     
1:30:27     
where no progress is made and I guess progress meaning persistence through mutation and I don't really know if     
1:30:34     
that's progress I guess you know we you know I guess the the mutation sort of     
1:30:40     
allow the common to persist and they build upon one another so in this case if we have onedimensional     
1:30:45     
array where we mutate some site it has a     
1:30:50     
downstream effect and so those in that sense these mutations can be     
1:30:56     
cumulative and the memory of that isn't in a genome but it's kind of read out in the cellular     
1:31:01     
autometer so it sort of unlocks certain uh Pathways certain patterns and then     
1:31:08     
allows those to persist so this is an interesting paper     
1:31:13     
and then this is where he kind of gets at some of these uh so so far we mostly been     
1:31:20     
talking about one particular random sequence of mutations but what happens if we look at many possible such     
1:31:26     
sequences here's how the longest lifetime or in effect Fitness increases for 100 different sequences and random     
1:31:33     
mutations and so this is the thing here where we have go out to certain number of time     
1:31:40     
steps and we get these mutations so we get this is mutations on the y- axis     
1:31:45     
this is time on the x-axis and so here we're kind of     
1:31:51     
exploring the space and exploring where we're getting uh sort of these adaptive     
1:31:57     
processes that emerge where the population doesn't get stuck it it allows it to     
1:32:02     
persist and then he talks about multi-way graphs which are these graphs     
1:32:09     
here that show all possible mutation histories and how they're connected to one another so a couple other really     
1:32:16     
interesting things from this article it's very long so I'm not going to get into all the details but he talks about     
1:32:22     
multi-way graphs and how that sort of shows the connections between different     
1:32:28     
types of rules and things like that then he makes the link of course the fitness Landscapes so a lot of this work is very     
1:32:35     
reminiscent of what we know from artificial life research and so makes sense to look at the fitness landscape     
1:32:41     
to bring that into play so he does some work on looking at Fitness Landscapes     
1:32:48     
and what he calls Fitness way points which are ways decisions or decision     
1:32:53     
points within the fitness landscape that allow population to move in One Direction or another sort of a I don't     
1:33:02     
know if you want to call it a decision point but it's definitely a you know a point of     
1:33:08     
contingency in other words if I I if this is the rule or this is the mutation     
1:33:13     
that's made the population will r one way versus another and then you can reconstruct that and show that sort of     
1:33:20     
contingency as the product of a series of contingencies as sort of     
1:33:27     
produ of evolution and so um we can also think of this in terms of     
1:33:34     
exhaustive versus adaptive search so exhaustive search would be where the space was searched uh randomly regr     
1:33:42     
force and we can bring in ideas about adaptive Evolution and adaptive search     
1:33:48     
to the space and so he talks about this a little bit     
1:33:59     
then he gets into some of the issues in computation so he talks uh the first     
1:34:04     
thing he talks about is this issue of undecidability um so this is where we talked about these Fitness Criterion and     
1:34:11     
how patterns either terminate or don't terminate and then we you know he ask     
1:34:18     
the question but how can we tell if a pattern is going to terminate so sometimes p patterns live for you know     
1:34:25     
variable amounts of time that they eventually terminate and you know we know that like     
1:34:31     
maybe I don't know if he tried it but you know can you evolve uh S L automa     
1:34:36     
for more than 700 Generations in this manner so you know there's a lifetime uh     
1:34:42     
to this process you know in the tree of life there's also sometimes a timeline     
1:34:48     
the lineages certain lineages die off and you know I don't think that's been     
1:34:53     
very well studied but um in any case um talks about this and of course     
1:34:59     
undecidability links into turning machines and the ability to Halt and so     
1:35:05     
he also gets into that a little bit later     
1:35:17     
on so you can actually boil this down to an even more metal and so you know again we're     
1:35:24     
thinking about computational rules and the sort of computational universality meaning that     
1:35:30     
I can take this and ask questions about other systems so if I find these     
1:35:36     
Dynamics in this minimal model of life it shouldn't inform things like machine learning um but in this case we want an     
1:35:43     
even more minimal model to boil down what we're finding in the model to something else and so uh this is     
1:35:49     
something he built called the mutation graph so it gives us a link between these mutations as they were introduced     
1:35:57     
and the fitnesses and it gives us sort of this ability to extract the set of     
1:36:02     
rules that allow us to find the best manner of sort of adaptive     
1:36:20     
Evolution so you know we might go back and ask like what does this mean for biology not in the minimal sense but in     
1:36:28     
the broad broader sense and so how do biological organisms manage to be the way they are with all their complex and     
1:36:35     
seemingly clever solution to such a wide range of challenges so are we just doing     
1:36:40     
the natural selection or is there something else going on and so this is     
1:36:45     
where he gets into biology and how you know it's not about sort of the     
1:36:50     
cognitive decision making but it's just kind of set of random processes and so     
1:36:56     
you know we have to think about not only these random processes but the way he views it is that there are these     
1:37:02     
randomly chosen programs and they have a behavior and those behaviors are really the basis of what we're talking about so     
1:37:09     
you know he thinks very much in terms of programs and their behavior so this is maybe a point a point of contention for     
1:37:16     
biology is biologists don't necessarily think of things as programs would think     
1:37:23     
about like like you know gene expression programs but those are sort of analogies     
1:37:29     
and so you know there there's there's something here that sort of an assumption about bi the underlying     
1:37:34     
biology but it's interesting um     
1:38:18     
so one thing he talks about here he talks a little bit about the complexity community and how you know people have     
1:38:25     
been using genetic algorithms to try to solve different cellular automa rules or     
1:38:31     
you know evolving different types of optimization procedures and so this this     
1:38:37     
has a lot of links to evolutionary computation as well as biology so it may be you know that we're looking at a     
1:38:43     
minimal model of evolutionary computation more than biology he also     
1:38:48     
brings up uh Greg chayton's work on metabiology uh Greg chayton published a     
1:38:53     
book I think it was in 2009 in metabiology where he wrote sort of a     
1:38:58     
proof of arnian evolution and so it's kind of an interesting book uh I     
1:39:04     
remember reviewing it in a blog post at the time and so it was really you know     
1:39:10     
it was very reminiscent of this work but also some of the work they do in Avita     
1:39:16     
which is a artificial life platform basically taking Evolution and boiling it down to some of these Concepts that     
1:39:23     
we see here and we see these discrete States we see natural selection and we     
1:39:29     
see evolutionary Dynamics and so it's a really interesting kind of linkage there     
1:39:34     
as well and of course FR Chon's done a lot of things with algorithmic information Theory and     
1:39:41     
so and so uh that's all this is a very long article if you're interested it's     
1:39:46     
on St from this blog
