     
Transcript     
0:01     
hello hi Hi how are you I'm on this on the compell 3D course     
0:08     
and that's what I did last week decides buy a new camera yeah     
0:15     
yeah well that's great uh how was the course um very     
0:20     
intense I don't know if it's very useful to me because there's not enough     
0:27     
um link with fores exact forces     
0:32     
and distances um it it does do uh code and     
0:38     
show cell Behavior it does that fairly well     
0:46     
but if you have to do something that's physical much     
0:54     
yeah so yeah I I could show it to you actually there their     
1:00     
Boolean um their Boolean part of the course would work for your most recent     
1:08     
paper you could take your paper and you could do uh the Boolean part of comp cell 3D     
1:17     
on your paper and it would make a very interesting uh addition I don't know     
1:23     
yeah I mean you have to think about like you know some of the things because this paper is so big and it has like a lot of     
1:30     
things that are sort of there and they could be followed up on so have to think about     
1:38     
that anyway comp yourself 3D is accessible and um I have a bunch of intro videos     
1:47     
there on YouTube okay yeah let's see I I'll I'll I     
1:54     
actually have the the web page it's very unapologetically 1990 design here but uh     
2:02     
this yeah this is the summer school I guess they're having a hackathon this     
2:07     
weekend and so they have like the uh the different links here so you have the     
2:14     
YouTube videos I guess from 2023 the'll probably post for 2024 they yes that's um I'm involved in     
2:23     
that because I proposed fix model where you have a line of Sals and if perturb     
2:31     
the first one um and um it turns into a column or cell how     
2:39     
does that affect its neighbor or neighbors yeah and how does that go down     
2:44     
the line so I've I've got a model for that     
2:50     
they've um looked at Suite division with the Pell 3D and I said had they     
2:58     
considered rally and stability like I really wanted to know where they had     
3:03     
Incorporated that and they knew about rally instability so I I relieved but I     
3:09     
still don't know how they managed to incorporate that into the model or if they     
3:16     
did yeah I'm out of sync so I G show off my picture because     
3:23     
I'm out of sink okay all right yeah so this is uh yeah     
3:31     
this kind of goes over some of the details of the uh what was covered in     
3:36     
the course so Monday through I guess Sunday that seems like a lot     
3:43     
yes it was I went to their um python     
3:48     
boot camp on Sunday so I went for eight days and I would basically exhausted     
3:55     
yeah I imagine yeah I hard to think yeah yeah so yeah so     
4:06     
python boot camp was before the course so you get a crash course in     
4:12     
Python yes it's very very SE like     
4:18     
um like I've certainly seen that before and if I uh wanted to program in Python     
4:25     
I should just kind of take a course in it and then I'd know all of the syntax foibles right for     
4:34     
it then i' be able to do it anyway the whole thing's exhausting     
4:43     
yeah yeah looks like there is a lot of stuff here you had the introduction to the platform virtual tissue modeling     
4:51     
introduction uh Basics and intro to Cellular pods modeling which of course is is kind of like a cellular autom but     
4:58     
it's it's a it's a physics model but it doesn't model mechanics     
5:03     
necessarily um and then the next day you had fundamentals of the modeling process     
5:09     
soell properties and constraints cell growth and division then chemical     
5:15     
gradients on Wednesday uh working with python     
5:20     
API uh building a tumor model and forces on cells     
5:27     
chemotaxis uh cell layouts so they're kind of working up with Concepts working up with uh different ways to implement     
5:34     
stuff in the platform I see Boolean Network modeling     
5:39     
yeah that was on Friday so yeah yeah Boolean networks are are interesting and     
5:45     
useful um something we don't haven't like explored a lot in the group but it's it's probably is useful for     
5:53     
especially like Gene regulatory Network modeling and things like that that looked very interesting Boolean Network     
6:01     
for especially like I said for your paper yeah     
6:07     
yeah so uh then there was of course uh compartmental     
6:13     
cells uh focal point plasticity data wrangling best practices     
6:19     
for parameterization so I guess these modules were different from these modules then Vector Fields planner cell     
6:28     
polarity issue folding and then they were prepping for the hackathon which of     
6:34     
course is probably just implementing some of these things     
6:40     
more I don't know if it's possible for me to do anything um I later on if you     
6:47     
want me to present what I did for a lightning talk I could do that okay that's that sounds good do you like you     
6:54     
want to do that later or later yeah all right maybe noise will have calmed down     
7:01     
never know okay it's a zoo here four dogs yeah     
7:08     
four hello all right well I'll I'll come back to you later we'll see     
7:16     
okay all right uh so thank you Susan for kind of talking about that and going     
7:23     
over that with us um yeah that looks like it was a full week and of course     
7:28     
you can catch up on YouTube and there are a lot of Concepts that you know we could find useful for our     
7:34     
work uh paper that we've recently put out on differentiation waves you know     
7:41     
there are a lot of other new other approaches we could use uh Boolean networks being one but there are other     
7:47     
things that we'd like to do going forward I mean last figure in the paper     
7:52     
is kind of this features this model of the dropa imaginal disc and it has like this     
8:00     
wave going across it and being able to model that more explicitly with uh gene     
8:07     
expression and with uh the morphological changes would be kind of interesting I     
8:13     
don't know how much of that you could model and copy s 3D the way we would like to model it but that would be     
8:19     
something uh to talk about for another paper s so um hello Jesse and Pocky how are     
8:28     
you hello yeah I'm I'm much I I I finally got     
8:36     
uh some good sleep and I'm ready to push forward on some things again that's a very difficult weekend to be honest uh     
8:44     
but I'm looking forward to things here I don't think Lucas is here uh I was going     
8:50     
to ask a little more about some of that Lucas project and see if for ways that I     
8:55     
can also help with um uh     
9:01     
Le using Dro to facilitate some Diva worm stuff too or like help advertise     
9:07     
things or if you want to do stuff or anything I know we had this conversation with uh Orel a big conversation for Orel     
9:15     
in the future and fall term and everything last last week U but anything     
9:21     
I can do to sort of facilitate that for um Diva specific projects too um     
9:30     
in hearing more about that orating what I can it's my brief update for today     
9:37     
yeah yeah I think that uh yeah Lucas's project is he gave an update in our     
9:43     
Saturday meeting because he's been having trouble finding time to update us on things but he's coming along on some     
9:50     
of the work that he's been doing so he he did uh some bioinformatics work for     
9:56     
uh the paper that we just published and he's doing the student ship through openworm to kind of make that more sort     
10:05     
of you know build a pipeline make that more computationally explicit he you     
10:11     
know got a lot of work kind of in the way that a lot of people will do work um     
10:16     
you know they'll like take they'll have their uh resources in different places     
10:21     
they'll kind of you know uh do very informal things steps in their analysis     
10:28     
and the idea is to build a reproducible pipeline using um a Jupiter note or you     
10:35     
know some sort of computational notebook or you know some API tools and building     
10:41     
you know a a set of commands and building a GitHub repository around them     
10:48     
so you can just go to the GitHub repository you know follow the readme go to the notebooks use those and be able     
10:55     
to execute a similar analysis and then of course fit that into to models of an     
11:02     
embryo which we have in part and you know it's it makes it it augments that     
11:08     
model quite a bit so yeah I think that's that's good and     
11:14     
then as as far as like um so he's making progress and I you know I said you know     
11:19     
we'll have to figure out how much is sort of the right chunk for a uh student     
11:25     
ship project because you can't do everything during a student ship but you can do like you know get to a point     
11:32     
where it's something you can showcase and then that's something you know also we can integrate into the things we're     
11:37     
doing so it's nice to have those kind of     
11:44     
opportunities um is for advertising Diva worm stuff yeah we haven't done that in a while uh we talked about this also     
11:51     
Saturday and uh you know there are a lot of ways you can advertise different     
11:57     
opportunities um and so let's I'm going to work on that um maybe you know we have we advertise     
12:07     
specific projects and then we have the Google summer code projects and some of these other student ship     
12:14     
opportunities but uh also you know other projects that are kind of more abund     
12:19     
right now and I'd like to see them kind of when when I'm done um which     
12:27     
hopefully will be through this week uh I'll show what I've done with jopo that     
12:34     
part of what I want Joo to be is is sort of a     
12:39     
um not quite like a jobs board or a quest board but something that     
12:46     
can uh offer a space to see     
12:52     
certain like up like almost like almost more like a     
12:59     
like uh showing projects that are being launched uh because I think that's     
13:05     
that's a key thing both in this space and in orl and in other other places too     
13:11     
like I know a lot of like smaller startup groups that are trying to     
13:16     
just um explain you know like so much of this     
13:22     
so much of the barriers just this explanatory or like     
13:29     
putting things in a way that's easy to understand and understand okay if I do this I can do that and that's that's     
13:34     
that's it's it's it's a challenging thing to do so um I think I think there     
13:40     
are some ways that um like it could be beneficial or at     
13:45     
least at least if not beneficial like     
13:50     
uh not you know like indirectly helpful too to have it uh so I more about that     
13:57     
ahead but I'm definitely and and yes um having the the history page which I'm     
14:03     
updating slowly the actual like history will somewhere go but then say okay here's Diva worm here's like some maybe     
14:11     
past projects or existing and forthcoming projects and and and and giving it a little more     
14:18     
um of a different kind of structure because I notice a lot of a     
14:24     
continuation there's sort of the existing de website     
14:30     
and kind of kind of what I tried to do for a while with that GitHub the GitHub thing forl was like give a framework to     
14:37     
what was going on and I think i' I've sort of improved upon that now there's a sense of okay um across up many     
14:46     
different fronts here's like different working groups and what they're different groups of what they're doing     
14:51     
and sort of project associated with that and I think that would be helpful too so I'm more about that hopefully hopefully     
14:59     
I can I can finally release this um through the end of this week uh but     
15:06     
I'm happy to talk more about that ahead yeah yeah that's good uh maybe we need     
15:12     
to have a reorganization a bit of things like in terms of advertising them     
15:18     
getting them out we haven't done that in a while we've done things like buil uh onboarding guides and other types of     
15:25     
things uh you know we've had the 10th anniversary uh     
15:30     
reassessment or to see where we've been so we have that stuff kind of out there     
15:36     
and it's just yeah this actually there's like a tremendous amount of great material already um so I think somebody     
15:44     
just putting at us in in one space too uh but yeah I I look forward to that and     
15:50     
I'm looking forward to I'm starting I watched the I watched I have a sort of like     
15:57     
movie playlist that I'm trying to watch in like my evenings to Hype me out for     
16:02     
fall I'm actually really excited about like allo semester and just things coming up and and this is part of that     
16:09     
so yeah times yeah yeah that's great um     
16:14     
yeah we should we should talk more about that um and try to get some things yeah     
16:20     
out there so thank you Jesse uh Jesse asked Morgan when are you     
16:27     
in Boston this week I know Morgan is in I guess he's going to Boston or he's going to Rochester New York and Boston     
16:34     
uh so yeah Morgan's here and Pocky of course we talked uh on I think you gave     
16:41     
presentation at Friday's meeting on some of your progress on the uh on the work     
16:48     
you're doing for gck yeah and um so yeah that looked pretty good I think you're getting     
16:54     
things together um I can offer you an extension for your project maybe two or     
17:00     
three weeks I'll have to communicate with people at incf about that but it looks like you're making some pretty     
17:05     
good progress on your project so yeah um I know Pocky can't talk today     
17:13     
but like did you want to put anything in the chat and and mention anything about     
17:19     
your project today put it in the     
17:24     
chat looks like uh Morgan isn't sure where he's going to be be this week so     
17:30     
it's fine yeah so Jesse and Morgan might connect this week we'll see in person     
17:35     
it's always fun to get together in person uh I don't know if Susan is able     
17:42     
to present yet uh yes I could okay yeah go     
17:47     
ahead okay so this is um about     
17:52     
contraction um to produce a wave that we've been talking about like we talked about waves and tissue     
17:59     
and it's basically self contraction or expansion     
18:06     
um and if you do this with epithelial tissue it's sort of     
18:13     
two-dimensional not exactly but you're you're dealing with fewer fewer cells     
18:19     
deep or at least I'm currently working with a model that's only one cell de     
18:25     
deep and I know that's not true uh most tissues but anyways um you can actually     
18:32     
physically make a tissue that's one epithelial cell deep so you could test     
18:37     
it in reality anyway so here we are um they     
18:44     
wanted to know the biological background so the contraction of one cell leads to     
18:50     
the contraction of the cells next to it and these contractions occur in     
18:56     
epithelial tissue and um our thought to lead to cell differentiation if the the     
19:05     
cell contraction lasts long enough for the forces to get into the nucleus and     
19:11     
rearrange the cytoskeleton of the cell and rearrange the some of the     
19:18     
nucleus um and an epithelial cell has an interesting structure it is an     
19:23     
actomyosin ring around its apex so this is a     
19:31     
um an interest interesting structure leads to interesting mechanical     
19:38     
properties yeah anyways so     
19:43     
um so this is a side view and the attachments are all along the top here     
19:49     
and there's actually a tri Junction in between cells a tri Junction by Junction     
19:55     
quad Junction whatever is need     
20:01     
and what I'm trying to model is a cell contraction in this     
20:06     
cell will make the top narrower and cause it to be more colmer and this will     
20:15     
affect the next cell um to do something and then that'll     
20:21     
affect the next cell so this that's what I'm trying to     
20:27     
model and here's an actual real model of an     
20:33     
epithelium um with a contraction going across the     
20:38     
top of the tissue um it says modeling epithelial     
20:47     
tissues as active elastic sheets yeah produce contraction pulses     
20:54     
and predicts rip resistance it's kind of an inter casting paper so what are they     
21:01     
doing here is they're trying to uh model like the effects mechanical     
21:06     
effects across the sheet of cells and right right right and this is across the     
21:14     
sheet of cells and for Simplicity sake I'm just going to do a row of cells okay     
21:20     
yeah for for the modeling I mean hopefully get to this stage     
21:25     
but uh to start with just a row of     
21:33     
anyway so the problem motivation the hypothesis and cell     
21:38     
contractions uh that last long enough for forces to be sustained in     
21:45     
nucleus um for long enough to affect genetic expression I could have probably     
21:50     
stated that differently um and basically this is the link     
21:56     
between mechanical force and genetic expression     
22:03     
yeah so modeling goals [Music] um would like to model the force and see     
22:11     
where it goes um I need a quantitative mod modeling so     
22:17     
that uh we can track the force between one cell and another     
22:25     
um so it's a simulation uh that models what a group of     
22:31     
epithelial cells experiences when one of their members uh     
22:39     
contracts and the energy of uh this um     
22:45     
wave or contraction is uh thought to be a phase     
22:50     
transition you're actually changing uh the cytoskeleton of the     
22:56     
cells from um sort of cuboidal to colum     
23:01     
or cell and cells have a refractory period     
23:07     
where they can contract for a while and then uh they can't do anything then they     
23:12     
relax and um can't do anything for a while because they've used up their     
23:19     
chemical     
23:24     
reserves and here is the um     
23:30     
cc3d simulation that they've already done of an     
23:35     
epithelial um string of cells so they figure they've already got this as a     
23:44     
model uh so this is the ECM or basement membrane in Gray and then the um bottom     
23:53     
of the cells is in blue the middle part of the cells is in green and the apical part part of the themselves is in     
24:00     
Red so they make it in three parts as three compartments     
24:07     
actually so I was a bit disappointed about that but it's true that the epical     
24:12     
part of epithelial cells is totally different from the rest of the cell yeah so the three compartments     
24:20     
there are what EP or the apical the basil and the what yeah this is so this     
24:27     
the basement membrane is in Gray and the basil part is in blue where it's     
24:34     
attached to the basement membrane green is the middle part of the cell and the     
24:40     
red is the apical part of the cell so you attach the apical U part to to each     
24:48     
other like that would be connected by a force and they've actually got that in a     
24:55     
model where you're disc connecting the apical of part of the cells to each     
25:02     
other okay and the rest are well the the blue the basement membrane to cell     
25:09     
that's connected to each other but they're not necessarily     
25:15     
connected across okay so they're connected uh vertically     
25:22     
and not horizontally or um the blue is connected ver     
25:28     
vertically okay and the red is connected     
25:33     
horizontally okay I see yeah and the green not at all that's     
25:40     
just the middle part okay     
25:47     
um so what do we need we need epithelial cells on a substrate you need a calcium     
25:54     
buildup and the actomyosin buildup and then you need to add ATP of course     
26:04     
um so the cells will cramp um on the at the top to make the     
26:14     
area of the top of the cell smaller and make the cells into a colmer type cell     
26:22     
and this should cause a Cascade of uh cell cramping in the epithelial     
26:30     
layer so to model this and model the epithelial layer one cell deep uh Place     
26:37     
attachment forces between the tops of the cell where the acting ring is and model a cell     
26:44     
crft and this is what I gave for resources oh yeah yeah that's good yeah     
26:52     
so so especially the first one that's so that people can see an explanation     
26:59     
um and get it for free and and the same with the last one here I said this is     
27:04     
more complicated the last one uh reference is more complicated than is     
27:09     
needed but the introduction um has lots of resources about this the waves the pramp pramp     
27:17     
waves that go through epithelial tissue and I actually had someone ask     
27:24     
me so L is that true of all epithelium tissue and I said yes it it's nonmuscle     
27:33     
mein and it's ass associated with actin and you add calcium and ATP and     
27:42     
you get a cramp it's similar mechanism to what happens in muscle but it it does it does     
27:50     
move right and they didn't realize that and she was uh very into genetic so I'm     
27:57     
glad I was able to point that out to her right oops Yeah myosin is just actually     
28:04     
just in muscle so like that's the thing that's responsible for the movement muscle or the right so it just works     
28:13     
it's a ratcheting action yeah and very very similar to what happens in     
28:20     
epithelial cells and it's non muscle mein but the ratcheting action of the     
28:27     
actin and meas is very similar associated with calcium and     
28:35     
ATP so anyway that that's my my um slides well any     
28:42     
questions yeah that's great uh did we have any questions from anyone     
28:49     
or so yeah if you could go back to slide seven I just wanted to make a point about that so that looks like a nice     
28:56     
like array like it's a two- dimensional array kind of cut a cross-section so you said that the red parts were connected     
29:03     
vertically and the blue or the red parts were connected horizontally across that     
29:09     
and then the blue ones were connected vertically up up and down from the bottom to the top yes so that's     
29:16     
interesting like from a network perspective how you could model this as like you know a model a network of     
29:22     
forces and how they're kind of not all going in the same direction     
29:28     
within that cell sheet so you have like you know forces kind of from the substrate forces from     
29:35     
the neighbor cells and what do how do those kind of work in a     
29:41     
network yeah I've been interested in working with with that actually I hadn't     
29:49     
thought of that but that's the way my Integrity     
29:55     
models are yeah they're very senstive itive to their attachments both the     
30:00     
basal and the side attachments and uh the pre-     
30:07     
string that seems to be what what they're sensitive to that's     
30:14     
what they need or something that's um that what that's what changes the     
30:19     
graphs right yeah yeah     
30:28     
yeah that's good um yeah thanks okay so this is something that     
30:34     
you could model in copy cell 3D I mean you could do it but they don't have you     
30:40     
said that they don't have like a a good model of like mechanics included in the     
30:47     
platform it's sort of there they he did say well that's almost directly related     
30:53     
to um the forces involved in cells and you can oh I have to get out     
31:01     
my notes to see exactly what he said but yeah some of it is directly rated to the     
31:07     
forces involved yeah but it's not like I said how many newtons is that like give     
31:14     
me hard facts Newtons how many newtons and they said well it's sort     
31:21     
of the forces and what yeah well I can't use this yeah     
31:29     
doesn't have like a in the force and Newtons and then yeah I know what you     
31:34     
mean yeah if it's not got um well I think you can do size um properly like     
31:43     
micrometers right but whether you can do Nano check with it or not meaning when     
31:51     
you get small enough um surface area forces matter more um oh okay I'm I'm     
32:02     
not thinking of it again um and I'm not going to right at the     
32:09     
moment um I have a interference here with the     
32:15     
dog oops the dog is helping okay uh so     
32:28     
there are effects at the Nano scale that are not at um our normal micr scale I took a     
32:39     
nano te course right hes that are stronger at that Nano scale     
32:48     
and they affect cells so you have to keep that in mind too going small you have to watch out     
32:55     
for that so I don't know if the compucel 3D has done that but I'm sure you could     
33:03     
just you could put it in your model like it's fairly flexible model and he says it's great to try things out with and     
33:11     
see if it's trending in the right direction and then you can think to go to uh their tissue Forge which they've     
33:18     
just developed um and do a more detailed model with more forces and more properly     
33:25     
modeled right but it's really great to try things out with that's what he     
33:31     
said yeah we covered you can do chemical Fields with compel     
33:38     
3D okay which is nice that's why I said oh there has to Beal a calcium     
33:44     
field and you can just put that directly into your     
33:52     
model yeah that's uh that's good yeah and of course we have these Nano scale     
33:59     
effects when you're at a nano scale That You Don't See in at the micro scale and     
34:05     
if you're in that regime then you have to model it it's a basically a different     
34:10     
set of things that you're modeling so you have if you're going to have like something that interacts between the     
34:16     
Nano scale and the micro scale you have to account for both types of effects and yeah     
34:22     
yeah yeah and then we talked about tissue Forge uh a while back and that's     
34:28     
like their new sort of where they're I guess they're doing multimodeling or     
34:34     
multi- scale modeling they're basically bringing together a bunch of different types of     
34:39     
models yeah um I if you want to hear about tissue     
34:46     
Forge they have the presentation about it um on YouTube that goes with this     
34:53     
conference they had um there's a a medical doctor who very involved with tissue     
34:59     
Forge and I don't know that I can get involved myself with my model because     
35:05     
I'm out of time yeah like I should probably just go with what I have write     
35:10     
it up and say here you go and then um and then mess around with tissue     
35:17     
Forge yeah I mean probably a whole new frontier so yeah it might take a while     
35:24     
to learn it but they were very pleased that I was interested in it so good     
35:32     
yeah yeah so yeah I've been trying to kind of figure out a way to like incorporate this sort of copy C     
35:40     
3D uh you know world into what we're doing and it's been hard because it's a     
35:45     
hard it's a hard learning curve as I understand steep learning curve for applying it to a specific problem and     
35:52     
then you know we have to have the specific problem that we want to solve with it so um and then as you're showing     
35:59     
it's not as straightforward as just taking a problem and sticking it in there yeah you need to     
36:06     
know uh what you've modeled and the fact that it's a model     
36:12     
and not reality and you've made these     
36:18     
shortcuts um one of them is that you're made the Affinity to each part of the     
36:24     
cell so that it'll stick together and then you have to make make it so it's     
36:30     
lesser between cells so that they don't mix and you can't put a strong force on     
36:40     
on the top I don't know I I should I'll try and do that model with them so I can     
36:48     
get a better handle on what's going on with compell 3D so that I can maybe     
36:54     
write a bit about it in my thesis [Music]     
36:59     
and that would be good enough I think it'd be good enough if I     
37:07     
had well I I have already written something about it but I might as well do     
37:13     
something um more comprehensive in a paragraph or two yeah you can see my     
37:19     
family in the background there my son at the door yeah anyway     
37:31     
all kinds of interesting things the four dogs and yes anyway so is good enough     
37:39     
yeah that's great yeah thanks for sharing uh it looks good and promising okay all     
37:46     
right all right all right so yeah thank you Susan     
37:53     
for that update on how we sell 3D and you know how to use that how to implement something in it or you know     
38:00     
how to pose a problem so that we can use some you know existing modeling platforms and kind of     
38:06     
fit that together so it's always a problem with modeling is you know you have like some problem you know you make     
38:13     
a pro you you have a hypothesis or you have a problem you want to put into a model and you have to figure out what     
38:21     
modeling Paradigm you want to use so you know we do a lot of stuff with machine learning and of course you have to train     
38:28     
those models you have to not violate the Assumption of those assumptions of those     
38:33     
models um and so you know we've been doing this at picky's project too where you have to fit the problem that you     
38:40     
want to solve into the model and and given its constraints and its assumptions how do you get an output     
38:47     
that tells you something so that's that's you know that's that's a big     
38:52     
that's the biggest challenge of modeling is to get you know pick the right type of     
38:58     
model maybe you don't have everything you want so you know maybe you have to add things in but that's you know that's     
39:05     
kind of hard to do in open source models we can theoretically add things in but it takes a lot of     
39:11     
overhead and then get an output that's you know that's reasonable it can tell us something about the system we're     
39:18     
interested in so that's that's how you you know approach modeling and it's not     
39:23     
not a perfect um exercise but it it should lend you know if you're successful it     
39:29     
should lend enough insights so that you can say something     
39:35     
interesting so okay so thank you for those uh     
39:40     
updates and presentations so yeah I'm going to talk about a paper here for the last maybe 15 minutes and it's a new     
39:47     
paper that's come out on cgans this is again kind of in the direction of     
39:52     
molecular biology and C elegans is a good model for molecular biology ology     
39:57     
because we have a full genome we have we know kind of the identity of every cell     
40:06     
and we know a lot about the organs we know a lot about the defined mutants and so forth and so it may seem like seans     
40:13     
is a really good system to build molecular models so we you know maybe know what certain genes do if you knock     
40:20     
them out if you mutate them and we know a lot of the gene regulatory networks     
40:25     
for certain things we have have the genes defined and so forth but of course     
40:31     
there's a lot of variation there and people have studied uh you know the sort     
40:37     
of the epigenomics of cgans extensively although there's still a lot more to be     
40:43     
learned and the epigenetic effects in celegans aren't as great as say in mammals but we still have those     
40:51     
so um there's this new paper that came out I think it's I think it's in it     
40:58     
bioarchive paper um this is called alternative splicing across the seelan     
41:04     
nervous system so as you're aware you know cgans has this complete     
41:10     
connectome and we know what the neural cells are and we know what the neural circuits do so we can identify things     
41:17     
like a movement circuit and we know kind of how that's activated during forward     
41:23     
and backward movements we have some you know we know something about the sensory motor Loops within the connecto we know     
41:30     
something about like uh things like chemos sensation and other types of uh     
41:35     
the effects of other types of stimulant but we you know we also have these things that are going on with     
41:41     
regulating the nervous system at the genetic level and so they're exploring alternative     
41:47     
splicing and you'll find that in seans people produce these papers that kind of     
41:52     
are this genome wide or this organism wide sort of attempt to character     
41:58     
something that we know you know like a concept from biology and we're applying     
42:03     
it in the celegans that's what they're doing here and they're applying this concept of alternative     
42:10     
splicing so let me go to the abstract to kind of explain what alternative     
42:15     
splicing is so alternative splicing is a key mechanism that shapes neural neuronal transcriptomes so what neur     
42:23     
What alternative splicing is is when you have a gene and a gene will often have     
42:28     
coding regions or open reading frames and those open reading frames will allow     
42:34     
a transcriptional Machinery to read certain parts of the Gene and so when     
42:40     
you get your you take your coding region of the Gene and you express it you get     
42:45     
this these transcripts that that sort of get produced as mRNA and those mrnas then can be spliced     
42:53     
in different ways so you have different ways that uh     
42:58     
mrnas can be spliced and that allows you to take a single gene or a single coding region     
43:04     
and produce multiple isoforms of that Gene so you know what you're basically     
43:10     
doing is in the genome you have these splicing sites identified they'll you     
43:16     
know different parts of the coating region will be cut out and then you'll have these different splicings of the     
43:22     
same gene or of the same coding region and so these different     
43:28     
isoforms uh have functional consequences often times different isoforms will be     
43:34     
used in different sort of regulatory uh settings you know     
43:39     
sometimes you know any one gene can be expressed and depending on the different isoforms that exist you know it can be     
43:47     
sort of regulated in a contextual Manner and this is especially important in the nervous system because you're doing a     
43:53     
lot of things in the nervous system that require the sort of grading expression when you need to have this     
43:59     
sort of graded expression of things uh for function so like if you're moving     
44:05     
forward versus moving backward you know maybe alternative splicings are useful there because it would be much less uh     
44:13     
useful to have multiple genes that are basically duplicates but     
44:18     
doing slightly different things it's more efficient to have one gene for that or you know having like duplicate genes     
44:24     
but that basically uh don't change that much across Devolution but can be     
44:30     
spliced in different ways and recombined in different ways so that's basically     
44:35     
what alternative splicings are and so that's why they're important um so these     
44:41     
alternative splicings help to Define neuronal identity in modulate function so you can also have alternative     
44:47     
splicings being involved in uh cell identity so when you have a certain cell     
44:52     
type you know that those alternative splicings might be different than in a     
44:58     
in a related cell type that has some different function so we can modulate function we     
45:03     
can define neural neuronal identity and those alternative splicings are     
45:09     
important here we present ancing what was that how do the     
45:14     
splicings come about again uh how do the splicings come about yeah usually if you     
45:20     
have like a transcriptional event you'll have your uh your your coding region     
45:26     
that's trans described and then there will be some sort of post transational mechanism that will splice those uh     
45:35     
those uh different parts you know they're defined within the coding region     
45:41     
and they you know the they're splicing sites and they're identified posttranscriptionally and then those     
45:47     
mrnas are spliced in different ways so you might take out a region of the coat you might take a chunk out of the     
45:54     
coating region and combine two parts of the coding region that are maybe far     
45:59     
apart or you might uh take out like the beginning of the coding region and it just depends on the     
46:07     
context so there are these yeah these post transational mechanisms that do     
46:13     
this does that answer your question or yeah thank you okay um I should only     
46:21     
watch a YouTube video yeah yeah that'll give you more detail I'm just kind of glossing over it right now but     
46:28     
um so yeah here we present an atlas so we have an atlas of alternative     
46:33     
splicings across the nervous system of cens our analysis identifies novel     
46:38     
alternative splicings and key neuronal genes so you know you can identify key     
46:44     
genes and those have an large effect but then the splicings give it this sort of     
46:49     
range of function that you wouldn't get by having like duplicate genes that have     
46:54     
like distinct functions themselves so that's basically it so un 40 DCC and sax     
47:00     
3 Robo are these some of these key neuronal genes that they're interested in um and not all genes have the same uh     
47:09     
potential for alternative splicings some have more of a tendency for it than others and again this has to do with the     
47:16     
functional imperatives of those of those uh genes uh globally we delineate patterns     
47:22     
of differential alternative splicing in almost 2,000 genes so you know you can     
47:27     
look across you know your Canada genes or you can look across the genome and     
47:33     
you look across the genome you see that there are these patterns across 2,000 genes so alternative splicings are     
47:39     
ubiquitous but only you know some genes are more apt to having a lot of isoforms     
47:45     
and others an estimate that a quarter of neuronal genes undergo differential splicing so this is where you have this     
47:52     
distinct functional um you know uh imparative for splicings or where they serve specific     
48:01     
functions we introduce a web interface to examine splicing patterns we explore     
48:07     
the relationship between neuron type and splicing patterns between splicing patterns and differential gene     
48:13     
expression so again you can look at different neuronal types and look at how splicing patterns exist uh within the     
48:21     
same gene or you can look you know at gene expression that's you know differential gene expression look at the     
48:27     
spacing patterns that result from that differential gene expression um and I guess one I guess     
48:34     
it's just because you know you have this in different neural types you have different things going on in a     
48:40     
regulatory sense and when you have differential gene expression you'll see that sort of turned on and off so it's     
48:47     
just kind of a difference in terms of you know what what the whether it's     
48:52     
maybe important for NE neuronal type or important for just for function or     
48:57     
whatever we identify RNA features that correlate with differential alternative     
49:03     
splicing and describe the enrichment of microexons which exons of course are     
49:09     
these regions you know you have introns and exons and so this is where you get     
49:15     
this where these become important um finally we compute a splicing regulatory Network that can be     
49:22     
used to generate hypotheses on the regulations and targets of alternative     
49:27     
splicings and neurons so they're working on the splicing regulatory Network so     
49:33     
there's a lot of content in this paper um and so this kind of goes over     
49:39     
differential alternative splicings as sort of their fundamental mechanism     
49:44     
elevating molecular diversity so the idea is to create a lot of molecular diversity from you know you don't want     
49:51     
to have an infinite number of genes um and there's people still are kind of     
49:56     
unraveling this idea of why genomes are a certain size right like why don't you     
50:02     
have a gene for every single thing in in the work and the answer of course is     
50:08     
because it would be you know very too large to fit into the to the uh nucleus     
50:15     
maybe they're also energetic imperatives we talked a couple weeks ago about um     
50:20     
cell division and how you know cell division takes as long as it takes to replicate the genome     
50:27     
so if you have a larger genome it takes longer to replicate the cell to to to replicate the DNA to have a cell     
50:34     
division event so your cell division event takes longer so you know we want to don't want to have an infinitely     
50:41     
sized genome and so this is you know one reason maybe why we have alternative     
50:47     
splicings because it gives us that diversity without having huge genomes that are you know take a long time to     
50:54     
replicate and to pass on to the daughter sub um and they kind of talk about     
51:00     
alternative spacings they occur when a premrna is processed in more than one     
51:05     
way so this is where uh different introns are removed so you have introns     
51:12     
and exons the introns are those spaces in between the exons the introns are     
51:19     
removed and the exons are left and so you you know as long as you have this     
51:24     
pattern of introns and axons kind of gives you an idea of how the altern the structure of the alternative splicings     
51:31     
in that coating region will uh you know it gives you sort of a combinatorial model of what's possible and then     
51:38     
depending on the situation different entrons are sped out the exons are left     
51:45     
and that's your alternative sping uh and then you get what you end     
51:52     
up with then are these mature mrnas with different sequences and those different sequences of course have an effect in     
51:59     
like building proteins and building um uh amino acids proteins and then     
52:06     
tissues um so you get different protein ISO forms as a result as can also affect     
52:13     
the stability of other features of the MRNA itself the vast majority of human genes undergo alternative splicing and     
52:21     
defects and alternative spicing have been linked to various diseases so you can have alternative splicings as this     
52:28     
sort of mechanism for diversity but you can also have defects in this process     
52:34     
that lead to diseases or other types of dysfunctional aspects of the cell and     
52:39     
the tissue so this is a process that's regulated spatially and temporarily so     
52:47     
you can have you know alternative spacings that vary across time you can have alternative splicings that VAR     
52:53     
across different cells so space so if you think about your tissue different     
52:58     
cells in the tissue might have different complements different sort of sto geometries of these     
53:05     
isoforms and so that may affect uh the phenotype you know you might also have     
53:10     
this over time so like if you have this sort of early in in the process of uh if     
53:17     
you if you say have like some movement and you have a forward movement a backward movement you know maybe     
53:23     
alternative splicings might play a role in uh you know switching from forward to     
53:29     
backward movement and of course we don't you know I don't know if there are any papers that specify that but this is     
53:34     
just an example of how this might be useful so in the nervous system where it's most prevalent and there's some     
53:41     
citations here differential as controls multiple aspects of neuron identity so     
53:46     
it controls identity and of course cell type identity is you know there's the     
53:52     
sort of the larger cell type identity like you might say something you know is a glutamatergic cell or it's a you know     
53:59     
some sort of inter neuron and then you also have sub sort of identities where a     
54:06     
subset say of inter neurons do something uh and so you know you can neuron     
54:13     
identity is can become very sort of specific depending on the function that     
54:19     
you're looking at so uh differential as controls multiple     
54:25     
aspects of neuronic identity including Global uh Alternatives placing switches     
54:30     
during development so there are different switches that occur uh and this is in citation five isopor     
54:37     
difference is a neuron type specification so depending on you know how this how different neurons are     
54:44     
specified or differences in isoforms that's citation six and seven Aon     
54:50     
specification guidance and synaptogenesis so the formation of synapses the formation of axons and how     
54:56     
they're guided to other neurons that's in citations 8 and N synaptogenesis is     
55:03     
particularly important in larvel celion but also you know with respect to neural     
55:09     
plasticity in terms of synaptogenesis you do have some plasticity um in the adult as     
55:17     
well and has been linked to several brain disorders and this is in 1 10 and 11 so this is uh stuff that's you know I     
55:25     
I don't know if this is specific spefic to C elegans but it does give you an idea of how alternative splicings     
55:32     
operate in the nervous system what they're responsible for uh alternative spicing have also     
55:39     
been studied in cancer inde needed some forms of cancer dependent on differential alternative splicings uh     
55:45     
because they're not cellp specific so you know you don't have like specific     
55:51     
alternative splicings for a cell type but you do have them in involved in different processes     
55:57     
they can be identified by methods such as sequence sequencing bulk cdna and so     
56:02     
this is a method where you can actually look at the cdna and get a sense of you     
56:09     
know where what these alternative splicings are so you know this it almost has the feel of like like almost like a     
56:16     
hammer trying to find a nail in a way like you have this mechanism and it's cool and you look for different things     
56:23     
that it's involved with and so this is why like we have all these different studies on these different things but uh     
56:30     
as to you know what specifically alternative the role specific roles of alternative splicing and where they have     
56:36     
the greatest effect that's still kind of an open question but that's why they're     
56:41     
building this Atlas so people can go back to the atlas and make connections and ask higher level questions and     
56:47     
things like that so they have you know this nice literature review I think it's um so in     
56:55     
this paper the analyze data generated by the senen project which is an important     
57:01     
project in cigan where they're doing things with cing genetics it produces     
57:06     
atlas of alternative splicings for 55 single neuron types we develop     
57:12     
analytical tools to make the study of this easier and they study differential     
57:17     
alternative splicings between neuron types so their data is shown as sort of     
57:23     
the differential expression of different or the different presence of different alternative splicings between neuron     
57:30     
types so if you have different neuron types you can see the difference so Alternatives these alternative splicings     
57:36     
are present in these cells but not these cells and so um focusing on canonical     
57:42     
alternative splicing events we establish overall patterns of differential splicing and then of course they     
57:49     
developed a computational approach to extract a regulatory Network for differential alternative SPL placings so     
57:58     
this kind of shows their methodology here where they're using facts to sort     
58:03     
cells so fax is a cell sorting technique they sort the cells by their identity     
58:09     
they extract the MRNA they use these RNA seek counts so they sequence and align     
58:16     
the rnas they RNA seek of course maps rnas to the mrnas to the genome to     
58:22     
different genes different uh locations and The genome but also counts the     
58:27     
number of transcripts so you get this quantification that tells you how much of a certain transcript you're getting     
58:34     
and of course in an alternative splicing you're going to have it mapped into that coding region you're just going to     
58:39     
exclude the uh the intron and so this kind of shows the     
58:48     
procedure this P this figure kind of shows uh a comparison to previous sping     
58:54     
data uh and so they show this in da 2 which is a gene uh in celegans and they show     
59:01     
kind of the connections between these different spacing or show the different splicing     
59:07     
patterns and what they might look like uh this is uh detection of a novel     
59:15     
cassette Exxon in UNCC 40 so they show in this un 40 they show this pattern of     
59:21     
splicing here and they show it the the evidence for in the uh in the blot here     
59:29     
where they have the different uh parts of the splicing down here and they can     
59:35     
show them sorted by molecular weight they show that Mel and then they show these exons     
59:42     
located in the coding region here     
59:49     
okay uh this shows you know some more molecular biology the differential as between neuron types so again you you     
59:56     
use a heat map to show the differential uh expression of different uh     
1:00:01     
alternative splicings uh so you have in this figure     
1:00:07     
here you have different cell types uh so you have il1 il2 Ava     
1:00:14     
a you see like of course the intersection of the same cell as gray     
1:00:21     
and then the expression of these alternative splicings in different shades of blue BL and so that's what     
1:00:27     
that's just showing they have this sort of by neurotransmitter type uh you know     
1:00:34     
they have you can identify different neurot cell uh you can identify cell types by     
1:00:40     
neurotransmitter you can and this is what they're showing in these colored bars down here so neurotransmitter type     
1:00:46     
cated cells either yes or no and they're modality so you have inter neurons motor     
1:00:52     
neurons Ral neurons and sensory neurons so you can sort all these cell types     
1:00:58     
that they these 55 cell types by their modality by their status of being cated     
1:01:03     
or not and the neurotransmitters they express and then look at those differences here so that's a nice uh     
1:01:11     
figure to look at some of these some of this     
1:01:20     
diversity okay so now they get into some of these networks here this is a splicing regulatory Network work so you     
1:01:27     
have the alternatively skipped exons the putative splice factors The covariance     
1:01:32     
Matrix and then the Precision Matrix they show these differ uh I guess     
1:01:37     
the spliceosome network they call it the spliceosome uh in other papers but     
1:01:43     
basically this shows uh your I guess your coding region linked to different     
1:01:49     
exons and then they show this for different genes uh I'm not really sure how this is     
1:01:55     
structured so um yeah there's a lot of methodology     
1:02:02     
that I'm skipping over here but basically they build these networks out of the spliceosome uh they use some different     
1:02:09     
algorithms to discover these uh spliceosomes and then to build these networks and then they uh you know     
1:02:17     
validate it through different     
1:02:22     
statistics so I think that's a good overview I know that's a pretty complex paper and you know it's I don't know how     
1:02:29     
useful it is there's an Associated Atlas online which I I don't I'm not going to go into right now but it allows you to     
1:02:36     
explore the data and usually the atlases are at a higher level than the paper so you can query specific genes specific     
1:02:44     
things that you want specific sequences and it gives you that information now linking this to the     
1:02:50     
phenotype of course they did a little bit of that in the paper with some of the uh different     
1:02:57     
uh you know neurotransmitter types and things like that but it it's maybe a     
1:03:02     
useful tool if you're interested in the connectome and you know thinking about some of the functions of cells in the     
1:03:09     
connectome and you know thinking about the connectum sort of as a functional entity not just as a structural entity     
1:03:17     
which a lot of people tend to think of it they think about the connections between cells and of course they're not     
1:03:23     
static they're varying all the time in terms of how they operate in a circuit or how they do things over long periods     
1:03:31     
of time so that's you know again I I don't know it's kind of hard to link a     
1:03:36     
lot of molecular stuff to um you know the phenotype of the connectone but     
1:03:42     
that's that's what we're trying to do here okay so uh     
1:03:51     
yeah okay if if there's nothing else I think we'll call it a day did we have have any questions before we go     
1:04:06     
comments no comments just um I guess thanks for the meeting and     
1:04:15     
um I'm going to have to explain the meeting to um Dr share because you have     
1:04:24     
my name on a paper     
1:04:29     
oh well yeah all right well well thanks for     
1:04:34     
presenting Susan that was a good present presentation of okay thank you right thank you all     
1:04:41     
right talk to you next week bye bye thanks all right now I'd     
1:04:47     
like to talk a little bit about morage ingredients and I have two papers here that sort of     
1:04:54     
reassess gradient based thinking in in orag Genesis and then some on more detailed     
1:05:01     
paper mathematical paper talking about interference and Lage of gradiance so     
1:05:07     
this first paper from the international Journal of Developmental biology this is from     
1:05:12     
2006 this paper is called the role and limits of a gradient-based explanation of Morag Genesis a theoretical     
1:05:21     
consideration so as we know you know radi are often what we use to describe     
1:05:29     
morphogenesis we have a source which is usually some chemical and that chemical     
1:05:34     
has a gradient and so if we think about this in terms of an embryo or maybe like the type of epithelial sheet that we     
1:05:41     
were talking about earlier let's let's do an epithelial sheet just to kind of give an idea so we     
1:05:48     
have these cells in an array two-dimensional sheet just just for demonstration     
1:05:55     
purposes and each one of these cells is going to have a chemical     
1:06:03     
source and so let's say that there's a chemical Source being expressed in this sub and the expression of this chemical     
1:06:10     
source is greatest in this cell and the chemical of course will     
1:06:17     
diffuse in this direction so this Arrow this vertical component is the     
1:06:23     
expression level and this horizontal set of uh arrows is the     
1:06:34     
diffusion and so our diffusion of course will go across the sheet of cells but     
1:06:39     
the level will decrease as you go farther away from the source so you end up with this     
1:06:49     
gradient that is basically shaped like o curve space and so you have your sort of     
1:06:57     
your diffusion that decreases as you go outward and eventually the amount     
1:07:02     
diffuses to almost nothing as you go for enough found now if all of our cells are     
1:07:08     
expressing this at the same rate then the gradient is sort of irrelevant because all of them sort of     
1:07:15     
have their own supply of this chemical but if not all them are expressing it then the gradient becomes     
1:07:22     
important and they get this chemical but different amounts so that means that the     
1:07:28     
effects of the chemical are going to decrease as you go towards the end of     
1:07:34     
this sheet of cells so these cells on the end are going to receive less of the chemical than the ones in the     
1:07:42     
middle okay now the thing is is that if we have this if we replicate the sheet     
1:07:47     
down here and we get uh let's do eight cells     
1:07:54     
here     
1:07:59     
and we have the same sort of situation but this time the cells on the ends are going to express some chemical     
1:08:07     
high this is these are going to be our sources of our chemical chemical a and     
1:08:12     
chemical B and so chemical a and chemical B are going to be sort of endogenous to these     
1:08:20     
cells only at the ends and they're going to diffuse outward so they're going to     
1:08:26     
diffuse the left into the right but in this case we're only interested in the right word component a and the left word     
1:08:32     
component B so we have a rightward and b     
1:08:39     
leftward and so these are going to continue out and this gradient will     
1:08:44     
Decay as we go outwards and as you see that they     
1:08:49     
intersect at some point so you can see that there at sort of this point they     
1:08:55     
sort of intersect you get this and it's not quite even but you get this effect where     
1:09:01     
you're sort of you know just kind of the minimal amount at this point but what happens in     
1:09:08     
this region so in this region you get an interference between the two chemicals     
1:09:16     
and often times in in chemical morphogenesis that's a gradient so you have if we draw our our bell curve     
1:09:26     
uh Tails here you have you know the uh     
1:09:32     
the Decay which decays almost nothing but you have this interference region which is here and     
1:09:39     
then often times in in um in like more     
1:09:46     
in and say something like an embryo you'll have a boundary that form     
1:09:51     
a sharp boundary that gives you some sort of     
1:09:57     
often you'll have a sharp boundary so if we have an embryo like     
1:10:06     
this we'll have two sources of a chemical these will     
1:10:12     
B fuse towards each other and you end up with a sharp boundary here because as they     
1:10:20     
interfere they kind of inhibit each other and you end up with this boundary     
1:10:25     
and so you this is how you get striping this is how you get boundaries between     
1:10:31     
chemical signals and this is the basis of chemical morog Genesis is trying to find     
1:10:37     
it as the basis for a lot of the sort of that pattern formation that     
1:10:44     
we talked about so of course this is just a model of     
1:10:49     
morphogenesis it's a model of chemical morphogenesis in particular and so just keep in mind that     
1:10:56     
you have this sort of uh diffusion you get overlapping diffusion you get this     
1:11:04     
interference and that can lead to some sort of interference pattern or can lead to this     
1:11:10     
boundary okay so that's sort of the context for both of these papers this paper here talks about the role and     
1:11:17     
limits of a gradient based explanation of work Genesis so this is kind of a theoretical consideration of these     
1:11:24     
limits and so the abstract reads development of an organism is a multi-dimensional     
1:11:30     
process leading to the generation of complex species specific structures so this is where you get different     
1:11:36     
chemicals being expressed different amounts of uh of mRNA or different types     
1:11:43     
of chemicals that get diffused across this phenotype and bound you know the     
1:11:49     
the maybe the striping patterns which are those boundaries or the distributions of these     
1:11:55     
what they call soluble substances which are these morphogens and this is a theoretical concept so any your when I     
1:12:03     
talked about chemical compounds you know I could mean morphogens and I could mean any soluble substance that can diffuse     
1:12:10     
in a cell cellular environment so the uneven distribution     
1:12:16     
of gradients and soluble substances morphogens and specific receptor Li     
1:12:22     
interactions are known to cause differential gene expression so radians can be evenly     
1:12:30     
distributed but they usually unevenly distributed so if we go back to our     
1:12:35     
image here we have sort of a bell curve in this example and generally we assume     
1:12:42     
diffusion to be the sort of uh I guess you know uh even sort of Decay this     
1:12:49     
linear Decay and so this is maybe one model that maybe one assumption that     
1:12:54     
doesn't actually exist in nature so you might have a skewed distribution here is     
1:13:00     
what I'm saying with you know diffusion rates That Vary sort of     
1:13:05     
nonlinearly so this is what it's kind of getting at with this is that the typical model Mor Genesis or at least this sort     
1:13:13     
of standard model maybe isn't as U amenable to reality as we like to     
1:13:20     
think uh so this can actually you know in in cases where you have have Radiance     
1:13:26     
and specific substances and receptor liing interactions we know that causes     
1:13:32     
differential gene expression so when we have these different gradients that interfere with each other we have these     
1:13:38     
different regimes it lead the differential gene expression between those regions therefore gradients and     
1:13:43     
morphogens are used as a causal explanation of development of processes so what I showed you in the gradients     
1:13:51     
these are causal so when there's a gradient of a chemical it triggers differential gene expression in these     
1:13:58     
cells and maybe in this middle region versus the end regions and that results     
1:14:04     
in differential gene expression which varies across this region so some Gene is being targeted by this chemical     
1:14:11     
signal it's being upregulated but it's being upregulated in a dosage dependent     
1:14:17     
Manner and so this is where you get this idea of about differential gene expression so that relationship is     
1:14:23     
thought to be causal and that's the assumption that he's talking about here however each attempt to describe     
1:14:30     
developmentally CA causality or developmental causality should take into account both the local fine organization     
1:14:38     
and the global robustness of morphogenesis so this is where we have to consider both this local example of     
1:14:46     
this uh gradient to cross saale like a sheet of Epal cells and This Global     
1:14:52     
relationship between gradients uh uh the classical view of the r morphogens will     
1:14:58     
be critically considered imp possible alternative proposed the core idea of my     
1:15:03     
proposal is that the main function of morphogenetic substances would be a context dependent modification of cell     
1:15:10     
Behavior so this is kind of what we're talking about with alternative spacings where alternative spacings are context     
1:15:17     
dependent and it can be mo they can be modified based on their where they're located in a sheet of cells for the cell     
1:15:25     
that exhibits alternative splicings those can be different in a sheet of cells and in a global context so if you     
1:15:32     
go from region to region they differences and you know in terms of cell type as     
1:15:37     
well both history and different features of morphogenetic fields create the framework for the activity of     
1:15:44     
morphogens so this is sort of a a critique of the standard     
1:15:50     
uh model of turning morphogenesis tur chemical morph     
1:15:56     
genis so this talks a little bit about causality and explanation and boundary     
1:16:01     
conditions so the issue of the causal factors in morphogenesis and development is both fascinating and fuzzy so you     
1:16:08     
know we talk about causal factors maybe in an oversimplified Manner and so we     
1:16:13     
have to think a little bit more deeply about these sort of causal factors and how they playable so he talks a lot     
1:16:20     
about causality and sort of how it operates in Genesis     
1:16:29     
um so it distinguishes between the what is and the what is the reason of so if     
1:16:36     
you look at a pattern you're going say what is a pattern and you can identify that and then you have to think about     
1:16:42     
what is the reason of a pattern so you know is it the pattern that's the causal     
1:16:47     
factor or is there something some mechanism that's the causal factor that leads to the pattern and so that's we     
1:16:55     
have to think about so sometimes when we talk about these kind of relationships we get them backwards or confused or     
1:17:01     
worse yet there's some sort of circular reasoning here where we think about the mechanism we think about the pattern and     
1:17:08     
then we think back to the mechanism and make this sort of I guess category error     
1:17:14     
I guess that's his position here so it is clear that the boundary conditions or contexts are a necessary     
1:17:21     
component for each physical event to be explained but one tends to deny their causal role by claiming that only an     
1:17:27     
efficient cause the source of the change is of explanatory relevance so this is     
1:17:34     
where you think about the efficient cause which is the source of the change and that's the only part of this that's     
1:17:41     
relevant and in fact in this case we have this mechanism and it sets up a     
1:17:47     
gradient and you have this pattern of course then we assume that there's some     
1:17:53     
effect in the genotype and that leads to uh the pattern that we see and of course     
1:18:00     
you know we have to think about that very carefully because maybe that's not     
1:18:05     
really what we're dealing with maybe there's a more Global explanation so we can think about this     
1:18:12     
uh the the the chemical signal and the gradient as being sort of an efficient     
1:18:18     
cause on the other hand it was emphasized that for real physical complex systems like the graner     
1:18:24     
convection boundary conditions play a crucial causal role so if we think about this at     
1:18:29     
the global scale they're boundary conditions namely the boundary conditions that are characterized by     
1:18:34     
this interference is just one example that actually also play a role in this     
1:18:40     
pattern formation so this mechanism may be important but it's not the only aspect that we need to pay attention to     
1:18:46     
it could be boundary conditions of the global     
1:18:52     
case as Michael Pani argued the same is true for machines in his view a machine     
1:18:57     
cannot be explained in terms of physics and chemistry principle of structure controls the principles of physics and     
1:19:03     
chemistry so the structure actually controls the principles of physics and chemistry maybe within a compartment or     
1:19:11     
you know at the global scale it could be suggested that the notion of causal form malice is of great     
1:19:18     
importance to biology since it explains how a certain change can occur even in the absence of an efficient clause so     
1:19:25     
even when you don't have an efficient cause you can have some sort of uh you     
1:19:30     
know have a change occur we have to think about this when we observe the efficient cause in conjunction with the     
1:19:37     
change we think it's obvious that there's some sort of cause of effect but     
1:19:42     
if the cause is not app not present and you also have the same effect then you     
1:19:48     
know we don't usually consider that case and this is uh the kind of counterfactual that you need in terms of     
1:19:54     
looking causality such situations are common in biology if some function could not be performed or if some substance is     
1:20:01     
not available so if our substance isn't available here if we have this effect when the substance is available     
1:20:07     
obviously we can say we can point to this kind of model and say yes this explains it but when we don't have this     
1:20:14     
kind of when the substance is unavailable we still get this kind of pattern then what's the consequence and     
1:20:21     
a lot of times in biological systems you'll get standin or you'll get other things that are involved and sometimes     
1:20:27     
that's you know that can confuse our sort of causal model and that's one of the problems in biology you do have     
1:20:34     
these uh sort of these uh shortcuts substitutions that can be problematic     
1:20:41     
from really getting at these counterfactuals so in such cases the boundary conditions play an undoubtedly     
1:20:47     
causal role so sometimes the boundary conditions can uh play a role in the absence of some sort of substance not     
1:20:54     
being being available or that mechanism not being available this is because development is     
1:21:00     
a process that not only uses previously existing conditions but also creates new     
1:21:06     
conditions so when you you know and we talked about this in uh the context of uh systemic closure     
1:21:16     
and so systemic closure is where you have systems that are sort of closed in a sense not necessarily energetically     
1:21:24     
closed will closed in the sense of causality so you have these causal Loops that kind of feed upon themselves and     
1:21:30     
that's really uh sort of an identifier of biological systems of living systems     
1:21:37     
that they can create their own Dynamics and so this is something that of course     
1:21:43     
throws a wrench in this model of causality where you have a mechanism and you have an effect at the global scale     
1:21:50     
you can actually have boundary conditions which reinforce these things and even in the absence of a clear     
1:21:55     
mechanism can produce these outcomes recently the idea has become     
1:22:02     
increasingly widespread that the role of causa for malice is the origin of morphog genetic morphological Novelties     
1:22:09     
and is crucial as an increasingly grown over the course of biological evolution and it sites Newman Mueller     
1:22:17     
2001 uh so this is where they kind of talk about this a little bit more     
1:22:22     
unfortunately the consideration of the notion of a context is outside the scope of most recent approaches to     
1:22:28     
morphogenesis instead the efficient molecular factors morphogens and morphogen mediated interactions so these     
1:22:36     
again are these chemical factors that play a role in this kind of sort of a     
1:22:41     
reductionist approach to looking at this uh are regarded as crucial     
1:22:47     
explanatory Tools in developmental biology this concept implies that uneven distribution of morphogens usually in     
1:22:54     
the form gradient which is what we saw here where the effects are on nonuniform     
1:22:59     
across space uh but of course you know we have of course different types of     
1:23:05     
non-uniformity in this gradient um lead to a stepwise symmetry breaking     
1:23:13     
in embryo in embryos into its subsequent morphogenesis and     
1:23:18     
differentiation so there's this idea that uh there's this uneven distribution     
1:23:24     
of Mor and this leads these step stepwise symmetry breaking events and that's it's     
1:23:29     
kind of what he's getting at considering alternativ stud so he talks about the history of     
1:23:36     
the gradient concept and this is in development this is not necessar where this is in chemical systems this is not     
1:23:43     
necessarily the same thing as energetic gradiance that we might use in machine learning although they're somewhat     
1:23:49     
related in terms of the mathematics um but so this is the FOC     
1:23:54     
Focus here is on chemical gradients the idea of gradients in developing organisms is very old Theodor     
1:24:02     
B seems to be the first who suggested that a gradient or differential concentration of substances determines     
1:24:08     
the cell's fate later child developed a detailed theory of gradients his idea was based on     
1:24:15     
experimental results showing different regenerative poent potentialities or potencies in planarians and other lower     
1:24:23     
invertebrates along the anterior posterior axis so this is showing this     
1:24:28     
but in sort of a very simple model in the planarian it's you know it's where you have this sort of anatomical     
1:24:35     
variation in this model so in the embryo you would have sort of the in the anterior pole you'd have a difference     
1:24:42     
from the posterior pole     
1:24:47     
um and so the gradient postulated by child is a metabolic one and assumed to be generated by unic specific external     
1:24:55     
excitation and its transmission therefore the formation of the gradient was possible without any     
1:25:01     
pre-existing differentiation so you don't need differentiation to have these gradients     
1:25:06     
formed and of course this is this reinforcing mechanism that he was stressing before that once you have     
1:25:14     
differentiation then you can get uh this sort these sort of gradient effects     
1:25:20     
sometimes without the gradient actually being I guess is the word but in this case they were able to show the     
1:25:26     
formation of the gradient without any pre-existing differentiation which means there's something driving this to start     
1:25:32     
off child also recognized the relation between gradients and genes if a     
1:25:37     
gradient extends over more than a single cell cells among the course represent different levels and these differences     
1:25:44     
provide a basis and difference in gene action certainly an essential factor in     
1:25:51     
differentiation and so this idea of Developmental potency Seas kept going uh throughout the 30s and 40s they looked     
1:25:58     
in 1939 at the animal vegetal axis and sea Orin embryos it was shown that     
1:26:06     
limitations of the fate of the embryonic substratum is related to its position within the embryo gradual manner so     
1:26:13     
these are these sort of L curves that I showed you uh this this result was interpreted     
1:26:21     
in terms of gradients and animalizing and vegetal substances so the differences in know the embryo terms of     
1:26:29     
these gradients between two different substances the gradient view is closely     
1:26:34     
related to the concept of positional information which would come later by wolber and the idea is that there     
1:26:40     
different positions in the embryo they have different sorts of uh stuff in them     
1:26:46     
so if we think about our model here you know we have these like say like uh in     
1:26:52     
this latest case we have the animal pole the vegetable pole and we have this     
1:26:58     
gradient that kind of forms where we     
1:27:03     
have animal stuff here and vegetable stuff     
1:27:08     
here and you know you can like Define animal stuff down here and vegetable stuff down here and there's either some     
1:27:15     
sort of smooth transition between this so you either have this sort of slope like this or you have this more     
1:27:23     
discont continuous uh transition here so that's     
1:27:29     
what we're trying to do we're trying to find the pattern of transition so you can either have     
1:27:34     
something that starts out here and gradually be goes from being animal to gradually becoming vegetable or you can     
1:27:41     
have something that's animal down here and vegetable down here and there's this very sharp transition     
1:27:48     
between this is the type of thing we're interested in gradients we're interested in the slope     
1:27:53     
of the gradient and of course gradient is this NAA     
1:27:59     
symbol interested in the slope of the gradient in the shape of the gradient so we're interested in you know depending     
1:28:06     
on what position we're in in the embryo we want to return sort of our     
1:28:11     
gradient value so we're in the gradient are so the gradient is very gradual we     
1:28:19     
want to know kind of where we are and that allows us to do you know     
1:28:24     
to look at like differences in position and ultimately you know we can     
1:28:30     
apply calculus to this some other types of differential and mathematical methods     
1:28:35     
which make this really interesting so this is kind of where we're going with uh this sort of history     
1:28:43     
in developmental biology uh so this concept implies that     
1:28:49     
the cells acquire a positional value with respect to boundaries and then interpreted in terms of a program     
1:28:55     
determined by their genetic Constitution developmental history the concept of     
1:29:00     
morphen was introduced a class of substances causing a specific howcome of     
1:29:06     
development so the morphogen was introduced by Turing but also refined     
1:29:11     
that concept positional information you have this class of substances that can be very generic you could like Define it     
1:29:18     
as any substance that you can think of and but it has to cause a specific outcome and development and it has to     
1:29:24     
have some positional uh you know specificity so     
1:29:29     
this is this is important Al important in terms of this concept of the positional value is assumed to be     
1:29:37     
generated by interactions of a cell with different concentrations of a morphogen morphogens are defined as soluble     
1:29:44     
substances that diffuse uh the first clear demonstration of orphagen Radiance of course in     
1:29:52     
ropa um and this was in 1988 so this is pretty late in this process but you know     
1:29:57     
this is showing kind of the striping uh this patterning mechanism in veropa during the last two decades this     
1:30:05     
knowledge of morphogens has been expanded enormously and this is led to this     
1:30:11     
gradient base Theory and so then you know he talks about the logic and natural necessity of     
1:30:17     
Morag and radiant formation and so you know the following     
1:30:23     
theoretical reasons imply The Logical necessity of a morphogen based Theory so     
1:30:29     
first is since the genomes are equivalent in all cells which is maybe not a fair assumption the cause of     
1:30:34     
heterogenity should be extrinsic so if we assume that the genomes of all these     
1:30:39     
cells are equivalent in our sheet then the thing that's affecting     
1:30:44     
the differential expression here is this external radiant and not variations in     
1:30:50     
in the genome and I guess another way to put that is is that there's no way to say because the cells have a specific     
1:30:57     
position that there's something in the genome that codes for this gradient the gradient is external and affects the     
1:31:04     
expression of genes in a way that like is kind of dose dependent so you can see     
1:31:10     
that in this example where you get this Decay over space so any one of these cells is going to have more or less the     
1:31:16     
same genomic complement there are no specific mutations or there no specific     
1:31:22     
recombinations or splicing events that are done in a patterned manner that allow for this gradient you get this     
1:31:30     
gradient from this sort of external stimulus and indeed the whole idea of     
1:31:35     
boundary conditions is consistent with this because the boundary conditions then also constrain these cells     
1:31:42     
externally so that's that's number one number two is it is known that specific receptor Lian interactions are able to     
1:31:48     
modify gene expression so this is where these signals are transduced into the     
1:31:53     
cell and this is where we know that that can modify gene expression as well as     
1:31:59     
getting information from that external gradient or this external sources into the     
1:32:04     
cell three that ligans are external signals so we can use ligans at The receptors and those     
1:32:11     
lians qualify as sort of these morphogens four is sort of the     
1:32:18     
concluding Point thus these molecular Lian should be unevenly distributed in space in order to activ genes in     
1:32:24     
different ways how will this distribution be generated so the theory of diffusible     
1:32:31     
morphogens which form a gradient has seen more success in recent years     
1:32:36     
mechanism of diffusion has been subject mechanisms of diffusion have been subject to intensive study different     
1:32:43     
experiments shown that have shown that concentration dependent actions of morphogens are important for example     
1:32:51     
activin which is a substance from the TGF group is able to act in a concentration dependent manner so we     
1:32:57     
know that there are substances that do these things it's not just a theoretical concept so then he talks about the     
1:33:04     
problems associated with morphogen based explanations so this is where he talks about morphogen based explanations and     
1:33:10     
then you know gets into the problems so I stressed above the explanation of     
1:33:16     
symmetry breaking exclusively by previously existing Mally symmetry is not     
1:33:21     
appropriate so this is where we have these you know potential mechanisms uh     
1:33:28     
this idea of symmetry sequential symmetry breaking is not appropriate and we need to kind of think about some of     
1:33:35     
the reasons why that is um and so uh basically he's arguing that some     
1:33:42     
of these like I said before there's this explanatory regress that occurs um     
1:33:48     
because you already need some establish the Symmetry to achieve this diffusion     
1:33:53     
model so you need to have some sort of structure already in place for this to occur and so this is what he's getting     
1:33:59     
at at this point again this has to do with causality um the discovery of Highly     
1:34:07     
robust character in of Developmental     
1:34:14     
uh the discovery of the highly robust character of development contradicts the specific nature of molecular     
1:34:20     
interactions within the cell um and so there's a lot of variability     
1:34:27     
of the mechanisms of gastrulation in the development of the clonial hydroid diopa so you have a lot     
1:34:35     
of different ways that you can produce gastation in this this is actually typical of biology so you have a lot of     
1:34:43     
things going on with in terms of cell number and the size of the embryo and     
1:34:48     
macroscopic mechanisms that can vary quite a bit and so you still get normal     
1:34:55     
development it's very robust and you know molecular Events maybe aren't as     
1:35:01     
robust and so there's this mismatch between what we see at the phenotypic level and what we see at the molecular     
1:35:07     
level that's coming into question we also have this role of     
1:35:12     
mechanical stresses so of course mechanical stresses are factored into this model at all and of course in in     
1:35:20     
chemical morphogenesis you know the assumption is is that it's operating on its own and so you know we have a lot of     
1:35:26     
mechanical stresses as we've talked about with uh respect to uh some of     
1:35:31     
Susan's work and so this is where we get this uh you know we can we can put that     
1:35:38     
into the you can put those factors into the model but then it kind of changes the model and so you know you have to     
1:35:46     
think about that     
1:35:52     
um and so you know you know it may be true that cell behavior is actually the tarom morphogens as opposed to sort of     
1:35:59     
the molecular aspect of it so w and inber in 2000 suggested that the     
1:36:05     
behavior of a cell can be described in the framework of an attraction based model they describe five different modes     
1:36:12     
of cell Behavior quesence apotosis proliferation differentiation of movement as stable States and so the     
1:36:20     
idea is that there's this there are these different homeostatic stat states that these chemical signals can actually     
1:36:27     
Target those cell behaviors rather than molecular circuits and that's maybe what     
1:36:32     
we're talking about so again we're we're considering different aspects of the cell mechanical Behavior it's sort of     
1:36:41     
phenotypic behavior and putting that into our modelis well     
1:36:54     
okay so basically in in conclusion from a theoretical point of view any local explanation work Genesis is not     
1:37:01     
appropriate we need to consider both local and Global interactions and that's really the way this works we also have     
1:37:08     
to think about the action of any morphogen in the context of the morphogenetic behavior of cells and     
1:37:14     
tissues as well as the mechanical aspects of cells and tissues so these interactions between mechanical and     
1:37:20     
chemical influences are really sort of at the core of any model of morphogenesis and of course chemical     
1:37:26     
morphogenesis is incomplete in that     
1:37:40     
respect I said the second paper is from 2021 this is from veteran Iber Roman     
1:37:47     
veteran vagar Iber and this is a a preprint um so I'm not sure where it was     
1:37:52     
eventually published but the title is interference and morphogen gradient Precision for     
1:37:57     
molecular noise data so this is where we talk with the morphogen grading very specifically and the role of molecular     
1:38:04     
noise so we're not talking about cell behaviors or mechanics we're just     
1:38:09     
focusing on this molecular gradient focusing on its Precision but also talking about molecular noise so this     
1:38:16     
requires us to think about molecular noise which is of course the noisiness of gene expression and the role of that     
1:38:23     
this process so in the process I showed here with this bell curve this gradient     
1:38:30     
with this gradient here this uh animal vegetal gradient you know this is where     
1:38:36     
we consider the gradient as this transition of this smooth deterministic     
1:38:42     
process but in a lot of and of course that's really kind of key to sort of the     
1:38:48     
uh replicability of development so you get this General replic ability um in     
1:38:55     
development where you get these gradients it have to be in the right place and you know things have to be     
1:39:00     
spatially distinct but we also have molecular noise and of course molecular     
1:39:05     
noise is noise and gene expression where you get you know different types of uh     
1:39:12     
variation in terms of the amount of morphen that's produced then that ranges     
1:39:18     
over time and there's so there's this noise and your process is either tolerant to the noise or     
1:39:23     
so that's what they're getting with here so let's read the abstract and see you     
1:39:29     
know what they're getting at this paper during development orphagen gradients provide spatial information     
1:39:35     
for tissue patterning so of course we've talked about this gradiance and readout mechanisms are inevitably variable of     
1:39:43     
course yet the resulting patterns are strikingly precise and this is what I mean by this replicability of     
1:39:49     
development these things are precise even though they're variable with respect back to cell SI cell number cell     
1:39:57     
size uh embryo size and so forth measurement limitations currently     
1:40:03     
preclude precise detection of morphagene Radiance over long distances so you know     
1:40:08     
we can measure uh morphing gradients it's hard to measure them over long     
1:40:14     
distances usually their local effects and then they're embedded in this global     
1:40:19     
system of Radiance that intersect in in other ones so it's hard to really measure them over long     
1:40:26     
distances um in with the way we do uh the measurement now here we develop a     
1:40:33     
new formalism to estimate gradient Precision along the entire patterning axis for measurements close to the     
1:40:39     
source using numerical simulations we iner radiate variability for measureing     
1:40:44     
molecular noise levels and Morgen production Decay and diffusion so our     
1:40:50     
you know our production Decay and diffusion are all by this sort of stochastic process we have to use that     
1:40:58     
to deter you know we can estimate the the gradient and the extent of the     
1:41:04     
gradient so when we get so we start off with a lot we end up with a little you     
1:41:09     
know this isn't a smooth bell curve this is a lot of variation across uh the     
1:41:14     
distribution but also at the ends of the distribution you have very little to work with so noise can really dominate     
1:41:22     
here and so no one what this diffusion looks like and the presence of noise is important and so what they're doing is     
1:41:28     
they're developing some mathematical techniques to measure to to sort of     
1:41:33     
estimate this gradient for molecular noise levels and the variability in this     
1:41:39     
gradient and produce it a gradient that can be predictive over the long distances within the     
1:41:46     
embryo the predicted Precision is much higher than previously measured so this is actually suggesting that despite all     
1:41:54     
the noise that you might have uh intrinsic noise in gene expression that     
1:42:00     
this predicted Precision is much higher or at least it's much more robust they don't really say that but the you know     
1:42:08     
we can use this model and it's actually pretty productive this model is precise enough     
1:42:14     
to allow even single gradient to define the central prenative BS during all tube     
1:42:19     
development they pick the specific event and devel NE tube development and they     
1:42:25     
look at the central progenerative Aries and they can actually predict those pretty well finally we show that the patterning     
1:42:32     
mechanisms optimized precise progenitor cell numbers rather than precise     
1:42:37     
boundary positions so it's actually like you know when we estimate these we can     
1:42:42     
estimate these for cell numbers or for boundary positions both are sort of spatial in nature but cell numbers are     
1:42:50     
less specific explicitly spatial meaning that maybe our you know our position     
1:42:56     
positional information is more about the number of cells in an array rather than     
1:43:02     
you know like a coordinate system as the progator domain sizes particular with bust gradient     
1:43:09     
alterations so our domain sizes really kind of can be you know variable     
1:43:16     
depending on how the gradient varies um and so you know there's this it's easy     
1:43:22     
to fit this gradient to different domain sizes we conclude that single gradients can yield the observe development of     
1:43:29     
precision which provides new prospects for tissue engineering so in Appliance to tissue     
1:43:34     
engineering you know we need to have this sort of model that's flexible because our course our systems are going     
1:43:40     
to be variable depending on you know what what side you know your tissue if you're     
1:43:46     
engineering tissues they're going to be of shapes and sizes maybe not seen in     
1:43:52     
nature so that's something that we need to have for these gradients um so in this paper and I'm     
1:43:59     
not going to go over a lot of it they really get into very detailed mathematics they basically talk about     
1:44:05     
this morphing gradient as approximated by an exponential function so in this     
1:44:10     
case I kind of gave an example of a Vell curve or of a normal distribution in this case they're modeling this with an     
1:44:17     
exponential function which has an amplitude c0 and a Decay length of Lambda so Lambda is the Decay um and     
1:44:26     
this produces an exponential distribution that decays with a tail of a certain length and the tail of course     
1:44:32     
can be long since in an exponential function the length of the gradient or     
1:44:37     
Lambda specifies the distance over which the morphogen concentration decays so this is where you know we have the tail     
1:44:44     
which is down here and that tail can be very long and it can extend outward so the tail is kind of what we're     
1:44:50     
interested in for some of you know the farther outreaches of this gradient as     
1:44:56     
it gets into these transition zones and interference zones and so that's kind of what they're doing here they're modeling     
1:45:02     
this uh with respect to concentration decays and how this Decay is exponential     
1:45:09     
the patterning domain is typically much larger than them such that the concentration gradient declines     
1:45:15     
substantially within so for instance the bicoid gradient length is a fifth of the     
1:45:20     
length of the early drop embryo so so this is where the bco gradient actually     
1:45:25     
goes out uh to a fifth of the length of the early embryo uh the DPP or     
1:45:33     
uh eent plgic radient and the verh wingis is even shorter at 0.11 the     
1:45:41     
length so this is where we can take this Lambda and divide it by the length and get these gradient sizes the relative     
1:45:47     
gradient of and the relative lengths of the Sonic Hedgehog and po morphogenetic protein     
1:45:53     
EMP and shh gradient declined from about 0.2 to 0.05 during Mouse neural tube     
1:46:02     
development so in in Mouse neural tube development Sonic Hedgehog and Bone     
1:46:07     
morph genetic protein are actually quite small relative to the embryo size and     
1:46:15     
this is during Mouse neural tube development so this is where we have the specific structure that's being     
1:46:21     
formed so we can see that there's variation in these uh gradients and they kind of serve the needs of what's going     
1:46:28     
on in the embryo so this is kind of getting into the maybe the more the math     
1:46:34     
of it the other paper he was speculating about the size and robustness of these     
1:46:40     
gradients in the developing ver vertebrate neural tube positional information is provided by opposing     
1:46:47     
Sonic Hedgehog and bortic protein gradients so this is where there is this     
1:46:52     
interference region and you know how this kind of is triangulated from these     
1:46:58     
different gradients even if each morphagene pattern is only one half of the domain     
1:47:04     
the morphagene concentration at the center of the final domain will be about 10 to the fourth fold lower than at the     
1:47:10     
source so again we're talking about sort of the uh the tals of this uh radiant     
1:47:17     
and where it's actually quite much quite a bit lower than the source um so this     
1:47:22     
is where you know how do you sense uh this this decayed gradient over     
1:47:28     
such distances and how precise such patterning information will be so when you have a very low level of the     
1:47:35     
gradient in a certain location how do you sense that like gradient Freo     
1:47:40     
concentration and how efficacious is it in doing something to that side so if we     
1:47:46     
go to bacterial chemotaxis and the responses chemotactic responses there     
1:47:51     
adaptational sense cells to sense concentration gradients spanning at least five orders of magnitude so this     
1:47:58     
is where you know cells different cells are sensing concentration gradients in     
1:48:04     
in the environment they're moving towards different uh sources chemical sources and so they have a really     
1:48:11     
sensitive uh mechanism for sensing these gradients so they get to the edge of a     
1:48:17     
of a chemotactic gradient so in the case of something like a bacterial cell you     
1:48:23     
might have a chemotactic gradient which is this patch of chemical a source     
1:48:30     
here and you have bacterial cells and they're moving towards the     
1:48:37     
source moving in here okay they're navigating across this     
1:48:43     
space they're navigating towards the source they're following and they have this very     
1:48:50     
specific uh sensory ability so the very sensitive to the edge of this gradient because they need to be able to get to     
1:48:56     
the edge of the gradient in order to get into this increasing concentration     
1:49:01     
remember they're moving uh across say a free environment this isn't a cellular     
1:49:06     
environment this isn't where cells are communicating with other cells as where cells are picking up a signal that's an     
1:49:14     
ingredient that's separate from their sort of there aren't any bacterial tissues I     
1:49:20     
mean they're bacterial colonies but they're going towards this chemical gradient independent at least nominally I mean     
1:49:27     
they may uh give information to each other about the best way to get to the gradient but that's not really what     
1:49:34     
otacc responses are all about at least the way we measure them uh so uh whether similar effects     
1:49:42     
are at work also when the neural tube is unclear that adaptation you know Sonic Hedgehog responsiveness has been     
1:49:51     
noted so so uh they talk about the French flag model which is of course a     
1:49:56     
famous model of of Genesis and the rout position ex     
1:50:04     
uh rout position is defined by the concentration threshold C by rearranging     
1:50:10     
this equation one one obtains this equation as a result of molecular noise     
1:50:15     
the gradient like Lambda and the ratio between gradiant amplitude and the rout threshold differ between embryos this     
1:50:23     
difference translates into shifts in the readout positions the overall readout position is typically calculated as the     
1:50:29     
mean of the individual positions in the different embryos I and then the positional error is typically Quantified     
1:50:36     
by the standard deviation so in the presence of molecular noise we can give     
1:50:41     
this uh positional error so you know there's this error that can occur especially when you have very low     
1:50:47     
amounts of some orphagen and this you can model this using a standard deviation so this gives you this noise     
1:50:55     
Factor here uh following definition the readout position and accuracy of two centrally     
1:51:02     
located regenerator domain boundaries with Quantified in the mouse to positional error of both the dorsal     
1:51:09     
boundary and the ventral boundary were found to be about 1 to three cell dieters this is where we have these     
1:51:16     
dorsal and vental boundaries of different factors in KX 6.1 andax 3 and     
1:51:22     
there's uh sort of this positional error that occurs so it's not you know it's not     
1:51:29     
absolutely precise but it is precise enough to be prople and you do have this     
1:51:34     
one to three cell diameter error that can occur in parallel also the gradient     
1:51:40     
variability was uh given the challenges of measuring morphid and gradients directly we used the transcriptional     
1:51:47     
reporter as a read out of signaling and so close to the source position I error     
1:51:53     
increased from a single cell diameter to about three cell diameters over time so     
1:51:58     
this error of course could be from within one cell to three cells three cell diameters which means that it could     
1:52:04     
be within one cell to over three cells in the center of the neural tube however the positional error was reported to     
1:52:11     
increase in three cell diameters in early stages to more than 30 cell diameters later on so what that means     
1:52:19     
perhaps is that at the edges of these boundaries of these radians it's more     
1:52:25     
important to be precise and in fact that's what you see you see more Precision you get these uh tolerances     
1:52:32     
that are in the range of micrometers or or you know like five micrometers to 15     
1:52:39     
m in this where it's less prec where it needs to be less precise we're in the     
1:52:45     
center of the neural tube where it's less critical to patterning the positional error can be greater and so     
1:52:52     
30 cell diameters uh is 150 M microns so you can     
1:52:58     
have a much larger cell diamet or you can have much larger errow where you     
1:53:04     
don't necessarily need specification combine readouts of the imprecise Sonic Hedgehog and uh borag     
1:53:12     
gentic protein gradients was proposed to yield the higher Precision of the central regenerator vales so this is     
1:53:19     
showing an example here something in in this uh 3D model here where you have the     
1:53:26     
dorsal side the ventral side you have these different factors being expressed with     
1:53:32     
gradients in particular you have the Sonic Hedgehog gradient and this B     
1:53:37     
morphogenetic protein gradient and they start one starts at the dorsal side BMP     
1:53:43     
ventral side we have Sonic Hedgehog and they kind of intersect in the middle and     
1:53:49     
you get this boundary so you get this sort of intersection between the two gradients and so this is just showing     
1:53:55     
this and this is showing some of the distance from source so it shows this     
1:54:00     
function with noise as it decays um and then this is the exponential     
1:54:07     
distribution uh then you see this in embrionic positional variability here where you get this difference uh in the     
1:54:14     
distance from source so sometimes you know you get a a greater distance from Source or a lesser distance from Source     
1:54:21     
depending on this interionic positional variability     
1:54:26     
this results in a distribution of readout positions with an interionic mean and a positional error given by the     
1:54:33     
standard     
1:54:40     
deviation okay so the rest of this paper is talking about these statistical     
1:54:45     
gradient parameters for the mouse neural tube looking at this example here and     
1:54:50     
working it out and then the of course they talk about the technical limitations of these measurements and so this is kind what     
1:54:58     
they're talking about here about gradient routing Precision in the developing neural tube and they kind of     
1:55:03     
give some examples of that here and so you know a couple words on gradient variability as a result of molecular     
1:55:10     
noise so in a cellular tissue the morphogen production degradation and transport rates vary from cell to cell     
1:55:17     
and that's a very important thing to remember that even though the genome is more or less     
1:55:23     
uniform the kinetics of the cell or not and so this variability ultimately     
1:55:29     
generates the variability in the steady state morphing gradient profiles we can     
1:55:34     
estimate this variability by simulating a simple reaction to Fusion model on a continuous one-dimensional domain where     
1:55:41     
these parameter values differ randomly from segment to segment to describe the     
1:55:47     
steady state morphogen profiles we solve the steady state reaction diffusion equation this Swit is here and a     
1:55:55     
one-dimensional domain that was split to Su two subdomains a morphogen source and     
1:56:00     
a patterning region okay then they use a heide step function to ensure that the     
1:56:05     
morphogen produced at rate p is only in the source where it degrades at a linear     
1:56:11     
rate D everywhere so you have this ruction rate p and you have this linear     
1:56:16     
Decay rate D morphogen transport is driven by fii and diffusion with diffusivity capital     
1:56:23     
D and zero flux boundary conditions uh this gives you a concentration profile that follows     
1:56:30     
hyperbolic cosign here and this Decay length which is large D / into small D     
1:56:37     
and that's the square root of that which is of course the equivalent or equal to     
1:56:42     
Lambda depends on the morphogen diffus diffusivity capital D and the turnover     
1:56:48     
rate small D so it's the the diffusivity over the turn turnover rate in a square     
1:56:54     
rout and so this is a numerical model that predicts gradient variability from molecular noise you get the source which     
1:57:01     
are five cells here and then the patterning domain which is 20 to 200 cells down here and you get this where     
1:57:10     
you see this numerical simulation here the coefficient of     
1:57:15     
variation here and this shows the results so for C cell variability     
1:57:21     
patterning link that s variability here     
1:57:29     
cbk so I think those two papers kind of uh maybe improve our understanding of     
1:57:35     
morphing gradients maybe challenge them a little bit but definitely I think the second paper will kind of answer some of     
1:57:41     
the questions the first paper but actually maybe raises more questions it's a good modeling paper and shows a     
1:57:48     
lot of quantitative context to a lot of these ideas thank you for paying attention and I hope you learn something
