## TRABSCRIPT

0:00     
hello hello hello morning morning hello     
0:06     
Susan hi so how how are things is anything you     
0:12     
want to share with the group or don't know if Susan had uh anything from last week that you want to talk     
0:20     
about I just like P Factor um with t     
0:25     
seges it if you um take it down     
0:30     
um by a factor of 10 uh the resulting     
0:35     
stress strain curve is also reduced by a factor of 10 kind of so I'm playing with     
0:43     
that seem S I have a question do you have an     
0:50     
ability to simulate any tensity     
0:55     
stru um I I can try but whether it will     
1:01     
stand up it it's it's a finite element program and so for instance that     
1:09     
that x x um structure where there's an X here and then there doesn't St won't um     
1:19     
compute doesn't compute I meant the standard one with the rod and uh     
1:26     
strings um if you're talking about the um one     
1:31     
that's yeah rods and strings that are kind of like this yeah okay you can good okay uh     
1:40     
something strange has happened in the world of nuclear physics okay yeah somebody tried to make     
1:48     
oxygen with a total of 28 neutrons plus protons     
1:55     
yeah you heard about this um just briefly okay they they got a weird     
2:01     
result they were expecting it to be very stable and it     
2:06     
wasn't and this caused me to think well maybe this structure of a nucleus is a 10 secrity structure and when they made     
2:14     
this one they made it in a very unstable form yeah     
2:21     
well up theories of nuclear structure and the whole field is a mess well it's     
2:28     
they've been doing 10 seg structures to figure out the magnetic fields of things     
2:34     
um some of the low temperature uh physics the um they've been working with     
2:41     
some sort of tensity structure okay well but that's probably     
2:48     
molecular um no just looked like it     
2:54     
was um low temperature physics um I I'll have to look it up um look I'd     
3:01     
appreciate that so anyway I came up the idea that the re the reason that they     
3:08     
didn't get the result they expected is that they made the wrong T secrity structure for the     
3:14     
nucleus and uh so uh this to do this     
3:19     
this requires the short range interactions between neutrons and neutrons and     
3:25     
neutrons and protons protons and protons plus the repulsion of the uh the     
3:34     
protons which strikes me as a t seg structure with a field so to speak     
3:40     
around it oh yeah yeah okay the to the     
3:45     
field or something so uh anyway if I can interest you in it maybe we can do it     
3:50     
after you get your Masters okay I wouldn't want to burden     
3:55     
you with it now oh well I'm I'm I'm annoyed with finite element analysis and     
4:02     
how it works with Tegrity structures there could be a lot more research done on 10 segy structures if     
4:09     
there was a program that wouldn't go all Bonkers because it's     
4:16     
because the thing moved and and um became stiffer just because you put a     
4:23     
finite pressure on it because that's what it does it does these increments     
4:29     
fin increments of force well this this might just be a matter of uh trying to uh get     
4:38     
the energy states and metastable States uh of uh given structure and then     
4:47     
uh it struck me if if we can do that then we can make something at least     
4:53     
analogous to the isotope     
4:58     
tables number neutrons and number of protons against each other and they they     
5:04     
uh they predict which nuclei should exist okay and which are very uh which     
5:11     
should be very transient well I K value what's a k     
5:18     
value it's the it's the elasticity times the area     
5:26     
of the elastic divided by the um elasticity of the rod     
5:34     
times the um area of the     
5:40     
rod okay well modeling this might be fun but I     
5:45     
think it could be fairly straightforward well well you should I     
5:51     
should probably explain K values um to to the group     
5:57     
sometimes value intens integrities that yeah I have a lot of nuclear physics to learn in the     
6:03     
meantime yeah okay and I'll look up this I mean     
6:09     
that struck me that something so fundamental as the structure of a an atomic nucleus is a total mess phys okay     
6:18     
yeah yeah well I still want I want better programming for 10 sties and     
6:27     
I there was something called grass copper that did 10 seges and the people     
6:33     
working with 10 seges even use Mathematica mind you was really the     
6:39     
program is really slow when you use Mathematica but it works at least oh     
6:46     
okay I have mathematic so yeah so you you can make math you can make 10 seg     
6:53     
structures in Mathematica okay if we get the code then I'll control okay I'll I'll try to find I think I have a sample     
7:00     
code okay okay I said after your Masters I have a sample code because I I     
7:07     
took the I went to the 10 segr Conference um it was started at 2 in the     
7:14     
morning and finished at about 10 in the morning and with big break for lunch at     
7:19     
about 6 in the morning I couldn't do I could not do the lectures at two in the     
7:26     
morning that much I can tell you I could wake up at 4 do do a lecture and     
7:34     
then when they had lunch I would go back to sleep and then I would wake up at 8 o'clock when they came back on so that's     
7:41     
how I attended this conference it was in um Italy oh     
7:46     
okay okay yeah well I haven't stayed up all night since I was an undergrad so     
7:54     
why I can't anymore and I certainly can't wake up at two o'clock and stay awake     
8:00     
started talking I tried it one the first day and back to sleep yeah stay up on I     
8:07     
had a class around nine o'clock or so and I I I realized I was Dopey and uh so     
8:14     
I never did it again oh yeah well this is this is not just dopy this is me     
8:20     
going to sleep okay well that's     
8:31     
there Bradley we're off on another field     
8:37     
yeah all right well I'll shut off my camera     
8:42     
because it usually lags for some reason it just doesn't work like it should so     
8:50     
it's working here fine yeah pardon it's fine here yeah um I don't know why mine     
8:58     
does this     
9:05     
anyway all right so yeah thanks for that update um yeah you should you should     
9:10     
present a little bit more intense secrity when you get a chance isn't that'd be great yeah especially this a     
9:17     
factor the strings versus the rods strings     
9:22     
strings over rods yeah um because I didn't know it existed     
9:32     
yeah all right so yeah it     
9:38     
is all right so um yeah thanks for that um so yeah today so I want to point out     
9:45     
that tomorrow uh is the annual uh open wor meeting the open General open wor     
9:51     
meeting every November we usually have a meeting where everyone from every     
9:57     
openworm project gets together and talks about their projects you know that     
10:03     
they're uh representing I'm going to give a talk on dorm we got a lot of     
10:09     
stuff for this year we've got you know we've I'll just give a quick short walkr     
10:15     
of this um so this was the 10th year of evil worm and of course I did a talk     
10:23     
earlier this year about 10 years of computational Developmental biology and     
10:28     
kind of you know what's been going on in the group for 10 years and where we might go     
10:34     
in the future um and then you know kind of reviewing the beginnings of the     
10:39     
project in April 2014 and then talking about some of the     
10:44     
Google summer of code stuff we've been doing this year and you know they always enjoy hearing about that um we had two     
10:52     
projects this year contributing to this D graph repository in the dvo Lear     
11:01     
organization um talking a b about picky's work on the Kepler mapper algorithm and that's that visualization     
11:09     
is a lineage tree so we have you know this sort of topological data analysis     
11:15     
approach we have these neural developmental programs that we forked uh and you know these are so     
11:22     
these are the two things that got forked for that project and you know I'm still thinking about what to do for next     
11:28     
year's if we should continue with stuff like this or maybe the hypergraph stuff or     
11:35     
maybe something else so this was the the the Pocky project on the H mapper and     
11:43     
NDP uh methodologies then you know we gave some lectures uh here's this lecture     
11:50     
dimensions of morphogenesis which was back at the beginning of the year mathematics a diva worm and then D.A     
11:57     
learn the 10,000 me viw I actually put together a talk uh going over a dvo     
12:03     
learn and where we've been over the last like five years and there have been a     
12:09     
lot of people contributing to this so this was a really interesting overview     
12:14     
of this it's probably somewhat out of date now but that's the way these things     
12:20     
go uh we also have the 20124 papers we have the paper we got into BIOS     
12:27     
systems uh several months ago on differentiation waves so the molecular basis of     
12:35     
differentiation wave activity in every Genesis okay Bradley yeah yeah comment     
12:41     
on this uh one thing we're lacking is a simulation of how a differentiation wave     
12:49     
could propagate from one cell to another okay okay and it seems to me     
12:58     
well buy a lot of this soft matter stuff uh it seems to me that the uh it could     
13:05     
be done in that sort of context it could be uh modeled either molecularly with     
13:10     
the cell State splitter or just by assuming that a cell can contract or expand yeah when a cell changes its     
13:19     
apical width or apical area it undoubtedly has an effect on its     
13:24     
neighboring cells which may cause them to do something similar     
13:30     
okay so exploring how you get a wave that propagates over a tissue     
13:36     
and do the whole tissue even though it's a stochastic     
13:41     
process uh would be very interesting and apparently no one it's looked into this     
13:46     
prop basically from Susan's point of view it's a propagation of a wave in an     
13:52     
epithelium okay okay and uh there's I've been     
13:58     
looking uh uh we're doing a book on uh the period     
14:05     
on Earth called the boring billion oh yeah which is     
14:11     
1.8 billion years ago to 08 okay yeah and this is a time during     
14:19     
which epithelia might have formed uh uh UK carots may have formed uh     
14:27     
multicellular organis it's very boring     
14:33     
yeah only only the major factors of life okay so uh at any rate if you have     
14:42     
any thoughts on that I might want to get involved in the book yeah that would be interesting I mean that's kind of where     
14:49     
we left it in this paper so you know we kind of got to like some sort of examples but not like uh going towards a     
14:57     
simulation the simul of course you know would be of course soft matter would probably be a very good approach because     
15:04     
that's what a lot of other people were you know kind of thinking about um in     
15:10     
terms of like sheets of cells and things like that uh the cell State you know kind of incorporating or having like     
15:17     
rival simulations with something like a cell State splitter would be interesting too and see how they kind of model those     
15:25     
sorts of waves so yeah and and so this paper you know we talked about the molecular basis for it and then of     
15:32     
course you have the phenomenon and there's still somewhat like you know disjointed like we don't have the we     
15:39     
didn't get the definitive answer in this paper we got some interesting results but it's still you know uh what is thing     
15:46     
that's driving this there there's sort of three questions how do you start     
15:53     
away uh how does it propagate and how does it desp it's being stochastic cover a     
16:01     
whole tissue yeah okay it's it's sort of sort     
16:08     
of like it's analogous to uh you're familiar with the simulations of uh fire     
16:14     
spreading in in a Woods yeah yeah it's sort of like that if you     
16:20     
have a given piece of woods the fire will eventually get most of it or all of     
16:25     
it yeah uh and this this is somewhat similar under what conditions does that     
16:32     
happen uh the the model for the contraction     
16:37     
waves the stretch activated contraction and that itself should have     
16:43     
some parameters and it should propagate under certain conditions and not others     
16:49     
right on the on the parameters so I I think it's worth     
16:54     
exploring and apparent nobody's done this W waves in epithelia     
16:59     
huh confluent tissue will definitely um propagate a     
17:06     
mechanical um mechanical wave or mechanical indentation like it's it's uh travels a     
17:15     
lot faster than defusion like it's um paper about it     
17:23     
somewhere um yeah fluent tissue and if it isn't nonfluent tissue then the     
17:31     
mechanical wave is stopped     
17:37     
or under which it stops also or not yeah well when there's some in development     
17:45     
for instance the zebra fish tail is becomes a non-confluent     
17:50     
tissue like it it breaks up it's actually oh yeah it actually breaks up     
17:58     
so there's in development there's both confluent tissue and non-confluent     
18:04     
tissue okay okay so you know basically we have     
18:10     
we have a model for it but it's never been tested its parameters have work out     
18:16     
yeah in the cell State splitter it it looks like my model and I making my model from um EP     
18:26     
epithelial tissue in the way it is with that with the contraction ring on     
18:32     
the top yeah yeah and and you look at any     
18:38     
any cell in well not any cell it propagates the qualitative model is if a     
18:46     
cell contracts at its ail gland it stretches a neighboring cell and that     
18:52     
triggers a presumably a stretch activated contraction similar to that     
18:57     
fing smooth nothing yeah that's but that's all we know     
19:03     
and it's just qualitative but it's obviously has a set of parameters that could be tested it's the when it works     
19:10     
and when it doesn't yeah well I have a a picture of     
19:16     
a epithelial tissue in my thesis from Cold Spring Harbor and it     
19:24     
shows microtubules um just under the surface     
19:29     
oh yeah and then I have some papers that say that actin     
19:36     
continuously uh contracts like the um top of the cell continuously has     
19:43     
contractions of actin and that helps circulate the actin in the     
19:51     
cell anyway okay well any rate the     
20:00     
problem nobody's tackled it might be quite interesting     
20:06     
yeah yeah sounds it sounds like it could be yeah I don't think it's hard probably     
20:14     
not but you never know yeah yeah yeah trying     
20:21     
to make a t segr that works is always fun oh     
20:27     
yeah okay so yeah that was the kind of the first     
20:33     
paper well this is more about the paper I'll tell them about the comparative developmental genomics approach where we     
20:41     
did you know things as C we sort of started in cgans we did things that so     
20:47     
we did Pro proteomic analysis across celegans dropa Mouse and yeast and then     
20:55     
we did a further uh genetic analysis within CL elans and then compared to the     
21:01     
out groups of drop Mouse and East and then looked at the sort of the     
21:06     
differences and similarities of those systems so it's interesting because you     
21:13     
know each of these organisms have different modes of development so Mouse of course has the Mamon mode of     
21:19     
development and uh dropa has uh motor development different than seans     
21:26     
although it's similar to celegans uh in a number of ways it you know as     
21:31     
opposed to Mouse uh development and then of course yeast even has a developmental     
21:37     
stage where the cells kind of change shape and they have budding and that's     
21:42     
that's sort of considered a developmental stage maybe quasi developmental the point is is that these     
21:48     
all exhibit different types of development and you can see in those candidate proteins how their abundance     
21:56     
changes and so forth so that's that's the one part of this that's interesting reason I'm mentioning this is because uh     
22:04     
Lucas who was an author on this paper uh did some a project for uh a student ship     
22:11     
and I'm going to talk about that separately um you know some of the work he did from this paper he he kind of     
22:18     
brought together in a repository so that'll be interesting these are the networks that you know uh     
22:25     
different networks that were derived in the paper so so there are different ways that we can look at uh the canidate     
22:32     
genes and celegans and um you know look at what how they function as like so the     
22:38     
candidate genes for different proteins how they kind of uh get     
22:43     
expressed uh over development so that required the proteomic data some genome     
22:50     
sequence data and then also some gene expression data so it was a lot of data integration there to build these     
22:56     
networks or these these uh models of you know what's going on in a in a to     
23:04     
produce a certain protein so this is all I kind of go over this in the in the um     
23:09     
presentation in a little bit deeper level um then this is our map here so     
23:16     
this is where we took the uh genes and SE elegans and attached like a protein     
23:23     
identity and then looked at their expression level and development and built sort of a     
23:29     
surface uh in development so it's like a developmental surface of these different     
23:34     
genes as they're expressed and what proteins they represent and so that's     
23:40     
that's kind of an interesting surface doesn't really tell us a lot about the     
23:45     
mechanism behind the um waves but it does give us some sort of starting point     
23:52     
and then this on the right is a model of the roph imaginal disc which will become     
23:59     
the compound I you see this process of differentiation going across the     
24:04     
imaginal dis along this Furrow so the furrow is in the middle on the right you     
24:10     
have the omia which are the differentiated cells on the left you have the undifferentiated cells and the     
24:16     
idea is that the wave goes through the tissue and activates the cells and then     
24:21     
you get this differentiation behind it which are these omia this is just a     
24:26     
demonstration of this but this is I mean these things aren't really linked but     
24:32     
that's where the modeling would come in I think that would be a good way to do that okay uh so then this is the other     
24:40     
paper uh hypergraphs demonstrated an asmosis this is something that actually     
24:45     
when um mahul was applying for his gso project he read this paper and that's     
24:51     
how he got interested in the hypergraphs world so that's why we did that this this last summer you that's why I got I     
24:58     
was able to get mahul and we did a lot of interesting stuff over the summer but this paper was written in 2020 late 2023     
25:06     
and it was kind of born out of some of the network science presentations I was doing at some of the network science     
25:12     
conferences uh in the last few years so this is an idea that's been developing     
25:18     
basically the idea is you have these hypergraphs with uh with where they're     
25:23     
hyper nodes with nodes inside of them and then you have these hyper edges with different Ed edges inside of them and     
25:29     
they represent different levels of a network so you have a regular Network Within These hyper     
25:36     
networks and you have these hyper uh edges which contain edges that go to     
25:42     
different hyper hyper nodes so you know you can represent a lot of biological phenomena in these you can represent     
25:49     
thing all sorts of things but especially development where you go from these kind of uh you know uh sh roughly shaped     
25:58     
objects to these things that have you know a structure and asmosis different     
26:04     
sorts of modules and so forth so down here this picture is of the zebra fish     
26:10     
heart forming so this is like this process that you could model with a hyper     
26:16     
graph uh then this is a hyper graph sort of based on celan's embryogenesis demonstrating the     
26:22     
different tissue types that emerge um then this is Mah's work that     
26:28     
he did he presented this in the group uh over like in late summer where he was     
26:35     
working out some of these different ways to do uh spatio temporal decomposition     
26:40     
so this is based on the celan's um uh lineage tree and you can see that you     
26:47     
know the identities of the cells are there and you get this kind of interesting structure both at the     
26:53     
lineage in terms of a lineage hypergraph and a spatial hypergraph so the lineage hyper graph is spatially invariant it's     
27:00     
just analyzing the lineage tree the ancestor descendant relationships mother     
27:06     
daughter relationships in the lineage tree and then plotting that out as a hypergraph the spatial hypergraph takes     
27:13     
the spatial locations into account and plots them differently and it's hard to see the     
27:19     
labels and everything here but you have these concentric circles around each hypernode and those are showing kind of     
27:26     
how those things are connected so you don't see a graph here you see a bunch of nodes large black dots with     
27:34     
concentric circles around them and those concentric circles are are sort of suggesting or describing potential     
27:41     
connections between them so that's that's kind of that work and there's one     
27:46     
more slide here and this is a closeup here where you can see the concentric circles     
27:53     
around the hyper nodes and how they might be connected so this is uh all     
27:58     
kind of you know this is even a bigger closeup where you can see the uh lineage tree identity markers on the hyper nodes     
28:07     
and this is taking in a lot of data so this is actually over many embryos so     
28:13     
it's a little bit kind of a weird way to do this but um you begin to see kind of the structure of many lineage trees kind     
28:21     
of laid on top one another or you know many lineage trees with cell locations and it's kind of     
28:28     
discarding that uh motherdaughter relationship okay so we we continued     
28:35     
with our weekly meetings uh I have an archive of these on on GitHub so you know I was just     
28:41     
always I'm always amazed at how much we've covered over say the last four or five years and we have all sorts     
28:49     
of um you know all sorts of topics that we've covered and really just kind of     
28:56     
you know everything from multif physics modeling to Tegrity to complex networks and     
29:01     
hypergraphs course Medical Imaging biophysics cellular and tissue     
29:08     
Dynamics computational models like cellular autometa AG based modeling uh swarms and Collective     
29:15     
behavior solons and differentiation waves transcription translation so there's a lot of stuff that we've     
29:21     
covered and I'm I'm very proud of that breadth of of topical um interest and so thanks to our     
29:30     
contributors we have that's our list of contributors for the year these are people have attended at least one     
29:36     
meeting and you know sometimes even more but I just wanted to give them an     
29:43     
acknowledgement so that's a very quick overview of that um and I'll be     
29:48     
presenting that tomorrow at the general meeting so I like the idea of doing a     
29:56     
simulation like that I think that's missing and it would definitely like you     
30:01     
know give us some a stronger results in terms of you know showing that and then     
30:07     
showing that in terms of what it would look like in a tissue and you could add on to it so you could start with a sort     
30:14     
of a biophysics simulation like a soft material simulation or a you know     
30:19     
something like that and then move out to the molecular aspect or to you know     
30:25     
tissues like a like almost like a differentiation tree where you have different tissues that     
30:31     
differentiate um and so yeah I mean it's interesting yeah uh Bradley by the way     
30:38     
I've been uh looking at a number of papers on uh with the title what is a     
30:44     
cell type oh yeah yeah and they are gradually coming to the realization that     
30:51     
you need a tree to describe that right okay so uh     
30:59     
uh again there might be it might be worthwh consolidating this stuff and say     
31:04     
okay it's time the different differentiation tree as a as a way of     
31:09     
defining cell types yeah instead of molecularly yeah well I mean you can I     
31:17     
mean the characteristics of a cell type are multitude and but the only way you can really understand all of that is     
31:23     
through a tree because you know you have like all sorts of different traits and the relationships are different based on     
31:30     
like if it's some set of gene expression markers or some set of     
31:35     
morphological Properties or some set of Transformations it's going to be a     
31:41     
different answer I guess so oh yeah they're many different answers and uh but they're getting very frustrated it's     
31:50     
obvious yeah a lot of the people who do a lot of these I throughput studies they're very good at that but then     
31:57     
they're like well how do we synthesize this their     
32:05     
hands just that they're actually mentioning trees as a way of doing it yeah     
32:11     
yeah yeah I mean that's yeah that would be probably the best way I guess     
32:21     
maybe I'm interested in the um um graphs hyper graphs for um the     
32:33     
mechanical portion of of a cell like I'm interested in how things are linked     
32:39     
together because there doesn't seem to be any data on how um what the area of the strings that     
32:49     
I'm using are and the or the area of the rods in my model and there doesn't seem     
32:57     
to be any information on the elasticity of the acting bundles like they they     
33:05     
isolated acting they figured out what the elasticity of it is but it's the     
33:12     
bundles are interlaced with other other molecules um you probably know them     
33:18     
better than I do but okay so where's the data on the elasticity like it's it's     
33:25     
starting to bother me because this K factor is very dependent on uh the     
33:32     
linkages between these bundles or linkages between the     
33:39     
microtubules and then the uh what the linkage um elasticity is like I have no     
33:47     
idea yeah and I tried to go in there with     
33:52     
some realistic data and I'm finding it's become extreme     
33:58     
to get uh epithelial tissue uh properly I'm using strings that are are um     
34:07     
elasticity of rubber and the rods are the elasticity     
34:13     
of Steel to get the right     
34:19     
um I don't know get the right strain curve     
34:29     
yeah and that's not biologically possible uh yes it is it     
34:35     
is just barely which is right on the edge of not     
34:41     
being real because tensies have to be stiff or they don't     
34:48     
propagate um mechanical stress     
34:54     
properly um Steven Len this is the main thing that I took away from talking to     
34:59     
him is that if it's an elastic and a rod     
35:05     
you will get a squish not a tensity okay the STS have to be     
35:14     
stiff in order to have a tensity that will propagate your mechanical     
35:22     
signal um I have a squish here and I have um I took some fishing     
35:29     
line and made one of my 10 segres into a proper 10 segr like my my little models     
35:37     
that I have here yeah and it's one one acts as a Squish and the     
35:44     
one with the um nylon uh     
35:49     
strings acts like a tensegrity I can show you if you want oh     
35:55     
yeah go ahead okay so this this is this is the dusty     
36:00     
squish right here okay and if you push down     
36:07     
push down on it or maybe I'll do it this way across yeah this is there's such a delay     
36:15     
on this thing okay so you push push on     
36:22     
it yeah if you can see that but if you push on it it's it just pushes part of     
36:29     
it and it doesn't really you can see the top two um balls there they're close     
36:36     
together yeah it hasn't really changed the structure that much a little     
36:42     
bit but if you this is the one where I     
36:48     
changed the elasticity of the strings and I don't know if it'll work     
36:56     
but okay well it's hard to see yeah on on here but um     
37:05     
sure the whole thing changes like the whole see the the whole thing is wider     
37:11     
and the whole thing is squashed together right and I'm just working with two     
37:19     
points yeah like that and the other one doesn't do     
37:25     
that What's the diff I mean what's the difference do you     
37:30     
think see that that sort of works right     
37:35     
but not as the other one I don't know if you can notice that it it really doesn't work as well and that's because these     
37:42     
are elastic the strings yeah the strings are elastic these where     
37:49     
I have my finger right that's an elastic right here the black     
37:59     
and over here I don't know if you can see it some of them are falling off but     
38:05     
I put um fishing line oh okay yeah there and because     
38:13     
these elastics actually were getting old right but I put     
38:21     
um the fishing line on it yeah and see the line and that's a     
38:29     
lot stiffer than the elastic so that's how I fixed that and     
38:36     
that was from talking to Steph Len oh okay yeah     
38:44     
yeah and and did too but it was this that's a Steven Len     
38:51     
contribution to things so I whether the rubber strings     
38:58     
are a good idea but um to get my model     
39:03     
to work properly I'm going to try it this afternoon     
39:11     
yeah to get a small K factor which one is a small K     
39:18     
Factor U the um elastic um elastic     
39:24     
strings okay as they have a smaller     
39:30     
elasticity but I hope rubber is is an okay elasticity for this you'd think it     
39:37     
would be but I'm not sure so I'd really like to know about these networks and     
39:43     
cells and whether um some are like uh     
39:48     
rubber elasticity and some are like steel elasticity and it on     
39:57     
I don't know it's conceivable that they're like steel because they're microtubule networks and they're     
40:05     
held by um the rest of the cytoskeleton so it's conceivable that there's some     
40:12     
really stiff um the parts there um that thread     
40:19     
through thread through the network in a Cell I don't     
40:24     
know I I'm interested in the network s and how they change yeah has a     
40:32     
consequence yeah that sounds like a really interesting question     
40:38     
um have looked at Dynamic instability     
40:43     
um yeah I had a look at it     
40:48     
um because if you make a Integrity structure with components are in Dynamic     
40:54     
instability then you get the question what is the whole structure     
41:02     
stable it has to be stable in order to uh model it in a finite     
41:09     
element um model okay then we need to generalize it for dynamic     
41:16     
instability where the individual components are not stable but is the overall structure     
41:22     
stable uh that that goes I got interested that in what of under what     
41:29     
conditions is the cell State splitter stable because all all of its components     
41:36     
may be fluctuating yeah that's true so I've decided that when I'm actually doing is     
41:44     
one snapshot in time yeah okay but then the the we     
41:51     
interesting this seg structure with that's not true     
41:58     
I don't know um using finite element analysis you have to have a very stable     
42:04     
Tegrity so what I used was what is called a T4 um     
42:12     
tensegrity and I used it for basically the whole structure I used it for the     
42:19     
the middle um Tri Junctions and I used it for the general     
42:25     
shape of the tagr yeah but might have to get something     
42:33     
better than final elment yes that's what I've been saying     
42:39     
yeah like for a master's thesis just gonna go ahead with it     
42:50     
okay yeah the the network connectivity thing is pretty interesting so I me we     
42:56     
could have like you know different sort of     
43:01     
topologies of network and look at like different sorts of properties for the     
43:07     
the edges so the nodes would be kind of you know components that are connected     
43:12     
together the edges then would have like some sort of set of mechanical properties elasticity or whatever and     
43:19     
then you know you might have and the nodes would probably have to be in some     
43:25     
spatial spatially explicit configuration and then you'd have to curve them in     
43:30     
some way so you'd have to introduce some external forcing or something and then see how they behave both in terms of the     
43:39     
mechanical properties but also in terms of the topology to see yeah     
43:45     
yeah yeah the uh uh with the dynamic and stability     
43:52     
there each Edge for example can be get getting longer or     
43:59     
shorter right that's also you have to go one more step when     
44:06     
they're connected to each other they might be connected via motor molecule so that the modes are actually     
44:14     
moving along the edges uhhuh yeah it's a motor molecule I mean     
44:19     
it's myin non-muscle mein doesn't matter whether it's     
44:25     
um non-muscle or not it still um moves the same way it's a     
44:31     
ratcheting yeah but I'm saying it can be connected the masin itself could be at     
44:37     
the node and therefore oh two     
44:42     
components yeah one Edge relative to another well two yeah two components     
44:49     
would be good because you can can um model that Wonder know if     
44:55     
that's been demonstrated can a motor molecule well I guess it     
45:00     
can a motor molecule in order to move something has     
45:05     
to move along something and carry     
45:11     
something I don't know in anyway um what was I going to say     
45:18     
um uh the configuration of the     
45:24     
uh Integrity inside is changes     
45:31     
um when you compress it because it'll make more attachments     
45:38     
and um I don't know when they measure when     
45:44     
they measure tissue it will change so they're they give it one compression and     
45:51     
measure and then they confess it again about five times and about the fifth     
45:57     
time it stops changing um the stain curve     
46:04     
slope so there there's a first time and then you compress it a bunch more and it     
46:15     
changes okay uh in the myosins are there any meins that move     
46:22     
along an active and also carry an active     
46:29     
I don't know if anybody's measured [Music] that I don't know we one way to make a     
46:35     
bundle you move to the end of an act and you flatten you you make and as a result you     
46:42     
make to get things parallel yeah when you when you flatten     
46:49     
things it it makes things parallel yeah it kind of it stretches them out yeah     
46:56     
yeah so so there might actually be a role for B that can both walk on an act     
47:02     
and carry an act okay maybe um I should I should hand you my     
47:10     
my um Library your library I have a library     
47:16     
yeah I need to go to the library and get them to help me categorize it     
47:29     
I have three bookshelves full of it because I printed out the most     
47:34     
important papers so I wouldn't lose them important papers because there's     
47:41     
also a print copy of [Music] them that's why dog     
47:49     
print never get filed back I can't read them on the screen     
47:57     
very well so I print off as I'm supposed to be [Music]     
48:03     
reading yeah read if I can't find them well that's true I have them in um the     
48:10     
the orange binders and yellow binders are the 10 segres and then I have blue     
48:16     
and and pink and purple purple is     
48:23     
embryology I don't know why yeah     
48:30     
yeah I'm interested in that Network inside cells because of some of this     
48:37     
yeah there's no data there's no data but we we can therefore how should I say     
48:46     
open to speculate yes I guess that's um a plus I     
48:54     
can just guess and say well there's no data so I'm trying     
49:01     
this yeah people estimate things all the time I     
49:07     
mean anyway I don't know the whole thing is um pain     
49:15     
but um I'm looking forward to the postmasters yes yes     
49:26     
yeah well I'll do a postmasters rather than a post talk that sounds fineing to me     
49:39     
yeah so okay that's that's about all all I've today other than Dusty T     
49:49     
wow did it ever get bad yeah well you don't you have occasion to use them that much so yeah at least not     
49:57     
the one that doesn't act like a t seity yeah sorry yeah making making uh making     
50:04     
lunches here okay all right so that's all for today     
50:11     
uh thanks for attending and you know I'll let you know if there's any I mean     
50:16     
usually there is pretty good interest in what we're doing in the annual meeting but if they have any specific things     
50:22     
that they uh want to inquire about I'll bring it up here     
50:29     
um and otherwise it's it's good so see okay yeah BR Bradley do you remember     
50:38     
that silly movie where uh motor molecules are     
50:44     
moving carrying huge burdens compared to their size yeah and they're moving to     
50:52     
Beethoven oh I think I've seen that yeah     
50:57     
okay but is the huge burden is another cytoskeletal molecule instead of a uh a     
51:06     
vesicle uh then uh it opens up the possibilities     
51:12     
tremendously oh okay yeah was then the Integrity structures are continually     
51:21     
changing yeah okay because uh     
51:27     
you know what and     
51:33     
what the structure is dynamic that right we don't just get time time     
51:40     
snapshot but uh if we would simulate the whole     
51:46     
motion but the question is does that does this     
51:51     
happen do any motor molecules carry cyos components     
51:58     
well they're certainly attached to um masin     
52:04     
[Music] um Myas sorry they're um they associate     
52:11     
with each other mein's attached to Acton but get a connection actin to mein to     
52:19     
Acton okay probably yeah just like they they     
52:25     
associate with each other they     
52:31     
um what can I say they always seem to be together well we need we need some     
52:39     
hope well I can show you some well in my library I have a paper     
52:45     
showing where the Acton and Myas and are in cells okay but you don't know what's     
52:52     
attached to what just because you know where they are doesn't mean they're     
52:59     
attached yeah I don't know they they say they're     
53:05     
attached that ring the tri Junction start with the tri     
53:11     
Junction in cells in there's Tri Junctions um in     
53:19     
epithelial cells okay what's that the tri Junction     
53:29     
um you want me to try to find it sure sure I'll even read it I     
53:40     
try okay well I have um     
53:48     
oops I have whole a whole PowerPoint     
53:53     
called some cell biology and it has um over 200     
54:01     
pages okay well send it to me I'll skim through it okay I'll just I'll just find the     
54:08     
slides that are acting in my and um okay I don't have it in the one     
54:15     
that's right here um so I I'll I'll find you I'll put     
54:24     
together a bit of a PowerPoint and maybe that can be for another day     
54:30     
since since this is kind of the end of the meeting I'll um yeah I'll find some acting and     
54:38     
biasing for you oh stupid question for     
54:43     
microtubules you get bundles of microtubules anywhere besides INF     
54:48     
fella you get bu bundles of microtubules inside cells well you do what holds them     
54:55     
together are they held together yeah um various     
55:02     
molecules various molecules yeah any M     
55:09     
molecules s are any of those motor     
55:14     
molecules um is there some masin in there um I don't know not my     
55:21     
well I no be kindan or dining oh I I see um I don't know don't     
55:31     
know I don't know I have an adherance Junction here and it's got um     
55:39     
actin and necton and eerin and beta caterin and Alpha caterin and golden     
55:47     
Rock and Myas and 2 attach like you want     
55:52     
that one I I'll just send the sure there put let me put it in the chat     
56:01     
okay find the chat and put it     
56:10     
in okay this paper there's one paper there's a whole     
56:15     
bunch of them okay thank you yeah yeah that's a     
56:23     
lead Fair good lead is it's recent paper okay now SC through     
56:30     
that PowerPoint okay well all right well I'll I'll I'll get     
56:37     
you some nice references and I'll show you I'll I'll get something     
56:44     
together okay thanks thanks a lot all right see you next week thanks yeah bye     
56:51     
bye bye so just a quick recap of some of the things we been doing and to talk     
56:56     
about another paper as well so so I was writing some pretty     
57:03     
extensive notes in some of our discussion I was going over some of the things is capturing them as I'm along so     
57:09     
I have a lot of notes here to synthesize for later I usually do this on a whiteboard you can see uh this case we     
57:16     
have um you know this this model of the sheet of cells where we might end up     
57:24     
modeling uh differentiate waves expansion and contraction waves building     
57:29     
a simulation having like a molecular component a soft materials component and     
57:36     
a propagation component so it's just a matter of kind of thinking about that and sorting out all the components of     
57:42     
the simulation and building this is like uh Susan's equation K I don't think I got that     
57:49     
right but that's something that um we'll probably go over later     
57:56     
uh talking about hypergraphs about the different materials of the edges on the network     
58:01     
elastic versus fishing line so you know that would be something we can Implement in a A hypergraph or in a graph     
58:09     
theoretic uh context and we I think we did Cover a paper where they tried to do     
58:15     
that with actin meios networks and and sort of abstracting them out to a     
58:20     
mathematical graph the difference here would be we can model multiple components in a node     
58:26     
we can model multiple aspects of the edges in hypd so you can have composite     
58:33     
uh material properties of an edge composite properties of a node so you     
58:38     
can have a lot of processes going on with the hyper node and a lot of different aspects of the     
58:46     
edge uh this is kind of like a early rendering of this kind of hyper graph I     
58:53     
I would you know this will be this would probably evolve significantly from this     
58:58     
image but it's just a pairwise uh I Jake     
59:04     
relationship and then this is uh talking about Tri Junctions modeling this at the     
59:09     
tri Junction all right now I wanted to get into two papers uh the first is the one that Susan mentioned in the meeting     
59:16     
she put it in the chat and then the second one is uh follow up on another     
59:21     
paper we talked about several months ago uh it's the live stream     
59:26     
materials paper we'll see what that looks like so the paper Susan uh put in     
59:32     
the chat is this paper that adherens Junctions as molecular Regulators of emergent tissue mechanics this was from     
59:40     
nature reviews molecular cell biology and this is um basically uh     
59:47     
having to do with these uh mechanical networks particularly adherence Junction     
59:53     
so Susan has a lot of lever review on this and in subsequent meetings hopefully we'll kind of sort through     
1:00:00     
some of this a little bit more so uh let's read through the abstract and then go through the paper     
1:00:06     
maybe a little bit so the abstract reads tissue and organ development during     
1:00:11     
embryogenesis relies on the collective and coordinated action of many cells recent Studies have revealed the     
1:00:18     
tissue material properties including transitions between fluid and solid states are controlled in space and time     
1:00:26     
shape embryonic structures and regulate cell behaviors so this is where we're     
1:00:31     
looking at transitions between different states so you know we have to in our     
1:00:37     
Network we have to think about what the material state of the network is or of specific nodes or specific edges so     
1:00:45     
that's something you know again with the mathematical graph you know you have to     
1:00:50     
it's kind of hard to implement unless you use something like a weight or something like a um value that gets     
1:00:56     
attached to that item so this is uh you know but but the idea of a mathematical     
1:01:02     
graph is that we can put all of these things in a set of relationships where we have these EJ     
1:01:09     
relationships and then we have that as some sort of indicator of connectivity     
1:01:14     
how are our nodes and and a matrix related to one and what's the strength     
1:01:20     
of that relationship and so the material properties can serve as that sort of strength or weight     
1:01:27     
at that time tissue material properties are important in terms of transitioning     
1:01:32     
to certain macro States or having the state of like in a local state of of the     
1:01:37     
of the network and so this is important for shaping embryonic structures and     
1:01:43     
regulating cell Behavior so sometimes if cells are connected through some uh you     
1:01:49     
know uh material property they can actually change their state so it's     
1:01:54     
often the case that if a cell on a certain type of substrate it can affect its fate or its diff of course a     
1:02:02     
differentiation so if you have sort of a rough substrate versus a smooth substrate but also different materials     
1:02:08     
that it interacts with can affect that as well um although the collective cellular     
1:02:13     
flows that sculpt tissues are Guided by tissue level physical changes these ultimately emerge from     
1:02:19     
cellular level and subcellular level molecular mechanisms so we have our uh     
1:02:26     
tissue level physical changes so we can have a macro state that changes from a solid to a liquid or you know in soft     
1:02:33     
materials language you know we would have some sort of phase transition um but those ultimately are     
1:02:41     
reliant on sort of the single cells or the representations of single cells     
1:02:47     
what's going on in those and then at the subcellular level ultimately the     
1:02:52     
molecular level so this is where this is the thing that we're trying     
1:02:57     
to well in the paper that we did in 2024 that's what we were trying to get at and     
1:03:03     
really did not have a strong connection with that's not that you can't say anything about it without that strong     
1:03:09     
connection it's just that we would like to have a model that sort of demonstrates a molecular change is     
1:03:15     
responsible for a cellular change and then perhaps it was responsible somehow for the macro State and understanding     
1:03:23     
that that's an emerging process and it's not additive uh is important but also that you can     
1:03:28     
trace some of these changes downward adherence Junctions are key subcellular     
1:03:35     
structures so this is where they focus on adherance Junctions built from clusters of plastical cadherin receptors     
1:03:43     
they mediate physical interactions between cells and connect biochemical signaling to the physical     
1:03:49     
characteristics of cell contacts so we're talking about taking this biochemical signaling pathway which is     
1:03:55     
one level of a model and connecting it to the cell contact so Susan was talking     
1:04:01     
about Tri Junctions and CIS where becomes important where you have these Tri Junctions where three cells come     
1:04:07     
together and you have this biochemical signaling that mediates that process     
1:04:13     
certainly it's not something you can incorporate easily into a mathematical     
1:04:19     
graph of cellular structure or of these Junctions but it's got to be a uh sort     
1:04:25     
of a sub cellular process so this is where our U hypergraphs come in because     
1:04:32     
we can actually have each node at multiple components we can have these molecular components in in a hyper node     
1:04:39     
the hyper Edge can then model the strength of the relationships based on the different molecular     
1:04:47     
relationships so if you have say for example an adherence Junction in one hypernode versus an adherence Junction     
1:04:53     
in another hypernode that might be a strong connection whereas the identity     
1:04:58     
of the cells might be a weaker connection uh molecular signaling in general might be a moderate connection     
1:05:05     
from that you can glean some of these relationships uh it's not that they     
1:05:11     
would be averaged out but it's just that they're multiple competing influences     
1:05:16     
and so you know this is something we have to deal with in terms of uh emerging structures is that you have     
1:05:22     
different types of processes running simultan ously and you have to sort of     
1:05:28     
sort out the relative contributions of those things sometimes things will     
1:05:33     
dominate sometimes things will be sort of part of the mix but getting that     
1:05:39     
better characterization of things in terms of emergences okay so we're talking about     
1:05:45     
adherence Junctions we're talking about uh how they mediate physical     
1:05:50     
interactions they connect biochemical signaling and physical characteristics of the cells and that means that they     
1:05:56     
play a fundamental little tissue M Genesis so we can go from these components of the cells to the cells     
1:06:02     
being connected to to one another to these macroscale states of tissue our MOG     
1:06:08     
Genesis in this review we take advantage of the results of recent quantitative measurements of tissue mechanics to     
1:06:14     
relate the molecular and cellular characteristics of adherence Junctions including adhesion strength tension and     
1:06:21     
Dynamics to the emerging physical state of embryonic tissues we focus on systems in which cell cell     
1:06:28     
interactions are the primary contributor of morphogenesis without significant contribution from cell Matrix     
1:06:34     
interactions so this is where we have you know systems where we can actually just see these cell cell     
1:06:41     
interactions on not thinking about the Matrix and kind of par parceling that     
1:06:47     
out uh we suggest that the emergent tissue mechanics are an important direction for future research bridging     
1:06:54     
cell biology velmental biology the canob biology to provide a holistic     
1:06:59     
understanding of morphogenesis and health and disease so this is the     
1:07:04     
paper and it's a nice review kind of talks about adherence Junctions of classic     
1:07:10     
adherence so these are you know this is figure one adherence Junctions are specialized membrane domains so we have     
1:07:18     
these uh adherence Junctions here that are kind of bridging two cells uh we     
1:07:24     
have this acist assembly here and then we have this other layer here on top so we have this     
1:07:31     
Junction uh we have the sheeta cells where we have interpolation where the cells move in to one another and they     
1:07:38     
form these Tri Junctions you can see the tri Junctions here here here here here     
1:07:45     
and they're constantly being remodeled as the interpolation process moves so if     
1:07:50     
three cells come together and one cell is moving inward it's pushing the other     
1:07:56     
two cells apart but that Tri Junction is still there and it's moving along some     
1:08:01     
axis this other cell is moving in so at some point it's going to form a quad Junction and that's going to change the     
1:08:09     
properties of this of the tissue so that's really you know kind of the     
1:08:14     
dynamic aspect of it we're talking about the meating where things are constantly changing these relationships are     
1:08:20     
constantly changing and the expression of these uh different you know caterin things are also changing with the with     
1:08:28     
the cell cell environment this epithelial monolayer we     
1:08:33     
have the tricellular adherence Junctions here bicellular adherence Junctions here so we have the corners we have the edges     
1:08:41     
between two cells and that gives us our adherin molecules their context so they     
1:08:49     
can join at a Vertex they can join at an edge so if you think about this set of     
1:08:55     
uh hexagons is a network the mathematical graph you have vertices     
1:09:02     
which are joining of two edges and you have your edges so your bicellular Junctions are the     
1:09:08     
edges the tricellular adherence Junctions are the vertex or the node and     
1:09:14     
that's how that's modeled and so they're different properties at each so in a hypergraph it could probably     
1:09:21     
characterize that fairly well you have different components that the vertex     
1:09:26     
different components of the node and so forth and finally there's this aspect of     
1:09:32     
mechanochemical feedback so we have this mechanical contractility and it's resulting in this     
1:09:38     
change in the biochemical millu and that's how we get our changes in this     
1:09:44     
tissue okay so that was a quick overview of that now I want to talk about this other paper this is following up on we     
1:09:52     
did a uh there was another paper on sauce streaming where we talked about sort of the idea of sof streaming and     
1:09:59     
this is uh from the domain of soft materials this is three-dimensional soft streaming so this is Matia Gaza runs the     
1:10:08     
lab at UI and uh this is a mechanical engineering lab but they also do of     
1:10:15     
course this biological stuff so uh basically what it is uh soft streaming     
1:10:21     
is what they call viscous streaming and so this is kind of talking about about different types of flows uh different     
1:10:28     
types of materials within the flows and this is really relevant to cell migration and these kind of the external     
1:10:35     
environment of these cells that cells find themselves in this external environment of liquid that has its own     
1:10:42     
sort of mechanical properties actually impacts a lot of what's going on in the cell so when we talk about these     
1:10:48     
different conditions we talk about interpolation in a Cell sheet we also have to think about the streaming on the     
1:10:54     
outside so the abstract reads viscous streaming is an efficient rectification mechanism     
1:11:01     
to exploit flow inertia at small scales for fluid and particle manipulation so     
1:11:06     
this is where they're doing this in a lab but this applies to our cell populations as well so we're at a very     
1:11:12     
small scale in in tissues we have fluids and particles and particles are the     
1:11:18     
cells the fluid is the liquid it typically entails a fluid vibrating around an immersed solid feature that     
1:11:26     
by concentrating stresses modulates the emergence of steady flows of useful     
1:11:31     
topology motivated by its relevance in biological and artificial settings characterized by Soft materials recent     
1:11:38     
Studies have theoretically elucidated in two Dimensions the impact of body tic     
1:11:43     
City un streaming flows so there's a two-way relationship there's the streaming flow and then there's the body     
1:11:50     
elasticity of the thing in the flow which affects the flow because you can have things that kind of are sitting in     
1:11:56     
a flow and the flow is affected by it so if you have uh you know rocks in a river     
1:12:03     
it can create turbulence from a lamin or stream and you know also the relative     
1:12:10     
turbulence of the flow can affect the in this case the cell because the cell is     
1:12:15     
taking in these mechanical inputs and changing uh respectively     
1:12:21     
so we have this uh B directional case here we first improve existing solutions for     
1:12:28     
the rigid sphere limit by considering previously on counted terms we then enable body compliance so this is where     
1:12:35     
you know we have the the body is has a certain sort of flexibility or a certain     
1:12:41     
um you know elasticity exposing a three-dimensional elastic streaming process available even     
1:12:48     
in Stokes FS such effect consist of a two-dimensional analysis an analytically     
1:12:53     
distinct is validated direct numerical simulations and shown to translate to     
1:12:59     
bodies of complex geometry and topology having the way for Advance forms of flow control so we have our geometry of our     
1:13:07     
cell we have our geometry of our cells in a sheet that means multiple cells that interact and then we have our     
1:13:13     
networks uh which are how they're configured together how there's like you     
1:13:19     
know how they exhibit adherence to one another how they stick together and     
1:13:25     
ultimately it can affect the cells the shape of the cells and perhaps even     
1:13:31     
their fate so we can look at flow control as a way to kind of get around this flow control in the mathematical     
1:13:39     
graph would be sort of buffering the topology of the graph so the topology of     
1:13:44     
the graph would influence the flow the flow would influence the topology and that's sort of an external variable for     
1:13:51     
a mathematical graph in other words it would act to sort of influence uence it and shape it but not be represented     
1:13:58     
directly in it so this in this simulation this is not in a network this is not in a biological context this is     
1:14:05     
just simply a sort of an isolated model of a flow in a in a set of Po particles     
1:14:11     
so we have our problem set up here uh in a we have a three-dimensional viscal elastic cellid sphere which has a radius     
1:14:20     
and a rigid inclusion so there's this radius and rigid inclusion aspect um     
1:14:25     
and then you know that's how we model this and then B is immersed in a viscous fluid so this B is where you have this     
1:14:35     
uh solid sphere and then in B you have this external field which is the fluid     
1:14:41     
so the viscous fluid is out here where there's an elastic solid fluid interface in a term here that models     
1:14:49     
this so we have uh a spherical coordinate system where we want to     
1:14:54     
measure the reactivity of the sphere uh and then the sphere is exposed     
1:15:00     
to an oscillatory flow with farfield velocity that's the term here so there's a farfield velocity which measures     
1:15:06     
what's going on in the fluid you have an illat flow so the flow is you know is     
1:15:12     
kind of along this oscillation um and then this is going in the X Direction along the axis of     
1:15:19     
symmetry which you can see in a is along the midline of this     
1:15:25     
particle so it's along this X and so it's moving in that direction everything is moving Dimension figure two is     
1:15:32     
elastic sphere and streaming flow response so we have this model here where we have     
1:15:40     
the system with different flows around it and then we have these results uh where we have different types     
1:15:47     
of elastic bodies which we saw there rigid bodies rigid body Theory and with     
1:15:55     
cow values so this is the elastic sphere and streaming flow response um a through C which is up here     
1:16:03     
these images represent a Time average lonian stok drift     
1:16:08     
corrected uh this depicts the streaming response at m equal 6 with increased     
1:16:13     
softness cow so you see that there's an increased softness going in this direction you can see the results     
1:16:21     
there um note that blue orange represents counter or it represents clockwise counterclockwise rotating     
1:16:28     
regions so you see these different regions of rotation in in the     
1:16:35     
experiment uh the non-dimensional radius of the pin zone is set at 0.4 throughout     
1:16:40     
the study maintain the tangential slip velocity magnitude uh the effects of the pinzone     
1:16:47     
radius on streaming flow is detailed in the supplemental material misus     
1:16:52     
streaming theories for a rigid sphere are represented in this graph here's we have this combination of observation for     
1:17:00     
different cow values and theoretical values here so um rigid body Theory     
1:17:06     
according to Lane is here according to Riley is here and then these experimental results come from this and     
1:17:16     
they seem to be kind of different from some of the rigid body theory that     
1:17:22     
exists so this is where we have an IC body and we have different cow values     
1:17:27     
for that basically as we increase the cow value this slope     
1:17:34     
decreases I'm not sure what that means but basically that's what the results     
1:17:39     
are of this paper and then an extension to complex bodies so here we consider     
1:17:44     
compliance and do streaming in a soft Horus so this is just a ring that's kind of in the uh fluid in the Stream but     
1:17:53     
complex shape entailing multiple curvatures and distinct topology relative to the sphere and so they're     
1:17:58     
looking at an oian FL topology for a Taurus of cor radius R in cross-sectional radius AAL     
1:18:06     
R3 at around M4 uh with varying body elasticity     
1:18:11     
C the Taurus is pinned at the center of its circular cross-section by rigid tal     
1:18:16     
inclusion of radius 0.4 and so this paper is you know kind     
1:18:23     
of Deep In The Weeds of streaming theor so you know I don't know how it connects to development it's just to say that     
1:18:30     
like we can model these rigid bodies or these soft bodies and then we can model     
1:18:35     
them in a flow field and we can say things about the mechanical Properties     
1:18:41     
or the mechanical perturbations of say like a cell or how the cell affects the     
1:18:48     
environment which affects the cell again later so their conclusion reads in     
1:18:54     
summary this study improves existing threedimensional rigid sphere streaming Theory expands it to the case of elastic     
1:19:00     
materials and further corroborates it by means of direct numerical simulations so you saw they were comparing with rigid     
1:19:07     
body Theory and it's different from that so that gives you this sort of expansion the case and then you get this uh these     
1:19:15     
numeric simulations which are important to show some of these results so we talked about this I think     
1:19:22     
we talked about this paper on two-dimensional soft cylinders where they did the streaming these streaming     
1:19:28     
simulations on soft cylinders uh an additional streaming mode accessible through material     
1:19:34     
compliance and available even in Stokes flow it further demonstrates how body elasticity strength and streaming or     
1:19:41     
enables it at significantly lower frequencies relative to rigid bodies finally we show how theoretical insights     
1:19:48     
extended geometries other than the sphere highing the Practical generality of our Theory so this understand are     
1:19:55     
fundamental understanding of streaming flows and implications are all biological and Engineering domains so     
1:20:03     
that's a quick recap of where we are with our tensegrity and     
1:20:08     
mechanical morphogenesis Explorations um so if you're interested in participating     
1:20:15     
please let us know thank you for paying attention
