     
Transcript     
0:00     
getting data at least     
0:06     
helpful okay W that sounds very good very super yeah hi Bradley Hi how are     
0:14     
you I am good it's been really long I haven't seen and join     
0:22     
Yeah Yeah well yeah hopefully you can give us an update this week and see where we are     
0:28     
and yeah see so yeah uh well yeah let's start uh so     
0:35     
yeah why don't you start and then we can go to Susan and then hello     
0:47     
Morgan okay I'll share my screen yeah that's good uh so Bradley uh so last time in     
0:55     
the discussion Bradley and I were discussing a data uh which was C     
1:00     
essentially it was this let me open that data also so it's good for that in the name is C raw data um it should be found     
1:09     
in data uh C raw yeah so I'm opening along     
1:15     
with it so this was the data last time Bradley and I were discussing and Bradley actually gave me a lot of     
1:21     
insights in detail and helped me understand uhuh so we have here cell column time column XYZ is the coordinate     
1:27     
of that particular cell and size and size also evolves along with time as we can see and yeah and then uh Bradley     
1:36     
actually directed me he told me that I can find lineage data also so I searched     
1:42     
on Devo open War uh website and I was able to find it and I have used it for     
1:49     
uh finding a daughter sales and parent Sals and all so mother sales yeah yes so     
1:55     
uh so in the discussion today I plan to cover the data understanding or data and     
2:00     
then I have done done some data preparation basically trying to prepare it for the graphs then I have made a     
2:06     
graph data based on the spatial so I plan to create graphs for spatial as well as cell lineage but right now     
2:13     
focusing on uh uh uh the the spatial graph then I also created a spatial     
2:21     
hyper graph as well I have couple of ideas to create and I have shown both the ways if we finalize with one we can     
2:27     
go with the that so proposing the two ways then I have other ways of creating     
2:34     
hyper edges based on the lineage data so I have a couple of ideas and then in the end introducing the dynamic graphs so     
2:41     
that is actually the last part but still keeping those things already in mind we will Design everything so I'll start uh     
2:47     
the discussion all right okay so in data understanding so     
2:52     
this is how data looks like we have cell time XYZ columns and size and I was just     
2:58     
reviewing the data so it is time is an integer float and XYZ is coordinate so     
3:03     
it's float and size is also in float so I just see saw the uh stats of the data     
3:09     
so count is like these many data sets these many points these are rows so uh for every time for every cell we have a     
3:16     
recorded data that is why this is so large I think it's two uh 25 to 100 so that's quite B and     
3:25     
then we have XYZ column so we have mean of it uh something like this I I will     
3:30     
tell how I will use this later on and then for size also correspondingly then     
3:36     
we had one interesting uh time right time column so I was I wanted to see what time column looks like so we have     
3:42     
time from 1 to uh 118 1 190 actually and this is a continuous data I think some     
3:50     
are missing uh actually nothing is missing from 1 to 190 uh I saw something     
3:55     
missing in between but yeah right now I'm not able to I hope it's continuous and probably you     
4:02     
can confirm from one to 190 time point but there was some negative points also     
4:08     
so - 16 -5 -4 and -3 so I saw which particular cells belong belongs to these     
4:15     
time so those are P1 actually P1 only and their size changes little bit across     
4:22     
the time so I was not sure how what is happening here but I think this can be no noisy data probably but uh because I     
4:31     
just because I was not able to understand but probably it might be con something so coming next to size of     
4:37     
cells so just to observe what is the Min and max value of size so the minimum value of a cell is 24 units I'm not sure     
4:47     
I think it's micro some micro Nano size of course and the max size is3     
4:54     
units and then we have frequency of counts so the total number of uh unique     
5:00     
cells are 786 in this particular data and I just did the value counts of it so     
5:06     
trying to observe how many times a particular cells is observed so this Capa CA PA is observed the most number     
5:14     
of times so I printed the top 10 which are observed so this observations is basically number of times it comes and     
5:22     
number of the frequency of that particular cell so multiplied by those we get this     
5:27     
806 so uh to understand this better I plotted this corresponding to time also     
5:34     
so when we see let's say this this is a new cell e e a r a this cell occurs at     
5:41     
at the one zero time it occurs 14 times that means there are 14 instances of this particular cell at the time 102 and     
5:50     
also at 101 it has 14 instances uh in the in the organism so this is how it     
5:56     
looks like so I think this gives me a good understanding of the and this gives us good understanding of data and then     
6:03     
uh yeah so then I wanted to map it along with the lineage data as well so in this     
6:08     
data we do not have the information who is the daughter cell of which particular cell right so for that Bradley yeah so     
6:16     
Bradley Bradley pointed me in the last meeting towards this particular data set I downloaded it uh so it looks like this     
6:24     
yeah I hope my screen is changed now yeah so we have yeah so we have cell name as this A B all these things then     
6:31     
we have location so location is nucleus for everyone I I checked it so not required then we have object subass of     
6:38     
this is also a not very useful column and then relation so relation tells us     
6:44     
daughter of so basically this is the cell and P0 is the mother of ab and ab     
6:50     
is the mother of ABA so that is what it conveys so using these two particular     
6:56     
columns uh uh we can see this columns here so this is the cell and these are     
7:02     
the mother cells of it yeah and then I uh wanted to see how     
7:08     
many data points exist actually I will go more deep into it I didn't do very much exploration of this particular data     
7:14     
so there are 1,800 uh rows so I don't know why it is     
7:20     
more than 786 not 786 the previous data     
7:25     
we had uh how many we had uh we had 786 yeah 786 but here we     
7:32     
have 1,864 which I am not sure about but I think it doesn't matter at this point     
7:38     
for us so I didn't explore that much right yeah so these were the yeah     
7:44     
this so this is about data understanding now coming to data preparation so uh uh     
7:51     
yeah now what I want is what we want is we want to make two types of graphs basically or hyper graphs one is spatial     
7:58     
and one is the line lineage so for the spatial uh we have to first little bit normalize pre-process data I mean do     
8:06     
something with the data so Bradley suggested in the last meeting that we should normalize the data uh at at the     
8:13     
zeroth point so that is what I did I calculated the mean of all the axis     
8:19     
which is XYZ and then subtracted uh from all the points so these three XYZ     
8:25     
columns are little changed now just the normalization is happening uh not normalization exactly we I just     
8:32     
subtracted the mean but uh yeah but I can also do normalization if requir     
8:37     
actually I thought this is okay but if required I can do normalization we just have to do divide by the Max and Min     
8:46     
value Max minus Min but still I haven't done it so I'll just comment     
8:51     
it uh     
8:57     
then uh uh what is this so yeah then I then basically I     
9:05     
wanted to include this particular uh the lineage information in the same data as     
9:11     
well so I did some code and this is what we have in the end so we have the final     
9:17     
data which we are going to use so we have here is sell information oh it is not yet ready so I added lineage after     
9:24     
adding lineage yeah this is how a data looks like so this is a cell information this this is the mother of each cell and     
9:31     
then we have this XYZ and then I thought of making the Min size and Max size of     
9:37     
every particular uh uh cell because we had a lot of time stamps right so I     
9:43     
thought Min size and Max size would be useful probably we can take more features for example average size also     
9:49     
so we can add them uh as required so when I printed the size of this data we     
9:56     
have 780 columns so one 780 rows I'm sorry so one row for every uh every     
10:02     
every unique cell so this is was the data preparation but I notic one thing     
10:08     
so initially we saw that we had 786 rows right uh but here we are getting 780     
10:14     
rows after combining with lineage data this is because I saw what are the missing columns in the lineage data so     
10:21     
in these uh rows were uh these cells were missing the lineage so probably     
10:26     
because P1 P1 P2 P3 P4 Z2 Z3 doesn't have any parents so that is why it was     
10:33     
missing so what I did was I included these six data points uh also as rows in     
10:39     
this particular data but I renamed their mother as n so just to remember that     
10:44     
these particular uh cells don't have any mother cells so these particular points     
10:52     
so what I did is Ion cated it with the previous one so we have now the new stats there the data frame we are going     
10:58     
to finally work on to create the graph so any particular Point still here     
11:05     
or should we should I should I complete everything and then we can dis oh yeah let's go through the whole thing     
11:11     
first okay okay then I was on graph creation part so we need two types of     
11:19     
graphs so the first is we need graph created from the cell positions and the     
11:24     
second is graph treated from cell lineage uh so uh referring to the es of     
11:30     
2022 by jang's project so what he has created is he has created a graph using     
11:35     
the cell lineage and cell lineage since it gives information from mother to daughter he made a directional graph     
11:41     
because there is directional information involved so uh but we don't plan to yet     
11:46     
use directional graph because I'm not sure how hyper graph can be directional but uh we can think about it at a later     
11:54     
point so but yeah but taking to this particular Point uh St to the first     
12:00     
graph uh the cell positions we will start going we'll start with that so I     
12:06     
have used T geometric uh to create that data uh to do the to create the graph so     
12:12     
I have used Cann graph so KNN graph is basically based on the coordinates of X and     
12:18     
Y um so this is basically graph creation what I have done is I created a fully     
12:24     
connected graph that means every uh cell connects to every other cell uh     
12:30     
basically they have a edge but the edge value will be different The Edge value will depend on what is the distance     
12:36     
between those uh between those cells in the coordinate system uh yeah and uh actually we can     
12:45     
threshold also basically based on what how many distance we want so I thought this is the most General graph we can     
12:52     
create at this point later on if we want particular things then we can uh uh     
12:58     
change this full connected graph itself right yeah so this is how yeah so this     
13:04     
is yeah so this is how our uh complete data looks like we have 786 comma 3 so     
13:11     
786 is the number of uh cells or the number of nodes and three is the number     
13:16     
of features so the in features I have included the size uh which is min size Max size and there was one more     
13:25     
thing so yeah we have also mother information here as as the node information and we also have the name of     
13:32     
the cell so these three information is Incorporated in this particular X and     
13:38     
then we have Edge index so this Edge index specify the shape has a shape of two comma uh uh six lakhs or six 61 6     
13:48     
617,000 so these are basically number of edges so this since it's a fully     
13:53     
connected graph that is why the edges are so many and here two represents basically uh that it's a pairwise uh uh     
14:01     
it's a pairwise graph pair wise Edge that means two particular noes belong and then we have Edge attributes also     
14:08     
Edge attribute is basically the weights between those particular nodes or the cells right and these are just the node     
14:16     
size node names and mother names just incorporating all the information then I     
14:22     
try to visualize this graph uh this which was a bad idea I try to visualize a fully connected graph and it took I     
14:28     
think 25 minutes to run then I probably it was way too tiring and not happen so     
14:34     
instead of creating the fully connected graph I just visualize the 2D uh only     
14:39     
the X and Y so this graph doesn't tell anything but just for the visualization I try to plot the self positions here it     
14:46     
doesn't convey anything and that is what I wanted to show that it's not coning anything to visualize a graph we have to     
14:52     
threshold and then we will plot something yeah so this is the graph     
14:57     
creation part but our main but our main task is spatial hypergraph this was just a simple exercise probably to get ideas     
15:04     
and all so now coming to special hyper graph we have two ways to create it so     
15:10     
first way is we can use a previous graph to create the hyper graph so I have named it as distance threshold base up     
15:17     
so thresholding is basically uh what does it     
15:24     
wait okay so here I'm giving 50 yeah so here we are specifying threshold so     
15:30     
threshold I will give give default as 50 uh so what I want is uh what we want     
15:36     
is we want to create many hyperedges and each hyperedge we are specifying how many cells are going to be there so that     
15:43     
is a threshold value so let's say we create we have to we will be creating uh     
15:48     
we will be creating uh number of hyper edges will be number     
15:54     
of hyper edges will be same as the number of uh cells which is     
16:01     
786 uh 786 so if the so if so if threshold is 50 then we will be creating     
16:08     
number of we will be creating 78 six hyper edges and each hyper Edge we have like 50 uh cells which is a lot so I     
16:15     
will show this is not a very good idea and we will have to reduce it so what I have done is uh based on the distance     
16:23     
proximity uh using somewhat like KNN uh KNN is nearest     
16:30     
uh we have created a graph with threshold value of Hy so this is how it looks like uh     
16:38     
pretty uh hard to understand but what it says is um but what it says is in the lowest     
16:46     
most Circle so we have different colors for different hyper edges so in the lowest most Circle there are like some     
16:51     
few nodes and as we go let's say in this pink one we have like a lot of noes so     
16:59     
this is a basically bigger hyperedge uh and these are the hyper Edge numbers randomly given numbers so     
17:05     
this really honestly is not telling anything so what I did was I reduced this threshold value to little to a     
17:11     
lesser value which is I think five uh yeah five and then we plotted this graph so I     
17:19     
think this looks little more sane and little bit Pro probable to     
17:25     
understand but still it's I'm not satisfied with this but go to next but just understanding this particular one     
17:31     
we see that we have a lot of hyperedges and uh and uh yeah and each hyper hedge     
17:39     
has has like five number of nodes here um so this looks very dirty in the     
17:45     
beginning but I have made a bigger when and downloaded so I can show it at a later point also decide in more B detail     
17:53     
uh so this is basically the first method by thresholding uh based on the distance     
18:00     
which we are using the pre utilizing the previous graph and now coming to Second method     
18:08     
that is actually little better I I wrote Somewhere the benefits on paper but I can't find that now uh so yeah uh here     
18:18     
uh we have this method is basically using the clustering approach and I have     
18:23     
used DB scan clustering because DB scan it has some benefits I forgot I will uh     
18:29     
mention them later in the meeting uh but I wrote like three four useful why I     
18:36     
went ahead with DB scan and by clustering based hypergraph is little better so here what we are giving in DB     
18:44     
scan is we have given two hyper parameters one is EPs and one is men samples so EPS means the     
18:51     
distance the max distance basically so for example a hyper Edge two two nodes     
18:57     
has have a distance of say 4.5 then they are going to be in that hyper Edge if     
19:02     
that nodes is little is farther let's say 5.5 distance between two nodes then these two nodes these two nodes are not     
19:09     
going to be in a hyper Edge and to be a hyper Edge there has to be minimum like     
19:15     
two samples they cannot be hyperedge with one sample I mean that we can change also I have changed withon and I     
19:21     
will show later and so I wrote a function to create it uh yeah this is     
19:28     
the function so the hyper parameters I have given is EPS uh Min samples and include noise I     
19:35     
will I will I will try to just tell what it is this     
19:41     
after so here we see that we have a much cleaner graph so here uh in in the     
19:48     
previous threshold based method we have to make sure that all the hyperedges had like uh 40 nodes right or maybe let's     
19:56     
say five nodes but here it is not the case here it depends on the density so DB scan basically I think it means     
20:02     
density based something something I don't remember what the full is D scan that's good so this it's a clustering     
20:08     
method uh so it it basically creates uh lesser number of hyperedges     
20:15     
so uh since and yeah and here uh this     
20:20     
include noise basically I have use as true so include noise     
20:26     
basically means there were a lot of no or cells which were not coming in any     
20:31     
particular hyper Edge they were not able to belong to any particular uh hyper     
20:36     
Edge so if if when I set it to true what they can do is they can create their own     
20:41     
unique hyperedge so for example only that particular node or that particular cell will be in that hyper Edge I don't     
20:48     
think that is a good idea but just for uh trying I did that I also tried with     
20:54     
setting it as false that means those cells are not being included in the visualization itself so uh that is this     
21:02     
uh diagram so this you we probably from visually we can see it has lesser number     
21:07     
of nodes because we are removing the noise cells from here which are like very far from every     
21:14     
nodes uh but but here we are losing some data so it has some pros     
21:20     
and um yeah so this is actually we can see it in much more detail like if we     
21:26     
focus on particular areas uh which we will do uh but I wanted to highlight the     
21:31     
benefit of this DB scan based hypergraph creation or clustering based hypergraph creation is uh so by the way just a note     
21:39     
uh here we only have I think 116 hyper edges compared to the previous method     
21:45     
where we had 786 hyperedges and it looked very very clut right so that is     
21:51     
the one benefit of this clustering based uh spatial hypergraph creation that we     
21:57     
have uh less number of hyperedges that means less overlaps and     
22:03     
less noise I mean uh if we have way too many hyperedges then information     
22:08     
propagation will happen a lot faster information propagation or we can say diffusion and and sometimes that much     
22:15     
diffusion is not required because it can do over smoothing also so it can     
22:21     
contribute to over smoothing also so I think we should do a balance between these two or probably I think this is     
22:26     
better if not the balance but visually if we try to see we can     
22:32     
make sense of it if we have like a bigger figure which we will see in the     
22:37     
end uh yeah so this is     
22:42     
about I okay yeah this is the false one yeah     
22:47     
so this is all I tried about the spatial hyper graph probably if we are good with     
22:53     
it I can create a proper API of it uh for like pushing the code and all this     
22:58     
was just a research aspect and probably if like Bradley suggest or if like you     
23:03     
if everyone suggest we can probably dive deeper into particular things and explore more ideas I have some ideas but     
23:09     
these two are like some good ideas which I we should try and if like in the end     
23:15     
we can go ahead with creating the API whichever method we found the best or     
23:20     
probably we can create apis for two three ideas which are good uh yeah now     
23:25     
coming to lineage hyper so this is is not uh was I plan to do till now right     
23:32     
now the plan was to do only St but yeah I wanted to discuss because it's coming     
23:37     
up uh in this July month uh so this uh yeah so how do we     
23:46     
create line lineage data so one approach we can do is we can create Hypes based     
23:51     
on common parent so for example if let's say a particular there are five cells     
23:57     
and five cells have a common parent so those five cells can be part of a single hyper Edge so just an instance uh this     
24:05     
was the this was my criteria for creating the hyper Edge in the lineage data so I have created using this only     
24:12     
and this is what we can observe so I have a so here we can see that uh uh     
24:19     
here we have the name for the hyperedge also and we have the name for the nodes also so black ones are the nodes and the     
24:26     
different colors for are the hyper I can draw much better figure I'm using some Library called hyper netex I found it at     
24:34     
some point during so this is actually very good for visualization and I can     
24:39     
change it to better colors and all this little more flexible so just to observe     
24:45     
this data in more detail I but it didn't take a lot of time to visualize I think it took 20 seconds to make this large     
24:51     
image so we can make even larger also     
24:57     
yeah okay yeah thanks Bradley yeah and coming to last point this last     
25:04     
point then we can discuss so then it comes to the dynamic hyper Ed which is     
25:10     
our last plan which we want to do but still I think we should plan things from now on so we have a good understanding     
25:16     
and things go very smooth so uh as we observed before like we have time points     
25:21     
from one2 18 190 we have continuous time steps right and then we have some     
25:27     
negative values so I was thinking whether we should create spatial graph and uh and and lineage gra for these     
25:35     
particular data points basically for all data points but let's say for the seventh time there are only like let's     
25:42     
say five six nodes existing so we create uh nodes only for those we create graph     
25:48     
for only those particular uh Hy uh cells that exist at the seventh point and then     
25:55     
we are going to have some a graph something like this uh so this is at different time points I     
26:00     
mean this is just for the idea this is some other data set and some other thing     
26:06     
but probably we will have some evolving graph at part at all the times from 1 to 190 or if we if if you think that 180     
26:14     
for 90 is a lot then we can threshold we can do for every 10th time yeah whatever     
26:21     
sounds good and whatever gives more clarity on understanding the data yeah yeah     
26:28     
so that yeah that looks good um thank you for the update it was very good and very complete um and you produced a lot     
26:36     
of visualizations which I love because it's you know we can see what a hypergraph looks like and from the data     
26:44     
so I mean you know one of the things about the plots is that we don't you know we can plot like every cell uh but     
26:51     
we you know are often interested in the the sort of the cells that are alive at     
26:57     
a certain time so you know if we think about the hypergraph as being present say at 180     
27:03     
minutes like what cells are there at 180 minutes then maybe what cells are like     
27:09     
the mother cells of those and the daughter cells of those so I think that might help with like creating     
27:15     
visualizations where you just maybe would maybe build a an all to all graph     
27:21     
or hyper graph and then maybe have one where you know you take a couple time slices where you go through like at     
27:28     
certain times and say okay at this time there are certain cells that are alive     
27:33     
certain cells that are the Mother cell certain cells that are the other cells and that's it so you have like three sets of cells and then see how those are     
27:42     
connected because you know what we're interested in ultimately is you know we're interested in all the     
27:47     
relationships but to see like at a certain point in time you know what the     
27:53     
mother's cells are what the current cells are and what the daughter's cells are so it would be like the current     
27:59     
cells are basically the spatial distribution of that level of the     
28:04     
lineage tree or the you know uh the the cells that are alive at that time and     
28:10     
then the mother cells of course are interesting because they're the um the     
28:16     
ancestors and so they're passing things onto the daughter cells and then The Descendants are interesting because     
28:23     
those are what's those are what are going to be born and then you can say things like okay how do those move in     
28:29     
space relative to the to the current cells which would be the mother cells for those cells so you would have this     
28:36     
nice three layered graph you know in terms of the lineage tree and then that would be the spatial distribution of     
28:42     
those and you could say a lot of interesting things you could say okay A Mother's cell divides into two daughter     
28:48     
cells those cells migrate out then the daughter cells of those migrate out so you have like this three layers of the     
28:55     
lineage tree that are represented there and you can say okay now what are the hyper nodes for those um those cells and     
29:03     
you know and that would be like because I I think you have especially like the last two hypergraph visualizations were     
29:10     
clean I mean I think the ones that are clean you know where you can see the hyper edges or the hyper     
29:18     
nodes um clearly the hyper edges are a little harder to interpret because I     
29:24     
guess the the question here is like you know if we visualiz the hyper edges it's like technically two hyper     
29:32     
nodes are connected right and then the hyper edges can be heterogeneous so the     
29:37     
hyper edges can be connected to say like multiple hyper nodes theoretically     
29:43     
because if we're using a clustering algorithm you know some of the cells are going to be um you know connected to     
29:49     
other cells in another hyper node but maybe another hypernode as well so you could have this sort of     
29:56     
heterogeneity um but this is this is nice uh just just being able to define     
30:02     
the hyper noes with their contents and then having these edges and I I don't     
30:08     
yeah I don't know how to define the I don't know how to visualize these really clearly and this is just something we're     
30:13     
going to have to work out but those visualizations I think are going to be important because you know ultimately a     
30:20     
lot of these Network science things there are two things that are really kind of are the outcome of a lot of     
30:26     
these studies one is like the visualization and a lot of like Network     
30:32     
science papers will just show like the Hairball and they might try to extract some communities from those and it's     
30:38     
hard to really visualize everything and get the the you know it's like you see the visualization that's one thing     
30:45     
understanding the sort of the cause of relationships that's another thing and people I don't think people have mastered that at     
30:51     
all um the second thing of course is the uh the content within the the so you     
30:58     
know you're basically looking at these all to all representations and it's like if I just     
31:03     
take a connectivity Matrix and dump it out into a visualization what do I get     
31:09     
so we want to take like maybe the most informative slices of the data and then plot them out and that that can help go     
31:16     
a long way towards sort of getting us to a point where we can say okay we're going to reduce the data Downs somewhat     
31:23     
because you can't just put everything to everything and go for it um but yeah I     
31:28     
think we should think about like these informative slices of time where you     
31:33     
know you have like a subset of cells daughter cells mother cells and     
31:40     
then the current uh you know Co cohort of cells and so yeah and     
31:48     
like you know the paper that we did on um spatial networks just the regular     
31:54     
networks you know you you plot these out and and you know you there are a lot of     
31:59     
things that are interesting with all cells and at all levels of the lineage tree but really you want to get     
32:07     
technically you just want to get the ones that are alive at a certain point in time um and and look at those so yeah     
32:13     
there are a lot of interesting things you can do here this is really good work um and you know I I like some of the     
32:20     
Python toolboxes you're using um and I think yeah the DB scan method is good um     
32:28     
and you know there there are going to be drawbacks I think what would be good for the uh sort of for gso and then when we     
32:35     
what we writing this up as a paper too would be to sort of have you know uh     
32:41     
some documentation so that you can like say this is these are the methods that were used these are the toolboxes that     
32:47     
were used these are the pros and cons of each toolbox and Method so like why did     
32:52     
you decide on DB scan and you just You' said that they're pros and cons why did we not you know we maybe we tried this     
32:59     
other method and that that has some pros and cons so if someone wants to say use     
33:04     
this method they can do so but you know it depends like if we're looking at celegans maybe this is best for celegans     
33:11     
we wanted to apply this to dropa maybe it was a better way to do it and there are reasons for that um and then of     
33:19     
course the toolboxes that we're using here you know are I guess they're open     
33:25     
source so we need to attribute them and inite them and everything so I want to have like some documentation with this     
33:31     
as well and definitely it'll help us later sure sure okay so the next things     
33:40     
I will try to do is I'll just J down the point so uh the documentation for all     
33:46     
the steps which we want uh like documentations of     
33:52     
decisions uh about uh libraries and methods     
33:59     
like uh DV scan yeah right and and the immediate next steps     
34:05     
what else can I do uh next steps you know would be to kind of like I said go     
34:10     
through the data and pick out subsets of the data but that are interesting so maybe like all cells that are alive at     
34:18     
time Point 200 take the mother and daughter cells of those cells and then     
34:23     
use that as like a subset and you can like subsample your data set in different ways and you know just test     
34:30     
out different subsets and you know maybe ideally we' like to have a gra or like a     
34:36     
report that says you know this is the hypergraph at 200 minutes this is the     
34:42     
hypergraph at 300 minutes this is the hyper graph for 250 minutes and it's     
34:47     
going to look different it's going to evolve so the idea would be to have like a hypergraph that evolves and you can     
34:53     
see over time as the embryo is changing what that hypergraph CH how the hypergraph     
35:01     
changes and you know it's up to you to see what that looks like yeah     
35:07     
yeah okay okay yeah thanks yeah that actually makes a that's a very good idea     
35:12     
I try this uh so uh and uh Bry about this     
35:20     
spatial graph uh so should we go ahead with creating um sorry this so we have     
35:25     
tried these two methods right distance pressure control based approach and this clustering based so should we start     
35:32     
creating it into API like Jiang has used and like py or py geometric exist     
35:38     
basically another person can directly utilize it should we do it at this point of time or should we push it for later     
35:45     
and right now do only the exploration part and documentation part yeah I think we'd say the API for later um and you     
35:53     
know that's like this is good and just keeping on this and trying to get this down to like where it makes sense and we     
36:01     
can get some good visualizations and then hopefully later we can do some apis you know and I don't     
36:07     
know what gahong had in terms of an API but maybe we can you know kind of use the same one or use a similar one so we     
36:15     
can integrate that so I don't want to like have you know two different apis for the same project I don't want to use     
36:22     
yeah I want it to all kind of be one thing but but that's I think that's something for likely     
36:30     
right right makes sense so he has the API basically for bidirectional graphs which was incorporating lineage     
36:36     
information Yeah well yeah I mean okay yeah so I     
36:41     
mean the idea would be someone could download maybe like I don't know like you know     
36:48     
basically they don't have to like uh be confused as to what to use for their problem so if we need to create it like     
36:56     
a maybe like a compatible API so we could take like you know we could tell     
37:01     
people okay if you want to build a bidirectional graph this is a tool if you want to build this hyper graph this     
37:08     
is the tool just so that it's easy for people when they Fork the repository to     
37:13     
sort of find what they need yeah yeah right right okay uh and     
37:22     
Bradly last Point uh so apologies for missing out a lot of meeting so but but     
37:28     
from now on uh like every week I'm I'm going to show some updates uh because     
37:34     
for a very important reason because I think uh this is more of like research and I need a lot of your feedback every     
37:39     
week so that would help a project run very smoother and also decide at every     
37:45     
particular steps why are you taking those decisions so yeah I wanted to mention that well yeah that's good thank     
37:51     
you that's a good update and um yeah I look forward to kind of getting this off the ground and getting some really     
37:57     
interesting stuff out of     
38:02     
it yeah one last question it's a technical one actually so I'm going     
38:07     
sorry you have to repeat this particular point I think you we have discussed this before but I can't remember it so what     
38:13     
is a difference between hyper node and Hyper edges like just in technical so as I understand hyper Edge hyper Edge is a     
38:19     
correction of nodes in a way but you have mentioned hyper nodes a lot of     
38:24     
times in the paper right so I didn't fully understand what hyper what is the difference between these two are they     
38:29     
like same things well hyper node is a single node that has a bunch of nodes in it so     
38:36     
like in the clustering method you might have like 10 nodes that cluster as a hyper node right so you might have like     
38:44     
the content inside the hyper node are individual cells with identities and then those things are     
38:51     
kind of clustered together and you can refer to those collectively or you can refer to those     
38:56     
as the individual nodes and then the hyper edges are the things that leave that hyper node and go to different     
39:04     
hyper nodes with their own Sal and so this is why you know you have hyper edges which are like a like a bundle of     
39:11     
nodes going from one hyper Edge to another but that means that you have hyper edges that go to different nodes     
39:18     
so like that's the idea that it's it's almost like you have like uh a super graph and then inside     
39:27     
that super graph you have parts of uh regular graph and you're kind of trying     
39:32     
to bundle those things together in the way that you know so they share like a     
39:38     
connection uh if a bunch of nodes are connected to a bunch of other nodes in     
39:45     
groups then those are all bundled as a hyper a hyper uh Edge and then the cells     
39:51     
that are clustered together bundled together as a hodde that's basically the way I think     
39:56     
of it yeah and then you know it's kind of tricky because it's hard to wrap your head around because you have this     
40:01     
regular Network and then you have this other network on top of it but um yeah I don't know if the tools are making like     
40:09     
I don't know if that's the same way they do they treat those in the tools that you're using but I     
40:16     
mean yeah okay yeah that makes sense that makes a lot uh but just for the tool     
40:24     
sake uh actually in tool the tools I Wasing using for visualization uh just     
40:30     
to give them credits and highlight them so I'm using this hypernet x they have a paper also in 2023 uh I think in some     
40:38     
conference so this is a very nice tool for uh for a lot of objectives for     
40:44     
basically uh visualization of course and then it provides a nice way to create     
40:49     
hyper graphs and then do basic arithmetic operations also so and and     
40:56     
and one more thing was there so it was related to paky actually so that's why I     
41:02     
wanted to highlight so it can help us uh understand the topology of hypergraphs     
41:08     
as so it has some metrics if I'm not wrong I was seeing some in in one of     
41:14     
these collab notebooks so it's a pretty good package to explore for later on     
41:20     
understanding those hyper graphs as we were thinking we have a big hyper graph but we try to want to understand it uh     
41:26     
through some Metric so this Library can help us in some ways at least okay yeah     
41:33     
that's good this looks yeah like a nice toolbox and they have tutorials and everything so that's good and I like     
41:40     
what are these like these uh drawings up here so these are like paths     
41:47     
yeah so these are like clusters of of nodes that they show up here yeah so     
41:54     
this yeah so this is like a nice example because it's like it's basically taking     
42:00     
the nodes and grouping them in different ways so like you have Slither and Hufflepuff griffindor and like it can be     
42:07     
nested or they can be separate and then those are connected so in Slytherin there three nodes those are connected to     
42:15     
Ravenclaw the the there's an overlap in this case and so that they can overlap     
42:21     
certainly um but then you have other nodes that connect other nodes in other     
42:27     
hyper nodes and so those are the hyper edges and so yeah this is this is a good     
42:33     
this is a different way of visualizing it because they consider overlap between     
42:38     
uh hypernodes but you can think of hypernodes categories that you fit your nodes into and then hyper nodes are     
42:46     
connections between those categories right but this Library hasn't     
42:53     
highlighted hyper nodes anywhere as well they yeah they don't they don't I don't     
42:59     
know what the terminology is I mean people are like this is kind of an area where I know people have different ways     
43:06     
of thinking about it so um and it's hard to really kind of get around like yeah     
43:12     
basically you know they're trying to classify nodes in different ways so     
43:20     
yeah right right yeah so thank you very much that     
43:26     
is from my side well yeah thank you yeah it was very good very good update all right um so Pocky do you have     
43:34     
an update in the meantime why don't I highlight her um the poll request she     
43:40     
made uh on the repository so this is yeah so this is uh her PLL request in     
43:47     
the D graph repository this is the kmapp implementation so this is and I I think     
43:54     
she said she was going to make another poll request soon but this is uh with     
43:59     
the K mapper implementation and some other details so this is the fork of the uh k mapper um     
44:08     
Tool uh toolbox and some of the things that she's been working on with respect to that and then she's merged it into to     
44:16     
daph so congratulations to Pocky for that contribution and it looks like she's     
44:23     
back yeah I'm out there we go all right yeah thank you yeah uh yeah so I've been working on     
44:31     
like this NDP function this is almost complete I just need to integrate like     
44:37     
the functions here with the other like files so in their repository they have     
44:42     
like a very organized framework so first it will call call like the train.py file     
44:48     
it will then call this one file in this file it calls like the MLP function and     
44:53     
the adding new nodes and the predicting new nodes and it also calls like The evolutionary algorithm I'm also working     
45:00     
on this one I need to like make some changes to this so like this integration     
45:07     
part is taking uh some time I also have like my exams going on right now so I     
45:12     
wasn't able to give much time to the project but like uh this file is almost     
45:18     
uh complete and I just need to like uh integrate and then train it it works but     
45:23     
it's showing that the graph is disconnected that's probably because in their architecture they are generating     
45:29     
everything randomly like the initial graph is random and then the new o new     
45:35     
nodes that get like added to other nodes are also random and then they like check if it's follows like the small world     
45:42     
Network topology but like in our case we have like the data set available and all     
45:47     
that so I need to make those changes and if uh like in the last meeting you     
45:54     
suggested like the small world network is uh similar to like the growth of C Elegance is what I understood so I'm     
46:01     
trying to uh like use the same functionality here as well just modifying it to our usage but if that     
46:08     
doesn't work out I will like go with my initial plan of like uh making the whole architecture like from scratch so I'll     
46:15     
give more time to this because I think I can uh like try to implement in this only uh I would like to show how it     
46:22     
would look like so this is the uh small world Network that they have generated     
46:30     
so once uh I'm able to train it then this is how it would look like so uh     
46:37     
this is what I'm trying to do right now uh this should take like a few more days     
46:42     
two to three days and yeah I can update you after that okay so yeah it's just     
46:49     
generating like nodes and then edges with as the nodes are added and it's     
46:55     
doing in that small world fashion correct yeah okay yeah yeah I mean yeah     
47:01     
there going to be some challenges because you know we can't say that cagans is a random Network I mean I I     
47:08     
mentioned I think last time about how the connectome is small world and we might assume that the sort of the larger     
47:16     
graph of cells is also has some structure that is maybe small world maybe not but but small world is a good     
47:22     
heris and then we can work from there um but I don't know like how it's handling     
47:28     
the incorporation if it's doing it at random that's maybe not going to fit the data we have for C elegans because it's     
47:35     
not really you know the the cells are U you know you have the mother cells and then you have the daughter cells and the     
47:41     
daughter cells uh you know migrate in a non-random fashion they migrate you know either they're Left Right pairs or they     
47:49     
move anterior posterior from the mother but they're not you know just generated at random there's some contingency there     
47:57     
so that's that's the thing and uh I don't know you know what you'll need to do to make that happen like you know I     
48:04     
don't yeah yeah I I have some idea in mind like uh in the paper also theyve used like two three MLPs like for     
48:11     
placing randomly and like generating randomly and all so I don't think we require all that because we already have     
48:17     
the data set we would need only one and it will like predict okay like maybe here the new no the child notes get     
48:23     
added and then I can like compare it with the data we have have and then back propagate the losses in order to like     
48:29     
train the model so that's what I'm planning so I'll let you know once I'm able go TR it okay well that sounds     
48:38     
great thank you for the update thank you and so yeah and mul uh you know     
48:46     
please uh issue a poll request for the work that you've been doing when you can you know whatever you think is a good     
48:53     
siiz package for the poll request and we want to keep everything up to date and and backed up and everything     
49:01     
so that's good thank you okay yeah right Bradley so for the PO request     
49:08     
yeah I'm sorry go ahead no yeah so in the PO request should I     
49:14     
upload this Jupiter notebook is it yeah please push the Jupiter notebook and any files that you have like you can push     
49:21     
image files as well and just put them like you know kind of put together the repository so that it has you know maybe     
49:28     
some maybe a read me because you'll be pushing to this again you want to have like your space and then you want to     
49:34     
have your you know you'll issue a pull request and then you'll update with new     
49:40     
files so please you know the notebook images maybe a small read me now for now     
49:46     
and then we'll add documentation as we go so yeah makes sense yeah     
49:53     
I okay uh so I don't know Susan or Morgan have updates um yeah we have some things on     
50:02     
in the chat and it's largely just kind of the co you said forgot to mention thanks     
50:09     
Mah for sharing the visualization Library I'll try using it too your visualizations look great so that's good     
50:15     
please share that with Pocky about you know where where to I don't know if that was the thing you were highlighting or     
50:20     
if it's a different package but     
50:27     
yeah good okay okay so yeah Susan I guess you want to have do an update oh     
50:33     
well um I think I mentioned before I've been accepted into compucell okay the     
50:40     
compucell 2D and 3D uh course and so I've been preparing for that I have to     
50:46     
program in Python and of course I haven't but I'm assuming it's SE like     
50:51     
people say it's SE like and it's much like that lab or C and I have programmed     
50:57     
in that so I'm just kind of preparing for that and I discovered that my lovely     
51:05     
uh console program was exaggerating the movement of the parts by 10 the minus5     
51:11     
so that's why I was getting these odd numbers yeah they were off by 10 to     
51:18     
the minus five for some reason yeah so I found out where to fix that and now I'm     
51:24     
getting proper numbers at least there's still a few problems but     
51:31     
at least I get data now that makes sense that's     
51:38     
good that's awesome though because I was really tempted to to apply to     
51:45     
that you so I I'm just glad you can say more about your experiences you know     
51:52     
what what they have do or yeah yeah yeah and I I was talking to them and they said oh     
51:59     
yes if if I can get my 10 secrity to work that maybe they could incorporate     
52:05     
it into pum cell would be willing to do something     
52:10     
like that I don't know if it'll work but um at least they they figure it's a a     
52:17     
good platform for integrating other ideas so that was nice great yeah it' be     
52:24     
nice to have like a good platform for Tegrity I guess well yeah at least have it out     
52:31     
there because I I think it's um I think it's a proper way to go about     
52:37     
looking at at cells because they're     
52:43     
intention well that's good thank you yeah any other updates     
52:53     
or Morgan did you want to give an up     
53:00     
I'm not sure I got anything particularly relevant     
53:09     
um no I mean you know check check chanels for any interesting talks or     
53:15     
papers that that I might have might have seen but that's that's about it yeah     
53:20     
there's a lot of interesting stuff coming up in the orthogonal lab slack that you've been putting in there about     
53:26     
different uh tools and and talks and things like that so yeah yeah you know     
53:33     
I'm not sure that that uh I'm not sure how well grounded they are no I mean you     
53:39     
know like like super super interesting stuff in MTH and in the formalization in     
53:45     
mathematics like like great sets of talks coming out of um that osdorf     
53:52     
Institute uh and they're they're still coming they're still dropping videos even like this     
53:57     
morning like like I just saw another Kevin buzzer one too um but but I     
54:04     
haven't been able to figure out how to connect those back to anything that kind of project related right     
54:11     
yeah um oh cool there was a biological physics uh webinar by dapping     
54:20     
B on um well it used comp comp youself     
54:25     
um three um and they're looking at the fluidization of of tissue um and it was     
54:34     
very good I need to watch it five     
54:40     
times um I wrote down the guy's name and biological physics but then I didn't     
54:45     
write down the name of the webinar anyway um if you want me to I     
54:51     
can uh find that for you yeah that'd be great     
54:57     
okay I I'll I'll find it and send it all right we can have a look at it yeah     
55:02     
maybe topic for next next week or something yeah that'd be     
55:08     
good okay yeah thanks looks like dick posted     
55:15     
uh this uh origin of Life via ARA shape drop woodster ARA so this is something     
55:21     
that was just published uh     
55:29     
oh being printed this week so that's good yeah that's coming out this week that's that's the shape droplets paper     
55:35     
that we featured a while a long time ago in in um one of the meetings and it's     
55:41     
basically you know looking at different shapes of our kls and looking at droplets and seeing     
55:47     
how they're of uh connected so that's that's good stuff     
55:52     
we'll have to go over that paper as well at some point     
55:58     
so all right so I think that's it for today uh if we had any other     
56:04     
comments where we     
56:09     
go okay all right thanks for attending on later bye bye take care     
56:17     
take care bye you too bye all right now I wanted to cover an article from azimov     
56:23     
press on breaking the speed limit of cell division or cell cycle so let's start with the breaking     
56:30     
the speed limit on cell division paper this is from Asimov press um this is by Kamal nah house so     
56:39     
Asimov press is a really interesting sort of uh blogging platform uh short     
56:46     
biological story platform and so they talk about a lot of issues that are relevant what we do here so this is an     
56:54     
essay from issue 2 three and so this is about breaking the speed limit on cell division so cell     
57:01     
division has a speed there's we've talked about this before and we'll get into this with the papers there's a you     
57:08     
know there are different Cycles uh there's a cycle of cell division there different phases that a cell can be in     
57:14     
and basically it has to be regulated in time so the cell has to complete one phase to move on to the next phase and     
57:21     
if you extend the amount of time that's in one phase it can uh disrupt cell     
57:26     
division and it can disrupt the survivability of a cell line or it can     
57:32     
affect things like differentiation and other properties of the cell if you increase this rate of cell     
57:39     
division you have faster proliferation of cells if you slow down cell division you can basically slow down the     
57:47     
proliferation rate of cells so all sorts of things that cell division is relevant uh too so the first part of this is     
57:55     
computer programmers have it easy so they talk about computer programmers being able to write a piece of code for     
58:02     
just a few minutes before running it and seeing the results now contrast this with bioengineers and biologists who do     
58:10     
different manipulations of cells and of genes so biologist manipulate code of a     
58:15     
different kind spending days modifying Gene sequences however they plot along as     
58:21     
they clone plasmids and so this is some something you do in a wet lab environment     
58:26     
ment um we're constrained in biology as opposed to computer science by the pace     
58:33     
at which cells grow and divide so in this case what they're going to talk about is how to increase the speed of     
58:39     
cell division to observe things more directly so due to the nearly ubiquitous     
58:46     
use of cloning in life science research this lost time adds up and so this is researchers use DNA     
58:53     
cloning to make numerous copies of a gene which can then be inserted into living cells to study how that Gene     
58:59     
functions or otherwise CES the cell to make proteins so this is important in     
59:04     
producing medicines and other types of uh biomaterials that you want to create     
59:10     
um using DNA cloning the ability to clone DNA faster would enable scientists     
59:15     
to run experiments make discoveries and produce drugs more quickly so if we can say divide you know     
59:25     
we can help cells and we can create clones of the DNA and different in uh a     
59:31     
larger and larger cell line we can actually uh speed up the amount of time     
59:36     
it makes it takes to make medicines or to create a cell line through other     
59:42     
various purposes so that's why we're interested in speeding up cell division     
59:47     
because it speeds up the time that we can get these things that we want to engineer and so they talk a lot about     
59:54     
cloning here uh the cloning process itself contributed to a delay in doing     
1:00:01     
the things that we want to do requiring multiple steps such as cutting and pasting sequences from one plasma into     
1:00:07     
another exercising antibiotic resistance genes or flipping sequences around so     
1:00:13     
when you make genetic manipulations you often make a manipulation in a single cell and then you allow that cell to     
1:00:21     
divide and this is in bacteria you can then get the uh copies of that and you     
1:00:27     
can do things with that line but you can't make additional edits until you     
1:00:32     
have clones ready to go and so you have to wait for those clones to sort of     
1:00:38     
generate naturally um for each perturbation researchers like me must reclone the     
1:00:43     
modified DNA sequences into bacteria such as the coli to mass produce the     
1:00:49     
plasma however E coli grows slowly doubling only every 20 minutes under the best conditions forcing us to overnight     
1:00:56     
for bacterial colonies to appear on culture qus so they're talking about     
1:01:01     
eoli they're talking about doubling time of eoli which means that when eoli is a     
1:01:08     
single cell it divides into two cells and then those two cells divided into two cells so you have four cells and you     
1:01:15     
have this geometric increase which is of course doubling at every generation or     
1:01:23     
every division event and so when you you have you know as as your eony grows you     
1:01:29     
get more and more cells but that's constrained by a doubling time so the doubling time is how long does it take     
1:01:35     
you to get from a single Ealy cell to two ecoy cells to four ecoy cells and so     
1:01:41     
on and so forth and so that doubling time as you can imagine if it takes 20 minutes to go from one to two cells then     
1:01:48     
you just keep adding 20 minutes until you get a colony of competent cells and that could be you know a long time if     
1:01:56     
you're looking for you know thousands of cells and so this is something that     
1:02:01     
maybe we have to wait overnight to do instead of having this done in in minutes so if you think of like you know     
1:02:08     
cell division is sort of a compiler for your genetic engineering experiments     
1:02:13     
it's like running a compiler overnight and of course in computer science that would be unacceptable but this is     
1:02:19     
something we have to live with in biological engineering we must then scale up colonies into large populations by     
1:02:26     
culturing them in liquid growth media adding another 24 hours after which we can finally isolate the DNA that that's     
1:02:33     
the stuff that you've manipulated and um analyze it so this is like this would be     
1:02:39     
analogous to a computer programmer taking code having to put it into the     
1:02:44     
computer and running the job overnight and then you know opening up the file     
1:02:51     
and then trying to manipulate or or look through the code to see what happened     
1:02:56     
um and so this is something that you know it's it's again it takes a lot of time the workflow is very     
1:03:04     
U you know it affects the length of the workflow so they just talk about this in     
1:03:10     
terms of this analogy with computer science this is a dividing eoli cell uh     
1:03:16     
this is a scanning electron micrograph and you can see that the cell was dividing here and of course this took a     
1:03:22     
certain amount of time to divide so you know it's going to take a lot more time to keep dividing into a competent line     
1:03:30     
with many many cells where you can Harvest them and of course you need to harvest them uh in in Mass because you     
1:03:38     
know a lot of cells you can't extract DNA out just like one cell usually it     
1:03:44     
takes a fair number of cell to get an amount of DNA that you can     
1:03:50     
recognize and analyze so they biophysical limits as well so in recent years researchers have     
1:03:57     
found biophysical limits and cell division speeds such as genome size or     
1:04:02     
the timec consuming sythesis of bulky ribosomes and so this is one of the things about cell division is it not     
1:04:09     
only relies in these different phases but it relies on the replication of     
1:04:14     
DNA you know and so if you have a larger genome it takes longer to uh copy all     
1:04:21     
the uh DNA create another genome and create another nucleus for that new cell     
1:04:29     
you also have genes being translated in or rnate being translated into amino     
1:04:35     
acids and then made into proteins and that's constrained by the ribosome and the capacity of the ribosome and so     
1:04:43     
ribosomes have a certain uh throughput uh after which you can't really speed     
1:04:48     
those up so there's a rate limiting process of the ribosome and so it takes a long time if you have a lot of DNA you     
1:04:55     
need to Sy syze uh into a protein then you're going to have to wait so there     
1:05:00     
are these mechanisms within the cell itself that slow down the uh speed of cell division and limit the speed of     
1:05:08     
cell division and so you know making cells divide faster uh such as simplifying genomes or     
1:05:14     
ribosomes or enlisting artificial selection favor genetic mutations that boost speed are maybe Pathways forward     
1:05:22     
but it's not clear that that's something that you could really speed up that much so you know we might be able to speed up     
1:05:29     
cell division a little bit but not really to you know at wor a     
1:05:35     
magnitude comparing the growth speed of different organisms might Ste us towards ways of jump starting this process the     
1:05:41     
rate at which organisms double is Central to how evolutionary biologist we up the competitive competitiveness of     
1:05:48     
individuals but then a species evolutionary Fitness boils down to two factors survival and of Greater     
1:05:54     
relevance here reproduction so this is again you know this looks this is looking at the fitness of cells     
1:06:01     
and you know looking at the survival of cells some cells over others and their     
1:06:06     
reproductive capacity so in uh bacterial cells their division rate is basically     
1:06:14     
their reproductive rate so you know some cells Will Survive over others if there's a certain mutation in a Cell you     
1:06:22     
know after it divides then that's that's that might make that cell more competent than a sister cell and it comes to     
1:06:29     
dominate The Colony um so you know these are this is a kind of model we can think of in terms     
1:06:37     
of uh you know the kind of colonies that we're producing and to see you know if     
1:06:42     
those if we can use a mutation based approach to improving uh or speeding up cell     
1:06:50     
division or improving the efficiency of cell division and so if we compare all life forms by     
1:06:57     
their speed of cell division multicellular eukaryotic cells reproduce the slowest so we were just talking     
1:07:04     
about procaryotes and eoli now if we consider eukaryotic cells which are may     
1:07:09     
be more interesting especially for biomedical applications this is something that um you know it gives us     
1:07:17     
even hard a harder time in terms of speeding up the repr uh you know the     
1:07:22     
division rate reproduction rate so m multicellular ukar carotic cells     
1:07:27     
reproduce the slowest and this is because of a number of things including that you have a lot of organel that you     
1:07:34     
need to reproduce to have uh two cells you have a lot more DNA that needs to be     
1:07:39     
uh replicated and so you end up with this situation where you have a much slower rate of     
1:07:46     
division so-called primary cells harvested from living tissues and cultured in the laboratory and take     
1:07:52     
weeks to grow and spread across the culture flask so the doubling time is much slower Prime in these primary cells     
1:07:59     
and eukaryotic organisms uh you know it takes weeks to to grow a culture um hampered partly by     
1:08:07     
their large genomes which take time to copy it takes about eight hours of the fastest dividing human cells to double     
1:08:13     
their DNA so that's even slower and so speeding up that rate of course is going     
1:08:20     
to be a harder lift because you have a lot of you know they're they're going to be there's going to be a lot more DNA to     
1:08:26     
replicate but also you have a lot more structure in the UK carotic cell to     
1:08:32     
reproduce in addition they carry out two intermediate steps between rounds of replication and division the one is G1     
1:08:39     
phase and G2 phase so in procaryotic cells they don't have these G1 and G2     
1:08:47     
phases and so in in eukaryotic cells we have four uh phases of cellion so having     
1:08:55     
four phases of C division stalls division by roughly 15 more HS primary cells also begin to divide more slowly     
1:09:02     
or come to a halt after a few rounds of doubling making them inconvenient for long-term     
1:09:08     
experiments so there's this uh thing called the hay flick limit which is that     
1:09:13     
the number you know in in some cell lines you can only divide so many times before you get some sort of apotosis     
1:09:21     
which is cell death program cell death or the Lin just don't want to divide     
1:09:27     
anymore so you have these limits on eukariotic cells which don't exist in E     
1:09:32     
coli and this makes it even harder it actually slows down cell division rather     
1:09:37     
than speeding up cell division so in a lot of experients people will use rapidly proliferating     
1:09:44     
cancer cells such as Hala cells actually Hala cells are not cancer cells they are these fused cells they're like cancer     
1:09:52     
cells and liver cells and they have different properties all kind of mixed together in the same     
1:09:58     
cell and so haa cells are known as Immortal cells so they can divide     
1:10:04     
definitely uh they double every 17 hours so they're very fast growing and they're     
1:10:09     
very useful for experiments where you need a a quick doubling to but they're not representative of a lot of cells in     
1:10:17     
the body and indeed they don't really help you in you know a lot of contexts     
1:10:22     
where you need to have you know specific specific types of cells that you     
1:10:28     
want some bacteria like microbacterium tuberculosis reproduces slowly as UK     
1:10:34     
carots taking 24 hours to double under the best conditions however most bacteria are built for Speed among the     
1:10:41     
fastest eoli asilia subtilus or subtilus     
1:10:46     
take a mere 20 minutes to divide so these are pretty fast dividing cells we     
1:10:51     
talked about ecoi previously Theus is is um you know it's also in this class of     
1:10:58     
very fast dividing cells Vio divides faster still doubling     
1:11:04     
in only 10 minutes bacteria benefit from being single celled having tiny genomes     
1:11:09     
hundreds of times smaller than those of viarot and lacking in intermission between rounds of replication and     
1:11:15     
division so you have basically in bacterial cells you have replication division in in carotic cells you have     
1:11:23     
these uh G phases that are intermediate um and so so when looking     
1:11:31     
to design Swift cells that can Elevate productivity these microbes make great     
1:11:37     
contenders what's the delay then this is where they're considering um sort of you     
1:11:43     
know how to speed this up so a bacterium's goal is to become bacteria this is a quote from the late     
1:11:50     
microbiologist Stanley Falco so this is goal of bacteria is to become bacteria so bacteria will divide     
1:11:57     
they need to remain dividing to become a bacterial colony and to behave like a     
1:12:03     
bacteria so this is uh a movie where they have a time lapse of bacterial     
1:12:08     
growth it's very rapid and you get a sense of what they do they're basically machines that divide and they replicate     
1:12:15     
that's what they do that's all they do um it's kind of like seans you know they basically eat bacteria and they lay eggs     
1:12:25     
that's largely at the day they move around a little bit and some organisms just do that they're just replicators     
1:12:31     
more or less they're no bacteria just replicators but that's mostly what they do so um so ecoi kind of exemplifies     
1:12:42     
this they copy their genome every 40 minutes yet divide in half that time     
1:12:47     
they actually divide faster and they copy their genome so this is interesting because you think you'd have to have a     
1:12:53     
full copy of the genome to divide but in this case the vision is such an imperative that it's selected for over     
1:13:01     
copying the genome and I guess what happens is they just kind of fill in the gaps after they divide so this is     
1:13:07     
interesting um but this is you know this is again this aspect of natural     
1:13:12     
selection at work in some lines you could really get rapid division in other lines you know it's it's very slow and     
1:13:19     
there are reasons for that that have to do with the survival of the cell uh they achieved this Fe by St     
1:13:26     
starting new rounds of replication even before previous ones have finished juggling up to a dozen rounds at any     
1:13:32     
given time unlike eukariotic cells that pause a replication for the majority of the     
1:13:38     
cell cycle these multitasking bacteria never take a break by marveling at fast lane Cruisers     
1:13:46     
like eoli we witness how Evolution favors quickly dividing cells so this is something that is selected for an     
1:13:52     
evolution in bacteria this rate of quick     
1:13:58     
Division and so but yet despite refinements over 3.5 billion years urces     
1:14:03     
creep in and limit natural selection in the same way that a race car is tires way at faster speeds cells struggle to     
1:14:10     
indefinitely increase their velocity without having it taking a toll on their health so this is sort of like the     
1:14:17     
example from Halo cells of course Hala cells are these sort of cancer cells that also have properties of liver cells     
1:14:24     
and these Immortal cells um but of course that has an impact on what they     
1:14:30     
do what they function so in bacterial cells they're very fast dividers but it takes a toll on what you     
1:14:37     
know their sort of their competency and their health I guess so this is something that you know we always     
1:14:43     
struggle with in natural selection we have some we have these trade-offs in terms of function that often times limit     
1:14:51     
or constrain you know how efficient some something could be or how fast something could be or even yet how fit something     
1:14:59     
can be they also can't come up with new strategies from scratch in away an engineer might they must retrofit     
1:15:05     
existing genes to adapt making it difficult to circumvent bottlenecks so this is a good point that you know     
1:15:13     
natural selection doesn't operate like you know uh engineer might optimize     
1:15:19     
something you have you have to work with the existing genes that you've gmin and     
1:15:25     
you may be able to retrofit existing genes to adapt but sometimes you have these bottlenecks that you know are     
1:15:31     
really kind of hard to overcome because you don't have the raw material of evolution you know to to overcome so you     
1:15:39     
know sometimes you know we talk about mutations being sort of like a way to to get adaptations but sometimes you know     
1:15:46     
those adaptations kind of you know there has to be something there and so when the     
1:15:52     
cells are dividing like that um they may have enough mutations to overcome these     
1:15:59     
bottlenecks they may genome size creates the first bottleneck to cell Vision so the genome     
1:16:07     
uh size of course is important to kind of speeding up cell division so if you     
1:16:12     
want to have fast cell division generally want to have a small genome but you know of course you can divide     
1:16:18     
faster than you can replicate DNA so that's there's that but of course at some point the genome size becomes     
1:16:26     
constraint and again large genomes generally take longer to copy because as a DNA po polymerase enzyme runs across     
1:16:33     
DNA strand during replication it pauses to proof read the copied sequences so     
1:16:39     
this is this whole process of of copying DNA you have a     
1:16:44     
polymerase runs along the DNA strand it's proofing the copy because it you know can introduce armful mutations and     
1:16:52     
so it can you know it needs to do that those kinds of checks and then assemble into a new bble Helix uh Evolution hits     
1:17:00     
the brakes to avoid burning its tires if poas has rushed this phase the cell might occur many harmful mutations so     
1:17:08     
you can occur random mutations during cell division during replication but you can also occur harmful mutations most     
1:17:16     
mutations that sort of uh you know get passed on or or neutral you're going to     
1:17:23     
get harmful mutations those usually get edited out and of course it's imperative for a competent cell line to have these     
1:17:30     
harmful mutations edited out at the source so that's what happens with um     
1:17:36     
with cell division and that's what slows down cell division in part although the rates vary a     
1:17:43     
procaryotic prase copies 1,000 bases per second on average but a UK carotic one     
1:17:48     
copies only 30 in the same time and so this is where you know you have this sort of cop this speed of the     
1:17:56     
Palmeras in per carots it's much more it's much faster and so we have this     
1:18:02     
speed you know differential those slower eukariotic polymerases make up for this by     
1:18:08     
initiating replication at multiple sites along chromosomes this dividing the JW between multiple enzymes bacteria     
1:18:15     
doesn't split up this task in the same way they begin replication of just one Locus with two polymerases running in     
1:18:22     
opposite directions to copy the whole Geno so this is where you know we have this UK carots have developed or evolved     
1:18:30     
these different mechanisms for speeding up cell division because you know eukariotic cell division could be     
1:18:37     
infinitely slow than uh but still it's slower than uh than uh procaryotic     
1:18:46     
cell and so this you talks about proofreading and you know kind of what     
1:18:52     
the mechanisms are there the rate at which polymerase enzymes fulfill their function illustrates that genome size     
1:18:59     
acts as a fundamental barrier to Speedy replication smallware genomes therefore could lead to faster growing cells so     
1:19:06     
these are the ribosome subunits which is the other bottleneck and so again rnas     
1:19:12     
that get transcribed from genomes they go through a ribosome they get translated into amino acids which you     
1:19:19     
get it's assembled into proteins which are needed for a lot of the structural     
1:19:24     
aspects of the cell and so but ribosomes have a certain rate that they can     
1:19:29     
translate things so they have this sort of it's like feeding things into a a paper shredder or feeding things into a     
1:19:37     
tape machine you can't overload it and stuff so there's this whole literature on the speed of     
1:19:44     
ribosomes and you know the that's that's where your other uh constraint comes up     
1:19:50     
so the biosynthesis of ribosomes the complexes that build all the cells proteins cause an additional bottl     
1:19:57     
ribosomes create all the enzymes needed to catalyze chemical reactions and po growth and ramping up enzyme production     
1:20:03     
could in theory speed of cell division times however this would require the cell to make many more ribosomes which     
1:20:10     
in turn with slow down division speeds because the production of these Behemoth complexes in the cellular sense they're     
1:20:16     
not very big requires copious resources in time so you know relative to sort of     
1:20:22     
DNA replication these are huge but they're not that big in the cell but but making more of them for sell slows down     
1:20:30     
Vision speeds and of course they're more complex ribosomes in eukaryotic cells and in procaryotic cells which of course     
1:20:36     
slows down their division rate yeah for for 2021 paper researchers at the California Institute of Technology     
1:20:43     
estimated how long it would take for eoli to churn out a single ribosome using a back of the envelope calculation     
1:20:50     
so they just wanted to know for our first principles how long does it take to churn out a single ribosome     
1:20:58     
equ RNA and proteins comprise each ribosome the cell synthesizes these two     
1:21:03     
biomolecules at different rates the RNA polymerase enzyme transcribes DNA into RNA at a rate of approximately 40 bases     
1:21:11     
per second but proteins emerge much more slowly only around 15 amino acids are     
1:21:16     
added to a bing protein chain in the same amount of time with approximately     
1:21:21     
50 proteins in a ribosome totaling approximately 1 or 7500 amino acids they     
1:21:28     
estimated that ribosomal protein synthesis is one of the slowest processes within e colon so it takes     
1:21:34     
eight minutes for a cell to synthesize the proteins needed to make just one ribosome significant constraint so this     
1:21:41     
is our minimal cell that we've talked about in meetings in the past this is a     
1:21:46     
j jcvi minimal cells where you have the molecular modeling and you have the uh     
1:21:55     
sort of the corar grained cell modeling and you're putting all that together um     
1:22:01     
and so this is where they can kind of do these kind of experiments they can simulate these things you know we might     
1:22:07     
want to ask the question if eoli possessed only one ribosome how long     
1:22:12     
would it take the cell to divide if this one ribosome added 15 amino acids to gr     
1:22:17     
protein chains every second and if total protein content of the cell amounted to 1 billion L amino acids it would take     
1:22:25     
over two years to build up all the cells proteins so you know this is kind of     
1:22:30     
interesting because um you know this really tells you something about like this kind of     
1:22:37     
speed of cell division what's all involved in building a cell so um you know we can do these back of     
1:22:45     
the anvel calculations we can do these simulations these molecular Dynamic simulations these cell these wh cell     
1:22:53     
simulations um but you this this is only part of the     
1:22:59     
equation so when eoli divides every 20 minutes then because it spreads its     
1:23:04     
molecular labor across anywhere from 10,000 to 100,000 ribosomes these     
1:23:10     
estimates match experimentally determined ribosome concentrations suggesting that ribosomes only already     
1:23:16     
work at maximum capacity in microbes when ribosomal proteins evolve     
1:23:21     
they become integral to cell survival but a cellular sped up synthesizing these proteins became a timec consuming     
1:23:28     
burden that cells couldn't abandon without losing other crucial functions so this is interesting because     
1:23:36     
cells can do other things right like an ecoi could do other things but the     
1:23:41     
imperative is the Divide so as its imperative is the divide it sort of devotes all of its resources to the     
1:23:49     
mechanisms for dividing so there there's this aspect of needing to sort of speed     
1:23:55     
up but also needing to have like G it replicated and uh things pass through     
1:24:02     
ribosomes to produce proteins ribosome synthesis may take most credit for limiting division rates     
1:24:09     
but the cell has yet other bottlenecks to condem with so we've talked about ribosome synthesis and genome length     
1:24:17     
which are two constraints on division rate so it slows down division rate but     
1:24:23     
we have other bottleneck that we have to consider so uh in 2011 biophysicist at     
1:24:29     
stonor University estimated how the rates of protein folding impacts eoli     
1:24:35     
growth they found that some proteins fold much slower than others depending on their three-dimensional structures     
1:24:41     
helical twists in a protein form faster than parallel running strands called beta sheets for example some proteins     
1:24:49     
take as much time to fold as an entire eoli takes to divide this suggests that even if cells     
1:24:54     
could Sidetrack timec consuming synthesis of ribosomes slow protein folding could be another road block to     
1:25:01     
faster division so protein folding rates are important of course proteins need to     
1:25:06     
fold in a certain way at a certain rate in order to become fully function so     
1:25:12     
protein folding usually doesn't take a long time but sometimes they can fold     
1:25:17     
slowly and you know if they have a a more complex structure then you know     
1:25:25     
that that's going to take longer to get those in place so finally you know we can ask the     
1:25:31     
question how do we make cells go faster so decluttering the genome is one answer     
1:25:37     
we can just remove all the extraneous genes leave behind only bar Essentials     
1:25:43     
so we have these what we call minimalist microbes and these are these minimal genomes uh so there's the gcvi sin 3A     
1:25:51     
which is the one that we just saw with the minimal cell here this is based on this kind of minimal     
1:26:00     
genome and you know that minimal cell is half of the original size of the genome     
1:26:06     
so we have a bacterium that has you know we have like a certain number of genes in a well type     
1:26:14     
we reduce that by half and we get this minimal cell it's minimally competent as     
1:26:19     
well so they don't always survive but you can they are living they they have     
1:26:24     
all the essential cell genes they need to you know reproduce which is their     
1:26:30     
imperative but they have a minimal set of gen so this is not like you know this     
1:26:36     
only gives you the ability to the blueprint of a minimal cell Min billus um and this is of course this     
1:26:43     
paper where they do this in billus uh where that we have analyzed     
1:26:49     
the Genome of be subtilus and selected all genes that require uh are required all life in complex medium at 37° C the     
1:26:58     
selection is based on known information on essential genes and functions as well as on Gene and protein expression data     
1:27:05     
and Gene conservation the list presented here includes 523 and 119 genes coding for proteins     
1:27:12     
and rnas respectively these proteins and rnas are required for the basic     
1:27:17     
functions of life and information processing so that's replication and chromosome maintenance transcription     
1:27:24     
translation protein folding and secretion so these are the key behaviors     
1:27:29     
you know you don't really get anything else um but you need these basic biochemical mechanisms to survive we     
1:27:37     
also need metabolism cell division the Integrity of the minimal cell which means just having a structural um uh     
1:27:46     
home for the for the other parts of the cell the completeness of the selected metabolic pathways reactions and enzymes     
1:27:54     
is verified by the development of a model of metabolism in the Minal cell a     
1:27:59     
comparison of many bacillus Gene genome to the recently reported designed minimal Genome of micoplasma mades which     
1:28:07     
is the jcvi S 3.0 indicates excellent agreement in the information processing     
1:28:13     
Pathways whereas each species is a metabolism that reflects specific Evolution and adaptation the blueprint     
1:28:20     
of minius presented here serves as a starting point for a successful reduction of the bpis     
1:28:27     
genome this is where they want to reduce this genome so much that they can really kind of get it down with the poor genes     
1:28:34     
it survives but does a little else it just produces enough to have a cell that     
1:28:39     
isn't you know Dead uh on arrival or you know can actually reproduce itself so     
1:28:46     
this is you know really kind of getting down to the bare minimal and so even then you know we still have problems     
1:28:52     
with getting cell division to go faster we can get these minimal genomes     
1:28:58     
but it still takes time not only to replicate The genome build all of the     
1:29:03     
proteins that are needed to build a whole     
1:29:14     
cell these case studies warn that minimizing genom don't necessarily for growth rates even though it should be     
1:29:20     
theoretically possible     
1:29:34     
and so in eoli we have one origin of     
1:29:39     
replication in Wild type but synthetic biologists at Yonce University in South Korea found that by adding two extra     
1:29:46     
Origins at different sites in the chromosome that Propel cell division so what they're talking about here is that     
1:29:52     
eoli of course is a circular genum and there's one site origin site or origin     
1:29:59     
fork for replication so it starts at that point goes around and it takes a while to go around and uh with the     
1:30:08     
polymerase and make a copy but if you have two extra origin sites different     
1:30:13     
sites than the chromosome you can actually copy The genome in less time     
1:30:20     
and speed abelation so they worked with a low dividing C ecoy string called mg1655     
1:30:27     
which normally takes one hour to double up by adding two more Origins they shave     
1:30:32     
four minutes off the doubling time so this is probably you know trivial in terms of doubling time because you would     
1:30:40     
think it might be uh you know cut it by a third but it only shaves four minutes off the double though they only achieved     
1:30:47     
a small boost the data is encouraging the cells were viable despite carrying extra Origins and the findings revealed     
1:30:53     
that tweaking the number and position of these replication start sites speed up cell division rates it'd be interesting     
1:30:59     
to see if adding Surplus Origins and fast fiting equiz strains could achieve a similar     
1:31:06     
effect we could also move genes around so scientists recently found that replication influences the order in     
1:31:12     
which genes of the bacterial chromosome are expressed so during a replication in cycle in eoli for example genes are     
1:31:20     
switched on in the order they are copied starting with genes located near the origin thus producing extra copies of     
1:31:27     
these genes sooner and boosting their expression so you can actually create     
1:31:32     
maybe more copies of a gene to boost expression of genes of that Gene or you     
1:31:39     
could Shuffle genes around so that they would position those accelerate growth     
1:31:44     
position those that accelerate growth near the origin so again you can you know select for certain types of genes     
1:31:51     
replication and play around with that this is interesting um and so for a 2021     
1:31:58     
study researchers used minimal micoplasma cells to pinpoint genes essential for cell division finding     
1:32:04     
seven so there's seven essential uh cells or the seven     
1:32:10     
essential genes in sort of promoting cell division the most notable genes     
1:32:17     
includes Fitz Z and SE F and Fitz Z is of course is involved in the ring that     
1:32:25     
that sort of pushes the cell into two different cells so like does this budding where it it produces this     
1:32:32     
protein ring and then it contracts and it it shifts the     
1:32:38     
single uh cell into two cells it it allows the cell to sort of     
1:32:46     
divide into two parts which so fits Z in SE F and code     
1:32:53     
protein means that split a dividing cell in two by assembling into constricting ring that tightens and splits apart the     
1:32:59     
cell membrane at the dividing line between buting cells without the cell splitting Machinery mop plasma cells     
1:33:06     
inflate and swell sometimes up to 25% 25 times their typical volume but do not     
1:33:12     
divide so this is important that we have these genes in place so this is an example here of fit Z proteins forming     
1:33:20     
this contracti ring it just kind of squeezes the cell body into two parts and then of     
1:33:27     
course it divides into two cells the DNA migrates into     
1:33:32     
each copy and that's what we have so this is a molecular model showing Fitz     
1:33:39     
proteins in green and red here extracted from a microb con uh constricting a liposome     
1:33:46     
which is blue so the liposome is the uh membrane of the cell and it's     
1:33:51     
constricting it into two different compartments additionally way rather     
1:33:58     
than trimming genomes perhaps doubling times could be reduced by trimming down ribosomes so maybe we could have fewer     
1:34:04     
ribosomes but remember we need more ribosomes to have more proteins produced     
1:34:10     
so that's always going to be a tradeoff recall that a typical ribosome in bacteria is built for 50 proteins and     
1:34:17     
three strands of RNA it is the RNA that catalyzes chemical reactions structural biology just     
1:34:23     
already designed blueprints for a minimalist functional ribosome so we're not only looking at minimal cells but     
1:34:30     
we're looking at these minimal ribosomes unor ribosomes might take less time to synthesize and therefore boost     
1:34:37     
speeds so this is where you know we're getting into a lot of these minimal models and seeing what kind of is the     
1:34:44     
lower bound of this of this speed how fast can we     
1:34:49     
divide and so researchers in this study up here     
1:34:55     
which're TR we get into identify 27 ribosomal proteins not conserved across     
1:35:00     
bacteria archa and UK carios how much time could the cell save in synthesizing     
1:35:05     
ribosomes if it relinquished these components that are not conserved so     
1:35:10     
this is where you know we're kind of getting rid of all the adaptations and we're getting like a poor ribosome     
1:35:17     
structure match these earlier estimat made by researchers at Caltech if you assume the ribosome is made up of 50     
1:35:24     
proteins of equal size fing 7500 amino acids then these 27 would account for     
1:35:30     
roughly 4,50 amino acids at a rate of 15 amino acids added to the growing protein     
1:35:37     
per second researchers could theoretically shave off about four minutes half the time taken to     
1:35:43     
synthesize a ribosome so now we have this four minutes Time Savings again but     
1:35:48     
instead of in The genome it's in the ribosome so this is actually about half the time it takes to synthesis size of     
1:35:54     
ribos so this is again we can shave time off of this process in this case it's a     
1:36:00     
little bit more uh relatively speaking it's a little bit more significant but uh this is of course     
1:36:08     
there are problems with this in practice some of these proteins may prove crucial to eoli ribosomes to function even if     
1:36:14     
they're not conserved across three domains of life so reducing the ribosome to the scale might not work and again     
1:36:21     
you know you're just trying to find sort of the uh sort of the common ancestral ribosome but of course in the context of     
1:36:28     
the cell it may be the ecoi needs the specialized ribosomes and you know     
1:36:33     
that's that's another thing that we have to consider with minimal cells what can we reduce without killing the ler making     
1:36:40     
it non-competent so finally there's a section going Offroad so this is more of this sort of     
1:36:48     
back at the envelope calculations of different systems looking at like uh this this     
1:36:55     
organism called vion nitrogens this microbes Hasty division has recently drawn the attention of scientists with a     
1:37:02     
minimal doubling time from 9.4 minutes was the speed to be COI builds the place     
1:37:07     
as one of the fastest dividing organisms known as science colonies of V nitrogens appear     
1:37:13     
on agar plates in as little of six hours as opposed to 14 hours fre Coline unlike     
1:37:20     
gridium uh refringens divides faster double L to approximately every 6     
1:37:25     
minutes of our clostridium refenes were stricky to work with in the lab so this     
1:37:31     
is this gridium which is hard to work with so we would like to have this fibrio nitrogens as the model organism     
1:37:40     
one thing being strictly anerobic it does not survive all and exposed to Oxygen Plus it's a human pathogen so     
1:37:46     
it's hard to handle uh but the nitrogens of course has none of these limitations it's easy     
1:37:53     
to culture and modify and so at synthetic genomics in California     
1:38:00     
they've minimized its genome for industrial purposes excluding 194,000     
1:38:05     
DNA bases so they don't delete the genes one at a time they don't do these combinatorial knockout experiments in     
1:38:12     
this case what they did here uh is to compare the chromosomes of two V     
1:38:18     
nitrogens strain side by side this allowed them to spot differences in their genomes highlighting regions that     
1:38:24     
weren't conserved and could be jettison so we're only interested in the conserved regions between individual     
1:38:31     
cells or celles and so unlike in the JVC 10 3A mini basilis models where they do     
1:38:39     
these combinatorial knockout experiments where they get rid of genes that are sort of are     
1:38:45     
non-essential uh they actually just get rid of Base you know uh DNA bases that     
1:38:51     
are not conserved and so so now you're actually discarding genes that encode     
1:38:56     
virus sensing Machinery um and so these are things that you don't need in the lab there all     
1:39:02     
sorts of things that you remove that um you know may be needed in nature but     
1:39:08     
aren't needed in a very fixed environment so this makes it easier to     
1:39:14     
engineer but also it makes it faster to divide so we're trying to reduce the time it takes to     
1:39:20     
divide and so this is where you know we just have to imagine you know how     
1:39:26     
efficient can we make these systems how fast can we speed up cell division and     
1:39:31     
you know when we're working in the lab we don't really need to have a robust Geno we just need to have something that     
1:39:37     
divides quickly so if we can engineer that then we're you know we're we're better off and so we'll probably never     
1:39:44     
match the speed of computer programming but we can improve using model organisms     
1:39:49     
and some of these bioengineering techniques so hopefully you found that interesting     
1:39:56     
thanks for paying attention and I hope you learned something
