## TRANSCRIPT

0:05     
hello hello welcome to the meeting before we start are there any updates anyone would like to     
0:12     
do um not much of an update I went to that course on T     
0:19     
segr um and discovered that some people are     
0:24     
uh programming T segres in Mathematica     
0:30     
although it's very slow slow doing um a routine triangle so     
0:39     
um not too sure about that um I don't know I guess uh there is     
0:47     
trouble um working with finite element analysis     
0:53     
oh okay um yeah I met the author of     
0:59     
someone who I read who has been using finite element     
1:04     
analysis with pens sries and she says there are um some     
1:10     
trouble spots yeah you can say that     
1:17     
um I could maybe do a more complete overview later right now I'm going on a     
1:23     
trip to to VC so British Columbia yeah     
1:30     
what was this meeting called is there a website or     
1:41     
www.c.org studies and yeah so this is their the     
1:47     
host of the event yeah activities events yes t i they wanted me to get up     
1:55     
at two in the morning so I sort of missed the first two lectures because I could only get up at 4 oh yeah I know a     
2:03     
sustained basis I was okay getting up at four but not at two yeah so they they     
2:11     
looks like they have a lot of uh good stuff going on here and the mechanical Sciences it     
2:17     
doesn't look like they have the event on their website but yeah I mean all the     
2:24     
course there's the courses yeah um of course they if it's over was last     
2:33     
so okay oh wow they have a lot of stuff here they have bone cell and tissue     
2:39     
mechanics uh hysteresis and functional materials uh drops Jets and     
2:46     
films T segurity Systems here we go Advanced courses this is T segurity     
2:52     
Systems from biomechanics to Mechanical meta materials and uh yeah so they have so     
3:00     
basically it's uh in tensegrity structure they     
3:06     
pressible trust structures obtained by connecting comprehensive members ORS your stret your pre- tensile     
3:14     
pre-stretched tenso elements cables or strings and but of course in your case     
3:20     
you're not interested in the trusses as trusses you're interested in in sort of     
3:25     
the cellular context so did did they have like a did you find what they were     
3:31     
talking about relevant to um what you're doing or was it like did you have yeah     
3:36     
think um they had a few lectures about     
3:44     
biomechanics um um yeah the last set of lectures was on Strictly on     
3:50     
biomechanics so the person who helped me was Anna and I can't pronounce her last     
3:58     
name I'm sor sorry I'll put it in the chat okay yeah right that sounds     
4:07     
good so yeah look interesting I have a number of things here they had     
4:12     
tensegrity metam materials basic concepts Tegrity structures or space     
4:19     
applications extremal mechanical properties of tensegrity structures     
4:26     
nonlinear biomechanical character of tensegrity and of course the basic sort of overview     
4:33     
of tensegrities so there's biotens segries which are the tensegrities     
4:38     
mimicking biological systems biotens segres mimicking the anatomy and physiology of living     
4:45     
organisms form Finding in the linear regime of self-equilibrated biotens     
4:50     
segries mathematical modeling of the shape change of biotens segres computational approaches and numerical     
4:57     
examples but that's good and then we can actually look at uh     
5:03     
the see if we have the suggested readings yeah they have a lot of     
5:09     
suggested readings here a list of readings if you go to their website and     
5:15     
so they have uh actually things like the Steven levven work um the 10 secrity     
5:22     
trust is a model for spying biomechanics um the significance of     
5:28     
closed kinematic chains to bi iCal movement and dynamical stability so it looks like there might     
5:34     
be kind of a gap at the cellular level at least like in modeling     
5:39     
cells um at least in the readings they have here but but it looks interesting it looks     
5:46     
like there's a lot to kind of uh learn from     
5:52     
that Pocky could you present your screen or can you present your screen and we can do an overview Stu     
6:00     
I'll just share my one second okay so I'll just uh work you     
6:08     
guys through the code so well uh for my project I was just uh     
6:17     
trying to create a growing graph neural network and for that I was using the     
6:23     
evolution algorithm so here I have just initialized the first two     
6:33     
notes     
6:41     
right and then uh uh sorry uh yeah so here I have uh     
6:50     
like I'm running through the uh data set which contains uh the parent cell along     
6:58     
with the daughter cells and also the division time of that parent cell which is like the birth time of the daughter     
7:05     
cells so uh yeah so here course it will like just     
7:11     
uh get the parent like labels and the daughter cells and the index of the     
7:17     
parent and daughter cells and then I'm calling the grow function in the ncf     
7:23     
file yeah yeah so uh what does grow function     
7:29     
basically uh does this it's like calling like functions to add the doter cells     
7:34     
and it is also like uh splitting it's using a split Network to like create the     
7:41     
like split the divide the parent into two daughter Sals so this like for the     
7:46     
arguments it is taking like the graph and the parent index and the daughter labels and it's returning like the     
7:53     
updated graph with the new DOA nodes and the list of the updated noes as well so     
7:58     
this add DOA cells function is uh in this file so this file is basically handling the uh growth of the graph     
8:06     
structure basically like adding new nodes and adding edges and plotting the     
8:12     
graph so and in this file we it's NC stands for neural cellular automator so     
8:19     
it is just basically handling calling the functions from the other other file and Performing like optimization     
8:26     
functions on the graph structure so uh we called the add da Sals in the     
8:35     
grow function where it's just a adding the daughter cells and I've also created     
8:41     
a dictionary for the new labels because in the graph I also     
8:46     
wanted to show like the label of the cells that are getting divided     
8:52     
so uh when uh the new cells are added it is also getting that uh     
8:59     
Edge dictionary which is like a The Edge dictionary basically contains the edges between every pair of cells so when like     
9:07     
a parent device it forms edges between the cells that the parent was previously connected to with the new daughter cells     
9:14     
and like a like an edge between the daughter cells as well so uh it's uh calling other functions     
9:23     
like update as dictionary and add or remove nodes so it was just removing     
9:28     
like the parent node and adding like the da nodes in place of that and then uh we     
9:37     
back here like it Returns the updated graph in the nodes and then this uh I've used CM function actually back     
9:45     
propagation works on a linear growing networks and here the parameters are also getting updated as new nodes are     
9:52     
getting added so uh I was U falling into errors with that propostion so I thought     
9:59     
using the CMA as algorithm so this     
10:05     
uh so this function basically uh it is applying the evolution strategy and uh     
10:13     
here it's U like I am giving random positions to     
10:19     
the new daughter cells that get added so it will like uh find the loss between     
10:25     
like the two distance that should have been there like distance between two no it's not learning position but like the     
10:31     
relative distance between the nodes because like the initial position that I gave was random so it is just like     
10:38     
learning relative to that so uh after the point it will like learn how to     
10:43     
place the daughter cells where so here also I've called this objective function     
10:51     
so which is basically optimizing calculating the loss and like setting     
10:57     
the parameters and all that for uh finding the relative distance I thought     
11:02     
of using a matrix so that it can optimize the distance between every pair and not just like the pays between the     
11:09     
new cells like at every iteration it is optimizing the entire Matrix to learn     
11:14     
like globally as well and then uh after like updating all this m is     
11:22     
like returning back the graph so uh there are two Loops here because at one     
11:27     
time stamp like two different can divide so I just wanted to take that also into     
11:33     
consideration so the second Loop is just for all the cells that are dividing at the same time stamp and then it's just     
11:40     
saving the lots of like various time stamps and uh those are then later used     
11:46     
to generate this 3D plot also like I due to like GPU     
11:53     
limitations like I don't have GPU on M so I just I couldn't turn it through     
11:59     
like like the entire data set I could only run it till like 134 times St after that like it was just going on forever     
12:05     
it was the kernel was crashing or it was just not learning so I like I'm stopping     
12:10     
it there but uh in this also we can see like it is uh like it is learning     
12:18     
something like around 57 58 mes are getting generated so it is getting sort     
12:25     
of optimized you can see that like within this much noes     
12:31     
s so this is like a gif and there is also like a final State like a static     
12:38     
image of like the last stage where the graph was at     
12:48     
so could you zoom in on that a little bit like zoom in your screen     
12:53     
like there we go yeah yeah so that's the sort of the     
13:00     
optimized uh lineage graph I guess you could call it and yeah this is just like     
13:05     
50 cell so we can't really see like the shape of the growing embryo so like     
13:11     
maybe with more nodes it would be better visible yeah yeah and that's just the ab     
13:17     
oh actually no it's it's both I see so it's just kind of in that     
13:25     
space yeah okay yeah that that looks good I mean I don't know     
13:31     
if was the input data normalized uh did was it centered on zero or is this like     
13:38     
just the raw position data oh sorry I didn't get that your     
13:44     
voice was breaking a little yeah did you have did you use the     
13:51     
data that was centered upon zero or did you use the raw distance de     
13:59     
I'm trying to what I use the draw distance data okay I'm sorry oh if you use the     
14:07     
raw distance data you you might get something that looks I mean because we     
14:12     
have the normalized data which is basically like it will give you the shape like a point cloud of the embryo     
14:19     
but I'm not actually sure what it's doing here it might be like producing some sort     
14:24     
of uh you know Force layout or something but anyways yeah I'm just curious as to     
14:31     
how it's working with the position data and how it's rendering all the nodes     
14:37     
but it'd be nice also like if we could pull out like different little pieces of     
14:43     
the network I don't know if it can do that um but like if once it grows to a     
14:49     
certain size if it can take like a snapshot and like take out certain does     
14:54     
it save like the when it grows the network does it save the output     
14:59     
somewhere or is it just kind of like generates the     
15:15     
visualization sorry I didn't get the last part I think because of the screen share can you repeat that uh so does it     
15:24     
uh does it uh allow you to save the positional one information from the data     
15:30     
or does it just display the visualization like is there an output file with positions in it in the sort of     
15:37     
the layout or the I guess the connections between     
15:42     
them oh not as of yet but like the positions are there in the embeddings of     
15:48     
the nodes like the XYZ coordinates are in the node embedding so I can like add     
15:54     
some code to like save that as well yeah yeah be good I mean you probably don't     
15:59     
have to do it for your final submission because I think that's today but just you know it it just makes it easier to     
16:06     
save at a certain point to see kind you can take it take that output and put it     
16:11     
into a a rendering program and you can do other types of network analysis on it     
16:17     
so that's the only reason I'm asking all right I'll add that as well     
16:25     
yeah after like today submission add that later okay yeah it looks good um yeah and     
16:32     
don't worry about the uh you know I know we had a problem with uh being able to run it optimally on a CPU so if we don't     
16:40     
have a GPU which is hard to you know find uh time to run on because it's     
16:47     
expensive um you know we'll have to just make do with a CPU and so it just has to     
16:53     
you know have to figure out a way to run the LI Brio on that maybe or uh maybe     
16:58     
get access to a GPU later but I think for the purposes of gso that you have is     
17:03     
perfectly good and I I think it it was a pretty good project I think you did a lot of work on getting these different     
17:11     
pieces put in place um and I think we've got a nice little tool here to build     
17:18     
networks to grow networks and then you know we can actually intervene in the networks once they're as they're growing     
17:25     
so you get them to like 30 nodes 40 nodes you can stop the process you can look at     
17:32     
what that process has yielded after 35 36 no well would be in multiples of two     
17:38     
so 36 nodes and uh you know and and you know     
17:44     
it'd be interesting because we can look at a lineage tree like that we have an interactive lineage tree and we can stop     
17:52     
it at like a certain number of cell divisions and look at what's there and     
17:57     
look at the structure of that tree um and but you know with a network it's     
18:02     
kind of hard to do you have to kind of work backwards and it's just not easy to do because you could you know so that's     
18:10     
that's one advantage of this kind of approach um now you know like for people     
18:16     
who aren't familiar with networks one of the things that you can do with a network is you can lay them out in     
18:21     
different ways so what Pocky you know was showing was like how the program is     
18:28     
rendering the network and you know I don't know if they're rendering how they're rendering it that's why I asked     
18:34     
about the force layout so there're these different types of layouts there like a circular layout where you can lay the     
18:41     
nodes out in a in a circle in a and then you know the nodes are kind of connected     
18:47     
around that Circle so you see these chords that kind of uh are displayed     
18:52     
along the edge of the circle and that's local connectivity between nodes uh you you know there's that kind     
18:59     
of layout and then there's this uh Forest layout which is where you have the nodes pulled out in different     
19:06     
direction so it won't look like an embryo um but it will kind of optimize     
19:12     
the layout and so you know like if you're doing these kind of networks I don't think there's a good     
19:20     
default visualization I mean they're all going to have their drawbacks um and you know so it won't     
19:27     
actually show the embryo necessarily the way that we would like but which is fine we're     
19:35     
really interested in the network structure um but it's you know it's it's something that's hard to really kind of     
19:42     
because if we do put it in this kind of Point Cloud you know we'll have uh we'll     
19:48     
have problems kind of looking at the connections and things like that so it's     
19:53     
always it's going to be a challenge kind of to build these kind of network visualizations     
19:59     
that might be a g project for another summer so I wouldn't you know I just wanted to point that out too but I think     
20:05     
Pocky has really done a good job here so congratulations and uh you know I are     
20:11     
you ready for submission or have you already submitted or what's the what's     
20:17     
the status on yeah like for the submission like there are some questions     
20:22     
that I need to like fill up so I have like them ready in a draft I just wanted     
20:27     
to get riew once before I like submit everything okay that's good yeah yeah     
20:34     
that's good so yeah uh please make sure you know what time the submission is uh     
20:40     
the deadline is I think it's coming up pretty quickly here and the next several hours     
20:46     
so and then I have H you have a poll request I'll accept that after yeah okay     
20:53     
after the meeting so very good thank you     
20:58     
so mahul you're next um are you ready to share your     
21:04     
screen okay uh mahul had trouble sharing his screen and talking at the same time     
21:09     
because of bandwidth issues so I'm going to go over his contributions to the daph     
21:15     
repository specifically the hyper daph stuff so this is hyper daph modeling C     
21:23     
elegant develop typograph this is mahul Aurora that's his presentation his uh     
21:29     
Google sumar code submission looks like he has analysis scripts he has some data     
21:34     
sets some visualizations and a readme so let's start with a readme and looks like he has uh you know     
21:43     
his data sources documented the self lineage data so his     
21:50     
one of his data sets was the CE raw source which is from our GitHub repository the people War GitHub     
21:57     
repository this gives you spatial coordinates of Developmental cells and     
22:03     
also gives you the lineage tree and then for self phenotype lineage data he has     
22:11     
self phenotype lineage data XLS which is also in from dorm repository this is     
22:19     
where we have the lineage information more explicit lineage information where the daughter cells and mother cells are     
22:27     
aligned and then we have of course data fields cell which is the unique identifier for each cell that's the     
22:34     
nomenclature time which is the point of observation so in these cell tracking studies they usually observe the embryo     
22:42     
at specific time points they don't necessarily match to the minutes of development but they do give you     
22:49     
independent and individual uh time points that you can then align over other observations and     
22:56     
given the nomenclature of the cells you can then match it up with other data that tell you when in terms of     
23:04     
Developmental time in terms of minutes when a certain cell is born and when it     
23:09     
dies um then there's of course the size of each cell and each time point this is     
23:16     
not necessarily very useful for one thing it's not the total size of the cell in terms of membrane volume it's     
23:23     
more uh some marker that they're using it's a fluorescent marker that extends beyond the nucleus but it's uh     
23:30     
indeterminate in terms of the actual cell size and then mother which     
23:36     
identifies the Mother cell which of course is useful for the hyper graph aspect of     
23:42     
this so he prepared his data by normalizing the spatial     
23:48     
coordinates uh normalizing to the center of the embryo uh so this is just uh taking     
23:54     
either like um what Pocky did was use like a standard     
24:01     
deviation you can also use zcore to normalize the data on each access to a     
24:07     
zero point and then have a right and a left side an anterior and a posterior     
24:14     
side and a dors on a ventral side for X Y and Z     
24:20     
respectively uh cells of missing mother data include P1 P2 P3 P4 Z2 and Z3 and     
24:28     
of course this is uh I should add in here that these are germine cells P1     
24:34     
isn't a germine cell but going starting I think at P4 as a germline cell maybe     
24:39     
P3 but well that we have mother cells but he didn't have that in the data so     
24:45     
it doesn't it's not really that important that we have mother cells for those but technically they're mother     
24:51     
cells for P1 through P4 um mean coordinate and size of each     
24:56     
cell are calculated across the time points so again you know working with the data and getting it in shape and     
25:04     
then the analysis involves graph creation so for the graph creation step is the statically fully connected     
25:10     
spatial graph and this is where a graph is created using the coordinates for each cell so finding that location of a     
25:17     
cell in the data set depending on you know how finally you slice things uh you     
25:25     
have sometimes you have multiple instances of the same cell from different parts of the data set so     
25:31     
what they'll do is they'll sample you know hundreds of embryos and they'll look at each embryo and so you know they     
25:37     
get parts of the time series and so aligning that sometimes you get many multiple instances of the same     
25:45     
cell sort of clean that up a little bit and normalize it uh so you did the     
25:52     
static fully connected spatial graph using the fman rle layout algorithm so     
25:57     
we were talking about layout in picky's work the algorithm she's using does some     
26:02     
sort of force layout which you know is good for maybe optimizing the shape of the graph but not necessarily for     
26:10     
mapping it to an embryo and a lot of layouts aren optimized for that because that's not what people use graphs ands     
26:18     
for typically but this per L wrangled layout that's a good choice I suppose um     
26:25     
we'll have to get a wink for that in in there um the spatial hypergraph creation     
26:31     
distance threshold based approach and clustering based hypergraph or DB scale so you talked about this in previous     
26:37     
meetings where he's using the distance threshold based approach which is where you have cells that are connected based     
26:45     
on their distance and so if they're a certain distance away from one another they're connected if they're farther     
26:51     
than that threshold then they're not connected and so that tell that gives you some information about cells that     
26:58     
are adjacent um and the cells that are far farther away you can also use other     
27:04     
types of threshold based approaches like nomenclature identity because sometimes     
27:10     
the ab cells kind of spread out across the anterior pole and some of the other     
27:18     
cells that are in the P1 sublineage do the same so you could actually use the     
27:24     
identity of the cell as a threshold based approach and see that yields but he he did the distance based approach     
27:30     
then the clustering based hyper graph clusters uh cells by their proximity so     
27:36     
this is again another way to do the distance threshold based approach it's a little bit more rigorous because you're     
27:42     
discovering groups instead of just using a strict threshold distance normalized     
27:47     
distance then he does the wi Edge hypergraph creation so you have the wi Edge base hypergraph which connects     
27:53     
cells based on their common mother links visual wise using the f rle L out a     
27:59     
dynamic hyper graph a conceptual model that dynamically represents changes and cellular relationships over time so this     
28:06     
is where take that linear hyper graph model so it's changing its um     
28:11     
composition and so in that sense it kind of links back to what uh Pocky is doing     
28:17     
with growing embryo networks so this is a nice uh sort of the other side of the     
28:23     
coin for gck so we had you know we had have this uh the growing networks the     
28:31     
growing sort of lineage networks developmental uh networks and now we     
28:36     
have this hypergraph and of course the hypergraph is different in that it nests nodes within hyper nodes and and edges     
28:44     
within hyper edges so you have a lot more complexity here um so this is the hyper graph     
28:51     
analyzer class this just goes through the analyzer class and python the     
28:56     
hypergraph Creations you have the proximity hyper graphs you have function create proximity hyper graphs with the     
29:02     
threshold will be determined by the user uh analyzer create DB scan hyper     
29:08     
graphs you have some parameters to enter in uh you can add noise in so you can find these clusters and assign     
29:17     
theming visualization where you have draw hyper graphs proximity hyper graphs     
29:22     
and DB scan Hy graphs okay so then you have to import things in Python 3.11     
29:28     
hyper hypernet xit scii at plot Li pandas     
29:35     
and um we have the data files as instructions on how to run it and then     
29:41     
notes on some of the ways to use this this is very good work uh from mul thank     
29:49     
you mahul and good luck on your submission okay uh so that's good thank you mahul uh we'll we'll come back to     
29:56     
that in a little bit when he pushes the code and I can go through it or     
30:02     
little um I don't know Hussein did you have an update I know you were     
30:07     
waiting and you've been putting a lot of things in the in one of the slack channels yeah sorry are you asking me     
30:14     
yeah yeah thanks thanks yeah thanks so much um yeah I can go ahead and share my updates that people that are ready for     
30:20     
that um so yeah okay excuse me so through orl through the do this work     
30:27     
have been working on the RCT project the breast cancer Community topography project making a lot of work looking into resources looking into applications     
30:33     
I've got to a past working to be able to form simulations but right now we're prioritizing our effort on two papers     
30:39     
right now the um a manuscript that's due um September 31st electronic steering of     
30:47     
uh electronic steering of xray sends um excuse me I'm just typing things up     
30:55     
noct steing of xrays those the first one as well as the     
31:02     
um the electronics steering of XRS and uh finishing up the uh the the book     
31:08     
chapters collectively that we've been sending out through the emails too so um     
31:14     
I think that's basically it I uh we got to PS working for the simulations but we     
31:21     
don't have enough time to put it on our manuscripts just yet it's not quite it's not quite ready     
31:26     
so um     
31:31     
we getting a lot of work together showing the graphs and getting the figures together for the manuscripts and um overall looking into the ideas of how     
31:37     
we're going to construct the computer tomography technology and the platforms that we use like the physical technology     
31:43     
the how to engineer things as well so um a lot of we're definitely thinking about a lot of things at once like not just     
31:50     
from the code like the intense coding sessions dick and I have been having but um you know how to     
31:56     
mathematically compute and com calc what exactly we're looking for the best ways to represent that how that can     
32:02     
be encoded into some sort of language or some sort of um some sort of to program for     
32:07     
scientists and Engineers to use and um you know showing for example how like the peak vary with the     
32:14     
angle like as as like like many many different products and many different ideas so like like for the angle     
32:20     
deflection like the paper of the angle deflection showing how long so reflection could be demonstrated as like the function of deviation from an ideal     
32:26     
angle so all of this by the way just backing up a bit for those of you guys aren't on the same page here it's like um all about um excuse me seeing how     
32:36     
well and what ways we can steer the x-rays and still maintain maintain feasible results in our technology so     
32:46     
yes yeah that sounds good yeah I noticed that you had uh in the orthoga web slack     
32:52     
you'd put a lot of things in um this breast cancer CT Channel     
32:58     
and like kind of like running notes for what you were doing so I found that interesting some of the stuff that was     
33:05     
going on there so yeah looks like you're doing pretty well on     
33:10     
that yeah we're we're stay we're staying pretty busy um progress can be progress can be pretty linear and Progressive at     
33:16     
times other times progress can be moving forward then getting stuck and moving forward and getting stuck more like um     
33:22     
things like that too like I can show the uh figure that I generated to demonstrate     
33:28     
what we are envisioning in terms of the what we are envisioning in terms of     
33:34     
the genos speres and how they reflect uh let me pull this up um like what     
33:40     
the excuse me what I'm just what I'm describing is what the setup would look     
33:45     
like in terms of the the figures themselves uh see it's inup notebook i' s the figure to dick so I can just open     
33:52     
it quickly onto my computer okay want to is your     
33:59     
okay hold on is your is your microphone working I I didn't actually see did you send the messages     
34:05     
uh is your microphone working yes yes okay it is working it is cool yeah the     
34:11     
microphone is but you're showing as a black screen here no problem I'll show     
34:17     
my uh the figures that had yeah excuse me the figure     
34:24     
sense okay uh let me share my screen okay     
34:30     
tab yeah see this right yeah that's good okay so you might be wondering what okay     
34:37     
yeah so um geospheres as before the types of particles that can be uh J     
34:43     
geospheres named after the Roman god Janice who is depicted having two places so as such the Janos speres were named     
34:49     
this way by being particles that had two different properties on one sphere and the other sphere so we can rotate them     
34:55     
in such a specific angle such a specific way when they're regularly spaced at at particular intervals we want to see how     
35:01     
when shooting x-rays from the top to the bottom we can pos position them ever just so slightly exuse me so slightly so     
35:09     
that we can um we can control them in a dynamically stable dynamically sted man     
35:16     
like this so um basically each subsequent row is spaced in such a way     
35:23     
that the the the sphere from the incoming one must hit the center of one for the below one below it and they need     
35:29     
to extend in such a way that we have a very continuous trajectory through all Janos speres and this is what it's this     
35:35     
is uh how it's demonstrated for the eight that the beams are being shot through they're actually supposed to be     
35:40     
rows of 10 so I can actually add it to this one I can actually add it to this one to make another row from the     
35:46     
beginning and another row from the end and um we're only we're only shooting eight beams um excuse me we only shoot     
35:55     
beams there's U excuse me excuse me yeah we shoot we     
36:02     
decid to we only shoot a beam skipping the first one skipping the last one and I think that was just for like practical     
36:08     
mechanical feasibility purposes so yeah yeah like um ideally deflection occurs     
36:17     
um IDE excuse me IDE the reflection occurs the reflection the actual     
36:23     
reflection occurs more effectively and readly when SP us first in the Spheres     
36:29     
that are within so there should be 10 spheres each mode but this is just to demonstrate what we are what we are what     
36:36     
we're doing and that's how that's how Janis spes work based off of our uh technology as we propose it so even     
36:42     
being able to visualize this in python or image J I think that's really really cool um to at least demonstrate and show     
36:48     
what we're looking for because that's that's that's mathematically what it would look like being able to have that much control over something on such a     
36:54     
fine tun level with the with that Precision it's just really really really ideal remember how we we had looked     
37:00     
through many different software and we're still using topos so as long as we can visualize things as we need to and then perform the simulations for very     
37:05     
high level high order of statistical cation calculations and computations that's ideally what the academics with     
37:11     
the Publishers are really looking for you know like um and then maybe if I want to get creative like making like an     
37:16     
animation and blender or something um be kind of fun to you to get creative but yeah yeah um I used to draw rats back     
37:24     
when I was writing papers about the Morris water maze I would spend my time drawing Rats on on my     
37:30     
remember draw pictures of rats swimming into tubs of water to demonstrate the Morris modern BS and the effects there     
37:38     
so this is not as exciting I I don't know if shooting shooting x-rays into     
37:43     
spheres is as exciting as walking R swim but you know anyways thanks thanks so     
37:50     
thanks so much guys overall um overall it's definitely an uphill battle like dick and I were talking about like I     
37:56     
we've been sharing many many resources of like other forms of breast cancer communter topography in the slack     
38:01     
Channel or other other examples of commun tography going on in general such as like one from     
38:07     
um excuse me one from these uh scientists based in like Germany for example But ultimately ultimately we're     
38:14     
just trying to put things in perspective that's all so yeah yeah well that's good that sounds     
38:20     
great thank you um so the Janos speres are particles of two different properties so like you would engineer     
38:27     
these or can you find these in nature yeah so jph is     
38:35     
um excuse me J janere josph themselves um like okay in a scientific context you     
38:42     
have the um there are yeah okay in in a real world scientific context you would look     
38:47     
at they they're synthesized they're synthesized you make them so you can produce like beads that would be you can     
38:54     
produce beads or small small spherical like glass particles of like a hemisphere even it's just like a physical object right with one side     
39:01     
hydrophilic and one side hydrophobic and those orig used to be used in um in     
39:07     
um goodness I think I think they were used by Xerox they were used by Xerox in the 70s that's really really cool like     
39:13     
there was a panel ballplay can I can I show it I can um I can show this this is all this is all like     
39:18     
part of History it's also so cool you know and you know like like in a true entrepreneurial fashion you always steal     
39:25     
from Xerox everything is stolen from Xerox Bill Gates steeve jobs they all stole from Xerox that's that's what they     
39:31     
all do that um but like the the original Xerox panels just plays and I'm just     
39:36     
reading this right from the Wikipedia stuff too um like the Nick Sheran from zerox Corporation a twisting ball panel     
39:43     
display so they would use a par particles with have electrical     
39:48     
an an anop EXC     
39:55     
me yeah so that's were using the original like zerox machines in the 60s and 70s and um like oh they have the     
40:02     
patent here cool yeah but any anyways so JIS particles themselves they're used in an     
40:08     
extremely wide range wide wide wide wide range of technology and the other     
40:13     
features that we are taking advantage of such as say like um you know like polycapillary steering sources     
40:21     
um qu steering technology sources like uh the Imaging challenges like we     
40:26     
discussed like the noise deconvolution um like br br studies work on PR noise like how you can simulate     
40:33     
breast texture with PR noise everything is all coming together and we're definitely looking at things from all these different angles but um that's     
40:39     
what makes this approach like not only so all-encompassing but also like pretty Innovative a fundamentally disruptive     
40:45     
way for breast cancer for targeting breast cancer in this particular electronic electronic paper electronic     
40:53     
paper Okay screen elic paper is older     
40:58     
than Nick that is older than I am that is that is older than me     
41:07     
um yeah don't give away your age and then we can um we can credit     
41:15     
Xerox just kidding just kidding yeah well they still haven't perfected electronic     
41:22     
paper yeah I mean um you know it's like like like um     
41:28     
okay even so even from a from a Computing point of view from a Computing point of view     
41:37     
um excuse me from from a from a from a Computing point of view full stack from     
41:42     
a from a full stack Computing point of view what technology we would use in what context we would be able to borrow     
41:48     
the scientific techniques from right now toos is the simulation software that we     
41:54     
that we're taking advantage of but we' even discuss say um Quantum Quantum     
41:59     
Computing or even Quantum bio Computing from a particular scholarship and fellowship that I'd come across too so     
42:05     
there are many many different Avenues and ways we can draw upon research and Technology to see the best ways to make     
42:11     
these things happen so it's really about keeping options open seeing what's the best um scientifically like like what's     
42:20     
best backed and stuff too so yeah yeah it is I I have the patent for um     
42:28     
I'm just reading the patent for for Xerox yeah for um the twisting     
42:37     
ball display sheets pretty cool that's what they called it what they called it     
42:45     
so cool stuff cool stuff really cool stuff good yeah it's not looks pretty good yeah so makes can redo a PhD in     
42:53     
physics just to do this yeah     
43:00     
okay thanks guys all right you're welcome yeah that's great great stuff so I'm G to present on a paper     
43:08     
um so this is a paper that uh I guess     
43:13     
this was from Morgan um spotted this uh it's a in the on the     
43:19     
archive and you know we're interested in simulating biology here in the group     
43:25     
we're interested in simulating organisms or at least things at the organismal level but you necessarily     
43:31     
have to sort of work on things sort of underneath that so things at the     
43:37     
cellular subcellular level and so forth what they're doing here is they're     
43:43     
trying to build a virtual cell with artificial intelligence now this is a     
43:50     
um you know this is sort of a vision paper it's not really you know they're not actually doing this but they're     
43:57     
basically giving the priorities and opportunities for doing this so there's a group at the University of Illinois     
44:04     
that's building virtual cells from you know data and observation so you know     
44:09     
you you've got like uh physics modeling multiphysics modeling that they're doing     
44:15     
of different parts of of bacterial cells um and you know they're actually     
44:21     
able to visualize the cell visualize things like the um gene expression the     
44:28     
proteins the sort of the components of the cytoplasm components of the membrane     
44:36     
things like that now in this case what they're trying to do is they're sketching out a a vision for building a     
44:43     
virtual cell with some AI technology so it's kind of trendy in that sense uh     
44:49     
they're you know they're kind of bringing to Bear some new techniques and seeing how they fit together uh a lot of     
44:56     
big names on this paper um I see people who I've I'm familiar with their work so     
45:05     
you know it's it it looks like a pretty big uh group of people so let's take a look     
45:11     
at the abstract and kind of go through what their perspective on this     
45:16     
is so they talk about the cell being arguably the smallest unit of life being     
45:23     
Central to understanding biology being able to accurately model cells is important for understanding disease it's     
45:30     
standard boiler plate for you know making this relevant to people uh you     
45:35     
know recent advances in AI combined with in the ability to generate large scale     
45:41     
experimental data present novel opportunities to model cells so their vision is of an AI powered virtual cell     
45:49     
where robust representations of cells in Cellular Systems under different conditions are directly learned growing     
45:57     
biological data across measurements and scales so I think what they're getting at here is that you can build a model of     
46:04     
the cell by getting data from different sources continually integrating the data     
46:10     
into the model and then that model learns from the data and produces an     
46:16     
output that improves the model with itself itself so you know it's like if we got some data from a celan cell and     
46:25     
we you know were able to model the basic structure of a seelan cell and we kept     
46:30     
feeding it data from continual observation maybe adding components in     
46:35     
we could you know better U approximate the Dynamics better approximate some of     
46:41     
the other aspects of the cell uh the physics the uh you know the the gene     
46:48     
expression Dynamics the movement Dynamics things like that so this you     
46:54     
know again we discussed desired capabilities of AI virtual cells     
47:00     
including generating Universal representations of biological entities across scales so again you know they're     
47:06     
working from a representation of a biological entity so you know so and U in I think     
47:13     
in a lot of previous approaches to modeling these sorts of systems people would work from some sort of ontology     
47:20     
certainly when we use bioinformatics data we have an ontology we have     
47:25     
functional on ologies where we look at like the function of certain genes or the function of certain proteins or     
47:32     
cellular components and you know that's that's what we're using for and those have to     
47:38     
be continually refined through experiment in this case what they're saying is they want to build representation of different parts of the     
47:45     
cell so you know what is it that characterizes say the membrane you know     
47:50     
is there some sort of data structure characterizes a membrane or is there some sort of framework we can fit the     
47:56     
data into so that we can scale up as needed uh so you know this is kind of     
48:04     
one of these Universal representations of biological entities so if we have     
48:09     
like have some change going on in the gene expression can we see that manifest in the protein or if we have a bunch of     
48:16     
proteins aggregated together can we simulate you know some sort of Dynamics     
48:21     
where it forms in a membrane or some sort of uh structure in in the cytoplasm     
48:28     
things like that then uh they talk about uh we     
48:34     
further address the challenges opportunities and requirements to realize this Vision including data needs     
48:41     
evaluation strategies and Community standards and engagement so they're talking about some of the you know may     
48:47     
maybe you know having people contribute data to this and standardizing everything so it just kind of plugs     
48:53     
right into the model um The Envision a future where AI     
48:58     
virtual cells help identify new drug targets predict cellular responses to perturbations as well as scale     
49:05     
hypothesis exploration okay so they're interested largely in building a model of cell that responds to different     
49:13     
experimental perations different environmental conditions so you can actually you know like basically provide     
49:20     
data to the cell it would learn from that and then that's it stor in a     
49:26     
represent presentation that can then allow you to build these components sort of on the Fly not only build the     
49:34     
structural components but build the functional Dynamics where you know you can see how the cell behaves but also     
49:42     
perturb that and see how the cell behave responds to that perturbation and of course that requires     
49:48     
you to do experiments where you do those same perturbations and provide the model with     
49:53     
these data so you know the model will learn from and produce some sort of similar output     
50:00     
so you know wouldn't be exact like we running the experiment but you'd get similar results and you should be able     
50:06     
to compare them with a biological experiment and they should be uh pretty much the same or very     
50:14     
similar um so that's that's a pretty lofty goal with open science collaborations across the biomedical     
50:20     
ecosystem that includes Academia philanthropy and biopharma and AI Industries a compreh ensive predictive     
50:27     
understanding of cell mechanisms and interactions as Within Reach so that's the abstract um you know     
50:35     
they kind of go through virtual cell models uh where they say to understand a     
50:42     
cell's function scientists have attempted to construct a virtual cell model to simulate predict and steer cell     
50:49     
behavior and they have these citations one through six we'll go down to those in a minute building on this vision we     
50:56     
use the term virtual cell the Divine computational model that simulates the biological functions and interactions of     
51:02     
a cell existing cell models are often rule-based and combin assumptions about     
51:08     
the underlying biological mechanisms with parameters fit from observational     
51:13     
data so you know people doing cell modeling uh usually if they're modeling     
51:18     
the Dynamics or even if they're modeling the structure you're using these rule-based systems so if you have uh say     
51:25     
for example you know some training data you might assemble the Dynamics or the     
51:31     
structure from these rules and um or you might uh model from first principles     
51:38     
what they call eono modeling so a lot of biochemical models are like that where     
51:43     
they have they work from like say chemical principles or physical principles and they say okay the way     
51:49     
that these things interact is through these kind of um you know abono principles so for example you know if     
51:55     
you have something that responds to gravity or something that responds to diffusion you     
52:01     
build that into the model particles diffuse at a certain rate where you can change the rate of diffusion but     
52:08     
basically the model will you know the the particles in the model will diffuse     
52:14     
according to that assumption and so these are assumptions that we build into the model and of course you know for the     
52:21     
most part models have to have assumptions you know you just have to be really good at what the assumption are     
52:27     
you have to be correct and I I don't know if they're arguing that you know you can get around sort of you can relax     
52:33     
assumptions through using you know generative AI or some sort of AI engine     
52:41     
that you know um kind of learns from experimental you maybe learns like     
52:46     
underlying principles from experimental data but it's it's interesting um I     
52:52     
could see that kind of going off the rails too I mean the one reason why we use sort of ammonal principles is     
52:58     
because you know cells have an underlying physics and underlying chemistry and so you know that's that's     
53:05     
a thing that's been pretty successful um but anyways um in terms of the existing     
53:12     
cell models that use these kind of uh rule-based in assumptions uh they rely     
53:18     
on explicitly defined mathematical or computational approaches such as differential equations stochastic     
53:25     
simulations or agent based model so they're talking about this is these are citations 7 through 16 so they're     
53:32     
talking about these very what to a very familiar model types you're using     
53:38     
differential equations to look at the Dynamics using stochastic simulations to generate these stochastic processes     
53:45     
within the cell or agent based models to look at the interaction of cells say within a sheet or you know some of the     
53:52     
things that we're doing with hity or with morphogenesis so this is you know again     
54:00     
I don't know if AI gets around this you know you may need to end up you know having like an AI constructed cell in an     
54:08     
agent based model and just kind of live with that because I don't know how you get around that with AI but in any     
54:15     
case um they they vary in complexity and cover different defined aspects of Cell     
54:21     
Biology such as transcription translation cytoskeletal driven Behavior     
54:27     
biochemical networks or metabolic flux and by they they mean these models up     
54:32     
here and that's citation 17 through 23 uh this is actually in reference to     
54:39     
the um project I was talking about earlier the first to cell model was developed in     
54:45     
2012 representing all 482 genes and molecular functions known for organism     
54:51     
the bacteria micob bacterium genitalium uh since this pioneering works genome wide models have been     
54:58     
developed to represent other bacterial organisms including Eola so that's kind of where they're     
55:05     
going with this um they're kind of giving the history but then saying we can do     
55:10     
better despite their widespread use in modeling biological systems these approaches have limitations in modeling     
55:16     
so okay well we're at sort of these bacterial cells these minimal bacterial cells meaning that they have the minimal     
55:23     
complement of genes the minimal complement of things like organel and structure and then you know we're able     
55:30     
to model the cell as sort of the system so it's a minimal system and it's in     
55:35     
bacterial cells which are much less complicated than eukaryotic cells so we're not quite at the level of doing     
55:42     
anything like cgans but it's you know a cell basically a functional cell um and     
55:50     
you know some of the models don't run as intended so that's maybe one of the limitations of them but still they     
55:57     
actually you know they they're achievable okay so they see some um sort     
56:03     
of shortcomings here um and this is maybe for bacteria and more complex     
56:10     
cells like eukaryotic cells so one of the shortcomings is being able to do multiscale modeling where you're trying     
56:17     
to model multiple scales or processes thatan multiple scales as we've said you     
56:22     
know you can use for example um you know sort of atomistic modeling on one scale     
56:30     
and then like some other type of physical modeling on another scale you can do that kind of a thing you can do     
56:35     
multiphysics type uh work you can also trans you know use transfer functions to     
56:41     
move from one scale to another that's a challenge that I don't know if they solve here but um it's it's something to     
56:48     
keep in mind the other one of other challenges are these diverse processes because in     
56:54     
cells you have a lot of different processes going on you have transcription you have translation you     
57:00     
have morphogenesis at the Single Cell level you have uh construction of     
57:06     
proteins you have other types of signaling mechanisms that you know go     
57:11     
through receptors so there are all these processes metabolic processes that we have to account for they don't all fit     
57:17     
together neatly either so we need to kind of fit those together and of course we're using different experiments to     
57:23     
calibrate this so it makes it even more challenging cellular function encompasses numerous     
57:30     
interacting processes such as Gene regulation metabolic pathways and signal signal     
57:35     
transduction each process involves a large number of biomolecular species and diverse and dynamic     
57:42     
configurations the third of course is nonlinear Dynamics uh which is where you have     
57:49     
cellular processes that are highly nonlinear such that small changes in inputs can lead to complex changes in     
57:55     
outputs thus despite progress in modeling specific C processes these factors     
58:01     
collectively pose a substantial Ro to the construction of a virtual cell so their AI virtual     
58:09     
cell our view of an AI virtual cell is a learn simulator of cells and Cellular     
58:15     
Systems varying conditions and changing contexts such as differentiation States     
58:21     
perturbation disease States stochastic fluctuations and environmental conditions and they have this in figure     
58:27     
one so let's take a look at figure     
58:35     
one down here more I guess not figure ones not in here     
58:42     
or something let's see we do     
58:48     
okay so this is figure one this is the these are the capabilities of an AI virtual sub so we have uh the first one     
58:56     
is a multimodal measurements across different scales of the cell we have uh     
59:02     
genomic information information of Mo molecule localization phenotypic     
59:08     
information and we can use different types of data to sort of calibrate our model and of course you have to merge     
59:14     
these types of data together so it's not easy we also have things like spatial transcriptomics which give us     
59:20     
information at the at spatial resolution we have you know temporal data that give     
59:25     
us information at the temporal give us temporal resolution that those cells form tissues which they kind of provide     
59:32     
is this little circle down here but you know hey it's just the next step right     
59:37     
um then there's the AI virtual cell Foundation model since they're doing AI they're thinking of a foundation model     
59:44     
because they're at Stanford and that means that they want to build this sort of piece set of universal     
59:50     
representations so they have the molecular Universal representation the cellular Universal representation the     
59:58     
multicellular Universal representation they have a section on universal representations up above which we can     
1:00:05     
talk about if you're interested uh but basically they're linking together things in these I guess Laten     
1:00:12     
spaces uh or these embeddings I don't know which uh where they're linking things together between different     
1:00:18     
physical scales so if something is in that space in the molecular Universal     
1:00:24     
representation they link it to the corresponding space in the cellular Universal representation sometimes     
1:00:31     
you're using different measurements to get sort of a location in that Universal representation so you're just kind of     
1:00:37     
linking those two locations and it kind build you this     
1:00:42     
model that you know otherwise would be very hard to sort of     
1:00:47     
interpret um then we have the capabilities and applications so you     
1:00:52     
know we have what they call reference Atlas so you have you know you can     
1:00:57     
contribute to a reference Atlas you have continuous Dynamics which you can model     
1:01:03     
you can model perturbations which you know are basically when you uh you know     
1:01:09     
you know you can do like Knockouts or you can uh you know modify sort of the     
1:01:15     
trajectory of a cell's differentiation or something like that basically they show here like a lineage tree where     
1:01:21     
you've knocked out a cell and so what's the consequence and then there's ex extrinsic     
1:01:27     
perturbation where you might give a drug treatment so you shut down some process or you modify the metabolism in some way     
1:01:34     
and you see what how the system responds you also have the ability to model spatial nich niches which are these you     
1:01:42     
know cells in different places in space and showing how say you have a bunch of     
1:01:48     
cells kind of assembling these multicellular Aggregates or sometimes     
1:01:53     
even tissues and how what that process this looks like you have novel cell     
1:01:58     
State Discovery so discovering different cell States cellular States they could be states that you observe in nature     
1:02:05     
easily or they could be states that we not really observe very much in nature but to have that ability to model     
1:02:13     
different cell States um and then we have in cical experimentation so you can     
1:02:19     
do experiments in in in this computational environment and then this digital twin     
1:02:25     
which basically means you're building a replica of the cell and digital twins is     
1:02:31     
a term from technology where you know they'll build like uh they'll try to     
1:02:36     
model some system um sometimes it's like you know     
1:02:41     
non it's mechanical systems sometimes biological systems with like enough     
1:02:47     
resolution to get an exact replica so they're trying to replicate in a hyper     
1:02:52     
realistic way the system in a digital form for them so that's what they're aiming all right let's look at some of     
1:02:59     
these references so these are some interesting references uh number we talked about that number one through six     
1:03:06     
or number one through seven were about virtual cells virtual cell modeling so     
1:03:12     
the first one is slepchenko at all quantitative cell biology with the virtual cell that's from 2003 so this is     
1:03:20     
like you know 20 years ago where people were proposing this talking about it     
1:03:26     
building the next generation of virtual cells to understand cular biology that's from Johnson and all 2023 it's in     
1:03:33     
biophysical draw how to build a virtual embryo this is marks or nature methods in     
1:03:41     
2023 emerging wholesale modeling principles methods current opinion in biotechnologies is Goldberg at     
1:03:49     
all uh George Julie at all uh multiscale models of wholesales progress and     
1:03:55     
challenges that's Frontiers and cell development biology um 11 Issue 11     
1:04:04     
2023 uh RCI at all computerated wholesale design taking holistic     
1:04:10     
approach by integrating synthetical systems biology that's in Frontiers in bioengineering and     
1:04:16     
biotechnology 2020 and then so that's one through six and then of     
1:04:22     
course uh seven through like 15 are other reference references that might be interesting here so you know we have uh     
1:04:31     
number of books or a number of references on receptors on systems biology um some other citations by     
1:04:40     
yurian uh full cell computational model predicts phenotypes and genotypes so you     
1:04:45     
know kind of Bridging the scales of organization scales of analysis uh causal protein signaling     
1:04:53     
networks derived from multiparameter single cell down so you know the the tools have been     
1:04:59     
sitting out there for a while they're just trying to put them together so from 12 to 16 these are     
1:05:06     
papers that kind of talk about the different components like cellular noise so the stochastic components of the cell     
1:05:15     
challenges in measuring and understanding biological noise and nature ofev these genetics in     
1:05:20     
2019 noise can induce B modality and positive transcriptional feedback loops of without by stability this is from     
1:05:27     
science in 2010 and just kind of talking about different types of modeling 15 is     
1:05:35     
advancing microbial Sciences by individual based modeling and 16 is aging based modeling and synthetic     
1:05:41     
biology those are in nature reviews microbiology essays and biochemistry     
1:05:47     
respectively and both from 2016 so a lot of references here um it's     
1:05:54     
an interesting paper check it out okay I just just wanted to add something to     
1:05:59     
that um mechanically a single cell is different from the tissue yeah they're a     
1:06:08     
different mechanical system yeah yeah yeah that I don't know how they're     
1:06:15     
accounting for that or other things because when you're mapping between scales yeah there are a lot of things     
1:06:20     
that are different at those different scales when I was working on     
1:06:26     
finite elements finite element modeling the guy I was working with had a book called computational     
1:06:35     
reality you know was before digital twins Morgan do you know anything about     
1:06:41     
finite element analysis programming uh yeah yeah we we     
1:06:47     
we were using it for electrical impedance demography electrical impedance really     
1:06:55     
okay um that's     
1:07:16     
something great intro so he was a he was a early user of     
1:07:24     
Phoenix yeah     
1:07:29     
so so would Phoenix do a better job than commo doing U finite element analysis I     
1:07:37     
don't know it' be good to be good to find out what the issues were that the the woman you mentioned     
1:07:44     
yeah well I can read them you want me to read read this I think I've got     
1:07:52     
her uh here it is     
1:07:58     
um okay we tried to simulate a structure of ab in Abacus using truss     
1:08:07     
element T3 D2 it's     
1:08:12     
a the simple T Integrity with added predefined stress     
1:08:19     
Fields uh we use the initial conditions type stress option to prede     
1:08:27     
um field to use this option you have to use ANL     
1:08:34     
Geo geome option in step we were     
1:08:40     
performing only static analysis and I found that the static     
1:08:45     
analysis is uh the only one I can get um console to work with um it says in order     
1:08:54     
to create the structure in Abacus um you can only use abacus     
1:09:02     
CAE whatever that is um but to add to this initial conditions you need to use     
1:09:11     
a input file there are two ways to do it easier in my opinion is to create all     
1:09:19     
structure in aboca without predefined stress Fields as you'll always do an     
1:09:26     
abacus then creating job in job module click     
1:09:31     
option right input then you need to find     
1:09:37     
thep file in directory you work and add     
1:09:43     
the lines which um add predefined stress     
1:09:48     
field anyways um name and set elements you want to preset pre- pardon me value     
1:09:57     
of pre-stress uh not the normal forces and     
1:10:03     
then it sort of tells me how to do it um and you can put it before the     
1:10:11     
material assign anyway she's English is the second language here um that's okay     
1:10:18     
a lot of people have have that in your um     
1:10:26     
anyway in both cases the two steps should be applied the first establish     
1:10:31     
balance and second to further analyze anyways it's complicated to create a     
1:10:42     
Tegrity um that will work in a finite element system needs to be very stable     
1:10:48     
that's my opinion any any addition to     
1:10:59     
this um I you know I don't know if there's really enough detail there the     
1:11:05     
CAE is is just computer AED engineering um which again I think just     
1:11:12     
kind of reflects their you know kind of like functionality set that     
1:11:17     
you you probably pay more for or something like that in terms of these these commercial     
1:11:23     
packages um yeah uh I the other thing is that uh we     
1:11:30     
were focused on electromagnetism Yeah     
1:11:36     
Yeahs to work better um and commo seems to do a lot of     
1:11:46     
electromagnetics which is more I'm more comfortable with electromagnetics rather than mechanics but oh well     
1:11:58     
um yeah yeah it's you know again like maybe try and find     
1:12:04     
someone who can share a model or share share a model that they're having     
1:12:09     
trouble with to see to see what the see what the issues are you know well I think the     
1:12:16     
main issue is that if you put a small force on a 10 segurity     
1:12:23     
model it may move a lot it's not broken it's just the way it is because you     
1:12:30     
pushed it the one last little time and it changed     
1:12:36     
shape because they do that and so their incremental     
1:12:43     
force is that does that does that match I mean is is there you know kind of like     
1:12:50     
discrepancy between a real world model and and computational one or well is     
1:12:58     
that is that does that reflect what yeah my little tensegrity here if I push on the     
1:13:05     
sides it also collapses on the top like     
1:13:11     
it um all of it moves you put a little     
1:13:16     
incremental force on it and the whole structure moves it is not broken it's a     
1:13:23     
tensegrity it does that so I think these     
1:13:29     
models the or console looking at the structural     
1:13:35     
mechanics thinks it's broken or computes that is broken yes     
1:13:42     
when you put a small incremental force on it and the whole structure     
1:13:47     
moves that's the only thing I can think of because it throws it into an     
1:13:53     
iteration where change the damping factor and the time time     
1:14:00     
length both like it it iterates and it'll iterate 200 times and it'll say no     
1:14:08     
it doesn't work well the thing is bouncing and recreating the same thing     
1:14:14     
each time in some cases and some times it um     
1:14:19     
actually um goes down to a very small error it just depends on on your     
1:14:26     
structure whether it bounces or or tries to approach a     
1:14:33     
conclusion by changing damping factors yeah yeah yeah yeah I mean again     
1:14:41     
like I think that reflects that it's a it's an interesting problem yeah like like whenever whenever     
1:14:49     
you're having trouble with something like it usually speaks to um yeah like that there's     
1:14:58     
some yeah something things not being taken into account or you know or it's     
1:15:05     
generating an interesting kind of like optimization problem that is is you know     
1:15:11     
super sensitive um yeah anyway um you know again we' love to     
1:15:20     
yeah yeah okay um     
1:15:26     
okay so anyways U it has to do with     
1:15:31     
um I dual pendulums and a constrained dual     
1:15:37     
pendulum two together and also when you put um a t segr     
1:15:45     
together where it's all attached by strings it's even     
1:15:51     
worse like the um console just does not compute I     
1:15:57     
tried to put a star T secrity together and it just nothing works yeah yeah yeah     
1:16:04     
yeah yeah again I think that's reflecting that it's it's actually you     
1:16:10     
know an interesting problem no just I should get you to     
1:16:15     
persuade my     
1:16:23     
professors yeah yeah well let me let me ask on the the Phoenix List um if     
1:16:31     
anybody's done any modeling like that thank you and see if you     
1:16:39     
know perhaps that they could explain why why that's you know what what the     
1:16:45     
issues are um yeah interesting     
1:16:53     
yeah yeah that's great um so before we go Hussein put something     
1:17:00     
in the chat um just wanted to update you all quickly I can I can I can share that no     
1:17:06     
probleme thanks thanks yeah yeah so uh met manifold research group uh just just     
1:17:12     
another in other news manold research group uh so they are they comp they're     
1:17:18     
excuse me a Computing group based on a lot of different areas of research in terms of AI and large language models     
1:17:25     
like multimodality continual learning modor interpretable phenomena like say understanding how why in what ways     
1:17:31     
people um want to understand adapt data like intervene with AI and um they had a     
1:17:39     
call out to see um for anybody who want to do some voluntary research and building the next generation of     
1:17:44     
intelligent systems which is really really cool so I think Moran put me in touch with them several months ago I     
1:17:50     
reached out and they did not have a project at the time that would be suitable for me but now they've gotten some some things together so um yeah     
1:17:57     
yeah they got some things together they reached back out to me and said um we can be discussing a physics based LM     
1:18:02     
project that I'd be working on so just to be clear it would only take like maybe five five or 10 hours a week of my     
1:18:08     
work at most uh so I can still leverage the other things I'm doing like the computer demography projects and stuff     
1:18:14     
too um so yeah yeah well congratulations     
1:18:19     
yeah it's good yeah super interesting so how are there     
1:18:25     
um what are their big projects again on their website they uh classify     
1:18:32     
things in three areas multimodal excuse me multimodal learning     
1:18:38     
uh Contin multimodality continual learning and modular modularity     
1:18:43     
interpretability those are basically three phenomena three areas that all Learning Systems should have so     
1:18:48     
multimodality meaning that Learning System should be able to handle objectives for different     
1:18:55     
different modalities different types of different ways of understanding stuff continual learning a system to update     
1:19:02     
and adopt to changeing DEA over time it should be continu should always always be continu Y and the the third thing is     
1:19:07     
being modular so humans should be able to understand and intervene in a way that they can do so     
1:19:15     
reasonably when they need to for updating things in ways that are different for things that are maybe subjective that are ethical not just     
1:19:21     
throwing everything onto a machine and the human sitting back and just you know drinking drinking coffee and doing     
1:19:26     
nothing no a learning system is something that we we work we     
1:19:32     
work alongside we work with it so um yeah the manifold research group um right I'm pretty sure it's all voluntary     
1:19:38     
I think it's all voluntary but they they have people across the country like there people in San Francisco I'm sure     
1:19:43     
there are people here in the Midwest too like um Henry the one I was speaking to Henry he's in Chicago I'm I'm near I'm     
1:19:49     
near Indianapolis so Chicago is within driving distance so yeah yeah     
1:19:55     
yeah no that's that's awesome did you say physics I I I thought it was going to be um math uh math     
1:20:03     
focused I think he's kind of using both interchangeably a bit here ah ah     
1:20:09     
mathematical physics okay undergrad was in physics I'm sure I could have declared a math major I just didn't so     
1:20:16     
yeah yeah yeah that's     
1:20:22     
awesome all right I think that's it for today um thank you for joining uh yeah     
1:20:28     
good luck to Pocky and mahul on their submissions to Google summer of code and     
1:20:34     
Hussein on his projects um yeah and then Susan of course on doing the ever Quest everl     
1:20:42     
last uh Ever Changing quest of 10 secrity okay well I handed in a rough     
1:20:49     
draft of my thesis already because they insisted so I am working on validation     
1:20:56     
of my method or verific verification that's it I'm not too sure what to say     
1:21:03     
except that it's within all the parameters that AFM measure for tissue     
1:21:10     
so yeah all right all right     
1:21:16     
thanks take care take care bye
