     
Transcript     
0:08     
hello hi Hi how are you so uh yeah hello     
0:13     
uh hello herit uh how are you hello sir hi I know you wanted to     
0:19     
give your presentation on your project so we'll do that um before we do that     
0:25     
are there any uh uh updates all right hello sir hello how are     
0:32     
you hi ma'am uh sir I am currently btech student in second year okay I'm from     
0:40     
India I I am studying in Valor Institute of Technology and I want to participate     
0:47     
in gof and I want to participate in machine     
0:54     
learning so I wanted to meet you yeah     
1:01     
welcome yeah so uh that's great uh yeah we'll be making decisions     
1:08     
uh in the next like this week actually coming up so this week we'll be uh sending out the     
1:16     
acceptances I have no idea how many slots we'll get from G um and if you don't get selected you     
1:24     
can still work on a project um you know just let me know and we can set something up because we always have     
1:30     
people working on projects in the Pro in the in the group and they you know they     
1:36     
work at their own pace if they're not on gck if they are on gck then they work according to their schedule and and     
1:43     
sometimes people you know continue on afterwards and so there are a lot of     
1:48     
opportunities um yeah so did you want to present yes I would like all right go     
1:56     
ahead and share your screen yeah yes I can see your     
2:03     
presentation sir my project was about graph neural network in a j problem     
2:11     
statement okay it it was a large project according to that given in the problem     
2:19     
statement so I have worked on GNN     
2:24     
earlier like I can show you one of my project okay     
2:30     
I read this research paper for making that project I am currently working on a     
2:36     
research project which is related to fluid     
2:42     
mechanics okay I can show you that I will show so sir it is     
2:50     
related to like Edge Focus then node Focus Network okay in Edge Focus the the     
2:59     
data is passed through Edge and node focused data is passed through nodes     
3:04     
directly Edge focused mainly focuses on specific types of which are specifically     
3:12     
should be shared through edges only like we can see in this     
3:19     
researcher I will show you     
3:25     
right so what happens is we can to recursion like in fluid mechanics     
3:31     
project what we have done is we have a one node nodes are nothing but neural     
3:38     
neurons there and those neurons can transfer their data from one to another     
3:45     
and the recursion happen one Edge shares data to nearby edges and that shares to     
3:54     
early others and that come to a specific node one by one after     
4:02     
Ur sir this is a research paper that I have worked on it is related to fluid     
4:10     
mechanics it is about the probe design which is increasing its volume with time     
4:17     
we want to increase its volume in such a way that its point force remain stable     
4:24     
in water or any fluids available so we are trying to to predict     
4:30     
how much volume it should expand according to the condition life conditions after the chemical reaction     
4:37     
happening inside and one more project that I have     
4:45     
worked on is about Urban heat island monitoring system sir Urban heat island     
4:51     
is as the name suggest heat island island means some area covered by water     
4:58     
and heat island means a area has more heat in itself than its     
5:04     
surrounding so that surrounding area is cooler and     
5:09     
that specific area is hotter that is called Urban heat island and for     
5:15     
predicting those areas we are telling which features like elevation of that     
5:20     
area and other features like air quality and other we are telling which feature     
5:27     
contribute more to that area specifically and we are doing same for     
5:33     
larger area as well I have learned Google Earth Engel for doing     
5:40     
soour right uh I have a video as well for that     
5:46     
all right yeah     
6:08     
you run that team     
6:21     
yeah so we're using a graph neural network on this or is just a separate project     
6:30     
sir graph neural network project is a separate one okay for that I have made     
6:36     
easier that one as well graph neural network is more complicated yeah yeah we     
6:42     
can make it easier yes this is the Google Earth engine so this is like     
6:50     
a yeah yes sir it is for Earth satellite     
6:56     
data we can use on Google Earth engine we have to learn for using data yeah that's     
7:06     
great     
7:20     
yes sir is video audible as well uh no but that's okay it's it's hard to it     
7:27     
gets it reverberates too much yeah     
7:38     
sir I can make website as well I know reactjs expressjs Jango as     
7:44     
well yeah yeah and and Sir like I have a project also     
7:53     
and on react yes I have a website deployed as well yeah on     
8:01     
get so sir these are my projects that I worked on yeah did you have a paper that     
8:08     
you were going to bring up as well or you have that I see a tab it says Graal     
8:15     
Network that one yeah yes sir this is yes so this is a paper that I that I     
8:24     
read to make uh that graph neural network are works     
8:31     
and one and our paper is also there that I can't show sir because it's in     
8:37     
publication process yeah uh so sir this paper tells us very     
8:44     
well about how graph neural networks one in work in two ways node focused and     
8:51     
Edge Focus like this shows how fluid mechanics node focused and Edge focused     
8:58     
and uh the data is transferred from nodes to     
9:03     
nodes specific type of data is transferred through nod and specific through     
9:09     
edges okay like uh for example in fluid     
9:16     
mechanics we can see that sir like advection advection net is     
9:26     
for calculating acceleration and uh then Collision Network then pressure network     
9:32     
two of them are are used in node Focus then one is Edge     
9:38     
Focus yeah what's the advantage of using what's the advantage using a node Focus     
9:45     
versus an edge focused or vice versa I should say because we kind of know what the advantages of a node Focus Network     
9:51     
are but what how would an edge Focus Network help uh do like deal with a     
9:58     
problem of flow is it the interaction that you're capturing or is it something     
10:05     
else s it is used pairwise Ed focus is more specific for pair wise inputs like     
10:14     
one input is dependent on another neuron in case of node focused it is not     
10:20     
dependent on another it is going directly and in case like when Collision     
10:28     
happen in fluid mechanics then that is dependent then we use Edge focus and in     
10:34     
other case we used n Focus for pressure and all other stuffs     
10:42     
yeah so that is how we use it and this makes it     
10:48     
faster G this makes GNN faster yeah if if we separate both if we     
10:57     
include all together then it will be much slower than normal this     
11:05     
one yeah it is for Accelerated Lang reg     
11:10     
fluid simulation     
11:17     
yeah yes that's good uh this is the graph     
11:23     
here graphical abstract could you click in on the graphical abstract there     
11:33     
yes sir yeah that     
11:38     
one yeah the graphical abstract is yeah right     
11:45     
there yes sir it is AD Direction net first which calculates     
11:51     
acceleration and then it is okay acceleration data is transferred to     
11:58     
poision net and collision net data is transferred to pressure net and in when     
12:04     
we separate them it becomes faster in case we calculate all together then it     
12:09     
is much slower if we separate them and gets the predicted output then we use     
12:15     
that predicted output and speed into another neural network then to calculate     
12:22     
another part of neural output then Del feeding to another it becomes     
12:32     
faster yeah yeah it looks in     
12:39     
case yes so you do this for every time step you evolve     
12:45     
it yes sir for every single node we have     
12:51     
to do yeah     
12:59     
all right the more number of neur the more time right     
13:04     
right well that sounds good yeah so this is actually an alternative to numerical     
13:10     
methods and yeah yeah we've talked about that in some of the work we're doing um     
13:17     
you know we we're interested in biophysics we're interested in these tensegrity structures where you have the     
13:25     
edges are important quite important maybe more important than the nodes and     
13:30     
so it's interesting to see some work on that what was the paper reference again     
13:37     
I just missed it yes I can share     
13:47     
it yeah here we go good oh thank you     
14:01     
yeah that's great thank you herit that's uh you know that's an interesting approach     
14:09     
so any other questions about uh the uh work there the     
14:22     
approach okay yeah thanks uh so let me share my screen now     
14:30     
um yes yeah right so yeah we got a lot of uh     
14:37     
things I want to talk about today uh I have several articles and     
14:42     
then I like little pieces of news I guess and then uh larger a larger paper     
14:50     
to talk about so this is uh the first one I wanted to talk about     
14:55     
here is this APS article on it's an opinion article that's not physics and     
15:02     
this is Andrew zagil or Zang and uh this is you know talking about things like     
15:08     
emergent systems and complex systems and this you know is kind of a interesting     
15:15     
topic in terms of complexity Theory and how we apply it to things like biological systems but it also kind of     
15:23     
takes the perspective from the physics side which is is this actually physics or not so it's a sort of a     
15:30     
a disciplinary boundary issue but anyways uh the question he ask the     
15:37     
question where do the boundaries of physics begin and end the debate has persisted for more than a century so     
15:44     
this is the picture of a starling flock which of course is consists of a bunch     
15:49     
of birds they uh participate in this uh     
15:54     
flocking Behavior which people have been able to capture using algorithms like     
16:00     
voids and you know you can model this in a computer you can also observe it in the sky and we've been able to do a     
16:06     
pretty good job of getting uh the details of these emergent structures down at least uh in terms of the     
16:14     
movements we don't know a lot about like the actual behaviors in the uh context     
16:21     
of these movements but that's something that you know is less understood in terms of algorithmics so this is you     
16:28     
know we can replicate these these kind of swarms and flocks and things like     
16:33     
that so this is a sterling flock and a so-called murmuration or Collective     
16:38     
behavior of interest and this is one of the many subfields of biological physics uh and     
16:46     
biological physics itself that don't always belong in physics so a lot of times you know biophysics is you know     
16:54     
they're they're doing they're answering questions in physics but they're also answering questions in biology they're     
17:00     
answered questions in computer science and so forth so if you've been in     
17:05     
physics long enough you've probably left a colloquium or seminar and thought to yourself the talk was interesting but it     
17:12     
wasn't physics so this is usually a problem uh you know I've had this problem in my career a lot where I've     
17:18     
been in different departments and you know you go to talks and they are very     
17:24     
different from the rest of the talks in the department and kind of situating work in different departments versus uh     
17:31     
you know the in sort of an interdisciplinary group and so a lot of times there there's some uh you know     
17:39     
boundary issues with that people have conceptual problems sometimes with with     
17:44     
thinking about things outside of their domain of expertise so you know um if     
17:49     
you get a biologist and a physicist in a room sometimes the biologist and physicist won't really see this problem     
17:56     
in the same way so you'll have a flock you'll see a flock in the sky and the biologist don't want to talk about     
18:02     
certain things and the physicist don't want to talk about certain for uh formalisms and you know it's they're not     
18:09     
quite on the same page and it's because their training is quite different and so one way that we've tried to overcome     
18:15     
that is by you know doing this sort of cross trainining this sort of interdisciplinary training but still you     
18:21     
have problems of people recognizing it as part of a a field or being a     
18:27     
contribution to science so this is you know a problem in terms of promoting and     
18:33     
hiring individuals uh and this can be you know it's kind of the it can shape the     
18:40     
opportunities for people to work on different problems so you know in our group that's not so much of a problem     
18:46     
but if you're in a more formal academic setting it can be a very big problem so you know a lot of times uh     
18:54     
questions that we've asked have happened in the context of or sometimes in spite     
19:00     
of where their field of origin so these debates over physics and what is physics     
19:05     
and not physics are important we've been doing this for over a century a key moment for not physics     
19:13     
occurred in 1899 at the second ever meeting of the American physical Society in his keynote     
19:19     
address the aps president Henry Ro Roland exhibited exhorted his colleagues     
19:24     
to cultivate the idea of the Dignity of physics which basically means ask physics like questions always and you     
19:32     
know that's uh of course you know as as the The Next Century would roll on the     
19:38     
nature of physics which change quite a bit but you know that's the sort of attitude sometimes you get in physics or     
19:44     
in any field really um much of the intellect of the country is still wasted in the pursuit     
19:50     
of so-called practical science which ministers to our physical needs he scolded and not to investigations of     
19:57     
pure ethereal physics which our society is form to cultivate so there is this     
20:02     
aspect of like the pure versus applied research and and that that's a different set of things where you know if you're     
20:10     
applying your physics knowledge to a problem like swarming or blocking you     
20:15     
know it's not viewed as real physics because it's not answering fundamental questions even though it kind of is answering fundamental questions but you     
20:23     
know so this is this is this you know has been an attitude that's persisted     
20:29     
another you know a good example of this is Acoustics so Acoustics started as a     
20:34     
branch of physics and it's become its own field since you know people have uh     
20:42     
investigated Acoustics in terms of its applications to you know sound     
20:47     
systems uh you know there have been applications to cognitive science applications to animal behavior and so     
20:53     
forth so Acoustics which there's an acoustical Society of America which     
20:59     
studies you know the the propagation of sound waves the interpretation of sound     
21:05     
waves by the nervous system and so forth those that started as a a physics Endeavor but now is this     
21:12     
multidisciplinary uh field that takes in a lot of different disciplines in those perspectives so it's interesting that     
21:20     
acousticians or people who studied Acoustics uh you know in in sort of the     
21:25     
early days of that area were given a cold shoulder at early APS meetings uh     
21:31     
at the time which is the early 20th century Acoustics research was not uncommon in American physics departments     
21:38     
Harvard for example employed five professors who uh studied Acoustics between 199 and     
21:45     
1950 you got more people with World War II and the needs of of uh things in that     
21:52     
era uh and then after the war of course you had other applied areas that started to pop up so so this is you know they     
22:00     
were kind of like separated out from the field or kind of expelled from the field     
22:06     
uh but they ended up forming their own sort of interdisciplinary field it's kind of an interesting exercise in     
22:13     
looking at how different academic Fields kind of form and uh     
22:19     
evolve so you know cosmology is another example where it goes from being kind of     
22:25     
not physics to physics so in this case cos ology started as not physics it was very much based in astronomy but then     
22:33     
kind of came into physics with Albert Einstein's general relativity Theory kind of brought that into the field and     
22:40     
then there were other physics questions that resulted from that which made it more integral in terms of the     
22:47     
physics um and so not all disciplines have had such a clean path in and out of physics     
22:54     
so that's where we get to biophysics uh where physicist their methods to the study of living systems     
23:01     
and so biophysics which is probably the closest to our interests you know it's     
23:06     
kind of like skirted that line between physics and not physics and so that means you know we have questions that     
23:13     
are maybe very physical in nature and questions that are very not physical in nature and we have our techniques which     
23:20     
some come from physics some might come from say computer science or some other area and it's very very eclectic in that     
23:28     
way and so you know it's an interesting kind of study of these different areas     
23:33     
especially with respect to biophysics some uh something in the chat     
23:40     
okay okay yeah thank you harit for attending if you need to leave that's fine but yeah thanks for presenting on     
23:46     
your project thank you sir yeah okay um so this is another article     
23:56     
it's interesting this is from astrobiology olog this is another um topic that we're interested in and     
24:02     
speaking of things that are physics and not physics this is sort of not physics but it's also physics if that makes any     
24:10     
sense so this is again astrobiology which is you know sort of thinking about     
24:16     
life in on other planets or on in other habitable zones other than Earth how     
24:23     
that relates to life on Earth and so there are a lot of interesting questions that have emerged from astrobiology     
24:29     
astrobiology being sort of involving physics but is also not physics so this particular article is on     
24:37     
the parallel evolution of gravity sensing so this is where you have organisms on Earth uh tinora nideria     
24:47     
Plaza these are all uh invertebrates that have different structures to their     
24:54     
phenotype and they have but more importantly they have different they've evolved different meth mechanisms of     
25:00     
sensing gravity so an organism has to sense gravity if you're in the water     
25:06     
column you know you have to know which which side is up and which side is down so if you're an organism that's     
25:13     
differentiated into this asymmetric form um you know you have uh different     
25:20     
ways of propulsion you have different you know sources where you know places where food     
25:26     
is different food sources you have to have some sense of gravity sensing so     
25:31     
you can move to get to that food or move up and down the water column as need be there might be chemical gradients that     
25:37     
you need to conform to and things like that so you need to have this gravity sensing uh ability and in these three uh     
25:46     
types of organisms we've had three different forms of gravity sensing     
25:54     
involved so if we look at this more closely and Tina for we have an arbor AR     
26:01     
Boral organ with a statolith which is this thing here you can't really see it very     
26:07     
easily but it's basically this uh there's this Dome that has a statolith     
26:15     
which is a little I guess a little Stone it's like an oolith in fishes it's got     
26:21     
these uh you know accessory cells around it and it has other structures around it     
26:27     
so it's this Dome with this these uh structures emanating from it so it's one     
26:33     
way to do this uh that's you know one way that this was evolved in tinora then     
26:41     
in nideria which is a different uh set of organisms we have the narian     
26:47     
neaty so this is a little bit different where you have the sensory epithelium here you have this concretion so again     
26:54     
it's like a stone but it's not the same because it doesn't exist in a Dome it's in this layer of epithelial cells have     
27:02     
this inner thin epithelium sensory epithelium out here and this structure     
27:07     
out here which I guess gets information from the outside world and so you have this it you know again like in fishes     
27:14     
and in uh vertebrates we have this thing called an ooth and the ooth is in a     
27:21     
structure and this is like our basically the H in humans this is the inner ear in fishes they call it the oo so it's the     
27:28     
structure in the head that allows us to sense um you know gravity and and our     
27:36     
sort of when we move our heads or when we move our bodies we can tell kind of the inertial sort of we can register     
27:43     
ourselves with respect to gravity so this is the same thing here it's using a different     
27:49     
structure and then in plaz we have these Crystal cells which is you know a type of cell     
27:57     
where you have it's uh has it has a very different structure than these     
28:03     
two so this is uh these are called statuses architectures and they do this     
28:09     
in a different way in each uh set of organisms so this is an interesting     
28:15     
thing you know we every organism has to deal with gravity and you know there are different ways that you can evolve this     
28:21     
organ obviously it's important because you get these convergent evolution you know this convergent evolution of the     
28:27     
organ how it gets kind of put in place in different ways in evolution so you     
28:33     
know gravity affects all living organisms you need this for survival because you need to navigate your     
28:39     
environment but you also need to know kind of even cells really need to have an account of gravity as they sort of     
28:46     
form structures but they don't really have a you know as far as we know they don't have like a specializ sensory     
28:52     
organ for this there's you know you have chemical signals and physical signals     
28:58     
that play a role in kind of maintaining the stability of the cells and maybe you     
29:03     
know we've talked about like epithelial sheets where we have uh sort of a a     
29:09     
mechano mechanobiology where you can build these hyper stable structures or structures     
29:15     
that are resistant to the forces of gravity so you know but this is more about sensing gravity in the context of     
29:22     
moving around the environment so these are specialized organs for that so you     
29:28     
know a lot of even a lot of bacterial sized microorganisms are subject to browni and motion and so are our our     
29:35     
cells in um animal cells and plant cells     
29:40     
uh they operate under distinct micro environment and viscosity constraints with a lack of detectable gravity     
29:47     
sensation so there there are these different physical aspects of the environment and you have to have this     
29:53     
sensation of gravity to know kind of how to move against the gravity um and so many unicellular     
30:00     
protests and multicellular organisms which are less than 50 microns in size     
30:05     
have developed remarkable examples of gravit perception which is what they call sort of sensing gravity and     
30:11     
responding to it gravity sensitivity often co-evolves with photo receptors as     
30:17     
one integrative system so this is interesting that you see often in evolution you see photo receptors evolve     
30:23     
along with gravity sensitivity and it's worth thinking if you have like a organism you know in     
30:30     
like if we had organisms on another planet where they were you know there was a very high gravity you know what     
30:37     
would those organisms evolve in terms of gravity sensation would it be a system like this or would it be much much more     
30:46     
complex you know what would it what would it look like so this is uh you     
30:51     
know they have there's a variety of these kind of uh you know organs and     
30:56     
ways of sensing gravity um and you know we also have the sort of psychophysics that's a part of this as     
31:04     
well uh in many planktonic and benthic organisms larval stages and land     
31:09     
organisms positive phototropism which is the sensation of white and negative     
31:15     
gravitropism which is the sensation of gravity so a tropism is where you're moving towards a source so if you have a     
31:23     
phototropism you're moving towards the photic stimulus if if you have a     
31:28     
negative gravitropism you're moving away from the source of gravity so you know     
31:33     
this is this is where they can mutually influence each other if you think about like uh different organisms in say the     
31:41     
deep sea or in uh some other Aquatic environment often times they're moving     
31:47     
up and down the water column and if they move up the water column it's usually towards light although it could be at     
31:53     
the bottom if you're in the very deep sea because you have uh you know vents     
31:58     
uh volcanic Vents and things like that but you're often moving in the maybe     
32:04     
against or in concert with gravitational forces so gravitational forces P down on     
32:11     
the organism and photot tropisms might make you want to move up so you need to     
32:16     
evolve something that counters gravity not only to keep yourself stable but to move up the water color so these are     
32:23     
things that you know all kind of co-evolve um quantitatively hawen hater     
32:29     
which they talk about up here 2006 uh they state that light     
32:35     
transduction appears to possess a logarithmic transducer whereas gravitation or gravit transduction     
32:42     
induces a linear so there are two different kind of responses to this you have this logarithmic response which is     
32:49     
the response to light and a linear one which is the response to gravity that's interesting over a range of kind of     
32:56     
values that you can get this response response by the organism so you know in terms of gravity If the gravity     
33:02     
increases uh or if the light increases there's a a difference in the     
33:08     
corresponding response by the organism so you know this is very important for both bioengineering and     
33:16     
astrobiology understanding the diversity of these grav gravity signaling systems presents a unique reference Paradigm to     
33:23     
understand transitions of homology which are these kind of independent uh     
33:28     
evolution of different types of systems to examples of convergent evolution within integrative systems and     
33:35     
so this is I don't know where they put the citations for this but uh this is an interesting     
33:44     
article um and you know there are a lot of questions in terms of uh astrobiology     
33:51     
that you know could be kind of answered by this paper okay uh then the next paper here     
33:59     
is this new paper on the path to complexity this is in npj complexity     
34:05     
which is a journal it's kind a nature uh Journal I don't     
34:11     
know and what PJ means but it's a nature-based journal     
34:17     
so this is the path of complexity and this talks a little bit     
34:23     
more about complexity Theory and kind of the origins of complexity Theory     
34:28     
and how you know the kind of how this is grown around different problems and     
34:34     
different questions that people have had and how they've tried to answer it so     
34:40     
another things about complexity theor is it's a very it's a departure from the way that we answered kind of     
34:57     
statistical methods that weren't particularly didn't capture things like emergence or dynamical systems but in     
35:06     
you know in the later 20th century these techniques have been developed uh that we associate with complexity Theory and     
35:14     
uh there's a lot of especially like kind of the late 20th century there was a lot     
35:19     
of hype around it and so Stephen Hawking was quoted as saying I think the 21st     
35:25     
century will be the century of complexity so that may or may not be true but there there was a lot of     
35:30     
thought that this would be the the sort of the science the unifying science of the 21st century well so far I don't     
35:37     
know if that's true but I guess uh you know it it's it's got its promise still     
35:43     
but also it's been applied quite a bit in throughout science so that's good um     
35:51     
so complexity science study systems were large numbers of components or subsystems at times of a different     
35:57     
nature combined to produce surprising emergent phenomena Apparent at multiple scales it     
36:03     
is these phenomena hidden behind the often deceptively simple rules that govern individual components the best     
36:09     
defi Define complex systems so if we go back to our swarm you know each of these     
36:15     
birds are one of these black I mean they look like black dots but they're a bird they're individual agent they have     
36:23     
behaviors and the behaviors aren't always the same and so you know modeling     
36:28     
this as like a set of particles you know might be inappropriate we might want to add in like individuality we might want     
36:36     
to add in like you know different perceptual systems or different types of decision-making systems we might add in     
36:43     
some noise to counter sort of this uh idea that every agent does the same     
36:50     
executes the same set of rules there are a lot of things we can add to the modeling this but we have to have a     
36:55     
starting point so we have this model that takes individuals looks at their interactions and then builds these large     
37:03     
scale structures out of it and says okay this large scale structure is a different thing and it you know is     
37:09     
different from the state of the individuals yeah tensegrities could be     
37:15     
thought of as a type of complex system you could do that yeah definitely like a tensegrity structure is different than     
37:22     
the component parts so the component parts all contribute to this structure which is a different state     
37:28     
than the individuals and so you know there have been a lot of uh this requires a sort of     
37:35     
non-physics or not physics sort of interdisciplinary collaboration studying a problem across     
37:41     
scales and disciplinary domains and so this is something that is you know was     
37:47     
back centuries but has really kind of crystallized in the latter part of the     
37:52     
20th century so in 1962 Herbert Simon laid a road map for the study of complex     
37:58     
systems in the architecture of complexity um and you know her Herbert     
38:03     
Simon was from the social sciences so he was doing a lot of stuff post-war stuff     
38:09     
with uh early artificial intelligence and organizational science so it's very     
38:14     
different from like the physics of a swarm or or a flock and and the sort of     
38:20     
biophysics questions that we we ask but the methods are very you know you could     
38:25     
have similar methods they're they're relevant to both areas uh a decade later the physicist     
38:32     
Philip W Anderson addressed how this philosophy clashes with the standard reductionist hypothesis in 1972 in a     
38:40     
1972 paper called more is different broken symmetry in the nature of the hierarchical structure of science so     
38:48     
this is a classic paper sort of that lays out this sort of initial approach     
38:54     
to complexity Theory as a field and so the people from the Santa Fe Institute love to cite this as the foundational     
39:01     
paper of complexity science um but you know it's there was a lot of things     
39:08     
going on at that time that were kind of in going in that direction so um now in this essay Anderson argues for the need     
39:15     
for multiple perspectives since the ability to reduce everything to simple fundamental laws does not imply the     
39:22     
ability to start from those laws and reconstruct the universe so you know you can have laws that sort of govern say     
39:29     
something like emergence or emergent complexity um they're different than the     
39:35     
laws that sort of govern the origins of say like life or the origins of a     
39:42     
physical system or something like that so you know a couple of Agents kind of     
39:47     
mucking around operate by a different set of rules than a in a complex emergence structure it's basically what     
39:53     
he was saying and you can't get there from looking at things using a standard reductionist approach you can't break     
39:59     
down the the Swarm or the flock or the emergent structure into parts and so if     
40:05     
you know we're modeling something that's an interesting thing to recall because if we model something we're modeling a     
40:12     
complex emergent structure a lot of times in especially in in biophysics especially in development and so you     
40:19     
know taking these parts of the system the cells or parts of cells and modeling     
40:25     
the whole is going to give you sort of a different uh set of principles that you     
40:30     
have to model so we talked about multiscale models and one of the reasons we want to build multiscale model is     
40:36     
because we have things like you know macromolecules or cells that contribute     
40:42     
to larger structures that are very different in in the way that they operate so we saw that last week with uh     
40:49     
the uh platforms the modeling platforms that we were talking about they treat these different scales very differently     
40:56     
in terms of their governing laws so you know there are different     
41:02     
types of ideas that were floating around in the 1970s and into the     
41:07     
1990s uh along with this there were a lot of popular books and and complexity     
41:13     
theory that came on around that time it was a very heady time for that area uh     
41:18     
but you know was it physics or not physics well they had a lot of it in physics departments but a lot of it was     
41:25     
also non-physics because you had uh you know you were dealing with Biology and     
41:30     
you were dealing with social systems and so it was very different from what people were doing necessarily in physics     
41:37     
department so uh yeah the     
41:44     
um yeah so at the same time John Von noyman and stennis La ulam were     
41:49     
developing theoretical machines which came found which became foundational to the Future field of computer science     
41:56     
that could build functional copies of themselves with hopes that they would eventually evolve ever increasing levels     
42:01     
of biological complexity so this is really like the sort of the beginnings at the same time in fact probably from     
42:08     
about the 19 early 1960s on you had people trying     
42:14     
to build upon the uh self-replicating aspect of biological complexity and the     
42:22     
uh generative aspect of biological complexity so you know we don't talk about uh genetic algorithms too much in     
42:28     
the group uh but we talk about it off and on but you know developing those kind of models the sort of replicator     
42:36     
models John V noyman did some interesting things with cellular     
42:41     
autometa that you know really kind of uh mimicked how biology and biological     
42:47     
complexity work so there were a lot of different uh kinds of self Arguments for     
42:55     
self-organization uh and genetic you know working with genetic regulatory networks and other types of physical     
43:02     
models for understanding self-organized criticality and things like that so     
43:07     
there was this uh really this explosion of work on uh bio complexity that that     
43:15     
really kind of persists to this day and you know they're they're very applicable they're very different from what came     
43:21     
before so this is this paper is uh interesting because it it it kind of     
43:26     
goes over a lot of the aspects of the diff early models in the early in the in that uh     
43:33     
kind of came out uh um kind of emerged pardon the upon in the late 20th     
43:39     
century uh and then they make a point about it being interdisciplinary not any disciplinary so it doesn't require you     
43:46     
to kind of just not have a discipline there is very a very much an interdisciplinary aspect where you're     
43:53     
combining things for multiple uh disciplines and this is again this sort     
43:58     
of you know you might find this a trivial argument but it's very important in terms of sort of focusing the work on     
44:07     
things that people are expecting to see so there's this focus on sort of     
44:12     
interdisciplinary work which means you're taking things from two Fields it could be like you know physics or some     
44:18     
non-physics field and putting them together and so this kind of talks a     
44:24     
little bit about this new journal or the the scope of this journal so I'm not     
44:30     
going to get into that but basically you know they're they're kind of talking about this aspect of     
44:35     
disciplinarity and how that relates to complexity SS okay so Susan said everything is     
44:43     
physics well I don't know if I'd go that far but uh that was that kind of the attitude I think of a lot of the     
44:50     
complexity theorists of the like the late 20th century so a lot of them came from physics and so they they said okay     
44:57     
everything is a physics problem but it's you know you also have to have these other methods thr you know you want to     
45:03     
study biology in a way that biologists haven't studied it nor physicists have studied it before but the the techniques     
45:10     
are from physics so yeah at the at the root I guess the the implication was     
45:17     
everything is physics but yeah it's it's sort of the the     
45:22     
Baseline and you can build actually from from physics to chemistry and from     
45:29     
chemistry to biology it's sort of a and you can go directly from physics     
45:35     
to biology yeah at least that's the way I I see it no yeah I mean it's there's     
45:43     
definitely like a line from physics out to     
45:48     
biology because I mean we've had physicist in biology or physical biology for a long time probably 100 years or     
45:55     
more but um yeah a lot of the techniques um have     
46:02     
developed in physics both the quantitative techniques but also the techniques like you know microscopy     
46:09     
techniques or you know uh DNA sequencing comes kind of from chemistry but also     
46:15     
physics and it comes into biology yeah you     
46:21     
can essentially describe a lot of biology using physics     
46:28     
yeah biology tends to be observational and is certainly written     
46:35     
differently than uh engineering or physics because they'll say oh it's this     
46:41     
way I think um or I might be wrong and     
46:46     
you're as an engineer you're going ah it might fall down yeah so it's it's hard     
46:53     
to read sometimes because did they mean what they just just wrote this whole paper about or why did they say that     
47:02     
yeah yeah it's has been difficult     
47:08     
Yeah well yeah Susan you have this problem in your own work where well in this case it's     
47:15     
engineering and not engineering right like     
47:22     
yeah um yeah I've run into difficulty oops     
47:31     
yeah don't think my I just G to turn this off it's not lined up     
47:38     
um anyway um yes and so I get asked by an     
47:43     
engineer uh is your construct right like for the for instance the tensity that     
47:50     
I'm doing yeah um and I say maybe and maybe maybe I'm wrong because     
47:58     
that's the way it's phrased by the biologist so then then he thinks I doesn't I don't know what I'm doing     
48:06     
yeah but yeah this is my best guess from reading the biology     
48:13     
right and it's all observational and I might be wrong     
48:19     
written right into their papers so what am I supposed to say yeah yeah     
48:30     
yeah it's definitely um H you know this persists I     
48:36     
guess uh yeah so I had posted something in the slack uh this was a paper from I     
48:44     
science uh this interesting we talked about Reservoir Computing a couple weeks ago this is Jack chinski and and some     
48:51     
co-authors uh this is on modeling non- gentic information Dynamics in cells     
48:57     
using Reservoir Computing so they're doing some interesting work with non-     
49:02     
gentic information in cells and this is uh I pulled this paper up uh put it in     
49:10     
this in the slack and this is the paper     
49:17     
here uh so they they first of all they introduce a non novel theory of non- gentic intracellular information     
49:24     
Dynamics which I assume is like things moving around the cell you have a lot of things like going on in the cell but I     
49:31     
think in this case they're focused on energy-driven ION specific membrane pumps so they're trying to maintain this     
49:40     
uh these transmembrane gradients and so they have this these information     
49:46     
Dynamics and then they say information is transmitted by ion fluxes across     
49:51     
membranes and along the cytoskeleton so they take a certain part of the inter C or meu and they sort of     
49:59     
uh focus on that they're focused on the information transmission there then they     
50:04     
take then they use a reservoir Computing model Bas based on the cytoskeletal     
50:09     
network so again this is a network approach but it's a reservoir Computing approach so the reservoir computer that     
50:17     
you know they build is taking like the C of skeletal Network and sort of throwing     
50:23     
random basically turning it into a random Network and then try they try to model it as a     
50:30     
sort of uh you know trying to find like the patterns or modes of information transmission or information spatio     
50:36     
temporal information distribution so this Reservoir model demonstrates efficient spao temporal information     
50:44     
distribution so this is their graphical abstract and again this is uh you know     
50:50     
these are very useful I like there's been a trend towards this um so so this is uh the non- gentic     
50:58     
communication mechanism necessary for Rapid cell response basically if you have a cell that has electrical activity     
51:05     
or you know electrophysiological activity there's this influx of ions and     
51:12     
there's this you know signaling that happens so you know briefly you have a r     
51:18     
you start with a resting cell you have this extracellular perturbation so you     
51:23     
per perturb the cell in some way there some transmembrane ion flock so ions can     
51:29     
come in but they can also leave and they're different types of ions so they're different uh you know there's     
51:35     
potassium ions there sodium ions and so forth and you can go through um so you     
51:42     
can model the the influx and outflux of those but this is also you know it's     
51:47     
it's not just happening in isolation there's this information flow along the cytoskeleton that allows for     
51:54     
these this membrane transmembrane flux to occur so the cytoskeleton essentially     
51:59     
has to remodel to allow for this and then there's his overall cell response     
52:05     
so is there's information flow along the cytoskeleton there's a whol response     
52:11     
that that results from so that's what they're getting at this kind of model and they're using uh Reservoir Computing     
52:17     
to sort of get us handle on this so this is this picture down     
52:24     
here is modeling intracellular information Dynamics and cell your     
52:29     
decision making with Reservoir Computing and deep learning so you have this model of the cell this 3D model the     
52:37     
cytoskeleton across the cell uh membrane you have you put you model this as a     
52:43     
reservoir computer and you come up with these uh components you you have this model of     
52:50     
information processing where there is this inhomogeneity along the     
52:55     
cytoskeleton and so you have like all these different parts that you have to model in this network the cell membrane     
53:02     
peripheral cytoplasm the cytoskeleton the nucleus plus     
53:08     
endocellular reticulum plus mitochondria those are all like one     
53:14     
class and then information signal so the information signal was in white     
53:20     
peripheral cytoplasm is in red cell membrane is in blue the nucleus ER     
53:25     
mitochondria is all one thing and that's yellow and so they they use those as     
53:30     
sort of the nodes of this uh Reservoir computer they throw them and they have these randomized connections and then     
53:38     
the reservoir Computing as we talked about and in previous meetings kind of Mak sense of this so     
53:45     
that's what they're doing in this paper I think it's interest an interesting approach to this problem again Reservoir     
53:51     
Computing you know you could argue that it's kind of this um sort of magic that happens because     
53:59     
you know we we know that like we can model well we know kind of how uh neural neural networks work we     
54:07     
know how deep Nets work uh sometimes they're hard to interpret what what's going on inside of them and the same     
54:13     
hold if not more holds true for Reservoir Computing so this is where     
54:18     
they talk about hoden and Huxley and some of the models that we have currently so in hodkin and Huxley we     
54:25     
model the um ion channels all sort of from different classes with different classes     
54:31     
of ions and we can model the flux depending on what's present in the cell and you can model things like uh you     
54:38     
model Sol depolarization that way but you can also model Action potentials anyway so there are a lot of ways you     
54:44     
can apply AA and Huxley uh we hypothesize that ion flows     
54:49     
and neurons represent a specialized adaptation of a broader information acquisition processing and response     
54:56     
Dynamics centered around the cell membrane so hin and hxy doesn't model the information content but and they     
55:04     
don't take it doesn't take into account a lot of the structures in the cell it just models the flux of ions that's all     
55:11     
it does but in this case what they're modeling is they're modeling the cellular response the phenotype and so     
55:18     
this is you know uh I think a nice addition to hkin huxing it tells us a     
55:23     
lot more in some ways what's going on and the S during these these ion     
55:31     
fluxes so that's uh that paper for in     
55:37     
iscience all right any questions or     
55:52     
comments okay so um again uh for the benefit of     
55:59     
repetition uh Google summer of code we're going to be making decisions this week or I rather incf will be making     
56:07     
decisions this week and we're going to live with the decisions they make so you know we if you get selected you'll get a     
56:15     
message uh sometime I think midweek and if you don't get selected you can still     
56:21     
participate in the summer uh once we get our people coming on board we'll have an     
56:27     
onboarding process we'll have uh what they call the community period which is     
56:32     
several weeks of getting to know the community and I'll try to make that easy for people now in our case we have two     
56:39     
communities or two sort of Allied communities we have dorm which is this     
56:45     
group and we have openworm which is the larger group of openworm where we have     
56:50     
people doing other kinds of activities modeling other parts of celegans and we     
56:55     
talked about this in previous meetings and so people are welcome to do things with Graal networks with dvo learn with     
57:04     
anything that we're doing in the D worm group and then anything open worm is doing so we have a lot of things that     
57:11     
you could do whether you get selected or not and if you're doing your project you know you want to stick to the schedule     
57:17     
on the project but you also want to maybe make some connections in the larger openworm community that would be     
57:24     
good I think for future endeavors but also for maybe some of the work you're doing in the projects because our     
57:31     
projects often times in past years we've had people uh want to model things like     
57:37     
movement of the worm or other uh aspects of worm behavior biophysics and their     
57:44     
Tools in open orm's pane of tools to do that and their data sets as well so we     
57:50     
can investigate those in the community period um but even if you don't use them     
57:56     
it's nice to know about them so that's coming up this week um so     
58:02     
yeah next week we'll continue with more things probably welcome our new uh     
58:08     
students for gso when we find out who those are so uh thanks for     
58:14     
attending and uh talk to you next week all right now I'd like to talk a     
58:20     
little bit about a technique called Tomo seek so this is a technique that looks     
58:26     
at spatial transcriptomics and you know there are a lot of advantages of spatial     
58:32     
transcriptomics especially looking at things like embryos and even individual cells where     
58:37     
you can look at the transcripts happening in a certain region and you can localize those     
58:42     
transcripts and see other changes over time so it's very useful but it's hard to kind of make sense the data people     
58:49     
use things like dimensionality reduction umap and so forth and there have been some debates as to whether umap is     
58:55     
useful at all um now I happen to think it is but I think often people     
59:00     
misinterpret what exactly is going on so we're going to talk about some techniques for this a book chapter on     
59:07     
the technique uh we're going to leave the uh analysis and the methodology for another     
59:14     
day but I just want to go through this method     
59:20     
now so this is from a book methods in molecular biology number 1920     
59:26     
vertebrate embryogenesis embryological cellular genetic methods so this is a book on has     
59:33     
a host of methods in it the method itself is in chapter nine     
59:39     
this is RNA tomography for spatially result transcriptomics or Tomos so they're using RNA tomography     
59:46     
and I think we talked about this in a paper last week and I wanted to follow up on but it may have been a talk I     
59:53     
attended to where we talked about this but anyways this is using Imaging to find RNA in     
1:00:00     
itself so this is really you know kind of an interesting technique especially     
1:00:05     
with respect to you know other structures so we're looking at cytoskeleton we're     
1:00:10     
looking at the shape of a cell or the shape of a population of cells we can     
1:00:16     
use this in tandem to find RNA expression and resolve that     
1:00:23     
spatial so the abstract reads embryo IC development is heavily dependent on temporally spatially restricted gene     
1:00:31     
expression uh spatially resolved measurements of gene expression are therefore crucial for identifying novel     
1:00:37     
regulators and their and an understanding of their function so we have onitu methods for     
1:00:44     
this we can use like different antibodies and stain for RNA and we can look under a microscope and see the sort     
1:00:51     
of where those stains are in the cells if we have like cell culture or if we     
1:00:57     
have uh a model organism we can look at like a slice under the microscope we can     
1:01:03     
see how it kind of you know how where it's expressed and so this you know this is     
1:01:09     
good for like looking at specific genes if we're interested in a gene of interest and how it's expressed maybe     
1:01:17     
across different time points or with respect to some you know disregulation     
1:01:23     
or adant expression or some sort of function but Inu methods do not resolve     
1:01:30     
Global gene expression and sequencing based methods usually do not provide spatial monage so     
1:01:36     
we have these Inu methods that are of course you know pretty established um we can't necessarily know     
1:01:44     
much about global gene expression we can only know about maybe a gene of Interest     
1:01:49     
or several gen pictures and so you know in a high throughput context that's not     
1:01:54     
very useful on the other hand we do have nextg sequencing and that does provide This     
1:02:01     
Global gene expression information but it doesn't usually provide spatial information so something like RNA SE we     
1:02:08     
get an abundance of transcripts we get u a sequence for different segments so you     
1:02:16     
know the RNA has different U you know sequences that can be mapped back to a reference genome but we can't     
1:02:23     
necessarily know spatial location if we do single cell RNA SE or single cell     
1:02:29     
sequencing we can get down to the level of a cell but beyond that we can't say within a cell or even within a     
1:02:36     
tissue uh where these uh rnas are being expressed differentially because in fact     
1:02:42     
we can take single cells but we often take a sample we dissociate the cells from its tissue and then we take the     
1:02:48     
single cells themselves and we usually average over single cells if we want to say something about a population so we     
1:02:55     
don't have that that spatial context here we described Tomos a method     
1:03:00     
that combines classical histological sectioning of embryos which is the thing that we're doing in Inu     
1:03:07     
methods with a highly sensitive RNA sequencing technique so they're taking or sequencing they're combining it with     
1:03:14     
the histological preps of insitu uh gene expression essays and they're using this     
1:03:21     
as a single method so in this paper in this chapter they're applying Tomos to zebra fish     
1:03:28     
embryos and it allows reconstructing the spatial gene expression of thousands of genes and so you know when we talk about     
1:03:36     
spatially restricted gene expression we're talking about the establishment of patterns that sort of play a key role in     
1:03:44     
the development and tissues of tissues and organs so when we're talking about spatial restrict to gene expression     
1:03:50     
we're talking about pattern formation we're talking about differentiation     
1:03:56     
of cells into different tissues we're talking about spatial phenomena we're also talking about temporal phenomen but     
1:04:03     
we can view this in terms of space so you know whatever processes happen in     
1:04:08     
time they become manifest in spatial arrangements and so uh you know the gold     
1:04:15     
standard div visualiz already localization has been home out or or section in citro     
1:04:22     
hybridization and these These are simple protocols but they don't get give us the kind of information we need for what we     
1:04:28     
want to do here so you know there are a lot of techniques like M fish and seat fish     
1:04:35     
that have been developed uh to overcome the limitations of insitu which has been     
1:04:40     
just basically the number of genes that you can look at at any time uh by sequential hybridization of     
1:04:48     
short probes coupled fluorescent dyes some hundred genes in their localization can be assessed in a single tissue     
1:04:54     
section so these are these fish methods uh this is again you know it's it's     
1:05:00     
getting better but it doesn't tell you a lot like you know it doesn't it doesn't square away with a lot of the next gen     
1:05:08     
sequencing while providing detailed spatial information these protocols put High requirements on the equipment and     
1:05:14     
currently do not scale the level of whole transcripts and then of course classical     
1:05:20     
R sequencing provides Crypton L expression data that's basal information     
1:05:25     
is lost so here they describe tomek method     
1:05:33     
for spatial way result transfer atomics combining RNA barcoding techniques which     
1:05:38     
is used in single cell approaches with classical tissue sectioning which is of course used in in     
1:05:45     
cun uh and of course Tomo uh doesn't require any real specialized equipment     
1:05:52     
it just can be used you can use standard lab equipment to do this this they provide a protocol in this in     
1:05:59     
this chapter so the steps from the Fresh sample to the sequenceable uh Library     
1:06:05     
can be accomplished in five to seven days it's a pretty good timeline for doing this you know where where you know     
1:06:13     
a lot of techniques will take you know weeks and weeks to perfect I mean I don't know depending on Whose hands this     
1:06:19     
is in how long this will take but ideally it will take five to seven     
1:06:24     
days almost is a simple but powerful technique for accessing spatial research     
1:06:29     
gene expression and is particularly suitable for applications and     
1:06:34     
development the spatial expression of thousands of genes can be mapped simultaneously and consequently can be     
1:06:40     
used to describe animal development so the sample preparation it     
1:06:45     
really doesn't involve very much stuff you need a poly acramite beads you need     
1:06:51     
the typical lab equipment you need uh trizol glycol blue these are all things     
1:06:59     
to extract RNA from tissues from cells so you know it's not that much more than     
1:07:04     
your typical sort of uh lab equipment and so you just need to     
1:07:11     
extract the RNA from a sample you need to run a cdna     
1:07:16     
synthesis and you know basically some PCR and you need to be hand to you PCR to do this run this     
1:07:23     
protocol and so what they they're doing is they're working with zebra fish embryos they're using the cryostat to     
1:07:29     
get thin slices and uh they're using a slightly modified version of the cells uh cells 2     
1:07:37     
protocol trip repair sequencing libraries a slice specific DNA barcodes     
1:07:42     
so that's uh citation eight and citation nine so this is celc2 this is a specific     
1:07:48     
protocol for this and so basically you're taking a tissue sample you're extracting the RNA you're building a     
1:07:54     
barcoding a library of cdnas you're then doing all these steps     
1:08:00     
that involve like amplifying RNA and checking for you know quality control     
1:08:06     
and then building a PCR library library check and sequencing so you're doing this uh you know you're going through     
1:08:15     
this protocol they introduce these stop points just as as a means to like do a     
1:08:21     
reality check to make sure that everything is working well so you're actually creating these cdna libraries     
1:08:28     
that can be used then to match up with the RNA and so then     
1:08:33     
we're you know we're talking about spatial resolution so we want to look at the zebra fish larvae at 48 hours so     
1:08:41     
natural if we look think of the zebra fish uh embryo we have this spatial     
1:08:46     
relationship between the anterior end and the posterior end we have this uh     
1:08:51     
we're trying to sequence uh different tissue slices so we're taking the slice from the anterior     
1:08:57     
to the posterior and we're kind of mapping out what this you know where these slices belong in along this     
1:09:05     
anatomical axis we're going in the direction of sequencing so we're sequencing from anterior to posterior so     
1:09:13     
in in terms of output material we have these uh cdna libraries we're matching     
1:09:18     
them up to RNA and this is the output material this is a spectrum uh for AR so     
1:09:25     
this is where you have typical Arna on an RNA Pico chip and this is a typical library on a DNA HS chip so this is in     
1:09:33     
base pairs and you have all these samples and you want to get a sense of you know how many bases longer the     
1:09:39     
different fragments and so we have this distribution here we want to be able to     
1:09:45     
match this peak of between three and 500 base pairs and we want to be able to     
1:09:52     
minimize the number of over thousand base pair size fragments so we want to find fragments that are small enough to     
1:09:59     
sort of identify unique RNA but we want things that are also clean we don't you     
1:10:06     
know we want to have a good sample a good representative library and so this     
1:10:11     
is an example here of Arna this is sort of been between 3 and 500 minimizing     
1:10:18     
everything over a thousand base pairs um this is the final Library size Distribution on a bio Ani     
1:10:25     
High sensity chip sensitivity chip the distribution should be have a peak     
1:10:31     
between three and 500 base pairs like the first figure and fragments larger than a th     
1:10:36     
base pair should be negligible so we have this Peak at about 1,300 base pairs so you know we can     
1:10:45     
actually also look at artifacts so this has a smaller Peak on 150 base pairs and     
1:10:51     
so this can be due to primary artifacts which is basically you know uh in the     
1:10:57     
step where you're amplifying your uh RNA and you can filter those out that's not     
1:11:02     
a problem but you want to have a clean curve and sometimes you have messy curves uh sometimes you have artifacts     
1:11:10     
so those you want to be able to check for that and so this is what the data should look like here so we have our     
1:11:16     
genes of Interest here we have again this is not you know a Canada a gene approach this is where we're getting     
1:11:22     
these fragments and they can be mapped to as specific Gene and so then we have this basically this list of genes     
1:11:30     
although it's not a canidate gene approach we just have our different fragments     
1:11:35     
representing what what you know that Gene the constituent of that Gene and then we have our sections so we have for     
1:11:44     
example the gene ink one and then we have section one section     
1:11:50     
two so forth and so recall that we had that figure of the zebra fici CH where     
1:11:55     
we go anterior to posterior so section one would be anterior section 96 in this     
1:12:02     
case would be posterior um and then there's a     
1:12:07     
number which is sort of your measure that your target measure and so what     
1:12:13     
you're measuring is you're measuring the presence of ank one in each anatomical section so you're taking each section     
1:12:21     
you're getting not only for anank one but for all your genes your data set and     
1:12:27     
sometimes you know you have fragments that don't map to a certain gen so that's what these dots are and sometimes     
1:12:32     
you don't get amplification you don't get any sort of signal so you have these dots     
1:12:38     
here there there's missing data in here uh the dots can be also you know this is     
1:12:43     
sort of uh showing like where you know you have in this case you have     
1:12:53     
U so that's you know and then there you know we have to clean up our data and things like that but this is what the     
1:12:59     
output should look like you should have different sections have different genes and then the intersection of those     
1:13:08     
two and so you know we're using a measure of normalized expression here so our Target measure is where we take the     
1:13:14     
raw number and we normalize it against some housekeeping gene or some other     
1:13:20     
method of normalized expression there are number of methods of normalizing gene expression we typically don't want     
1:13:26     
to take the raw number because the raw number doesn't really mean anything we can't compare it to anything especially     
1:13:33     
cross-sections so we usually normalize it against something um and so this is uh on the right this     
1:13:41     
is a graph of normalized gene expression against section number so here we have     
1:13:46     
our section number again section one is anterior most and then the highest     
1:13:52     
number is posterior most and this is for one gene exd2 or at least uh transcript     
1:13:59     
that belongs in that Gene so this is spatial expression of     
1:14:04     
xd2 this is the normalized expression and you can see that in basically what     
1:14:09     
this graph is telling us is that this Gene is expressed largely in the anterior portion of the embryo and so     
1:14:17     
you have some expression down in the posterior end that could be noise that     
1:14:23     
could just be you know um random gene expression whatever but we can say     
1:14:28     
basically that there's maybe something going on in the anterior end that's     
1:14:33     
interesting that involves xd2 and then since we have all of our genes uh and     
1:14:40     
we're assaying them in the same context we can then put together maybe like a gene regulatory number and say okay what     
1:14:47     
is xd2 you know what is it related to what is it in a network with what is its     
1:14:52     
function we can use all sorts of bioinformation itics tools to augment this method and ask questions of so we     
1:14:59     
can generate a number of graphs for different genes and different RNA uh     
1:15:04     
sequences where we can say you know all these are upregulated in the anterior     
1:15:10     
portion of the embryo so there must be something going on so this is how we can go from this kind of you know wouldn't     
1:15:17     
say it's high throughput data but it's also it's kind of a shortcut to a lot of     
1:15:23     
the uninterpretable Big Data of a nextg     
1:15:29     
sequenc so this is a an interesting approach     
1:15:35     
um it certainly is I think a complement to maybe nextg sequencing approaches it     
1:15:40     
certainly also gives us uh really useful information that we otherwise wouldn't     
1:15:45     
have and so that's uh all I wanted to say about that paper     
1:15:51     
uh we'll probably talk more about this in future meetings so thank you for     
1:15:57     
paying attention and I hope you Le something
