     
Transcript     
0:03     
hello hi Hi how are you hello hello how are     
0:11     
you so we have yes Michael and Dick and Susan and     
0:16     
Jess welcome so Michael uh would you like to introduce     
0:22     
yourself yeah Michael burer U software engineer just been interested in the     
0:28     
open Worm Project for several years now my interest has mostly been from soft     
0:33     
from neural architectures what I was trying to do is take the the XML data and convert it     
0:40     
into or write a piece of code that would generate some list code that would Implement some sort of framework I was     
0:47     
just just basically trying to come up with something that would you know make it easier to hack around on on some neural structures     
0:54     
yeah yeah that's great um welcome yeah we we do a lot of different things in     
1:00     
this meeting we have uh like uh we talk about biology but we also talk about the     
1:05     
computational stuff we do a lot of stuff with like machine learning and deep learning but we also are interested in     
1:12     
just any kind of thing that people want to do so yeah I've done some I've done some work on um machine learning and     
1:20     
also as far as um neuro Neuroscience that's kind of like a side hobby     
1:26     
something I really when I'm sitting at the bar and boardless just out of Neuroscience book kind of a geek that     
1:32     
way yeah yeah great thanks all     
1:39     
right so anyone else how are you doing Susan oh I managed to make one of my     
1:47     
models work the simplest one because I put it in what's called a T4     
1:53     
configuration and then I realized oh it doesn't like sequential forces     
2:00     
uh added together on it so if you just take one force at one time and let it     
2:06     
settle then you get a result so I have to do each Force individually because it     
2:12     
has a refractory period And I don't know how to program or refract rep period into this stupid console program oh so     
2:23     
it's you just have to do Force by force by force     
2:31     
yeah I don't know it works so I'm I'm just gonna do     
2:38     
that it's G to take all week but I'll get my data and then maybe I can quit     
2:45     
well yeah hopefully well if it works then you're at a stopping point so yeah then you do what you have to do     
2:52     
to get the data right yeah only need like 50 data points     
2:59     
or something because doing more than that will drive me nuts yeah what kind of data are you     
3:05     
using for it what kind of data are you using for it um just to get a stress strain curve     
3:13     
okay for a Tegrity that I've built that has a top part that's acting like the     
3:20     
actomyosin ring it's in the top part of epithelial epithelial cells pardon me     
3:28     
and um it has a center that's uh     
3:34     
triangular also Tegrity which I should probably switch     
3:39     
up into different configurations to see what works to see     
3:45     
how it changes things and uh it thank goodness it's only a three     
3:51     
cell um construct of tissue probably should do like a seven     
3:58     
cell one but oh probably not to start I mean I would get the three cell working     
4:03     
and then move out then I'd have some data and I have a     
4:09     
model and things would go better yeah but yeah it it needs a settling     
4:17     
period before it um before it'll give me data otherwise     
4:22     
it just bounces I guess all over the place and goes doesn't compute or that's     
4:28     
how it says but not resolved yeah well at least you got it     
4:35     
working take a while yeah well took me a while to     
4:40     
realize that oh yeah that works okay so let's do the the series well the series     
4:47     
doesn't work and I'm going oh okay well let's     
4:53     
try 100 and 200 separately and then it worked     
5:02     
so wow yeah is the it's a finite element     
5:09     
analysis program and 10 segres don't work that     
5:14     
well with finite element analysis because they have a large     
5:19     
movement whereas the the finite element analysis is trying to do is a small     
5:25     
movement to measure things and you you push a tensity um quite often with a     
5:33     
small movement and it changes everything all the tensions in the T seg because     
5:40     
the forces get distributed throughout     
5:46     
it see this with this this Tegrity object if you pull on it the     
5:56     
pulling uh gets distributed throughout it and if you push it the it changes     
6:03     
configuration everywhere like the forces get distributed so that seems to drive my pal sole     
6:12     
program that's okay like said it it also doesn't     
6:19     
happen all at once like it kind of changes shape and changes     
6:24     
shape and then it changes shape and changes shape again like it does this and then that and yeah so it's not a not     
6:33     
really linear sometimes yeah it's going to be a lot of nonlinearities in there that you have to     
6:40     
or I guess the whole mode is nonlinear yeah well and it changes um too fast     
6:47     
and I don't know anyway it has this refractory period where it needs     
6:53     
to um finish Contracting or something     
7:01     
yeah it's uh put in push me pull me into the isn't that a program um try to get     
7:08     
push me pull pull you on my computer and I keep downloading Java because that's     
7:14     
all I can get like it doesn't download anymore like it won't won't work     
7:21     
somebody's left neglected it yeah it looks like it     
7:31     
well it's it's that's good that you got it working at least in in a very small way and then you know yeah it just needs     
7:38     
to sort of work that's all that's required because I'm switching this PhD to a master's degree and they said it     
7:44     
could be Discovery and so I discovered how to make it work because it doesn't     
7:50     
fit with finite element analysis very well yeah in some instances so good uh     
7:58     
I'm I'm good that's good do anything     
8:04     
else looks like ved's here and hansu hello where are you Ved uh it's a great     
8:14     
background in a star     
8:22     
[Music] okay been working both uh topics that we     
8:30     
have been discussing so far uh the one with uh p uh I could resolve some     
8:39     
problem that they have been dealing with h so I'm     
8:46     
conation part uh from now on and uh also the other     
8:54     
part I've been work uh Garden and uh mamad that we have     
9:03     
been presenting some some ago uh some to     
9:09     
the IM construction yeah     
9:15     
I we working by some suggestion that muhamad and     
9:23     
been I     
9:29     
IM con I could as well and yeah     
9:40     
maybe okay so you're it's coming along you're not you don't really not in any sort of     
9:48     
not experiencing any uh obstructions or     
9:57     
anything well you're everything's going smoothly you're not having any     
10:04     
problems yeah but uh trying not     
10:09     
to yeah have some issues     
10:23     
maybe okay well good luck yeah I don't know uh I don't know when you want to if     
10:30     
you ever want to present in weekly meeting I would I would you know even if it's not finished because you know we     
10:35     
usually like to Bradley     
10:40     
yeah when he's ready it might good to have a general uh introduction to vs oh     
10:50     
yeah like so like yeah you know what you're trying to do is     
10:57     
use right yeah     
11:07     
yeah mamed seems to be the expert on point oh yeah okay uh yeah yeah I mean I     
11:16     
I I know like the origins of voids and some of the work because there people have been doing a lot of work on voids     
11:22     
like you know like perfecting how they all sorts of like different points of     
11:28     
view of the agent and things like that so it's really interesting work but uh yeah I'd like to see like kind of     
11:36     
the state-ofthe-art there hello Luke uh welcome and Morgan     
11:43     
welcome hey uh thanks     
11:48     
for for us yeah yeah all right uh let's     
11:57     
see okay so yeah we we'll talk a little bit today about some uh I have a couple     
12:03     
papers on simulations and large language models being used for biology I also     
12:10     
wanted to go over I have the slides on um uh yeah Michael asked who's had a     
12:15     
question in the chat I also have some slides on the 10th anniversary of dorm and I wanted to look uh we wanted to go     
12:22     
through them and see if I missed anything so I'll go through kind of the slides that I have and I don't know if     
12:28     
I've missed anything it may be something a dick could refresh my memory on or maybe other people if     
12:34     
I'm missing something that is you know pretty obvious but I I missed it so     
12:42     
let's let's get into that so last week I told you that like this is the this week     
12:48     
this past week was the 10th anniversary of Diva worm back in April of 2014 we     
12:54     
had our initial meeting and we had our initial GitHub uh issue that we filed     
13:01     
and basically you know we've been going ever since so I think we've maybe you     
13:06     
know we take breaks from week to week but it's not like um you know we haven't taken a long extended break which I'm     
13:14     
not saying that we will I'm just saying that you know this is pretty we've been going for 10 years so it's like we got a     
13:20     
lot of under our belt and you'll see that in the slides so yeah in the beginning in April     
13:26     
2014 uh Nick and myself and Steve Larson from openworm we got together and we     
13:34     
discussed this idea of modeling development in SE elegans and there's the openworm project     
13:40     
that's ongoing of course or that started openworm started in 2011 so at the time     
13:46     
it was pretty new and so we at the time you know we said okay we need a developmental component to this so there     
13:53     
was this original issue issue 179 on the openworm um     
13:59     
GitHub and it was add cell lineage info to Pi openen so this was the cell lineage of     
14:06     
cigan and putting that into a pi open room format so a lot of the early work was like trying to do some uh well data     
14:13     
structure stuff or or you know um that sort of thing so there's Steve Laron in the middle we were using Google Hangouts     
14:20     
at the time because that's what we did and uh then we had our first Public     
14:26     
Announcement in June of 2014 this is where I put this announcement on     
14:31     
on a Blog and asked you know it was sort of a call for involvement we had a few more people come in at this point and     
14:39     
you know we're really working from open data sets so the seans community has a really nice collection of open data sets     
14:48     
uh you know from like uh sell lineage information to sell tracking     
14:54     
information to other types of information that you know you can use     
15:00     
uh and you know the SE elegans Community is well known for maintaining those resources so it's the perfect sort of     
15:07     
model organism for this kind of endeavor uh then we did this paper in     
15:13     
2014 in October where we did uh we we proposed this four-dimensional model for     
15:20     
the seans lineage Tre using an rdf framework so this is uh data like     
15:25     
proposing a data structure for the lineage tree and then having like a theoretical skin over it explaining why     
15:32     
we're interested in this and very early on WE expressed an interest in looking     
15:38     
at differentiation and partic in particular what we call differentiation waves and     
15:43     
this is something that uh dick Gordon has worked on his entire career at least     
15:49     
he's sort of the originator of the concept and so we worked on this uh     
15:54     
problem and you know it kind of put out a first uh proposal for how this might     
15:59     
be approached so uh about five years ago     
16:05     
now and 2019 was five years ago uh we did a I did a report on the fiveyear     
16:11     
sort of anniversary of dorm so if we go back to five years in to the project we     
16:18     
have a lot of things a lot of papers that we did a lot of Journal clubs open worm doesn't do Journal clubs as much     
16:24     
anymore but we we did present at the journal clubs there     
16:29     
uh I've presented on different things at different uh locations such as the seans     
16:36     
meeting uh and uh you know we we have attended that as sort of you know been     
16:43     
represented there a couple times uh we published a number of papers uh and you know so we've we've     
16:52     
really kind of had been productive over that 5year period And so this is at the     
16:57     
five-year Point kind of where we were um and so yeah we've had a number of     
17:03     
logos throughout the years you know from the early days of the project we just kind of we made up a logo to different     
17:10     
logos for some of their side projects like the dvo learn platform uh and this is a software open     
17:17     
source software been working on over the last few years and so you know we we've     
17:23     
got a lot of tentacles in a lot of places at this point so you know we started out very small we've end up kind     
17:29     
of uh not only in cgans but in other organisms as well so um and and going     
17:35     
back I was looking and I was like you know this is incredible how much work we've done in 10 years so um we use a     
17:43     
lot of kind of secondary data we have this is an example of Point cloud data so in the embryo you can take uh     
17:50     
microscopy images of the embryo and you can track the cell nuclei or at least     
17:55     
markers that represent the cell nuclei and you can calculate cell centroids so     
18:01     
you can calculate out the 3D position of those centroids you have the stack of images you have the the width and the     
18:08     
depth or the width and the length of the images of each focal plane and you can get this three-dimensional Point Cloud     
18:15     
so this three-dimensional Point cloud is in a you know they have a a reference a reference point to each other's each uh     
18:22     
you know one cell to another cell and there's this a data set for Murray at     
18:27     
all uh from it's from 2006 but it's perfectly good data uh that allows us to     
18:34     
get like probably I think they have about three or 400 worms or worm eggs     
18:40     
that they've sampled in that data set and you know you can take those data and plot out this kind of Point Cloud that     
18:47     
shows every cell centroid in development so you can see it roughly forms this sort of embryo shape so you have all     
18:54     
these cell centroids and each one of these points is in a 3D location but it also has information attached to it so     
19:01     
you have uh position the 3D position you have information like the label you in     
19:09     
celegans if you have the label of the developmental cell you know what it will become in the adult so you can map this     
19:16     
mass of points to an adult phenotype of the worm so that's really nice uh and     
19:23     
you can do that in 3D because we have a 3D model of the adult but you also have other things that we can extract ract     
19:29     
from the data so you have time and migration angle so you can do this over     
19:35     
points and time so this point cloud is not differentiated with respect to time     
19:40     
if we wanted to we could take different time points and point you know plot out the points we have uh when cells divide     
19:49     
they divide at a certain angle they divide and go to certain places they migrate you can you know extract those     
19:56     
kind of information and what we've done create this data structure which is uh     
20:01     
multi-dimensional so we can have the cell volume the time and migration angle     
20:07     
and some other properties that we can modify with respect to our data set so     
20:13     
we've we've developed this and not just for seans but we use this basic kind of approach for other organisms as well     
20:21     
um now in 2016 there was the openworm openhouse which was an event that uh we     
20:27     
put on with our community committee uh and we did like a review of all the     
20:32     
projects so and someone even prepared some uh art for this so we had some     
20:38     
person who was developing art for open worm at one time there some nice pieces     
20:44     
of art that exist if you're interested I can connect you with this um and then     
20:50     
you know each project got an update we we did our update for our project uh at     
20:55     
the time we had a number of projects going on in openworm and this is a a point that I'm making because there are     
21:01     
a lot of things going on in open worm that people may not be aware of so you know there there are different ways of     
21:07     
modeling neurons we have uh different ways of modeling and measuring Movement     
21:14     
we have different ways of simulating the biophysics of a of a a worm and so forth     
21:20     
so there are a lot of different uh sub projects and one of the things and we     
21:25     
need to do this again is to kind of get in touch with lot lot of these projects so there's a     
21:31     
nice set of resources on this open house that if you want to go back and look at that it's a little dated now but it     
21:38     
basically gives you an idea of what kinds of things we've been thinking about okay so in 2018 then two years     
21:45     
after we had this event at the Royal Society in London and they they they     
21:50     
sponsor events uh on different topics in science and this uh event was on our     
21:56     
work in openworm on the CL connectome it called connectome to behavior so it was     
22:03     
people in openworm some labs in London that work on celegans um you know they're they're     
22:10     
biological Labs that work on these uh these problems and they have many more     
22:15     
resources than openworm but where openworm fits into this is that that openworm uses a lot of the secondary     
22:22     
data and does a lot of computational stuff so I know Steve Larson and some other people are     
22:28     
interested in formatting data sets from stuff that's experimentally collected to     
22:35     
something that can be used in a in a computer program so you know developing     
22:41     
data structures for biological data uh uh porg who was     
22:47     
mentioned uh is also interested in this type of thing and you know we we we kind     
22:52     
of have ways to like map those data sets to simulation efforts other things so     
22:59     
this this event was really nice because it gave the the project a lot of exposure uh Dil worm was featured in one     
23:06     
of the papers on the special issue that resulted from this so we had some good exposure     
23:13     
there um and I gave a talk at artificial life 2020 on this idea of open worm     
23:20     
being kind of misrepresented in a lot of people's minds a lot of people think that openworm is about like uh The     
23:27     
Singularity or like simulating like you know simulating life     
23:32     
in ways that we're not actually doing so I I did this kind of humorous talk on     
23:38     
how you know what we're actually doing in open worm and how it's relevant to like different scientific Endeavors     
23:45     
because I I I don't have time to kind of look at it here but uh you know and I     
23:50     
may add a slide in because I have some slides on this but there have been some pretty wild stories in the media about     
23:56     
what open worm is doing so it's um it's good to have a reality check on that so     
24:01     
that was something else uh that we've we've got um and during the pandemic we     
24:07     
were doing a lot of virtual conference participation so you know this is something that again is good to get the     
24:13     
word out it's a good motivator for different projects to get to a a point where you produce like a paper or a     
24:21     
presentation and you know so it's always a good thing to to S of Target these kind of things and so we've done a lot     
24:27     
of those in the history project we've also done a lot of things with respect to different topics so we've done things     
24:34     
uh with uh cellular aoma and this is Moroso and this is something that's open     
24:39     
source uh you know it's it's a a platform that Tom pigies uh sort of     
24:45     
developed on his own but we were using it in the project as a way to model     
24:51     
morphogenesis uh we've had other things like we've worked with datom uh which are different type of     
24:57     
model organism we worked on developmental connectomes uh machine     
25:02     
learning uh differentiation waves and trees and those sorts of theoretical uh     
25:08     
topics and then developmental data science where we've done data science     
25:13     
data analysis with developmental data as I showed with that slide with the point CL so you know we've done at different     
25:20     
points in the project we focused on these different uh topics but we've always kind of maintained a lot of the     
25:27     
stuff and you know I find this is a good opportunity to go back and look at what you know like if we stopped using morph     
25:34     
aoic well could we use it again could we do things with it you know just to make sure we don't forget about things and     
25:40     
they don't get swept under the rug so the point here is there are a lot of things that we've done we have a lot of     
25:46     
resources and you know if people are interested in working on something even if it's something we worked on a long     
25:52     
time ago we can pick it up and revisit it and come back to it so we worked with the lot of model     
25:59     
organisms we've worked with a surprising number of organisms uh we worked with uh uh zebra     
26:05     
fish data uh this organism that is the C     
26:10     
squirt uh worked with diatoms and of course nematodes we've done a little bit     
26:16     
of work with dropa with uh Mouse data uh with     
26:21     
theoretical data and and simulation and then comparative development where we've     
26:26     
compared different uh embryo you know the embryos of different species so in     
26:31     
one paper we did work with nematodes and with C Squirts and comparing those embryos so we've done a lot of work and     
26:37     
I was surprised to see this wide variety of organisms here represented um so     
26:43     
again you know we're interested in kind of not just kind of these uh questions about data and computation but actually     
26:51     
you know maybe like comparative biology and getting that fundamental aspect you     
26:56     
forgot the salamanders oh okay I'll put those in yeah and so yeah we done a lot     
27:02     
of things with things like differentiation trees of Mosaic development as this is a slide just     
27:07     
describing one of our papers on this where you know it was basically a reanalysis of the lineage tree where we     
27:14     
had to take the tree and sort it every node of the tree uh by a different     
27:20     
Criterion than what is typically done in a lineage tree and then be able to take     
27:25     
that and compare it across species and and say different things about and there are theoretical reasons why we would do     
27:31     
it this way so you know a lot of the original Sean lineage tree was basically     
27:38     
organized uh anterior to posterior and if you know the history of this it was     
27:43     
done by hand so uh you know solsten who was the person who developed or mapped     
27:50     
out that tree sat at a his desk and he drew it out by hand and you know that     
27:56     
that has a lot of utility but there are other kinds of questions you can ask and different ways you can organize the data     
28:02     
that are also very informative um and then this is a differentiation tree for c squir as well     
28:10     
so this is our general methodology for these kind of what we call Mosaic     
28:15     
development organisms where they um they don't have like uh these Mass tissues     
28:22     
you know like where you have a bunch of cells in a tissue their differentiation occurs on a cell-by-cell basis     
28:29     
and so in C squirt you see this early in its development and we have these kind of trees where we can you know kind of     
28:36     
do some uh we can model it we can render it classify the different cells the     
28:41     
different nodes and then map that to the uh to the phenotype which you can see     
28:46     
here and so we talked a little bit about that in this paper um and so you know     
28:53     
we've been working on a bunch of things we've also been working on cellular automa for uh looking at development and     
28:59     
pattern formation so there are these one-dimensional cellular autometa that     
29:05     
evolve over time and then we have these two-dimensional cellular aom this is morphic where you have a focal cell     
29:13     
which is in the middle cells around that cell that you know they have interaction rules so the cells around this focal     
29:21     
cell determine the state of the focal cell in the next iteration of the simulation and then in Moroso these are     
29:28     
nested so you can actually look at these kind of global effects and reason I     
29:33     
bring all this about cellular aoma up is because there's so much here that we've not done we we've kind of scratched the     
29:40     
surface of it and then left it sitting there um and we've had from time to time people wanting to do work on it but it's     
29:48     
really I don't know what it is but like there's something here that really could be like something if you want to do this     
29:55     
and you can do this with you know you can take take data from a data set or you can just model it with pseudo data     
30:02     
or whatever but there's something here fundamental pattern formation I think would be really     
30:08     
interesting um so anyways we've done that work then     
30:13     
you know we have our community we have our open source Community we do we release all of our code in open source     
30:20     
so we have the Moroso work is open source we have all the stuff we've done     
30:25     
with machine learning that's all open source and so we have this nice Community uh where we have different     
30:32     
resources our website our GitHub uh site which has a number of data sets uh we     
30:40     
have a a presence on GitHub and hugging face hugging faces where we host our     
30:45     
machine learning models and then we have our meeting spaces jitsy and also on YouTube where we do a lot of things as     
30:52     
well and you know this is sort of semi-independent of the openworm community because     
30:58     
in our community we're kind of doing the developmental stuff in open Worm but also doing other things so we have our     
31:05     
own presence in on YouTube for example where openworm as a channel and we have a channel and we do these things where     
31:12     
we've done a lot of things over 10 years I used to host an open coffee hour uh and then we also have you know     
31:20     
some of our work posters talks uh special things that we do and so there's     
31:26     
a lot here if you want to go to the YouTube channel it's just Diva worm on YouTube and you you know there's a lot     
31:33     
of content there we used to have Talks by people so there's a lot of stuff there to explore and maybe follow up     
31:40     
on then the the International seans Conference which is this conference so     
31:46     
celegans has a really nice community of people who do you know they collect data     
31:51     
they they have they're very open about like curating the data and sharing uh     
31:58     
data and it's largely you know it's really it's really uh not just kind of thrown up on the web people have worked     
32:04     
hard to develop what they call defined mutants so you can get celan mutants     
32:10     
where they have a gene that's mutated and it has some functional consequence     
32:16     
to it so you know there there tools like that but then there's also data sets like you know uh lineage trees and other     
32:24     
types of uh open data movement data and things like that and so we go to this conference we have a presence there and     
32:32     
last well we were there in 2015 I was there they used to host it in Los Angeles but now they've moved over to     
32:38     
Europe they moved over to Edinburgh and that's where porg is always the person going there because that's sort of where     
32:45     
he's from you know and so uh in 21 and     
32:50     
23 we did a poster and uh Diva worm was in that poster so we have a very small     
32:56     
space to kind of advertise it we're doing but uh you know we we get exposure in that     
33:02     
Community then of course we do our our get our summer of code stuff so we have our many many summer of code interns     
33:09     
that we've had since 2017 working on a variety of topics and the way we     
33:15     
approach uh summer of code is we you know we take a project one year people     
33:22     
you know dive into a topic they propose a topic they do the work we you know     
33:27     
develop some code it's usually some sort of application and then the next year we follow up on that project and you know     
33:35     
kind of pick up where we left off and get new people in so we've had quite a     
33:40     
few uh interns over this over the course of this program it's been very successful for us in developing a lot of     
33:48     
the machine learning stuff that we're doing which I haven't even talked about in here I'm not going to talk about that too much but I think it's worth pointing     
33:56     
out and then pointing out that like also if you're one of these people who     
34:01     
participates in gach you come into our community you learn about biology and     
34:07     
computation and those kind of linkages but also people you know they're very ambitious and they go on to some great     
34:13     
things so uh osid Singh who was in our group several years ago he's now a     
34:20     
graduate student at Georgia Tech uh mayuk Deb who's been in our group uh quite a while off and on he's he's been     
34:28     
doing some great things he wrote an article in the gradient which is a machine learning blog uh Journal kind of     
34:35     
thing and so you know there you can see them around uh Beyond divil worm so it's     
34:41     
great to see people doing stuff like that um and so yeah we in in 2019 we had     
34:48     
this course sto learn ml so there's an educational component here that we focus on and we did this 16we course which is     
34:56     
all the materials for this or online where we we did a series of talks on     
35:02     
different topics in machine learning so um these are all kind of available and     
35:08     
you know it's a little dated now but I I'd like to do this maybe again at some point but this is the kind of thing we     
35:14     
do with their educational initiatives we've developed this software D learn which is uh you know     
35:22     
where you have machine learning applied to biological data specifically     
35:29     
microscopy data so we have all sorts of microscopy data Susan's been doing     
35:35     
things with things like the ball microscope we have data sets that we can mine to get these kind of uh we we     
35:43     
interested in segmentation of data but we're also interested in building higher level models from the segmentation Maps     
35:50     
so you know we have to deal with all sorts of uh issues with input data such as cleaning it up defining what we want     
35:58     
to get out of it and then building higher level models and so in doing this we're building this pipeline of     
36:04     
different tools and you know it's open source so anyone can contribute to it     
36:10     
but we're we're you know we're making progress on this um and this is largely through Google summer of code so that's     
36:17     
that's nice too we've developed things like the pre-trained segmentation model the     
36:23     
embryo generator model and the lineage population model these are all kind of parts of dvo learn and you know     
36:30     
they use different techniques so I'm not going to get into the techniques but um they're really you know their Forks of     
36:36     
things like Gans and the segment anything model from meta and so they're     
36:42     
very state-of-the-art but you know there you know there's a lot there to kind of     
36:48     
if people want to participate or contribute to this it's it's open source so it's something we can     
36:55     
do um yeah so we have a presence on GitHub of     
37:00     
course we have a for Dil worm which is like where we have some of our data sets     
37:07     
some of our Legacy things we also have Diva learn which is a separate organization where we have this     
37:14     
software and then we have you know but the problem is we have a lot of technical debt and this happens over the     
37:21     
course of 10 years where you you know have different like I showed you that D.A wor ml example     
37:28     
there's it's it's kind of dated now so you know technology changes at such a     
37:33     
pace both in like computational science and in biological science that there are     
37:39     
all sorts of new data sets there new techniques there's new technology and we need we you know we struggle to keep up     
37:46     
everyone does and so when we do something sometimes it's not the state-ofthe-art sometimes our     
37:51     
documentation isn't in sync with things so there's a lot of technical debt in DVA worm um and one of the things we do     
37:58     
is how we have this read the docs so this is one piece of documentation we have which is basically as been updated     
38:05     
since 2018 these resources on read the docks and hasn't been updated since 2018     
38:11     
and you know we haven't really used it that much but the point is is that you know we have resources that people will     
38:17     
come across coming into our community that are not you know in sync with things so we recognize that and it's     
38:23     
always something it's an interesting aside for the 10 years of doing this that you'll have things like this in a     
38:30     
project and so it's not to say that we need to like I guess we can tighten things up but it's also kind of     
38:37     
interesting to go back and look at where we've kind of gone and maybe revisit or     
38:42     
maybe align ourselves with things that happened in the past like these categories that are built in this uh in     
38:49     
this scheme you know maybe our work is better organized this way and we've just     
38:55     
not organized it that way in a while or you know changing this to reflect what we're doing now there are different ways     
39:01     
you can do this this is more of an open- Source curiosity but it's an interesting point when you're going over 10 years of     
39:08     
work to see this uh in action so we you know sometimes we focus     
39:14     
on open questions so dick Gordon is good at this coming up with open questions to     
39:21     
address um you know some of the open questions that he submitted this was back in December 20 20     
39:28     
23 is where you know one question is is that you have uh in in biology especially in like     
39:35     
developmental connectum we have changing networks of neighbors from the point of     
39:40     
view of one cell in an embryo so when I showed you that point Cloud you can see it here if I were sitting here as a cell     
39:48     
and now we're looking out at my neighbors and now we're looking out at these connections what would it look like what would be the view from where     
39:55     
I'm sitting you know this is something we a model with voids this is something we can do with network science and you     
40:02     
know especially with connectomics it's very relevant because one neuron is Du influencing other neurons and     
40:09     
understanding what that Viewpoint is I guess it's metaphor but the Viewpoint being what am I influencing what am I     
40:16     
maybe you know trying to do in a in a biological system as part of a biological system that's an interesting     
40:23     
question and it's something you know someone can pick up a question like like that and interpret it the way they     
40:29     
want uh and so another thing is can we shift diatoms or which are individual     
40:36     
cells from one category to another with what we call a compuse stat which is     
40:41     
something dick wrote a paper on and I think the 1960s but it's you know something that     
40:46     
is relevant to say like cybernetics so it's something we can look at from a     
40:52     
modern perspective and then there are other model organisms these Sicilian     
40:58     
zapota which is a type of uh worm and they have this uh literature so there's     
41:04     
a lot of stuff in biology that is like very obscure that we can pull forward     
41:10     
into the present there may be data on this that we can analyze like images that people have published it may be     
41:16     
that we can you know pick up on a very obscure model organism that would be very useful for what we're doing in open     
41:23     
worm and devil so there are a lot of things like this that are kind of buried in the biological literature that we can     
41:30     
ask questions about we also do things like Vis revisit     
41:36     
Concepts so back in 2021 we had uh open questions about what     
41:42     
we call environments so sometimes you'll see things in papers called the environment and you ask what is it what     
41:49     
is its relevance and how do you define it so one thing is that the environment is permissive so you know we we have     
41:55     
that and then we have the environment being instructive so we can build models around that and use that in our models     
42:01     
and in our data analysis and so we've lost people during the 10 years Steve McGrew died uh a     
42:09     
while back a couple years ago and he did this uh talk principles of uh F     
42:16     
phography which is this technique that he worked on developing and this is a     
42:22     
talk this is the link to the talk it's very very popular I don't know how many views it has has hundreds of views for     
42:28     
something like that that's pretty good so you know there are people who we miss who started in the group and then passed     
42:36     
away and there are a lot of people who have left the group you know it's not a it's not we have core members but we     
42:41     
have people who come in and out and that's perfectly fine now I'm going to put a slide in here unlike our     
42:46     
contributors over the years it's going to be pretty hacked with names um and so     
42:52     
the next 10 years you know we have a number of topics we'd like to explore and so I'm going to maybe add a few more     
42:59     
things to this or you know modify it but that's basically the talk you know I     
43:04     
went over a lot of stuff there and so you know that's I think I'm going to do that long version of this on YouTube and     
43:10     
and release a video on this so we can get a handle on where we've been in our ten     
43:17     
years so any comments or     
43:23     
questions I don't know any I know dick wanted to go over     
43:29     
these to hear oh was that     
43:37     
Susan I'll just I'm popping in and out because it keeps freezing oh okay I'll     
43:42     
just be stubborn and keep coming back in okay well anyways yeah I think that'll     
43:49     
be the way I'll present it I don't know if I missed anything you know obviously there's just so much to go over it's but     
43:56     
I think that's good you know and then like you know I hope that people can get     
44:01     
you know say Okay I I I identify with that or that's something I want to do or     
44:07     
this is something that I can't believe we did so it's always good     
44:13     
stuff okay yeah so we had this whole muting thing up now Jess in the chat     
44:19     
here so I'm going through the chat U so Michael asked Susan about her uh project     
44:25     
he said what about a forc directed graph implementation and then Susan said I'm looking at the mechanical properties of     
44:31     
a model of epithelial tissue so and just you know uh that so Susan's been working     
44:37     
on this model of something called tensegrity which is like super stability     
44:42     
and so she's working on like cells and in epithelial sheets so it's not really something you see in C elegans as much     
44:48     
but the idea is that you have these cell populations they're hyper stable under     
44:54     
uh forces during development and that that sort of thing so yeah there are a lot of methods you could try on that the     
45:01     
there's a literature on modeling Tegrity but it's like people are approaching it kind     
45:06     
of you know they do a lot of things at matrices and uh accounting for different     
45:13     
forces and it's not as advanced as you would like     
45:18     
actually it's um there's even they even get into crystallography to describe the     
45:25     
where the model is yeah I like parts of the model yeah and     
45:31     
yes definitely heavily into linear algebra but it's     
45:36     
um deeper into linear algebra than my um     
45:43     
adviser he's an electrical engineering Prof and known for his linear algebra     
45:48     
yeah expertise yeah so it goes beyond something um and it's gets in to     
45:58     
mechanics and I'll eventually sort some of it out     
46:03     
but I've got a couple of books and a nice paper or two on the mathematics of     
46:09     
tensegrity and I can only read so far into them like I'm into the third     
46:15     
chapter in the book and I'm starting to not understand it and I have part two of     
46:23     
a series of papers and I don't understand that although it's giving me nice keywords so I should be able to     
46:29     
look that up if you're interested I can give a presentation on like the math of     
46:36     
T because it's it's beyond me yeah oh     
46:42     
that would be great an engineer it's beyond me I have to study some more yeah     
46:49     
it's like a yeah it's not obvious what they're doing and I mean they could use better models I think definitely it's     
46:55     
just kind of like this is a network and this is the these are the     
47:01     
forces um I'll uh I'll post this paper the two-part     
47:10     
paper the first part was excellent and it helped me out I can actually end my     
47:18     
study of this as a master's degree using just part one but part it's part two     
47:25     
like I said that's giving me issue so I'll try and type that in there and hope that my computer decides to     
47:32     
work yeah I'll put it in the chat okay yeah     
47:37     
so the uh Jess said plus one for updating the course yeah this is something that I think Jess was involved     
47:44     
in back then Deva worm ml that might be something I'm interested in at the educational level and might also be a     
47:51     
good Avenue to have JT trainer experience as well yeah we have a an a live     
47:57     
group uh oraga a lab and we have people interested in like educational stuff     
48:02     
there and then yeah and then this whole thing and yeah so that's that's     
48:09     
good uh yeah so I'm going to finish up here with a couple of resources on     
48:14     
computational modeling there's a nice uh paper that came out on modeling and yeah     
48:20     
there's the dynamic stability of Tegrity structures so there two papers there so     
48:25     
there's this new plat form that's been released um and it's on modeling cell po     
48:31     
or cell it's kind of like what you're trying to do intens segr it's like modeling tissues as a series of cells     
48:38     
and their physics and then uh some work with large language models in biology     
48:44     
which we've talked about in the last a number of meetings uh where they're using large language models to build     
48:50     
models of the genome and they're like analyzing you know genes in terms of     
48:55     
their structure and generating genes Gene sequences and things like     
49:01     
that so let me see if I can pull this up here there we go part one and part     
49:08     
two uh so this is uh biomodeling with 3D and large language model so the first     
49:14     
thing here is zimu cell 3D so this is from nature computational     
49:19     
science and this is uh three-dimensional simulation of tissue mechanics with cell     
49:25     
polarization so this is is actually very relevant to uh differentiation waves and     
49:31     
we've been talking about this recently because we're writing a paper on this where you have you know differentiation     
49:38     
waves or waves that go through tissues and they trigger     
49:43     
uh expression changes in the cells that allow them to change their fate so if you're you know you have a bunch of     
49:50     
cells that are going to become muscle cells there's a differentiation wave and then there's some activity in the cell     
49:56     
the cell picks up the signal and then it changes its shape and then it becomes     
50:02     
the new cell and it changes its shape in different ways so you have to model the the shape shape of the cell and the     
50:08     
changes in that shape and some of the sort of the mechanics that go along with that so in this case what they're doing     
50:15     
is they have this three-dimensional platform where they're building models of three-dimensional tissue so they're     
50:22     
very simple you know like 3D models uh they don't include every aspect of the     
50:27     
cell they include enough about like say the cytoskeleton that you can model     
50:32     
these kind of changes so this is the three-dimensional organization of cells determines tissue     
50:39     
function and integrity and changes markedly in development in disease so there are a lot of changes relevant to     
50:46     
development disease but also to uh you know to other things as well just     
50:51     
basically when a organism moves for example or when there's like tissue remodeling or something like that     
50:58     
cell-based simulations have long been used to define the underlying mechanical principles however high computational     
51:05     
costs of so far limited simulations to either simplified cell geometries or small tissue patches so we can't really     
51:12     
I mean we can model organisms but there's a tradeoff between the number of     
51:17     
cells we're modeling and the detail we can have and so this is a problem that     
51:24     
is maybe overcome by better modeling Tech techniques may be overcome by bigger computers but it's a it's a     
51:31     
tradeoff so we present simell 3D an efficient open source program to     
51:37     
simulate large tissues in three dimensions with subcellular resolution so they model things within the     
51:43     
cells growth proliferation extracellular Matrix fluid cavities nuclei and     
51:50     
non-uniform mechanical properties so this is like we're talking about with uh     
51:55     
t seg only that in this case it's like you might experience a force in one part     
52:00     
in one cell and in another cell you don't and then this maybe propagates     
52:06     
across the cells but it doesn't it's not like all the cells experience the same thing at     
52:11     
once so you find this in polarized epithelia so we have these epithelial sheets we can model which is a just a     
52:18     
layer of cells these epithelial cells and they go on to form like skin and other things like that um so you can get     
52:25     
all sorts of weird shapes as we know from some of our meetings uh spheroids     
52:31     
vesicles sheets tubes and other tissue geometry so we can find these in     
52:37     
microscopy images we can explore them but modeling them you know it's either an exercise and     
52:43     
visualization or you know we can model the the sort of the physics as well but it's hard to do it at a h at a detail     
52:50     
you know in a high resolution so that's what they're trying to do here um so they want to really incorporate     
52:57     
biomechanics into the visualization and do this with respect to the D underlying     
53:03     
data doing so we show that 3D cell shapes and layered and pseudostratified epithelia are largely governed by a     
53:11     
competition between surface tension and intercellular adhesion so it's like     
53:16     
where you have adhesion between the cells whether they stick together or not and then also the surface tension so     
53:23     
they're pushing against one another uh they're creating tension there's the surface which is the cells kind of     
53:30     
interlocked and producing this tension force but also there's adhesion so     
53:35     
they're adhering to one another and there's you know there's they're packed in but they're also and they stick     
53:42     
together but they also have to move and we've seen examples of this where they kind of move across you know one another     
53:50     
they kind of break selectively break that uh adhesion where they come     
53:56     
together and adhere under conditions that we this is why we're doing this modeling because we don't really     
54:01     
understand the full reasons why that happens and so this is something that we     
54:07     
can you know this allows us to do this kind of large scale modeling now this is a good uh figure     
54:12     
one here is a good example of what they're trying to do so we have these large tissues which is a collection of     
54:19     
cells they exist in these sheets so they might exist in like these little tiny     
54:24     
three uh neighbor you know uh clusters they might exist in larger clusters and then they might     
54:31     
exist in these sheets so they have all these different it's heterogenity of things sometimes those can be deformed     
54:38     
and there forces involved in those deformations so they expand they contract they kind of change shape     
54:44     
depending on the forces they're experiencing the membranes are polarized     
54:49     
so there's an apical end and the Basil end the Basil end is usually attached to some floor the apical part usually     
54:56     
deforms upward so you have these kind of little hats that they wear I guess and     
55:02     
so there there's a lot of shape change with respect to their other polarized uh then there are things     
55:09     
within the cells that we want to know and we want to model and have high geometric resolution because cells are     
55:15     
not circles we I showed you the centroids but that's not what a cell is     
55:21     
a cell is this complex geometry and they pack together in a way that's m     
55:26     
mathematically interesting so you can do this with these kind of meshes that they have here     
55:32     
so I don't think this is fine an element analysis per se but it's this mesh that you can build to the cell the cell     
55:38     
consists of a bunch of triangles the triangles then are independent they change their sort of orientation and and     
55:45     
size and they build these meshes that can then be manipulated you can model     
55:50     
different sorts of things like surface tension membrane elasticity pressure differences and bending elasticity so     
55:57     
they're all these governing equations that allow you to do that you can also have cell division where you have one     
56:04     
cell which is a Mother cell you can have a division event where the cell pulls apart and creates two daughter cells so     
56:11     
you can model like division of cells and you know there's this whole process within it where DNA is being segregated     
56:18     
into the different holes and then apart and you get these two independent cells     
56:23     
all they're doing here is they're modeling the geometry of the cell like they're able to just kind of pull the     
56:30     
sphere apart into two parts so I mean you know that's that's useful it doesn't describe the whole process but it's     
56:37     
useful and so they show the computation time here there's cell Sim 3D which is     
56:43     
another platform semi cell 3D and IAS so I guess these are different platforms     
56:48     
this is not uh uh compuse cell 3D they don't show that and they know that like     
56:54     
Morgan is interested in comp cell 3D we've done stuff with copy cell 3D in the past but that's that's a different     
57:01     
type of model I think it's and I don't know if that's what we call a cellular pots model so I don't think that's what     
57:07     
they're doing here so but it's a it's a a similar approach and then finally we have this     
57:15     
um uh these different types of structures that we can model we have a     
57:20     
vesicle with a lumen we have a spheroid with an extra cellular matrix around it     
57:26     
in the Deep Red we have different nuclei within the cell so we can model that in     
57:31     
a sheet of cells so this is a an epithelial sheet with a nuclei inside so     
57:37     
you can model like a sheet and its Dynamics but also what's going on inside the cell and then you have these tubes     
57:43     
which you see with neural tube and development basically where Shea cells rolls up and the ends join together and     
57:50     
form this tube and a lot of times that that process is dynamic so you might want to know for example     
57:56     
how you go from a sheet to a tube and you want to model that continuously you don't really want to model the end     
58:02     
points because that's not the interesting part of that the interesting part of it is how it kind of moves     
58:08     
together and what are the physics so this is a nice approach now they have a     
58:14     
get or they have a get lab repository so this is the open- source aspect of this     
58:19     
paper this is uh run out of the run I guess runer simio cell 3D I don't know     
58:26     
what lab this is they're out of uh uh Zurich I think and so they're doing this     
58:32     
this is their repo uh I think you can like clone it and run the model I think     
58:37     
on your computer they have a Docker file so they have a they run it on Docker uh the read me here is basically     
58:45     
showing this this is a C++ cellbase simulation framework you can install it     
58:51     
on Linux or Windows you know just basically uh follow the     
58:57     
instructions uh you can do I guess you can run it in Docker if you would like or you can launch a     
59:03     
simulation um and yeah and it basically creates a number of output files that we     
59:10     
can then check and you know use that in in the way that you want to use it so it     
59:15     
gives you uh I think some visualization files a CSV file where it gives you     
59:21     
simulation statistics Excel area and volume so this is something that we create with um in in D learn from like     
59:30     
raw microscopy data but you can actually run a simulation and then get the same information out of this but this has now     
59:37     
been simulated uh and then of course we have these different meshes that we can do we     
59:43     
can visualize this using par view so it produces these meshes and these output     
59:48     
data in a CSV file you can use piew then to visualize it so I don't think it     
59:53     
visualizes it within the software but this is you know this is good this is the kind of software that we have to     
59:59     
simulate cells and and development so I don't know how useful this would be for     
1:00:04     
people but definitely it's something to look into more if you're interested and we have a bunch of things in the chat     
1:00:11     
here so um yeah so we have the first of all the two uh tensegrity papers from     
1:00:19     
Susan uh hansu said a full-fledged course catered towards computational     
1:00:24     
developmental biology with AIML will have some novelty and sounds interesting that was with respect to the DOR ml um     
1:00:33     
course uh Susan also dropped another link to 10 segurity structures the book     
1:00:39     
which is Jiang and osaki which we actually think ran through because they give some matlb code in that book on 10     
1:00:45     
secrity so that's it's it's largely Matrix algebra so it's not too difficult     
1:00:51     
but of course Matrix algebra is not maybe not the best way to Model T secrity but they give some code in that     
1:00:57     
and then this T1 transitions thing Jess has a hashtag cells are not     
1:01:02     
circles uh Susan gives another citation here how Dynamic pre-stress governs the     
1:01:08     
shape of living systems from the subcellular to tushu scale that's from the Royal Society this is yeah this this     
1:01:15     
paper is I guess a paper on physics of fgr and why we're interested in it     
1:01:22     
because when we see these models in 3D we're modeling the physics sort of of     
1:01:28     
like as they might be encountered in a process like you know we can Model A     
1:01:34     
process and we can have a certain efficiency but that doesn't necessarily reflect what's going on in the biology     
1:01:39     
there's a lot going on in the biology that you know if you have say like a pre-stress where you have a certain     
1:01:45     
structure it can give you uh stability longterm or you know it can have certain     
1:01:50     
properties that we're not really modeling so yeah this is all good stuff     
1:01:57     
so again yeah there are a lot there are a number of platforms for 3D modeling of cells and cell populations and this is     
1:02:03     
something that again is a long-term interest of the group and I should also put that in the in the 10th anniversary     
1:02:10     
talk as well because I I I didn't think about like you know bringing that back     
1:02:15     
up because we haven't really done anything with copy cell 3D so much as we'd like H so but that's interesting we     
1:02:22     
should talk we should revisit them okay let me oh there's Hussein no     
1:02:31     
Hussein let me share my or I sharing my screen so I'm going to go over to this     
1:02:36     
other paper and this is um so we've talked about large language models and     
1:02:42     
modeling genomes before and it's a really interesting area because we have a lot of genome sequences full genome     
1:02:50     
sequences now where we can just take the the sequence data and we can analyze it     
1:02:56     
and find different things in it there annotations in it now so people know kind of what the coding regions are what     
1:03:03     
they maybe do we're not you know our annotation isn't great but we what we     
1:03:08     
can also do is plug it into a large language model because genomes have kind     
1:03:13     
of a structure like natural language and so we can do those kinds of things um     
1:03:18     
what they're saying in this paper is they're talking about Transformers which are a typical Technique we might use to     
1:03:24     
analyze something like you know microscopy data or we might model     
1:03:30     
genomes with Transformers to find patterns and in classification but we also use large language models and so     
1:03:37     
what they're saying here is that this is beyond Transformers to Transformers and Beyond large language models for the     
1:03:45     
genome so uh you can use Transformers and large language models but there's this large language model aspect that's     
1:03:53     
interesting so this abstract reads uh in the rapidly evolving landscape of genomics deep learning is emerged as a     
1:04:00     
useful tool for tackling complex computational challenges so this is the stuff we're doing with dvo learn so this     
1:04:06     
is not you know these are techniques that people are using neural networks different types of neural networks     
1:04:12     
Transformers things like that this review focuses on the transformative role of large language models which are     
1:04:19     
mostly based on the Transformer architecture in genomics so large language models are are like a language     
1:04:26     
model where you take a data set of of well in language natural language it's     
1:04:32     
words or phones or you know uh like n     
1:04:38     
engrams and then you uh take those and you train the model on that and it finds the structure and it generates uh     
1:04:45     
potential outputs like if you query it for something it'll put an give you an output it's like if you wanted to find     
1:04:51     
something that had a certain function it would give you an output now now you can     
1:04:56     
use Transformers and large language models as sort of one of the parts of it     
1:05:01     
the other part of course of the language model is the training set which you can use from like a a full Genome of an     
1:05:08     
organism or a set of organisms or individuals from an organism and so this this gives you uh you know a lot of     
1:05:15     
power potentially to do bioinformatics type stuff building on the foundation of     
1:05:21     
traditional convolutional neural networks and recurrent neural networks so this is CNN and rnns and and you know     
1:05:28     
this is very found traditional stuff now we explore both the strengths and limitations of the Transformer and other     
1:05:35     
large language model types for genomics so some uh large language models use     
1:05:41     
Transformers other large language models use other kinds of networks and the question is what's the best technique to     
1:05:48     
use for genomic data because it's not natural language but we know that these models can deal with like sequences of     
1:05:55     
symbols and you know they can put them together in in a way that's you know makes sense     
1:06:00     
it's there's a sort of uh I guess you could say a semantics but probably more of a syntactics of the genome that we're     
1:06:08     
interested in uncover um additionally we contemplate     
1:06:13     
the future of genomic modeling beyond the Transformer architecture based on current trends and research the paper     
1:06:20     
aens has serve as a guide to for computational biologists and computer scientists interested in learning     
1:06:25     
language models for genomic data we hope the paper can also serve as an educational introduction for     
1:06:31     
biologists and so this kind of goes through uh some of the ways in which we can use large language models um you     
1:06:40     
know and like some of the types of data that we can get so we can get things like chromatin accessibility information     
1:06:47     
methylation information transcriptional status chromatin structure bound     
1:06:53     
molecules so those are all types of data that are sort of epigenetic or you know     
1:06:58     
like you have a gene sequence how is it expressed there are different properties that the gene sequence there's different     
1:07:04     
properties of the structural biology that interact with that how can we uncover like you know incorporate those     
1:07:11     
data and then uncover the the relationships there so this this talks     
1:07:16     
about the promise of large language models combining them with other types of models um and getting you know some     
1:07:22     
of the utility of that um you know with train Transformers uh applying Transformers to     
1:07:29     
large language models and especially genomic data offer New Concept a novel     
1:07:34     
conceptual framework we can use the attention mechanism which is something in Transformers you know we talk about     
1:07:40     
attention as all you need and all of that that's part of that whole framework     
1:07:46     
you can use this attention mechanism which is basically where you identify certain things in the data and ignore     
1:07:53     
other things and that's that helps you s of find the structure you're looking for     
1:07:58     
we can use attention to study the organization and grammar of the genome so we can use things like that you can     
1:08:05     
use other types of approaches in machine learning as well and we want to have larger models with large High predictive     
1:08:12     
power and so that's you know there's a whole literature on that but it's it's     
1:08:18     
worth considering you know how these things kind of you know operate and and you     
1:08:24     
know finding the best Solutions and so you know but also we have a lot of emerging techniques coming     
1:08:30     
out so you know the Transformers have been wildly successful um but there's     
1:08:36     
probably some sort of next Transformer that might come out something better to replace and to use in large language     
1:08:42     
models that's even better and so this is you know kind of a call for being on The Cutting Edge to say you know we have     
1:08:49     
these techniques that people probably aren't familiar with there's this very uh good technique that a lot of people     
1:08:56     
maybe aren't using right now and then their up and coming models that we should be on the lookout for um so you     
1:09:04     
know large language models what we're basically doing is pre-training our models on sequences of biological tokens     
1:09:12     
such as DNA sequences what they call cers which are sequences of a certain size or Gene identifiers which are maybe     
1:09:19     
labels of what certain sequences or genes do or what their extent is so like     
1:09:26     
in bioinformatics you often have like you know a a start site and an end site for a gene we can use that to train the     
1:09:33     
model we adopt the term large language model traditional we used in uh natural     
1:09:38     
language programming due to the pre-training approach analogous for the text data so we're building analogies     
1:09:45     
with like natural language and we're using this in uh you know molecular     
1:09:50     
biology now you know that logic is is good and it's largely correct but we     
1:09:56     
have to be careful because uh natural language is a lot of properties that your uh genome does not in terms of the     
1:10:03     
sequence the structure of the sequence and so forth so this is a caveat that you know is something that they kind of     
1:10:10     
talk about here but so they talk about in this paper they talk about an overview of the Transformer     
1:10:16     
architecture they talk about this hyena model now we've talked about the hyena model in the past this is a model that's     
1:10:23     
been released specifically Al for analyzing DNA and genomes this is an     
1:10:28     
implementation of a large language model and again it's it's specialized trained     
1:10:34     
on on molecular data not on natural language so it's a you know this is     
1:10:40     
something that I don't have the link right now but if you look up DNA and hyena or hyena llm you you'll find the     
1:10:47     
link to that that software I don't know if it's open source I can't remember how     
1:10:53     
much of it's open source but it's something that is uh definitely uh being developed and you     
1:11:00     
know it's it's like a very fast moving area uh you know they they then they talk about different deep learning     
1:11:07     
architectures that people have used in bioinformatics there have been some novel architectures they talk     
1:11:14     
specifically about the Transformer Hybrid models which is where you use a Transformer and the attention mechanism     
1:11:21     
to predict assay data and then applying transform based large language models     
1:11:27     
and other types of large language models to the genome now I'm not an expert on     
1:11:32     
large language models but I noticed that like if you go to hugging face there are a lot of different open- Source models     
1:11:39     
and those models sometimes are General but sometimes they're trained on specific data sets the techniques that     
1:11:46     
they use are specialized to analyze specific types of data and so it's it's     
1:11:51     
you know it's interesting how this area is kind of exploded in like the last year it in like you know what can be     
1:11:59     
done how people are doing it so there's a lot here that you know is is kind of     
1:12:05     
out there on The Cutting Edge so you know it's something that at this point you know just simply kind of reviewing     
1:12:12     
the different types of techniques that people are using and evaluating them is probably useful um and I don't have any     
1:12:19     
clue as to how much computational uh what kind of computational resources you'll need     
1:12:25     
I know you can run some large language models on like a Max silicon but you     
1:12:30     
know you probably will need some sort of like you know AWS time for to run them     
1:12:36     
well so yeah they give like a nice glossery of terms here if you're interested in applying large language     
1:12:43     
models and what you know what how Transformers work and all this they have all these different terms that are used     
1:12:50     
um and it's yeah it's it's it's quite involved um but this is basically the     
1:12:56     
approach they're using so yeah it's a nice little paper again revisiting large language models and applicability to     
1:13:03     
like genome which isn't cell modeling but we have to be able to find patterns     
1:13:09     
in the genome to say something about how it's related to proteins and the     
1:13:19     
phenotype so yeah that's that's all I have on that I don't know if people had anything any any comments on that paper     
1:13:26     
on the other paper as     
1:13:34     
well yeah really really interesting I'm gonna follow     
1:13:40     
up taking kids to school here but um uh yeah just just wanted to add one thing     
1:13:47     
that U the net Pine uh by monthly meeting is this week it's the yeah the     
1:13:54     
18 so Thursday yeah um if anybody wants to     
1:14:01     
join and U uh I'm definitely gonna be bringing up these kinds of you know     
1:14:09     
additional developmental um yeah like basically     
1:14:14     
kind of organoid you know who's who's interested in organoid modeling and who's potentially already working     
1:14:21     
on yeah I I can post in the SL     
1:14:27     
details yeah yeah this is of course open source for uh net Piney so it's like the or net     
1:14:35     
Pine which is a python uh package and not I can't     
1:14:41     
remember what it's what what it does what so so um it's kind of like a bill     
1:14:48     
lton um and and others I think he's passed the torch now to U     
1:14:56     
I forget it's name now sorry it's it's a a blanking on it t in the car but     
1:15:05     
uh it's a uh kind of an extension of     
1:15:10     
neuron to to handle larger multicellular     
1:15:16     
models or like larger you know area models um and yeah so it started out as     
1:15:26     
some simple scripts to kind of you know take a take a Neuron model and then you know kind of duplicate it yeah uh and     
1:15:35     
but you know it's it's U it's definitely grown over the years to to be kind of     
1:15:43     
its own its own package now yeah or you know kind of     
1:15:50     
own have its own Focus but yeah but these these you very very well aligned     
1:15:56     
with say organoid modeling for instance yeah yeah because you're modeling cell     
1:16:01     
without the yeah well you know you're modeling like populations of cells     
1:16:09     
right and and yeah what comes out of that Dynamics like the it it strikes me     
1:16:16     
as similar to the kind of stuff that Stephanie Jones is doing a brown with     
1:16:21     
the human neocortical solver     
1:16:27     
um there should be another I know it's hnn     
1:16:33     
brown. um and uh but again like trying to model     
1:16:39     
populations of neurons as opposed to you know each each model being essentially a     
1:16:46     
cell yeah oh okay yeah so it's not just the cell isn't the unit it's the population activity right right right     
1:16:54     
yeah yeah start starting to think at that you know I mean you know Stephanie Jones's     
1:17:01     
case it's like it was very much like trying to model you know um     
1:17:09     
egm population scale right yeah net net Pine was definitely more you know lower     
1:17:17     
level or you know lower smaller populations um and     
1:17:25     
very like intercortical and other scenarios but I I can when I     
1:17:33     
when I get home I can drop some links and slack of okay that sounds good as     
1:17:39     
well as the the med details but yeah and looking forward to looking at     
1:17:44     
those those cellular papers they sound like variant of kind of     
1:17:50     
morous yeah and it with those kind of models it's like imperative that they work like you know we've talked about     
1:17:57     
some of these uh modeling software that like either very poorly maintained with     
1:18:03     
respect to 2024 or that they're very opaque and you don't know how to set up a model that's     
1:18:09     
always a problem because yeah it's and the best way the best kind of software     
1:18:14     
for that is where it's very much like you can set up a model maybe even visualized set the parameters it's very     
1:18:21     
clear what you're doing when you make manipulations to the model because that's what you basically are interested     
1:18:26     
in and you know knowing what those outputs are going to do and mean and you     
1:18:32     
know so this this can be a problem with padon     
1:18:39     
solware yeah okay well that sounds great uh     
1:18:45     
thank you for attending everyone uh any other comments or questions before we go     
1:18:51     
I would say there's a problem with some commercial software to oh yeah of     
1:18:56     
course yes true true I've been dealing with console and     
1:19:03     
um for some reason when you get it through the academic institutions there's no help desk so I've been using     
1:19:10     
chat GPT as my help desk for console so than this to say just about     
1:19:18     
had enough yeah yeah you know you know I think a lot of the excitement around chat GPT is companies excited that they     
1:19:25     
don't have to do support oh maybe that's it yes definitely     
1:19:32     
yeah all right well thank you for attending and we'll be back next week um     
1:19:39     
and know we're we're in the slack as well so if you want to join the slack and comment on things um be be be glad     
1:19:47     
to hear we're coming up on our decisions for gck so you know that's coming up in     
1:19:53     
the next few weeks so beginning of May we'll make announcements as to who we're     
1:19:58     
having this summer for Google summer of code uh we're still working on selection     
1:20:04     
and finding slots so hopefully the people will be uh satisfied with that     
1:20:10     
and if you don't get selected for Google summer of code you can still work on a project this summer because we're that's     
1:20:16     
what we do in the group um you know we can maybe work on the project if you want to find funding somewhere else or     
1:20:23     
if you want to apply again next year those are all things that we can do so don't feel bad if you don't get selected     
1:20:29     
it's it's it just really depends on what how many slots we can get pretty     
1:20:38     
much okay thank you have a good week bye     
1:20:43     
bye okay so I'm gonna talk about a topic that is close to what we were talking     
1:20:48     
about with sem cell 3D and that is talking about cellular     
1:20:54     
she sheets and tissues as bi soft materials the soft materials that have     
1:21:01     
symmetry they have some geometry they have a set of properties physical properties but     
1:21:08     
also you know their own Dynamics and interactions and also uh signaling from     
1:21:16     
molecular mechanisms because remember their cells and they're not like tilings     
1:21:21     
or other types of generic geometric objects their cells with a     
1:21:27     
biology so why don't we get started here and I had three papers in this collection so the first one is this     
1:21:35     
paper here from physical review letters minimal model of cellular symmetry     
1:21:42     
breaking these are people from the max plun Institute and so the abstract of this     
1:21:50     
paper reads the cell cortex a thin film of AC material assembled below the cell     
1:21:57     
membrane so this is a a specific structure within each cell called the cell     
1:22:02     
cortex it's a thin film of active material so when we say active material we mean that it responds to different     
1:22:10     
physical stimuli and it responds in a way that is dynamic so we'll talk about     
1:22:17     
the cell cortex in a little bit we talked about the uh cell body and how     
1:22:23     
it's not round it has these you know it can conform to different kinds of shapes and and so the cell cortex is part of     
1:22:30     
that the cell cortex plays a key role in cell cellular symmetry breaking     
1:22:35     
processes such as cell polarity establishment and cell division so when     
1:22:40     
we establish the polarity of a cell we talked about the basil and the apical end or you know could be an anterior     
1:22:48     
posterior orientation those are all aspects of cell polarity and then cell of course we     
1:22:55     
talked about how you could model cell division as this process of a sphere     
1:23:00     
being pinched in the middle ping up each end apart is you know with a cell a     
1:23:07     
sphere that develops Two Poles develops this asymmetry and then breaks apart     
1:23:12     
into two cells and so there's a whole mole Suite of molecular Transformations that occur but if we want to just model     
1:23:19     
the cell itself uh then we have this model of symmetry break so let me show     
1:23:27     
you what we're talking about here so remember recall from our previous discussion on modeling cells if we have     
1:23:35     
a cell here and we'll model it as a sphere unfortunately because we talked about     
1:23:42     
how cells are not spheres but for the sake of argument here we'll do a     
1:23:47     
spherical modeling so this is a cell sphere this outside is the membrane     
1:23:56     
and we have the cortex underneath which is a secondary     
1:24:04     
layer and then you have the uh inside of the cell nucleus and the DNA and the nucleus     
1:24:13     
and of course you have the inside of the cell the cytoplasm and organel and     
1:24:21     
things like that so that's what we're do now now a cell can have if it's arranged     
1:24:27     
in in a sheet like we saw with the epithelial cells you have two poles here     
1:24:35     
we have an apical end and a Basil end and the Basil end is usually anchored     
1:24:41     
with some substrate and this is very important in physics because what happens is say I introduce a force to     
1:24:48     
the cell the left and it can bend the cell with respect to the     
1:24:55     
substrate it can also attach it from the substrate but in biology usually we have     
1:25:00     
some substrate and that's the basil portion of the cell body and then the apical portion can deform as well so     
1:25:07     
sometimes you can get a peak or some other type of deformation in that direction     
1:25:29     
now when we're talking about cell division we're usually talking about the spherical     
1:25:35     
structure gets constricted along the central     
1:25:42     
axis and that Central axis then serves to pinch the sphere together so if you     
1:25:48     
go to the next phase here you get something like this where it's being     
1:25:53     
pinched in [Music] Center and you start to get segregation     
1:25:59     
of Chromatin the segregation of the components of the cytoplasm and of     
1:26:07     
course the cortex has to deform to conform to that sort of activity these     
1:26:13     
Dynamics ultimately this becomes two cells each with the room cortex     
1:26:20     
cytoplasmic contents and nucle     
1:26:26     
and so they move apart accordingly these are the daughter cells that come from this initial cell division event here so     
1:26:33     
here are two daughter cells in     
1:26:40     
context so those are the kinds of asymmetries and symmetries we're talking about there are these asymmetrical     
1:26:46     
processes that can happen during so when a cell is anchored to a substrate there     
1:26:52     
is symmetries that can happen when a cell is dividing things can segregate in completely or you can have daughter     
1:27:01     
cells that are larger than other you know one daughter that's other than another or you can have other     
1:27:06     
asymmetries that result from a cell not being SPH here we present a minimal model the     
1:27:14     
self organization of the cell cortex that is based on a hydrodynamic theory of curved active surfaces now we talked     
1:27:21     
about a a physical Theory uh a couple weeks ago it was live streaming or something like that and it sounds like     
1:27:28     
it's it's it's some internet technology but it's actually uh biophysics     
1:27:33     
technique actually it's not biophysics so much it's it's Material Science but this idea that you have the sort of     
1:27:41     
active material that they Dynamics and that it's involved in liquid in in being     
1:27:46     
transported through liquids and in indeed uh cells are very liquid you know     
1:27:51     
we have to use Liquid theories to describe some of these Dynamics because cells are immersed in liquids we're     
1:27:58     
dealing with liquids and we're dealing with the cytoplasm we're dealing with liquids when we're doing with cells     
1:28:04     
vibrating and things like that so hydrodynamics is the appropriate way to view that and we're talking about curved     
1:28:10     
active surfaces again we talked about curvatures uh previously in the group we     
1:28:16     
talked about sort of how membranes are curved and so a lot of things happen on these curves if you look back at our     
1:28:22     
cells we have the sphere and of course I said cells are not spherical but cells often have a curvature of some type even     
1:28:29     
something like this cell here which is the epithelial cell it's sort of like u     
1:28:36     
a rectangle that's upright but of course it has curvature and I'm just idealizing     
1:28:42     
here but there are a lot of curvatures that happen both on the surface of the     
1:28:48     
membrane uh but also within the cytoplasm in terms of migration in terms     
1:28:54     
of organal structure in terms of sort of like you know navigating a liquid and so     
1:29:00     
there are a lot of things that have to navigate liquids within the cell and between     
1:29:08     
cells so active stresses on the surface are regulated by a diffusing molecular     
1:29:14     
species so you can build these models that are hydrodynamic in nature that involve curved active surfaces but also     
1:29:22     
are minimal models so the minimal model is simply a model where you have a cell     
1:29:28     
and that cell is has components as many components as you want to specify and you know if you have the     
1:29:35     
right conditions at the right framework then it should give you an answer that's acceptable it's not going to model     
1:29:41     
everything but it'll model to give a good answer to the question that you're     
1:29:48     
asking so the active stresses are regulated by diffusing molecular species and so a lot     
1:29:55     
of times in cells you have these active stresses they're mediated by things and     
1:30:02     
this is what we're trying to we show the coupling of the active surface to a passive bulk fluid so we     
1:30:09     
have this passive component and this active component the active surface is     
1:30:14     
uh the surface of the cell this curved active surface and then we have the passive bulk fluid so we have the     
1:30:20     
surface and we have the fluid dynamics the fluid d being the environment which is passive the surface the substrate     
1:30:28     
being the active thing enables spontanous polarization     
1:30:33     
and the formation of a contractile ring on the surface biome mechanochemical     
1:30:38     
instabilities now we've talked about contractile Rings both in terms of cell division so this is a contractile ring     
1:30:46     
and indeed if we look at some of the models of cell division in bacteria they have these contractile rings and we've     
1:30:52     
talked about how uh genes like fit Z were invol are involved in the sort of form formation of the ring so you know     
1:30:59     
you have molecular models of this but you also have more generic models so you can actually have this as a product of     
1:31:06     
physiochemical interactions and not necessarily specific gene expression     
1:31:11     
although in biology we do often see these genes reg U regulated in times     
1:31:16     
where you get these contract a rings that you need to have cell division and other things like that so you need to be     
1:31:23     
so these contract out Rings they will induce curvatures like in this case where     
1:31:29     
you're pinching this uh the membrane cortex of the cell and those are all     
1:31:35     
mediated by molecular mechanisms but in the minimal model we we don't worry about the specific Gene we just worry     
1:31:42     
about the process we discussed the role of external fields and guiding such pattern     
1:31:49     
formation our workor reveals that key features of cellular symmetry breaking and cell division can emerge into a     
1:31:55     
minimal model by a General Dynamic and stability so again they're just showing that this minimal model can replicate     
1:32:03     
the biology you're not interested so much in the specifics specific species specific     
1:32:09     
genes all they're worried about is the is the mechanics the     
1:32:17     
mechanical the mechanochemical environment or the mechanochemical driver     
1:32:24     
and so this is a great model uh for looking at these things very generically     
1:32:30     
you know maybe being able to derive some insights from this because it's a very     
1:32:36     
hard thing to model at a very specific level because you need a lot of experimental information you need a lot     
1:32:41     
of data but even when you do have those data sometimes it's very hard to really     
1:32:46     
kind of figure out what's going on because there's so much variation across species and across different types of uh     
1:32:53     
cont track out behaviors the other comment I'd like to     
1:32:59     
make is that you know this is very similar to a lot of the morphogenetic models that we talked about so during     
1:33:05     
morphogenesis of course is a reaction diffusion model that has two components a reaction component in the diffusion     
1:33:13     
and in fact here we also have a reaction component in the diffusion we have a diffusion component     
1:33:19     
that's regulating active stresses and we have a reaction component where the ACT where we have this curved active surface     
1:33:27     
that where uh the changes occur so we have this sort of two     
1:33:32     
compartment model as well so even when they have the minimal model we have this basic     
1:33:38     
structure so they talk about the cortex of animal cells being this dynamically crossling polymer Network it's located     
1:33:45     
between beneath the cell membrane and as we know from some of the work we've done on differentiation waves you know that     
1:33:52     
polymer network is very important differentiation so when that polymer network is deformed or there's you know     
1:33:59     
a need to sort of differentiate into a certain cell type that polymer Network     
1:34:04     
needs to be modified and so there has to be a set of mechanisms that do that now what's     
1:34:10     
interesting of course is that the signal to differentiate could come from the environment but it could also most     
1:34:15     
likely come from genes and proteins being expressed that are very specific to the cell type so we know that     
1:34:22     
depending on the destination cell during differentiation the cell has to you know make different     
1:34:31     
changes multitude of changes to get to its destination and so if you're say for     
1:34:37     
example turning the cell into a neuron versus turning the cell into a muscle cell or turning it into some sort of     
1:34:44     
like you know skin cell that's a different geometry the you know the polymer     
1:34:50     
networks have to contract and they also have to conform to different types of     
1:34:56     
shapes and so there's a lot there are a lot of interesting questions here it's not just about pattern formation it's     
1:35:01     
about triggering those changes and triggering them in such a way so that you can get characteristic     
1:35:10     
structures so the comment here is that the cortex is interesting because it you     
1:35:17     
know the cortical flows and cell shape changes are connected the cortex has to interact with the material that     
1:35:23     
surrounds it so you know on one side you have the cell membrane on the inside of     
1:35:29     
the cell and in contact with the cytoplasm you have a crowded viscous fluid so you have this viscous fluid     
1:35:34     
it's kind of like jelly it's crowded with things it's not like just a typical     
1:35:41     
homogeneous material it's not a typical homogeneous fluid it's got a lot of aspects so it's viscous so it doesn't     
1:35:48     
behave like a winar fluid but it also is crowded with other things in it so it has a lot of turbulent     
1:35:54     
we can use you know turbulence if even at very small scales turbulence can play a role in     
1:36:00     
morphogenesis it's much different than say a laminer flow and we know that from     
1:36:07     
uh from microfluidic chambers and how liquids behave in     
1:36:15     
those by manipulating the cytoplasm mechanically has been shown that cytoplasmic flows can directly affect     
1:36:21     
the Dynamics of the cortex and the distribution of proteins within it so     
1:36:26     
this is where we know that like proteins are kind of manipulated by flows and that that's that can play a role in     
1:36:34     
differentiation the rever so this is one scenario now the reverse scenario in which active fortical flows so the     
1:36:40     
cytoplasm fluid in motion has also been observed so we've got we have the cytoplasmic fluid cortical flows meaning     
1:36:48     
the activity of the cortex can push setop plasmic fluids into motion as they're Contracting along with the     
1:36:54     
membrane and these contractile rings are acting it can actually serve as a causal     
1:36:59     
mechanism for cytoplastic fluid fluid flows so this is all you know these are     
1:37:04     
things that happened in the cell we've observed them empirically now we have to model them and so you know we like to     
1:37:11     
take uh the material as a whole we don't you know we need to figure out a way to model it so the cell cortex has been     
1:37:19     
successfully described as a fin active fluid foil this is from citation seven     
1:37:24     
so a thin active fluid film is a sort of soft active material that behaves like a     
1:37:30     
fluid but is also many aspects of the cortex's emerging Dynamics can be accounted for     
1:37:37     
by considering its generic mechanochemical organization citation 8     
1:37:43     
the concentration of a diffusible chemical species regulates the amplitude of active stress but also changes     
1:37:49     
dynamically to to infection of the stress regulated by material     
1:37:54     
so we're interested in these diffusible chemical species this is what they were talking about about this diffusion     
1:37:59     
compartment where we have this diffusion component where we have chemical species moving along at some you know in some     
1:38:07     
concentration at some rate and so it's moving through uh and so it actually     
1:38:13     
regulates the active stress on the surface so you know if you have a flow of course is producing a stress to the     
1:38:21     
surrounding fluid or the surround     
1:38:35     
tissue so in spontaneous pattern formation uh such as in active     
1:38:41     
self-organized active fluids these have been studied on fixed domains with and without substrate friction and on     
1:38:48     
deforming surfaces in an environment with a homogeneous pressure so if we're looking at spontaneous pattern formation     
1:38:54     
we can look at this in a number of ways spontaneous pattern formation is again     
1:38:59     
like this tring reaction diffusion we have the diffusion term and the compartment now we need a reaction     
1:39:05     
compartment and that's what we get with some of our you know what's going on with the surface whatever's going on on     
1:39:11     
the surface is the reaction part and with these you know with the substrate we have the     
1:39:17     
diffusion so the diffusing molecular species regulates active tension on Surface     
1:39:23     
and we show that coupling of the surface to the enclosed fluid gives rise to a hydrodynamic screening length that     
1:39:29     
guides mechan chemical instabilities to generate well- defined patterns on the surface these patterns can govern shape     
1:39:35     
changes so you can have these changes in shape and which lead to maybe cell     
1:39:41     
division or something else and of course because our cell is in a sphere we can     
1:39:46     
modify the shape of cell such as having an apical extension we can also do things like     
1:39:53     
form actual patterns and that can be done in the same way so this is a very generic model but it also allows us to     
1:40:00     
model wide variety of symmetry bra okay so that's that paper now I want     
1:40:06     
to get to this other paper called biological hyper crystals and this is in Journal of     
1:40:12     
physics and uh so this is a 20 2023     
1:40:20     
paper and so the abstract reads the no of a biological hyper Crystal can be     
1:40:25     
regarded as a step towards a broader Crystal notion and so we talk about uh     
1:40:31     
soft active materials and this is based on the idea of liquid Crystal biology or     
1:40:36     
liquid Crystal Dynamics uh back from uh Dean who actually proposed this back in     
1:40:43     
the 60s and so this is uh you know when I talk about crystals I'm not talking about you know some sort of woo or you     
1:40:50     
know something that's Fantastical this is actually uh you know something that     
1:40:56     
we use as a metaphor you know that this biological system has these properties     
1:41:02     
that behave like a liquid Crystal type interface so this is uh a top a very     
1:41:08     
specific Topic in physics that they're kind of talking about here and they're diff you know if we're familiar with     
1:41:13     
liquid Crystal TVs we know that you can use this to as a conductor uh but what     
1:41:20     
we're going to talk about here are some aspects of soft active materials in this contribution I     
1:41:27     
consider the geometry of cell patterns and tissues described in terms of fororo tessellations and cut and project     
1:41:34     
techniques so we're talking about um the geometry of cell patterns in voro     
1:41:40     
tessellation so we're modeling it with a voro Tessellation and cuton project which is a you know two different ways     
1:41:46     
to do with modelings in this way we realize that one foro tessellations early used in the     
1:41:53     
description of atomic and molecular building blocks distributions in qc's     
1:41:58     
which I guess are crystals     
1:42:05     
um can be extended to describe the geometry of cell arrangements and tissues of biological     
1:42:10     
interest and two the recourse to higher dimensional spaces can be fruitfully exploited to describe complex order     
1:42:17     
designs in biological systems so they're using this mathematical tool physical tool to     
1:42:24     
describe some of these kind of systems and the pattern formation that occurs     
1:42:31     
within so you know we're kind of in this way we're kind of looking at sort of the boundary between non-living systems of     
1:42:38     
crystals and living systems like biological pattern formation and so we're trying to explain these ordered     
1:42:44     
designs or ordered patterns and complex biological systems and we can do so with     
1:42:49     
this analogy of a crystal and so this this is uh where they start here they're     
1:42:55     
talking about um sort of in in crystals or non- living ordered compounds where     
1:43:01     
periodic order dominates most biological structures exhibit a gallery of hierarchically based structures ringing     
1:43:08     
over several orders of magnitude and size so you know instead of having these sort of periodic structures as you've     
1:43:14     
seen in like different tilings or you might see in different types of inorganic     
1:43:19     
compounds um you see all sorts of different types of patterns formation or biological structures they're     
1:43:25     
hierarchical because we have these hierarchical systems that have a molecular basis that emerge as these     
1:43:33     
sort of interactions between cells and and you know having tissues and things     
1:43:38     
like that so we have this hierarchical organization vertical organization that     
1:43:43     
results in these structures that range in different you know at different magnitudes they also combine so you look     
1:43:51     
at a pattern it's a iotic in the sense that they're there's different order at     
1:43:56     
different spatial scales so thus going from the Mac molecular level to the full-size     
1:44:03     
organism scale we observe the emergence of Novel structural designs at a given scale which are not present at the     
1:44:09     
scales below or above so if you think about the route of cells to tissues to     
1:44:15     
organs to organisms you know that structure isn't self-similar in the it's     
1:44:21     
self-similar but it's not selfsimilar in a direct sense so in other words the     
1:44:26     
organization of cells can be different than the organization of tissues different the organization     
1:44:33     
organs and organisms so an organism looks very different than its component cells it looks very different than its     
1:44:39     
component tissues there are a lot of things that are pattern formation at organismal level that maybe isn't     
1:44:45     
present at the cellular level at the tissue level and that's because we have this hierarchical sort of a periodicity     
1:44:54     
okay so the presence of characteristic scale transformation in complex biological systems reminds the geometry     
1:45:00     
of or reminiscent of the geometry of fractal geometric patterns and say is     
1:45:06     
fractals you know where we have self- similarity we have you know we have these fractional dimensions and things     
1:45:13     
like that uh nevertheless ideal mathematical fractals or homogeneous structures     
1:45:20     
exhibiting scale and variant symmetry it's not generally the case in most biological systems so again when we have     
1:45:26     
a mathematical fractal we have these patterns beautiful patterns uh that     
1:45:33     
occur that are just basically tilings they're not driven by anything with mathematics biological systems we have     
1:45:39     
these different scales of organization and of action so it's not going to look the     
1:45:45     
same okay so soft matter based quazi periodic crystals or qc's display long     
1:45:51     
range orderings of molecular motives through the space which can be properly described in terms of a systematic     
1:45:57     
application of inflation symmetry operations so from a structural Viewpoint qua periodic crystals can be     
1:46:04     
regarded as self-similar molecular Aggregates for the translation symmetry characteristics of periodic crystals is     
1:46:12     
replaced by a scale and variant one recourse to n dimensional crystallography allows one to describe     
1:46:19     
quasi periodic crystals in a ukian space of dimension ion and larger than three     
1:46:24     
large enough for recovering periodicity so we can actually have these spaces     
1:46:29     
that can give us information about this     
1:46:41     
periodicity so the long range quasi periodic order present in qc's can be envisioned as a natural consequence of     
1:46:49     
the incommensurate orientation between the periodic hyper lus in the 3D physical space so this is just where we     
1:46:56     
have this sort of ordering of the space which is the sort of uh sort of inactive     
1:47:03     
aspect of it or this passive aspect of it and the active aspect which is the periodic hyper lus interacting with that     
1:47:10     
physical space thus hyper Crystal would probably be a more appropriate term for condensed matter phases which can be     
1:47:17     
described in terms of projections from a higher dimensional geometric space and so in this cont contribution     
1:47:24     
we illustrate the hyper Crystal notion and the study of the geometrical Arrangements of cells and tissues of     
1:47:29     
biological interest which can be properly described making use of General     
1:47:35     
cut and project techniques originally introduced in the study of atomic and molecular Cas quasi periodic     
1:47:42     
crystals in particular we will describe the significant role of waro patterns     
1:47:47     
for histological features and curv testation stemming from tissue development so voro pattern are     
1:47:53     
interesting because you describe a space in terms of it centroids and then you     
1:47:58     
find the packing around it so you set find the optimal distance from each centroid and between each centroid and     
1:48:05     
then you find the boundaries so it's like a tiling where you have a bunch of centroids and you find the lean distance     
1:48:11     
between the centroids and that changes as you move across the lattice and so     
1:48:17     
you end up with these circles or other shapes so we have uh a which is the     
1:48:23     
simple epithelium tissue and we have B where we have this uh kiling this uh     
1:48:30     
Forno tiling where we have the centroids for each cell and then we find the optimal distance between each centroid     
1:48:38     
and we have these boundaries that form so you have this sort of lattice of different geometric shapes and so you     
1:48:45     
know it's not like a it's not a perfect lattice of things of all the same shape it's just the distance mean between the     
1:48:53     
veins and then C we have this forid di based on the location of Central veins     
1:48:59     
combined with the original amuno stained image so this is where we have this overlapping with the B high and you can     
1:49:05     
see how it represents the pheny so that gives us an idea of waro tilings and how     
1:49:11     
they relate to these quazi crystals and looking at the periodicity now I'd like to finish up     
1:49:17     
with this paper on uh this is from nigarian so this is a uh Marine     
1:49:23     
invertebrate and we're looking at the cnem in particular and we're looking at Target     
1:49:30     
genes for the novel uh BMP signaling module and so this is a little bit     
1:49:36     
different than the other papers because it's looking at this molecular uh level of     
1:49:41     
organization and it's looking at pattern formation with respect to that so let's     
1:49:47     
read the abstract and think about how that relates to the other papers so the abstract reads BM P signaling as a     
1:49:54     
conserved function in patterning the dorsal vental body axis in bolaria so this is our sort of our anatomical     
1:50:02     
symmetry breaking we have this axis uh we have this orientation and the     
1:50:08     
directive access in antho narian so far narian Studies have focused on the rule     
1:50:14     
of different VI U BMP signaling Network components and regulating the pmad 1/5     
1:50:21     
gr and form much l is known about the target genes Downstream BMP signaling so we know sort     
1:50:28     
of the uh different aspects of the main pathway but we don't know what's going     
1:50:34     
on down the street to address this we generated a genome wide list of direct P mad one5     
1:50:40     
Target genes of the anthone the mastell infensus several of which were conserved     
1:50:46     
inop and zenopus so we have these Target genes from this mad complex in narian     
1:50:52     
but we can actually look for ones that are conserved between narian and ropa     
1:50:58     
and narian and zenopus so we can capture larger philogenetic set of relationships our chipseek analysis revealed that many     
1:51:05     
of the regulatory molecules with documented bilaterally symmetric expression and nnat Stella are directly     
1:51:11     
controlled by BMP signal so this is interesting this means that you know if we're looking at symmetric expression     
1:51:17     
and the Symmetry we can learn a lot about regulatory molecules that are involved in information so what we know     
1:51:24     
about BMP signaling is that it's sort of the impetus for pattern formation in     
1:51:29     
different ways in different organisms but we can look at the symmetric expression as a a sort of an indicator     
1:51:34     
of What kinds of pattern formation is going to be symmetric and what kinds of maybe topological defects we might     
1:51:41     
expect and so we can base this all on the expression of genes and tissues and     
1:51:46     
in cells that you can see these differences and it's may be conducive especially when we know that the MP has     
1:51:52     
a lot of Downstream targets conducive to these kind of changes we identified several so far uncharacterized BMP     
1:51:59     
dependent transcription factors in signaling molecules whose bilaterally symmetric symmetric expression may be     
1:52:06     
indicative of their involvement in secondary axis pattern so this is where we have this anterior posterior axis and     
1:52:14     
we have this expression of EMP in the cells and we have this bilaterally symmetric expression because the cells     
1:52:21     
will go to their position positions on the left and the right and you'll have this expression of symmetry but then     
1:52:27     
sometimes you have symmetry breaking within that uh basic framework so you have a left to right Axis you have an     
1:52:34     
anterior to posterior axis sometimes you have things that are asymmetric so like     
1:52:40     
the human heart is asymmetric with respect to our midline and our you know our anterior posterior orientation which     
1:52:48     
is our head or our feet and so if you have things like a heart it has to develop     
1:52:55     
asymmetrical you also have things that are also involved in this sort of uh secondary axis patterning meaning that     
1:53:02     
they're other axes that form so the left right Axis of course is maybe a secondary axis to the anterior posterior     
1:53:10     
axis um because in say in celegans you have the anterior posterior axises     
1:53:15     
established first and the two cell stage and then eventually as you get uh cells     
1:53:22     
have pairs they usually have left right Pairs and that happens later in development so you this is a secondary     
1:53:28     
axis and so you get patterning in that way but you can also have dorsal vental patterning so that's another actus that     
1:53:35     
emerges so this is all like kind of where we're leading to we have this this     
1:53:40     
BMP expression through a simple molecular complex there's a gradient     
1:53:46     
that forms you get then the downstream targets of BMP which be the pattern     
1:53:51     
formation and then you have secondary Axis or secondary AIS patterning which     
1:53:57     
leads to the sort of the refinement of the basic body one of these molecules is z swim     
1:54:04     
4-6 which includes a novel nuclear protein that can modulate the pem one5     
1:54:09     
gradient and potentially promote BMP dependent Gene regression the editor's evaluation says that this important work     
1:54:15     
presents a systematic survey of Downstream Target genes of the BMP pathway during body axis establishment     
1:54:21     
in the night Darian nomat Stell vectensis combining genomic approaches and genetic manipulations across species     
1:54:30     
they the author's present convincing evence that zwi 4-6 acts as a negative     
1:54:35     
feedback regulator of BMP activity so we have BMP the downstream targets Z swim     
1:54:42     
46 which then becomes a negative feedback regulator these body axes form a cartisian coordinate system which the     
1:54:49     
location of different morphological structures are specific by gradients and morphogen signaling in case of nidaria     
1:54:56     
it's actually outside of Bia but it has a lot of it's an evolutionary sister     
1:55:01     
group it's an outgroup that is very similar and it has a lot of the same molecular um mechanisms so we should     
1:55:09     
expect to see in Nigeria the sort of the precursors to things that lead to pattern formation in biolan species so     
1:55:17     
this is what they're trying to do they're trying to find sort of the phenetic origin of pattern formation and     
1:55:24     
I think taken with the other papers I think it has a good explanation of the sort of multi- range or multiscale     
1:55:32     
pattern formation some of the mechanisms we both physical and so thanks for paying attention and     
1:55:40     
hope you learn something
