## TRANSCRIPT

0:04     
Hi how are you all right welcome     
0:10     
uh so any updates uh anything we want to talk about before     
0:17     
we get started now working on my thesis oh yeah     
0:23     
write up yeah yeah how is it going just     
0:31     
I had to find a validation for my measurements     
0:36     
and um so I I did um sort of basically     
0:43     
I'm having to write up a section like a a chapter in my thesis for validation     
0:49     
because there is no direct measurements that I can get that are exactly the same     
0:59     
as what other people have measured I'm going to call it a fibro     
1:06     
blast these are all blast if you pull on they're stiffer so that's that one and     
1:13     
this is that one my conclusions I just call them all     
1:22     
fiop that's what everyone else does I guess it's just kind of like the     
1:28     
standard uh cell type yeah and if you want lung tissue well     
1:36     
you can't uh stretch it more than about 10% or it fluidizes apparently so     
1:46     
okay yeah fluidize and     
1:51     
tension that's what they did anyways     
1:58     
so and and it's really interesting their um their curves they they do have sort     
2:05     
of A J curve but they're all over the place like they're like this and that's what I'm     
2:12     
getting for um all my data they start off going     
2:17     
every which way and it's very very deterministic like you can measure it 10 times and you     
2:24     
will get the exact same graph oh wow     
2:31     
yeah very stable um hexagon graph     
2:37     
okay it's interesting yeah why should a hexagon be     
2:45     
stable like that I well yeah I think well it's geometric stability of course     
2:52     
but I don't know yeah anyway they are at least that     
3:00     
um I had another setup where I had the um microtubules going the other     
3:07     
direction so that it wouldn't stand up at all and um I would get all kinds of different     
3:15     
results all the time from [Music]     
3:22     
it interesting yeah I did see that you sent some papers to dick and I I'll have     
3:28     
to look through those I didn't I mean I saw the email I didn't get a chance to look through     
3:33     
them yeah that's okay I I was writing another chapter for my thesis and I ran     
3:41     
across a few things so I thought you would like them oh yeah     
3:47     
yeah yeah that's good yeah we'll have to look at those uh pretty soon     
3:54     
so well good luck with the thesis thank you hopefully we can yeah     
4:01     
see some of the data later when you make sense of it more and well I I'll write     
4:07     
it up hand it in they'll probably want to rearrange it and um when     
4:14     
I handed it in I can present it here okay it sounds     
4:22     
good thanks Susan looks like Hussein is here if he wants to give an update     
4:33     
oh yeah I can give it update thanks so um uh I've been working with uh with Z     
4:39     
on the breast on the on the rbct projects yeah yeah we got published we got     
4:46     
published really really happy yeah yeah I saw that paper excuse me yeah     
4:55     
yeah um     
5:01     
yeah um that's basically it yeah yeah yeah did you want to put a link to the paper in the chat so you can see sure     
5:08     
sure I mean it's not it's not it's not public it's not public yet oh okay we have our PF we have the paper all     
5:16     
right okay yeah but you're doing pretty well otherwise yep yep yep yeah happy to also     
5:23     
happy to also contribute to double worm to openworm as well too yeah um if I need to if there's any way to yeah     
5:30     
I I will just say with with the word projects um at times it can be difficult to figure out what really needs to be     
5:35     
worked on you know uh sometimes have been raised like maybe four four or five     
5:40     
years ago and it's like it's like tough to know okay but what's relevant right now so     
5:48     
yeah well I mean it's kind of hard to say we're having an annual meeting this     
5:55     
next month so maybe in about a month from now um okay yeah and I think like a lot of the     
6:02     
things revolve around the uh the containerization so Docker     
6:09     
container have some other things like simulations we've been talking in the     
6:14     
group here about you know doing things with uh copy cell 3D and some of the     
6:20     
other multiphysics models and you know there are other things in open worm I'm not sure exactly     
6:28     
what uh you know I know porg Lon is probably the most active person     
6:36     
right now well yeah I'm also I'm also definitely open to um to the contacting individuals who may know of something     
6:42     
and dire need of issue or something I'm also happy to suggest my own my own changes like if we have if I been     
6:48     
directions to the project do are things you want to take things research research software AI machine learning     
6:54     
perspectives to you know obviously I don't just want to like air out random ideas and you know I want to make it informed and inform pitch about     
7:01     
something that may be relevant you know that's uh really important um but um the other thing I'll mention is that um I've     
7:07     
become a lot more uh stable with my career at the edtech startup in California for that educational technology startup so I feel a lot more     
7:14     
confident and comfortable taking on um side projects within within limits so     
7:20     
you know once you once you like feel like like stable with that then making open source contributions to voluntary     
7:25     
products like open worm actually feels a lot easier you     
7:31     
know like um you know other than when I was applying to like 20 jobs a day sometimes and it's taking time to to to     
7:39     
taking time out of my day to go to the gym felt too difficult because I'm like I should be firing off job applications     
7:45     
but you know the recent project progressed both on RB on the on the uh the Janice for the RBC project with with     
7:52     
zck and um the work I do with a startup with alter learning so for those of you who don't know I've been working at a startup in California alter learning and     
7:58     
I spent my time between both of these projects both of these recent successes have been really amazing we've been     
8:04     
using toas and I got toas working uh so so toos is a uh excuse me toos is a     
8:09     
simulation tool for um multi excuse me multi using the multi Cara method for     
8:16     
radio therapy like radiation radiation uh radiation uh radiation therapy     
8:21     
simulations excuse me and um we looked into this one among many other software     
8:26     
many other programs and things and uh is working now so now I can be a lot more creative and uh liberal with the best     
8:33     
ways to visualize the best ways to move forward with the's data like what sort of simulations are important for us and     
8:39     
um I've even uh contacted a patent lawyer for our publication and they've     
8:45     
gotten back to me they're happy to do an art search um an art art so that's     
8:51     
called a prior art search to test of uh just see excuse me a prior a prior art search     
8:58     
to deter DET um the qualities of our technology from a patent perspective you see it's relevant it's novel if it's if     
9:06     
there's utility with this and um I'm going to look at my options with them there and with them on the JIS Tech J     
9:14     
technology we'll be patenting electronic steering of x-rays that would be the patents you know so and if we can patent     
9:22     
that in establishing multi patterns IP portfolio you know that's probably the I     
9:27     
would see that as the best thing moving forward so yeah     
9:33     
yeah that's good um yeah I mean and you're and you're welcome to propose     
9:39     
your own contributions to either openworm or Diva worm I know you had the     
9:45     
open enclosed worm repository were working on a while back um yeah I mean     
9:50     
you know like a lot of machine learning stuff I think would be good as well I mean there are a lot of problems you     
9:56     
could kind of approach using those methods so yeah I welcome you to or we     
10:03     
can talk about it more or you know just kind of come up with your own thing and you know just go     
10:11     
ahead just just always saying you know these products they they come and go but as long as you like have decent followup     
10:17     
every now and then I find things very productive and practical even just like coming back to something like several months later you know we get back on     
10:25     
board with things you know because because sometimes things happen Things fall to the wayside it happens here and there you know know I think like even getting to past running took took a     
10:32     
month or two of me just messing around with it until I got working so things like that happened it's still a big deal it's all right um but yeah yeah yeah     
10:41     
really happy to keep moving forward so yeah yeah great thank you     
10:48     
um Morgan had a update or if he's in the morning     
10:55     
schedule and I was like you no just uh just hoping to get everybody     
11:02     
to school on time yeah good so yeah actually given what     
11:09     
Hussein was just talking about um we do have a new data set to look at here so     
11:15     
there's a group of people who uh it's actually uh let's see this is uh a group     
11:24     
of people from I guess MIT and then some other people so Ed Boyen is involved in     
11:30     
this and I know that um Conrad cording actually talked to me a little bit about     
11:35     
the uh seans connecto at one point but anyways this is a group that is looking     
11:41     
at the seans connectome and you know they have this uh new data set this new connectome so     
11:49     
you know the connectome and seans is well known we know all the cells we know     
11:55     
their nomenclature uh and we can model the entire ire connecto and that's what's     
12:01     
one of the things that's appealing about celegans is that you have this relatively small connectome we know the     
12:07     
cells and it produces behaviors that we can recognize as you know like something that a complex animal would produce     
12:15     
anyways um so this is a data set I think that they've either generated or     
12:21     
compiled and so this is on hugging face so you know they have repositories on     
12:28     
hugging face and this is celegans connectome data Q Simeon is the     
12:35     
contributor and the is sort of the this is from I guess a conference Southeast     
12:41     
con uh and the paper was called scaling properties for artificial neural network     
12:48     
models of a small nervous system so the figure is here compiled     
12:53     
labeled connectum of celegans from multiple experimental sources standard I     
12:59     
to a common graph tensor organization format these tensor files are formatted as following with the relevant     
13:05     
accessible attributes and methods so this is where they've taken different uh connecto models from the     
13:14     
literature you know we have a hermaphrodite connectum we have uh a     
13:20     
male connecto and um we even have like a Developmental connecto and usually you     
13:27     
know most of the data is based based on uh Gap Junctions so the idea is that if     
13:33     
two cells have a gap Junction they're connected and so you can draw out this     
13:38     
this graph um and of course they're also like synaptic uh based connectomes as well     
13:46     
but this I think this is just basically from different Gap Junction connectons     
13:52     
with what they call electrical connections and it's from multiple experimental sources so they're kind of     
13:58     
compiling everything and then they're trans transforming this into a graph tensor form so they're getting this     
14:05     
ready for machine learning basically so this is where we have this is an example     
14:10     
of the data structure here graph tensors there's an edge index Edge     
14:17     
attributes uh some information about the graph the number classes the X and Y     
14:24     
coordinates and some other labeling information there uh some important     
14:30     
notes number one is The Edge index attribute or edgecore index attribute     
14:36     
contains directed Edge connection between information between node pairs so Edge index is like where you get the     
14:42     
connectivity information so two cells are connected they have this     
14:48     
value number two is that the edge ATT attribute or the edgecore ATR attribute     
14:57     
contains Gap Junction and chemical synapse weightage so they do include chemical synapses as well as the Gap     
15:04     
Junctions or the electrical Gap Junctions so they have both of those in here um and so this is uh it it contains     
15:14     
Gap Junction and chemical synapse weightage values aligned with edgecore     
15:20     
index directed edges so you're they basically have this connectivity     
15:26     
information they have the weights and aligning it with directed edges so you     
15:31     
know kind of which in which direction there's a connection and what the weight is and then the third Point here is that     
15:38     
the positive attribute or POS attribute contains three location 3D location     
15:44     
information for each neuron so this is where you have I guess it's position not     
15:50     
positive but it's this uh you know basically the I guess a a three Tuple     
15:56     
where you have the 3D location so this is how you might execute this in     
16:01     
a c laab notebook and this is the kind of the code graph tensors you load P torch I     
16:09     
guess uh you go through this and you sort of Define everything and you draw     
16:16     
the connectome in graph plot 3D false so     
16:22     
let's see this is a visualization of said Network so this is how they're     
16:27     
visualizing it uh they have the labels they have the     
16:32     
categories they have the connections in red and you can see the connections how     
16:38     
they're directed as you have arrows going from one set of neurons to some     
16:44     
other neurons out here and you have either you know you have both types of     
16:49     
connections Gap Junction synapse the red ones are synapse the blue ones are Gap     
16:55     
junctioned I don't think they're showing Gap Junction in this image but the idea is that and then this this kind of     
17:02     
circle here is the edge of the dorm I guess so this is just kind of showing the uh connectum sort of in place in the     
17:11     
Anatomy showing the connections taking all these data and putting them     
17:17     
together which is nice because you know a lot of the there are a lot of data sets out there that are kind of you know     
17:25     
independent and so it's nice to have this here so I I guess this is kind of their first take at um you know building     
17:33     
this for some sort of machine learning purpose and getting the data into shape     
17:39     
for that any questions about that or     
17:51     
comments okay so that's available um that is on hugging face it's data sets     
17:59     
qiman and then celegans connect out data the other things I want to talk     
18:05     
about so this is actually um the turer lab and they're based I think in     
18:10     
Paris um and they do they have some interesting stuff on their GitHub and I     
18:16     
wanted to point this out so they're actually the turer lab they uh it's called ver they sort of username is     
18:24     
virtual embryo on GitHub and so they have a a lot of different repositories     
18:31     
they have this uh fo embryo or FBO uh repository which is this work that they     
18:38     
did in a number of papers where they were able to infer tensions and     
18:44     
pressures from 3D microscopy images of embryos and tissues so they wrote a     
18:50     
paper on this a while back and I'm trying to remember what the citation was     
18:55     
citation is here yeah where I think it's a a bioarchive paper yeah this is the     
19:03     
paper embryo mechanics cartography inference of 3D Force atlases Fromm fluorescent microscopy so     
19:10     
this is Herve tur turlier and uh     
19:16     
colleagues and you know this this is the paper here where they do this measurement of uh forces at the whole     
19:25     
embryo scale so they do these foam simul ations so uh basically they want to     
19:32     
understand the morphogenesis of tissues and embryos and the role of mechanics in     
19:38     
them and so we can get sequencing we can use sequencing methods to get spatio     
19:44     
temporal maps of gene expression but we can't necessarily get a map of cell mechanics in the same way so that's what     
19:51     
they want to do here they're basically treating the the uh cells in the embryo     
19:57     
as a foam like geometry so they're using the soft materials approach to modeling     
20:03     
the embryo cells in the embryo so they call it the foamlike geometry of cells and embryos we propose a robust end to-     
20:11     
end computational method to infras spatio temporal atlases of cell cellular     
20:16     
forces for fluorescence microscopy images of cell membranes our method     
20:21     
generates precise 3D meshes of cell geometry and successfully predicts relative uh cell surface tensions and     
20:29     
pressures in the tissue we validate with 3D foam simulations study its noise     
20:34     
sensitivity and proove its biological relevance in Mouse aidian and celegans embryos so they actually use a wide     
20:41     
variety of embryos including the acian or the C squirt and the mouse so we have     
20:48     
uh you know a number of different uh modes of development there uh 3D     
20:53     
inference allows us to recover mechanical features identified previously but also predicts new ones     
21:00     
unveiling potential new insights on the spatial temporal regulation of cell mechanics and early embryos our code is     
21:07     
freely available and paves the way for unraveling the unknown mechanochemical feedbacks that control embryo and tissue     
21:14     
Mor for Genesis so if we go back to the     
21:21     
repository uh we have this is the breed me and if you go here I guess we have     
21:28     
the uh I guess we have information about their model     
21:34     
organisms and yeah so they have some images here this is     
21:41     
equilibrium this is the force viewer this is the sort of the software     
21:46     
they're using is polycope where they're able to kind of model the cells     
21:54     
um and I think maybe it'd be better to go back to the uh me to see if they have     
22:00     
any information about how to run this or what to download so basically uh you     
22:06     
build these simulations um uh basically you uh have to run foam     
22:14     
embryo with uh python it's python code so you plug it in you can use you know     
22:21     
some uh algorithm to do image segmentation so you have to segment the     
22:27     
images of the embryo you're interested in you have to create masks then you     
22:32     
have to um basically uh run the software     
22:38     
run the analysis and then you can view it in polycope which is there's a link to it here so this is way to visual wise     
22:44     
this 3D geometry so it's the software you know you basically do the     
22:49     
segmentation use the software to do the analysis and then you visualize it in polycope so that's basically what we're     
22:56     
doing so you know it's pretty easy to install pip install foam embryo uh you install the package and     
23:05     
then you start doing your work so you're taking things from a segmentation mask     
23:10     
and you're doing the analysis and then you dump it into this viewer so this is where we have the API     
23:18     
and documentation so they have actually pretty good uh layout of how to use this and showing the examples and everything     
23:26     
so this is uh nice software here I just thought it might be interesting     
23:31     
especially for Susan um and they have some other things here as well going on a lot of things     
23:38     
with mechanics and uh the physics of the     
23:44     
embryo okay third thing I want to talk about was this embryo multiscale modeling course this is something that     
23:52     
um Morgan pointed out I think in the slack and he gave a link to it so this     
23:57     
is a multi scale modeling course this is mrio membr R iio which is an     
24:04     
abbreviation of some type so uh basically it's a modeling course where you can go through and you know follow     
24:11     
along and you know create your own sort of get a lot of knowledge about     
24:16     
multiscale modeling and uh apply it to your favorite system so you know     
24:22     
basically you go through the introduction there's this introduction to Scientific computing     
24:29     
uh partial differential equations and biological systems so a number of examples of that uh and then equations     
24:36     
of State going through that signaling pathway     
24:41     
Dynamics so we're going through kinetics biochemical kinetics reactor stability     
24:47     
and sensitivity uh looking at energy balance and steady state energy     
24:54     
energetics uh and then morphogen gradients and of going through that uh     
25:00     
reaction to Fusion in an antibody detection device that's a project that they're offering here kind of a sample     
25:07     
project to go through and work out on your own so you kind of get the background up above and then you start     
25:14     
doing this uh project there's another project Max toxin     
25:20     
manufacturing which is an antitoxin another project on circadian     
25:25     
rhythms where you can model you know feedback loops and oscillators phosphorilation     
25:32     
States uh and other things like that model chemot taxis adaptation where     
25:38     
you're you know you have different models for chemotaxis the Spyro model the barai     
25:44     
model and you know information about the chemotaxis     
25:50     
network uh then they have this model paper ethanol and U aceto heide uh metabolism in human     
26:00     
beings uh this is this kind of a paper you can go through another model paper     
26:06     
smad nuclear uh cytoplasmic shuttling and mutated system so this kind of goes     
26:12     
through that paper and then you have this catalog of Jupiter notebooks and     
26:19     
that's the whole CL course so this is a nice course to go through if you're     
26:24     
interested in multiscale modeling kind of using a lot of different physics tools and other types of tools     
26:31     
to do modeling so any comments or questions     
26:37     
about     
26:43     
that just to say that um that was from     
26:48     
um that that course website was from a postto and James glazers lab okay yeah     
26:56     
yeah they host oh sorry is it a course that's     
27:07     
available is it available online well it it's it seems like it's     
27:14     
the course material that they're putting together anyway that that was all it it     
27:20     
was like you mentioned this at the beginning of one of their     
27:26     
um uh what's doing he he does this graduate course like     
27:33     
network network Dynamics or something like that it's basically going through um so     
27:42     
how parents it's Soros um system system biology     
27:50     
text um I I'll I'll try and find     
27:57     
more something on youtubeer has     
28:04     
um well a YouTube     
28:11     
channel yeah that they do these     
28:19     
days yeah that's um yeah I don't know like I know that that's you mentioned     
28:25     
it's from James glazier's group so yeah it's it's kind of that take on it     
28:32     
very much uh biophysics and also a lot of you know there's some dynamical systems in there there's some kinetics     
28:39     
and other types of you know systems biology analyses and so     
28:46     
forth really be really fun to go through to see yeah for     
28:54     
sure this is um sort of sort of on top um it's     
29:03     
uh been doing a lot of uh like physics based modeling um we     
29:11     
we're using that for uh in this case for ultrasound um but really interested     
29:19     
in you know because of because it's so computationally demanding really interested in um following up on some of     
29:27     
these um physics and informed neural networks and you know the     
29:36     
um anyway I I this weekend I put together a couple pages of of kind of     
29:41     
notes and and references um and uh trying to I'm sorry I'm not at my     
29:50     
computer but uh like     
29:55     
multiscale Behavior is one of those things that um that the clac professor I'm I'm     
30:04     
blanking on her name it's like en     
30:10     
Ania um again I I'll I'll find slinks um but um if anybody's interested um yeah     
30:19     
I'm I'm definitely looking for some help and um uh you know these are these are     
30:26     
interesting ways to solve finite element problems um you know we're currently     
30:32     
using these finite difference um domain models uh but uh but it's     
30:39     
really yeah yeah really interesting for modeling multiscale Behavior     
30:48     
te right and what are the connections there like in terms of the the connections to     
30:55     
the embryo uh yeah as an example like why would what's the connection between     
31:01     
finite element and multiscale models I mean well just     
31:07     
just so finite elements you might um well the multikill you you the models     
31:16     
get quite complex right because you've got to have kind of like a mechanistic model at at every level and and you know     
31:26     
they they the the real benefit with these um kind of neural operators is     
31:32     
that you're you're turning it into a machine learning problem you know that     
31:38     
that you're solving with this kind of automatic differentiation that um that     
31:44     
is actually just learning the mapping between between domains and     
31:51     
um uh the you know there's some some real     
31:59     
efficiencies um and in terms of doing this you know     
32:05     
computational efficiencies um uh that you preserve the     
32:10     
kind of multiscale behavior um you     
32:16     
know whether it's necessarily as interpretable or you know     
32:22     
um that's yeah I'm I'm interested to see what what the Es are that um as we     
32:31     
just you know I'm still focused on implementing the boundary element and finite element cells to kind of complete     
32:38     
out the the normal kind of physics models yeah yeah     
32:45     
sorry dogs dogs and kids have arriv     
32:52     
Rush my finite element analysis experience     
32:59     
um just putting the tri Junction into my     
33:04     
cells and um means that oh maybe you see     
33:09     
me better here um is um a problem like I'm kind of at     
33:16     
the edge of it saying that my I have to refine my mesh because I'm I've got     
33:24     
things that are too dispar in size like the the side of the cell is too large     
33:33     
compared to the tri Junction okay I've had some trouble in     
33:39     
that that with that size difference even though     
33:46     
um um the finite element analysis you're supposed to be able to do chemical     
33:53     
chemicals or um uh look at the physics of um I I'll     
34:01     
say proteins um that's two different too big a     
34:07     
difference in size to translate that back to um the cells that I'm I'm     
34:14     
drawing anyway that's that's my experience with it is uh trying to do     
34:20     
the multi-size multi-layered model at least in console     
34:26     
doesn't seem to want work     
34:32     
yeah all right so that's good um yeah thanks for that uh discussion that was     
34:39     
good um especially putting the last thing that we're talking about in context and how that's     
34:48     
useful so let me move on to some papers um okay so let's see     
34:55     
um we talk about this one um so this is a paper uh from biological     
35:02     
Theory um this is uh a special in a special issue uh called Darcy Thompson's     
35:09     
conceptual Legacy so Darcy Thompson of course was uh alive you know over a     
35:16     
hundred years ago he wrote a book hold on growth in form and he proposed that a     
35:21     
lot of growth processes and a lot of these uh anatomical structures in     
35:30     
seashells and in other types of uh systems were they had this mathematical     
35:37     
aspect to them so there were mathematical scalings and you know you could use the golden mean to say describe a seashell     
35:45     
and things like that so he was very much mathematically oriented and you know he     
35:50     
wasn't necessarily A darwinian but he did kind of bring     
35:56     
these uh ideas is up in in sort of a math put a mathematical cast on so you     
36:02     
know that's that's interesting and people have tried to follow up on his work there's been a lot of stuff in     
36:08     
terms of simulation in terms of the mathematics and you know there's been     
36:14     
some interesting work on this so This thematic issue is kind of revisiting this conceptual Legacy so this is a     
36:21     
paper um called biological theories of morphogenesis based on holistic     
36:26     
biophysical thinking and this is from uh Amy Shire and alen Rodriguez who um I     
36:33     
can't remember they work at a university in New York City um not sure if it's Colombia or uh one of the other     
36:40     
universities there but they do uh have done a lot of really interesting stuff uh with respect to morphogenesis     
36:48     
so this this paper is relatively new it was I think accepted last month so um     
36:56     
let's let's read through the abstract here uh so the abstract reads the roles played by physics in the study of the     
37:02     
life sciences have taken on many forms over the past 100 years so this is again     
37:08     
this sort of physical mathematical approach to I guess you know life     
37:14     
sciences in general but developmental biology and uh you know biology more     
37:20     
General more specifically here we analyze how physics can be brought to bear on the Contemporary study of     
37:26     
morphogenesis were new tissue scales form uh tissue tissue scale forms arise of simpler more     
37:35     
homogeneous initial structures we characterize how morphogenesis has been studied through     
37:41     
reductionist approaches and discuss their limitations we suggest an alternative     
37:47     
way of approaching morphogenesis that begins with a consideration of the whole which may     
37:53     
also serve as a fruitful mode of scientific inquiry so what they're proposing is that you know that there's     
38:00     
been a lot of a lot of the work on morphogenesis has been this uh sort of     
38:05     
reductionist approach where you're looking at specific genes and manipulating them and seeing what their     
38:11     
effect is on the phenotype where you're looking at specific manipulations and looking for     
38:17     
mechanisms um which are usually you know molecular but you can also see them at     
38:23     
the morphological level and so you know if you look at the reaction diffusion     
38:28     
model that's I guess reductionist because you have you're looking at different morphogens and sort of their     
38:34     
behavior in specific places in the embryo so um basically though I think a     
38:42     
lot of the experimental work is what they're critiquing um so they're proposing this     
38:47     
alternative way of doing this uh by considering the whole so through     
38:53     
historical analysis of Concepts relevant for contemporary systems biology we illustrate how physical thinking     
39:00     
oriented towards the biological whole or holistic biop physical thinking was     
39:05     
exemplified in the biological theories presented earlier by Dary Thompson and     
39:11     
per alberg so per Alber actually was another person has done a lot of things     
39:17     
it uh did things with uh like uh eletric     
39:22     
scaling looking at that published a number of works with um     
39:28     
you know like Stephen J G and other people so it's interesting he had an interesting um career as well in any     
39:36     
case they're drawing back to those works and they're trying to kind of trying to bring that work     
39:42     
forward we contrast the use of physics to develop the conceptual foundations of     
39:48     
holistically oriented systems biology with the more prevalent contemporary use of physics that focuses on technological     
39:55     
development and quantification embedded with reduction of strategies so     
40:00     
they're they're kind of saying you know you know you might use something like nextg sequencing or some of these     
40:06     
Imaging tools where you're looking for you know patterns in the whole embryo     
40:12     
but you're kind of looking at features they looking at you know they want to kind of bring these theories to the kind     
40:19     
of techniques that we use today to look at uh morphogenesis so that's kind of where     
40:25     
they're going with this an in-depth contemporary case study as an in-depth contemporary case study we describe how     
40:31     
in our research we have used holistic biophysical thinking to develop biological theories of vertebrate     
40:38     
morphogenesis that account for and extend beyond genetic molecular and cellular processes we propose a theory     
40:45     
of skin development where both molecular and morphological patterns were orchestrated by emerging physical     
40:52     
processes at the supracellular scale so this is this work they do on skin     
40:57     
development where they have like a model of the cell like a layer of epithelial     
41:03     
cells but also have this uh genetic circuit that fits into the model and so     
41:10     
that's what they're going to talk about in this paper uh but basically you know there's this submergent physical process     
41:17     
that happens at the layer of of cells at the tissue scale and that's kind of what they focus on WE further pause at that     
41:24     
holistic biophysical thinking at the super so scale May Advance theories of disease diseases such as cancer so     
41:33     
that's the obligatory call for applications in any case they talk about     
41:38     
holism and they talk about reductionism and so you know uh the idea     
41:44     
is that of course uh you know biological systems or whole takes precedence holism     
41:52     
and where physics or what we will refer to as biophysical thinking is Central to the test     
41:58     
to conduct an analysis on what we call Holistic biophysical thinking we will Trace two biological theorists we are     
42:04     
pioneers and the use of observation combined with physical thinking to characterize the dynamic structures     
42:11     
leading to the creation of animal forms so this is where they talk about Darcy     
42:16     
Thompson and per Alber and kind of going through that work um so first we     
42:23     
discussed the two forms of reductionism time and scale so this is where we have     
42:28     
reductionism in terms of time and reductionism in terms of scale so this is an interesting way to divide this up     
42:35     
so of course we we're familiar with reductionism uh with respect to scale     
42:41     
which is basically spatial reductionism or like you know reducing things from     
42:47     
cells to genes to you know who knows what but you know it's basically going     
42:54     
from the hole down to the mechanism and so we're familiar with that and it's usually spatial in an embryo because we     
43:01     
want to know like maybe in a specific cell what the mechanism is or in a you     
43:07     
know maybe in a bunch of cells what the specific mechanism is so there is a spatial component as well but this also     
43:14     
there's this idea of reductionism is is sort of reductionism in time and so that     
43:19     
means that you know you're trying to narrow down the window of some change so we might want to know say for example we     
43:26     
see a change in the embryo some stage of development we want to kind of reduce     
43:32     
that down to the exact precise mechanism and time that happens so there's some     
43:38     
mechanism that we want to know that kicks everything off and that's what we want to know and so you know that that     
43:44     
doesn't necessarily leave room for things that are sort of these uh systems     
43:50     
that have feedback a Perpetual feedback or systems that sort of are     
43:55     
self-propagating are kind of left out of that kind of analysis so you know I mean we've     
44:02     
covered papers where they talk about these kinds of things in more detail so     
44:07     
this is just one take on this I think it's an interesting take but um just be     
44:12     
be aware that this isn't like kind of a novel thing but it's interesting um the way they're doing this so uh we suggest     
44:21     
that a reductionist approach precludes the possibility of accounting for properties that are created across time     
44:27     
or properties that must be elaborated temporarily or across scale properties     
44:32     
that emerge at a scale or level of organization larger than the constituent     
44:38     
Parts in contrast to reductionist approaches we then articulate an alternative investigative approach that     
44:45     
we call Holistic so Guinness is where you know it's greater than the     
44:50     
parts uh we provide our view of its key elements number one is the the     
44:57     
importance paid to direct observation of a biological system so this is where we     
45:02     
have holism where you can observe the biological system as a whole and see what that looks like number two is a     
45:10     
focus on processes and relationships among the parts of the system which can lead to awareness of emerging properties     
45:16     
of the whole so the first key element is observing sort of     
45:21     
the uh sort of the overarching scale of something that's emerging so for example     
45:28     
I can look at an embryo and I can see like movements in the embryo I can see changes in the embryo that are sort of     
45:35     
where the emergent properties of some bio or some developmental process occur     
45:42     
and I can observe that I can't really observe any of the mechanisms or processes or relationships I just     
45:48     
observe it from sort of the you know the sort of the 10,000 meter view so the     
45:54     
second part though is to kind of focus on the process and relationships amongst the parts of the system so when you see     
46:00     
this phenomena happening you want to know what parts are involved how they're     
46:06     
involved and so you can't really say something is emergent unless you're aware of its emergence and that's kind     
46:13     
of the point uh emergent properties are those that cannot be characterized or     
46:18     
predicted solely by an understanding of the intrinsic features of the parts of the system so we know that emergence is     
46:25     
sort of the greater than the sum of the parts so if we were to take like a bunch of reductionist studies looking at     
46:32     
specific the action of specific genes or the action in specific cells we can't     
46:37     
simply put them end to end and get a a complete description of the emerging phenomena there's something else there     
46:43     
we need to describe so this is kind of the problem that we've had with you know     
46:49     
a lot of things in biological modeling but also a lot of things in     
46:54     
understanding um you know how say can move from the the scale of individual     
47:00     
cells to the scale of say tissues or embryos okay so this is the standard     
47:07     
definition of emergence putting it in the context of Developmental biology     
47:12     
then they talk about Darcy Thompson who pioneered a holistic approach that applied biophysical thinking to the     
47:18     
consideration of systems level morphological properties so he adopted this as the approach where he took you     
47:25     
know a whole organisms or you know things like that and he did a     
47:31     
mathematical analysis and described sort of the emergent properties of it using     
47:36     
mathematical properties didn't really think too much about what was go what was going on underneath that whole you     
47:44     
know the entire emerging process but gave a description of the emerging process     
47:50     
itself Thompson adopted his this approach because for him the whole is not merely the sum of the parts rather     
47:56     
rather the result of relationships amongst them so you know again you have     
48:02     
say like a sea shell or like a fish that was you know growing and you can     
48:07     
describe that with mathematical laws and then you can maybe make some statement     
48:12     
about the underlying biophysics um Thompson's work was predicated on the     
48:19     
use of observation reminiscent of how natural historians look at specimens as     
48:24     
the basis for generating physical insights and inferring mechanisms and so     
48:29     
Thompson's holistic biophysical thinking can be brought to bear on the study in many processes and development as well     
48:35     
as disease however to concretely contrast holistic biophysical thinking and     
48:40     
reductionist approaches we focus here in one particular biological context and     
48:45     
this is where we get vertebrate organogenesis uh this is the onset of     
48:51     
ver organogenesis specifically so for that sort of model system we can turn to     
48:58     
the work of per Alber who uh s we suggest can be seen to     
49:04     
exemplify holistic biophysical thinking in contrast to perspectives that focus on the instructive roles played by     
49:10     
chemicals and molecules and patterning which include the Turing reaction diffusion model as well as some other     
49:18     
models um yeah okay so in contrast to     
49:26     
perspectives that focus on the instructive rules laed by chemicals and molecules on patterning which includes     
49:32     
Trang morphogenesis and some other related approaches um Elbert posited that the     
49:39     
physical processes am collectives of cells play a fundamental role in generating form during     
49:47     
organogenesis so basically you're looking at the physical processes like we talk about with respect to um     
49:55     
tensegrity you have these physical processes that play a role in collectives of cells and shaping those     
50:01     
collectives of cells and then this contributes to organogenesis and the idea is that     
50:07     
that's that has emerging properties which we can then describe you know by     
50:12     
looking at lower and lower scales but the sort of the level of analysis that's     
50:17     
most interesting is at the biophysical but why don't we go yeah why don't we go     
50:23     
to this part here where they where they talk about um temporal length scale reductionism so the process by which     
50:30     
living systems create their forms or morphogenesis are commonly acknowledged     
50:35     
to be nonlinear and complex they are nonlinear because for instance a two-fold change in one protein and a     
50:42     
pathway does not lead to a twofold change in the final output of the pathway so you can have all sorts of     
50:49     
fluxes and things within different Pathways and it can lead to some final     
50:54     
output that is either greater than the sum of its parts or less than the sum of its parts um     
51:02     
they're complex because the components within the system are interrelated such that changing one variable leads to the     
51:09     
automatic changes in other variables so this is where you have um you know all     
51:15     
these different variables that um you know lead to if you modify one thing a     
51:20     
bunch of other things get modified and this makes it difficult to isolate the     
51:25     
effect of varying one component while holding all other components fixed so typically in a in an experimental     
51:32     
setting where we want to sort of make a manipulation and we want to test this     
51:38     
with a control we usually fix everything except for one intervention and we make     
51:44     
the intervention and we observe the output and we can say that the thing that we intervened with the thing that     
51:50     
we manipulated is the thing that caused them or at least was responsible for that and then this is a path towards     
51:57     
determining causality so it's very important that we get that kind of uh you know we get those kind of things you     
52:04     
know we control everything else in the system we focus only on the thing we're interested in of course the problem is     
52:11     
morphogenesis is that there are a lot of things that are changing all at the same time and it's very hard to fix those     
52:18     
things uh you know and and make them sort of constant and then do some     
52:24     
targeted intervention so that's that's why you know we have morphogenesis being     
52:30     
both nonlinear and complex and why it's very hard to use sort of a reductionist     
52:36     
technique to get at these problems when faced with a poor poorly     
52:42     
understood complex living system an investigator must adopt a research strategy in order to further their     
52:47     
understanding of the system while they are often only implicit research strategies used to investigate the     
52:54     
emergence of living form have become dominated by reductionism so this is     
52:59     
some these are some references that talk about the different types of reductionism so of course there's     
53:06     
temporal reductionism which we talked about uh which is of course you know     
53:12     
where we think about focusing on finding an initial state or source that holds     
53:18     
the information necessary to provide instructions to generate the final form     
53:24     
that is an interest to an investor instigator so basically having this sort of initial     
53:30     
state that will then give you the output that you want so like I just said     
53:37     
sometimes processes are nonlinear with respect to output and so you know having     
53:42     
an initial State isn't always going to give you that final output or that final     
53:47     
form so you know so for example this type of reductionist notion is present     
53:54     
in the idea that the genome there is information sufficient for specifying the skeletal     
53:59     
pattern we see in our hands and feet so you know reading things out from the genome is one example of this where we     
54:06     
don't really have a set of blueprints in the genome we have genes that are     
54:11     
expressed and those genes can be expressed in different ways furthermore you can interfere with that to modify     
54:18     
your hands and feet so that you know you may have uh fewer or more fingers than     
54:24     
you should and these are comp Lex processes of gene expression and Gene     
54:30     
regulation but of course using this reductionist approach we can't necessarily get at this you know we     
54:36     
still have to think about it in terms of you know reading out the genome and reading out the     
54:42     
instructions um in such a view individual cell biology as well as collective cell processes that occur as     
54:48     
a tissue takes on its structure are seen in processes that merely execute the     
54:53     
information stored in the initial state so you thinking about biology as an executable and this is actually relevant     
54:59     
to some of the things you've been talking about with synthetic biology because you know we talk about executing     
55:06     
things from the genome you know there there's uh been a lot of work in bacteria where you're executing sort of     
55:13     
programs in the genome you're reading out things and for simple if you're you know trying to express simple compounds     
55:20     
maybe that's a good way to do things maybe it's easy to sort of execute the genome and get like a some compound or     
55:28     
some uh you know enzyme that you want to produce and that's fine but like when you're doing something in realm of     
55:35     
morphogenesis maybe this is a little bit harder because you're not just uh you     
55:40     
know you know building chemicals in the system but you're also regulating their     
55:46     
concentration and you're reg regulating the silver space and many cells are doing this collectively so it's hard to     
55:53     
get this sort of executable model to work in patter you know to explain pattern     
55:59     
formation uh concentrating efforts on identifying regulatory elements within the genome and the supposition that     
56:06     
their identification will reveal such instructions as determinative is therefore an instance     
56:12     
of temporal reductionism so you know they kind of go through temporal reductionism in this     
56:18     
section and they give some more examples then they talk about talk about length scale reductions which is where you you     
56:26     
have this uh reductionism uh with respect to scale so this is where we go     
56:32     
from you know like the genomic scale to the morphological scale and there are     
56:39     
different ways you can think about this so from molecules to cells to tissues to     
56:44     
organs to organismal this is called the hierarchy schel uh hierarchy Chanel of     
56:51     
scales which I guess is from Chanel 2007 and they talked about these different     
56:56     
scales and their properties so you know there are a lot of ways you could I guess     
57:01     
parse uh the hierarchy of biology but schow provides this scaler approach or     
57:08     
this scale uh dependent approach that might be interesting uh much of experimental     
57:15     
development of biology today has been influenced by the seminal genetic screen in veropa identifying key signaling     
57:21     
genes and transcription factors involved in structural patterning today 40 years     
57:27     
worth of genetic and subcellular scale Focus has led to a meticulous     
57:34     
catalog has led to a meticulous catalog of key genes as well as of corresponding non-coding Gene regulatory regions     
57:42     
biochemical signaling Pathways in cyos skeletal Machinery so we basically you     
57:49     
know in a lot of ways we think of events at subcell Scales is being considered to     
57:55     
be full responsible for mechanisms generated uh generating structure at larger scales and this is from nout in     
58:02     
1990 for example Master gene expression patterns or morphagene gradients were     
58:07     
often viewed as responsible for setting the pattern for structure SE structure seen at larger scales however the     
58:14     
possibility that Mor morphological features emerge at the tissue scale that are not reducible to smaller scales as     
58:21     
receive far less attention so they talk about this you know sort of struggle     
58:27     
between thinking about you know things that are happening at the molecular scale translating to the morphological     
58:34     
scale and whether they're you know sort of one thing is responsible for the other or merely if they're sort of     
58:41     
influencing the other and so you know sometimes you have things that happen at the molecular scale that merely     
58:48     
influence what happens at the tissue scale and there other things that we're not really describing that play that     
58:55     
give maybe a more uh full description of what's going on so they kind of go through this this     
59:01     
is a nice paper because they kind of go through this history and they talk about     
59:06     
the contributions of Alber and Thompson and bridging that to the modern era and     
59:12     
then talking about epigenetics and kind of thinking about this you know kind of giving this new     
59:19     
way of thinking about it or at least proposing a way at least to experimentalists so they can think about     
59:26     
emergence and they can think about their model systems but also in this context     
59:31     
of this multiscale set of relationships so is this something that is just merely repeating what we've been     
59:39     
talking about with respect to multiscale models or does this have something to contribute to this area I think it's the     
59:45     
lad I think you know they're kind of coming up with a number of really good     
59:50     
points here that kind of fit into this multiscale modeling effort you know     
59:56     
because we're thinking mainly about how to structure the computations and structure the problem     
1:00:01     
but this is really kind of linking this to actual systems and developmental     
1:00:07     
biology so that's that's the paper um any comments or questions about that     
1:00:30     
okay um well there are no comments or questions I guess that's it for this     
1:00:36     
week um anything else we want to talk about before we go     
1:00:43     
or tensity um the Tegrity that changes     
1:00:50     
shape um like you push it and completely     
1:00:56     
change a shape is there is there um another name for it     
1:01:03     
or keyw well keyword so I can look it up I don't know     
1:01:09     
met meta stable     
1:01:14     
okay that might work say anyway I think I should mention it     
1:01:22     
in my thesis all right that sounds good all right thanks for     
1:01:30     
attending and talk to you next week by thank you thank you all right     
1:01:37     
now I'd like to talk about a few papers and tissue biology DS so the first one is about Dynamics     
1:01:45     
from a single snapshot and the other paper is review that kind of calls into question the way we study     
1:01:53     
tissue biology and in fact connects to the paper     
1:01:58     
on uh sort of emergent properties and holism that we talked about earlier in     
1:02:04     
the meeting so the first paper is this     
1:02:09     
bioarchive preprint on temporal tissue Dynamics     
1:02:16     
from a signal snapshot so this is Jonathan summer uh sh Manor and Yuri     
1:02:23     
Elan and the systems biology approach to this     
1:02:29     
problem so the abstract reads uh physiological and pathological processes     
1:02:36     
such as inflammation or cancer emerge from the interactions between cells of however methods to cell populations     
1:02:44     
over time within the naive native context of human tissue are lacking     
1:02:50     
since tissue biopsy offers only a single snapshot so you know we have these     
1:02:55     
static snapshots of tissues usually tissue biopsies and we can look at them     
1:03:01     
under a microscope they don't tell us a lot about how a process like inflammation works or how cancer sort of     
1:03:10     
metastasizes or even how wound healing proceeds although we do have some good experimental systems of wound healing uh     
1:03:18     
usually those aren't sort of the you know the stand for kind of looking at these     
1:03:24     
things here we present one shot tissue Dynamics reconstruction or osdr an     
1:03:30     
approach to estimate a dynamical cell populations based on single tissue samples so this is again using a     
1:03:37     
dynamical model and modeling these cell populations and is using a single tissue     
1:03:45     
sample osdr uses spatial proteomics data to learn how the composition of cellular     
1:03:51     
neighborhoods influences division rate providing a dynamical model cell     
1:03:56     
population change over time we apply osdr to human breast cancer data and     
1:04:02     
reconstruct two fix points of fibr blasts and macras in reactions these fixed points correspond     
1:04:08     
hot and cold fibrosis in agreement with culture experiments that measure     
1:04:13     
Dynamics directly we then used ODS osdr to discover a pulse generating excitable     
1:04:20     
Circuit of T and B cells in the tumor micro environments these are two types     
1:04:25     
of cells cells in the tumor micro environment and that there's an excitable circuit that generates a pulse     
1:04:33     
suggesting temporal flares of adaptive anti-cancer responses osdr can be applied to a wide     
1:04:40     
range of spatial transcript toic or proteomic assays to enable analysis of tissue Dynamics based on patient     
1:04:47     
biopsies so patient biopsies are very easy to access and you would like to model things sort of not as a     
1:04:54     
theoretical thing but things that might happen in in a single patient or a     
1:04:59     
process that is sort of otherwise hard to image so this is where we can model     
1:05:06     
different aspects of the Dynamics of these tissues from the single input and     
1:05:11     
it gives us this you know pretty reliable set of outcomes including this     
1:05:17     
sort of excitable circuit Which models these temporal pulses so again this is an overview of     
1:05:24     
what they're doing this one shot two Dynamics reconstruction you have a biopsy of some piece of tissue you get     
1:05:30     
the spatial proteomic snapshot which is basically like Imaging the protein     
1:05:36     
distributions within the tissue then you get this dynamical model estimation from     
1:05:41     
the single static model and then you have the cell population dynamics that     
1:05:46     
you can mod from the differential equation     
1:05:52     
um from the D the of the dynamical model estimation which is u a function uh and     
1:06:00     
then you can extract cell population Dynamics from the phase portrait shown in three the second part is neighborhood     
1:06:08     
composition maps to a location in state space so you look for neighborhoods which are these local groups of cells     
1:06:14     
usually within a certain number of millimeters uh and so of a radius and so     
1:06:20     
you can actually model each uh you know neighborhood and its composition     
1:06:26     
and this maps to a location State space you see up here the phase portrait number three and a you have this you can     
1:06:33     
also map single cells to this pH portrait and get a state space estimation you also have celf Division     
1:06:40     
and death markers determining change in tissue composition so again you have this neighborhood you have different     
1:06:47     
activities in this neighborhood so for example you might have division events in the neighborhood and you might model     
1:06:54     
those as cells that are or cell populations that are moving around in state space uh so this neighborhood     
1:07:01     
evolves in state space and so this all lends itself to control for because we can see we have x dot as the dynamical     
1:07:09     
estimator and we have the states space and so we can actually do some states     
1:07:15     
Spas based control theory and so this is where this is going we also have breast     
1:07:20     
cancer tissue with neighborhood mapal location in the phase portrait estimated by OS s strr so again we have the static     
1:07:27     
snapshot the spatial distribution of cells fiber blast and macrofagos so it's     
1:07:32     
a mixed population you have these neighborhoods that are locally Salient spatially and then those map to this     
1:07:39     
phase portrait where you have fiberblast density versus maccrage density so it's     
1:07:46     
a little bit different than the number of cells of different types in the first     
1:07:51     
examples you now have this density uh phase portrait and you can map the     
1:07:57     
spatial distribution of cells to this finally we have probabilistic     
1:08:02     
Division and death events for each cell and if we do this we can look at the stochastic trajectory of tissue     
1:08:09     
composition over time so basically we want to know how many macrofagos and how many fiber blasts are present at any     
1:08:15     
given time and we can model that by looking at the birth and death dynamics     
1:08:21     
of each cell population type so we know what the start ing number of cells is we     
1:08:27     
look for doubling events we also look for uh eptic events necrotic events but     
1:08:34     
also you know cell divisions and then we calc we put all that into a a trajectory     
1:08:40     
which has the stochastic Dynamics and we can look at the proportion of microage     
1:08:45     
macrofagos and fibroblasts in the tissue the other one some of the other     
1:08:52     
things we look at we can look at uh division similar thresholds so we can have uh markers     
1:09:00     
such as ki67 which is a marker of cell division we have this amount of ki67 in     
1:09:06     
the cell over time and we can look at like the amount of this over time relative to cell fa uh a certain phase     
1:09:14     
in cell cycle so the g2m transition in cell cycle which is between 24 and 32     
1:09:21     
hours we can see ki67 concentration rais Rises threshold     
1:09:26     
during that period and so we can say that when we have those dynamics that we     
1:09:31     
will have a division event so we can actually calibrator division events to actual events in the molecular millu and     
1:09:39     
we can link that to cell cycle and we can link that to these population size based models because we can generalize     
1:09:47     
this across many cells or we can alternatively model ki67 is a stochastic     
1:09:54     
process so AI 67 concentrations might shift over time depending on where the     
1:10:00     
cell is in the phase space um so we can do all this you know this is nice     
1:10:07     
modeling work um you can also predict the cells probability for division using     
1:10:12     
the number of cells of each type within radius R so in other words if we have a lot of cells of a certain type within     
1:10:19     
radius R we can predict the cell's probability for division so is it being     
1:10:25     
crowed it out is it something that is in you know are there cells nearby that are     
1:10:30     
inhibiting its division or if there no cells nearby is that facilitating is     
1:10:36     
division and so we can actually look at these neighborhoods and we can get counts and we can predict probability of     
1:10:43     
division in that way uh this whole technique osdr is     
1:10:49     
based on a probabilistic rate equation so this is where we have a differential differential equation that has a couple     
1:10:55     
of parts so the first one are deterministic rate equations so this is a differential equation which has     
1:11:04     
divisions minus deaths so we have divisions which are births minus deaths     
1:11:09     
and the balance of that is the net growth of the population or the     
1:11:16     
deterministic rate at which the population expands the second differential equation     
1:11:21     
is a stochastic equation which is where we have this change in time we have the     
1:11:29     
number of division and death probabilities within a tissue so we uh     
1:11:35     
basically sum these up in uh for the entire tissue so we take each     
1:11:41     
neighborhood we take the probability of division and death and we then sum that and get some     
1:11:50     
sort of amount of net growth over time or net uh reduction over     
1:12:00     
time a deterministic ordinary differential equation can be derived from the stochastic models using the     
1:12:06     
expected direction of change so this is with respect to the phase portrait so we     
1:12:12     
can actually go from the static image to the differential     
1:12:18     
equations and these are ordinary differential equations and we can take that information and determine a     
1:12:25     
direction of changeed within the phase portraits so again our phase portrait example here is fiber blast density     
1:12:31     
versus macras density and we know that like when fiber blast density is at a certain density maccrage density is a     
1:12:39     
certain density there's going to be a certain expected Behavior so we kind of figure out where that is in the phase     
1:12:45     
space and from that we can determine this direction of change which is basically the slope of the change     
1:12:52     
between two time points     
1:12:58     
and then they say their method osdr accurately reconstructs Dynamics from simulations of known dynamical circuits     
1:13:06     
so you can look at different uh circuits so for example if we have one stable point in in a population of two cell     
1:13:15     
types A and B and there's a density of each we can know kind of what this one     
1:13:21     
stable Point uh phase portrait looks like we can also infer face portraits     
1:13:27     
from that um so we can actually measure it directly or we can infer it and they actually match up quite well we can look     
1:13:34     
at this for two stable points as well where you have multiple stable points you can do this for three stable     
1:13:42     
points and for four stable points and of course some of these points are not stable so you can look at     
1:13:49     
unstable and semi-stable States as well and it seems to work pretty well well it     
1:13:55     
seems to you know kind of replicate the Dynamics and so with respect to the hot     
1:14:02     
fibrosis and cold fibrosis they've they refer to these in     
1:14:07     
vitro experiments which established hot fibrosis and cold fibrosis fixed points     
1:14:13     
so they're looking at fiber blasts in a Cell culture so there's a number of     
1:14:18     
fiber blasts in a Cell culture a well and then you have macrofagos which are     
1:14:25     
wow so you have this macro fages versus fiber blast comparison so when the number of macras is low you have what     
1:14:33     
they call cold fibrosis which is where fi there you know it can be from very few fiber blasts to very many fiber     
1:14:39     
blasts and you can have this mode of cold fibrosis then this mode of hot fibrosis where there are a lot of fiber     
1:14:47     
blasts but also a lot of macro fages and so this is we're getting this growth in     
1:14:52     
in um basically in fiber blasts going from very few to very many fiber     
1:14:57     
blasts without macres so it's basically fiber blasts dividing without     
1:15:05     
very much um death or consumption in the case of when there a lot of macras is     
1:15:11     
you have a lot of fiber blast being produced but a lot of fiber blast being consumed or dying off so the raids are     
1:15:19     
you know uh well it's R independent it's really about like how many cells are     
1:15:25     
dying versus how many cells are just dividing and have a lot of division with a lot of death versus just a lot of     
1:15:31     
division and that's kind of the point that you have these fixed points that are stable and you know it just depends     
1:15:38     
on different aspects of macras growth as well fa Blass growth and the conditions     
1:15:45     
under which they find themselves which I think is kind of summarized in this point here where fiber blast and macras     
1:15:52     
is communicate bya growth factor so the growth factors     
1:16:09     
also and so we see some more phase portraits of fiber blast and maccrage     
1:16:15     
Dynamics um and we see these phase portraits so we see actually in DNA we     
1:16:20     
see this ki67 above threshold for fiber blast and for maccrage so again this is where you     
1:16:27     
get a division event where your ki67 if you're above threshold for KI 67     
1:16:33     
you get a division event so you can see that when you have this fraction of k67 above     
1:16:40     
threshold um you know there's a change in terms of how many fiber what the     
1:16:45     
fiber blast density is in the macro page density so you can see that this fits into these different modes of hot and     
1:16:52     
cold fibrosis and so you replicate this for T cells and B cells they interact in     
1:16:58     
the cancer micro environment and you get these face portraits for cancer micro environments as well where you look at     
1:17:04     
this k6 ki67 where if it's above threshold you get Division if it's a below threshold     
1:17:11     
you don't get Division and so you see this for T cells versus B cells okay     
1:17:16     
that was the that was a nice paper to show that like you can take a single     
1:17:22     
input and predict output and in in the first paper that we reviewed today uh     
1:17:28     
they talked about nonlinear outputs so for example if you have an input to a     
1:17:33     
process that input is not you know the components leading to the output are not     
1:17:39     
always stable and in sometimes in some cases like in in W Genesis it's kind of     
1:17:44     
hard to predict an output given the input U they were talking more about like if you're reading out information     
1:17:51     
from the genome how that's not a linear process how you might have something that like a     
1:17:57     
gene for something or something encoded in the Gene and it either gets degraded as it gets expressed and incorporated     
1:18:05     
into a tissue morphology or it you know it interacts with so many things that     
1:18:10     
it's early recognizable so you know there these different aspects but in the case of the last paper we saw that you     
1:18:17     
can use differential equations to predict some of these interactions and to understand that system a little bit     
1:18:24     
better so when we think about holism you know we not only have to think about sort of this fundamental problem of     
1:18:32     
interactivity but also of you know the tools that we bring to bear sometimes we     
1:18:37     
can bring to bear differential equations and that's enough to describe you know some of these     
1:18:43     
phenomena in terms of phase spaces phase PS uh but in this Cas this paper they're     
1:18:49     
talking about tissue biology in search of a new paradigm so this is annual reviews of     
1:18:56     
cell developmental biology and the abstract of this paper     
1:19:03     
reads animal tissues are made up of multiple cell types that are increasingly well characterized yet our     
1:19:10     
understanding of the core principles of govern tissue organization is still incomplete this is in part because many     
1:19:16     
observable tissue characteristics such as cellular composition and spatial patterns are emergent     
1:19:23     
properties and as such that cannot be explained through the knowledge of individual cells andone so again we have     
1:19:29     
to have that population perspective we have to have that tissue level perspective to really understand the     
1:19:36     
different aspects of in this case tissue organization but its Origins come from     
1:19:43     
from Genesis here we propos a complex systems theory perspective to address this     
1:19:49     
fundamental Gap in our understanding tissue we introduced the concept of cell categories     
1:19:55     
which is based on Cell relations rather than cell identity so this is interesting that they're doing it     
1:20:00     
different way kind of approaching this from a different way they're not thinking about what the cell is or what     
1:20:05     
it'll become they're thinking about the interactions and of course that's kind of what we do in uh that's the one of     
1:20:12     
the primary concerns pism is this interactivity so this is an interesting new think based on these Notions we then     
1:20:20     
discuss common principles of tissue modularity introducing composition structure and functional tissue     
1:20:27     
modules cell diversity and cell relations provide a basis for a new perspective on the underlying principle     
1:20:33     
tissue organization in health and disease so in this journal they have a     
1:20:39     
table of contents so they talk about tissue organization through animal Evolution cell types and evolution and     
1:20:47     
then cell categories based on Cell relations so they propose a number of different types of cell relations they     
1:20:55     
propose the primary supportive relation the complimentarity relation the     
1:21:00     
instructive relation the supplier consumer relation a hierarchical relation and the mutual exclusivity     
1:21:10     
relation so this is where you know we have these sort of trophic goals instead     
1:21:15     
of thinking about what the cell is if it's a fiber blast or a maccrage we     
1:21:21     
think about the relationship between and of course not not all fiber blasts will have the same relation to each other nor     
1:21:29     
to all macres and vice versa so we can take these relation and break it down we     
1:21:34     
think about those spatial maps that we saw phase portraits we think about this across space you know there's variation     
1:21:42     
across space there these different relations maybe that are more uh     
1:21:47     
relevant to density than to subs or more relevant to isolation than types so this     
1:21:56     
is uh you know a way to think about this very differently than cell types of     
1:22:02     
course cell types might be it might be conditional in cell type so you might have compliment complimentarity     
1:22:09     
relations that are different between fiberblast and macro phages than you would find within and between fiber     
1:22:16     
blasts so it's it's an interesting way to kind of look at this um     
1:22:26     
and so then then they go on to talk about the modular organization of tissues talking about minimal tissue     
1:22:32     
units and higher order modules minimal tissue units are groups of tissues that are basically you know like we talk     
1:22:39     
about these sheets of uh epithelial cells or Fiber blasts and that's sort of     
1:22:46     
like a basic level of minimal tissue unit and then the higher order module is     
1:22:52     
something where you get these compositional structure on functional tissue modules basically where you get     
1:22:59     
different types of cells in a tissue different structures and different functions so this is an interesting     
1:23:05     
figure two modes of cell type diversification and evolution so this shows two different     
1:23:10     
ways in which you get new cell types in evolution across     
1:23:15     
species and so the first example is induced constitutive     
1:23:22     
transition where you might get the origin function and this is where you get a new     
1:23:29     
cell type where there's an set of environmental interactions there might be an     
1:23:34     
irreversible transition from one cell type to another or there might be genetic assination you have thing you     
1:23:42     
know where you have a convergence of phenotype given different genetic     
1:23:48     
backgrounds uh then you have tradeoff resolution by division of label so in this case you're also getting a new     
1:23:54     
function this is not induced but constitutive so in the first example we     
1:23:59     
had ways of inducing uh different types of new F new cell     
1:24:05     
types in this other model we have the cell type diversification the     
1:24:10     
introduction of new cell types as the origin of a new function and an increase     
1:24:15     
in functional complexity so we have sort of new cell types constructed to sort of     
1:24:22     
form more diverse tissues um you might think of like     
1:24:28     
macras as being part of them in relation to fiber blasts where fiber blasts are kind of     
1:24:34     
pushed you know sometimes you're trying to heal a wound you want to produce a lot of fiber blast but you also want a     
1:24:40     
lot of those fiber blast to eventually you know die off and so you need macro     
1:24:47     
pages to sort of take care of that uh you know you can think of different ways     
1:24:53     
in which that works but the more functional complexity you have the more cell types you     
1:24:58     
need um then you have this Continuum A variation due to tradeoffs so this is     
1:25:05     
where you have different functional tradeoffs and you have different numbers     
1:25:11     
of cells or different cell types depending on you know what types of     
1:25:16     
activities functional activities adaptive activities the organism is     
1:25:22     
engaging and then of course you can have the segregation of gene expression programs where you have specific genetic     
1:25:30     
modules for specific cell types or to Define different cell types and this is     
1:25:35     
based on an increase in morphological complexity so first part is functional complexity where you have a change in     
1:25:41     
functional complexity or an increase in an organism and then you have morphological complexity increase where     
1:25:49     
you get diversification of cell types in morphology due to different types of     
1:25:54     
gene expression programs that specialized or different tradeoffs that     
1:26:00     
need in evolution so this these are the two different states here or ways of     
1:26:06     
thinking about tissue organization so on the uh left we have this uh cell type     
1:26:14     
Paradigm so this is where everything is based on cell type you have your different cell types you have your     
1:26:19     
epithelial cells your fiber blasts some other types of fiber blasts     
1:26:24     
and maybe some phages in there where they're all kind of in relation to one another it maybe in layers and so they     
1:26:32     
might be doing thing they might have a specialized role in this sort of uh     
1:26:39     
ecosystem of of tissue um but you know they're all based it's all based on the     
1:26:44     
cell type so you know you maybe there's some process that involves multiple cell     
1:26:51     
types and you know they have their own orties uh the second is celf relation     
1:26:58     
parm so this is where we have the same set of cell in the same organization but     
1:27:04     
we're interested in the relationships between say like fi L and pages so we     
1:27:09     
can actually look at that relationship and say that's a relationship that counts there's in fact maybe a different     
1:27:17     
way to describe this relationship where instead of you know maybe cell type c     
1:27:22     
and cell type D being the same layer and they're digitated with each other they're actually paired off as different     
1:27:30     
types of you know cells that rely each other so you can think of this as relational instead of typ     
1:27:37     
typological and of course you can bring things like relational biology to Bear into this as     
1:27:42     
well so then that's to these different types of relations that they propose and of course you could map those onto this     
1:27:49     
type of organizational scheme finally we go over the cell relation table so this is kind of laying     
1:27:56     
out the different types of relational you know this different sets of relations     
1:28:04     
that they proposing for cell uh for understanding cells and tissues instead     
1:28:09     
of thinking about this typologically we're thinking about this functionally in terms of     
1:28:15     
relations so for number for letter A we have the primary supportive relation     
1:28:21     
where function a supports function B or function B is supported by function a     
1:28:28     
and function B of course is the primary function function a is a support of function so one cell supports another     
1:28:35     
cell they both have to be there the second is the comparity relation where     
1:28:41     
each cell type contributes to a common function and so the oals need to be in that tissue to produce that output     
1:28:49     
function C is the instructive relation where you have enal sensor cell but     
1:28:56     
sensing the environment s sending a signal to an affector cell and then that     
1:29:02     
gives that affector cell the ability to uh differentiate into different types of     
1:29:07     
cells so there are these cell fate choices that are made that are contingent on a signal sent by a sensor     
1:29:15     
cell sensing then there's the supplier consumer relation where you have a     
1:29:21     
supplier cell it's providing a growth factor to a consumer cell or sometimes     
1:29:27     
the consu you know both cells are both supplier and consumer cells so they're providing each other withow growth     
1:29:33     
factor but in this case they both dependent on each one's one is a CLE dependency relationship and the other is     
1:29:40     
where there's a codependency relationship and they call this asymmetric and     
1:29:46     
symmetric there's the hierarch relation where you have say like a T Cell that's     
1:29:51     
relying on a mass cell the mass cells L on the udip aite and the smooth muscle cell and then you know or I guess it     
1:29:59     
would be the other way around where the t- cell the mass cell depend on the te- cell the smooth muscle cell and adipoid     
1:30:06     
are dependent on the mass cell the stromal cell the capillary endothelium     
1:30:12     
the maccrage and the epithelial cells are all dependent on the adipocytes and smooth muscle cells so you have this     
1:30:19     
hierarchical relation of you know where you can actually take the cell take the     
1:30:24     
tissue and sell that these cell types are dependent on others and so you can actually think then about how this     
1:30:31     
tissue might have evolved what needed to be there first what needed to be there     
1:30:37     
after the first cells and how did the tissue come to be finally they talk about the mutual     
1:30:43     
exclusivity function the mutual exclusivity relation which is where you     
1:30:49     
have cell type A or cell type B so you have so you can't have one without the     
1:30:55     
other or you can't have one in the context of the other so I guess it's like you can     
1:31:02     
either have cell type A or cell type B but you can't have both in the tissue     
1:31:07     
for various reasons so if you look at these different relations you see that they're     
1:31:13     
kind of grouped into categories cell relation a and cell relation B have this functional dependency cell relation C     
1:31:20     
has this information dependency cell relation D cell relation e and some relation F have this existential     
1:31:28     
dependency and so it's interesting you know how you have this sort of thinking in terms of primarily function thinking     
1:31:36     
in primarily in terms of information or thinking primarily in terms of survival     
1:31:41     
or conditional surv so that's an interesting way to think about tissue because it takes what we know about self     
1:31:48     
Fates and expands this to like you know given certain cell Fates or other cell     
1:31:54     
Fates necessary or their cell types necessary and what is their role functionally and     
1:32:01     
structurally so that's all for this uh group of papers hope you learned some
