## TRANSCRIPT

0:00     
hello welcome to our meeting for this week um because of some technical issues     
0:05     
I'm doing the update by myself so uh welcome I'm going to cover some tools     
0:12     
that I learned about at the open room annual meeting I'm also going to go over several papers that we uh have in our     
0:20     
queue so let's get started so this last week was the open or manual meeting the     
0:26     
dorm presentation went successfully we got a lot of good feedback people are     
0:32     
positive about our work which is good this is a meeting of the word of Trustees of the organization plus all of     
0:39     
our senior contributors so we have uh some interest in our work     
0:45     
now uh there is another group there's other two groups that are really doing a lot of work in open home right now our     
0:52     
group and por Gleason's group por gleon is based I think at the University of     
0:57     
College London and he's been doing a lot of work with uh the     
1:03     
connectome and getting together a lot of secondary data so he's involved with     
1:11     
incf and their different initiatives for data sharing and then he's also been     
1:18     
working with someone on building a large language model for the sea elegans     
1:25     
literature so the first tool here is the seans connecto tool box this is from por     
1:31     
EGS group uh they've been working on bringing together all of the major data sets for the celegans     
1:38     
connectome we have the Gap Junction data sets we have the synaptic data sets we     
1:44     
have the neuropeptidergic data sets we also have the hermaphrodite data set and     
1:50     
the male specific data set so this is the tool here open.org connectome     
1:56     
toolbox you have an overview which is here this is just talking about sort of     
2:03     
example plots so when you have a a connectome data set how do you plot it I think that's a interesting problem in     
2:10     
light of some of the the discussions we've talked about about plotting network data out so you know at the top     
2:17     
we have sort of our hairball slash uh and it's not really a     
2:22     
force directed graph but it's in some sort of hairball slforce directed     
2:28     
context you have Hive plots on the lower right you have a circuit diagram in the     
2:34     
lower left and you have sort of this heat map at the upper right so it just     
2:40     
shows different ways to plot these interactions these IJ     
2:46     
interactions and so that's uh I think it's an important thing to think about especially with connectomes because     
2:53     
you're trying to communicate two things you're trying to communicate the interactions between neurons and you're     
2:59     
trying to communicate the function perhaps you're also trying to communicate something about the     
3:04     
structure and so if you have different modules different circuits if they     
3:09     
interact with cross talk if they're separate or Inseparable then those are all things good things to know from the     
3:16     
data some plots are easy to interpret and some are not so this is an ongoing     
3:23     
area of Interest then we have these historical Publications so this is a nice chart     
3:29     
this actually uh goes into sort of the different categories of uh     
3:38     
connectum and you know data sets and then the Publications associated with them so we have the anatomical chemical     
3:46     
electrical networks we have theep extra synaptic networks we have the gene     
3:51     
protein expression networks and we have the functional networks on the right uh so we start with Albertson and     
3:59     
Thomasson 1976 we moved to White     
4:04     
1986 Durban 1987 Holland Russell     
4:10     
1991 that's all in the uh anatomical chemical electrical category so this is     
4:16     
where they're mapping out the basic features of the connecto the cells the     
4:21     
connections Etc uh then we have in gene protein     
4:26     
expression we have altun at all 2009 goes down to Taylor yamini and beats in     
4:35     
2021 and 2023 so that's actually quite recent     
4:41     
work and a lot of these other works here uh on improving the connectome are     
4:46     
also uh from 2010 on so uh varshney 2011 cook     
4:53     
2019 this is the gap Junction uh connectum of the hermaphrodite in the     
5:00     
connectum of the male respectively then you have cook 2020 Britain 2021 and Whit     
5:06     
Fleet 2021 we have um additional extra synaptic networks or     
5:14     
actually we start with our extra synaptic networks in 2013 Serrano SE     
5:20     
Carrera tendal Atlas and Hobert Bentley and roli Sanchez so this is actually     
5:26     
where we're talking about the peptidergic networks uh then in functional networks we     
5:36     
have uh these four papers so you know they're all from 2020 on and so they     
5:43     
have some nice visualizations of of the literature of of the data its providance     
5:50     
what it is and so forth so I leave you to explore that on your own um we have     
5:58     
access actually to the data sets here uh we actually have I think you can get to     
6:04     
the actual data set here okay so this is like a stub for the data set we actually     
6:10     
have uh XLS files or Excel files I think these are I think they they're also data     
6:17     
readers and things like that that were available from different     
6:23     
sources uh yeah so a lot of these are stored in Excel files and there's a reader here     
6:30     
as well so if you want to play with some of these connect HS they're all connect they're all collected here in the     
6:37     
connectome toolbox okay so that's the connecto toolbox now we have what they're calling     
6:44     
openworm Ai and so the idea here is to take a large language     
6:51     
model and you know this runs on this runs actually on hugging face the URL is     
6:57     
open or. so you can reach it through there but it's actually run off of hugging face and it's hosted there with     
7:04     
a number I guess a number of larger language models that you can use that are trained for this purpose which is     
7:12     
ask individual large language models questions about celegans you ask them about something from the seans     
7:17     
literature and you get an answer coming out so this is the individual llm     
7:23     
stub uh like if if I ask for example what is the primary role of the celegans neuron AV     
7:31     
vbl and I want to use Gemini and the temperature which is a     
7:36     
parameter is 0.1 and I can you can't really check     
7:41     
uncheck this box but only answer questions related to cgans because sometimes you can get things that are     
7:47     
not related to celegans so if we submit     
7:52     
this we get an answer um the primary role of the seans are on avbl     
7:59     
is to mediate thermotaxis Behavior which is the movement of the worm towards or     
8:05     
away from a thermal stimulus so gives you the function of evbl I can also choose other larger     
8:12     
language models llama 2 let's try Lama 2 just just to see what     
8:18     
happens okay now that's a little bit different question or answer a great     
8:23     
question the primary role of the celegans neuron avbl known as a left is     
8:29     
the function as a sensory neuron involved in identifying and responding to a broad chemical chemosensory cues     
8:37     
particularly food sources such as bacteria and their Associated volatile     
8:42     
compounds uh vwfa specifically avbl neurons are responsible for detecting these chemical     
8:49     
signals through specific receptors on their surface and transmitting its information to other neurons in the     
8:55     
neural circuitry of seans to guide locomotory and then it stops so you know     
9:02     
you can see where you can find out a lot of different types of information a lot of things in parallel and then that's     
9:10     
kind of your answer okay this is a little bit different but it basically gives you the     
9:16     
same answer so this is you know playing around with large language models asking questions about cigan biology and I've     
9:23     
tried it with other questions and it works adequately there's this panel discussion     
9:30     
so this is where I guess you can actually have a panel of larger language     
9:35     
models so let's pick a oh I guess you can only do one to lead the panel so     
9:41     
let's have llama 2 lead the panel and then Gemini GPT     
9:47     
40 uh GPT 4 and coh here let's have them so it's like you have this leader of the     
9:54     
panel and then the attendees and I guess they're all going to give sort of some sort of consensus this answer so let's     
10:01     
submit that we'll see what happens it might     
10:06     
take a while to do okay so it gives me a number of answers uh so llama 2 will say based on     
10:14     
our expert panelist responses it appears there in consensus that the typical     
10:19     
length of C elegans worms is approximately 1 mm uh I don't know how that relates to     
10:25     
our question here the typical length of the worm C elegans so this is the     
10:30     
question they're trying to answer however I noticed a slight inconsistency in Dr Geminis and GPD 4's     
10:36     
responses which do not explicitly state they're referring to adult worms so there's a lot of sort of semantics and     
10:44     
semantic guessing here uh which is typical of a large language model so yeah Gemini says 1     
10:51     
millimeter the typical length of the adult seans worm is about 1 mm GPT 40     
10:57     
the typical length of the nematode C elegans is about 1 mm and you know so just it's just kind     
11:04     
of giving answers that are kind of more or less specific to say     
11:10     
like an adult or you know some no you know variations in the name so it's it's     
11:18     
just interesting I think if I asked it a more complex question it might be able to give me a more interesting     
11:25     
answer okay so now you can actually search for publication Sean Locomotion     
11:31     
if I want to get 10 results on Sean Locomotion then I have this these search     
11:38     
results basically goes through the literature it's trained on and pulls out literature and it gives like a sort of     
11:44     
an annotated uh literature review where it says you     
11:50     
know uh it gives some description of the article and how relevant it is to     
11:57     
you structured data so you can actually query different     
12:02     
types of sources this is the worm neuro Atlas which neuron would you like to     
12:08     
know more about avbl so if we look at the answer here we     
12:13     
see that it gives uh information about neuron avbl from worm neuro Atlas and it     
12:19     
looks like it's formatted data uh just giving information about the anatomical     
12:26     
connectum its inputs it it's uh putative inputs in the     
12:31     
neurop peptidergic connectome and then the anatomical connectome and then it's     
12:36     
giving uh outputs as well so this is really interesting I hadn't realized     
12:42     
that there was this kind of structured data aspect to it the open armm Corpus     
12:48     
of course it will give you a list of specific cells for say like the ferx so     
12:53     
if I say what are the main types of neurons in the seal against ferx it gives me these neurons     
12:59     
and uh you know it has it sites it's where it's growing the data from the worm at was     
13:06     
handbook so this this benefits from the vast number of Community Resources we     
13:12     
have for SE elegans in the community we have all sorts of     
13:18     
resources shared data sets and other things that we can actually put into the     
13:24     
larg language model the query so this isn't coming from nowhere this isn't just coming from the literature the     
13:30     
community has a lot of tools that they can use the problem of course is that they're all kind of uh     
13:37     
siloed and sometimes it's hard to know kind of how to query them or how to     
13:43     
access them and so I think this gives you like a good way to access them all at     
13:49     
once uh you can run different models you can run a cell model so this is     
13:55     
a a muscle model     
14:02     
and that muscle model gives you an output so it actually gives you a graph I can run a muscle model with     
14:10     
parameter 4.1 PA and it gives me this finished simulation so it's     
14:15     
actually you know I don't know what the inject this is injecting something into the     
14:20     
muscle uh there's a Neuron model where you have an injection level as well you can run the Neuron     
14:26     
model so it's running their models that's doing this uh so it's giving you     
14:32     
like electrical activity in SI units over time all right now let's go to some     
14:40     
papers and there are two papers here on what I'm calling developmental trajectories as inner product space and     
14:47     
this is where these in these two papers they basically treat developmental     
14:53     
trajectories as sort of this inner product space and I'll describe what that is in a minute so when we talk     
15:00     
about an inner product space usually what we mean is like we have these two functions that     
15:06     
intersect and we have this angle at which they kind of form once they cross     
15:12     
and go in other directions so this is function a and function [Music]     
15:18     
B all right and this might be a graph with time so you     
15:23     
have and then some variable here and so this is the inner product space     
15:31     
here and you're like what does this have to do with development this looks like something that's     
15:37     
mathematical well the idea being that in development you have these trajectories so if you think about like the uh the     
15:46     
epigenetic landscape of wadington you have this landscape that kind of comes down like this and you have these     
15:54     
what he calls a c uh he calls this process canalization     
16:00     
or developmental channels basically these are developmental trajectories that sort of get reinforced     
16:07     
by the environment by genes interacting genes being     
16:12     
expressed and you know the sort of the the body plan of the organism and so forth so you start up here at the top     
16:19     
which is like the single cell embryo and you go down this path and you end up     
16:25     
with these sort of switch points or decision points where you go down different     
16:31     
channels and you make your way down to the bottom where you end up in a unique     
16:37     
location sort of at the end of development so this is     
16:42     
adulthood or I guess     
16:48     
maturity and this is the juvenile here juvenile stage or the embryonic     
17:00     
and so you can see that there are these unique Developmental Pathways that kind of result from this process you have     
17:06     
these branching points and you have these Developmental Pathways so these developmental     
17:13     
trajectories are sort of you know you can plot them out like this you might     
17:18     
have one that goes like this one that goes like this one that might go like     
17:24     
this and so you can see where you get these inner product spaces     
17:30     
in different places so you can use these to sort of     
17:35     
understand uh the Divergence of development so if you have like     
17:40     
something like this trajectory here versus this trajectory here that I've     
17:46     
kind of defined in these arrows uh those are different those are going to maybe not have an in product     
17:53     
space they might diverge like this they'll have an inner product space at the origin point so kind of like right     
18:01     
here but not later on as opposed to maybe a trajectory that intersects with     
18:08     
this see we have a trajectory that kind of comes like this if you have polyphenisms often times you'll have     
18:15     
alternate paths for the same organism depending on its environmental context     
18:21     
so if we have a polyphenic organism we might have something that maybe goes down this route during normal     
18:27     
environmental times and this route during like stressful environmental times and that gives us     
18:32     
a uh inner product space here so these are Developmental     
18:43     
trajectories and then we have an inner product     
18:53     
space and this could be due you know this is we'll see in the papers that they use these for different reasons you     
19:00     
know it could be like morphological it could be life history     
19:07     
it could be even gene expression a lot of times gene expression     
19:12     
patterns people will use this to interpret like NextGen sequencing data and things like that where they can form     
19:19     
a trajectory or a vector of some type and then two vectors have a crossing     
19:24     
point and then they have an inner product space you can determine that you you can Define that you can quantify a     
19:30     
lot of these changes in uh gene expression so but of course it does     
19:35     
don't have a developmental context what I'm saying is this is the broader developmental context of all     
19:42     
this so let's start with this paper here in biosystems uh this is called the indispensable role of time and     
19:48     
autonomous development This Is by William Leto he's from New York Institute of     
19:54     
Technology and I actually reviewed this paper uh for publication I thought it was good ah heads you know we had a     
20:01     
conversation back and forth about what we were talking about here and you know nailing down all the theoretical points     
20:07     
but I think it's a very good paper uh especially for some of these issues with     
20:13     
respect to quantifying development so let's see if I can read the abstract it's a little hard to read but um the     
20:20     
abstract reads advances and single cell analysis have led to a picture of development largely in agreement with     
20:27     
watton's uh epigenetic landscape in which a sulate is determined by its     
20:32     
Basin of Attraction and a high-dimensional gene expression space so again those uh Landscapes I was     
20:39     
showing you can think of those as sort of like dynamical systems the the channels or the the canalization process     
20:47     
it goes down these sort of you know long thin retractor basins where you know     
20:53     
it's easy to kind of find a path forward once you make a a bifurcation     
20:59     
and then you end up at the bottom so it's kind of like if you roll a ball down a hill and it follows a path of     
21:05     
leas resistance and you keep rolling balls down the hill it tends to form a little Groove into the ground and it it     
21:11     
makes it easier to go down that way the next time and then pretty soon your ball     
21:17     
gets buffered into that Canal or that channel and it always goes down that     
21:22     
route unless it's really forced out by some sort of energy and so same thing     
21:28     
with development we usually have these uh canalization is a process of buffering it's you know phenotypic     
21:35     
buffering where the phenotype is sort of kept within the constraints of that     
21:41     
particular path and it requires a maybe something like a macr mutation or some other type     
21:48     
of perturbation to drive it out of the of that trajectory so that's kind of     
21:55     
where we're going with thinking about watton's landscape and these spaces of attraction in this sort of high     
22:01     
dimensional Gene Space wadington Landscape yeah conceptual gaps remain as to how a single progator can     
22:07     
simultaneously generate multiple end points and why time should be required of the process at all we propose a     
22:13     
theoretical model based on hamiltonian mechanics of n-dimensional rotational     
22:18     
motion which resolves these paradoxes we deriv the result that systems which     
22:24     
become different for themselves over time must initially move in a direction not towards their ultimate end points     
22:30     
and propos that this process of resolving ambiguity can be Quantified in an information theoretic sense by the     
22:36     
volumes subtended in gene expression Space by the trajectories taken by the system towards its end point so     
22:42     
basically he's describing the process I just diagrammed out it was a little bit     
22:47     
less uh ver Bose and what I showed you was kind of like the mathematical     
22:52     
interpretation and then the developmental context we discussed the implication of this Theory for the     
22:58     
analysis of single cell gene expression data in studies of development so I've     
23:04     
highlighted a couple of points here uh the first one is that he talks about the     
23:10     
landscape model and he's kind of getting at the sort of maybe limitations of the     
23:15     
landscape model and the uh sort of the benefits of the landscape model and so     
23:21     
he's so he's been talking about the landscape model and then he gets into this discussion here     
23:29     
so it all right so furthermore the landscape model has even been refined to capture some aspects of indeterminancy     
23:36     
and simultaneity Wang and colleagues first introduced the so-called curl flux     
23:42     
Force to account for limit cycles and non-equilibrium non-gradient behavior so these are things that are     
23:49     
non-gradient non-e equilibrium so landscape models you know if you you see the connection with machine learning     
23:56     
we're dealing with a gradient basically and things move down a gradient and down the landscape as they move down to the     
24:03     
bottom they sort of minimize their developmental potential or you     
24:09     
know you start at the top you can go in any direction you want you can take any     
24:14     
set of paths you want but as you get further and further down the developmental landscape or the     
24:20     
epigenetic landscape your sort of your decisions have already been made your potential is     
24:27     
less because you've taken all the available paths and you end up at the bottom where you have maturity and you     
24:33     
have that phenotype that you're sort of limited to you're buffered into that that attractor     
24:40     
Basin and it's very specific for your genetic and environmental     
24:45     
context so that's that's kind of what they're talking they're actually talking here about some of these issues with     
24:52     
respect to non-equilibrium non- gradient behavior um these and other authors also     
24:57     
rejected the idea surfaces could be static so this is where the surface is dynamic it's always     
25:03     
changing uh you know depending on differences in the genome depending on differences in the     
25:10     
environment by their demonstration of how tuning parameters in a couple differential equations governing the     
25:16     
mutual expression levels of various genes May generate bifurcations in the landscape so again the wadington     
25:23     
landscape it's all underlied by gene expression like different coordinated     
25:28     
gene expression you know and and there's a picture which I think I've shown in a     
25:34     
previous meeting where waddington's drawn the landscape and then he's drawn like a network of genes underneath and     
25:41     
back in when he wrote the book they didn't really understand genetic regulatory networks so it was interesting that he had this conception     
25:48     
of it but basically it's controlled by genetic regulatory networks underneath the surface and that's what determines     
25:55     
these switching events these bifurcations and how these things are buffered into     
26:00     
these non-equilibrium retractors this is where we have sometimes we have     
26:05     
progenitor states that become unstable sometimes you know we get this buffering there's a lot of sort of a rle     
26:12     
of stochastic fluctuation and gene expression um and this is something     
26:18     
that's also captured in this model um and this is an interesting     
26:23     
point here uh so the elucidation of the RO stochastic fluctuations and gen expression play a bias in biasing a pair     
26:31     
of dividing cells into alternate Fates was another important experimental and theoretical advance for it provided a     
26:37     
non-teleological mechanism by which such bifurcations could arise so this is     
26:42     
interesting if we go back to our uh landscape we saw that there were these     
26:48     
bifurcations and you might ask how did they happen and I said well it's coordinated gene expression it's the     
26:54     
role of environment but it still doesn't explain why the these are sort of adaptive I mean why they should be     
27:00     
adaptive I guess if you generate a embryo that isn't functional that kind     
27:06     
of selects for things that are functional but also that we you know there's a sort of Temptation to say well     
27:12     
it's a teologicos that development unfolds by some sort of uh you know blueprint or     
27:19     
plan and that's actually not what we we know that not necessarily to be true so how do we get these bifurcations in the     
27:26     
landscape how do we get to phenotypes that are alternate phenotypes but also are functional     
27:32     
phenotypes and so you know one way to do that is to of course have these     
27:38     
stochastic fluctuations in gene expression which allow for cells to     
27:43     
bifurcate into alternate Fates it doesn't require any sort of architecture     
27:48     
or blueprint it just requires this sort of noise with you know Gene regulatory     
27:54     
networks with interactions with the environment and so on     
28:00     
all right so more recently the introduction of RNA velocity as a method for tracking the     
28:07     
instantaneous Direction single cells take in a gene expression space has led     
28:13     
to a new understanding of the Wasington landscape in terms of vector field so this is where we get into these inner     
28:18     
product spaces so we think about RNA velocity as sort of like what is what     
28:27     
what is the RNA complement over time and how fast is it be are things     
28:33     
being expressed and so we can actually measure all that as sort of this velocity     
28:39     
vector and so we want to know kind of what a gene expression space which is     
28:45     
abstracted from even from the uh epigenetic landscape because it's just     
28:51     
describing uh gene expression as a vector and then gene expression change over time as a vector     
28:58     
uh so we get this sort of velocity approach which is very similar to like a     
29:04     
velocity approach of motion where you're looking at sort of the position and then     
29:09     
the derivative position we have Vector fields which are these the velocity of     
29:17     
RNA and we have this uh where they have these inner Vector spaces because they     
29:25     
cross each other and so or they're different from each other and you can measure that difference in that way this     
29:32     
is an interesting approach um so people have used Vector Fields before course     
29:37     
and siga 2017 Schmid 2020 H 2022 and go     
29:43     
2023 in these studies there is shown a cleer flux in different directions for progenitors to differentiated sulfates     
29:51     
there's even evidence of cancer cells metastasized by both their reversing and recapitulating the developmental TR     
29:58     
trajectories taken by nearby cells of a related lineage so this is where uh you     
30:04     
know you're kind of they're either replicating the developmental trajectory     
30:09     
or reversing the developmental trajectory and these are cancer cells so these are cancer cells that are growing     
30:15     
much like a tissue with the types of cells that were hijacked by the cancer     
30:21     
um you know so we have they also have this sort of Developmental     
30:26     
trajectory um so so this is interesting uh and also the dedifferentiation     
30:31     
proceeds along the reverse of the paths taken during differentiation so sometimes cells     
30:37     
dedifferentiate you have a number of model systems where cells can go from say like a muscle cell or some type of     
30:44     
neuron and then reverse course and become like basically a stem cell when     
30:49     
we have cellular reprogramming we kind of do this T differentiation process it's a little bit different than what we     
30:55     
see in nature um but you have these cases where cells will dedifferentiate     
31:01     
and they'll just become like sort of multi- poent again basically they pop out of their     
31:08     
differentiated State and back into a multi- poent state and so it's interesting that they     
31:14     
follow the same sort of they they get buffered into the same sort of Developmental trajectory it's just that     
31:21     
you're running it in reverse collectively these results     
31:27     
suggest that tracks laid down in the landscape are well defined and stable     
31:32     
and more broadly that a combination of deterministic and stochastic Dynamics underly cell decision- making during     
31:39     
development so this is a combination of deterministic and stochastic Dynamics we have these deterministic Dynamics in     
31:46     
these in these canalization uh trajectories and then we have the stochastic component that can     
31:52     
pop them out of the trajectory it can make you know these bifurcations happen     
31:57     
it can also reverse course and result in dedifferentiation or something going     
32:03     
down the cancer pathway so it's really interesting uh so then you know he talks     
32:09     
a little bit about uh you know waddington's hypothesis of canalized     
32:15     
development um and then if correct this analogy would res so he talks about the     
32:20     
hypothesis of R velocity Vector fields which are effects not causes of changes     
32:26     
in gene expression so this isn't a cause of gene expression it's an effect so if     
32:31     
I look at the velocity Vector it's basically what's being expressed over time how fast it's being     
32:38     
expressed and so you're looking at the effects not the causes if correct this     
32:43     
analogy would resolve the Paradox in the wadington model forced Upon Us by the existence and uniqueness theorem of     
32:49     
dynamical systems theory namely the stipulation that a single progenitor cell have but a single fate allotted to     
32:57     
it by the integral curve traced from its starting point which are cells which     
33:02     
under the gradient model would not be permitted to start at the end despite their knowing the end at the     
33:09     
start this would under the new model only learn their ultimate end point upon achieving it similar Notions have been     
33:17     
recently expanded expressed that cells do not contain a blueprint or program in their DNA but rather a generative     
33:26     
autoencoder which is set of to adapt and constrain the several potentialities of the cell without being specified ahead     
33:32     
of time this is Mitchell and Cheney 2024 so this is what I was talking about that     
33:38     
you know there's this idea in developmental biology that there's this sort of blueprinter program that tells     
33:44     
the cell where where to go to the end point from the beginning and so the cell     
33:50     
knows kind of what it needs to do it just needs to get there now that of course invokes thology it invokes a lot     
33:58     
of sort of assumptions about what's going on in the cell what's going on in the DNA uh so but what he's saying is that     
34:05     
there's some sort of generative Auto encoder and so this is the paper Mitchell and     
34:10     
Cheney which is here and this is the genomic code The genome instantiates a     
34:16     
generative model of the organism this is actually this is a archive preprint this     
34:22     
is Kevin Mitchell is the first author and this is the genomic code this I think we went     
34:28     
through this uh preprint before but basically talk you know using the sort of     
34:35     
generative autoencoder variational autoencoder um metaphor to describe     
34:40     
what's going on in terms of gene expression and that combinatorial gene expression you see in watton's     
34:47     
landscape so again you know this might suffer from the same problems as we have     
34:52     
with the blueprint model namely we're using a metaphor that's sort of current     
34:59     
and you know even hip to describe what's going on in the cell when we don't maybe     
35:05     
actually know what's going on the cell so in a you know to recap in a     
35:11     
variational auto encoder you know you have this Laten space the Laten space     
35:16     
sort of describes all the potential changes and then the cell would reference that Laten space which I guess     
35:22     
would be the genome and all of its states of instance iation all its     
35:29     
possible states of instantiation and then it would select a subset of those to go forward with to     
35:35     
achieve what it needs to achieve it's a lot more plausible I suppose than the uh the blueprint model     
35:43     
which basically implies that everything is sort of pre-ordained okay     
35:58     
so the next paper so the next paper is this pnas     
36:04     
paper um utilizing developmental Dynamics for evolutionary prediction and     
36:09     
control so this relates to what we were talking about in the first paper you     
36:15     
know uh can we predict Evolution can we control Evolution well can we use Evol     
36:20     
the developmental Dynamics to do this so with this journal we have a significant     
36:26     
statement and an abstract read the significant statement first uh A variation in traits is the     
36:32     
raw material for evolution yet the complexity of development makes it daunting to predict control an engineer     
36:39     
evolution by manipulating the generation of phenotypic variation so that's what we mean by predicting and controlling an     
36:47     
evolution or engineering Evolution we want to manipulate the generation of phenotypic variation much like what we     
36:54     
want to do with the epigenetic landscape we want to start the single cell and move down to a very refined     
37:01     
phenotype by representing development as a dynamical system we build a bottomup     
37:06     
theoretical framework to predict the effects of genetic and environmental perturbations on the resulting traits we     
37:13     
find conditions for when different perturbations result in concordant changes in the traits and show how this     
37:20     
could be used for evolutionary production and to steer evolution in a preferred Direction so it's that's the     
37:26     
significance um and then the abstract says     
37:31     
understanding predicting and controlling the phenotypic consequences of genetic and environmental change is essential to     
37:37     
many areas of fundamental and Applied Biology so we want to be able to do this     
37:43     
understanding predicting and controlling aspect of this we don't want to just kind of watch it happen we don't want to     
37:50     
have this sort of process that's you know an opaque process we want to be     
37:57     
able to manipulate it in evolutionary biology the generative process of development is a major source of     
38:03     
organismal evolve ability that constrains or facilitates adaptive change by shaping the distribution of     
38:10     
phenotypic variation that selection can act upon so if we think about our you     
38:17     
know landscape again we have these trajectories that is sort of like you     
38:22     
know maybe like stripes on a distribution and we want to be able to     
38:29     
kind of shape the distribution in ways that is you know uh understandable     
38:37     
whereas sometimes if we're doing genetic engineering want to shape in ways it's functional and of course when we do     
38:43     
artificial selection like domestication or something like that we likewise get that sort of     
38:49     
effect while the complex interactions between genetic and environmental factors during development may appear to     
38:55     
make it impossible to infer the concept consequences of perturbations uh the persistent     
39:01     
observation is that many perturbations result in similar phenotypes and this indicates that there     
39:07     
is a logic to where what variation is generated so that kind of almost speaks to the the model of you know the the     
39:15     
variational autoencoder that it's you know there's this variation that's sort of latent but     
39:22     
then there's some structure to how it's you know put into practice in the world     
39:28     
here we show that a general representation of development as a dynamical system can reveal this logic     
39:33     
we build a framework that allows predicting the phenotypic effects of perturbations and conditions for when     
39:39     
the effects of perturbations of different Origins are concordant we find that this concordance is explained by     
39:44     
two genetic features of development namely namely the dynamical dependence of the phenotype on     
39:52     
itself and the fact that all perturbations must affect the developmental process to have an effect     
39:57     
on the phenotype so this is interesting because     
40:03     
you know we have of course in development     
40:08     
the the phenotype you know starts off at a single cell and we talked about how     
40:13     
you get these bifurcations and you increasingly as you go down the slope of the epigenetic     
40:21     
landscape you become more locked into certain phenotypes simply because you've already made this decision and you can     
40:27     
dedifferentiate your cells you can do that but you have this sort of     
40:33     
increasingly constrained landscape so phenotypes are dependent upon their sort     
40:40     
of Prior developmental history and then the fact that all     
40:45     
perturbations must affect the developmental process to have an effect on the phenotype so if it doesn't affect     
40:50     
the process itself it doesn't really show up in the phenotype     
40:57     
we apply our theoretical framework to classical models of development and show that it can be used to predict the evolutionary response to selection using     
41:05     
information of plasticity and accelerate evolution in a desired Direction the framework we     
41:12     
introduced provides a way to quantitatively Interchange perturbations opening an Avenue of perturbation design     
41:19     
to control the generation of variation so this is interesting um they     
41:25     
they kind of talk about this process that generates variation it's known as development     
41:31     
notoriously complex and nonlinear interactions between genes cells tissues and environmental factors during     
41:37     
development make it difficult to grasp the phenotypic consequences of genetic and environmental perturbations indeed     
41:45     
the diversity and complexity of Developmental systems could be feasible uh or could be taken as     
41:50     
evidence that a prior influence on the consequences of perturbations rarely will be feasible so     
41:57     
um you know we look at the diversity and complexity of development systems and we can't necessarily say that there's any     
42:04     
sort of like uh you know a prior influence or     
42:11     
blueprint orological foreing or any of that a pessimistic conclusion is     
42:17     
therefore that the best one could hope for is to demonstrate the generative processes and principle could impact     
42:24     
evolutionary trajectories while studies that demonstrate how development affects Evolution remain a collection of     
42:30     
idiosyncratic case studies this perception that generative processes are intrinsically     
42:37     
unpredictable and that selection is the only reliable force in evolution it's also reflected in biotechnology and     
42:43     
Medicine where attempts to direct evolutionary processes emphasize control over selective regimes rather than     
42:50     
control over generative processes so this is interesting     
42:55     
um I mean yeah we're we're going in the same direction in terms of generative processes and how they influence     
43:01     
phenotype so their methodology here is interesting but they essentially what happens is they end up with these     
43:07     
different Developmental trajectories and then of course by virtue of having these developmental trajectories that diverge     
43:15     
that intersect that you know have features of sort of this generative     
43:20     
process you get this sort of these these inner product spaces and this is the same sort of approach we a little bit     
43:27     
different approach than we saw in the last paper but the same in the same direction I think so this is figure one     
43:32     
from this paper um this is a general framework to study the phenotypic effects of perturbations of different     
43:40     
Origins so the the top image here it looks like two developmental trajectories that are diverging and you     
43:47     
can take a Time slice through that and like look at these other sort of graphical representations down here so     
43:55     
the the one at the top here or the dev Vel m al trajectories that go over time     
44:01     
and then you have this time slice here where these trajectories cross through the time slice and then you can see that     
44:07     
these uh trajectories the black and the uh purple one respectively are shown on     
44:14     
this graph with respect to the perturbations so on top which is this     
44:20     
large uh figure with the trajectories the reference developmental trajectory     
44:26     
which is is at the bottom which is the black one uh in black and the pert perturbed     
44:33     
trajectory uh in purple through developmental time T so the purple one is the perturbed one black one is the     
44:42     
reference trajectory the panel in the middle shows that at any given time and     
44:48     
this is the panel in the middle here the effect of the perturbation on the trajectory given by the sensitivity     
44:55     
vector which is this sensitivity Vector down here or actually this purple one I     
45:03     
believe um is a linear combination of The Columns of the Matrix a to the power     
45:10     
of1 that's the transpose M or A1 of the uh     
45:19     
reference so the three panels at the bottom show the sensitivity vectors for different perturbations at a given     
45:25     
developmental time and that these are linear combinations of The Columns of the same uh     
45:33     
Matrix uh so S1 and S2 which are these two are largely     
45:40     
aligned but S3 is not so you can see if we zoom in these two are largely aligned and     
45:48     
this one is not so you can see that you have this sort of intersection of these different     
45:54     
trajectories showing the forcing mechanism and showing how they sort     
45:59     
of uh operate at that time slce okay now they talk about actually     
46:05     
explicitly about these uh inner product spaces and how they're doing this in the     
46:11     
context of aligning the effects of perturbations of different Origins so we saw in the last image up here where we     
46:19     
have these intersections of the reference and the     
46:25     
perturbed developmental TR trajectory now they're going to talk about sort of the formalism and how we might     
46:32     
understand this in terms of these inner product spaces so we now use the formalism     
46:37     
introduced above to study the relationship between the phenotypic effects of perturbations of different     
46:43     
Origins given two developmental parameters we say that their effects are totally aligned that the associated     
46:50     
sensitivity vectors have an angle of zero meaning that the two perturbations     
46:55     
result in pH changes in exactly the same direction so we saw that there were different angles of     
47:02     
sensitivity uh where the two perturbations will go in the same direction more generally we say that     
47:09     
there is some degree of alignment if the two sensitivity vectors have an angle that is significantly smaller at a given     
47:16     
confidence level than the distribution of angles between independent random vectors of the same Dimension which     
47:23     
approaches 90° as the number of Dimensions increases so basically if they're totally aligned     
47:29     
they're an angle of zero we can say that there's some degree of alignment if they're they have some     
47:36     
angle of you know they're subtended at some angle but they're not uh you know     
47:42     
maybe they're under 45° and then the distribution of angles between     
47:50     
independent random vectors so these are just random vectors that have no developmental trajectory if you perturb     
47:55     
random vectors they kind of do whatever they want and they don't really have you know any sort     
48:02     
of alignment at all uh which approaches 90° as the number of Dimensions increases so this     
48:09     
is where we have high dimensional spaces where we have random vectors versus high     
48:14     
dimensional spaces where we have uh developmental     
48:20     
trajectories so they talk about figure one giving an example for three developmental parameters which can     
48:26     
correspond for example to the Affinity of a co-actor modulating gene expression temperature and salinity or other types     
48:34     
of parameters that are not genetic or environmental so we can map these     
48:40     
perturbations uh or we can map these developmental parameters to different     
48:45     
sorts of things going on in the world so you know this isn't just like     
48:51     
an exercise in mathematics it's an exercise in like saying what are the effects of this     
48:57     
thing the effects of modifying each of those parameters at time T is given by the vectors for 1 2 and     
49:04     
three and the angles between them determine alignment from equation 5 we know that all the sensitivity vectors     
49:10     
can be written as linear combinations of The Columns of the same Matrix the inverse of the Jacobian which where each column is     
49:17     
weighted by the elements uh this provides the basis to derive sufficient conditions for     
49:24     
alignment between the effects of different perturbations so if two perturbations have a dominant     
49:29     
component in the direction of the same column of the inverse of the Jacobian then the perturbations will be     
49:36     
aligned that is there will be some degree of alignment between perturbing one and two if there is I such that this     
49:45     
expression furthermore there will be total alignment between the perturbations if B and B B1 and B2 are     
49:54     
proportional so we have this sort of you know this mathematical     
50:00     
model and you know we can extend this to reaction to Fusion models so reaction diffusion models are a set of models of     
50:07     
pattern formation that have been widely used to represent diverse developmental processes including digit formation and     
50:14     
hair follicle placement these models consist of a physical representation of the tissue and a set of molecules called     
50:21     
morphogens morphogens diffuse and interact with the tissue leading to the emergence of patterns zemal in specific     
50:28     
spatial regions so they will use the gray Scot model which is a reaction diffusion model to illustrate how we can     
50:34     
study the alignment of phenotypic effects for different Origins using the framework induced introduced in the     
50:41     
previous sections so this kind of goes over the developmental function for the gray Scot     
50:48     
model uh X1 and X2 are these uh dynamical systems that describe the     
50:55     
cellular concentrations of more oragen 1 and two and then the components D1 and D2     
51:01     
within the equations X1 and X2 respectively are their diffusion rates to neighboring cells which is described     
51:08     
as this gradient the actually the gradient is the lassine operator f is the production     
51:15     
rate of morphogen 1 and K is the rate of degradation of morphogen 2 so we have     
51:21     
morphogen one being produced degradation morphogen 2     
51:27     
these parameters have been shown to have the genetic to have genetic basis in some systems so the perturbations on     
51:33     
these parameters can be understood as resulting from different genetic changes so this is where we have genetic     
51:39     
mutations or something where we're changing the concentration of     
51:45     
morphogens and how they're diffused so they model this on a grid of     
51:51     
50 a 50x 50 grid of cells and each of these cells are a given amount of two     
51:57     
morphogens so we can do that to sort of approximate the genetic variation of the     
52:06     
cells so they run this uh     
52:12     
simulation they run the simulation with small perturbations and the developmental parameters and compare the     
52:18     
resulting phenotypes so we'll see in figure two what that looks like um the     
52:24     
phenotypic effects either decreasing it show an decrease in K or an increase in     
52:29     
F and this is up here the developmental parameters uh these are     
52:35     
largely you know they are largely aligned resulting in the formation of a connected set of connected dots rather     
52:43     
than dots in the reference unperturbed phenotype so we'll see what that looks like um so the perturb phenotype should     
52:51     
not be identical because the angle between SK and SF is not zero so now     
52:56     
they want to build a genetic regulatory Network to study the alignment between     
53:03     
phenotypic effects of perturbation we use a common representation of Gene regulatory networks found on the     
53:08     
literature with continuous representation of time and the conventional formulation of these models     
53:14     
the phenotypes are the expression levels of end transcription factors that regulate each other's     
53:20     
expression we represent these phenotypes at States X so they're     
53:27     
NX phenotypes with XI being the expression level of the E Gene this     
53:33     
function f giving the change in States during development for this example is encapsulated in this     
53:39     
equation where we're looking at the i j f into the Matrix and gives the     
53:45     
regulatory effect of Gene J and the expression of Gene I so Gene J is regulating geni we have activation     
53:52     
inhibition and no interaction the expression of each gene can also be activated or inhibited by     
53:58     
environmental inputs U and so we have U of n uh gene expression follows the     
54:03     
michalis M dynamics of the coefficients Ki and Gene product as a degradation     
54:09     
rate given by UI mui in this way the developmental parameters for this     
54:14     
example are here um so this is basically what they're doing with the gene regulatory     
54:20     
Network so this is figure two this is alignment and a reaction to     
54:26     
Fusion model so on the left we see in purple the angle between 30° and     
54:34     
90° between SF and SK remember we said SF and SK don't equal zero they're not     
54:40     
totally aligned but they're you know going to vary between 30 and 90     
54:45     
basically this is showing this through developmental time so you have a lot of fluctuation early in developmental time     
54:51     
in terms of this angle and then it settles down after I don't know uh 300 unit of Developmental     
54:57     
time so this is again the angle between SK and SF those vectors and then we have     
55:03     
this random which is a control which is at 90 just about 90° throughout developmental time so it's like un     
55:09     
alignment and then what's aligned relatively speaking so you can see that the uh angle kind of is quite High early     
55:19     
in development reaching 90 at some point and then it settles down to an     
55:25     
equilibrium at about 50 50° or so yeah so after a transitory period with oscillations following the introduction     
55:31     
of the perturbation so that's what this is where the perturbations are being introduced     
55:36     
early the angle between SF and SK is significantly smaller than the angle between random vectors which is this     
55:43     
orange uh Line Orange Orange concentration of points at the top indicating alignment between the     
55:50     
phenotypic effects of perturbing knf the three panels of the right which are these three     
55:56     
here show the phenotypes plotted as the concentration of morphogen one with     
56:03     
higher concentration and lighter color for the reference developmental parameters perturbed k&f respectively at     
56:11     
developmental time 500 so they're looking at the end of this graph here at the end where it's settled     
56:17     
down and they're looking at the reference which is this I guess uh where     
56:22     
the the dots are aligned you can see that um you have perturbed K and perturbed     
56:29     
F and perturbed K being in the middle perturbed F at the     
56:37     
right it is indicated by the small angle between F and K the phenotypic effect of the perturbations is smaller resulting     
56:45     
in Connected dots as opposed to the dotted pattern of the reference so the reference shows this complete sort of     
56:52     
gray Scot uh morphogenesis where you have the indep dots line aligned the perturbed K you     
57:02     
see the you know sort of that the dots are connected through this sort of drift     
57:08     
so sometimes you see some connections sometimes you don't I mean this one here doesn't look connected     
57:14     
this one here doesn't look connected but you get this connection and then in perturbed F you get more     
57:20     
connection it's just kind of a different looks kind of blurry and K more blurry than K than f but basically you get     
57:27     
these connected dots as opposed to the dots in the random pattern so basically the reference has     
57:35     
no perturbation K and F have a perturbation that perturbation results in drift of the morphogen between the     
57:43     
dots and it ends up with this so you basically get the connected dots which     
57:48     
are sort of this regular pattern and then you get these sort of almost like     
57:54     
Stripes which are a little bit different so you know I don't necessarily know how to interpret that other than to say it's     
58:01     
an interesting     
58:07     
result so finally let's talk about figure four so figure four shows these     
58:12     
trajectories it shows Fitness over a number of generations so this is evolutionary and this is showing how     
58:20     
plasticity predicts evolvability and how this can be used for evolutionary control so you can see that you you have     
58:26     
these trajectories here you have these trajectories in evolution so let's look     
58:32     
at this uh pan a which is on the left shows an example population of the left     
58:37     
right and up down sets in orange and black respectively do represented represents unperturbed individuals no     
58:44     
environmental inputs so you can see the dots in lying that uh an arrows represent the     
58:50     
direction of the change in steady States when an environmental perturbation is introduced and one of the first two     
58:56     
states panel B in the middle shows the Evolution for Left Right and up down     
59:02     
populations towards an Optimum located at X3 X4 uh 12.5 and 7.5 represented with the     
59:11     
purple Arrow so that's up here um the Left Right population in     
59:17     
Orange I'll compete the up down transparent lines are the average     
59:23     
among the 25 evolutionary lines so you can see these transparent lines uh initiated from a single     
59:30     
individual for each of the 15 populations in each set panel C on the right shows that an additional     
59:37     
mutational input directly on the first two rows significantly accelerates     
59:42     
Evolution towards the optimum of the Left Right populations so the fitness is increasing     
59:49     
faster when you have an additional mutational input so this is these are numeric     
59:55     
simulations they're using this model to sort of you know look at developmental     
1:00:02     
perturbations look at the you know how they're sort of affecting the developmental trajectory then they're     
1:00:09     
looking at this in terms of evolution of development looking at uh single     
1:00:15     
perturbations and then looking at multiple perturbations okay so I think that's a pretty complicated paper and     
1:00:22     
you know it's something that I will link to in the description of the video but I think that gives you an idea of     
1:00:28     
how people are using these sort of mathematical tools to look at development and some of the problems     
1:00:35     
AV so that's all for today thank you for paying attention and I hope you learn     
1:00:40     
something
