     
Transcript     
0:00     
right looks like we've got some people here got dick and Morgan and Jesse     
0:06     
hello hello good morning     
0:12     
morning so uh any updates before we get     
0:17     
into it the uh the Met Pine meeting this     
0:23     
week conflicted but I I did bring up an email with them the U     
0:30     
div Harmony and was asking them about you     
0:35     
know how much overlap there was or yeah how how much interest they had in that     
0:41     
kind of you know at what point does computational Neuroscience merg with     
0:46     
computational system biology oh yeah and     
0:52     
U as well as like being really interested in terms of their     
0:58     
um uh just their their overall like long range plans     
1:05     
um uh and I got got a little response back from from one of the grad students     
1:12     
there uh I'll care dig that up but but not much I mean basically like that that     
1:18     
sounds cool uh but I still think that like     
1:25     
um you know looking at the the workshop that's the summer that James Glazer     
1:31     
putting together oh yeah the compies so 3D yeah yeah yeah yeah and I forget if     
1:40     
they fill it as I think they're also doing a meeting     
1:46     
at that time or something see seems to be a couple events back to     
1:54     
back um and uh that that might be the     
1:59     
excuse that might be the meeting to to go to to to talk more about that but     
2:05     
just just related you know one of the things that they brought up was that it sounds like you might be interested in     
2:12     
the kind of thing that human neocortical neuros solver group is doing with     
2:18     
brown and um and that was that was funny because I was just you know like was     
2:26     
absolutely just talking about that but they're gonna do an event like May     
2:33     
second I think um and yeah I'll probably     
2:39     
post some of this stuff in like brain organized yeah like for for     
2:45     
lack of and uh anyway the the only other     
2:52     
thing is I I did drop a link in in Divo worm orthogonal um just some somebody     
3:00     
had posted this uh what I would call an Ode to neurobi developmental neurobi     
3:05     
developmental biology Hacker News oh wow     
3:12     
okay you know just a just a aren't embryos amazing     
3:20     
yeah like yes yes they are yes they are so and and you might um well I'd love to     
3:29     
pick your brain bit about incf uh contacts and things at some point for but that this is kind of narch     
3:36     
X related but yeah we're you know I think I told you that the we got some board changes and things like that     
3:45     
um but no there was something else oh yeah I I met with a developer from     
3:52     
metacell okay and and metail do you know kind of contract     
3:59     
suffering engineering for a lot of scientific projects right I mean that's their that's kind of their thing and so     
4:05     
it was interesting because it was like well this is the people behind like the M Pine     
4:10     
gooey these are the people behind virtual fly brain you people behind yeah     
4:17     
and so that that was also just cool contact yeah uh who was that do do you     
4:24     
know the name or uh go go go Paul um I'm sorry     
4:30     
uh gopal Krishna okay it just asked because there are a lot of open Word people in in     
4:39     
metl it's you know Steve Larson is the CEO and Steve lson of course is he's     
4:45     
he's the sort of the one of the people in open worm who sort of started things     
4:51     
uh you saw in in the talk that I gave couple weeks ago yeah the the uh     
5:00     
the [Music] EI yeah yeah I think so well I mean uh     
5:06     
Steve Larson he he's he's done a lot of openworm talks if you go on YouTube or whatever and you'll see him so that yeah     
5:14     
that's uh but yeah so yeah he's also at medicel and and a lot of there are a     
5:20     
number of people senior contributors from open worm who are also sort of metacell adjacent or medicel involved so     
5:28     
gotcha yeah gotcha okay okay okay well it's     
5:35     
u in trying uh I don't know if this sorry     
5:40     
this might not be DEA but in trying to explain you know     
5:48     
uh the difficulty with brain-to-brain communication I was trying to get at the     
5:53     
fact that there were multiple you know there were probably multiple languages of of of different brain     
6:00     
especially like different modalities and basically trying to use you know the embryo and the kind of the collective     
6:08     
intelligence that is required for kind of spatial morphogenesis as like     
6:14     
obviously you know at least it's a good analogy for what's probably happening in     
6:20     
you know the brain um as well and that     
6:25     
you know that again not no one you know one cells in charge     
6:32     
yeah you know they they all develop together and they're all know changing     
6:37     
and learning each other's languages things like that any just just a great     
6:43     
you know o to the the the wonders of Developmental biology     
6:51     
yeah well that's great um yeah thanks Morgan that's great actually talk about     
6:56     
combine Harmony in a little bit because I was looking at the uh materials and I thought it was kind     
7:03     
of a nice conference I don't know I told I i' se you know it was     
7:09     
great that she joined because I I had seen it months ago yeah and as an     
7:16     
aspirational thing and because of course it was free I signed     
7:22     
up you know I I I've got kids I got responsibilities I can't I can get jet     
7:28     
off to London yeah uh but uh but I did I did want to go I mean it looked like     
7:33     
such a great agend you know it had such a great agenda things     
7:41     
happening next time or hopefully you know they'll be in the states     
7:47     
or I'm I'm joining the compus cell 3D um     
7:53     
conference oh yeah so yeah this summer it's like summer late late a     
8:00     
July early August July August yeah yeah I decided I should do that that's     
8:06     
awesome that's awesome I'm gonna try if I um I'm not shuttling kids places um     
8:15     
there's there's super close to that is computational um cognitive     
8:24     
Nur oh yeah conference that's like that's like six to the nth or something     
8:30     
like that that that's awesome because if I can't make it I'd love to     
8:39     
hear his his you know all his coursework is online all his talks are they're just     
8:50     
great multidisiplinary mix     
8:56     
yeah hi I'm being bugged by a dog     
9:07     
okay brainin communication with the dog yeah so I know what she     
9:20     
wants I got my 10r model to work using a what you call a key4     
9:27     
configuration stud any 10 but anyway I I got it to     
9:33     
work so now I can get some some data and that's all all I need     
9:40     
okay what was special about the T4 configuration um it's it's um it     
9:48     
shouldn't be stable but it is okay uh it's um they come the compression     
9:55     
elements come together in a V all right so it's like stacking one going this way     
10:01     
and one going the other way and you're putting them together and I think they     
10:07     
just push on each other the right way so they stabilize I guess I should analyze     
10:12     
that but yeah if I don't have to I won't yes they keep they keep telling me well     
10:20     
you have to make a diagram and show the forces and for for your thesis we want     
10:27     
to see this and I'm going hell great what I wanted to do yeah     
10:36     
anyway well that's great yeah yeah I got the exagon model to work good there's     
10:46     
three hexagons that joined together in four places with     
10:53     
um a tri Junction and the tri Junction is also a T4 configuration and that     
11:01     
works it's stable enough I can get uh     
11:07     
data so yeah so you when you model it you what what are you using to model it     
11:13     
what software oh just I'm just using console okay and so you're getting like numbers     
11:20     
out of it like yeah I changed the the strin um on the members and     
11:30     
and I can get a stress strain curve from it that's all I need to do I'd like to do some frequency analysis with it but     
11:38     
um and then they said what you have to relate it to cells and I'm going well     
11:44     
cells cells are not a T4 configuration not really they they do     
11:51     
other things including actively rearrange their their parts so um yeah I'm not sure     
11:59     
what I'm going to say about that but     
12:04     
um at least I can say that this simple um stable tensegrity can be used to     
12:13     
model that that cell configuration so     
12:18     
yeah yeah so it's nice that that's     
12:25     
working well that's great that I can write it out yeah     
12:31     
see this works I'm done you want to add active matter to     
12:38     
it give me five years and a salary and we'll talk     
12:49     
yeah well congratulations you got that done yeah it's     
12:56     
good all right looks like Mah's joining us well how are     
13:05     
you uh yeah so oh hello hi Bradley hi     
13:11     
everyone yeah I'm doing good I actually saw this message of uh course material     
13:17     
which I mentioned so yeah this looks interesting so is there a course from     
13:23     
2019 which teaches computational biology something like that in respect Dev     
13:31     
uh yeah I don't think we're doing anything with with if with uh you know in parallel with that so it's just kind     
13:37     
of that thing that was posted Morgan's really good at finding events and things like that     
13:43     
so okay sure sure sure I thought it's it came from a previous meeting which I missed so oh no no what was the     
13:51     
question oh you I guess you posted some things on uh the conference and and things like that and he wanted to know     
13:58     
if D.A was part of that and I said that really know okay sorry in like calls     
14:05     
calls for involvement yeah yeah something like that oh yeah I I I again     
14:12     
these are these are things that I aspire to go to but have no no Direct     
14:22     
evolvement that's okay um but but I do I I try to check to see if there's a you     
14:28     
know entially a remote track or any any you know will they film things or things     
14:34     
like that oh yeah yeah definitely the these Zoom meetings are always good to catch up     
14:40     
on yeah yeah yeah yeah so yeah we we're not     
14:47     
doing anything in conjunction with that but actually you know we should try to do more of that I don't know we can get     
14:55     
involved somehow but we'll see which one which one was     
15:03     
interesting uh so you are talking about specific for so I was actually looking at the website only I was looking at     
15:09     
Somey I didn't     
15:14     
see okay yeah yeah so um yeah we we should talk     
15:22     
about that today we're going to talk about the combine Harmony Workshop that that'll be interesting uh in itself and     
15:30     
one of the uh people who was at combine Harmony apparently is James Glazier we     
15:35     
just talked about him with respect to cyell 3D so compell 3D I I think it's it     
15:42     
would be a great thing to kind of get to use in the group but it's hard to get to     
15:47     
you know to sort of get it up and running for specific models and I actually visited uh where they were at     
15:55     
Indiana University back in 2015 so was a pretty ear pretty early days for Evo     
16:01     
worm and I went there you know was trying to figure out how to use Cy cell     
16:06     
3D to model something and they have stock models in copy cell 3D so I should     
16:13     
say that copy cell 3D is this platform it's a agent based modeling platform where they use cellular pots modeling     
16:20     
which is a physics technique um to model cells and cell populations sheets of     
16:27     
cells things like that so a lot of the things that you know we talk about in development like epithelial sheets or     
16:34     
embryos or you know whatever those are the kinds of things you could model cells that are in sort of connected to     
16:41     
one another in a configuration of some sort and you know you what you can do with cellular pots models is you can uh     
16:49     
introduce physics simulations you can specify the shape of the cells and their interactions and it     
16:56     
could be like physical interactions or chemical interactions or whatever so they they actually have done a lot of     
17:02     
stuff interesting things of different types of templates uh one of them being they I     
17:08     
think they've done a chick embryo and I'm not sure like what their state-ofthe-art is in terms of embryo     
17:14     
Genesis that they've looked at they've done a lot of things with like human organs and cancer research so they've     
17:21     
done a lot of things with tumors tumor Invasion uh you know you have things     
17:26     
like uh the liver model the different tissues and cell types in the     
17:32     
liver um and then you know you can actually use uh real world data to train     
17:37     
the model and what I mean by train the model is not like you do with machine learning which is to you know train a     
17:44     
network it's more like you give you specify things you parameterize things with real world data so you know if I     
17:51     
have some microscopy data that tells me about you know what the cells look like or you know maybe some videos of how the     
17:58     
cells migrate across uh a space you can use those kind of data to uh sort of     
18:05     
parameterize your model and that's basically uh you know I mean that's the best we can ask for in     
18:12     
simulations is to get that parameterization so so yeah that I mean you know that's     
18:19     
something that's kind of in the midst of the past so I don't know um you know     
18:24     
I've never found anyone who really been able to get it to you know to get compell 3D up and running and really     
18:31     
kind of attach it to a project so I mean you know this is I don't think it's a something you can     
18:37     
tack onto another project it's something you really need to spend some time learning and uh you know kind of finding     
18:44     
the right model system and the right model so you know we talk about all     
18:50     
sorts of different uh embryogen genetic systems here in the group um you know     
18:56     
we're interested in salamanders which might be a good candidate for this but there may be other systems as well that     
19:03     
you could use so yeah so that's that's compy so     
19:10     
3D um but um actually you know there's this combine Harmony event that happened     
19:18     
about two weeks ago and it sounds like a weird name and I I don't know I think it     
19:23     
was in the UK uh and you know had a bunch of people who who are doing modeling doing     
19:30     
computational biology and so I thought we should take a look at what they have what they they're up     
19:39     
to so this is uh actually I think the whole thing is the combine computational     
19:45     
modeling and biology Network so this is an acronym for this group uh     
19:51     
computational biology and or computational modeling and biology Network combine as an initiative to     
19:57     
coordinate the development of various Community standards and formats and systems biology so if you know anything     
20:05     
about incf um that's what incf does for computational Neuroscience so um Morgan is asking     
20:13     
about that earlier you know what what are the connections between computational neuro neuroscience and     
20:21     
then computational systems biology and I think one of the areas I think where they kind of come together is in this     
20:27     
community standards formats area so you know you'll have a lot of um data     
20:33     
collection and a lot of groups are doing data collection and then they want to     
20:38     
share the data sometimes that's by uh Mandate of the funding agencies     
20:43     
sometimes it's just because that's what you do in a community is you share data and you make it available to other     
20:48     
groups and so you know you can do that but you have to be able to format it in     
20:54     
the right way so that other people can access it they can understand what's going on in the data what the variables     
21:01     
mean you know and Etc and so these are you know there's a lot of formatting     
21:07     
involved so having formats that are both accessible to other people and formats     
21:13     
that don't like you know make the file obsolete in 10 years because we want to use these data for the long term we want     
21:21     
to have a Long View of data sharey yeah am I right Patrick Le is uh     
21:30     
a contributor to open world uh porri Leon you mean is it par por     
21:36     
yeah okay is I think I see his name GitHub on openware yeah so he is     
21:43     
actually an openworm contributor yeah and he's here too so that's good yeah     
21:49     
yeah okay so H so the formats and then the community standards which are of     
21:55     
course you know we all want to do things in a similar way because because we don't want people pursuing wildly     
22:01     
Divergent methods and then people don't know what to expect so like if if 10     
22:07     
groups are generating data they should all use generally accepted standards so that we can if I'm the end user of those     
22:15     
10 data sets I can put them together pretty quickly I don't have to question     
22:21     
whether I got you know whether I extract all the data out of the format or not     
22:26     
and I can really kind of merge the the data sets easily so that's what incf does as well um so yeah you mentioned     
22:33     
that uh por Leon is on one of the local organizers uh yeah he's in openworm he's     
22:41     
he does he's also an incf uh person so they do a lot of things with standards     
22:47     
and a lot of people in openworm um openworm kind of comes out     
22:52     
of that sort of tradition of data sharing so incf is uh sort of populated     
23:00     
by people who are openworm adjacent um so like for example uh Steve Larson's     
23:05     
doctoral adviser Marian Marton she's in incf uh she's a per member of incf like     
23:13     
a lot of neuro informatics type people and that's really been a core thing of open wor really kind of you know one of     
23:20     
the sort of core things in openworm that allow you to do the things that you want to do in open worm are from uh     
23:29     
this sort of thing so um yeah Harmony is a code Fest type meeting with a focus on     
23:35     
development of the standards interoperability and infrastructure so this is again this sort of data sharing     
23:43     
uh Big Data sort of thing there are generally not too many General discussions or oral presentations during     
23:49     
hermony instead the time is devoted to allowing Hands-On hacking and interaction between people focused on     
23:56     
practical development of software standards so this was held at University College London earlier this     
24:02     
month if this is the okay this is the schedule we have uh we get to go to the     
24:09     
break day this is Monday     
24:15     
Tuesday uh so we have uh neuroml we have a breakout book for each     
24:22     
one so I can send I can put this link in uh in the slack and you can check this     
24:29     
out on your own but uh there's neuroml which is a hackathon for neuron and     
24:35     
network models uh annotating systems biology markup language models with sbgn which     
24:44     
is uh Gene regulatory networks or Gene networks so it's a uh systems biology     
24:50     
modeling of that uh there's Dem demo of simmune     
24:56     
Monitor and analyzer then on Wednesday they had Vel which is     
25:03     
another model modeling platform uh exploring copac from     
25:08     
JavaScript and collaborative opportunities for combine we had the PAB     
25:15     
breakouts uh yeah so there's a lot of stuff here open VT which is a     
25:21     
multicellular simulation then Thursday we have uh sort     
25:26     
of the end of the conference there's this one continuing work towards     
25:32     
reproducible stochastic biological simulation so you can see there's a lot of stuff here that's uh in that     
25:40     
area uh let's see uh you know there's the abstract book I didn't want to get into that too much but I noticed some of     
25:46     
the people here um they have power outlet instructions for 220     
25:52     
volt so if you're coming from the US you know kind of what to do um yes so I noticed there are a     
25:59     
number of people here I didn't get a uh I don't know where I found the list of participants oh here's people I guess     
26:06     
this I don't know if this is all the people attending but these are the coordinators um so there a number of     
26:14     
people who participated in this uh were doing the were the systems biology stuff and then systems neuroscience and the     
26:22     
all the adjacent areas um I know that James Glazier was a     
26:28     
part of this and of course por gleon and another person who was     
26:33     
involved in this was this person here TJ SEO who's from uh the University of     
26:40     
Florida and this is a paper that they wrote um I think this came out in     
26:46     
23 and I think dick sent me this paper and then I think uh Morgan also     
26:52     
expressed interest in it um so this is a this is TJ SEO and James glazure on this     
27:00     
and so this is uh you know kind of a group effort it's a an approach called     
27:05     
tissue Forge interactive biological and biophysics simulation environment so     
27:11     
this is a a specific type of thing that they're developing for looking at     
27:16     
particle physics uh models to and chemistry models and biology models to model tissues at the tissue level so     
27:25     
tissue Forge is an open- Source interactive environment with particle based physics chemistry and biology     
27:30     
modeling and simulation it allows users to create simulate and explore models and virtual experiments so this is not     
27:38     
just modeling it's you know virtual experiments based on soft condensed matter physics at multiple scales so you     
27:45     
have different scales of of soft matter physics or condensed matter physics     
27:50     
going on in a Cell a in a tissue so you have cells interacting you have things interacting molecules interacting in     
27:57     
cells and between cells you have then tissues that interact and so forth so     
28:04     
you go from the molecular to multicellular and one of the benefits of     
28:10     
tissue Forge is putting this in a simple consistent interface so you know people     
28:16     
have tried to approach this multiscale modeling Problem by taking different     
28:21     
types of models and combining them so like you might model things like in a molecular simulation environment and     
28:28     
then you might model something else in a cellular modeling environment and you might model tissues in a different     
28:34     
Environ like if we want to look at T segurity like Susan we might use uh you     
28:39     
know something like console but console doesn't give us that sort of the any of the biological stuff that we might want     
28:45     
to model as well so it's you know yeah there there's no um sorry there's no     
28:51     
material in it that says this is a uh microtubule yeah it isn't doesn't have     
28:57     
that and you enter it in yourself and it's still not right because     
29:03     
microtubules can be pulled apart and the cell still functions just grows     
29:10     
microtubules in between itself or something or something shifts so and     
29:16     
console calls that an error yeah and it really isn't an error and     
29:22     
the same with the microfilaments if they're compressed you're not supposed     
29:28     
to be able to push a rope but in a Cell it doesn't care it just tightens that up     
29:34     
itself like it says that slack and it     
29:39     
um there's a mechanism it has for     
29:44     
um pulling that taunt again think there's several mechanisms     
29:50     
so yeah I can't remember the name of the molecule but yeah yeah I I do a talk on     
29:59     
it actually oh yeah how does it do this well um it's     
30:08     
not just pushing a rope in a salad and actually has mechanisms for taking care of     
30:14     
that yeah and including the masas and motors     
30:25     
and anyways yeah console is very     
30:30     
crude method for looking at cells right yeah it always it's a bridge     
30:41     
yeah so yeah one of the things about this is you can um of course you can you     
30:46     
know monitor the simulation and change things in real time but then there's this growing Library built-in model     
30:53     
components along with support for user specified models during the development     
30:58     
and application of custom agent based models so this you know this is where you have these components that aren't     
31:05     
just like physical or just biological it allows you to really kind of span that Gap in the subcellular cellular     
31:12     
multicellular modeling domain most spatio temporal agent based biological simulation tools only support one     
31:19     
cellular Dynamic simulation methodology and focus on a particular problem domain     
31:25     
with a particular length scale so this is where we have you know we we have these modeling platforms that generally     
31:31     
focus on one type of process or one type of um uh problem or one type of scale so     
31:40     
they they kind of bring up their own you know work copy cell 3D and then they also talk about Morpheus so for example     
31:48     
in copy cell 3D and Morpheus Implement cell model objects using the cellular     
31:53     
pots model so Morpheus and Cy cell 3D use either the cell wheeler pots model     
31:59     
or the Glazier Grainer hog formalism which is the ggh     
32:05     
formalism and so this only supports oian lattice based models While others like     
32:11     
physicell or chest which is an acronym support modeling cells with leonian     
32:17     
Lattis free particle based Center models as simple Point like cell particles so     
32:22     
you can see that you have these two different sort of approaches you have the sort of this uh physical model uh     
32:30     
using sort of physics principles and then this other one which is uh which     
32:36     
which is kind of a ulian lattice based thing and then you have physicell or Chast which supports a lonian lattice     
32:43     
free particle based Center model so you're just using different types of approaches oan or lran and you make a     
32:51     
bunch of assumptions going into the model so if your problem doesn't isn't really suited for those uh     
32:58     
uh assumptions then you know you you're not going to get the kind of thing that you want it's kind of like with     
33:06     
uh uh with Susan's uh situation where she's trying to use a program that would     
33:12     
you know it's probably the only kind of program that can model uh 10 segres but it's really a mechanical modeling tool     
33:19     
so it doesn't give you that support for biology and in this case you know you're thinking about how you want to uh sort     
33:27     
of the perspective you take on how the biology works so for example you could use a particle model you could use sort     
33:35     
of a you know a lattice based model and so     
33:40     
you have to make those kind of decisions and you know depending on the the system it may one one approach may be     
33:47     
appropriate or the other uh this uh so this the second uh     
33:54     
version can be extended to include subcellular details using the subcellular element model which could     
34:01     
support modeling the spatial complexity of shell cell shaped cytoskeleton and extracellular     
34:08     
Matrix now extending the CPM ggh which is this first one cellular pots model     
34:14     
Glazer Grainer hoga formalism to include cellular compartments allows representation of     
34:20     
subcellular components like the nucleus critical molecular species or regions     
34:26     
with specific properties but does not support specific representation of maum molecular     
34:31     
Machinery so this is again you know we have these sort of uh subcellular     
34:37     
components we can model but it doesn't really help you too much you know with other types of things going on in the     
34:43     
cell macro molecular phen uh typically modelers who are     
34:49     
interested in subcellular and cellular detail must use an adapt general purpose     
34:54     
molecular Dynamic simulation tool such as lamps H HED blue named or gromax so     
35:01     
these are these different molecular Dynamic simulations that people use uh     
35:06     
for example uh shaffi customized lamps to model cells as cluster of particles     
35:12     
simulate spheroid Fusion during spheroid dependent bioprinting so you can use these kind of molecular Dynamic     
35:18     
simulations to do a lot of things you know sort of particle based problem so     
35:23     
you know again it depends on the problem you're trying to approach uh in your biological modeling if it's     
35:30     
uh something that's amenable to this the sort of particle simulation or these molecular Dynamic simulations then     
35:37     
you're unlocked there a lot of nice models if that's not exactly what you're doing then you have a     
35:43     
problem most molecular Dynamic simulation tools are designed to parse and execute models that are     
35:49     
theoretically well defined and molecular Dynamic simulation specifications and engines tend to be well optimized for     
35:57     
computational performance most assum a fixed number of objects within a model and do not     
36:02     
support runtime object creation destruction or modification so this is something you know where you have sort     
36:09     
of a limited degree of expressivity of your code uh that means that you can only do     
36:16     
a finite you know number of things you can't do a lot of uh operations that you     
36:22     
might be able to do if you just simply coded it in python or C++ you know     
36:28     
directly and use like objectoriented programming to do some of these things you can build things at runtime and you     
36:35     
know you can't really build things at runtime and you can't really modify objects so uh that's that's a problem     
36:42     
for a lot of these tools in molecular Dynamic simulations you know they come out of you know a very limited set of     
36:50     
use cases where you want to say model you know a bunch of molecules bouncing around in a container or something and     
36:58     
they're very good at that they're very optimized for that they don't have a lot of the details that we need for general     
37:03     
purpose model so that's basically do they include sorry do they include gravity in     
37:09     
their models that's what I oh uh for these kind of models I think they do but I don't know how they put them in I     
37:17     
think they're what they're talking about here is just kind of like the basic uh sort of approach to the problem     
37:24     
so like can you represent your problem on a ladder would you put like you know is it a     
37:30     
bunch of cells interacting on a sheet or is it like you know something     
37:35     
where you have molecules bouncing around or moving freely uh that AR aren't on a lattice     
37:42     
you know so you have the physics the physics are kind of implied in the model but the idea is that you're     
37:48     
trying to specialize it for specific types of thing I me there's some basic physics modeling in in this but yeah I     
37:56     
mean again with molecular Dynamic simulations I don't think you can get very sophisticated with that like you     
38:03     
know I mean like you're I asked someone one time if they had included gravity in     
38:10     
their molecular model and they said no no no gravity doesn't have anything to     
38:16     
do with it and I'm going yeah there's no convection yeah in micro gravity you have to     
38:23     
include that if you're going to look at yeah gravity as a an input anyways right     
38:32     
well yeah that I don't know I mean that very well could be that they don't have like they haven't specified a lot of those interactions but I'm not sure like     
38:39     
if you use the package I'm not sure how specific you can make the any one model     
38:44     
so they'll have you know they oh go ahead you might be able to change the     
38:50     
physics so includes it yeah but you know there's also this     
38:57     
issue of heterogeneity so if you say go across your simul your model you know     
39:03     
what what is a specific gravity what is a specific gravity doing and so forth how is it represented in the model and     
39:10     
does it represent everything equally you know so you can have yeah you can have     
39:16     
different convective forces as well and that might be uh homo heterogeneous across the simulation so there are a lot     
39:24     
of things going on there um that I think the Point here though is that a lot of     
39:29     
the papers or a lot of the models that have come before this are sort of you know not really they're not even really     
39:36     
standardized in the same way that you might expect a modeling package to be     
39:42     
they're specialized for specific problems but if you go outside of that domain then you know that's that's your     
39:47     
problem and then you know they make a lot of assumptions that a lot of times people you know have to really kind of     
39:55     
think deeply about to get you know their their problem to match with the software outputs so you know it's it's a lot like     
40:03     
mathematical modeling or like you know thinking about like how to represent     
40:08     
some biological thing right we want to represent a epithelial sheet for example what how would we want to do     
40:16     
that would we want to use an oan approach or bran approach or um some other kind of approach or maybe     
40:22     
something that's you know uh a minimal physics Model A minimal model there are     
40:28     
a lot of options but the point is is that you want your model to be reproducible you want it to be sort of     
40:34     
interpretable in other words you want to know how it's operating and you want to know if it's     
40:42     
if it's well suited to your problem so uh and and of course if you stack models     
40:47     
on top of one another they have to be interoperable because it doesn't make any sense to use like three different     
40:52     
tools for three different scales and then say I'm attacking a a multiscale problem     
40:58     
and you can't fit them together very easily so these are all problems they they that's how they kind of motivate     
41:04     
tissue Forge so this paper presents tissue Forge and open source real-time     
41:09     
modeling and simulation environment for interactive biological and biophysics modeling     
41:15     
applications tissue Forge is designed to address many of the affir mentioned issues and challenges it enables agent     
41:21     
based spatio temporal computational modeling that scales from the molecular to the multicellular     
41:28     
uh is designed for ease of use by Mo modelers research groups and collaborative scientific     
41:34     
communities uh here we describe the philosophy mathematical formalism and basic features that demonstrates     
41:40     
usefulness across multiple disciplines in the physical and Life Sciences we present representative examples of     
41:47     
advanced features at a variety of Target     
41:53     
scales so yeah uh let me go down so they do do actually they have open what they     
41:59     
call open science support which is the sort of aspect of data sharing and you     
42:05     
know model uh sort of modeling standards uh you have a number of     
42:12     
computational benefits like interactive and batch execution which allows you to sort of     
42:17     
develop your models and run them and modify them in real time     
42:24     
um so they introduced the Concepts here uh so for Newtonian Dynamics the     
42:31     
position of I particle in R is updated according to its     
42:36     
acceleration which is proportional to its mass and total force exerted on it     
42:41     
and for uh levian over damp Dynamics m subi is a drag coefficient and particle     
42:47     
velocity is proportional to the total Force so we have Force calculations here Susan this where they're this is sort of     
42:55     
what they're working from in terms of the kinds of physics that are represented in this     
43:01     
model uh then they also support three broad classes of force generating     
43:06     
interaction um there's the implicit Force which is the force due to implicit interactions between the E and J     
43:13     
particles so if you have two particles they're interacting that's the intera implicit interaction     
43:19     
Force then you have a bonded Force which is the force due to Bonded interactions     
43:24     
between the E and J particles and then uh the explicit Force which is     
43:31     
uh acting on the I particle so you have implicit interactions between I and     
43:37     
J uh bonded interactions between I and J and then an explicit force acting in I     
43:44     
so that's how they do that and then the implicit interactions result automatically from interaction potentials between Pairs of particles of     
43:52     
given types bonded interactions act between specific pairs of individual particles     
43:57     
and then explicit forces act on particles they explicitly defined Force descriptions do not necessarily     
44:03     
represent interparticle interactions so for example gravity internal noise     
44:09     
system thermal equilibri so this is again the question you asked um this is     
44:16     
where you have these explicit forces that you can represent you're not representing them like you're not     
44:22     
representing well I guess you could represent gravity specifically but you know you want to turn turn things on and     
44:27     
off depending on what you want so I think that's why they make that leave that open um so sometimes internal noise     
44:34     
and some systems that's really important uh system thermal equilibrium sometimes     
44:39     
really important uh sometimes gravity's you know you might want to turn gravity off or turn it on depending on what     
44:45     
you're doing uh tissue Forge provides built-in force and potential based definitions     
44:52     
and permits applying an unlimited number of executable force and potential objects     
44:57     
they individual particles and particle types so again you can PL plug these in if you're going to subject yourselves     
45:05     
to Cent fuging yeah you need to include gravity yeah     
45:11     
well that's a force I mean it's bad like you can certainly wreck spells by the     
45:18     
centrifuging you do it deliberately um but sometimes people separate cells by     
45:25     
using centrifuge and I still think that has an effect on cell     
45:32     
type when you're all set said and done with the center yeah and then of course     
45:38     
if you're in a Cell culture environment you also have substrate so if you're on a certain substrate you can affect the     
45:44     
fate of cells or at least the way they interact so but all these things     
45:50     
yeah I mean you can model them in different ways uh using different explicit forces that's the idea and I     
45:56     
don't you know you could argue that it's not enough but you know that's what we do     
46:02     
within modeling sometimes we're just dealing with like throwing in a generic thing and then saying it represents     
46:11     
something uh yeah so this is a a diagram here figure one where they talk about uh     
46:17     
the different modeling features here uh so you know in a we have um five     
46:24     
superimposed snapshots of a double pendulum implemented in tissue Forge so they have     
46:29     
you know double pendulum is where you have a double jointed uh Rod that comes down and swings on like a center of uh     
46:38     
gravity you know around a center of gravity on this thing doesn't that's that's sort of stable and then you have     
46:44     
these two things that move back and forth so it's like a pendulum but there are two independent parts to it with a     
46:51     
joint in the middle and so you can have uh a large number of configurations that result from that that's what they're     
46:57     
doing here so they're able to demonstrate that you can model kind of like these interactions between linked     
47:03     
particles and so that's that's kind of what they're doing there yeah a double pendulum like that is actually a chaotic     
47:11     
system yes yes it is yeah interesting so the Dynamics are     
47:17     
going to be chaotic you know if you plot them out at at the different joints um     
47:23     
and so yeah and so uh B is uh four tissue forg clusters so these     
47:29     
are the particles that are different colors and they're clustered representing biological cells each     
47:35     
consisting of 10 particles whose color demonstrates cluster membership potentials describe particle     
47:42     
interactions by whether they are in the same cluster intracellular or different clusters intracellular so a blue cell     
47:49     
and a yellow cell have a relationship those are those can be an i and J but they're in different clusters so you you     
47:56     
want to treat those different differently um C is tissue forg     
48:01     
simulation of chemical flux during fluid droplet Collision so this is where you     
48:06     
have blue and red these two particles I and J they kind of come together and     
48:11     
they Splat and they're kind of overlapping here and then those parts are kind of coming apart so they can     
48:18     
model sort of uh particles that come together and they form like patterns     
48:23     
they sort of form droplets from the drops and splatter out and you can see     
48:29     
sort of this the simulation of this process here so this is chemical flux during fluid drop Collision each     
48:36     
particle represents a portion of fluid that carries the an amount of a diffusive chemical the amount of which     
48:41     
varies from zero which is blue to one which is red so you have zero and one     
48:46     
and they Collide and then they create all these intermediate sort of patterns     
48:51     
and and colors you know amounts when two droplet carrying different uh two droplets carrying     
48:58     
different initial chemical amounts Collide resulting droplets tend towards homogeneous chemical distributions so     
49:05     
you can see that here so they can do a lot of interesting things the physics here um definitely more than you can do     
49:11     
with uh copy cell 3D so you know this is something that we were interested in     
49:16     
with like you know at the molecular scale but also you know maybe relevant to the cellular scale as     
49:24     
well uh so they they kind of go through some of the more of the math here in terms of what you're uh kind of the     
49:31     
different functions that you're using within the different types of [Music] interactions     
49:36     
um since tissue Forge enables the implementation and execution of models at different length scales particles in     
49:43     
a simulation may represent objects with a wide variety of possible behaviors a     
49:49     
particle could be Atomic and subject to energy conserving implicit interactions so these different types of potentials     
49:56     
that you can use here as in classical molecular Dynamics     
50:01     
particles can also represent portions of material that constitute larger objects for example portion of cytoplasm you can     
50:08     
carry quantities of materials within it and that's an example of that is the convection of a Sol solute chemical and     
50:15     
a portion of fluid tissue Forge provides built-in features to enable particle based modeling and stimulation of fluid     
50:21     
flow based on transport dissipative particle Dynamics or t dpd and smooth     
50:28     
particle hydrodynamics including a predefined t dpd potential that can be     
50:34     
applied when describing interactions of the simulation they have an equation here and so you know there are all sorts     
50:40     
of ways we can treat these kinds of Dynamics we can also do things allowing     
50:45     
particles to carry embedded quantities or treating particles as constituents of     
50:51     
larger objects so that's a cluster for embedded quantities like we saw with the     
50:57     
uh the uh droplet splattering we can actually uh attach to each particle a     
51:03     
vector of states that can evolve during a simulation you see that you can actually have it within a particle you     
51:09     
can have a lot of uh different states and then you can actually break those     
51:15     
apart as you need to so there are a lot of ways you can represent this mathematically in in terms of code and     
51:22     
you know so that's that's really nice some of the stuff that they have here and you can represent flux you can represent different types     
51:30     
of uh like chemical species transport um and then they kind of get     
51:35     
into the code here where you know we have these uh we have we're working     
51:41     
actually on modeling and simulation specification using classes objects and     
51:46     
functions so we're using basically object-oriented programming and so in     
51:52     
you know this is a a very powerful technique where you can assign different     
51:57     
uh sort of conceptual categories to objects and then break that object down     
52:02     
into instances and other things so you know we can build a model of um you know     
52:08     
classes objects instances things like that uh you and that can be related to a     
52:14     
real world system so if we want to Model A Cell you know we might have different classes of things in the cell we might     
52:21     
have different objects those objects are going to have instances and we can use     
52:27     
uh we can actually just write this in code to simulate this so this is you know something you can do in either C or     
52:34     
Python and they show some examples here where you can this is in Python where you're importing tissue Forge so you     
52:41     
know they have the package available you just import it into Python and then you start running this     
52:47     
objectoriented uh um you know this this sort of framework so specifying a     
52:53     
particle type with a particular radius using a class um from the     
53:00     
toolbox and so you can you know initialize the simulation retrieve an executable particle type and create     
53:07     
particles on Python and it's not that much code but they give you these examples so that you can do things in     
53:15     
code and then this shows you sort of an example of what a simulation will look like and I don't know how they're uh     
53:23     
visualizing this but basically you have these two particles you've set up a simple oscillator     
53:29     
interacting via harmonic potential so you know we're us putting all that stuff that we've talked about together to do     
53:39     
this and then of course we have the uh this is at the I guess this is     
53:45     
a let see this is actually using the python     
53:52     
API specifying interactive simulation of a convection of a species near a     
53:58     
species sync in Python so this is actually uh where you're using uh something in a python     
54:06     
interpreter and then you can also use this in a Jupiter notebook so Jupiter notebooks are very lightweight compilers     
54:12     
later while You' execute things in kernels and we can put them on GitHub and you can see that the quality of the     
54:19     
uh simulation is basically equivalent so if I have I can build something in a in     
54:24     
a in Python interpreter and I can put it online in GitHub and people can work on     
54:31     
this um you know kind of interactively so this is very powerful Stu it allows     
54:37     
you to do a lot of things not just in terms of simulation but also in terms of like uh implementing it in code and     
54:45     
doing it in a in a good way that we can then share with our community and uh     
54:50     
play with the the the parameters and it's it's pretty transparent     
54:58     
okay so I had a a comment thank you for that um that     
55:04     
reference write it up in my presentation because it does a nice summary of a     
55:10     
bunch of the simulation packages so and adds a new one     
55:18     
yeah so I guess yeah all of that very nice yeah so yeah I guess the point the     
55:25     
lesson here is that you know you know we have these very opaque specialized tools and we're kind of bringing them all     
55:30     
together into this very interactive easy to use package and that's why they     
55:35     
they're kind of pushing this and so you know you should maybe look into this as a way to model some Pro questions of     
55:42     
problems that we have in the group I'd be interested to see people can do it does do um convection it just said yeah     
55:52     
that was a model using convection yeah so you could do it where you have no     
55:59     
convection or very little and then um a lot more convection than you get at     
56:07     
Earth's gravity and you could do simulations on the moon and you could do     
56:13     
simulations on Mars and like you could it could make it interesting that way     
56:19     
yeah by changing their convection convection right     
56:28     
yeah that's great yeah so I always should do a project yeah yeah see what it could     
56:38     
do right well uh any other questions or     
56:44     
comments before we go yeah great package though great great project follow yeah     
56:50     
yeah I don't know where it is on GitHub but is that     
56:59     
that's okay I just I'll look for it okay yeah I'm interested all right sounds     
57:06     
great all right well thanks for attending and um yeah all right see you     
57:12     
next week thank you bye care bye bye so for those of you     
57:19     
interested in tissue Forge in the GitHub repository here it is it's github.com     
57:26     
SL ti- Forge and this is their they what they have on their readme here is they have a     
57:33     
number of really interesting uh particle simulation examples so this is kind of an example     
57:39     
of everything we're talking about we have Collision splattering sorting into     
57:45     
these groups you can see all of it right here in this top image and so they have     
57:51     
a build status here for the binaries they have a set of documentation examp     
57:56     
examples so you know this is for the API and for the basic program and for python     
58:01     
API C++ API and so forth they have a gallery here of specific things these     
58:07     
are animations of things we saw in the paper so this is uh Collision of     
58:12     
droplets in the resulting splattering this is where you have the     
58:17     
cell sorting in a tissue by differential adhesion this is where we have a Vertex     
58:24     
model which is conf figured uh from of uh sort of a model of local vasculature     
58:30     
that you know you can build a model from components and you know you're using     
58:35     
this mesh to build the model and you're exporting the mesh and you can actually so this object files generated in     
58:43     
blender and you can use that as part of the model uh then you have these     
58:49     
two-dimensional agent based models so this is a two-dimensional sheet of cells that uh it's a two-dimensional uh     
58:56     
sort of kind of like a cellular pots model uh this is self proliferation and differentiation in the colonic Crypt so     
59:03     
this is a specific organ where you have a bunch of cells and they have these different states so they're doing this     
59:09     
in a very similar manner to the molecular Dynamics or the molecular simulation but they're using not     
59:16     
molecular Dynamics nor cellular pots modeling they're using this approach uh then you have this uh     
59:24     
simulation of single cell migration over an extracellular Matrix across the fibr     
59:29     
so this is actually a nice example of like a cytoskeletal structure and uh cell     
59:36     
migration within that structure and so you know this is something I think that might be useful to some of the modeling     
59:43     
Susan's doing or some of the modeling of within the cell that we might want to do in our group okay one thing I'd like to     
59:50     
talk about with respect to combine Harmony is that combine of course is a set of standards that are sort of the     
59:57     
where they're going so they actually have this nice resource they have these     
1:00:03     
Jupiter notebooks that show how to create simple models from each of the combine standards so it's one thing to     
1:00:08     
have standards written down in a place another thing to have like a notebook that shows you how to execute the     
1:00:14     
standards and how to put your data into the standards so there are a number of Standards here there's     
1:00:20     
cellml there's neuroml there's om     
1:00:27     
there's sbgn there's     
1:00:32     
sbml there's SBO systems biology open language and there's satl so this is uh     
1:00:40     
simulation in SPM so each of these have their own GitHub repository they have actually a     
1:00:47     
binder server so this is really interest this is really good and then they have this uh main examples repo here which is     
1:00:56     
uh combined. org on GitHub so this is combined. or this is their these are     
1:01:01     
their notebooks and if you go to their organizational front here this is the     
1:01:07     
computational and biology Network and these are their repositories here so they're doing a lot of stuff with um you     
1:01:15     
know educating people and uh putting out the     
1:01:20     
specifications and other things so another thing I want to talk about uh is is this dorm ml repository     
1:01:28     
so this is if you go to github.com devor there's this repository dorm DML     
1:01:36     
and so this is this dorm ML Class that we I talked about in the 10th     
1:01:42     
anniversary talk uh you know is something we did in the fall of     
1:01:47     
2019 and this is the last time it was offered and you know this was separate from our regular meetings so this was an     
1:01:53     
hourong format where we would go over pretty much the 16we fall term of a     
1:02:00     
standard university and um you know going through different topics uh from week to week     
1:02:07     
and so you know this was something that was meant to be educational or biologists who wanted to learn about     
1:02:13     
machine learning but also for machine learning people wanted to learn how to apply their models to biology so we have this repository and     
1:02:21     
you're welcome to issue poll requests to it you're welcome to go through the materials you're welcome to so we had     
1:02:27     
for example we had these five minute reviews that people contributed to we     
1:02:33     
had current topics which of course since this was offered in 2019 you can contribute to as well with new content     
1:02:40     
and hopefully at some point we can do this again because this is something I think that needs to be updated and as evidenced by     
1:02:49     
conversations and slack and in the meetings there's some interest in doing this again so we have this stub on our     
1:02:56     
website this is dorm. we.com and it kind of goes through the syllabus here so we     
1:03:01     
had a 16we schedule it starts with sort of this informational first meeting     
1:03:07     
where I introduce this knowledge map of machine learning and you know there are a number     
1:03:13     
of different things that we talked about in that meeting the second meeting was on a project that we've done called     
1:03:18     
digital Bassel area which is modeling of a very small organism it's a diet type     
1:03:25     
of dietone that lives in colonies that they had this really interesting set of movement     
1:03:32     
patterns the third meeting was on something called open dvo cell which was really the precursor to Deva learn so     
1:03:38     
this was pre-o learn in fact dvo learn came out of this effort so this is an interesting kind of historical     
1:03:45     
connection within the group and open Devo cell is really you know about applying RNN cnns to microscopy data so     
1:03:54     
this was the main thing about the third meeting this is the hype cycle for     
1:04:00     
artificial intelligence for 2019 this was I think part of the first session but it's interesting to look at this     
1:04:07     
both in terms of where these Technologies are on this hype curve and historically what is the hype curve look     
1:04:13     
like for 2024 and not going to get into this curve but a hype curve is basically     
1:04:19     
something that uh the Gartner consulting firm puts out on different Industries     
1:04:25     
and the idea is that you know there are these things that sort of get these innovations that occur these models that     
1:04:31     
get released and they have this there's sort of this peak of inflated expectations where everyone thinks that     
1:04:38     
it's going to solve all the problems in the world and then there's this trod disillusionment where people say well     
1:04:44     
this this model is pretty much flawed maybe fatally so and then there's this slope of Enlightenment and this plateau     
1:04:51     
of productivity and so reason we have this in here is because a lot of the techniques that we talk about a lot of     
1:04:57     
the models follow this kind of Hye grth so you go from a model being released to     
1:05:04     
something that everyone adores and they think it's going to be applied to every problem in the world and then when it     
1:05:10     
invariably does not do that then people kind of go overboard and saying that     
1:05:16     
it's not worth our time then eventually we figure out the best kinds of models for different problems and we kind of     
1:05:23     
adapt those and use those and so you saw in the meeting with uh some of the models of cellular physics that we have     
1:05:30     
different models that people use and they don't necessarily go through this hype cycle because it's not you know a     
1:05:37     
commercial Endeavor it's an academic Endeavor but I would say that a lot for a lot of biological modeling we could     
1:05:43     
build a similar curve sort of expectation cycle so that's something     
1:05:49     
that's another aspect of this course that might be of interest to people if you're interested in technology     
1:05:55     
um then we had a very important fourth meeting lecture on input data and this is something that I presented and this     
1:06:02     
is a very important thing for a lot of modeling in general so when you have a model you want to have good input data     
1:06:10     
especially for machine because you need to be able to train your model you need to have it reflect the real world but     
1:06:15     
you also want it to reflect a lot of the edge cases and out of distribution aspects as well so biology is rif with a     
1:06:22     
lot of behaviors that are hard to actually you know classify very neatly     
1:06:28     
and even when you have a lot of data you won't observe some mods of behavior so this is you know talking about input     
1:06:35     
data um this is something I'm probably going to update but you know if you're interested in machine learning is is an     
1:06:41     
essential top but it's also as we saw in the uh     
1:06:47     
you know biological modeling space this is also something we need to think about when setting up our models or     
1:06:53     
initializing our models so in machine learning we don't worry about this as much but maybe we should because machine     
1:07:01     
learning you know we need to have a model that captures the right assumptions about the system we also     
1:07:08     
need to have in biological modeling we need to pick the right approach the right assumptions to match what we want     
1:07:13     
to model we can't model physics correctly if we don't have the right sort of framework so this is a very     
1:07:21     
important uh topic and if people are interested in contributing in any of     
1:07:27     
these areas I suggest you know maybe issuing a poll request in the repository     
1:07:32     
and we could maybe put something together some sort of parallel activities put them on the     
1:07:39     
syllabus uh so the fifth meeting was about pre-train models for biology and this is again you know kind of where we     
1:07:45     
were trying to go with this in part this is the thing that actually led to Diva Le so um we thought about like you know     
1:07:54     
what's the best way to approach biological modeling and using machine learning one of the ways to do that is     
1:07:59     
to pre-train so what you do is you take a bunch of data and you put it into a     
1:08:05     
model and you train it and then you deploy it in different contexts so for example if I wanted to look at cigan     
1:08:12     
development I could train it on a lot of the labels that we know a lot of the basic data and then we can then take     
1:08:20     
that model as it's pre-trained and apply it to uh you know different data sets in     
1:08:27     
C Elegance we can also apply it to other organisms as well and while that has a limited utility pre-training aspect it     
1:08:35     
does give you some information that you can then use to improve the     
1:08:40     
performance so pre-train models this something that we wrote a blog post on     
1:08:46     
um and we posted it and then we ended up pursuing that for the next year's Google summer of code project and we ended up     
1:08:52     
building D from that so this is know sometimes these topics     
1:08:58     
can lead to big things um yeah this is you know a random     
1:09:03     
kind of thing that was put in here this is actually in cognitive computational Neuroscience so interestingly enough     
1:09:10     
Morgan and the me main meeting asked the question about the connection between     
1:09:15     
computational Neuroscience computational systems biology this is actually maybe a     
1:09:20     
connection here so this is from this Paper Co cognitive computational     
1:09:26     
Neuroscience it can be found on the archive and uh this is the figure that     
1:09:32     
they're this is sort of their idea of modeling and this actually applies to     
1:09:37     
not just computational Neuroscience models but modeling more General so on the     
1:09:44     
x-axis we have what we call biological Fidelity which is the sort of the     
1:09:50     
truthfulness of the model or the realism of the model     
1:09:55     
and you know we can go from like the hodkin and Huxley model which is a very basic minimal model mathematical model     
1:10:02     
of conduction and neurons we can go then to a multi-compartment single Neuron model     
1:10:08     
where model a neuron is a bunch of processes that are in different compartments this is much like the the     
1:10:15     
the stuff we were talking about with reaction diffusion modeling and then the blueb brain project which is actually     
1:10:22     
like modeling a brain with neurons with interactions between the neurons and so     
1:10:27     
forth and so you can see on this side we have model complexity so the smaller uh     
1:10:33     
circles are sort of the minimal models the big circles are the of the hyper realistic     
1:10:39     
models and on the y axis we have cognitive fidelity meaning that we have     
1:10:44     
in this case things that incorporate cognition more or less and so we can see     
1:10:50     
that there's we go from the Gabor wavelet V1 model which is a minimal model Ma medical description of V1 which     
1:10:56     
is the visual area in primates we have these uh DNN models or     
1:11:03     
dcnn models these are deep neural networks and they're kind of at a mid-range of cognitive     
1:11:09     
Fidelity and then we have cognitive models which are a little bit more complicated and these are just kind of     
1:11:15     
uh subjective sort of rankings I think cognitive models maybe a little bit higher on this axis but the point being     
1:11:22     
is that you have this cognitive Fidelity of biological fidelity and you have this model complexity and so we can plot all     
1:11:28     
these different types of models out in this way uh and then we have these processes so a lot of cognitive models     
1:11:35     
are modeling two different things they're modeling top- down processes and     
1:11:40     
bottom up processes so top- down processes where we have mental models that you know kind     
1:11:47     
of inform what cellular models are doing so you know we might have labels we     
1:11:53     
might have Concepts we map them them to cells and how they're kind of interacting in a     
1:11:59     
network bottom off on the other hand is where we have the activity of a network or of constituents and they inform the     
1:12:07     
labels and they inform a virgent phenomen so we have this top down and bottom up aspect and you know models can     
1:12:15     
incorporate both but generally they're in one of these two streams and then you have what we you know want to sort of do     
1:12:21     
which is model the brain model cognition and we do this computationally and so we     
1:12:26     
reach this gray circle at the end so that's kind of one of the aspects of you     
1:12:32     
know sort of the meta aspects of machine learning that you'll learn in the course uh then on in the sixth meeting     
1:12:39     
we had a tensor flow tutorial so this is a little bit dated because you know we don't have a s kit learn tutorial but we     
1:12:45     
also tensor flow has been improved since this uh is given so we probably need a     
1:12:51     
new tutorial on that but I think you know the basics of this are all basically going to be correct uh just     
1:12:58     
setting up an environment and running a model in it and that's kind of what we want to do we were involved in hober     
1:13:05     
Fest that year so this was we wanted to solicit some contributions through hober Fest we solicited a few if you look in     
1:13:13     
the repository you know we have our five minute reviews and these are reviews of     
1:13:19     
different topics that you can you know kind of summarize and present to the group and and then you become a     
1:13:27     
contributor in the seventh meeting we had a general discussion this was kind of strategizing about this blog post     
1:13:33     
which is published on the Node which is a Blog for developmental biology uh     
1:13:39     
pre-trading models for biology so this is a something that led to our gso project in     
1:13:45     
2020 we also in eth meeting talked about computational parolia which is this phenomena where models will see patterns     
1:13:54     
in brand sets of of data so you know if you think about like if people see Elvis     
1:14:01     
Presley on a piece of toast or the face of like um you know some famous     
1:14:08     
celebrity on the surface of a tortilla or something and it's just like the like     
1:14:14     
the toast marks where the burn marks on the on the on the uh food you know and     
1:14:20     
but you see this pattern and so there's this phenomenon and it's not not necessarily model hallucination which is     
1:14:27     
a very popular topic but it's very specific to like image data where you'll see patterns in in the image data that     
1:14:35     
look like something else and the computational model picks up on that misclassifies the thing so misclassify     
1:14:43     
some feature in the in the data set as something else we want to avoid that it's you know it's sort of an analogous     
1:14:50     
to hallucination but it's very important it's a very interesting topic too so so     
1:14:55     
we we cover topics like that that are kind of outside the mainstream of a traditional curriculum but are very     
1:15:01     
relevant to theological model then we have developmental Gans     
1:15:07     
and Gans and Medical Imaging so this is uh we had someone present on this where     
1:15:12     
they talked about medical imaging and using Gans and developmental Gans and we've we use Gans in uh Dil learn to as     
1:15:21     
a generative model of embryos is very interesting thing that's been underexplored we do have that capability     
1:15:28     
in the software but we haven't really done a lot with it we can do things with developmental ganss as well and this is     
1:15:35     
Gans are kind of you know have kind of gotten lost in the shuffle of machine learning and deep learning but they're     
1:15:41     
very useful there's a lot of gas left in that tank um they're actually not that     
1:15:46     
old of a method but they really do some interesting things especially adversarial ganss so I guess they're all     
1:15:53     
adversarial so this is something that we could follow up on as a project Medical     
1:15:59     
Imaging of course is just taking IM you know modeling uh you know Finding     
1:16:04     
features and biological image data but developmental Gans are more about like     
1:16:10     
modeling this sort of emerging process that we Define as embryo     
1:16:16     
genetics so we have different slides for that all the slides and videos are available online and I'll show you in a     
1:16:22     
minute where that is on our YouTube channel this is our blog post we had another General discussion in the 10th     
1:16:29     
meeting the 11th meeting we had Game Theory and machine Learning and Development this is actually draws from     
1:16:35     
the Gans literature because in Gans we're basic we're kind of implicitly     
1:16:40     
discussing Game Theory and some of the aspects of uh game theoretic approaches     
1:16:46     
and so this is something that I talked about in a lecture and you know we not only can do this in machine learning but     
1:16:53     
there applications of game Theory to development as well so there's this intersection of game theory in both of     
1:16:59     
these areas then on in the 12th meeting we talked about reinforcement learning in     
1:17:05     
the 13th meeting we talked about biological metaphors in machine learning     
1:17:10     
reinforcement learning and cybernetics so we brought cybernetics in uh in terms     
1:17:16     
of biological metaphors so biological metaphors are very important when thinking about     
1:17:21     
larger systems at the larger system scale because you know we can model     
1:17:27     
things at sort of the feature level we can model things in terms of labels but     
1:17:33     
in development we're dealing with these emergent features that kind of come out     
1:17:38     
of a bunch of cells interacting so as you saw with tissue Forge you have uh     
1:17:44     
two two droplets that splatter and they you know and their components end up all     
1:17:50     
over this field that you're moding so they emergent p patterns that result from that splatter that's a very simple     
1:17:58     
action and has a lot of consequences and it would be hard to identify the splatter that whole process is a set of     
1:18:05     
features and so we need to use metaphors and maybe you know using metaphors like the splatter is like something and that     
1:18:12     
something can be modeled using this method it helps you do a lot of problem solving and finding the right model but     
1:18:18     
it also uh allows you to sort of model the process with know sort of a general     
1:18:25     
uh approach in mind instead of just saying a modeling splattering splattering is also similar to some     
1:18:31     
other thing that's happening biology and you know maybe we have this model or     
1:18:36     
this technique that can sort of you know cross a bunch of     
1:18:42     
areas the 14th meeting uh T talking about this technique called Deep phobia     
1:18:48     
Medical Imaging and Peter's Rule and these were these uh actually these different topics these five minute uh     
1:18:54     
reviews use in kind of going over some topics some techniques like deep phobia     
1:18:59     
and Peter's rule which is a specific uh type of thing in machine uh and then in the 15th meeting     
1:19:07     
we talked about invariance and Universal features and again this is a lot like the input data leure and that it's very     
1:19:14     
much like you know how do you model and what are you doing when you model in this case you know thinking about     
1:19:20     
invariant sometimes we're modeling systems that are invariant so in other words if I go to development I can use     
1:19:28     
techniques that maybe are used for medical imaging or use on embryos from     
1:19:33     
different species and use them in different you know they're they're generalized in that way but sometimes     
1:19:40     
they're not and sometimes features are very specific to a specific data set or a specific organism so we want to think     
1:19:48     
about invariance in Universal features a bit differently in biological modeling     
1:19:53     
and in applying machine learning learning and deep learning to biology because in a lot of that literature you     
1:20:00     
in the Deep learning and machine learning literature we assume invariance or we assume Universal features across     
1:20:06     
data sets and we use benchmarks but you know the assumption is is that we can get features out of the data in much the     
1:20:14     
same way all the time and so what I'm saying here we need to think about that and really kind of test that     
1:20:21     
assumption and then here we have this paper hydron dynamics of emban cell positioning as an example of what's     
1:20:28     
going on here so you know we we covered papers as well as     
1:20:33     
topics so this is um another paper kind of you know excerpt uh artificial neur     
1:20:40     
architectures and a minimization function for the brain and that was posed as a question because you know can     
1:20:48     
we use minimization to model neural     
1:20:53     
architectures does it work on the principle of radiant descent and so this is an example here in this paper a deep     
1:21:01     
learning framework for Neuroscience Nature Neuroscience um and you know this is     
1:21:07     
kind of answers this question so you have this it's theoretical but you have     
1:21:13     
these architectures and you consider you know what is going on in this gradient     
1:21:18     
descent architecture so if you have this architecture you have this space it's a gradient descent um space you use an     
1:21:27     
architecture and you end up with these different aspects of the space so for example when you have an objective     
1:21:33     
function the objective function can either get better or worse and it's either climbing or descending this     
1:21:40     
gradient we also have different parts of the gradient that represent different synapses and how strong they are so you     
1:21:46     
can look at different learning rules and it shapes where you are on the gradient and then of course if you can get some     
1:21:53     
sort of optimization what does that mean in terms of the architecture and how does that map to     
1:21:58     
braate so this is all interesting stuff then the 16th meeting we kind of     
1:22:06     
summarized uh hierarchical temporal modeling in New Directions so this is where we're going in different     
1:22:11     
directions other than just machine learning hierarchical temporal modeling is this approach uh developed by Jeff     
1:22:18     
Hawkin and the numenta group so they're doing things modeling neocortex and the     
1:22:24     
uh layers of neocortex and so you know we we can use that as inspiration for models     
1:22:31     
and the Numa model actually they you know they don't just you know put some     
1:22:37     
sort of Bio inspired model into practice but they're able to tackle a wide range     
1:22:43     
of problems so we we consider some of these alternative models we I think we got a little bit into bio inspired     
1:22:50     
models uh but and neuromorphic models but that's something that maybe we could update the course with uh we also have     
1:22:58     
rethinking the bias variance tradeoff which is a paper which is also important in looking at machine learning     
1:23:04     
performance you want to have a lot of variance in your data set without having bias and so there's this tra famous     
1:23:11     
tradeoff that we discussed and then there's skepticism of back propagation in AI which was a     
1:23:17     
discussion on Kora this was Jeff Hinton posted a question about back propagation     
1:23:23     
and or I don't know if it was Jeff Hinton but Jeff Hinton was the sort of person who develop or did a lot well I     
1:23:29     
guess it was Jeff Hinton so uh Jeff Hinton uh you know poses this skeptical     
1:23:37     
view of back propagation back propagation being sort of this feedback between layers in a neural network and     
1:23:45     
you know whether it's something that is you know either observed in biology or really is something we could rely on so     
1:23:52     
this is some really interesting things bringing things from different areas we also have additional content     
1:23:59     
which we're kind of you know after the course so a lot of things in 2020 and 2021 where we kind of did little     
1:24:06     
lectures and different features on things that were outside the scope of the basic curriculum so these were parts     
1:24:13     
of the lab meetings that we do weekly so we talked about Transformers for biology     
1:24:20     
spatio temporal feature extraction something called torch dreams which was something that was developed by one of     
1:24:26     
our G so students which is interpretable deep learning developmental data for     
1:24:32     
model training neural style transfer resonet voice recognition dietram tracking using     
1:24:39     
object detection this is again that digital Daton project reconstructing     
1:24:45     
continuous biological processes with deep learning neuros cellular autometer     
1:24:50     
ncas this is differential morphogenesis and deep learning for the life     
1:24:56     
sciences so again all these are like you know application domains some of them     
1:25:01     
are maybe less relevant to biology than others but I think any we're really interested in any model that exists we     
1:25:09     
haven't talked about large language models at all but this is something we talked about more recently in our group     
1:25:14     
meetings and so you know I may actually add those segments because these are     
1:25:20     
just segments of the the lab meeting uh so you know it like starts it it's queued up to that discussion so I may     
1:25:27     
add some things into this but I'm looking for people I'm looking to sort of update this curriculum build uh maybe     
1:25:35     
some new lectures or you know really something that's outside the scope of modern sort of pedagogy in this area     
1:25:42     
because a lot of the educational opportunities one has is to learn about machine learning at a very technical     
1:25:48     
level learn maybe about biological modeling somewhere and then you have to put them together what are group does is     
1:25:54     
we put all this together immediately and you know in some way that's uh unique     
1:26:00     
that that you learn insights right at the class instead of having to go outside of the     
1:26:06     
class so H as a finale here I'd like to highlight D ml lectures on our YouTube     
1:26:14     
channel so we have a playlist called dvo ml if you go to our YouTube channel dorm     
1:26:20     
and you look for dorm ML and under the playlists you'll see that we have all these lectur     
1:26:25     
from the course and so these are uh I think all of the week well some of the weeks are not on here but most of the     
1:26:31     
weeks are um we have the inaugural meeting week four week five week six week seven week eight week nine week 10     
1:26:37     
week 12 week 13 week 16 so um if you're interested check it out uh check out     
1:26:44     
some of these video segments as well and some of these models and these are just by our contributors and their     
1:26:49     
discussions on the stuff so it's not you know the ultimate author Authority by any means but it's really our sort of     
1:26:56     
perspective on things and so H I look forward to people contributing to this     
1:27:02     
and thank you
