     
Transcript     
0:00     
leaky uh had to pass a uh a language     
0:06     
exam and uh so they pushed the whole world for an expert in the language he     
0:13     
wanted to pass it in and the only person they could find was     
0:24     
LE um the um Imaging of     
0:31     
the uh salamander eggs would be a good thing to study because um I could I could show you my     
0:40     
PowerPoint on it yeah it's um at some point I can can show it to you sometime     
0:46     
uh I was going to pair it with the uh sucrose gradient and watch the the     
0:54     
salamander eggs rise in the sucos gradient and take image     
1:02     
of my sample salamander eggs as that was happening and then you can see the     
1:09     
inside what the inside's doing while it's um becoming less dense yeah in the     
1:17     
suc so it's a simply fairly simple experiment we have all of the pieces     
1:24     
that you need like I have the sucrose gradient yeah so I thought it was would     
1:30     
be an amazing experiment publishable of course and uh I want to go do that     
1:36     
instead of T segr although I'm keeping on with the T segr if anybody's     
1:42     
interested there's a uh a course in it um in Italy but you can take it     
1:49     
virtually and it's in English um and it's uh let's     
1:56     
see called tens segr system it cut it off     
2:01     
um September 16th to 20th it's a week long and it costs     
2:08     
about $400 Canadian or 300 American     
2:14     
count of course for the course for a week long course I'm taking it virtually     
2:19     
from my living room oh you are yes if they approve me I don't     
2:27     
know they should take it not you oh yeah yeah I'm gonna tell Dr Zang this     
2:33     
afternoon there's his course that he can take he's retiring so he he could take     
2:39     
the course I'm gonna give it to     
2:44     
him yeah welcome to the meeting um looks like ved's here little     
2:51     
Ved hello hello     
2:56     
everyone well Mah and Morgan     
3:03     
dick yeah so I I don't know if anyone wanted to present anything or or discuss anything before     
3:09     
we go ahead okay I've been working on on the     
3:14     
problem that for par that we have been talking about in the past regarding that     
3:21     
those two cells that I'm trying to simulate to be able to as a general     
3:27     
sample or a use case other cell models in C I've been trying     
3:33     
to simulate those two cells and uh yeah I've been working with p and actually     
3:41     
the parer that uh some other projects uh in a recent study they have     
3:48     
published some data model for two uh sample models for two sales in s so I'm     
3:55     
trying to simulate those two cells in neurl and LS which are actually the     
4:01     
basics of the basics of uh our simulations in in uh     
4:09     
world uh I I could somehow simulate     
4:14     
uh try to understand what is the problem because was working on it in the past     
4:20     
but uh I'm trying to solve some problems that we have and uh maybe in the near     
4:27     
future we could add those two simulation of the cells to the open one project and     
4:33     
after that we could uh maybe have have a     
4:39     
great platform for some uh solar simulations that could be uh added uh as     
4:46     
a Dev or any developmental processes going to be     
4:52     
simulated yeah that would be great     
4:57     
um yeah I saw a little of an email chain here with um orig from     
5:04     
openworm so uh is that that basically what you describing yeah yeah actually the     
5:12     
basics yeah good uh well that's great oh hopefully     
5:20     
yeah I move along on that and I don't know if you wanted to you know if you want to sh any     
5:26     
results as you go along you can do it in the meeting here we can I guess we can chat on on the side as well if you     
5:36     
need for me or for oh I mean well for yeah for your     
5:41     
project oh for sure okay uh I was a little uh uh having some     
5:50     
holidays and some relocation so I'm I'm under rooll from     
5:56     
this on and uh yeah I be glad to present     
6:01     
it in the future yeah yeah it's good for sure good luck yeah it sounds good     
6:07     
sounds like uh something yeah uh mahul how are     
6:17     
you uh hi Bradley I'm good so actually     
6:22     
I'm like currently I'm in Hyderabad and next week I'm going to Delhi like uh so     
6:28     
I actually told I'll be going to Japan for the conference in June so I'm preparing visa     
6:33     
and all next week so I'm traveling to capital of India get so a lot of things     
6:39     
are going on in that side like it's my it will be my first offtime conference to attend so I'm very     
6:45     
excited about it I'll be look out for like computational biological networks as well in the conference there any     
6:51     
happening there yeah keep you updated on that yeah what conference is it uh it's     
7:00     
wcci IG CNN so igcn is a conference but like three four conferences are happening together okay actually I was     
7:07     
taking that only if anything is related to like development biology but I didn't find anything related to it but there     
7:14     
are many brain related uh uh keynote speakers and workshops are happening so     
7:19     
brain development and all those things so I think Morgan will be interested in that oh yeah yeah that sounds     
7:28     
great that's great and yeah oh go ahead yeah nothing else I'm just like     
7:35     
into that stuff only not doing anything cool right     
7:40     
now yeah well that's good yeah so thanks for that update yeah your presentation last     
7:48     
week was pretty good um yeah I'm looking forward to going over the     
7:54     
proposals and seeing who we can get and U definitely want to work on that um     
8:01     
project um yeah so let's see I I have a     
8:06     
a special here and I don't know if people realize it well dick probably is the only person who would know this or     
8:12     
maybe remember this but it's the 10th anniversary of dorm so really yeah this     
8:21     
week so I I got some cake to eat today oh     
8:28     
yeah good so yeah I'm gonna why we've been at it that long huh yeah yeah 10 years     
8:36     
2014 so yeah 10 years of computational Developmental biology so I'm going to     
8:42     
make us uh talk on this and put it on YouTube like it's going to go over kind     
8:47     
of the activities over the 10 years a recap of things you know all the different papers that we put out all the     
8:53     
different things that we've done we've done uh Google summer of code for probably about     
9:00     
um maybe about seven years now so it's been a while I think this will be the     
9:05     
seventh year and so we have a rich history there we have a lot of presentations that we've done a lot of     
9:12     
Publications a lot of meetings where we've talked about things and and gotten things off the ground so yeah I'll be     
9:18     
putting a a longer probably about an hour long talk on YouTube sometime this     
9:24     
week I have it almost ready but I have to record it as well so this is uh you     
9:30     
know we're we're doing this computation of Developmental biology it's this open- Source participatory model we have our     
9:38     
different modes of doing things so we have GitHub YouTube hugging face the     
9:45     
openworm foundation the orthogonal research lab and then use jitsy as a way     
9:51     
to do video conferencing so um go ahead maybe we should have maybe     
9:57     
we should have a meeting next week didn't let you have a practice maybe     
10:05     
yeah okay then we can put in our true     
10:10     
yeah uh so this is the first so the way this started um this started like in     
10:15     
April 2014 so I think dick actually got in touch with Steven Larson or something     
10:22     
like that and uh I don't remember okay yeah well Steve lison you know back then     
10:28     
was all committed you know he's very committed to openworm and he's been spending more time with his startup meta     
10:34     
cell but back then he was managing like issues on open worm and uh and so they     
10:40     
were doing things with the worm simulation and they're doing things with pi openworm which is uh just the python     
10:49     
version of openworm uh so there was this open issue in their GitHub repository was issue     
10:56     
number 179 and uh this was actually something he opened in response to what we wanted     
11:02     
to do so we had a meeting with Steve lorrison and he talked about the you know basically incorporating something     
11:10     
into openworm that was developmental so there's this classic data set from     
11:17     
um uh Solon and you know he he actually sat there at his uh in the lab and he     
11:24     
drew out a a lineage Tree by hand so he looked under the microscope and he     
11:29     
looked at celegans the different cells and he was in in a an embryo and he was able to actually map out the different     
11:36     
cells in the embryo so um and this was you all done by hand     
11:42     
it was drawn so if you see a uh any sort of lineage tree know that that was done by hand uh and and back in the 70s so it     
11:50     
wasn't that long ago actually uh you know you'd think it' be a little bit higher Tech back then     
11:57     
but so this four years ago yeah which you know your age yeah I mean you know     
12:05     
historically like it wasn't in 19th century or something um so we we had a     
12:12     
an issue here that we wanted to build here and uh there were a couple things     
12:18     
identify the best data sources to begin working with write a simple python script that pulls out the names of the     
12:24     
cells and the cell division relationships and displays the console     
12:30     
augment the script to dump the data into Pi openworm SQL light database augment     
12:36     
pi openworm to pull these data from a SQL database and construct an rdf graph     
12:42     
which is a data structure out of it and then build accessor functions in pi openen orm so we     
12:48     
actually uh didn't actually go in this direction but this this issue was sort     
12:53     
of the beginning part uh what ended up happening was we ended up kind of diverging from like the openworm     
13:00     
foundation a bit and doing your own thing which is fine but you know um that's what happens a lot of times in     
13:07     
open source you get Forks we took a fork off of this uh you know I don't know     
13:13     
whatever happened to this issue but you know it's it's a good when you want to start something like this it's good to     
13:19     
lay it out in an issue and then see where people want to go with it so you     
13:24     
know we uh we actually that's what we did with uh D learn is we've kind of     
13:29     
forked some of the work we're doing in um and I'll and I'll show this in the     
13:35     
presentation with uh like kind of our machine learning applications and kind     
13:40     
of G went off in a in a forked Direction instead of like just assigning uh     
13:46     
in-house issues so so this was our first meeting April 11th we used Google     
13:52     
Hangout so that actually was pretty long ago because Google Hangouts is no longer a thing uh but that's that's how long     
14:00     
ago it was and still Steve Lon is here myself and and Dick were in the meeting     
14:06     
and uh and then we expanded our group over time to include new people some     
14:12     
people left some people uh came in you know there have been a lot of people who     
14:17     
have uh gone you know come and gone and so and that's the nature of this type of     
14:23     
group you know you get people visit for a while and then get what they need and leave and then some people stick around     
14:28     
for a while while and and so that's but that this is where it started on April 11th     
14:35     
2014 so we made a I think I made the first public announcement on my personal     
14:40     
blog on June 3rd 2014 that was in the summer uh you know this is a     
14:46     
presentation I think we did with uh the openworm foundation so back then they     
14:51     
used to have talks public talks that they did um you know where they would     
14:57     
have some of the uh sub projects give talks on their progress and it was much     
15:03     
more interactive than it is now but um you know that was sort of the nature of the project at the time and uh so we did     
15:10     
a couple on Diva worm um and so you know we took that solsten data set and ran     
15:16     
with it we had other data sets as well like cell tracking data and other types of data and so that really built the     
15:23     
early days of the project uh now we had also talked about things like um     
15:30     
uh finite element analysis and cellular autometa and things like that and we did actually follow up on the cellular     
15:37     
automata stuff you know uh we we did some work there we could do more uh we     
15:43     
never really did anything with the fin an element analysis stuff but we did you know do some things with the uh with the     
15:51     
lineage tree data with some of the cell tracking data so you know they're always     
15:58     
uh really based on the secondary data sets that other people have collected and that's really what open worm's done     
16:05     
as well they've done a lot of things as secondary data then celegans you have a a wealth of that kind of information     
16:12     
people have collected a lot of data but not really done anything like theoretical with it or you know the     
16:18     
modeling is you know there's a lot of modeling work that should be uh done     
16:23     
that people aren't doing for various reasons so that's uh the public announcement by then we had Mark Watts     
16:31     
who was a senior contributor at openworm I think he quit openworm a while back     
16:36     
but he's he worked on a lot of stuff with respect to like modeling uh like     
16:42     
building uh data structures and working in Python and things like that and then     
16:47     
Steve McGrew who sadly has passed away by now uh he also joined the group and     
16:53     
he was kind of uh one of these people who you know had a lot of talents     
16:59     
and so he was really interested in the project and he worked on he joined our group by that point as well so there's a     
17:07     
lot of stuff that came after that we had an open house in 2016 we started uh Google summer code in     
17:14     
2017 we got some papers on 2016 so there were a lot of things going on and     
17:19     
throughout it all I think we've kind of maintained a you know um I think we've     
17:25     
maintained things pretty well um and then our first publication which was October 3     
17:30     
2014 this is on the bioarchive uh this is differentiation waves and computation     
17:37     
it's kind of like you're putting our uh flag down and and marking some territory     
17:43     
so this is you know kind of the first thing we did uh this is where we proposed a 4D model for the seans     
17:49     
lineage tree using the rdf framework which is a tabular data structure that     
17:55     
sort of you know ties together a lot of the lineage data with some other data     
18:00     
that exists for SE elegans development so I haven't gone back to that paper and     
18:06     
read it again but I remember that was the first thing we did and so I had uh     
18:12     
you know after that we did a you know we did other presentations before that but     
18:18     
so so Bradley yeah the uh uh the musing thing it was     
18:24     
it took uh oh for 24 years to get something like this going     
18:31     
because uh in my 1999 book uh hierarchical genome you'll find a     
18:38     
description of cyber worm oh right yeah cyber worm was in a sense its     
18:45     
predecessor which I couldn't get off the ground I had somebody interested in then fade it     
18:50     
away yeah okay so it took me and you to get it going yeah yeah     
18:59     
it's interesting is there have been a lot of people proposing that you do things to see elegans like there are a     
19:06     
lot of I I went through like the web and I knew this was happening because I've seen papers on it but we did this paper     
19:13     
a couple weeks ago in meta worm which was an attempt to take some you know different tools and apply them to     
19:19     
analyzing celegans and it turns out there have been a lot of uh attempts at this over over the years not so much     
19:26     
development but like modeling seans and so like you know there was the perfect sea elegans project question yeah Tom     
19:35     
pus create a uh a robot that was like uh     
19:41     
SE yeah he did well yeah that was something that uh so there was an open house at uh the Royal Society in 2018     
19:50     
and that was openworm uh openworm sponsored that and he actually developed some robots that that time 2017 2018     
19:59     
he was really deep into working on robots and they actually did develop a robot um that was you know was like a     
20:06     
big it wasn't like the size of seans it was scaled up so you can interact with it um on the tabletop but uh it uses the     
20:15     
algorithm that they have for movement so you you put in the algorithm you could run this robot and it would have like a     
20:21     
lot of the same behaviors as the sea El and so and there's a lot there to do Cu     
20:28     
uh there was there were a couple people who were doing work they were contributors well this was back in 2018     
20:34     
where they were looking at Soft Robotics where you have soft bodies so you know     
20:39     
cgans of course is a soft body like every other organism and you know you could build robots with you know um     
20:46     
metal or plastic but it's like you can use more exotic soft materials to get uh     
20:53     
sort of compliance you need to make some of the you know movements like Omega turns and things like that     
20:59     
and so they they were working on that I don't know if that uh went any further because people were working on it for a     
21:05     
while and then it stopped and I know they were talking about selling a kit I don't know if that's still around but uh     
21:12     
yeah it's it's definitely been explored and it's it's an interesting kind of model because you know you can test     
21:19     
hypotheses about movement about biophysics that you couldn't do just by doing things in a computer you know like     
21:26     
the whole thing you a comp computational simulation you can actually take the uh     
21:33     
the movement program you can put it into a body of some type and it's you know     
21:38     
interacting with the world and so it's you know seans isn't moving around a tabletop but you can do things that are     
21:47     
you know you can actually see things it's it's they they were using it for Education uh that was the initial     
21:53     
application domain but you could easily use that for research I you know it's not     
21:59     
you probably learn quite a few things from     
22:06     
that yeah so yeah so U that's that was the     
22:12     
first publication yeah so that was uh that's I'll have a a a full presentation     
22:17     
you know pretty soon I'm going to finish it off um and then we'll we'll see we'll     
22:24     
kind of go over it and see how much progress has been made CU it's it's     
22:29     
pretty surprising to me anyways going back over it and seeing all the things     
22:34     
that we've done so yeah and how far we have to go oh yeah     
22:42     
yeah there are a lot of things there um yeah there's always more to do I     
22:47     
guess but     
22:53     
um that's great um okay well maybe we can get a uh     
22:59     
practice session next week yeah yeah so we had a couple things here     
23:09     
um so there's this interesting paper that came out uh I think it was like a couple weeks ago     
23:15     
maybe uh I can't tell where it was maybe March 22nd or so so it was wait last month and     
23:24     
it's in the pnas uh it's a research article in pnas and this is cell reprogramming designed     
23:30     
by transfer learning a functional transcriptional Network so this is cell reprogramming which is a little bit     
23:37     
different than what we talk about in cgans it's uh where people are do you know they're trying to transform cells     
23:44     
from uh like something like a skin cell to a stem cell or a a stem cell to a     
23:51     
neuron or something like that and what they usually do is they use transcription factors to drive that     
23:58     
process so they introduce transcription factors into the cell they transfect them in and     
24:04     
then they can get the cell to transform in some way and so the problem with that     
24:09     
technique is it's it's great because you can create cell populations in the lab     
24:14     
that you couldn't create uh otherwise sometimes you can't get access to     
24:20     
certain cell types uh but the problem is it's somewhat inefficient and people     
24:25     
have not like people have not I don't know if they've not been honest about the efficiency but um it's not something     
24:33     
that's you know easily uh it's not easy to do in the lab and you know people     
24:38     
want to use this eventually for therapies you know for for uh cell     
24:43     
transplantation or things like that but also for research and having like a stable model so that's that's where     
24:51     
they're coming from this Adon modder is is well known in network science for some of the work he's done with uh you     
24:58     
know networks but also biophysics then Thomas White talk is an author on this     
25:03     
uh so this is uh so they're talking about cell programming design which is where you're designing sort of a     
25:10     
protocol for the cell reprogramming and then transfer learning of functional transcriptional networks     
25:15     
this is where you're trying to learn you know these transcriptional networks and     
25:21     
how they can be manipulated using key transcription Factor so if you upregulate a certain transcription     
25:27     
Factor you'll push the cell towards a certain fate and so you know if you go     
25:33     
back to the work of yamanaka and the four transcription factors that's the work that won a Nobel     
25:40     
Prize they basically took four they they did a bunch of uh screens of stem cells     
25:46     
and they found that they're four uh genes that control what they call stemness in cells and so they applied     
25:54     
those four transcription factors to a cell they put them in the cell they over     
25:59     
over regulated those genes and they were able to push the cell into a new fate which means that there are these     
26:05     
transcription factors that sort of control the process and and ultimately     
26:10     
control sort of the nature of these of these uh uh reg regulatory Network these     
26:17     
transcriptional networks and you know it's not it's not common you have to do     
26:22     
a lot of screening but you can find genes of large effect that do this so that's kind of what they're trying to do     
26:29     
and that's just for stem cells if you want to transform say stem cells to neurons you have to have a whole     
26:36     
different set of transcription factors and then you know it gets a little tricky because there are a lot of     
26:41     
different types of neurons and you you know there are a lot of different ways to do it so this is where they're coming from on     
26:48     
this so if we look over here at the significant statement of this paper uh     
26:53     
over here you'll see that the they say the lack of genome wide mathem matical     
26:58     
models for the gene regulatory Network complicates the application of control     
27:04     
theory that manipulates cell behavior in humans so what they're trying to do is they're trying to develop some sort of     
27:09     
genomewide mathematical model and this is to look at Gene regulatory networks so you know you want     
27:17     
to build a mathematical model of a gene regulatory Network and typically this is some sort of model where you have genes     
27:25     
that interact in some way so you know it's usually like sort of causal so you     
27:32     
have one gene that maybe upregulates another Gene maybe another Gene that regulate down regulates another Gene and     
27:38     
so forth and so you can see these you know you can draw these huge networks on the board where one gene affects another     
27:45     
Gene but the thing is is that you know they're not just simply one thing     
27:50     
affects another thing there are a lot of intermediate uh things going on there's     
27:55     
a lot of mixed uh veilance so for example you know one gene might     
28:00     
upregulate something or it might uh you know cause another thing to be upregulated but there might be other     
28:07     
genes that downregulate that thing so it's hard to know which thing has the     
28:12     
most effect if you if you manipulate it in some way or if you overexpress     
28:17     
something in some way how much of an effect does that have so that's what they're trying to do build a     
28:23     
mathematical model of this uh so they're trying to be able to do this uh by     
28:28     
developing a transfer learning approach that leverages genome wide transcri transcriptomic profiles so they're using     
28:36     
data from uh you know transcription so they're looking at genes that are     
28:42     
upregulated or down regulated in cells and you can get these you know     
28:47     
genomewide uh transcriptional profiles and you can say if something is up regulated or down regulated in a cell     
28:55     
and so those profiles you can use those to sort of learn what you know constitutes the cells what     
29:02     
what kinds of genes a cell is expressing or you know how much uh you know if     
29:07     
they're genes that are really highly expressed that's usually a signal that they're involved in some sort of     
29:13     
function or you know something like that so that's what they're trying to do here     
29:18     
so they're trying to build these cell type ATT tractors and perturbation responses so they're trying to figure     
29:24     
out also not only the identity and the function but what happens when you perturb a cell you know what are the     
29:31     
things that change what are the things that need to change if you're going from one state to another state what are     
29:36     
these kind of things so uh that's that's where they're kind of coming from they're using this transfer learning     
29:43     
approach these responses are used to predict a combinatorial perturbation     
29:48     
that minimizes the transcriptional difference between an initial cell and a     
29:53     
Target cell type so what they're doing is they're building this model where you have uh let's say a stem cell on the one     
30:01     
hand and a neuron on the other hand and you want to find the transcriptional     
30:06     
profile for each cell type and then you want to find the difference between those two cell types in terms of their     
30:13     
transcriptional profile so which genes when you look at their transcriptional     
30:18     
profile change the most so some of them are going to change a lot some of them     
30:23     
are going to change a little bit and some of them are going to maybe change a little bit but be negligible and so     
30:30     
forth so we want to identify the things that are sort of you know uh different     
30:37     
but also the same and then we want to be able to predict perturbations that minimize that     
30:44     
difference so we can actually once we do that then we can build a model of regulatory Network we can bring that     
30:51     
Network to the Target cell type based in the retraction we anticipate that this approach will enable the rapid     
30:57     
identification of potential targets for treatment of complex diseases so this is where I was     
31:03     
saying that they want to do uh things in the clinic with cellular reprogramming     
31:09     
but there are a lot of barriers to that in terms of knowing how the cells are going to behave especially in Vivo and     
31:15     
so it's very hard to know these things and that's why we need mathematical models for that while also providing     
31:21     
insight into how the Dynamics of Gene regul regulatory networks affect phenotype so this this is something else     
31:28     
we don't really know a lot about is how these networks affect the phenotype um you know how do you you     
31:34     
know when you how do you go from a Network that has you know uh all these     
31:40     
genes fluctuating up and down how do those translate into a phenotype and     
31:45     
obviously you need to have go through several more steps you need to go through translation you need to go     
31:51     
through post transational modifications you need to go through protein construction and protein folding and     
31:57     
things like that to get a phenotype but this this is a you know a good step in the right direction     
32:04     
here so in the paper they talk about synthetic biology and some of those     
32:09     
applications and they just basically talk about you know some of these protocols to control cell behavior that     
32:15     
people are starting to develop with respect to synthetic biology so if you're going to do synthetic biology you     
32:21     
need to be able to control your cells and know where you know if you make a manipulation what the effect of that man     
32:27     
iation will be need to be able to predict that um and so you know it's hard to do this     
32:34     
with Brute Force trial and error so a lot of times the way we do these kind of     
32:40     
uh assays of sort of perturbation assays is to do like knock knockout experiments     
32:46     
where you look you knock out different genes and you look at their effect and but that can be very time     
32:52     
consuming same thing goes with like screens so if you do a genetic screen you look at like you know uh maybe you     
33:01     
test for a bunch of different transcription factors at once and you try to isolate the ones that are the     
33:08     
most that change the most and you do more tests where you uh try to manipulate its expression to see you     
33:14     
know what kinds of under what conditions it's it's having a large effect those     
33:20     
things are all very timec consuming and so it would be nice to have a mathematical model that would streamline     
33:25     
that process and so uh they they talk about control theory being you know an     
33:31     
engineering discipline and how in biology you need to you can use control theory but you need to have you need to     
33:38     
approach in a different way so unlike engineering Sy engineered systems in     
33:43     
which control theory provides an equation-based framework to design interventions so in control theory and     
33:50     
Engineering we have equations that predict the outcomes of systems that we can control something like an industrial     
33:57     
process we can control you know speed of something or the trajectory of something     
34:03     
but you know in biological systems we don't really have those descriptions we don't have like equations of motion for     
34:10     
transcription or for you know cell behaviors uh we'd like to be there but we're not there yet so you know one of     
34:18     
the problems with uh building such models is that you you know it's hard to     
34:24     
constrain the numbers actual degrees of freedom in terms of genes uh to describe     
34:31     
a cell so when you look at genes you have a lot of genes they're being expressed in different ways those are     
34:38     
basically akin to degrees of freedom so you have this massive number of degrees of freedom that you need to constrain uh     
34:44     
as you uh try to you know do engineering or build a mathematical model and so     
34:49     
it's very hard to control those because you simply don't know what you know if you hold something constant whether     
34:56     
that's the thing you shouldn't be holding in constant so it's very hard to do this um in an ed educated way I mean     
35:04     
you can take educated guesses but it's you know that's it's like sticking your finger in a dam it's just not not     
35:11     
feasible uh Without Really some good models so the the features presented a     
35:18     
challenge given that underactuated control is owner us even in physical systems that admit a close for     
35:24     
mathematical description so this is where talk about underactuation which is the way they     
35:30     
view biological systems um and you know you could say that they're also under specified another way of saying this and     
35:37     
this is a problem of course in modeling where you know we'd like to have the right level of specification things are overp specified     
35:45     
there's too much um you know there's maybe the system is too rigid if it's     
35:52     
under specified then the system isn't very predictive and so this is something     
35:57     
we can't really find the uh you know middle ground on so this is this is a     
36:03     
problem as a result the biological control problem is often relaxed to steering between natively stable States     
36:11     
so you know in other words we know kind of how what the stable states are maybe     
36:16     
which is not really biology uh so we know say what a we know     
36:21     
maybe what an idealized stem cell is we know what our model stem cells are we know what like the wild type is uh that     
36:29     
we work on in the lab but if we look at like biological diversity we know that there's a lot there are a lot of these     
36:35     
unstable states that are also very useful and interesting and especially when you move between different cell     
36:42     
States you move between a stem cell of some type and a neuron of some type well     
36:48     
that you know is those are Le nominally stable States but you actually go     
36:53     
through a series of unstable States furthermore a stem cell could be sort of an uh you know kind of an off stem cell     
37:01     
or maybe an unstable type stem cell and you could get you could end up with an unstable type neuron and you don't     
37:09     
really want to be in there or maybe you do depends on what your end goal is the point is is that we know how maybe how     
37:16     
to go from the idealized stem cell to the idealized neuron but we don't know a lot about how to control the system to     
37:23     
move to maybe unusual States or un stable     
37:29     
States so they you know they they talk about their goal here is to design a general data driven control approach     
37:35     
tailored to these aspects of the data and their approach contrasts with related control theoretic formulations     
37:42     
in the literature which cannot be easily deployed as they usually require targeted experiments to design the     
37:49     
controller temporarily Rich data individually or resolved system trajectories and the availability of     
37:56     
microscop agent based models so this is these are citations 18 through 22 so 18     
38:04     
is this paper Danny uh Danny bass it's an author on it data driven control of     
38:09     
complex networks so this is this area of network control where they're looking for certain nodes that control the     
38:17     
network that's an interesting area of research uh 19 is generalizing coup and     
38:23     
Theory to allow for inputs and control this is dynamical system uh Theory but this is also something     
38:30     
that is not you know this is uh Steve Brunton is an author on this uh so again     
38:36     
it's it's you know generalized sort of uh control theory but you know it's it's     
38:41     
maybe something we can use in biology um this paper here model free     
38:46     
control of dynamical systems with de Reservoir Computing so Reservoir Computing you know for the machine     
38:53     
learning people you know it's it's a it's an upand cominging technique that people are using to model biological     
39:00     
systems connectomes things like that I've seen a number of talks where they kind of you know highlight Reservoir     
39:07     
Computing as an upand cominging approach I've seen talks where they've done you know things where they've modeled     
39:13     
connectomes where they've modeled in the inside of cells and basically what a reservoir computer is is a sort of a     
39:20     
randomized neural network and the idea is you run things through a reservoir computer and it has a reservoir of nodes     
39:27     
and and and weights so it puts together the answer it emerges from this sort of     
39:33     
reservoir and then it gives an answer and it's something that is I mean at     
39:38     
least to me it's a black box I'm not quite sure I understand how to implement it but it's something that is you know     
39:44     
people are kind of there's a lot of promise there uh this is 21 this is Danny bass     
39:51     
it's also an author on this teaching recurrent neural networks to infer Global temporal structure     
39:58     
from local examples so this is where you're trying to get uh predict Global     
40:05     
temporal structure uh again this is just generalizing things and then this 22 is     
40:12     
data driven control of agent based models an equation variable free machine learning approach so this is again uh     
40:20     
you know working now with agent based models and using data to control those     
40:25     
and and control theory and all that so there's a lot of stuff in here a lot of different topics Network control we     
40:33     
of course a reservoir Computing makes an appearance dynamical systems and even some other types of uh     
40:42     
you know mathematical approaches like agent based modeling so there's a lot of math and a lot of uh computation here     
40:50     
that they bring to this paper uh it's yeah it's somewhat inscrutable in Parts I guess I didn't really go through the     
40:57     
det details but um this is the kind of work that people are doing especially like in uh you know in in     
41:04     
biophysics on this on these topics so um I don't know if anyone had any questions     
41:10     
Let's see we have some things in the chat oh yeah mayle said uh there is a student working in     
41:17     
Reservoir Computing in my lab yeah it's a topic people are picking     
41:22     
up on it but it's not I I don't know how far we are along it's not as common as networks and things like that what what     
41:30     
is Reservoir um Computing so it's yeah so     
41:37     
it's kind of like you know it's based on neural networks and you have this uh     
41:42     
this model that's a reservoir and it has like a lot of nodes in it and a lot of edges that are unorganized so it's like     
41:49     
there I don't want to call it there's a another technique that uses that terminology but basically you feed data     
41:55     
into it and it makes connection based on the what what's coming into it uh so     
42:01     
it's learning basically the network topology from the data coming in and so     
42:07     
you know it sounds like you know your typical neural network but it actually has like this Reservoir where you're     
42:13     
just kind of drawing from what's available the computational resources that are available and it's uh exporting     
42:20     
an answer and so you know I guess it doesn't have as much I don't know what     
42:26     
the the uh the advantages of it are but     
42:31     
um yeah so it's it's it's that kind of a thing it's sort of an un supervised     
42:36     
model I guess um but it it's it's it's related to neural networks but not quite     
42:42     
the same thing and people are making a lot of connections between that and complex networks where we have you know     
42:49     
networks that are based on some sort of data set where you're finding the connections you're finding like no uh     
42:56     
hubs of no you know nodes that are hubs and nodes that control processes so in     
43:02     
the seans connectone for example they're hubs that control different parts of the     
43:08     
connectone so a lot of nodes that are Downstream of that Hub have to go     
43:13     
through the Hub to get to another part of the connectone so it controls that part of the connectone and so I think in     
43:22     
uh Reservoir Computing the neural networks can actually resemble that kind of hierarchical control I'm not really     
43:30     
sure how people are are implementing it though a lot of different ways it's kind of one of these newer techniques where     
43:36     
there no rules well I guess there's some rules     
43:41     
but it certainly isn't as established some of the neural network     
43:52     
techniques okay oh thanks that's quite involved yeah I     
44:01     
know so yeah um I know a bit about control theory but just sort of on a practical     
44:09     
level yeah yeah yeah you can build pretty involved models of     
44:17     
course some they're definitely being used like     
44:23     
like I'm not exactly sure either but there's there's some connection with r Matrix Theory yeah and um and it it is     
44:32     
used in kind of specialized um high performance Computing scenarios too great like like     
44:40     
it definitely has a connection to recurrent neural networks but but there's there's there's definitely some     
44:47     
because again like the light on folks and Paris are using this with their     
44:53     
photonic Computing and um and then we saw it with the the brain Tome or go forget what     
45:01     
that awful name was the organoids the organoid doing Reservoir     
45:09     
Computing to do like sound sample     
45:14     
identification okay that was a recent paper if if you remember that I remember anyway but again it seemed like like     
45:22     
they were kind of using the organoid to do this this r     
45:27     
random uh reconfiguration of the inputs right and     
45:34     
then you know having the original and this this new configuration was part of the part of     
45:41     
the processing stream which again I'd really like to     
45:47     
I'd like to know more but yeah it has some useful properties     
45:55     
but they're they're seem to be quite spec you know certainly not as like     
46:03     
a as as kind of you know wide purpose as as general General networks right well     
46:12     
yeah I mean that like they they have that aspect where you're building up from random so you're not like it's almost like a complex Network where you     
46:19     
compare the topology to a random topology and you say what is the structure of this thing given the random     
46:25     
topology is like kind the ground state in a uh Reservoir computer you have this     
46:31     
random set of connections and you're saying what is the structure that the     
46:36     
data is introducing to this so it's kind of like you're building the structure from     
46:42     
that yeah it's like almost like a dynamical you're throwing a dynamical system at it     
46:51     
yeah yeah yeah um yeah     
46:57     
it's like I I see National Labs using it and like not a lot of other groups and I don't know what that means     
47:06     
well probably means there's a lot of computational power that's required to really run them     
47:11     
but well yeah it does I mean again like I I'm curious what the light on folks     
47:18     
would say because it's like I I do feel like it's like you've got to do this at     
47:23     
a scale that makes it useful you     
47:30     
know and and this this kind of light parallelism allows you to do that you     
47:37     
know to to somehow exploit this random this random projection but again yeah     
47:45     
love to know more love to have somebody who understands this     
47:50     
explained yeah I I I thought that they had some pretty good insights about uh     
47:56     
control theory and not being an expert at control theory but I I think you know     
48:01     
that's my cave but I mean the idea that you know if you have in biology and we've actually talked about in in the     
48:08     
other group we've course talked about you know uh the cyberneticist and how they kind of viewed biological control     
48:15     
and it wasn't really a for I mean it wasn't as formal of a theory as control theory is now but there's special     
48:22     
dispensations you have to make for biology as opposed to like control a dam or controlling a nuclear power plant     
48:29     
that you have these parameters that are you know it's a really UND specified system so we don't know like a lot of     
48:35     
things about uh like transcription and sometimes those are just stochastic variables or parameters so you can't say     
48:43     
like this is a you know we can just learn more about it because it's just inherently stochastic so that's that's     
48:49     
something you have to you know but we don't really know which things are which so it's very underspecified so that's an     
48:55     
interesting point that you know in the sense that you can actually apply control theory to     
49:01     
something like that how would you do it yeah yeah well I'm I'm curious how     
49:07     
how this relates to this this regul paper I put it in the um uh Diva Worm     
49:16     
orthogonal but I I'll put it in the D.A worm slack too yeah and I'm I'm     
49:24     
curious you know this this uh Gene regulatory Network you     
49:31     
know framework that or you know analysis framework that they they put together     
49:38     
that that seems Seems related yeah there anything about Gene regulatory networks is that like a lot of people have like     
49:47     
uh kind of run them like they kind of designed them and they say this is how it works but it's hard to simulate them     
49:54     
and it's hard to apply them to like real world data so there's a I think people     
49:59     
are making progress on it but like traditionally it's been very hard to get like good solid answers on it     
50:05     
so hopefully you know some of the techniques people are     
50:10     
using again this is this is why we need a lot of good a lot of good organoid uh     
50:16     
you know publicly available organoid work yeah so a few words on genetic regulatory notebooks so as I mentioned     
50:23     
in the meeting a genetic regulatory notwork something like this where you have     
50:29     
individual genes and the genes are expressed usually you have this model where you     
50:36     
have some sort of promoter at the front you have a gene which is the square any     
50:42     
one of these rectangles and then you have this arrow that comes up which denotes the expression product comes out     
50:50     
of so D drawing these bubbles kind of as a you know a cheeky cartoon     
50:57     
but these are these genes are also connected so this Gene up here is connected to this Gene down here and     
51:03     
it's a causal relationship so this Gene a will cause Gene B to be upregulated     
51:09     
Gene B will then will Express something so a proceeds B but a triggers B to be     
51:17     
expressed now you can also have another Gene here C which is here can     
51:24     
have products and you can see that c is induced by B     
51:31     
so we have a inducing b b inducing C and you have this circuit that is sequential     
51:38     
um gene expression but it's not so simple because first of all we can have     
51:44     
inhibitory relationships as well so we can have an inhibitory relationship between B and C and that will shut down     
51:51     
expression of C so if a is upregulated it's expressed then it will trigger B to     
51:58     
be expressed as well but the expression of B then inhibits the expression of     
52:04     
C so we can have our diagram here where a contributes the expression of b b gets     
52:12     
expressed but then it inhibits the expression of c and so a is going up B     
52:18     
is going up and C is going down and so a lot of times in gene regular Network     
52:23     
models you can get real data to plug in your gene expression data and you can say that you know a B and C are related     
52:30     
and you can see the sort of polarity of the model as you work through it to see what's being inhibited and what's being     
52:37     
sort of encouraged or excited by by Downstream or Upstream     
52:42     
components but it's it's even less simple than that because you actually have other     
52:48     
genes that are affecting things so a lot of times gene expression networks or     
52:53     
Gene regulatory networks are you know kind of extracted from real data they're     
52:59     
extracted from the literature and studying the effects of single genes and sometimes people will build models that     
53:05     
just are kind of like maybe a few genes and maybe the genes of largest effect but you have a lot of other genes here     
53:11     
you might have a prime you might have B Prime and you might have a lot of minor     
53:19     
players that actually maybe aren't expressed uh but they are still     
53:24     
contributing expressed at least in terms ter of their own function but they still play a role in the expression of B so     
53:31     
maybe B Prime is inhibitory so if B gets expressed which isn't very often it     
53:37     
inhibits B so a can contribute a signal a prime can contribute a signal and B     
53:43     
Prime all contribute signals the signals remain a prime or or facilitory uh and the uh signal from B     
53:51     
Prime is inhibitory so you can actually add those together in a in a model where     
53:56     
you consider all the effects collectively and of course A lot of times different genes will have     
54:03     
different contextual effects so for example if you're in a certain environment B Prime might be expressed     
54:09     
an inhibit B in in the face of the expression of a a might also get shut     
54:15     
down on other environments which affect the expression of B and so on and so forth so these can become very complex     
54:22     
and very detailed very quickly now we also talked about resir computers and so     
54:28     
to be very graphical about this Reservoir computer is in a reservoir     
54:35     
it's sort of a blackbox model and So within that black box or at     
54:41     
least coming into the black box we have an input and we have an output coming out of the black box so we have an input     
54:48     
and an output and this input and output then are shaped     
54:54     
by what's in the black box so the black box contains this randomized network     
54:59     
this is what Morgan was referring to when he talked about the connections     
55:05     
to random Matrix theory is that you have basically this randomized network in     
55:10     
here and it's basically represented by a randomized matrix now what happens when     
55:15     
you get an input of data is that it shapes this network so that it takes on some sort of uh topology that actually     
55:23     
can process this information and provide some feedback so this is something that     
55:29     
you know can be shaped from a random     
55:35     
Network to something that's non- random and so we think of our you know     
55:42     
weight matrices and our neural networks is being non- random it's it's selected by you know     
55:49     
the data and shaped by the data and so we have this transition from random to non- random and this is what we see in a     
55:56     
resir computer is we have this reservoir of nodes and this reservoir of edges and it get shaped by the data and it gives     
56:02     
us an answer which is of course the answer uh that we might expect from like     
56:08     
sa with anal Network now there's some also some interesting Connections in the in the paper with     
56:15     
with network control Ability Network     
56:21     
control and and sort of these changing topologies     
56:28     
and so there was actually um paper just came out was from uh a group of people     
56:34     
including Mike 11 where they're looking at genetic regulatory networks which we     
56:39     
talked about earlier and their competencies and so their competencies are really based on this notion of     
56:45     
network control and changing topologies so if you have a network that uh you     
56:51     
know is has a number of components and they have a number of uh effect sizes so     
56:57     
say for example this is this a is uh feeding into B the expression of a is     
57:04     
you know or the expression B is contingent on a and so forth but they have different effect sizes so there are     
57:10     
other things feeding into b and a only has a small effect on the expression B     
57:16     
although it does contribute we can find the nodes that have the biggest effect     
57:22     
and the things that affect Downstream things so we have like in a in a complex Network we have hubs like this and they control     
57:29     
all of these these Downstream nodes these nodes are sort of dependent on     
57:35     
this hub for connection to the rest of the network to The Wider Network and so this Hub is said to be a control node     
57:42     
and you see these control nodes in this in the connectone where you know you have different neurons that have a large     
57:48     
effect on the topology it might be in like movement or controlling some other     
57:55     
function and so that means that there are a lot of nodes that are feeding into this Hub and if the Hub is removed or if     
58:02     
it's changes its function that can affect large portions of the network and     
58:09     
so there's some interesting connections between connectomics and Reservoir computers     
58:15     
because a lot of you know people think that you can use uh connectomics to sort     
58:20     
of or you can use Reservoir computers to sort of find connectone uh different     
58:27     
types of functional conects so one other thing I'd like to go over refer we end for today and that is this     
58:34     
uh this thing we've had uh so dick actually uh put together a     
58:41     
paper uh in 1994 this is with uh Ryan drum and I think this is in response to     
58:48     
something that came up in a meeting a long time ago but I can't remember the exact context to this but this is on     
58:55     
branching more Genesis so this is on a Dix's papers this is the chemical basis     
59:00     
of datom morphogenesis so in the group we've worked on diatoms and single cell     
59:06     
morphogenesis and datom so these are diatoms uh one variation of     
59:12     
diatoms where you have this structure it's a single cell or I guess a considered a single cell organism some     
59:19     
diom come in colonies but you have these cells that have this structure and they they they kind of grow from     
59:27     
uh you know throughout their lifespan and they have these accretions and they have this     
59:33     
structure here this branching structure and so this is an example of a datom they're kind of talking about in     
59:39     
this paper and so this is uh the chemical basis of Dion     
59:44     
morphogenesis which of course is a takeoff on chemical morphogenesis of Turing and some of the things that he     
59:51     
talked about in his famous work um so this is uh just to talk about     
59:57     
this abstract an examination of the art of the artificial diatoms shows that Pur     
1:00:04     
chemical and physical considerations will account for the varieties of pattern we notice in natural     
1:00:10     
Dions uh and their living structure appears only to provide the conditions under which this uh silicus     
1:00:17     
precipitation takes place so they have this this uh they silicate organisms     
1:00:23     
they have this precipitation that contributes to the growth of cells and the the shells and so there there's this     
1:00:29     
morphogenesis that takes place through via silicon deposition uh according to the ordinary     
1:00:36     
laws of chemical action and molecular coalesence I have tried to make the subject more intelligible to that I fear     
1:00:43     
very numerous classes of microscopists who have not paid attention to Schultz's artificial diom     
1:00:49     
the attention they deserve so I don't know dick could you provide more context to this um paper     
1:01:06     
uh oh are you there yeah yeah I had the I was muted okay okay uh yeah it's it's     
1:01:14     
not Turing Turing it requires at least two substances uh which interact the fusion     
1:01:23     
the aggregation is basically precipitation     
1:01:29     
uh which leads to patterns uh the simplest example which people everybody     
1:01:37     
well everybody who lives in cold climate familiar with this     
1:01:42     
Frost yeah uh snowflakes is another example     
1:01:48     
but you have to be in a cold place yeah so it is yeah it's not like     
1:01:56     
reaction to Fusion it's actually just kind of a sing the hypothesis     
1:02:03     
basic Li agregation now uh there was     
1:02:09     
there a very recent paper which given I can dig out which is starting to become     
1:02:16     
more sophisticated than that uh and uh     
1:02:22     
uh theion liation may just be a first     
1:02:36     
yeah so diffusion limited aggregation is where you have like some some particle     
1:02:42     
in the middle of a field and you're throwing things at it and it sticks to that thing and then it builds up over     
1:02:48     
time as you have more things that go across at random and you build up the structure it's kind of like a snowflake     
1:02:55     
where it starts from a sort of a single point and then it grows out and     
1:03:00     
sometimes you can get some pretty uh impressive pattern formation through     
1:03:05     
that process um bacteria grow that way     
1:03:12     
yeah and uh there there recent articles about condensates in cells right oh yes     
1:03:22     
condensates yeah so I don't know if any of that but I threw some bacteria one time for a     
1:03:31     
project and they they grow like kind of like the snowflake they're     
1:03:38     
um um anyway in fractal     
1:03:44     
Theory there's um they're called a what diffusion limited     
1:03:50     
aggregate I don't know yeah     
1:03:59     
so yeah this is this so these are uh the Dions we're talking about this is the     
1:04:05     
mom morphogenesis uh you can have this process of diffusion limited aggregation     
1:04:11     
which is where these structures build up through deposition you know you just have you don't need any special process     
1:04:18     
in turing's model you have these two compartments which are reaction and diffusion in this case you just have     
1:04:25     
diffusion fusion and that's what the primary driver was so um yeah so this     
1:04:32     
talks about the external form of crystals is Prismatic and bounded by Straight surfaces which cut each other     
1:04:38     
at certain angles but the same form is seen in skeletons of many of the protests especially the flinty shells of     
1:04:45     
the datom and radal area which are related sort of organism stru organismal     
1:04:51     
structure their silicus coverings lend themselves to mathematical determination     
1:04:56     
just as well as the inorganic crystals and this goes back to HK in 1905 so this     
1:05:02     
is again you know where you have these kind of silicate morphologies and they     
1:05:07     
have this different mode of morphogenesis uh so this is an interesting form of morphogenesis here     
1:05:15     
um there's also this type of uh this this is a paper uh that's related to     
1:05:21     
this mechanism of branching morphogenesis inspired by diom so formation so this is a paper what was     
1:05:29     
that that's a latest the latest paper this stuff this is yeah 20 23 so     
1:05:35     
2023 so this is where they talk about uh silic formation as a form of branching     
1:05:41     
morphogenesis and so this is a potential mechanism so the abstract here it talks     
1:05:47     
about the Sila based cell walls of diatoms which we saw in the image they're Prime examples of genetically     
1:05:54     
controlled species specific mineral architectures so again this this is this     
1:05:59     
deposition of silicate and this is driven by uh the expression of genes and     
1:06:05     
it's species specific so you get different deposition patterns in different species it's not just a     
1:06:11     
physical process but there's this interaction between the physical aspect and the genetic aspect uh the physical     
1:06:18     
principles underlying morphogenesis of hierarchically structured silica patterns are not understood yet such     
1:06:26     
insights could indicate novel routes toward synthesizing functional inorganic materials recent advances in imaging n     
1:06:34     
and datom silica so these are these structures that I showed allowed rationalizing possible mechanisms of     
1:06:41     
their pattern formation so we can look at these images and sort of analyze the     
1:06:46     
structure the branching structures the sort of the depositional structures that     
1:06:51     
get a sense of the pattern formation mechanisms here we combine Theory and     
1:06:56     
experiments on the model diom the uh the     
1:07:02     
last pseudonana to put forward a minimal model of branched rib patterns so     
1:07:08     
there's this whole class of models of branching morphogenesis and if you think about branching you can think about     
1:07:13     
something like the lung where you have branching tubes that are you know they     
1:07:19     
they kind of repeat down the structure of the wung and so there are these     
1:07:24     
signals that tell the system went to branch and so we have this minimal model of Branch rib     
1:07:30     
patterns which are actually a fundamental feature of the silicus cell wall so you saw in that in that U shell     
1:07:38     
that you had these sort of uh radial sort of these radial patterns that repeated across the the body of the the     
1:07:45     
shell that those are these branched rib patterns and so physically you can build a minimal model you don't have to worry     
1:07:51     
about the gene expression or anything you just build a a model of those things     
1:07:57     
and you look at their frequency and you look at their sort of formation so what are the physics behind this so we     
1:08:04     
quantitatively recapitulate the time course of rib pattern morphogenesis by accounting for silica     
1:08:11     
biochemistry with autocatalytic formation of diffusible silica precursors followed by conversion into     
1:08:18     
solid silica so this is this uh uh this diffusion limited     
1:08:24     
aggregation where you have these things that are diffusible they're moving across the cell and they're aggregating     
1:08:31     
um in a in place so in turing's model this is sort of the diffusion part but     
1:08:37     
we don't have the reaction part we just have the diffusion part we're explaining everything through the diffusion part     
1:08:44     
where diffusion is being limited by something that some like uh thing that     
1:08:50     
it's sticking to or it stops its movement and it forms this structure uh     
1:08:55     
we propose that silica deposition releases in releases an inhibitor that     
1:09:00     
slows down Upstream precursor conversion thereby implementing a self-replicating reaction diffusion system featuring a     
1:09:08     
non-classical turning mechanism so this actually is a two compartment system reaction diffusion because you're then     
1:09:15     
adding in this uh conversion step uh with an inhibitor so you know you're     
1:09:21     
you're saying that these uh these diffusible substances have some there's     
1:09:27     
some control there and so you have the two compartment system but this is actually a nonclassical turning     
1:09:33     
mechanism so it's kind of a special case of of this sort of     
1:09:39     
morphogenesis the proposed mechanism highlights the role of geometrical cues for guided self organization     
1:09:46     
rationalizing the instructive role for the single initial pattern seed known as primary silification site so the pattern     
1:09:54     
seed is the thing that we start with that seed and then all the diffusible factors kind of bind to that and form     
1:10:01     
this pattern and that's how you get what you know the pattern that you get the     
1:10:06     
mechanism of ranching morphogenesis that we characterize here is possibly generic     
1:10:12     
and may apply also to other biological systems so while it's species specific     
1:10:17     
in terms of how the architecture is organized this process is generic so you     
1:10:23     
know we can say that there are a lot of generic physical processes across Development Across     
1:10:28     
morphogenesis they don't you know you have like variation across organisms but     
1:10:33     
you have these you also have these processes that are very generic and allow you to sort of extract them and     
1:10:40     
say that these are things that occur across the wide variety of organisms in     
1:10:46     
variation so we have the all right so we have this citation by dick this is a     
1:10:51     
paper from 2021 this is with Dick and Joseph sbach and another author dietel     
1:10:59     
morphogenesis this is from this volume here so this is the link this kind of     
1:11:04     
goes over more generally about this is actually a book on diet tomor of Genesis this goes over a number of     
1:11:11     
topics um so there's some interesting papers here on sort of some of the     
1:11:19     
puzzles in morphogen in Daton morphogenesis uh you know dealing with     
1:11:24     
single cells to chains of cells so you have these clonal chains you have vve     
1:11:32     
morphogenesis you have uh geometric models of concentric and spiral areola pattern or areola patterns of centric     
1:11:40     
diatoms uh Daton pore arrays periodicities and     
1:11:46     
symmetries uh a lot of different uh something unbuckling a geometric and     
1:11:51     
biophysical multiscale feature of centric diet morphogenesis is that's actually By Janice Papas and     
1:11:58     
Dick um and then another paper on buckling     
1:12:04     
forces there's a lot of interesting biophysics I think in diom dick introduced me to diom a couple years ago     
1:12:10     
of course and we've been working with diom off and on and I I think you people are interested in diom we should start     
1:12:17     
working on those again we haven't uh done a good diom research area we've done a good     
1:12:24     
diet research project in a while so I think it would be really interesting to revisit uh then dick says I have     
1:12:31     
electron micrographs with aberant branching patterns I could be used to test this PN ass paper yeah I'd like to     
1:12:38     
know more about that I don't know um not I mean obviously data is good um and you     
1:12:45     
know uh show aberin seeds in this theory morphogenesis of single cells not done     
1:12:52     
yeah so there's a lot of opportunity for looking at single cell morphogenesis here and you know it's I don't know how     
1:12:59     
generalizable it is but there are a lot of you know there aren't really I guess a lot of models of single cell     
1:13:04     
morphogenesis that are especially for Dions because they're very unique in that sense so a few notes on Def Fusion     
1:13:11     
limited aggregation so this is something we talked about in the meeting this is a classic biophysics Model that you can     
1:13:18     
use to model aggregation so we talked about morphogenesis we talked about rea     
1:13:25     
reaction diffusion models reaction diffusion models of Turing and the reaction part and the     
1:13:32     
diffusion part of two separate components so the reaction part of course is where you have a chemical     
1:13:38     
reaction and the diffusion part is this part where you have a diffusion of     
1:13:44     
particles across sort of a space and in this case you have diffusion limited     
1:13:49     
aggregation so as I described in the meeting you have some sort of stationary particle     
1:13:56     
that sits in one place and then your particles is moving randomly across a     
1:14:01     
space and those particles if they hit the stationary particle attach to it and     
1:14:07     
so it forms a cluster and then that cluster grows because the particles are moving randomly and running into the     
1:14:14     
stationary structure or what is now a structure So eventually you end up with large scale patterns that aggregate just     
1:14:22     
simply through diffusion and having some limit on that diffusion and so this is a model by wol     
1:14:29     
Fromm demonstrations project this is diffusion limited aggregation realtime     
1:14:34     
agent based simulation so we have our agent based simulation uh this is of course are     
1:14:41     
particles that are going to move around and we can actually we can't we can modify the level of noise so we can     
1:14:47     
modify the stochasticity of the model we can modify the number of particles so     
1:14:52     
let's start with uh try about half of the number of particles if we can okay     
1:14:59     
there we go and then we give it a random seed so that's just any number we'd like     
1:15:04     
and then we can run the simulation by ticking this box and so now the simulation's running as you can see we     
1:15:12     
can put it full screen so this is now our simulation as     
1:15:17     
you can see in the middle that uh that red dot that red structure or these     
1:15:24     
particles that are hitting they the stationary aspect of it and so the particles are hitting they're moving     
1:15:30     
around randomly they're hitting this structure and they're forming a larger     
1:15:36     
structure so we ran into some problems here let's reload     
1:15:42     
this now we see a larger structure here it's     
1:15:47     
grown which it's hard to see run the     
1:15:53     
simulation you can see the structure starting to     
1:16:02     
grow and my level of noise is pretty low in this run so you're not going to see a lot of aggregation you can see here     
1:16:09     
these particles are moving around randomly it's this sort of brownie and noise but the level of noise is pretty     
1:16:14     
low so you're not getting a lot of aggregation you're getting gradual aggregation if we increase the level of     
1:16:22     
noise we start to get faster aggregation because the particles are moving around more and their chances of bumping into     
1:16:29     
the structure increase and of course as the structure grows the chances of any     
1:16:36     
particle uh being limited by that stationary structure also increases so     
1:16:42     
you can see we're starting to get a some sort of pattern some sort of branching morphogenesis here where you get two     
1:16:48     
branches you can already see actually kind of three branches in this structure     
1:16:56     
and if we increase the level of noise some more let's do that so we can speed things up we start to     
1:17:03     
get an increased a accretion of some sort of     
1:17:09     
uh ranching morphogenesis so you see this is simply through accretion with brownie a noise     
1:17:17     
or some other type of noise you have random movement of particles AC in a in a space you have some stationary     
1:17:24     
component which you start with it's your initial condition and as things proceed through the simulation as diffusion     
1:17:31     
becomes limited by the structure you not only get branching morphogenesis but it starts to become     
1:17:37     
easier to grow the structure so you can see our structure is growing fast now we're getting more and more particles     
1:17:44     
stuck to this structure and we're getting branching morphogenesis now we have like at these five branches so you     
1:17:50     
can think of like buting and branching in a plant is may be pursuing this kind     
1:17:56     
of strategy now in Plants they don't actually do it this way because they internal mechanisms there's oxin growth     
1:18:03     
factors like that we also have cells that are sort of branching out from the interior of the     
1:18:10     
structure but you can replicate plant growth this way you can replicate uh bacterial growth in bacterial colonies     
1:18:17     
and you can replicate condensates intracellular     
1:18:22     
condensates and so we can see here here our structure we already have five branches we have more kind of coming up     
1:18:28     
and so this is what's going to happen we're going to get a couple of maybe two or three branches and then we're going     
1:18:34     
to get sub branching and the whole structure is going to look fractal by the end of the simulation it's going to     
1:18:40     
have this almost like a snowflake where you have intense branching patterns that uh     
1:18:48     
are self- similar and of course if we run the simulation again and again we     
1:18:53     
get a self similar pattern we get a pattern that's different but it's similar in in the structure so you could     
1:19:00     
analyze it and say that it's part the same class of structure even though it's totally different in terms of the     
1:19:06     
individual Parts uh well yeah they're about I guesstimated 200,000 species of     
1:19:14     
Dion okay Le the taom we've only published on     
1:19:22     
12,000 yeah uh but the general problem is there it's     
1:19:28     
in Dion in silus radiolarian also util but they're     
1:19:34     
entirely different from di yeah uh and     
1:19:39     
there Des I you there lots of single cells     
1:19:45     
where their patterns on on this on or well of the Single Cell Which shapes the     
1:19:52     
cell itself but uh p within the cell or on a Surface which are basically not     
1:20:00     
understood yeah okay and uh how far back this goes I'm not     
1:20:06     
sure uh I know there there are some weird     
1:20:12     
bacteria also that uh that have some morphogenesis uh or even Colonial uh uh     
1:20:23     
multicellular groups of bacteria yeah uh so uh it's basically an     
1:20:31     
unexplored area yeah now uh the the electron     
1:20:38     
micrographs I have are by Ryan drum unfortunately can't participate because     
1:20:43     
he has dementia oh okay okay uh so uh he gave me these     
1:20:51     
micrographs in the 1960s all right okay yeah so it's it's     
1:20:58     
completely open it requires some serious simulation uh it could     
1:21:04     
simulate the LA model it could simulate the model with just uh that you     
1:21:12     
discussed uh and see if it actually works because uh I have a hundred of     
1:21:17     
them yeah oh wow okay and they're all different and uh uh each one has     
1:21:25     
branching patterns that look uniform but come off of an aberant uh seed what     
1:21:31     
these people are calling seed yeah okay so uh there's there's an opportunity     
1:21:37     
here to try to simulate them with different models and see if to see what model     
1:21:43     
works I tried in uh that paper uh that I     
1:21:48     
did did with drum we tried but computers were too slow yeah yeah     
1:21:56     
that okay but computers are much faster now so we uh any rate uh     
1:22:07     
the simulation simulation from the atomic level to the electron microscope     
1:22:13     
level proba gives you an idea of what what's     
1:22:19     
going on but uh uh it requires simulations that uh that can     
1:22:25     
move single molecules around space and see what that is develop     
1:22:31     
yeah I me yeah even if you take the uh non-living world I don't know have there     
1:22:36     
been I haven't seen any uh simulations of snowlakes for example oh     
1:22:43     
right okay yeah you know and there's this hypothesis every s is     
1:22:49     
different yeah well I think there's self-similarity so it's not they're not exactly the same but they're very     
1:22:55     
similar in the structure I've seen DOA simulations online yeah just that it's     
1:23:01     
kind of a a p a thing I guess uh people are not as fascinated as they once were     
1:23:07     
with it but there's still a lot of interesting stuff there um yes yes I     
1:23:16     
agree anyway the diatoms are fascinating because they have so many different     
1:23:21     
structures which are unexplained okay say if you go across different     
1:23:27     
species yeah okay and uh it's individual variability     
1:23:34     
with the aberin diom is unexplored okay     
1:23:40     
I'm I'm waiting for a very strange thing special kind of paper that allows you to     
1:23:47     
turn a scanner into a film digitizer I don't want to buy a whole hold the     
1:23:53     
machine for a film digit digiti right okay uh so as soon as I have that     
1:24:01     
I can then copy these micrographs and put them on     
1:24:06     
L yeah but it might be a good thing for Deo because somebody's interest in     
1:24:13     
getting into the single self orogenesis problem and uh this new Theory uh could     
1:24:20     
be right could be wrong we don't know but it's not tested yeah     
1:24:26     
it's full of uh it's full of parameters which are speculations about what might be happening     
1:24:34     
right yeah okay I think that's it for today I thank     
1:24:40     
you dick for the uh kind of exploring that     
1:24:50     
area oh Susan hi hi um did you     
1:24:55     
um did you get my uh USB stick with the images from the microscope uh well how     
1:25:03     
long ago is that like the I've got a couple mon or two a month or two oh     
1:25:09     
maybe yeah I think so but I don't I think that was more than a month though yeah yeah I sent them a while ago yeah I     
1:25:15     
did get that yeah yeah I do have them yeah all right     
1:25:21     
yeah I just still want to know what you think of the quality of them because they're more blurred than I would like     
1:25:29     
yeah they were a bit blurred but I mean you know it's like something that we have to kind of smooth out and and     
1:25:35     
postprocessing so sometimes it's not but yeah I mean you     
1:25:40     
know I could try sticking them in mat lab and doing some image processing on     
1:25:45     
them I guess maybe but yeah I mean I don't know I don't know if you can get it any better it's kind of     
1:25:51     
a hard thing to yeah well it's um yeah     
1:25:57     
the quality of the microscope I guess well yeah I mean the depth the field     
1:26:03     
it's a depth of field thing more more than anything yeah     
1:26:10     
yeah okay well yeah oh yeah go ahead no that's all I just don't know how to find     
1:26:18     
something with uh something greater than three megapixel resol ution oh anyway     
1:26:26     
yeah it's um you'd have to have better lenses on the piece of equipment as well and     
1:26:34     
then the cost would go up I don't     
1:26:40     
know okay well okay thanks for attending the meeting uh again let's keep in touch     
1:26:46     
over the week and see everyone next week okay great I I'll leave this open     
1:26:52     
all right thanks take care bye everyone bye bye     
1:26:58     
bye
