     
Transcript     
0:00     
hello welcome to dorm uh today we not not going to have an in-person meeting     
0:05     
I'm giving an update on some things uh that we have in the group uh we have new     
0:12     
data sets that I'd like to share these are secondary data sets from different data sources so I'm going to go over     
0:18     
those and then I'm going to go over some papers so the first thing we have are these larel data sets so in last week's     
0:26     
meeting we talked about larvel stages for different organisms we of course we     
0:31     
have laral stages in celegans we have laral stages in other organisms and we     
0:37     
so what we have some zebra fish larel data that we've been working with for a couple years in the group and we have     
0:44     
some of those analyses published at least in one paper and then we also have some new data from cigan L1 stage so L1     
0:54     
is the first larel stage right after hatch of the egg this is actually way     
1:00     
well mapped both the adult and the embryo so let's take a look at some of these data sets so this is the uh cigan     
1:08     
L1 data set this is the first larvel stage so we have a list of cell names     
1:16     
and those cells are adult cells so these are the cells that result from development they've differentiated at     
1:22     
the tips of the differentiation tree and now every cell in this are considered     
1:28     
adult cells so what we have here are adult cell names we have their XYZ     
1:35     
position uh we have a normalization of that and then we have some intensity     
1:40     
data which have to do with uh gene expression and some other things here uh     
1:47     
color information for those measurements and then a mass which we typically don't     
1:53     
use uh but this is basically the data you want so you have the names of the     
1:58     
adult cells and of course we have um an adult celegans model which we can map     
2:04     
these cells to so you can look for example at these cells in laral     
2:09     
development versus the adult and then we also have data on Cell position for     
2:16     
developmental cells in the embryo and because those names because there's a deterministic mapping between the     
2:22     
lineage tree and the adult cells we can actually you know overlay an embryo with     
2:29     
an L and point Cloud so all of these uh positions if you plot them out they     
2:35     
yield a point cloud and that can be uh basically it's the shape of a of an L1     
2:42     
worm and that can be mapped to an embryo as well so you can get information about     
2:47     
what's going on with that corresponding developmental cell in the embryo and its     
2:53     
position and information about migration and other things so so this is also good for     
3:01     
training uh machine learning models so we have you know a good I think a good     
3:06     
data set here uh several thousand cases so that's going to be of interest I     
3:11     
think to people working on different problems in celegans then we have two zebra fish data sets so we have uh cell     
3:19     
larvel data sets and this scna seek data and so the cell lval data set is similar     
3:26     
to the data set we have for C elans L1 we have actually two stages in zebra     
3:32     
fish we have the zygote stage and the cleavage stage so these are just different stages of development of the     
3:39     
larvae in zebra fish and so we have here a a cell ID number which is kind of if     
3:47     
you go back to the original data set and I'm not going to show that here but there's an ID for the cell um you know     
3:55     
at least you know if it's a unique cell or not you have this position data and     
4:00     
then you have time a time stamp which is this minute data so basically we can go     
4:06     
through we can plot this out in time we can plot this out in space and we can     
4:11     
get an account of what the zygote looks like the shape of that point clot what that should look like we also have the     
4:17     
cleavage stage which is a little bit later in development so the zygote is     
4:22     
early the cleavage is a little bit later again you can see that starts at 46 minutes and it goes down uh uh you know     
4:31     
I don't know how many cases we have here but quite a few and we of course have the unique cell ID and then uh XY Z     
4:39     
coordinates and of course we can do things like look at the zygo versus the cleavage look at the cleavage stage or     
4:46     
the zygote stage on its own or even compare these to seans in some way so     
4:51     
it's very nice data it's all these Point cloud data these positions describe a     
4:59     
centroid of a cell and you know that gives us a point clo which we can then     
5:05     
do things with we can also use this these data to train machine learning     
5:10     
models and you know this sort of three-dimensional shape Discovery so we     
5:15     
also have the scna seek data and this is also for zebra fish so scna seek it's     
5:22     
this next gen sequencing technique and in this data set they've used this to uh you know kind of build these m     
5:29     
apps these visualizations and so this is where we have this is these are new numeric data     
5:37     
that you know you have to go back and look at the original paper to know kind of what's going on but basically you     
5:43     
know we have data for the wild type cells and we have two variables here this is going to yield a tne map so     
5:50     
these are the tne components that uh if you plot them out you'll get this standard tne map then we     
5:58     
have this developmental landscape uh from drop seek so this is actually these are XYZ coordinates and these are the     
6:04     
same uh I think these are uh Olo sequences that they have and these are     
6:11     
from the wild type so you can go through and look at those so they're probe IDs     
6:17     
with the cell ID and then you have three dimensions you so you plot this on a     
6:22     
three-dimensional map and you get this landscape of these uh cells and the     
6:30     
sequences that are being expressed in those cells okay that was a little bit about the new data sets um you know find     
6:39     
to find out more we'll have to go into the GitHub repositories and look at these data sets     
6:45     
uh where it has the reference for the original paper and um the CSV files we     
6:50     
always work from CSV files those are the simplest way to do a lot of these data     
6:56     
so let's move on to the papers now I think the papers are interesting this week uh we have a series of papers on     
7:04     
generativity and development that's going to be very interesting we also have this paper on     
7:10     
transcriptional uh accounting and I'll show you what that that it is in a minute so this paper is called     
7:16     
transcript accumulation rates in the early celegans embryo so this is uh from     
7:22     
the group of John Murray who of course has generated a lot of the cell tracking     
7:27     
data for celegans embryos um and so he's doing this kind of work now with these two uh     
7:36     
collaborators and so uh what is transcriptional accumulation     
7:41     
rate well that's what we're going to talk about so Dynamic transcriptional changes are widespread and rapidly     
7:48     
dividing developing embryos where self-hate decisions are made quickly so     
7:54     
what they're saying here is that you have an embryo you start with a single cell the cell divides but of course at     
8:02     
the same time genes are being expressed in that cell and there's a transcriptional complement of our mRNA     
8:09     
and there's an mRNA content in that cell and as the cell divides that mRNA     
8:14     
segregates into one of two cells it could segregate into the daughter cells evenly or unevenly and so this is what     
8:22     
he's talking about with respect to you know uh these transcriptional changes     
8:29     
and you also have dividing cells rapidly dividing cells in an     
8:35     
embryo the seelan embryo over comes these constraints partly through the rapid production of high levels of     
8:40     
transcription Factor mrnas so in the embryo because the cells are dividing so quickly there's a lot of     
8:48     
mRNA being made at a very high level and so you know we think about gene     
8:54     
expression in adult human cells or adult Mouse cells or what whatever and there     
9:00     
are these experiments you can do where you can actually turn off the transcriptional Machinery with the drug     
9:06     
treatment and you can watch the decay of mRNA over time and it takes like uh you     
9:13     
know maybe 20 or 30 hours to completely deplete and of course the cell dies so     
9:19     
you know these there's a transcriptional rate you can establish how long does it take to you know kind of make mRNA and     
9:27     
how big is that complement Within the cell of a certain mRNA so you know this is something you     
9:33     
can approach quantitatively and so in celegans embryos because you're kind of you keep     
9:41     
dividing and you need to have enough uh mrnas to do what you need to do but you     
9:48     
also have to keep fractioning it away to put it into your daughter cells um you     
9:54     
need to have this high level of synthesis so     
9:59     
this is also important for Sate decisions because you need to have things upregulated at the right time     
10:05     
they need to be available for you to use as a cell so these are all you know     
10:11     
things that involve mRNA mRNA pools within the cytoplasm and the ability to     
10:17     
like utilize that pool when needed transcript accumulation rates for some developmental genes are known at single     
10:24     
cell resolution so we know kind of what these accumulation rates are in other words     
10:30     
transcription proceeds at a certain pace and there's an accumulation of     
10:35     
transcript and there's a certain amount of time that the cell has to make it     
10:40     
before it divides into daughter cells and so if the daughter cell is going to differentiate one or both of the     
10:46     
daughter cells you need to be able to produce enough transcript and accumulate it so that that daughter cell will have     
10:54     
the transcript available for it to use so in the past several weeks we've     
11:00     
talked about cell division we've talked about cell cycle we've talked about how     
11:05     
you need certain amounts of mRNA or certain amounts of gene expression to occur before you can move to the next     
11:13     
stage of uh cell division or sometimes you need to shut off uh RNA mRNA     
11:21     
production to proceed to the next level of cell division and so because of those     
11:26     
transcription accumulation rates are very important whether you you know how long transcription goes on for what the     
11:33     
rate is and what that accumulation is in in the uh     
11:38     
cytoplasm we also talked about I think it was last week or the week before     
11:43     
about the speed of transcription or the speed of cell division and so the speed of cell division really is about the     
11:49     
speed of transcription and the accumulation of transcription so that's something to     
11:55     
keep in mind that this is closely linked to cell division this is but this this is also closely linked to uh diff cell     
12:02     
differentiation so we know some of these transcript accumulation rates we can do experiments to to Quant quantify these     
12:09     
uh but genome scale measurements are lacking so we know kind of what some of these are for certain developmental     
12:15     
genes key developmental genes and maybe that's all we need but we don't have these genome scale measurements we don't     
12:22     
know across the genome what's going to happen and so that may be important because you know sometimes you have you     
12:30     
know all sorts of transcripts that are needed for sort of to keep a cell viable or to uh you know     
12:38     
allow it to differentiate to a new state we've talked about genes needed for differentiation but you know we're only     
12:45     
really kind of interested in the genes of largest effect sometimes you're going to need other genes of lesser effect to     
12:54     
um you know the transcripts of those genes to be there in order to differentiate properly     
12:59     
so it would be nice to know all the details of that we estimate zygotic mRNA accumulation rates from a single cell     
13:06     
RNA sequencing data calibrated with single molecule transcript Imaging so     
13:11     
they're able to do this kind of Imaging where they kind of get these uh they establish these accumulation rates from     
13:19     
the single cell data so we just saw some single cell data but we're going to do     
13:25     
uh I think a different method in this paper rapid transcrip rtion is common in the ear elegans embryo with rates     
13:32     
highest soon after zygotic transcription begins it's interesting that you know we     
13:37     
have this very early establishment of very rapid     
13:42     
transcription and of course it seems like it's it's right at the sort of the root of the of the lineage tree so as     
13:50     
soon as you sort of start to divide into two cells that's when you have the     
13:55     
fastest transcription and then it kind of tails off after after that but you know in this early celegans embryo you     
14:02     
get this rapid transcription it's very common and so that makes sense if you think about what's going on in the early     
14:08     
developmental lineage tree you start off at the one cell you go you divide into     
14:15     
two cells you establish your basic polarity so you have your uh AB lineage     
14:21     
and your P1 lineage and then you in a couple division events later you end up     
14:28     
with your eight founders cells those eight founder cells are sort of key in     
14:33     
the establishment of sublineages that will turn into different tissues or you     
14:38     
know be different regions of the embryo and so it's important to sort of get all     
14:45     
the transcripts in place to get a large pool of transcripts and as we said     
14:50     
before you know transcripts Decay at a certain rate but they don't Decay that quickly mRNA     
14:56     
decays around you know maybe the 12 12 to 24-Hour period uh so you know the     
15:02     
halflife is what I'm talking about so if you start an early development and in     
15:07     
the celegans embryo you know our development uh is you know not that long     
15:14     
um you end up with you know being able to preserve a lot of that Airway     
15:20     
transcription uh throughout the embryo life of the embryo so by the time that we get to a you know a lot of these     
15:27     
cells being differentiated you know maybe some of that mRNA sticks around and but we definitely have enough where     
15:35     
you know the rate of division is greatest so we can get to those EV those differentiation events and have the     
15:42     
genes or at least have the MRNA complement at the right amounts to make     
15:47     
a difference to support that differentiation event okay     
15:53     
so high rate genes are enriched recently duplicated Sate regulators and share     
15:59     
common genomics features so what he says High rate genes that is uh where you     
16:05     
have this High rate of uh transcription this is enr for recently     
16:12     
duplicated sufate Regulators this is very tightly uh linked to sulfate and     
16:20     
for you know duplicated Regulators so this is you know maybe important for     
16:26     
cell duplication in our new paper we talk about we speculate about how you     
16:32     
know cell duplication might be important for differentiation how you know we have     
16:37     
these specific genes we're looking at for selfie specific things so this is     
16:43     
maybe an important thing to look into with respect to what we were talking about with differentiation     
16:50     
waves um we identify cor promoter elements associated with high rate and     
16:56     
measure their contributions for two early and mesodermal genes     
17:02     
c51 and sdz 31 so these two G these two     
17:07     
genes are involved in endomesodermal uh sorts of things so     
17:13     
early tissue specification individual motifs modestly     
17:18     
affect accumulation rates suggesting multi multifactorial control so this is     
17:24     
where we have motifs of genes and how they affect accumulation rates uh so     
17:30     
that means I guess that genes have to be regulated together and this is something that we you know maybe there's some sort     
17:37     
of multifactorial control uh these results are a step towards estimating absolute transcription kinetics and     
17:44     
understanding how transcript dosage drives developmental decisions so this idea of dosage is this same thing as     
17:51     
halflife the dosage is the amount that's there the halflife is the Decay over     
17:56     
time so over a certain amount of time you'll get half of that dosage and then     
18:02     
a certain amount of time later you'll get half of that half of dosage so it it decays sort of linearly over time and     
18:11     
not always sometimes you know especially when you don't shut off the transcriptional Machinery you get     
18:17     
nonlinearities where you get these uh AC Mass accumulations and then Decay and     
18:24     
then maybe some more production and so forth uh I can also say that in mRNA dek     
18:29     
experiments we can actually uh Force the cell to sort of adapt to its in um adapt     
18:37     
to its environment so we can do certain things to the cell before we shut off the transcriptional machinery and it     
18:42     
shows sort of an accumulation of mRNA way over Baseline and then you get this     
18:49     
Decay so there are a lot of things going on in the trans crypto and in this     
18:54     
cytoplasmic complement they kind of make the point here that absolute transcript accumulation rates where the rate of     
19:02     
change of RNA copy number that has led to a timely production of transcripts at     
19:07     
the appropriate level are rarely explicitly measured at a global scale uh during development so there are some     
19:14     
experiments that have been done under dropa where they've done quantitative Imaging they provided detailed     
19:20     
measurements of the transcript production rates or some early patterning genes so these are genes that     
19:25     
are involved in morphogenesis in uh striping and so forth one such     
19:32     
study found the accumulation rates of the Gap Gene Hunchback kple uh gurps and giant to be fairly     
19:40     
similar 33 transcripts per minute per nucleus so this is where we have these     
19:46     
genes that are involved in this Gap genes are involved in different uh     
19:52     
pattern formation uh activities that have very similar trans     
19:58     
transcription rates so this is 33 transcripts per minute per nucleus so     
20:04     
this is actually pretty useful uh in this is of course over a 15minute nuclear     
20:09     
cycle um these rates are high with levels approaching the theoretical     
20:15     
steric limit based on uh RNA pull two footprint so this is where you know the     
20:21     
polymerase can only move so fast and it can only produce so many transcripts and     
20:26     
so this is basically there's a sort of an upper bound and we talked about this     
20:32     
in this article on uh increasing the speed of cell division for     
20:38     
bioengineering and so you know we were looking there at bacterial cells in this     
20:43     
case we're talking about softw cells but in both cases you have this limit rate limiting factor of the polymerase and so     
20:51     
that's interesting that you know you're able to get up to that theoretical limit um so this this would allow for     
20:59     
one pull two every 70 to 80 base pairs and so this is something that we've     
21:04     
gotten from live Imaging of mrnas uh but there are some limitations to that approach and so that's going to set up     
21:11     
their approach using nextg sequencing and so they pick the celegans embryo     
21:16     
because it provides an ideal developmental system to answer these questions it develops with an invariant     
21:22     
lineage where division patterns are highly reproducible across embryos allowing     
21:28     
equivalent cells to be compared between individuals this robustness results in     
21:34     
part from a high level of precision and expression levels and this has been observed in intestinal transcription     
21:41     
factors these citations 11 and 12 the intestinal specific uh     
21:46     
specification G factors N3 and N1 accumulate to very high maximum     
21:52     
transcript levels and in this case this is over 300 mRNA molecules     
21:59     
per cell in short cell cycles so this would be a 15 to 20 minute     
22:04     
cycle so this is where you have these two genes that are Gata factors they're     
22:09     
in these intestinal uh what are going to be I guess intestinal cells they're     
22:16     
specifically intestinal transcription factors you go in the adult and check to see that they have this function and     
22:22     
they actually accumulate reproducibly meaning over many animals uh to very     
22:28     
high High maximum transcription level so I guess this is an upper Bound for the number of molecules per cell that can be     
22:34     
produced in about 15 to 20 minutes proper gut formation requires these transcripts to surpass a threshold level     
22:41     
emphasizing the importance of transcription rate and developmental progression they want to measure High transcript accumulation rates and     
22:48     
characterize these rates genome wide so they actually look at the read counts in     
22:53     
SE and a seek data so actually they are looking at the same technique SE seek so     
22:59     
what you have an scna seek is you have these short uh sequences uh that you     
23:05     
actually will try to sequence and then count the number of sequences of the     
23:11     
same type align them with the genome identify what the sequences are and then     
23:16     
get a count of those common sequences and when you have     
23:21     
a lot of them of course that means that you have high expression when you have very few to none that means you have low     
23:27     
expression so this is what they're doing here and so they're able to uh gain you     
23:34     
know gain insight into absolute transcript levels and C Elegance embryonic cells genom distributions a     
23:40     
transcript accumulation rates vary substantially across cell types and lineages but are highest at the eight     
23:46     
cell stage of development so this is eight cell stage where you have your founder cells and this is exactly what     
23:53     
we might expect given what we know about the lineage tree that this is when the sublineage ages are being founded and so     
24:00     
we expect this sort of uh you know transcriptic cumulation be highest when     
24:06     
we want to start our sub lineages IR rate genes tend to have more     
24:12     
recent duplicates and are enriched in chromosomal clusters indicative of a rapidly evolving class of genes we     
24:18     
identify Gene features associated with rapid accumulation including short primary transcript length and lower     
24:25     
intron size and count so this is where you know we get into this paper a little     
24:30     
bit and they're using SM fish to normalize or to compare with the rnac     
24:36     
data so SM fish is a sort of an imaging technology where they can use     
24:41     
fluorescent probes to validate a lot of what they're getting with the scna seek     
24:47     
so this is where they talk about how scna seek measurements can be noisy and     
24:52     
they validated everything on SM fish uh so this is where they get these uh oo     
24:59     
probes to uh Target the mrnas and so they can actually visualize the mrnas in     
25:05     
the cells so the idea is that if the fluorescent signal is brighter or greater in the SM fish and the number of     
25:12     
transcripts is high in the scna a seek then that's uh you know validating the     
25:17     
technique and it's sort of giving uh support lending support to what they're finding with a CR and a seek so it's a     
25:25     
nice design where they can actually kind of validate what they're getting they're not getting you know because what you     
25:32     
get with SC AC is you just get a set of sequences short sequences that are uh     
25:38     
sequenced from uh bulk uh sampling and then you have to map that to a genome uh     
25:45     
genome draft so you have to find the sequence you know and align it with     
25:50     
something in the genome draft so you know you might have misalignments you might have some other things so there     
25:56     
are a lot of sources of noise and CR RNA seek and so it's nice to have this     
26:01     
validation method this is interesting they talk about transcription rate varying across stages so embryonic cells     
26:09     
change markedly across stages so when the embryo under goes a reductive     
26:14     
cleavage uh where where cell nuclear volumes decrease by about two fold each     
26:20     
division then we get this difference in uh transcription rate so as you know     
26:26     
like you could sell you know you have certain volume in the embryo and as cells divide they get     
26:34     
smaller with respect to the embryo so their size reduces the daughter cell     
26:39     
size reduces and each round of cell division the daughter cells tend to get smaller so this is where you know you     
26:47     
don't need as much trans you don't need to have as high of a transcription rate if your daughter cells are getting     
26:52     
smaller and smaller when you get maybe when you get up to the eight cell stage where you're founding your uh line sub     
26:59     
lineages you know you may need a lot of transcript and the cells don't really get smaller until they get past that     
27:05     
eight cell stage so once you start to get this reductive cleavage you start to     
27:10     
get you know past the need to to establish these sublineages then the     
27:16     
transcription rate can fall it can it can lower itself uh because remember the     
27:21     
transcription rate requires a lot of metab metabolic resources so there's something going on there too we have to     
27:28     
take that into account um so there are a lot of things of uh sort of affecting     
27:34     
transcription rate across stages and what they did here in the paper is uh they found that while high     
27:41     
transcription rates occur at the 48 and 16 cell stages uh the overall distribution rates     
27:48     
is highest at the8 cell stage this seems broadly true across most lineages so this is lineage     
27:54     
invariant sublineage invariant but it's just having to do with the round of cell division and the     
28:03     
size of the cells that result so this is really not not anything that's like linked to cell or sublineage or to what     
28:12     
the cell type is going to be but it's really related to the size of the cells the daughter     
28:18     
cells so the median accumulation rate of medium and high rate genes is higher for cells in the eight cell stage compared     
28:25     
with their daughters at the 16 cell stage overall median rates of the cell stage are about two to three fold higher     
28:32     
compared to the four and 16 cell stages so this is a nice figure that kind of shows in graphical form what we've been     
28:40     
talking about so again this is uh the uh eight cell stage     
28:46     
here and this is the 16 cell stage on the bottom so the eight cell stage is here this third layer and then the 16     
28:53     
cell stage is the fourth layer and as you can see establish basic polarity up here in the second layer you     
29:01     
get your founder uh cells in this fourth layer and then you know so you have your     
29:07     
different AB cells you have your ab suppl ages you have MS which leads to     
29:12     
muscle and fering the E sublineage which leads to gut the C sublineage which     
29:17     
leads to muscle and epidermis and then P3 which leads to muscle and germ line     
29:23     
and so these are very on the right hand side which is the posterior end of the embrio you're going to get these very     
29:30     
highly specialized uh sublineages on the at the     
29:35     
anterior end of the uh embryo you're going to get these very general     
29:41     
sublineages which produce all sorts of different types of cells uh so this is     
29:47     
you know an interesting thing you might expect to see some asymmetry here but you really don't really see it in terms     
29:54     
of sub do you see it in terms of depth of the the tree so as you as the cells     
30:00     
the daughter cells start to get smaller and smaller that's when you see this reduction of transcription rate and when     
30:06     
you're establishing up to the point where you're establishing your sublineages you have this maximum     
30:12     
transcription rate okay     
30:17     
so you know we can look at this in terms of absolute change and full change so fold change is just in comparison with a     
30:25     
reference where you're counting against some housek keeping gene or something that's     
30:30     
invariant and your absolute change is just the change of that Gene in terms of     
30:36     
the transcript count so this is where you know this difference comes in for these graphs in B you get the number of     
30:43     
genes log scale so copy number uh increases here with absolute     
30:49     
change in RNA molecules so you get uh you know a number of them that are     
30:56     
really highly expressed um with not really a long tail but it's like you know you     
31:02     
get uh you know you don't have as so the what they're saying here is that the     
31:08     
very high number of uh copies is restricted to a subset of genes and so     
31:14     
that's what we might expect uh that's with the absolute change measure with the full change measure we have a little     
31:20     
bit longer tail which you still have the same pattern where you have you know a smaller number of genes     
31:27     
that are really upregulated so this is like you know you imagine probably functional genes that     
31:33     
are necessary for differentiation for sort of establishing things in the early     
31:40     
parts of the lineage tree and so forth so not everything is really highly     
31:46     
upregulated some things are many things aren't um it's interesting though as you     
31:53     
get this uh change in time so to cell most in the two cell stage which is the     
32:00     
abp1 uh stage almost everything is low in terms     
32:06     
of its uh expression you get this medium     
32:11     
very small number of medium uh Express genes and this is very a very small number this is the     
32:18     
proportion of genes so this is like 0.25 this is about a quarter of the genes and you can see that when you get to the 8     
32:24     
and 16 cell stage at about a qu of the genes you start to get this medium to     
32:30     
high regime so this is this part of the uh distribution here for full change and     
32:36     
absolute change this part up here so that translates into about 25% of your genes being either medium or high highly     
32:45     
expressed so and then of course in the E cell we get a little bit more in terms     
32:50     
of very highly expressed genes than the 16 cell and I think that decreases over     
32:56     
time so you get this decrease over time in the uh in the amount of uppr     
33:06     
Regulation uh we have this other graph here which is the number of genes low medium and high is just breaking this     
33:11     
out into the sublineages so your founder cells in or actually these are a Ms e Ms     
33:19     
C D and P these are the eight founder cells and you know this is where you're     
33:24     
getting your um differences across the different sub lineages so you get this     
33:31     
two cell here um p and ab uh then four cell you start to get     
33:38     
changes in EMS eight cell you start to get changes here and 16 cell here and so     
33:44     
you can see this for low medium and high for high you know you you don't have any in the two cell stage by the time you     
33:52     
get to 16 cells you know you have MS e     
33:57     
uh C a little bit in C and then AB so C is and c and MSR we go back up here     
34:06     
muscle fance and muscle epidermis so it means that you're     
34:11     
prepping these cells for muscle Feats uh but you're also prepping them for these AB     
34:17     
Fates which include neurons and other things so really you're you know maybe     
34:23     
some of these types of Fates are where it's important to Prime them with a lot of     
34:28     
mrnas and so forth so this is and then this is just a heat map of the genes     
34:34     
versus the lineage tree and showing some of these uh changes so to finish up this     
34:41     
paper they talk about the influence of transcript Decay and posttranscriptional processing on an RNA     
34:48     
output so if we measure the RNA we're only measuring sort of gene expression     
34:53     
at the first stage of its transformation to something like protein so we get gene     
34:59     
expression and that tells us something about like maybe you know how much transcript you need but ultimately you     
35:07     
know we want to know what the proteins are and so you know we need to know first of all the decay of transcripts     
35:13     
over time how much of those are going to contribute to trans translation then you also have this     
35:19     
posttranscriptional processing which happens uh you know to the RAS so that     
35:24     
some of them are functional some of them are not so it's not just enough have a lot of mrnas in a pool you know     
35:30     
some of them are going to be functional or change their function for the next step which is translation and so you     
35:38     
know these are things we have to think about in context and it's beyond the range of the study but for the paper we     
35:44     
just published on looking at gene expression proteins there's a very it's     
35:50     
very poorly correlated when you look at like protein counts and you look at gene expression counts it's very poorly     
35:56     
correlated as to how much transcripts are contributing to proteins and their     
36:04     
abundance so you know there are a lot of inter that's because there are a lot of intermediate processes going on and we     
36:10     
have to be able to understand those a little bit more and especially in something like this to really understand     
36:16     
their effect on say like the phenotype so we use to embryo or an a data to estimate the contributions of synthesis     
36:23     
and Decay to accumulation rates and our analysis suggests that transcript production has a larger impact on total     
36:30     
rates this is consistent with studies of malean cell culture systems during the     
36:35     
response to stimuli where transcript synthesis contributes more to changing RNA levels than     
36:41     
degradation so this is where you know you get this response to different types of things like lipopolysaccharides or     
36:49     
hypoxia which are stimuli that are like kind of challenging to the cell um and     
36:55     
so this is where they have to sort of regulate their RNA levels this is where     
37:00     
they might overproduce certain rnas over others to ensure that they can adapt to     
37:06     
their conditions and so this is where you know degradation isn't the dominant     
37:12     
thing it's actually changing RNA levels so Gene you know different genes are being upregulated and down regulated     
37:19     
with this stimuli many of The Regulators that we analyze here are transiently expressed     
37:25     
with most transcripts disappearing within one or two cell cycles of the time of maximal expression so this is     
37:31     
where you know we get into this issue of mRNA Decay and over time you get Decay     
37:38     
and of course a lot of the transcripts in this case are disappearing over just a couple of cell cycles from the time of     
37:45     
their maximal expression so this is an interesting thing     
37:50     
um and just thinking about like the timing of all this in the lineage tree     
37:57     
in accordance with this genes with higher total accumulation rates tended to have shorter half- lives so this is     
38:04     
where you aumu white a lot for a gene and they tend to have a shorter halflife     
38:10     
so it's it's kind of paradoxical you'd think if they accumulated more genes     
38:16     
that or more transcripts that it would Decay slower but in fact they maybe are     
38:21     
accumulating more transcripts because their their halflife is shorter would run out faster if they     
38:29     
didn't produce so much so that kind of makes sense you know if you have a lot of if you have something that runs out     
38:35     
very quickly you would produce a lot of it and if something has a longer halflife then maybe you don't need to     
38:41     
produce as much however there's a possibility for Decay rates to be different during the appearance and     
38:46     
disappearance phases uh so this is where this is something they talk about in the paper we didn't cover although we     
38:53     
estimate that the noovo transcription rates are modestly higher than than the observed total accumulation rates when     
39:00     
adjusting for Decay more direct measurements of Half-Life during the time of transcript increase versus     
39:07     
decrease will determine more precisely how much higher and even more impressive     
39:13     
to the total accumulation rate can be so there are a lot of things that     
39:18     
govern transcription rate increase and decrease and this factors into     
39:24     
accumulation rate so this is you know something that I guess is maybe     
39:30     
work for later or some sort of computational study or something but in     
39:35     
any case they get into post-transcriptional processing as well and of course this includes things like     
39:41     
RNA splicing and other types of modifications and this is another phase     
39:47     
of gene expression where rates may be tightly controlled we found a number of unspaced     
39:52     
transcripts of the MS specific Gene c51 outside n transcription sites and     
39:58     
early muscle cells which could mean that post-transcriptional splicing plays a role in how quickly mature transcripts     
40:05     
of ch51 are produced I think this whole passage is really interesting um so the idea is is     
40:13     
that if you have a bunch of mrnas and you splice them when you when     
40:19     
when mrnas are being spliced they're being you know chopped up so that certain pieces are remain functional and     
40:26     
some pieces are taken out and the reason you do that is because you want to create ISO forms so you have different     
40:32     
forms of the same transcript that do like sort of very specialized functional things and these are isoforms are very     
40:40     
important in muscle as we know from like other studies and so you know think about like     
40:47     
if you you're producing a lot of mrnas and you need a lot of splyce fering it     
40:53     
so you need a lot of different types SP of different uh different types of spliced mrnas so many different     
41:00     
varieties of them so you can create many different ISO forms you need to produce a lot of mRNA     
41:06     
to do that regardless of the Decay rate because you just need a lot of different     
41:14     
uh versions of something that are going to become different specialized variants     
41:20     
and you need to have enough so that you can have you know a pretty decent number of each so this is interesting from the     
41:28     
perspective of functional modifications direct R seek has broadly     
41:33     
revealed that many transcripts and diverse cells are not splice Co transcriptionally although the importance of post transcriptional     
41:40     
splicing and the rates of other processing steps during Sate specification still need to be worked     
41:46     
out and so they also have a statement here on cell size and transcription rate     
41:51     
so to finish up with that you're really dealing with cell volume and reduction     
41:58     
of cell volume and development for individual cells uh we know that transcription rat     
42:04     
scale with cell size across organisms the celegans embryo develops by reductive cleavage in a constant volume     
42:11     
eggshell so that means that the daughter cells get smaller and smaller and the eggshell doesn't vary that much so the     
42:18     
eggshell remains constant but as you get more and more cells they get smaller and smaller and crowded into this eggshell     
42:26     
with cells reducing about twofold in size with each developmental cell division we see the highest maximal     
42:33     
transcript accumulation rates in the eight cell stage where cells are larger compared to the 16 cell stage suggesting     
42:41     
that transcript rates in the cigan embryo may also be driven by cell size     
42:46     
so we have Decay we have functional requirements     
42:51     
and we have cell size and so these are the three kinds of things that are controlling our M production mRNA Decay     
43:00     
and what this mRNA balance looks like and and what it does functionally so     
43:06     
finally they they finish up with talking about maximal achievable transcription rates so we talked about two weeks ago     
43:13     
about the speed of transcription or the speed of cell division which is also the speed of transcription because you need     
43:20     
to be able to uh not only transcribe things but you need to replicate DNA so     
43:26     
that you can have two genomes for two new cells and so these are all     
43:32     
interrelated and so we have this discussion here on theoretical limits of transcription rates talking about pole     
43:40     
polymerase 2 pole two another transcription Factor molecules rates of     
43:45     
initiation and elongation Gene length and pause sequences so for short genes     
43:51     
such as N3 which is around 1.3 kilobases of primary transcript     
43:58     
at the commonly assumed elongation rate of 1.5 kilobases per minute the entire     
44:03     
Gene length would need to be completely occupied by pole two complexes during maximum expression so what they're     
44:10     
saying is that you have these genes you have the genome you have these genes there a certain length the transcripts     
44:17     
what we're really interested in where the transcripts are are sort of the things that code for the     
44:23     
transcripts and the pimas 2 has to be on there moving along producing transcripts mRNA out into the     
44:31     
cytoplasm you can't do that you know it has a certain rate so one pum ra 2 is a     
44:36     
certain rate and what they're saying is that to do this efficiently in the time     
44:42     
required you need multiple P twos on the same uh sequence and so what they're     
44:49     
saying is is that sometimes you get to these rates that you're predicting that require you to just have so many P two     
44:57     
that's sort of Impractical for what we're dealing with now you know we can put as many I guess     
45:03     
we can put as many ptos as we want on the on the coding sequence the question is is is that what biology is doing is     
45:11     
that something that's you know metabolically feasible so um that's the     
45:18     
kind of the point that they make here uh so there is a limit there is a limit to transcription rate practically speaking     
45:25     
SM fish analysis of early zygotic genes in the drop embryo has also measured High rates of transcription potentially     
45:32     
nearing the steric limits of P two suggesting that rates approaching theoretical maximum could be a common     
45:38     
feature of gene expression and rapidly dividing embryos so this is where you just you reach this theoretical maximum     
45:46     
related to the you know how many pole tws you can get onto a uh onto a     
45:52     
chromosome or onto a coding region of a gene and how fast it moves     
45:57     
and all of that so that's how you determine that uh so this is actually maybe a     
46:04     
common feature of rapidly defining embryos and so we don't know too much about the elongation     
46:10     
rates uh but we know that elongation speeds as high as six kilobases per minute are seen in Mouse uronic stem     
46:18     
cells so we know that this is possible it's just you know WEA we actually can     
46:23     
observe it so the next thing we're going to talk about is uh category called ensembles and     
46:30     
generativity so there have been a couple papers that I've run across uh this week that have to do with the generativity of     
46:38     
development and then there's this paper on ensembles in development so the first     
46:44     
paper is this paper the genomic code and this is by Kevin Mitchell and Nick     
46:49     
Cheney Kevin Mitchell is a genetics and Neuroscience person from Trinity College     
46:55     
Dublin and this is you know having to do with the genome instantiating a generative     
47:02     
model of the organism so the abstract reads how does     
47:08     
the genome encode the form of the organism and this is of course a question that has to do with the     
47:15     
genotype to phenotype mapping in other words if we take our genotype we produce     
47:20     
transcripts who produce proteins proteins produce tissues the tissues     
47:25     
then form this Phenix type and so how do we encode the form of the organism and     
47:31     
that's of course something that when people started sequencing genomes we     
47:36     
expected to be somewhat straightforward and of course it has not been that so we need to understand the nature of what     
47:44     
they call the genomic code and a lot of people have referred to what the genome is as a code and of course that's a a     
47:52     
metaphor that's fraught with a lot of problems so they actually speak to that     
47:57     
here where they say common metaphors such as blueprinter program fail to     
48:03     
capture the complex indirect and evolutionarily dynamic relationship     
48:08     
between the genome and organismal form so you know this idea of a blueprint or a program if you think     
48:15     
about what computer programs they're executed line by line so you have things in there that are line by line the code     
48:22     
you run through the code you can debug the code you can uh do things with uh     
48:27     
rewriting the code and we talked about this two weeks ago with that uh you know     
48:33     
some of the synthetic biology stuff we've talked about where you know people are actually doing this with bacterial     
48:40     
cells are able to take a gene they're able to rewrite the gene they're able to knock genes out or uh do other things     
48:48     
and they can execute you know different commands basically so you have uh groups     
48:54     
that have created bacterial colony that can produce different shapes like you     
49:00     
know faces or patterns or whatever or they can do things uh you know uh     
49:07     
metabolize different things on command you can put a switch in that can be activated so you know bacterial cells     
49:16     
we've kind of taken the genome and we've kind of created programs very simple programs uh the question is is you know     
49:23     
a does that Translate to say uh animal cells does it translate to     
49:32     
say like complex modes of development or you know and we know we can manipulate genes we can do crisper     
49:39     
we can do other things but you know what is that sort of in spite of itself in     
49:44     
other words is the complexity of those systems really not organized like a     
49:49     
program or a blueprint we are just able to do that because we're sticking our finger in a white socket or is this you     
49:56     
know maybe maybe things are programmatic so this is an interesting question and     
50:01     
we don't really have the answer to that but that's why we're talking about this and especially this paper because they     
50:07     
talk a little bit more about the constructive interactive processes that produce a developmental     
50:15     
phenotype so such metaphors are also not readily formalized either to treat     
50:20     
empirical data or to simulate genomic encoding of form in silico so this is where we want to be able to produce     
50:28     
forms on a computer we want to take a bunch of say uh DNA code which could     
50:35     
just be a series of letters or a string or a set of strings transform that into     
50:41     
some shape that's what we want we want to do that in a computer but it's not really easy to do in a computer so you     
50:47     
know we can do that with genetic algorithms to some extent but it requires you to have this sort of uh     
50:53     
blueprint on top of a blueprint and it becomes pretty you know pretty hard to connect to what's going on in nature so     
51:01     
uh that's that's an important set of points here we propose a new analogy inspired by recent work in machine     
51:07     
learning and Neuroscience and that is that the genome encodes a generative model of the organism so this is where     
51:14     
you have the genome encoding a model of the organism that's generative basically is encoding all these different     
51:21     
possibilities and then it's you know the developmental process select     
51:27     
from those different model different models or different instantiations of what's     
51:33     
possible in this scheme by analogy with variational autoencoders which is a tool of course     
51:39     
in machine learning the genome does not encode either organismal form or developmental processes directly but     
51:46     
comprises a compressed space of latent variables so this is otherwise known as the weight in space and so the weight in     
51:53     
space is basically all possible combinations that you can sample from to     
51:58     
get some sort of answer and you sample and Sample and Sample and you optimize     
52:06     
until you get to the thing that's you know most likely so that's an interesting approach it's using this     
52:12     
latent space to sort of get at this um these latent variables are the DNA sequences that specify the     
52:20     
biochemical properties of encoded proteins and relative affinities between transacting regulatory factors and their     
52:27     
target sequence elements so this whole passage basically says that we're     
52:33     
basically using DNA sequences in our latent space but not every DNA sequence     
52:39     
just the ones that specify certain biochemical mechanisms properties Etc so you know we're getting like we're     
52:47     
basically pairing down our genome to something that's functional or maybe you     
52:53     
know something that's relevant to a phenotype we have all these different configurations and then we can sample     
52:59     
from those and we can then select from those to get our um developmental     
53:07     
phenotypes um so you know this is not I don't know how to put this but basically     
53:13     
there's a connection here with epigenetic Landscapes so in epigenetic Landscapes we represent you know all the     
53:21     
possible uh forms and then we kind of see how     
53:27     
uh the developmental trajectory moves through that space so it's almost like we're taking this we're representing the     
53:34     
genotype as this latent space but the latent space is a landscape and then we're moving through that landscape with     
53:40     
a trajectory and we're finding an endpoint where we can then say this is the outcome of development so it's very     
53:47     
you know there's a connection there but I don't know exactly what it is just very interesting uh parallel there     
53:55     
collectively these comprise a connectionist network with weights that get encoded by the Learning algorithm of     
54:01     
evolution and decoded through the processes of development so this is where we have our connectionist Network     
54:08     
which you're familiar with a neural network are the uh cells the connections     
54:14     
the connections have weights you go through different layers your weights are a series of connections across     
54:21     
layers these weights are encoded by a learning algorithm of evolution so basically Evolution tells this uh     
54:29     
connectionist Network you know what connections are relevant which ones are not and then you get this decoded     
54:36     
through the processes of development meaning that it's then you know the development is where you're getting the     
54:41     
output in eval valuating it the weight and variables collectively shape an energy landscape that     
54:48     
constrains the self organizing processes of development so is to reliably produce a new individual of a certain     
54:55     
type this is is actually where he mentions the where they mention the uh     
55:00     
waddington's famous Epi genetic landscape so this is the direct analogy which is this energy landscape so these     
55:07     
latent V this latent space shapes the energy landscape or latent variables more specifically so there's this sort     
55:14     
of connection between the weight and space the energy landscape and then this     
55:19     
sort of selective mechanism that's in in this case encoded in a connectionist     
55:26     
network with weights um you know this is an interesting way the way I was thinking     
55:32     
about it more though is that there needs to be this feedback between     
55:37     
selection and representation The genome and of course in the last paper we saw     
55:43     
that there you know we have certain genes that are upregulated uh due to their functional     
55:49     
significance and so that's something that you know isn't something you can derive directly from development just     
55:57     
unfolding you need to have some encoding of what these cells are going to become     
56:02     
in order to have this upregulation of genes that are relevant for later development so you need to have this     
56:08     
feedback between the outcomes of development and the genes that en code development in the first place and you     
56:15     
know I don't know if they've explicitly kind of discussed this here but um I     
56:20     
think it's it's a pretty good model for getting what you want out of this the generative model analogy accounts for     
56:26     
the complex distributed genetic architecture of most traits and the emergent robustness and evolvability of     
56:32     
Developmental processes and so this is where of course you can provide an new explanation for     
56:39     
independent selectability of specific traits drawing on the idea of Multiplex     
56:44     
disentangled representations observed in artificial neural     
56:49     
systems so finan offers a conception that lends     
56:55     
itself to form formalization so we have the ability to formalize these you know observations of     
57:02     
makeup development and sort of How It's generating new forms so this is a really     
57:08     
interesting     
57:18     
paper so one part of this paper they talk about the current metaphors like     
57:23     
program in uh blueprint and and those metaphors fail to provide insights on     
57:29     
certain questions these are the questions that people typically ask uh     
57:35     
in development or that lead from those metaphors so the first one is how does     
57:40     
information get encoded in the genome and what is this information about so what is exact you know when you just say     
57:46     
there's information in the genome that's not really uh good good enough because     
57:52     
you know what is the information is it just kind of like you know transcribe     
57:57     
this don't transcribe this stop transcribing keep transcribing or is there just in is the entire form encoded     
58:05     
in in the genome so this is a question that we don't really know people throw the term code around a lot or encode but     
58:14     
it doesn't mean anything unless there's actually something that's being encoded the second one is what is the     
58:21     
data format of this information The genome so I guess what that means is like what you know what what is the     
58:26     
format is it um you know coordinate instructions is it um simpler than that     
58:33     
is it like migration information is it like you know something that has to do with Co the collective behavior of cells     
58:41     
what what is the data format so this is an important point because we don't know um I don't know if it even tells     
58:48     
you that much because maybe it just specifies what the protein should look like it's really not clear     
58:56     
how much s how does such information get decoded through the processes of development so even if you have     
59:03     
information coded in the genome how do you decode it how do you decode it in development so that it's useful it's not     
59:11     
clear how does genotypic variation relate to phenotypic variation so if you     
59:17     
have genotypic variation say if you have the splice variant we talked about earlier or if you have a mutant form how     
59:24     
does that relate to phenotypic VAR iation is it a direct mapping so when you have a point mutation do you get a     
59:31     
specific mutation that may hold true in some cases but not in others the same     
59:38     
thing with just you know your splice variance the splice variants don't automatically map to variation     
59:45     
functional variation um it's it's hard to say you know if we could build a map     
59:51     
between all the SP variant sa for muscle and you know what the functions of muscle are     
59:56     
it's you know it's not very clear what those relationships are so linking genotypic to phenotypic variation is     
1:00:03     
hard and I like the idea they have of using um a connectionist model I actually did something similar to that     
1:00:09     
in a paper a long time ago where used like a connectionist model to look at sort of how this variation is processed     
1:00:18     
and it's basically the idea is that you get this combinatorial mixing so you go from like different variants in the     
1:00:24     
genotype you combine different types of variation across genes and you get these     
1:00:30     
different phenotypic outcomes but there isn't this one toone mapping that you can say this genotypic variation that     
1:00:37     
you see here it relates to this phenotypic variation you see over here mean it it can happen but it's very rare     
1:00:44     
and in seans you'll sometimes see you know certain mutant phenotypes where a     
1:00:49     
gene has a certain set of point mutations that lead to some phenotypic deficit but that's that's not the norm     
1:00:57     
for most of biology um you get this in in uh you get this in specific     
1:01:05     
mutants because you know it's it's very rare you know that Gene has a pretty large effect on the phenotype so most     
1:01:11     
genes don't have that kind of an outsized effect how does the genetic architecture     
1:01:18     
of various traits arise so how do we even get a genetic architecture in the first place we know there's a lot of     
1:01:24     
junk in the genome we know there's a lot of like conserved elements in the genome so genetic architecture should be     
1:01:31     
conserved but it also should be sort of have a lot of redundancies and other types of things so in that case how do     
1:01:38     
we get a genetic architecture that maps to specific types of phenotypic     
1:01:45     
variation and then finally how do the underlying en codings evolve through time so if we have an encoding that's     
1:01:52     
matched to some you know functional aspect ECT if we change it if we evolve     
1:01:58     
It Through Time how does how do we ensure that it's you know produces a viable     
1:02:03     
phenotype and these are all things we can't really answer with the uh with     
1:02:09     
sort of the blueprint model nor with the uh code     
1:02:20     
model so the second paper here is so the second paper is generative     
1:02:26     
models of morphogenesis and developmental biology so this is in the seminars in cell and developmental     
1:02:32     
biology this is by Stillman and mayor and uh this is you know kind of     
1:02:38     
following up from this last paper talking about developmental biology and generative morphogenesis more     
1:02:46     
specifically so the abstract reads understanding the mechanism by which cells coordinate their differentiation     
1:02:51     
and migration is critical to our understanding of many fundamental process processes such as wound healing     
1:02:57     
disease progression and developmental biology of course mathematical models have been an     
1:03:03     
essential tool for testing and developing your understanding such as models of cells as soft spherical     
1:03:10     
particles reaction diffusion systems that couple cell movements to environmental factors and multiscale     
1:03:16     
multiphysics simulations that combine bottom of rule based models with Continuum laws so mathematical models     
1:03:24     
can often be Loosely related to data or of so many parameters that Model Behavior is weakly     
1:03:31     
constrained so we can build mathematical models that kind of describe these processes but you know sometimes they're     
1:03:37     
very Loosely related to the real world but sometimes we can also have so many parameters in our models because there's     
1:03:43     
so many things to take into account that we can't really ground our Model     
1:03:49     
Behavior too well so these are all things that we have you know when we use     
1:03:55     
sort of precise simulations we often need to you know propose precise     
1:04:02     
equations and it's not easy to do that in developmental biology it's easier to do that in physics maybe not in like     
1:04:09     
soft matter physics but definitely in like you know mechanics but in development we you know     
1:04:17     
doing that it comes at a cost recent methods in machine learning     
1:04:22     
introduced new means by which models can be derived and deployed in this review we discuss examples of mathmatical     
1:04:28     
models of aspects of Developmental biology such as cell migration and how     
1:04:33     
these models can be combined with these recent machine learning methods so this kind of goes through um some of these     
1:04:40     
mathematical models and then using uh some of the generative techniques to     
1:04:47     
sort of say you know can we do things better by using these generative     
1:04:52     
techniques they're also making this analogy to development H but you know it's not like     
1:04:59     
they're not talking about development itself being generative they're talking about the model being     
1:05:09     
generative so in this case they're looking at say mathematical modeling     
1:05:14     
versus maybe something like a deep learning model where we have this generative aspect to     
1:05:20     
it so however as we shall describe both modeling paradigms can be understood     
1:05:26     
under the broader framework of generative modeling and they talk about different types of generative modeling before this     
1:05:33     
generative models seek to model the underlying stochastic procedure for generating data opposed to discriminate     
1:05:39     
models which seek to separate data into subclasses or categories so you know sometimes in     
1:05:46     
equations we want to divide things up into categories or subclasses or sets or     
1:05:52     
other things that are you know we can use as is sort of a general input so we     
1:05:57     
want to have like we'll say okay we want these cells of this type we want these cells of this type we multiply them     
1:06:04     
together we integrate their trajectories and we get this answer uh with a     
1:06:09     
generative model you're not doing that that that's more of a discriminant model where you have you know specific types     
1:06:16     
of you might get categorization as an outcome you might get a predictive uh     
1:06:21     
set of values as an outcome those sorts of things and generative model you don't get those you're generating Solutions     
1:06:29     
and you're evaluating those Solutions so that's what they're getting at here uh I wouldn't have really described this as     
1:06:36     
just kind of a deep learning technique because in fact you can do Implement generative models of the wide range of     
1:06:42     
techniques and really it's that difference between generating Solutions     
1:06:47     
and having like you know Exact Solutions that's kind of the difference here     
1:06:53     
um and in fact sometimes s with uh differential equations you get it's it's     
1:06:59     
sort of generative in nature because you're generating answers you're you're Proxima answers and you're generating     
1:07:05     
you know different uh types of answers so that line isn't very you know it's     
1:07:11     
not hard and fast uh when understood as generative models both approaches share many     
1:07:17     
similarities and can in principle be combined we posit that doing so would allow for interpretable models of high     
1:07:23     
dimensional data or ultimately low dimensional models that provide highly accurate predictions of experiment     
1:07:31     
observation so in this paper they describe how biophysical models have been used in developmental biology     
1:07:37     
describe some of their limitations as traditional forms of generative models we describe     
1:07:43     
biophysical models of three processes fundamental to developmental biology cell cell     
1:07:49     
interactions Collective cell migration and physical morphogenesis we then describe gener     
1:07:55     
models that use neural network models so-called deep generative models we set out the main forms of deep generative     
1:08:02     
models and provide recent examples of their use as a nent field we are limited by the number of applications of deep     
1:08:08     
generative models to developmental biology specifically um and then they talk a     
1:08:14     
little bit about um tra using traditional and deep generative methods     
1:08:20     
to be combined as a form of Hybrid models that both retain the explainability of biophysics and the     
1:08:25     
flexibility of deep learning models so this is where they so they get into the biophysics     
1:08:31     
models interactions between cells cell cell interactions Collective cell     
1:08:37     
migration and morphogenesis so here you have these different mathematical models across     
1:08:43     
length scales cell cell interactions are these bidirectional interactions Collective cell migration is where     
1:08:49     
you're looking at cells moving in certain directions if they're coordinated or not and you know each     
1:08:56     
cell you have to account for each cell in their trajectory and sort of a     
1:09:01     
measure of you know at any point in time are they acting collectively or are they     
1:09:06     
acting in different ways there different phases of behavior here and then morphogenesis where you     
1:09:13     
have different components that interact but you know their components vary     
1:09:19     
across the model and depending on their interactions it can affect State changes     
1:09:24     
so the get into deep generative models here so they talk a little bit about deep generative models and so um they     
1:09:33     
talk a little bit about model accuracy is being EV valued by comparing model output to some specific observables or     
1:09:39     
summary statistics in this sense your biophysical models in the last section can be understood as describing a     
1:09:46     
generative process namely a process which leads the generation of a measurable quantity Where we measure the     
1:09:52     
system or output using summary statistics now this is these biophysical models     
1:10:00     
they distinguish this against generative models which are distinct from discriminant models which separate data     
1:10:06     
into subsets for example there are classification or clustering Al algorithm instead generative models aim     
1:10:13     
to describe the entire data space and the underlying data generating process     
1:10:18     
so this is where we have again this latent space where this uh energy     
1:10:24     
landscap Cape or whatever and then we want to take that and find solutions from that space and what the generating     
1:10:34     
processes so if we take our biophysical models and ground them in generative models we gain insight into the main     
1:10:41     
process by which a model might be developed so they talk about implic of generative models and they talk about this uh they     
1:10:48     
give two examples one are generative adversarial networks or Gans and the second is denoising diffusion model     
1:10:56     
diffusion models and Gans basically uh that have shown extremely     
1:11:01     
strong performance at generating things like text to image models that produce highly expressive images from text Proms     
1:11:09     
and so this is you know we can M map this to development in these kinds of models so implicit models such as Gans     
1:11:15     
have been used in cell biology as well for segmenting and tracking     
1:11:21     
cells um     
1:11:31     
and so we can do certain things like you know predict 3D cell geometries from 2D images and there's this expectation that     
1:11:38     
this will grow in the future um but of course implicit models well you know have having a lot of     
1:11:44     
Promise are particularly prone to limitations of interpretability as a learn generative process is typically     
1:11:51     
not straightforwardly accessible so you know just because we can sort of characterize that space and generate you     
1:11:59     
know outcomes it's not easy to sort of get an idea of what the process is     
1:12:06     
behind this generation and even if we're getting the right answer so there a lot of problems that we might have with such     
1:12:13     
a type of model in development uh then we talks about     
1:12:19     
explicit generative models and for this he uses this autoencoder variational autoencoder example so of course we     
1:12:27     
talked about variational Auto encoders in the last paper that's what they propose for     
1:12:32     
development um and so both AES which are Auto encoders and variational Auto     
1:12:38     
encoders or vaes follow a bottle neck like approach to     
1:12:45     
approximating this data space so this is where we basically take a neural network     
1:12:51     
we in we in input data we encode the input data into a low dimensional space     
1:12:57     
or Laten space and then decode this Laten space into these different types     
1:13:04     
of outputs and so we're trying to minimize the difference between input data that     
1:13:09     
is passed to the encoder and output data which decodes samples from the weight in space and so this is in variational Auto     
1:13:18     
encoders we have this probability density function that we have to worry about so we're actually building this     
1:13:24     
probabilist istic model that we sample from and so this is exactly what we're talking about with the weights in the     
1:13:30     
last paper so this is really you know this may allow us to build a lot of     
1:13:35     
synthetic data that we might be able to uh exploit but also you know we may be     
1:13:41     
able to build uh you know models of biophysical properties that may not be     
1:13:47     
observable so this is another um you know benefit maybe to     
1:13:52     
this and then finally we we talk about these hybrid approaches so to develop     
1:13:58     
models directly from data explicit generative models can be combined with methods for inferring symbolic equations     
1:14:05     
and so this is where we have uh this ability to synthesize uh symbolic equations or you     
1:14:12     
know build symbolic equations for systems but also have this generative model where we're producing different     
1:14:19     
possibilities so there's a whole literature on um sort of producing     
1:14:26     
scientific laws and people applying machine learning techniques to Scientific data sets to find candidate     
1:14:33     
laws and those laws are described as equations so this is an area that's you     
1:14:39     
know people have been revisiting this for decades and not making you know     
1:14:45     
making some progress but it's been kind of rooted in physics mainly so you know     
1:14:51     
and you know so you know having a role you know for this in developmental     
1:14:57     
biology is really exciting so a very specific example here are reaction diffusion systems where we have only     
1:15:05     
partial model equations and you know we can use artificial neural networks to augment     
1:15:10     
model equations where only partial Dynamics are known but a full model output is available so this something     
1:15:17     
that's you know exciting in terms of like getting you know connecting the     
1:15:23     
equations with these outputs hybrid overall hybrid learning     
1:15:30     
resolves the tension between modeling paradigms by restricting the application of neural networks to a subset of the     
1:15:36     
broader biology under     
1:15:42     
investigation finally we have this paper uh from Stuart Kaufman from physica     
1:15:49     
a and this the title of this paper is The Ensemble approach to understand genetic regul atory networks so they     
1:15:57     
don't get into regulatory networks uh too much in the last two papers Al the first paper I think that's the direction     
1:16:04     
they're going this talks about understanding genetic regulatory networks which are often kind of opaque     
1:16:12     
in terms of how they map to actual biological systems but what what steuart's going to do here is give us an     
1:16:18     
ensemble approach to this so the abstract is very short and it reads     
1:16:24     
understanding the genetic regulatory Network comprising genes RNA proteins     
1:16:29     
and the network connections and dynamical control rules among them it's a major task of contemporary systems     
1:16:36     
biology I focus here on the use of Ensemble approach to find one or more well- defined ensembles and model     
1:16:43     
networks whose statistical features match those of real cells and organisms such ensembl should help to explain and     
1:16:49     
predict features of real cells and organisms a couple things to say about this paper um first of all he's using at     
1:16:59     
least he's made attempts in the past to use random Boolean networks where gen is models as if it were an onoff device so     
1:17:07     
your network has n genes with K inputs so you have a certain number of genes     
1:17:13     
each of those genes have a certain number of inputs and the time is updated synchronously by a central clock so     
1:17:20     
basically this is not something that you see in cells this is just kind of an abstract ction where you have a certain     
1:17:26     
number of genes a certain number of inputs uh time uh component and then     
1:17:31     
this switch that goes on all so this you know may be useful in terms of     
1:17:37     
understanding very simple uh networks where you know you're turning on genes     
1:17:42     
or turning off genes like in the first paper where we talked about you know building an mRNA uh pool you know you     
1:17:49     
might be able to do something like that but beyond that maybe it's not it's not so good for example to understand     
1:17:55     
uh fine fine tune regulation of specific traits or something like that um and     
1:18:02     
then at least three initial ensembles of Boolean networks distinguished by their wiring diagrams are now available so we     
1:18:09     
have these classic random Boolean Nets each of the enges receives the same number K inputs randomly chosen among     
1:18:16     
the ends so you have these random Nets where the inputs vary across     
1:18:23     
genes you have these scale-free networks in which there are power law distributions of inputs and     
1:18:29     
outputs or both from the genes so you just have a distribution of inputs and outputs that follow a power law uh where     
1:18:37     
you know you have some genes with many inputs some gen and outputs uh many     
1:18:43     
genes with many fewer inputs and outputs and this is sort of this it there's no     
1:18:49     
specific uh you know scale to it it's just kind of in that     
1:18:55     
according to the tendency and again that's useful again for some of the things we saw in the first paper or we     
1:19:01     
saw kind of what looked like they could be power law distributions of the amount     
1:19:07     
of transcript produced by a certain number of genes uh then we have these Medusa     
1:19:14     
networks and which a small regulatory head where like maybe one or two units     
1:19:19     
that are regulatory contain a network among transcription factors that regulate one another and an Asic look     
1:19:26     
directed graph hanging off that head which contains genes that are regulated but not     
1:19:32     
regulating so you have like genes that are regulating each other in these small modules you have a head or a couple of     
1:19:38     
heads that do the regulating uh of these modules and that's how you know it's     
1:19:45     
kind of like uh very highly modular Network the spse data available tend to     
1:19:52     
support either a scale-free network and this is TR in eoli or a Medusa Network     
1:19:57     
diagram which is TR in East so these are citations 11 and 13 so     
1:20:03     
basically uh you know you have examples from ecoy and East which tend to suggest     
1:20:09     
that it's either scale for your Medusa in addition to these structural classes different biases in the classes     
1:20:16     
of Boolean functions can be introduced particularly including canalizing and hyp functions in certain post classes     
1:20:24     
non Boolean Nets one can consider discrete estate networks that remain synchronous Boolean or estate     
1:20:32     
networks that are randomly asynchronous probabilistic Boolean Nets     
1:20:37     
Boolean Nets in which genes have a distribution of time scales networks of piece-wise linear equations networks     
1:20:44     
with continuous hell functions these are equation-based networks or govern by equations networks with more detailed     
1:20:51     
and realistic equations are representing both RNA and protein synthesis and then finally networks's governed by     
1:20:57     
stochastic equations of motion so The Ensemble approach can     
1:21:03     
include all these different types of genetic networks the Medusa networks have     
1:21:08     
particular interest and that Evolution can add new genes to the C cyclic graph of genes that are regulated by the head     
1:21:16     
so you have this uh basically this uh sort of or you know ordered uh set of     
1:21:23     
connections that get regulated by a head and so the head actually serves to regulate the     
1:21:30     
entire set of genes that are connected in a certain direction so we have this uh evolution     
1:21:38     
canand new genes but plays no regulatory role without altering the attractors of the regulatory head so this is where     
1:21:45     
evolution is acting on the network merely in terms of selecting the components not the regulation itself the     
1:21:51     
regulation can emerge linear actions okay so I think that was a fun tour of     
1:21:56     
those papers hope you learned something
