## TRANSCRIPT

0:01     
hi than here um yeah I'm supposed to     
0:06     
have something written by now and I'm still missing a     
0:11     
page well it's I'm dealing with 10 sties and it's not that simple like they     
0:18     
behave like they behave yeah you know that is sometimes yeah     
0:27     
so need I I thought maybe I would listen in     
0:32     
today and and do do some graphing okay     
0:38     
yeah I just fill with the graphs get them well I I need     
0:44     
to put three of them on like one one graph yeah yeah see if I can I can do     
0:53     
that because uh then I can show the differences in slope right now I'm     
0:58     
dealing with a slope     
1:04     
problem and a couple other things it's strange you you put um a     
1:11     
stronger um attachment holding the model down and sometimes the     
1:20     
slopes increase as you increase um the     
1:27     
pre-rain and sometimes they get     
1:32     
smaller they can do both yeah or wow that's     
1:38     
interesting it does it depend on where you put it in the network or well that too um I tried taking     
1:48     
the attachments off the middle and the thing becomes basically a     
1:54     
straight line okay so you have to have some sort of gradation     
2:00     
of attachments but I I tried to make a model that would do that and it won't     
2:07     
attach to the um attachments I made in between like you make your grid work and     
2:12     
then you try to attach in between and no won't do     
2:20     
that does not compute this is CMO for you you think     
2:26     
you can do something and you can't I don't know so I I couldn't really     
2:33     
investigate that that much like I can mess around with the middle attachments but that's     
2:43     
all so maybe I maybe that's what I should explain today I need I need to explain     
2:51     
things to uh a pair of professors who want me to be done     
2:58     
yesterday oh what would you do in this case yeah yeah I don't know the answer I     
3:04     
guess I mean you could play around and see like if you make stronger attachments in certain locations if that     
3:12     
has an effect on it because you know Network you're dealing with a lot of     
3:17     
nonlinearities sometimes you're dealing with like dependencies so it's kind of hard to like isolate it's all intentions     
3:25     
so if you change change the um attachments in one     
3:32     
part it's it's going to change the whole system yeah so I just like there's so     
3:37     
many things to investigate with the model and I haven't I've only made it partway through and then I'm supposed to     
3:44     
present it yeah     
3:50     
yeah yeah I don't know it's an interesting problem I mean because you could probably spend your you could     
3:56     
write an entire thesis on just like evaluating I modified this little part and it did     
4:03     
all these things and then I modified this little part over here and it did all these different things yeah like you     
4:11     
could could spend a couple of years mon just the one model modify things because     
4:18     
it doesn't um I I can't automate it like I said     
4:24     
yeah so it just takes forever that's science I     
4:31     
guess land or whatever it is eus land     
4:37     
yeah okay so I've got Canadian news on in the background I don't can you hear     
4:43     
that or no no it's okay     
4:49     
anyway hi Dick hi hello hi yes I I can hear it but I can't understand     
4:56     
it oh okay     
5:01     
right something to do with um was it RF Kennedy being uh the head of health and     
5:09     
not believing in vaccines and stuff that's what's on the     
5:17     
news anyway uh so yeah good luck with that uh     
5:24     
project yeah um I I don't even know how to explain it to them like I can only     
5:30     
investigate part of this model and this is what I have investigated and it's     
5:35     
this one particular model is close to what's actually happening in the excise     
5:41     
lung model that I have that they have anyway that's I     
5:52     
know yeah okay um yeah so how are you doing     
6:00     
dick what's going on there uh pretty good we had a turkey dinner yesterday     
6:07     
okay so I'm sure it's slow     
6:14     
today right yeah so I guess we can talk about     
6:20     
a few things today uh I have an interesting paper on Sean's uh selfie and then I have another     
6:30     
on um some things on the cell's operating system it's kind of an     
6:35     
interesting blog post but we'll see let's see if this     
6:40     
is um if it'll turn on Okay slowly you heer slow     
6:49     
um I've got this um explanation for how     
6:55     
the stationary solver solved my equations     
7:01     
and it says compile equation stationary in study study one solution     
7:07     
one and then it gave the time it started geometry shapes function linear     
7:14     
lrange and then it is running on Intel 64 family 6     
7:20     
model 158 stepping 10 genuine     
7:27     
Intel and then using one socket with six cores in total on desktop such and such     
7:36     
anyway what I wanted to know is what on Earth do they mean by stepping     
7:43     
10 uh well M maybe it's like step size in the     
7:49     
simulation could be what does that translate into um seconds for instance     
7:58     
uh let's see I'm not sure I might look it up     
8:05     
here usually the step size is like arbitrary units of time it could there     
8:11     
could be in the documentation like a mapping between seconds and and units a     
8:16     
time but it's hard to say uh let's see what is this is are you     
8:22     
using console yes right     
8:33     
so it says a controlling okay yeah here's the okay     
8:40     
I found in the documentation controlling the time dependent solver time steps I don't know if that's what you're     
8:47     
looking for but basically they talk about like in the solver it's uh make it estimates an     
8:55     
initial time step size it take based upon the total simulation time plus if     
9:01     
you've solved the same model twice for different total times the initial time step can be different this can lead to     
9:08     
different results if there's something significant happening immediately after the start time then you can specify the     
9:14     
initial time step size go to the time step solver settings time stepping     
9:20     
section enable the initial step option and enter a fixed initial step size okay     
9:26     
where where did you looked at that from how did you yeah I'll grab it here and put it in the chat oh okay is from the     
9:35     
console uh documentation oh really as I for it well     
9:41     
I just searched on on um I think Bing or something and uh came up with this but I mean this     
9:48     
is like yeah it looks like there's a screenshot in there where they just talk     
9:55     
about the time units so like you can set the time units and the     
10:01     
range and then you know it has like it goes through from one point to another but of course it takes a step size every     
10:08     
time there's an iteration okay well you came up with     
10:13     
that faster than I certainly did so well I don't know it's useful yeah I mean it     
10:18     
probably is but like yeah step size is always kind of like you know arbitrary     
10:23     
units but you usually Define it in the simulation or usually you can define it     
10:29     
in terms of a process all it is is just that iteration uh yeah this is like the     
10:36     
description you put in the chat so like uh let's see so yeah it's just describing like what it's doing stepping     
10:43     
10 why is it a genuine Intel I guess that's the thing that they     
10:51     
yeah yeah okay yeah so it's a linear lrange     
10:56     
running an Intel 64 yeah okay so I mean that's all like information you need because every     
11:03     
computer is going to run it differently uh we've been talking of course uh Morgan and I have been talking     
11:11     
about a lot about uh virtual machines and running simulations of virtual machines and the advantage there is that     
11:19     
you can run them independent of the hardware like you can run them on different hardware and get the same     
11:24     
result but that's you know you don't need to do that it's just kind of one of these things we've been talking     
11:30     
well um the stress strain curves of tissue are very dependent and tanies are     
11:39     
very dependent on the um how the forces applied whether     
11:46     
it's um fast step slow step continuous step like a like a     
11:54     
staircase step like it's all it's I don't know what it is I don't know if it's a     
12:01     
staircase or if it's a square wave and what's the ramp on the Leading     
12:07     
Edge of the square wave I don't know I can't tell I don't     
12:13     
know so yeah ask well what's the rate here well the computer says it's 10 yeah     
12:24     
10 so yeah I mean I wonder if people have done like experiments in soft matter like to look at some of these     
12:31     
parameters like you can get you know parameters and biological materials but     
12:37     
also from materials that have basically the same hardness and things like that     
12:42     
and you just experiments I've got uh examples of     
12:48     
how the applied force rate changes the curve I have that okay yeah I mean     
12:57     
that's fine so I I'm I have a thesis with I have no     
13:04     
idea what the rate is of the applied force that's a     
13:10     
main main feature of     
13:19     
of oops anyway so that's um that's my     
13:25     
complaint today I guess CU it doesn't say yeah then it's     
13:32     
10 yeah yeah it's hard to get all these things sort of calibrated and figuring     
13:39     
out what's like realistic and what's not but yeah and if you go     
13:46     
into um time stepping like if you then they will give you ways of altering     
13:53     
altering the Leading Edge of the curve um for your experiments but um     
14:01     
this is a stationary study so     
14:10     
no there there might be a way of altering um stationary studies time step     
14:17     
but it's maybe I should ask ask home so     
14:23     
that yeah you could do that but I     
14:29     
just I just need to know what it is I at this point it was done at this rate I     
14:35     
don't I don't care yeah I just need to know what it is and     
14:42     
T doesn't cut it in the engineering yeah well I don't know what people are doing     
14:48     
in other studies with console if people are doing anything close to this like I     
14:54     
don't know what thing it's a building and it doesn't matter you hit it with with a little hammer or a big hammer it     
15:00     
still sit sits there if you hit it with an earthquake     
15:06     
well that's different well that would be paper     
15:12     
earthquakes and embryos yeah earthquakes and earthquakes     
15:19     
and integrities they use yeah modeling for buildings for uh earthquakes right     
15:27     
to make them earthquake proof     
15:33     
yeah that's they do you have to yeah anyway um yeah so I still anyway it     
15:40     
doesn't solve the problem of what is 10     
15:49     
yeah so I have a 10 problem yeah 10 units 10     
15:58     
units what are the in yeah can I can I ask Susan a quick question yeah yeah     
16:04     
Susan uh are you familiar with spherical tensegrity     
16:11     
structures um sort of sort of okay there the spirit     
16:17     
cells uh if I have a question uh if I designated n     
16:24     
struts okay and put them on the surface of a spere could you calculate whether     
16:30     
or not the structure is stated um I could throw it in Pomo it     
16:37     
would tell you immediately whether it was how stable it was oh okay there are     
16:43     
some 10 seties that are not stable INT in console or quite a few of them but if     
16:49     
they are stable in console they are definitely stable but there's there's some     
16:56     
structures that are not stable and com that in real life would stand stand up     
17:03     
okay okay good for future reference yeah got a very strange project I'm     
17:11     
trying to determine whether or not Tegrity has any relationship to the     
17:16     
structure of the atomic newness oh yes I was supposed to send you the Mathematica code but it's four     
17:26     
triangles oh okay well the the the problem is in uh in atomic     
17:36     
nuclei the uh there seems to be what they call Magic numbers which make them particularly     
17:42     
stable and my question is whether not there something similar in configurative     
17:50     
fce okay but it's still it would be a big project because the number of struts     
17:57     
would go up to uh 100 so that's that's 100 is doable oh yeah     
18:06     
100 is doable but one to 100 one all of     
18:12     
them okay yeah like I said it's not you can't     
18:18     
automate if you're using finite element analysis on the Tegrity you probably     
18:24     
can't automate it oh I can't use their normal method of um     
18:32     
giving a a step uh changing a step size in console because it wants to use the     
18:39     
automated method to change the step     
18:44     
size and um the whole thing falls apart oh     
18:49     
wonderful so it becomes unstable even more stable well it doesn't give it enough time to settle I think that's     
18:55     
what it is okay got you okay okay Bradly carry on all     
19:05     
right well yeah I guess I just yeah it's about as far as I can go with helping you on that I guess okay well thanks     
19:13     
yeah if other like I've asked this of a couple other people and they have similar issues so okay I'll give it to     
19:22     
the my professors this afternoon and um I asked some people are fairly     
19:29     
knowledgeable about computer systems and they don't know yeah well I mean yeah     
19:35     
it's probably a console specific thing so and console necessarily     
19:41     
but well there it is I mean you found the section where they mention it but     
19:50     
anyway well good luck with that and you know hopefully things pan out yeah thank     
19:56     
you sometimes they just tell you what they want to see they don't care if it's like you know you have to do a work     
20:03     
around and the committee has to decide what that is so yeah or I could send the     
20:08     
whole quote from console to console and say okay so I have 10 what is that     
20:17     
yeah I think I think I'll do that send them a help request yeah they hate those     
20:24     
but that's okay     
20:29     
yeah okay so the first thing I want to talk about was this Artic it's a blog post that kind of goes you know kind of     
20:37     
rambles a little bit but it's basically this uh consideration of your operating     
20:45     
system on a thumb drive so they're talking here about um uh     
20:51     
ribosomes and sort of the RNA associated with ribosomes and the genes associated     
20:57     
with those ribosomes so this is in bacterial I think they're focusing on bacterial cells in microbiology and then     
21:04     
you know sort of thinking about that in terms of uh Maman cells or or eukaryotic     
21:10     
cells more generally so uh you this is by Kristoff this is I     
21:17     
think the American Society for microbiology blog um so there it's a blog post uh so     
21:26     
starts off at the heart of every cell's operating system you'll find the ribosomes these ingenious nanom Machines     
21:33     
of about 20 nanometers in diameter that translate messenger RNA or mRNA into all     
21:39     
kinds of proteins the cell needs to build and maintain itself so if you know the biology you have mRNA molecules that     
21:47     
are basically transcripts from a gene they get fed into this Machinery the     
21:53     
MRNA are transcribed instead of translated at this stage to amino acid AIDS as amino     
21:59     
acids build a chain which U go you know contribute to proteins and so ribosomes     
22:06     
are those sites where translation occurs there's a very um you know highly sort     
22:13     
of um highly um evolved mechanism of     
22:19     
different types of uh trnas or translational rnas which can take those     
22:26     
uh the codons from the MRNA and translate them into amino acids and so     
22:31     
you know there there's studies where they look at the different pools of trnas and how they     
22:37     
uh you know work to sort of uh transcribe some things over other things     
22:44     
and that sort of thing so there's not only translation going on at ribosomes but selection of different uh codons and     
22:51     
selection of different amino acids and they contribute to proteins and so the     
22:56     
other thing to know about uh ribosomes is there are a lot of them in a Cell so it's not just one ribosome there are a     
23:03     
lot of ribosomes in different places in the in the in the cell in the     
23:10     
um you know where mrnas can be fed into this machine and turned into     
23:16     
proteins so this kind of goes on about this uh see ribosomes busy at work here     
23:23     
this is another image where they show kind of the I don't know if this is a good picture this is just kind of some     
23:30     
micrographs of uh eoli chromosomes being transcribed and then simultaneously     
23:37     
translated and then this is a picture of the RNA genes that are co-transcribed so     
23:44     
that's another blog post on that same blog uh ribosomes are first described by     
23:51     
the electro electron microscopist George pad in a paper from 1955 which he p on electron dense     
23:59     
particles lining the endoplasmic reticulum of rat pancreas cells then the quote is the relationship     
24:05     
between the membrane of the endoplasmic reticulum the small granules deserve special     
24:11     
consideration indeed they deserved and got consideration so we have at least     
24:17     
50,000 papers in PubMed dealing with ribosomes and that's about the same     
24:23     
number of ribosomes we have in eoli cells so these are bacterial cells ecoli     
24:28     
cells and even just in bacterial cells we have 50,000 and in eukariotic cells we have     
24:33     
more of that so this is a ubiquitous uh machine I guess you could     
24:39     
put it uh with such numbers it's no wonder cells spend roughly two-thirds of their energy Budget on the synthesis of     
24:46     
ribosomes in Translation so that's an interesting statistic that that's uh     
24:52     
really you know impressive that your the cell spending most of its or maybe     
24:58     
2third of its energy on translation and synthesizing ribosomes so you know that's and of     
25:05     
course it's important to to make proteins but it's a lot you know so it's     
25:10     
it's a really important process and then he says and yes cells really tend to stick to their operating     
25:18     
system so I guess what he's referring to as the operating system a ribosomes and ribos RNA and trnas     
25:28     
translational rnas and the sort of the Machinery that it is involved in so uh organel such as chloroplast and     
25:37     
mitochondria have ribosomes and they're genes encoding their components that resemble those of their bacterial     
25:43     
ancestors uh well that you know your ribosome is basically information and     
25:49     
you can put it on a thumb drive which is its whole other analogy but anyways continue on here and     
25:57     
then arranged into subunits and the you know the subunits that are in there differ from     
26:04     
procaryotic cells and eukaryotic cells with the sites of protein synthesis and mRNA binding at their interface it was     
26:12     
assumed for decades that the ribosomal RNA provides a scaffold on which the orderly bound ribosomal proteins perform     
26:18     
their job of joining amino acids to a grin peptide Gene the first hint that this might be less than half the truth     
26:25     
came from deciphering the genetic code when it was shown that codon anti-codon     
26:30     
interactions of rhos bound tnas which are the translational rnas and mRNA     
26:36     
which is the messenger or drive translation so this is where we have these codon anti-codon interactions with     
26:44     
proteins me casual bystanding so this is where the model where you have the you     
26:51     
know the Mr uh being read off is sort of these interactions then the proteins     
26:57     
kind of being the result of that but it still came to some surprise when The elusive     
27:03     
ribosome uh peptidal transferase the enzymatic reaction that joins amino acids by a peptide bond turned out more     
27:10     
recently to be actually a ribosome type activity located in the RNA mu of the     
27:16     
large ribosome subunit so we have a remnant of the RNA world as key     
27:21     
component of the cell's operating system maybe so they're making this kind of statement about how the ribosome is     
27:29     
doing maybe more than just kind of producing proteins that the proteins are     
27:34     
actually inter interacting with the ribosome itself we have this sort of RNA World Remnant where RNA was really the     
27:42     
thing that was uh you know involved in uh passing information on in terms of     
27:51     
parity in life instead of De so intuitively you would probably     
27:57     
guess that procaryotes the gene encoding such vital parts of the operating system are encoded on the chromosome or the     
28:04     
hard drive again another metaphor um they are in almost all known     
28:10     
cases in Archy and bacteria alike in bacteria the three ribosomal     
28:15     
rnas 16s 23s and 5s are rnas are encoded     
28:21     
by rdna gen that are organized as an operon and transcribed from a common promoter so the operon is this sort of     
28:28     
set of promoters and genes that are uh ubiquitous in the bacterial genome and     
28:34     
so they're talking about these R DNA genes and you have an operon where you     
28:40     
have multiple genes transcribed from a common peroor um and that's figure three which     
28:46     
is down below a long primary RNA transcript which includes the two so-called internally transcribed spacers     
28:53     
that in many cases in code tnas is processed by specialized RNA     
28:58     
which are these enzymes to yield the individual RNA species that are then assembled with the ribosomal proteins     
29:05     
into precursor ribosomal subance so this is figure three here this is a schematic     
29:11     
representation of sequence variation showing these different genes for these different     
29:17     
subunits in anopa from five Prime to three     
29:22     
prime okay um so kind of talks a little bit about     
29:29     
um how deviating from the rule that in compact bacterial genomes most genes     
29:34     
come in just one copy per genome one finds between only one RNA operon uh and uh so in in some species     
29:43     
of bacteria we have only one RNA operon and this is for a genome size of 9.1     
29:50     
megabases and a whopping 18 in cyanobacterium the species of cyanobacterium with a genome size of     
29:57     
10.6 me basis so apparently the number of RNA genes is not related to a genome     
30:03     
size but may reflect different ecological strategies of bacteria although mostly confined to a     
30:09     
chromosomal localization a few cases are known where additional copies of the RNA     
30:14     
opon are located on large plasmids uh which are present present in     
30:19     
Alo species in numus Alpha and beta proteobacteria many cases provides     
30:25     
another example for an operating system on a thumb G so this is again I guess where they're     
30:31     
going with this is like there's this can you describe the entire operating system of a bacterial cell and then is it     
30:39     
compact enough to fit on a thumb drive and we're thinking in terms of what genes that are encoding so they're gen     
30:46     
there's an operon that encodes all these RNA subunits then those subunits contribute to this mechanism and that's     
30:53     
the thing that you know so we can we put all of that on a thumb drive you know this is important for synthetic biology     
31:00     
because we want to be able to build you know maybe a minimal cell where we have a minimal set of components that can     
31:07     
actually self- sustaining transcribe genes produce proteins and do so maybe     
31:14     
like you know you know some sort of container and so that's that's kind of     
31:21     
where they're going with this um so does this mean that also our     
31:26     
RNA genes participate in horizontal Gene transfer there is some evidence that this is in fact the case therefore R RNA     
31:33     
based phog genetics The Marvelous idea that allowed Carl W to proposes three domains model which of course was a     
31:40     
famous study where Carl wo laid out the framework for ukar procaryotes and Ary     
31:49     
bacteria may turn out to be really messy in some niches at least this would then be the normal State of Affairs in most     
31:56     
most of biology so it's an interesting post um basically the idea is you can     
32:01     
take these RNA genes create ribosomes and then those ribosomes if you have     
32:07     
some genome you can transcribe that genome and produce proteins and so how big does that need     
32:13     
to be and that's what they're kind of talking about     
32:18     
okay all right so then the next thing I wanted to get to today was this paper um     
32:24     
this is a bioarchive preprint uh this     
32:31     
is a paper on early embryogenesis and seans it's by a Chinese group and     
32:38     
they're talking about quantitatively resolving sulfate in the early embryogenesis of     
32:44     
celegans so what they're going to do is they're going to try to figure out how     
32:49     
Sate is determined in where we C elegans we know that for example it's a deterministic     
32:55     
process meaning that you start out with like a two cell embryo where you have     
33:01     
basic polarity at an anterior and a posterior end and those two     
33:08     
cells produce eight founder cells then those eight founder cells go on to     
33:14     
produce different types of tissue different types of organs and you know you can you can't     
33:21     
really obliterate those early cells without basically creating a non-viable     
33:26     
Embry so you can't just have cells that kind of compensate for other cells not     
33:32     
being cgans produces certain number of cells for the adult and if those cells     
33:38     
are missing in the embryo then they're missing from the adult or at least they're sort of The Descendants are     
33:44     
missing from the adult so what they're going to do is they're going to talk about Sate and how we can resolve this     
33:50     
quantitatively so let me go into the abstract so the abstract reads the nematode seans exhibits an invariant     
33:57     
cellon during its development or the G molecular Network that regulates the development is crucial for the     
34:03     
biological process uh well there are many molecular cell atlases describing the phenomena     
34:09     
and key molecules involved in cell transformation the undering mechanisms for a from a systems biology perspective     
34:16     
has received less attention based on an endogenous molecular cellular theory     
34:22     
that relates the molecular mechanisms biological phenotypes we've constructed a model core endogenous Network to     
34:30     
describe the early stages of embryonic development of the nematode so what they're going to do is they're going to create a gene regulatory Network model     
34:37     
of this of these early stages and they're going to pick candidate genes that are involved in this early process     
34:45     
and they're going to look at they're going to build a genetic regulatory Network that shows how those things are     
34:52     
sort of uh you know U regulated and down regulated that     
34:58     
uh so that's what they're doing here different cell types and intermediate cell States during development from zygotes to founder cells Orr respond to     
35:06     
the steady states of the network as a nonlinear stochastic dynamical system so     
35:11     
the other thing they're doing is they're taking this and they're building a nonlinear stochastic dynamical     
35:17     
system and they're going to try to uh that's how they're going to try to get some insight on this connections between     
35:25     
steady States from a topological landscape ape that encompasses known developmental lineage trajectories so     
35:32     
you know we have these steady cell States and we have these transitory States or these transitions between     
35:38     
states and you know sometimes you can treat that as a sort of a Markov model     
35:44     
where there's certain probability of transition uh but you can also treat it as this sort of nonlinear stochastic     
35:52     
dynamical system that you can plot on this topological landscape basically this differentiation landscape that     
36:00     
allows you to describe known developmental lineage trajectories by regulating the     
36:06     
expression of agents in the network we quantitatively simulated the effects of the win and Notch signaling pathway on     
36:13     
Cell fate Transitions and predicted the possible trajectories of trans differentiation of the ab cell which is     
36:21     
one of these founder cells across the lineage so they're going to look at the     
36:26     
wi not win and Notch signaling Pathways they're going to look at that effect on sulate transitions win and Notch are     
36:34     
sort of like at the top of the network and then there are other genes that are below that that are going to be affected     
36:40     
by that activity and then you're going to look at they're going to look at the different trajectories of trans     
36:47     
differentiation in the ab cell across the lineage the success of the current     
36:52     
study may help Advance our understanding of the fundamental principles of Developmental biology and self     
37:00     
determination so they kind of talk about the importance of these early stages of     
37:06     
development um and kind of how that is you know cgans is an important model for     
37:13     
that they talk about the sources of single cell RNA sequencing data so this     
37:19     
is RNA seek data that they're uh focused on and how you know we can make really     
37:26     
high definition maps of this process um so the Single Cell trans     
37:33     
cryptone of the L2 larel stage of cgans as will more characterized along with the construction of cell maps at various     
37:39     
stages of early embryonic development and the creation of spao temporal atlases so this is all in you know     
37:46     
different aspects of development so of course we have the larvel stage L2 LEL     
37:52     
stage it's been well characterized in terms of single cell transcriptomics we also have some data from embryonic cells     
38:01     
but it's you know that's still kind of a work in progress uh we have cell Maps so we have     
38:08     
like the data it's kind of in pieces and so you know we can get access to some of     
38:14     
that data to inform you know this process so you know molecular structure     
38:22     
generates vast amounts of information but we have to sort of make sense of it so that's what they're going to do     
38:29     
here then they talk about waddington's landscape and how that might end up     
38:34     
being this sort of way to kind of make sense of a lot of this these data for     
38:40     
different cells so you know during the developmental phase cells reach their final differentiated types by distinct     
38:47     
fate decisions exploring the changes in cell States and the process helps understanding the relationship between     
38:53     
different cell types waddington's epigenetic landscape con conceptualizes developing cells those balls are on down     
39:00     
a hillside uh and then they talk about like the minimum and maximum of that landscape the shape of these hillsides     
39:07     
are determined by Gene regulatory networks uh and so you can work our Gene regulatory networks for this     
39:14     
process um and muray all which is a data set that actually the Murray 2012 data     
39:20     
set is important uh for understanding what the key Regulators of different lineage Fates are well buau at all     
39:28     
established the gene regulatory Network governing self fate specification in the endomesoderm during     
39:34     
the embryonic development so this is where they kind of piece together a lot of the available     
39:40     
data okay so they talk about the founder cells and then they kind of get into     
39:48     
this sort of endogenous Network modeling what they're doing they have to pick a number of uh different genes for this     
39:55     
they actually want to uh uh pick like four modules of functional genes so they     
40:03     
selected four key modules cell polarity cell cycle signaling Pathways and     
40:09     
lineage specification so this is where you can do you know go into the data look at the     
40:15     
genes look at their function or at least their annotated function and you can     
40:20     
construct networks based on this function so you know there are a number of genes that are involved in cell     
40:26     
polarity the number of genes involved in cell cycle signaling Pathways lineage     
40:31     
specification and those are all sort of the core parts of these modules so we     
40:36     
have four modules and we we presum they interact they're genes within those     
40:42     
modules and then we want to look at how those modules interact so when they talk     
40:47     
about an agent here they're talking about a node which is basically just Gene and then its expression state is     
40:55     
contingent on the expression state of other agents or nodes and then you know     
41:01     
whether that relationship is supposed to be sort of suppressive or facilitative U     
41:08     
you know whether it leads to up regulation or down regulation then that's how the network     
41:13     
function so uh an agent on the network in the current context can represent     
41:19     
typically a cluster of relevant genes molecules and or a signaling pathway the     
41:25     
genes or proteins regulate each directly or indirectly through transcription translation or signaling hence we take a     
41:32     
chus grain approach to abstract these regulations into activation or inhibition interactions so it's like     
41:39     
activation of the U regulation inhibition inhibition of be down     
41:45     
regulation so then they talk about the polarity during asymmetric cell division being controlled by the par polarity     
41:52     
proteins and then you can map that to different genes that are involved in that protein so     
41:57     
uh we have par six which is the representing the anterior protein complex par six Par 3 PK     
42:05     
C3 well lgl1 represents the posterior complex lgl1 R1 and R2 so you have this     
42:13     
set of proteins that represent the anterior and posterior ends you have par     
42:20     
six and lgl1 and then those uh the those two     
42:26     
genes represent this protein complex okay and then P plk1 stands for     
42:33     
the complex pk1 ma3 and X5 which links polarity protein to the asymmetric     
42:39     
distribution of Agents within the cytoplasm so this is where we have the     
42:45     
polarity genes polarity proteins and then the distri asymmetric distribution     
42:51     
of different nodes or or whatever within the cyop so the distribution within the cytoplasm and the     
42:59     
basicity okay so then the daughter of cells of ab divided more rapidly than     
43:04     
was of P1 so basically you know you can explain that polarity through these     
43:09     
different activities in the in the uh in the trans crypto leading to the     
43:15     
production of proteins so in in this case AB which is the anterior end is     
43:20     
favored over the posterior and P1 at least in terms of cell division and you     
43:26     
know how that is that asymmetry is being produced     
43:31     
cdc25 and cdk1 are chosen to represent the promoters of the cell cycle so if     
43:36     
we're talking about cell division we're talking about cell cycle well we 1.1     
43:42     
represents the Inhibitors so we want to have the promoters and Inhibitors in this network during the division of ab and     
43:50     
EMS to generary daughter cells so ab and EMS are two founder Cells Two key inell     
43:57     
signaling transducers are involved glp1 receptor represents a notch pathway and     
44:03     
TCF transcription Factor pop one represents the wind signaling pathway so     
44:09     
we have uh gop1 which is the notch pathway pop one which is wi signal imp     
44:15     
pth and so these are involved in growth and differentiation uh if this is you know     
44:22     
of course a cross uh taxa you see this in drop see this in     
44:28     
mammals um so okay key factors that     
44:33     
determine cell differentiation include maternal transcription factors like skn1 for mism and endoderm specification Pi 1     
44:42     
for germline sulfate and pal one for the somatic descendant of the P2 blastam additionally lineage specific     
44:49     
factors include tbx 35 and tbx 37 mism ph4 for the ferx HOH one for muscle N3     
44:59     
for intestinal endoderm Lin 26 for ectoderm and and so all those are sort     
45:04     
of in this network as well the above agents are listed in table S1 and supplemental     
45:10     
materials and in 1A so this is actually our gra our figure one this is the     
45:16     
network so you see that you have these different uh proteins and they just     
45:21     
described what the functions are they're also colorcoded here so we have cell cycle and blue sudden Pathways in yellow     
45:29     
this is wi in uh Notch respectively we have cell polarity par6 lg1 pk1 and then     
45:37     
lineage specification which are the things that specify these different lineages down     
45:43     
here and these connections the red connections are inhibitory the green ones are Activation so like for example     
45:51     
if we take N3 if more N3 is produced and there's more N3 protein it goes up to we     
45:58     
1.1 and it it facilitates the production of we 1.1 uh but if we 1.1 is uh you know if     
46:07     
it's facilitated then cdk1 is inhibited if cdk1 is uh you know if it's     
46:15     
encouraged to rer for it then we 1.1 is inhibited so you have these     
46:20     
relationships in the network and of course when you have these kind of you know like if we think about about a     
46:27     
neural circuit as well it's the same thing we can have these inhibitory relationships we can have these     
46:32     
reciprocal inhibitory relationships we can have relationships where we have a     
46:38     
positive uh feed forward relationship and a sort of an H inhibitory feedback     
46:44     
relationship so we can have a lot of interesting Dynamics come out with these models so that's uh Figure 1 a figure 1B     
46:52     
just simply shows the lington landscape on the left we see the sort of the phase     
46:58     
diagram on the right and we see that there's this potential so potential is     
47:03     
going upward basically where we have cells that are in steady states which are in blue and some transitional state     
47:11     
which is in pink the idea is that on this energy landscape on the right the     
47:17     
steady states are in the wells which like they're in the chanels of the lington landscape and then when it when     
47:24     
they need to transition from one state to another they need to climb this energy barrier to the top Reach This     
47:31     
transitional State and then drop back down to this new steady state which is     
47:36     
another energy BL and so you know you need to have this sort of potential that     
47:42     
rises to go from one state to the transitional State and then a loss of     
47:47     
potential to go back down to the new steady state so there's this metaphor of hill climbing and descent gradient     
47:54     
descent not the gradient something we talk about machine learning it's sort of related it requires some sort of maybe     
48:02     
an energetic input maybe a transcriptional input something to drive it from one point to     
48:09     
another so that those are that's your figure one that kind of describes what they're doing with this     
48:14     
model um so that's those are the interactions of the genes and the the     
48:20     
gene regulatory Network and then they work on building this non-equilibrium stochastic dynamic     
48:27     
so this is where we have these stochastic differential equations that they build so the Dynamics of n agents     
48:35     
within the endogenous Network which are these nodes with their level of expressions or activities denoted as a     
48:41     
set of x uh can be represented by a set of stochastic differential equations so     
48:47     
this is their stochastic differential equation where you have this component that's     
48:53     
Force which is this function here uh the deterministic part is     
49:00     
force um which is a genetically or a generically nonlinear function is the     
49:05     
rate of change for the expression level of the I of the U XI of the i i     
49:13     
agent um and you can also then the other part of that it's additive the other     
49:18     
part of that is multiplicative gusy and white noise so that means that we have this multiplicative noise factor and     
49:25     
this sort of Force Factor the force factor being deterministic the multiplic of gousy and white noise being     
49:31     
stochastic and so you add those two together and you end up with your stochastic differential equation where     
49:37     
you're going through and you're you know it's generating sort of this force in     
49:43     
the stochastic component for each time step uh so the you know there are some     
49:51     
parameters in here where there's noise intensity which stands for the average over the fluctu     
49:57     
ation uh DX denotes the diffusion Matrix and then finally we have a term that's a     
50:03     
direct Delta function and then this uh such a general form of a stochastic differential equation be further     
50:10     
decomposed in the following form so you can also write it like this is a bit longer where we have a friction Matrix     
50:17     
which is corresponds to dissipation a transverse Matrix corresponds to oscillations there's a scalar function     
50:25     
representing a generalized potential and then there's normalized zero white noise     
50:30     
so you can you know take this basic uh stochastic differential equation which     
50:35     
has a deterministic and a stochastic component and you can break this down     
50:40     
into specific types of things going on so there might be oscillations there     
50:46     
might be dissipation there might be these uh this sort of potential that is     
50:51     
you know we we don't describe it in terms of molecular biology but it's in the equation     
50:57     
and then this normalized white noise which is the stochastic component so all of this is basically to say that there     
51:03     
are a lot of ways we can model this Gene Express the outputs of this gene expression Network there are a lot of     
51:09     
like things that we can connect to the function molecular function in the cell     
51:14     
and describe with these stochastic differential equations so you know we're looking for     
51:20     
steady state Solutions we're looking for these transitional States and so a lot     
51:26     
of this goes through the math there's this fixed points of the ordinary differential     
51:32     
equations here nonlinear ordinary differental equations where they kind of Define identify the fixed points of     
51:40     
these and then of course they at the results where they have this endogenous Network     
51:45     
simulation so this is where they build this core endogenous network with     
51:50     
relevant modules and core factors involved in the interactions between them it's a Clos loop system where each     
51:56     
agent is regulated by other agents foring feedback loops with no pure Upstream Downstream relationship that's     
52:02     
what we saw in Figure 1 a which is is network with these kind of uh relationships that don't really form any     
52:09     
sort of linear or sometimes even coherent um set of relationships but     
52:14     
they Pro produce this dynamical out uh so then we can model the network     
52:21     
Dynamics as a set of nonlinear Odes uh we can also look at the Boolean     
52:27     
Dynamics and then this is their sort of Matrix of results so this is the 10     
52:32     
steady States obtained by ordinary differential equations so if I can zoom in on this let's see if I can zoom in     
52:40     
on you can see that there is these these steady States these 10 steady States S1     
52:47     
through S10 going right and then this is the these are the I think the genes here     
52:54     
in the network so these are the Jes is the steady States and then this is the     
53:00     
relationship uh I guess I don't know if it's correlational or what     
53:07     
um let see I think they're just expression levels but the point is is they go from zero to one so you can see     
53:14     
that there's U you know we have this heat map here from zero to one zero is     
53:20     
dark blue one is dark red so you see that there's you know cdk1 for example     
53:26     
has a lot of one values or 0.99 values for the different     
53:31     
um different steady States then you have some you know like uh steady states that     
53:37     
have a lot of inactivity of genes some steady states that have a lot of high activity of genes so it's interesting     
53:43     
some of these steady States but they're you know but they're kind of coming out um and then we have this other one     
53:50     
where we have these Boolean Solutions with the same thing so you have     
53:57     
uh par six you have these same genes here and they have zero to one and then     
54:02     
some of them are colorcoded for some reason well I think they're at uh 0.5 so they're white that's how this works and     
54:09     
so you see a lot of sort of uh these Boolean states where you have sort of a     
54:15     
similar distribution of things done um then you have these uh see the t     
54:22     
is zoom back up so a is the 10 steady States obtained by     
54:29     
ordinary differential equations B are the 33 transition States identified by OD so the Boolean states are the     
54:35     
transition States uh actually no the bottom one is B so that is the one where we have the     
54:42     
33 transitional States we have T1 on the left t33 to the right and so we have     
54:48     
those solutions for this and then this top one uh are the steady just another way     
54:54     
to uh obtain the States uh and then C okay now C is the     
55:01     
top here never mind uh those are the 10p point attractors and eight linear attractors based on Boolean Dynamics so     
55:08     
you have 10p point attractors and eight linear attractors they just different     
55:13     
ways in which they are stable uh red color is in the grids Mark High expression and hean for low expression     
55:21     
okay so then they did that then they had the phenotypes identified with experimental data this is where where     
55:26     
they went and they looked at these steady States and they tried to find these in experimental     
55:32     
data so uh they were able to identify um let's see so they had both     
55:39     
S1 and S4 steady States exhibiting High expression of the posterior factor lgl1     
55:45     
and the germine factor pi1 indicating their correspondence to the P lineage     
55:50     
cells and so in S1 we also see cell cycle promoting factors that are highly     
55:55     
exess rest well the levels of an inhibitory refactors remain low so we     
56:00     
can actually identify this in P1 to P4 cells S1 and then we know that that's     
56:07     
true because they continuously divide with stem cell characteristics during early develop while the S4 which is the     
56:14     
S4 steady state corresponds to Z2 Z3 cells which are uh a different part of     
56:21     
the germine that enter a quiescent state during late embryo embryonic development     
56:26     
in S3 S5 and S8 which are all steady States 35 and 8 the endomesoderm     
56:32     
activator skn1 is highly expressed Additionally the mesodermal factor tbx     
56:38     
35 is highly expressed in state steady state 3 therefore it can be determined     
56:43     
that steady state 3 corresponds to the MS cell the founder cell one of the founder cells that mainly divides to     
56:50     
generate beod the inmal factor N3 is highly expressed in both steady state or     
56:56     
in steady state 8 considering the transition Pathways between steady states only steady state 5 can be     
57:03     
directly converted from S1 representing the P lineage for a single transition     
57:08     
state th S5 is identified as the EMS cell and si8 is identified as the E cell     
57:15     
which develops into the intestine so what they're doing is they're looking at these founder cells they're looking at     
57:20     
these Trends in expression they're looking at the steady states that they found in their mapping steady states to     
57:27     
the founder cells that's basically what they're doing so they have eight steady states that they're looking at they map     
57:33     
them each one to a Founder cell and then they say okay these founder cells are     
57:39     
producing something Downstream so some in Ms for example that founder cell     
57:45     
which is in the uh posterior end of the worm so that starts with this posterior     
57:51     
asymmetry that's going to mainly generate to divide to generate me so we     
57:56     
can actually identify from these steady States the appropriate um founder cell     
58:02     
and then from that we can uh say something about the downstream effects of that that's basically what the logic     
58:09     
here so a lot of simulation a lot of like getting these     
58:14     
steady States mapping them to these founder cells and then validating them with high throughput data so they use     
58:20     
some data sets from um genbank gsse 779443 GSA     
58:27     
83523 so these are transcriptomic data sets where they look at um single cell     
58:34     
transcriptome expression developing from the one cell stage to the 16 and 102 cell stages respectively so these are     
58:41     
just kind of like going through the data sets and mapping this so This is actually a very similar exercise to what     
58:47     
we did in the um differentiation waves paper where we looked at different     
58:52     
candidate cells or candidate genes we didn't look at single cell expression too much but we looked at this across     
58:59     
the embryo and then mapping this to proteins and then saying something about you know the in the in this case it was     
59:07     
like the prevalence of certain you know genes in the developmental stage and     
59:12     
then mapping that to a larger set of statements here they're looking more internally at celegans and they're     
59:18     
trying to really get a handle on some of these changes in the lineage tree and     
59:24     
how you know how how they can interpret that so that's good that's a lot of the     
59:30     
work that they did there they also have this low throughput data here it kind of gives you know similar results to what     
59:37     
they did in the first figure and then they talk about how that maps to a uh to     
59:45     
an epigenetic landscape and so this is quantitative realization of watton's epigenetic     
59:52     
landscape kind of like going through 10 different different cell types so it's the eight founder cells and then two     
59:59     
additional cells where you have um ab and Z I think are the two     
1:00:07     
additional cells uh addition in addition to the eight founder cells and looking at the possible switching Pathways     
1:00:13     
between cell types which allow this these to be connected by these transition States so all these uh eight     
1:00:21     
founder cells are connected through the lineage tree and we want to find these     
1:00:26     
ways in which they are transitioning between them so they all s of start with     
1:00:31     
P0 where the egg is fertilized and then there's this two cell stage where you get these initial asymmetries and then     
1:00:38     
you go to the eight cell stage where you get these founder cells and they all kind of have to switch their program     
1:00:43     
from the one cell down to the eight cell and so the question is what are these     
1:00:48     
switching Pathways what are these transitional States okay so then they find that     
1:00:54     
there's they 24 Developmental Pathways between early embronic development nine     
1:01:00     
of which align with experimentally observe phenomena so that's that's an interesting paper I think uh you know     
1:01:08     
that's that's focusing on the lineage tree and the early lineage tree so I you     
1:01:13     
know that I think that's that's well characterized getting down past that might be a bit of a     
1:01:20     
challenge um I think their work is interesting especially the dynamical modeling that they do     
1:01:32     
dick are you gonna say something all right yeah could you put the reference     
1:01:38     
in the uh chat please yeah     
1:01:49     
backi here this     
1:01:56     
I'll just pull up the link     
1:02:05     
here and this is quantitative resolving selfie early embryogenesis of C     
1:02:12     
on here it     
1:02:18     
is okay yeah so that's that paper I hope I have better news next     
1:02:25     
week yeah like I've handed the whole thing     
1:02:30     
in this is the last little bit like the     
1:02:36     
validation of my model and can it be used     
1:02:41     
for other situations yeah     
1:02:46     
yeah yeah that's good I think that'll be a interesting to see what your committee says     
1:02:52     
and you know I I know you're kind of work working like in a new area like     
1:02:57     
you're working on 10 secs and this is sort of Bio secs and of course there are     
1:03:02     
things that you can't easily translate yeah you always you're running into Road     
1:03:10     
roadblocks every time every time you turn around it seems like yeah like this     
1:03:16     
morning oh what is 10 yeah     
1:03:24     
sure the time is 10 yeah     
1:03:31     
yeah all right well thanks for meeting uh see we're on next     
1:03:38     
week right take care all right bye
