     
Transcript     
0:00     
bottom of of the consol's iterative programming this week     
0:06     
or really we do oh yes it's     
0:12     
uh it iterates and causes my mou my construction to bounce around     
0:22     
and then gives me well off results I think it's adding     
0:29     
them together rather than um just telling you the final um movement it's adding all the     
0:36     
movements together so it's it's a problem I've been trying     
0:42     
to read the manual all else read the     
0:49     
directions the directions are obscure and in tiny print so I've taken to     
0:56     
putting them in a Word document so I can read them     
1:09     
py all right uh welcome to the meeting um I don't know if Pocky wanted to give     
1:15     
an update on her progress if she's ready to do that yet     
1:21     
hi hi uh yeah so I've been working on like generating the graph and stuff I'm using     
1:29     
like the code from from the NDP um repository and like I've edited some     
1:35     
functions like the chain file and NDP file but like I'm facing a lot of Errors     
1:41     
right now so I'm trying to fix that so uh nothing much to update like     
1:48     
I'm just been I've just been fixing errors for the past the weekend     
1:54     
so right I will let you guys know once I have some all right uh um so yeah you're     
2:01     
going to work on that this coming week then     
2:08     
yes are there any other things that you have on your timeline that you need     
2:13     
to address or uh nothing as of now according to the     
2:21     
timeline I would be like I will work on the dra mural networks only and like     
2:27     
I've been writing some content for the blogs as well which I'm planning to post in the coming weeks okay yeah that     
2:38     
sounds good all right that's great thanks for the     
2:46     
update um now I don't know where um mahul is I know he's been busy the last     
2:53     
couple weeks but I hope we'll get an update from him as well we got an update from him like two     
2:59     
weeks ago uh you know it's always good to keep up with uh keep going on the project     
3:06     
because we don't have a lot of time this summer uh to get things accomplished     
3:12     
so uh yeah so I guess I'll start off with um a data set that I found this     
3:20     
past week and is just published recently and what they're doing in this uh paper     
3:27     
and data set is they're trying to characterize uh the L1 worm and so the     
3:33     
L1 worm is right after hatch and so this is when the egg hatches and it unfurls     
3:40     
and it becomes this very small version of the adult and you know then of course the uh     
3:46     
worm goes through several laral stages L1 L2 L3     
3:53     
L4 uh this is cgans although some of the other nematodes do similar things they     
3:58     
have similar laral stages uh so that's you know and then L1     
4:05     
is the site of uh some sort of plastic response so there set of experiments you     
4:12     
can do where you can uh immediately when the or actually just right before hatch     
4:19     
you can put the eggs in uh like 15 Celsius buffer which is like a a stable     
4:28     
liquid and then put the out on a plate and then they'll hatch and they'll stay     
4:33     
in this arrest state so they won't eat any food they'll they'll go undergo     
4:39     
starvation but they can live like that for a while and so this is this has     
4:44     
consequences later in life you can extend the life of the worms and they have this uh plastic response to     
4:52     
starvation um and then there are other some other uh you know plastic responses     
4:57     
later on in laral development but this is focusing on L1 this is just I think     
5:03     
just normal L1 celegans biology that we're going to be talking     
5:09     
about so let's see uh so this is the paper     
5:15     
here a full body transcription factor expression Atlas with completely     
5:21     
resolved cell identities and C elements so the significance of this     
5:26     
paper is that it adds to the data set that we have for the embryo so we went in the last couple     
5:32     
meetings went through the embryo what we have in terms of data which is this three-dimensional positional data on     
5:39     
each cell and they're these developmental cells so we you know have a     
5:45     
developmental lineage and then at the end of last meeting sort of after the meeting if you     
5:53     
go to the recording I went over the interactive celegans lineage tree that's     
5:59     
something something that can be found online and that will allow one to map from those developmental cell lineages     
6:06     
to the adult cells okay now I'd like to talk a little bit about L1 arrest or L1     
6:13     
and properties of an L1 room so we talked about L1 in the main meeting is being the first Lal stage of sea     
6:21     
elegance and actually this applies to all caner htis species NC Brigg is has     
6:28     
an L1 and so this is uh so this is a picture of the adult worm and has eggs     
6:36     
and most of them are herid so they fertilize their own eggs they lay a bunch of eggs and when they're     
6:43     
fertilized and they start developing you get this early embryo which is a cluster of cells the later embryo you start to     
6:50     
get the shape of a worm it's coiled up in the egg and then the egg hatches and     
6:55     
you get this L1 phenotype immediately upon hatch and then of course you get     
7:00     
your stages L2 L3 and L4 and of course L1 is much smaller than L3 or L4 and so     
7:07     
each one of these stages of course we get the potonic morphogenesis we add cells so L1     
7:14     
starts with like 663 cells and you move up to the 959 in the     
7:20     
adult and this is for the hermo so you gain a lot of those cells in L3 and L4     
7:27     
that's the the difference in size uh actually the size is due to growth     
7:32     
there's some cells along the seam that are actually gr so this is this is the we add being added to the wor so um but     
7:41     
L1 of course has some interesting properties as I mentioned in the main meeting you have this phenomena called     
7:47     
L1 arrest so this is where you hatch the eggs in an environment with no food and     
7:54     
you can arrest this phenotype you can arrest the L1 phenotype for a long while     
7:59     
you can starve the eggs in a medium and a buffer and you can put them on a plate and they will     
8:05     
survive without food and they go through the starvation phenotype and it's a form     
8:10     
of plasticity and so they can persist in that one but then it has effects down     
8:16     
the line later in different laral stages and then in the adult uh there's also this dower stage     
8:23     
that begins at L2 and goes to L4 where they grow a very thick cuticle and     
8:28     
they're able to protect themselves from adverse environmental conditions the same with L1 arest that's due to adverse     
8:36     
environmental conditions so you have a lot of plasticity in the uh larvel worm     
8:42     
that you don't necessarily see in the adult and you don't necessarily see in the EM as we mentioned in the main     
8:48     
meeting the uh lineage tree is invariant     
8:53     
in other words unless they're uh genetic mutations or gen we're dealing with genetic mutants the pathway to different     
9:01     
cell types is uh basically deterministic but you do have this uh organism level     
9:08     
plasticity that's very interesting in celan's development in the Marvel stages     
9:15     
so this brings us to this paper by El Ryan B and he's done a lot of work on looking at L1 rest this is to grow or     
9:22     
not to grow nutritional control development during cigan el1 arrest and     
9:28     
he's done these experiments where they look at out want AR rest this is a response to starvation conditions when     
9:35     
eggs are hatched in adverse environment where there's no food or where there's limited food and this is a form of     
9:42     
diapause so in invertebrates and in insects we have different forms of     
9:47     
diapause and so this is a part of life history where growth is arrested and it     
9:53     
has effects in later life but you can there's a trade-off between survival and     
10:00     
growth and so uh he says here in the abstract seans larvae can also be     
10:07     
reversibly arrest okay he says in the abstract     
10:12     
seelan larvae can also reversibly arrest development earlier during the first Lal stage L1 in response to starvation L1     
10:20     
arrest also known as L1 diapause occurs without morphological modification but     
10:25     
is accompanied by increased stress resistance caloric restriction in periodic fasting can extend adult     
10:32     
lifespan so this is a form of theoric restriction and as we know in humans they're mixed effects with respect to     
10:39     
that um but but this is something that you know we want to maybe study and see     
10:45     
elegans to see what the effects are for humans U so we can study nutritional     
10:50     
control and development using the L1 arrest model we can also look at aging     
10:56     
and diseases like diabetes obesity and cancer of course these are all sort of NIH Centric interests so these are     
11:04     
things that we want to know we want to use a system with well worked our genetics and so that's why we're doing     
11:10     
this making this parallel between seans and and humans and so he talks of     
11:19     
this paper kind of goes over a lot of the work that's been done on L1 arrest and you know what goes on at the     
11:25     
molecular level and then what goes on in the phen typical so um the nervous     
11:32     
system epidermis and intestine contribute systematically to regulation of arrest but so autonomous signaling     
11:38     
likely contributes to regulation in the germine and so there are different effects in both the germine and in the     
11:46     
you know in in metabolism in cellular growth and so we know for example that     
11:53     
there a small number of genes affecting starvation survival many of these genes     
11:58     
affect are involved in L1 rest but they also affect the do FL fan so then we     
12:04     
have this paper which is the L1 stage so we have the data for the embryo we have     
12:12     
positions we have the lineage and then we have for this we have cell identities     
12:18     
in the adult form so uh you know at L1 stage you have all of your all of the     
12:25     
cells are differentiated during embryogenesis are present there going to be more cells that are     
12:31     
born that are post embryonic but that's that's not really important for this     
12:37     
what this allows you to do is map L1 to the embryo with the positions of the     
12:42     
cells uh and then also it offers gene expression information so I'll show you     
12:48     
some sample data and maybe what you can get out of those data later so this is     
12:54     
nature Communications uh this is of course in     
13:00     
cgans and so the abstract reads invariant cell lineage and cagans enable     
13:07     
spatio temporal resolution of transcriptional regulatory mechanisms so basically because you have these cell     
13:14     
lineages that always unfold in the same way we can then look to like the     
13:20     
resulting adult cells and their position in the embryo and we can do single cell     
13:26     
Gene sequencing or or labeling and we can get information about um you know     
13:33     
gene expression both in the embryo and then in the larvel stage in the adult so     
13:38     
that's what they're that's where the the power of kind of putting together the seans data sets comes     
13:45     
from um so this this paper talks about a tool and they're developing this tool     
13:51     
called rrap cat which is robust Point matching and peace wise apine based cell     
13:57     
annotation so they they use that to ident automate cell identity assignment so what they do     
14:03     
is they have these microscop the image Stacks they take the all1 larve they put     
14:09     
it under a microscope they take these images and then they're doing this point matching and piecewise saify based cell     
14:16     
annotation so they look at the position of the cell and they can identify it and they put a label on it and so that's     
14:24     
what they're doing here now in in the embryo embryogen     
14:29     
uh embryogenetic data set we have um     
14:34     
what we call cell tracking and we do this here too and the cell tracking is basically where you take microscopy     
14:41     
images and you take the movies and there's a software uh that allows you to sort of     
14:48     
reconstruct the the paths of the cells and it tracks the cells and it can     
14:53     
identify them and so that's a a feature of a lot of these uh you know     
14:59     
full embryogenetic full laral data sets so that's uh you know that's that's     
15:07     
convenient for us because it gives us data that we can actually use and build images of and I'll show you what those look like later so do they have the do     
15:16     
they have the sizes of the cells uh we'll see in a minute yeah is     
15:24     
somewhat well yeah so uh but with they're getting at here is they want to     
15:30     
get like they want to identify the cell position of the cell relative to other cells and then the uh expression values     
15:38     
so they're first you know they're not like segmenting cell boundaries in in     
15:44     
this kind of work they're just getting a down and dirty uh sort of information about the positions of     
15:51     
the cells and and what's going on in them how can you label them so yeah they're not that interested in getting a     
15:58     
really refined size measurement or shape measurement for that matter which would be nice     
16:04     
because I don't think the data set exists uh it may be a little bit beyond the technical scope of what kind of     
16:11     
tools we have to really get a good map we could probably get a a lousy map I     
16:16     
don't know if that's helpful but they are they 100% correct in     
16:21     
identifying themselves um they get pretty correct but I don't know if they're absolutely I have to look into     
16:28     
the method but um they do this you know on a number of specimens     
16:33     
so they're able to get like pretty decent information I'm not sure what     
16:39     
their accuracy is though or how they calculate     
16:44     
that uh so basically they have these image Stacks they have profile reporter     
16:51     
Expressions so this is where they have a fluorescent signal for a certain type of     
16:57     
uh certain Gene and then you have an intensity of that uh fluoresence signal     
17:04     
and that tells you what you know the relative expression of a certain Gene and what they're looking at here are 620     
17:11     
transcription factors so transcription factors are just things that get expressed um by a gene you can have     
17:18     
multiple transcription factors per Gene uh but you know these are important     
17:23     
transcription factors important mrnas that we know you know maybe are well     
17:29     
well characterized um and so that's that's maybe the most interesting stuff     
17:35     
although you know there all sorts of transcription factors and there's this whole project called encode where they     
17:41     
look at uh you know gene expression and function and all that and some of those     
17:47     
data are Inc elegant so that's why I bring that up um transcription Factor     
17:54     
profile based clustering analysis defines 80 cell types distinct from     
17:59     
conventional phenotypic cell types and identifies three General phenotypic     
18:04     
modalities related to these classifications so this is not like the labeles you know you have the sort of     
18:12     
the the waeles that we talked about in the wage tree and then it becomes like the adult cell becomes like a neuron or     
18:19     
a muscle cell and there's a nomenclature there what this is is they're actually identifying the cell based on     
18:26     
transcription Factor profiles so this is is where you get the transcription Factor activity in each cell and then     
18:33     
you try to classify them not based on where what tissue they're in or their     
18:39     
sort of uh lineage but what they're expressing in terms of transcription     
18:44     
factors so this this gives us a couple of ways to classify cells um and so you     
18:51     
know they found a couple of patterns or a couple groups that are you know interesting uh first transcription     
18:58     
factors are broadly downregulated in quient stage hermaphrodite specific     
19:03     
neurons suggesting stage and cell type specific variation in transcripton size     
19:09     
which is interesting because that suggests that you know during transcription there's certain uh thing     
19:16     
you know depending on the cell you're in um uh you know or even like when the     
19:22     
cell is active versus quiescent that there differences in what gets transcribed and it's just you know     
19:28     
something you can identify from a study like this um and you know that the the     
19:34     
issue of function is another question uh but that's that's what they found second     
19:40     
transcription factor expression is more closely associated with morphology than     
19:45     
other phenotypic modalities different pre- and post- differentiation developmental stages so this means that     
19:52     
transcription factor expression is associated with morphology so if you     
19:57     
have like I guess you know a certain type of cell with a certain shape you know and we know kind     
20:05     
of from not not characterizing it with sort of in a numeric way but just     
20:11     
knowing the morphology of the cell because we know the morphologies of the cells we can describe them in terms of     
20:17     
morphology it's just that we don't have the data set where those things are numerically     
20:22     
represented uh but anyways this is something that we can use to sort of     
20:28     
class speci ify cells as well finally embryonic cell lineages can be associated with specific transcription     
20:35     
factor expression patterns and functions that persist throughout postembryonic     
20:40     
life this study presents a comprehensive transcription Factor Atlas for     
20:46     
investigating intra cell type heterogen A so this is they're using single cell     
20:52     
RN a seek which is this next gen method and they're able to take single cells     
20:58     
and are able to do this R seek uh or this is the typical method for this what     
21:04     
they're doing here of course is this a different type of method using um uh like fluorescence microscopy     
21:13     
basically and so uh this basically they're saying here while cell typing is     
21:19     
historically relied on cellular phenotype uh which could be you know uh     
21:24     
functional physiological developmental single cell a seek trans cryptone     
21:30     
capture with unsupervised or marker guided clustering organized cells into biological meaningful groups is     
21:36     
Revolution revolutionizing our definition of perspective of cell types so phenotypes of seelan cells have been     
21:44     
well characterized at single cell resolution including the invariant cell in age throughout     
21:50     
development um however accurate and comprehensive whole body determination of individual cell identities by     
21:57     
transcriptional Pro file is a challenge that we don't have you know we don't have the answer to so people have done     
22:04     
uh single cell RN seek and cgans and there are some thorough     
22:09     
molecular atlases of orogenesis in the adult nervous system     
22:15     
and these are not you know these are kind of in some ways inaccessible because what they tend to give you are     
22:21     
these umap image or visualizations and what I'm going to     
22:26     
show you in a bit is is this sort of table I can extract which is very     
22:32     
different from that kind of an atlas like they and we had this conversation on Friday in our open source meeting and     
22:38     
I know dick was there at the end but I don't know if he caught it but um it's basically where you know a lot of groups     
22:46     
will do a lot of data collection and they'll make the data public and they'll put it in this Atlas that is maybe what     
22:52     
you might call a high level representation and what that means is that I can query a     
22:59     
umap uh visualization for a certain Gene and I don't necessarily know how to     
23:05     
interpret the umap visualization and it doesn't give me numbers but I can see     
23:10     
that it's you know in the mix which is a little bit you know I don't know how     
23:16     
useful that is I mean it depends on what you want to do if you want to be uh you know comforted in the knowledge that     
23:22     
you're going to have like that that Gene is in that cell or what the you know     
23:28     
what how the genes are clustered in a certain Cy then that's great but if you want to get you know if you want to     
23:34     
build alternative models or you want the numbers then you're out of luck um and     
23:39     
you know a lot of groups will you know build these atlases and then maybe they'll build an API but probably not     
23:47     
and it's hard to access the data and we had it back and forth about this Friday     
23:53     
so U this is not something that no one thinks about it's just something that's a unfortunate fact of working with data     
24:01     
and making it public so yeah and then of course that     
24:07     
aside from that problem they point out that in these molecular atlases the number of resolved transcripton     
24:14     
represents approximately half of the total cells in each investigated system and are considered robust largely due to     
24:20     
the transcriptomic homogeneity among symmetric cell pairs so they might take     
24:25     
like one side of the you know like what usually the cells and celegans the adult     
24:31     
cells are symmetric you'll have one on the left and one on the right so um and     
24:37     
then you know those are sort of sister cells they may or may not come from the     
24:42     
same lineage but they they function sort of on the left and on the right you see this in the seans connecting a lot with     
24:51     
neurons but you know they might take like one of those pairs and sampled them and say Well it serves basically the     
24:57     
same function so let's try it and see and so this is this is a problem of     
25:03     
sampling um so they're offering kind of a different way of doing this in this     
25:09     
work we applied a topology preserving matching preserved matching model to automatically annotate all 558 cells in     
25:17     
3D image stacks of L1 larve so they're 558 cells in the L1 or at least you can     
25:24     
annotate 558 they're actually I think 600 and some but that's that's what they're using here     
25:31     
then use lmage analysis to profile reporter expression for 23s of worm     
25:37     
transcription factors so image analysis is being used instead of the RNA seek     
25:42     
method and the RNA seek method of course gives you really good resolution in terms of uh you know different     
25:49     
transcripts but what this does is it ties it more closely to The annotation of cells inside to which is good and     
25:56     
then you can actually get you know you don't maybe get as much specific information about transcription factors     
26:03     
but you can you know create a probe that has a specific sequence and and get that     
26:09     
so they're able to do this in our transcription Factor profiles we found numerous instances of differential     
26:16     
transcription factor expression within cell types defined by cellular     
26:21     
phenotypes and so there's this infotype heterogeneity which is really     
26:26     
interesting you know you you may or may not think that's interesting but you know uh if you think about like the way     
26:34     
we understand cell differentiation of course you know we think well certain     
26:40     
things need to be upregulated or down regulated to have a cell you know be stable in its fate maybe but of course     
26:47     
you know neurons are very diverse muscle cells maybe less diverse but what they're saying is that you have     
26:54     
different things that are up or down regulated depending on different aspects of the that morphology so depending on     
27:00     
function but also maybe depending on you know cell cycle or time of day or whatever and this is not a great Insight     
27:08     
but you know having the numbers to back this up is good     
27:13     
um and so yeah so they talk about their results and you know I want to get into     
27:21     
some of the actually I'll get into some of the images here so this is the worm here this is a microscopy image     
27:28     
this is I guess a side view this is where they can create the point cloud or     
27:34     
the point matching so they have these Point matches from their fluorescent     
27:40     
signal and cell bodies they can match those they can uh update the     
27:46     
transformation function and they they build this map between these two sets of     
27:52     
images and they get I guess they get a pretty good accuracy here annotation     
27:57     
okay here's the question Vick asked annotation annotation accuracy it's     
28:03     
within between maybe like 0.9 and 999 but then there's some variation here     
28:11     
uh depending on how they're doing this so you know it's it's um this is the     
28:17     
bipartite graph here and they're just matching the cells and they're getting this kind of accuracy so not horrible     
28:24     
accuracy but not perfect accuracy um you know     
28:29     
okay so I guess I want to get on with some of the other things I'm I'm interested in with this paper why I'm     
28:34     
interested in it uh one reason is because they have data that they offer     
28:40     
um and so they have a couple of zenodo repositories uh this is their cell     
28:45     
Explorer version one this is their Atlas so they actually have this in a Docker     
28:51     
file and they have it set up so that you can download it and run it um and this     
28:57     
is on the notto this is like you know just kind of an archive this is rap cat version one this     
29:04     
is robust Point matching in piey Zine a cell annotation this is the I guess they're     
29:11     
running this in in mat lab because they're m files but this is where you     
29:16     
know they're running some of these they have sample data this bipartite graph that they're running the RPM the     
29:23     
calculations for that um Transformations so forth so you can do all those things     
29:30     
uh okay then you can transfer from mat lab to basic data types and then you have a training set     
29:37     
where you have training data and you have testing data and everything you     
29:43     
need so that's great uh but one thing you can do of course also is you have     
29:49     
access to some of the training data and some of the testing data and I'm going to show some examples of that in a     
29:55     
little bit here so there they give they have a number of data sets where they're     
30:01     
basically taking the cells they're tracking them in the images and then they have these output files that are     
30:07     
training their model and so what you can do is take those data you can extract     
30:12     
them and and turn them into tab delimited data and once you do that then     
30:18     
you can get it into tabular form which I have here so the reason I'm using a     
30:24     
spreadsheet for this is because it makes it very intuitive and easy to see what kind of data we have so what they have     
30:31     
is they have basically very similar data to the waterers and data set that we have for celegans on     
30:38     
renesis um we have the name of the cell and now in the waters and data set which     
30:45     
is the one I've pointed people to for gug uh you have the developmental cells     
30:52     
and those are going to be like AB d c you know the sublineages and then     
30:57     
the the n and clature these are different these are adult cells and these at some point those developmental     
31:06     
lineages basically there's a differentiation step and then that's it     
31:12     
and you end up with these adult cells so it's actually somewhat easy to take your     
31:17     
Developmental lineages and map them to these adult cells and they're always the same they always have this unfold in the     
31:24     
same way so that map is more or less invariant uh there are some exceptions but that     
31:30     
you know it's not important right now you also have your XYZ coordinates so     
31:35     
when they take images from the stack you know they'll have like a an image in that two-dimensional space of the image     
31:43     
focal plane and then you have a stack is basically a series of focal planes stacked on top one another so in the     
31:50     
focal plane you get your XY coordinate and then in the stack you get your Z coordinate so wherever it is you know     
31:56     
going up or going down so that gives you a number that you can then have that gives you an XYZ     
32:04     
representation of that cell that's the centroid of the cell uh but you know we     
32:09     
don't want to say it's the uh nucleus because the nucleus is pretty large     
32:15     
compared to what they're characterizing here and it's where the fluorescent signal is sort of the center of M for     
32:22     
that so that may or may not be directly in the center of the nucleus so we'll call it a centroid in this sense that     
32:28     
it's that Center of mass or center of gravity for that uh fluorescent signal     
32:34     
and so that's what we have here then we have intensity data so we have maximum     
32:40     
mean and standard deviation of intensity and this is just the fluorescent signal     
32:46     
and then you have this RGN size which is the region size this is inv voxal so     
32:51     
this can be maybe interpreted as well this is basically the fluorescent signal     
32:56     
in voxel so I didn't run the analysis on this it doesn't look like there's a     
33:03     
correlation between mean intensity and our region size but you know I don't     
33:10     
know exactly what the corresponden is but they have this this is not cell size     
33:18     
necessarily but it's just the region that you have in voxel so this is an     
33:23     
potentially interesting information number of voxel potentially interesting information you have this Mass I don't     
33:30     
know what this is uh then there's color so you have red green and blue and these are the fluorescent signals and the     
33:38     
intent or the the sort of the color components of those so we have a lot of     
33:43     
information here and then this is just one piece of data so there are other types of data you can get for this they     
33:50     
have other um I didn't I didn't put it in here I just wanted to show an example     
33:56     
of what these data look like now remember I said this was a larel stage see Elegance this is     
34:03     
L1 so if we go back to this uh I have I actually put this into     
34:10     
a visualization put this into silab and I actually prepared some plots where I you     
34:17     
know graphed out some of these so this is kind of a kind of a messy graph this is supposed to be like a surface graph     
34:24     
but I didn't get it quite right but this just kind of shows some of the geometry of the L1 so this is the anterior end     
34:31     
this is the posterior end and you kind of see how it looks kind of like a a worm um this is another angle of that so     
34:41     
you know I have to perfect this one I have to get like make make a surface and see if I can do     
34:47     
that uh testing it out this is a point Cloud for the L1 marel worm and so again     
34:54     
you get this sort of you know the anterior end here the posterior end here and it doesn't have a lot of structure     
35:00     
kind of almost looks like the embryo but it'd be interesting to map from the embryo to this L1     
35:08     
stage um this is another perspective on the point Cloud so you can take these     
35:13     
Point clouds you can rotate them once you plot them out and see what they look like they're going to be areas that are     
35:19     
more dense with these centroids these fluorescent centroids and some that are     
35:24     
less dense and that relates to the geometry the cells and the distribution of cells and so forth you expect there     
35:33     
to be uh it to be more heterogeneous in the laral and adult     
35:38     
worms simply because they have tissues you know so you know you have a lot of cells and you know you have a lot of     
35:45     
nuclei in muscle you have a lot of nuclei in the connectome and so forth so     
35:51     
you can see that kind of structure already there and then this is another     
35:56     
perspective of the L1 again not like high resolution when you have Point clouds but you know you     
36:04     
can there's more data to be added here I didn't plot every piece of data available but this is interesting so     
36:11     
then you know you can actually color code these points with different     
36:17     
attributes so if you're interested in gene expression for certain genes or INT     
36:22     
in a florescent intensity for whatever reason you can you can get that information out and ploted out and get a     
36:29     
sense spatially of what's going on maybe some also some spatial     
36:34     
statistical like if using spatial statistics to find out things about certain regions of the of the worm like     
36:40     
you know if they're things along the body wall or in the connectone that are interesting that might also help     
36:51     
that so looks like we have uh Alexander hello     
37:00     
welcome um and so yeah so did we have any questions or     
37:07     
comments on     
37:17     
that Bradley yeah and I'm little confused if you know the     
37:25     
transcript on the way they got yeah when you identify the     
37:31     
cell uh well probably not I guess that's the question yeah     
37:38     
well I mean you can get a pretty decent probability of knowing it but um the     
37:44     
thing about the transcriptome yeah it's like there's a lot of heterogeneity and it's like predicting a     
37:49     
cell from the transcriptome is probably going to be really hard to do um I know people want to sell it as     
37:55     
being easy but it's not like because you have like you know things are expressed differentially in different cells of the     
38:03     
same type so if you have a neuron you know there're all sorts of different expression profiles for neuron even for     
38:09     
the same cell over time if you're measuring it during the day or in the in     
38:15     
the evening or you know when the worm's quiescent versus active and then of course if you um you know depending what     
38:23     
part of cell cycle the cell is in uh well not so much of a for the     
38:28     
differentiated cell but yeah um and then you know so there are a lot of     
38:33     
conditions where the transcript can vary it's kind of hard to make that prediction on on what specific cells the     
38:40     
extreme of saying that the transcript is irrelevant to S no well no uh well we'll talk about     
38:48     
another paper where that they're kind of making the case that it's essential the differentiation but U it it's really     
38:55     
it's just kind of hard because it's kind of like you know if I show you uh a bunch of     
39:00     
beads in a jar and I say you know is there a pattern identify a pattern in     
39:06     
this bead the set of beads you know like you can it you can identify a lot of     
39:11     
potential patterns and maybe none of them are relevant maybe one is relevant     
39:17     
there's a lot of conditional expression so like if and remember I said that L1     
39:23     
you have this environmental plasticity that happens where the cell undergoes starvation and it in the in where the     
39:31     
worm undergoes starvation and the worm can persist sort of in a in a rusted state for a long time well that that     
39:38     
involves a lot of gene expression uh it induces a lot of gene expression change so the gene expression     
39:46     
of a starved or an arrested L1 worm is going to be far different from a normal     
39:51     
L1 worm and that's you know and there are probably gradations within them so     
39:57     
it's not that it's relevant it's just extremely hard to know kind of what the conditions are and it's very uh that     
40:05     
that measurement is very susceptible to any set of environmental conditions that the worm is     
40:14     
in okay I'm reading a book that was published last year called how life     
40:21     
works oh yeah like B you know the book I've heard of it yeah oh okay okay I'm     
40:27     
I'm reading it and I'm it's inspiring me to write a book of you little     
40:34     
but I'll see if I how it comes yeah yeah but at any rate it's     
40:41     
uh it seems to me that the book is questioning very much the genric view of     
40:47     
development yeah okay and transcriptomes if anything are supposedly an expression     
40:54     
of the genes uh so uh that's why I raised these     
41:00     
questions whether this is uh going in the wrong direction no I mean it's you know it's     
41:08     
like we have a lot of people have done you know people have built all these     
41:14     
methods for doing gene expression profiling maybe understanding gene     
41:19     
expression and it's a really hard problem and it's like we haven't made a lot of progress on it um but that's to     
41:26     
be expected but still it's like you know again I think the thing to stress there is that     
41:31     
it's just so highly conditional on thing on the environment and something we're     
41:37     
really used to we talk about jeans especially in se elegans you know we think well okay celegans is a pretty     
41:44     
easy lift but it's not necessarily     
41:53     
that okay um yeah it's worth a lot more disc discussion but yeah so Alexander     
42:00     
hello uh I sorry I can't speak right now I'm in a quiet space I'm new to the openworm project and so I thought i'     
42:06     
join a meeting yeah welcome Jesse said welcome glad to have you     
42:13     
here okay so uh the next thing I want to do I want to do another paper and this     
42:18     
paper is on um cell cycle and differentiation     
42:23     
so if we go to uh this paper we see that what they're proposing is     
42:29     
that there's an alternative cell cycle and that coordinates multisol cell     
42:35     
differentiation so this is interesting uh because you know they're proposing that there's this type of cell     
42:42     
differentiation that's corated by the cell cycle and um so they're laying out     
42:49     
this idea this paper now before we get into the paper I did want to talk about cell cycle a little bit because you know     
42:57     
you're not going to necessarily get where this is going unless we get into it a little bit so this is cell cycle um     
43:05     
you have basically a cell will go through different stages as it divides     
43:11     
it will it you know a single cell with a single genome will divide into two cells there     
43:18     
will there will be some sort of ring in the middle that constricts the cell it creates two uh basically two     
43:26     
compartments that become their own cells and then the genome uh doubles in size     
43:32     
and then those chromosomes segregate out so that uh at the end of the cell cycle     
43:38     
the cells there will be two cells that are that have the same complement of things and some you know whatever's in     
43:44     
the cytoplasm sometimes that segregates sometimes it doesn't so for example you can get maternal factors that go through     
43:51     
the cytoplasm you can get sometimes um proteins that go maybe to one daughter     
43:56     
cell another daughter cell but that cell division is really kind of coordinating     
44:01     
all of them and so we know that cell cycle is also involved in oncogenesis so     
44:07     
for different cancers cell cycle can be uh reg you know regulated or involved in     
44:12     
that process and so also you know with cellular uh reprogramming you know that     
44:18     
cell cycle is important um so there's this you know this whole role of cell cycle and there but there are these cell     
44:25     
cycle genes of course that regulate this process so you have these different stages you have g0 G1 s G2 and M and so     
44:35     
a cell cycle will enter G1 it'll go through S it'll go to G2 it'll go to m     
44:42     
and then exit in g0 if it's successful and so you know there are these     
44:48     
different state different phases of cell cycle that involve different things uh but the molecular network is actually     
44:56     
the interesting thing here here because it involves some well-known genes so you     
45:01     
know G1 you have these P factors it regulate cin D and CK C cdk6 and     
45:10     
cdk4 and then as you're entering into s you get uh I guess repression of cin e     
45:16     
and cdk2 you get this repression in S phase of s 1A and     
45:23     
cdk2 in G2 you get this repression of s 1A and cdk K1 and M you get the     
45:29     
repression of cin B and C K1 and then you except the cell cycle so it's it's     
45:35     
basically about building up transcription factors building up gene expression for these different genes and     
45:41     
then repressing them at the right time so that you can enter the next phase of the cell     
45:49     
cycle okay so yeah so and then the timing of this is important because you     
45:54     
don't want to spend too much time in G1 or else you know you have aberant cell cycle or too much time in space because     
46:02     
you have something that's aberant and of course that can lead them to cancer phenotypes but also you know you can     
46:08     
have uh you can modify cell cycle for different cell types so sometimes you     
46:14     
know s might be extended uh and G2 might be shortened in evolution as long as all     
46:19     
this is coordinated you know you can have different cell types um but yeah     
46:25     
it's so people have worked out the gene expression Pathways here and so um in     
46:32     
this paper you know they talk about this alternative cell cycle that coordinates     
46:37     
multi- solated cell differentiation this is actually from uh     
46:43     
I think nature looks like a nature paper um but anyways the abstract reads the     
46:49     
Canon canonical mitotic cell cycle coordinates DNA replication cental     
46:55     
duplication cyto that generate two cells from one um so     
47:00     
this is where they're basically everything gets duplicated and then you get two cells some cells such as     
47:07     
mamalian trophoblast giant cells use cell cycle variants like the endocycle     
47:13     
to bypass mitosis so what we're describing as mitosis in the last image     
47:18     
we know the cell cycle genes we know how that works and that's involved in some forms of you know helping along     
47:25     
differentiation and cancer phenotypes which is a form of differentiation bad form of     
47:31     
differentiation um and then we have these cell cycle variants uh that are     
47:37     
maybe things that are not um mitotic so this is endocycle as an     
47:43     
example uh differentiating multiciliated cells Bound in the mamalian airway so     
47:48     
this is in the long brain ventricles uh and reproductive tract are     
47:54     
postotic but generate hundreds of of centrioles so they don't use mitosis for their uh cell cycle they use a different     
48:01     
pathway they're able to divide and produce these uh cental Each of which     
48:08     
matures into a basil body and nucleates a motile pum so they basically are     
48:14     
functional um in that way several cell cycle Regulators have previously been     
48:19     
implicated in specific steps of multi- solated cell differentiation so this     
48:25     
this is where you know we know the cell cycle for like mitosis we know the genes     
48:31     
that are involved in that we know the timing more or less and then now what they're saying is that you know this     
48:37     
multi- solated cell differentiation has a similar type of structure where there     
48:43     
are these cell cycle Regulators that we you know we can identify here we show the     
48:49     
differentiating multi- solated cells integrate cell cycle Regulators into a new alternative cell cycle which we     
48:57     
refer to as the multic silation cycle so this is in a specific cell type this is     
49:03     
an alternative to mitosis but it operates in a way similar to regular     
49:10     
mitotic cell cycle and this is um you know what's this is a it's an     
49:17     
interesting sort of specialized system uh the multi silation cycle     
49:22     
redeploys many canonical cell cycle Regulators including squin dependent kinases cdks which we saw in the uh     
49:31     
figure and they're cognate cins so they're these squins and cdks that get     
49:36     
coordinated and they get coordinated in time so that's that's where these things kind of come together is that they use     
49:43     
sort of the same mechanisms but they use them in a different     
49:48     
way for example S1 D1 cdk4 and cdk6 which are Regulators of mitotic G1     
49:56     
to s progression so we saw in the image we saw G1 that these things get     
50:01     
suppressed they get like suppressed or shut down and that allows the cell cycle     
50:08     
to move from G1 to S and if they don't get shut down then G1 persists and     
50:13     
something bad happens or something weird happens so there's this reg you have to     
50:19     
regulate the progression to each cell cycle phase um but anyways these factors     
50:25     
here cyle D1 cdk4 and cdk6 are required to initiate multi- solated cell     
50:34     
differentiation the multic silation cycle amplifies some aspects of the canonical cell cycle such as cental     
50:42     
synthesis and blocks others such as DNA replication so you don't get DNA replication in uh this alternative uh uh     
50:52     
cell cycle pathway but you do get um     
50:58     
you do good cental synthesis um E2 F7 a transcriptional     
51:04     
regulator of canonical s G2 progression so this is a different part of cell cycle where you go from s to G2 it's     
51:11     
like the next step is expressed at high levels during the multic silation cycle     
51:17     
okay so again there's this this parallel between the two types of uh cell cycle     
51:24     
in the multic silation cycle e2f seven directly dampens the expression of genes and coding DNA replication machinery and     
51:31     
terminates the S phase like gene expression program so again you have to shut down S phase to enter G2 phase loss     
51:39     
of E2 F7 causes aberant acquisition of DNA synthesis multi- solated cells and     
51:46     
disregulation which is of course where you have something that's supposed to be     
51:52     
maybe regular like if you want UPR we a gene it's not up regulated when it needs     
51:57     
to be and so you have often times people Appo to disregulation as maybe a symptom of     
52:05     
disease or a symptom of some disorder or something going wrong in terms of function that term is used a lot in     
52:11     
medic medical literature but it basically means that it's not regulated in the way we expect it to for this a     
52:19     
specific function um uh so disregulation of multic ciliation cycle progression which     
52:26     
disrupts centreal maturation and ciliogenesis so again you get this E2 F7     
52:33     
you get a loss of it and then when you get a loss of that it affects other things Downstream it it     
52:40     
disregulation cell cycle progression that disrupts then C cental maturation     
52:45     
and ciliogenesis so it's this cascading effect we conclude that multis solated     
52:52     
cells use an alternative cell cycle that orchestrates differentiation instead of controlling proliferation so     
52:59     
this is where in cell cycle you know you have this sort of tradeoff almost     
53:06     
between differentiating and then proliferating so a cell can proliferate you know in     
53:13     
terms of the same cell type and just divide and divide and divide or it can     
53:18     
differentiate into a different cell type and cell cycle is sort of that hinge     
53:23     
that's what they're saying here is that in this system you have that trade-off you have differentiation or     
53:29     
proliferation either proliferation is dampened or it's accelerated and that's you know that     
53:36     
that plays a central role in the whole system soy have a picture of this well let's     
53:44     
see we can go     
53:50     
through picture of of cells uh so this is uh I don't know what     
53:55     
this image is a this is a good one this says cell cycle phase of Basil stem cells oh this     
54:03     
is umap so it's not necessarily helpful this is a u map of different types of cells so again we said that it may or     
54:10     
may not be helpful uh these are the like stains here if you I don't know if you can see it very well uh this these are     
54:17     
the different stages so you have these at different stages of cell cycle you     
54:23     
have these stains where you have cells in there that are of entering different stages of cell     
54:30     
cycle they show this generic graph here where they show well this might be     
54:35     
helpful they just kind of show a sort of a cube a     
54:40     
cubical cell a generic cell but they show the different stages so you go from     
54:46     
precursor to stage one to stage two so you have precursor which is just kind of     
54:51     
like there's nothing cogenetic about it you have this protein build up in stage     
54:57     
one which is S phase or corresponds to S phase so you go from G1 to S phase and     
55:03     
you see this little cyia coming out the top and then stage two you get cental synthesis so you get a protein build up     
55:10     
in the cell and then when you and it enter G2 you get this these     
55:16     
centrioles that sort of form in the cytoplasm then in stage three which is     
55:23     
sort of more mphase disengagement migration and docking so you get these little uh particles in     
55:32     
here stage three and then stage four you get pyogenesis where it migrates up to     
55:37     
the top of the cell and you get these CYO okay stupid question where are these     
55:42     
cells from uh this this particular type of cell is in Mamon I think it's in     
55:51     
the uh esophageal tract and the long they don't really I don't know if they     
55:56     
mention like a specific organism here oh Al on a specific organ well in     
56:03     
the like in the uh respiratory tract in the yeah so     
56:09     
it's like I guess it's a mix of different organs but yeah do those still     
56:15     
you move together or they're independent uh I think they move together I'm not really sure um     
56:23     
yeah let's see if they have any other images a vag vague memory of it having to do     
56:30     
with a small intestin they probably have Sil yeah they're probably silly uh solated cells     
56:36     
there too it just basically helps things move in system like the respiratory     
56:42     
system you have air move in and out in the small intestine you have you know     
56:49     
food that needs to move in and out so you know so forth yeah so     
56:56     
um yeah I don't know if they're they have any really good pictures of the cells in maybe these where you get like     
57:02     
the stains and you get kind of the morphology of the cells here um there     
57:09     
you you don't clearly see these multicilia yeah you     
57:18     
don't okay so these are not independent cells like uh AR or something like that no these     
57:25     
these are just in a Cell population so I guess it's from like a mine model or a mouse model rat I don't know this is     
57:32     
Mouse um and then they show harvesting from that yeah they they     
57:38     
work in in an organ system so they're a lot of them in the in the system     
57:49     
um just looking for some other images I don't think well this shows in cell cycle so we already saw this where you     
57:57     
have these different stages of morphogenesis during in this you know link to cell cycle so a cell cycle     
58:03     
proceeds these are essentially epithelial cells I think they're yeah they may be epithelial cells that's why     
58:09     
they have sort of the cubic cubical shape um and they're just showing one     
58:15     
they're going to be a population of them yeah but that's not during embryogenesis     
58:22     
during adulthood um I'm not sure when this happens     
58:28     
exactly um oh let's see mention it up here     
58:42     
um yeah I don't I don't think they mention exactly what it happens I have to dig into the paper     
58:48     
more for that but you know they may be like s     
58:54     
they may be renewing you may need a bunch of you may need to have CED cells born because they     
59:01     
probably die off and yeah this seems more like a repair process yeah yeah a replacement     
59:14     
process yeah so yeah I don't think there are any other really interesting images here but     
59:20     
just to show that you know that's a that's a paper I wanted to present uh to     
59:26     
get a sense of that kind of system and see I know we've been working on this uh     
59:31     
you know a lot of stuff with differentiation and talking about you know how what the molecular mechanisms     
59:37     
of it are and so this is a very specialized system but I think it hits on some of the role of cell cycle and     
59:45     
what it's doing in differentiation those cells also have if     
59:52     
they're inter epithelium they'll have also Junctions between neighboring cells yeah yeah okay uh so is this an     
1:00:03     
individual cell phenomen or or a whole cluster of cells oh it's going to be a     
1:00:08     
whole cluster yeah there there are a lot of cells the same steps at the same time uh     
1:00:13     
well maybe but maybe not I I would imagine they're somewhat synchronized but that's the thing too is that     
1:00:20     
sometimes cells are uh you know in different cells you're going to have aberant processes some some are going to     
1:00:26     
fail at differentiating properly some are going to die during the process so     
1:00:33     
it's probably you know usually they're synchronized but not often I mean you know one of the things     
1:00:39     
about like working in cell culture is like people synchronize their cell like synchronize cells by division so you     
1:00:47     
don't want cells at different stages of cell division in your culture and you do an experiment but in tissues of course     
1:00:54     
the cells are going to be a different stages cell division and I'm not sure about this system it might be a little bit more coordinated not     
1:01:01     
necessarily Tropa when a sheet of cells     
1:01:06     
differentiate they do so in a wave of mitosis right yeah well that's     
1:01:12     
coordinated right and vers this mentions nothing about the coordination mechanism     
1:01:18     
for yeah     
1:01:24     
yeah all right that was okay that was interesting so do     
1:01:29     
you have any questions or comments about     
1:01:35     
that otherwise I mean it has asked good questions     
1:01:43     
here okay uh any other things we' wouldd like to mention before we     
1:01:51     
go you know Susan was talking about her ventures in     
1:01:57     
multiphysics you might want to have a look at that book how life works yeah     
1:02:02     
okay okay it's uh it's interesting but it's     
1:02:08     
sort of self-contradictory yeah in what     
1:02:14     
way it seems to be an attack on the genomic basis of everything based on     
1:02:19     
genomic oh yeah yeah     
1:02:28     
oh well um my Adventures Continue with the     
1:02:33     
um console and other programs I'm going to take the comp     
1:02:40     
youell 3D uh course this summer it's um end of     
1:02:45     
July early August and so I'll get a look at that side of     
1:02:53     
things yeah that's sounds good when does that start um July 28th I think with uh     
1:03:03     
python course just a overview and then the 29th that week on     
1:03:10     
the 29th okay anyway so yeah I'm hoping     
1:03:17     
to well learn something from it maybe that I can add to my     
1:03:24     
thesis yeah at least I get a look at the     
1:03:31     
competition I put that details on the book in chat okay that sounds great     
1:03:38     
yeah right uh well I think that's it for today um we we have another meeting next     
1:03:46     
week um talk to you then okay bye you bye bye bye
