Transcript:        
0:00     
hello hi Hi how are you okay I guess I don't     
0:10     
know not busy anyway oh okay well I am because I have to catch up to things     
0:15     
around here but     
0:20     
yeah how how is everyone uh looks like we got a couple people mahul     
0:28     
welcome haven't seen you in why hi Bry hi everyone     
0:34     
Hi hi I hope everyone's doing     
0:40     
that yeah just got my thesis published it's actually online you can download it for     
0:48     
free for some reason yeah popped up my feeds and I     
0:56     
went is it for free and yeah it downloads right     
1:03     
away oh that's very nice yes oh yeah so Bradley actually     
1:09     
soam and I have been talking for some time on lindin uh we were discussing although I couldn't contribute much to     
1:16     
the discussion so was asking a few questions and I think yes last week he had a meeting also so he briefed a     
1:22     
little bit what was there in the meeting okay yeah so I thought like I'll join I     
1:29     
also told to him and also I discussed with you about the paper so yeah     
1:34     
yeah I yeah yeah okay that sounds good um so     
1:43     
you were talking on LinkedIn you said or okay what were you talking like yeah what were you talking     
1:50     
about so uh mainly it was a discussion like how uh soam can contribute so like     
1:58     
uh we I have been poting him towards the hypergraph part and I shared I shared him the paper     
2:05     
your paper basically which like the original hypergraph C Elance paper and     
2:11     
so has is trying to actually find like the downstream task and that is what I     
2:16     
was struggling also so we were discussing little bit on that so soam     
2:22     
actually brought a good point uh but the issue was there was there's not much uh     
2:27     
papers he can find so one of the application is a of course cell tracking okay and another thing which soam     
2:34     
mentioned was let me just check it soam you can just unmute and tell yeah um it was something dick     
2:42     
mentioned in the last meet about error accumulations in cell so yeah so if we could model that     
2:48     
using hypergraphs I couldn't find much literature on that topic yeah I don't yeah I think that's     
2:55     
something that we kind of were talking about in the meeting I don't think there's going to be a lot of references     
3:01     
on it but yeah that would be good uh any other points or those just the     
3:10     
main two yeah that is that is it about discussions and we thought we'll discuss     
3:17     
more once we hop on the all right that sounds good yeah um yeah so yeah cell     
3:25     
tracking of course is a part of the data sets that we get so we we're able to get     
3:30     
data sets for different biological systems you know uh we might take some     
3:37     
microscopy of a of an embryo or something and then we can track the     
3:43     
cells basically using Machine Vision it can pick figure out where the location     
3:48     
of the cell It's usually the cell centroid or some sort of feature in the cell or cells and you can use that to     
3:56     
track things over time so you know we could do something thing with tracking over time depending on the data set the     
4:03     
data set we usually use is uh has uh it's uses like n it didn't really use     
4:11     
nuclear cell tracking it's sort of figuring out the centroid of the cell and then that's what's gets reported but     
4:18     
you can do this uh you know you can do this sort of on board with a cell     
4:23     
tracking package or you can do this after the fact as well with like a machine learning algorithm it's not I     
4:30     
mean it's hard but it's like you know you can figure it out um and so that's kind of where that is and you know     
4:38     
that's that's interesting for a number of reasons we can track sort of the uh     
4:43     
centroid of the cell we can track maybe things like the nucleus or things like the membrane or other things in the cell     
4:51     
that we might be interested in and of course since it's not just one cell it's a population of cells then you know we     
4:58     
have all those data and you know tracking especially with cell division where you get you have one     
5:05     
cell and then suddenly it's two cells and you track that over time because you have time lapse     
5:12     
microscopy you have this Divergence of one cell into two or two cells into four     
5:18     
and you know that's something that you can put together we've done that over several years trying to work out     
5:25     
different methods for doing it so actually if you go back to the DVA learn uh     
5:31     
repository you know the dvo Learned software actually does a lot of stuff     
5:36     
interesting stuff with that working with cell segmentation and and cell tracking     
5:42     
and trying to figure out how to put those things together yeah segmentation     
5:50     
yeah then you know other yeah I'll go ahead no go yeah so I mean like with     
5:59     
error accumulations and cells the thing we were talking about last week I mean that's something that we can model it     
6:05     
wouldn't necessarily come from the data per se but it would be like this you know we we know kind of what the cells     
6:13     
you know the cells were dividing where they're dividing from things like that and you just you know you could build a     
6:19     
computational model to you know pull out maybe a model of     
6:25     
error accumulation for example um and you know that's something we could work     
6:30     
on uh but that could be incorporated into a a hypergraph model as well I     
6:38     
think and it's just a matter of knowing the L in SE elegans we know the lineage tree where we know if a cell where a     
6:45     
cell is in the embryo if it divides we know basically what the N we know what     
6:52     
the name of the cell is we know with it properties other embryos were not as     
6:58     
lucky because the cells just kind of differentiate um you know in in situ     
7:05     
they don't have a uh they don't have a strict lineage relationship so see     
7:12     
that's why we work in celegans a lot of times but this is something we can do as well um I would think yoube of a few     
7:20     
other things we might incorporate into the model um and you know I don't know     
7:25     
maybe that's two things is enough but um I think the cell tracking     
7:30     
is pretty much you know a matter of going back to the data set and you know     
7:36     
because we we kind of train the model on a data set and it's just kind of like we can do more with that we can     
7:44     
train it on other types of data we can get other types of things in there so     
7:50     
that's that's something we can work out I think it's more about playing with the data itself with error accumulation I     
7:57     
think that's more of like we'd have to build a separate model of that and use     
8:02     
that to uh you know characterize each point in time each set of cells that     
8:09     
divide and things like that yeah yeah um yeah and I think     
8:16     
that's probably and I I assume this is like a summer timeline a gck timeline so     
8:23     
probably that would be enough for that for like a summer project uh there may     
8:28     
be some other things lead out of that I'm not sure what those are yeah yeah yeah that's very helpful     
8:36     
unad definitely helpful for me and hopefully for also     
8:43     
yeah other thing which I wanted to like discuss like was about paper so I really     
8:48     
love the paper honestly like it was like a great uh read uh but like one thing I     
8:53     
felt was uh we haven't put a lot of empirical results I mean we did not had     
8:59     
much on also but it will be good that if we can put a lot of metrics right now we     
9:05     
are on the stationary part so put a lot of hypergraph metrics in it but also extend this like extend this paper only     
9:13     
not like create a new draft maybe also add dynamicity since we already have     
9:18     
cigance data uh putting the analysis of the dynamicity uh like how it is evolving     
9:25     
over time I think that will be pretty good and it will make a more solid paper then already is so yeah what I was     
9:33     
thinking is like uh this January and like by the end of February I'll try to     
9:38     
complete as much analysis as I can like on the hyperr and like show up in the     
9:43     
meetings discuss what how should I like which direction should I proceed taking     
9:49     
feedbacks and all that is what I plan to do okay that sounds good     
9:54     
yeah yeah I think that's true it's you know we could use a lot of uh we could     
9:59     
use a lot more information about how they unfold over time so F says uh there     
10:08     
is a literature on bilateral fluctuations during development so yeah if you have like an     
10:14     
embryo when you have like you know you you establish poles in the embryo and     
10:20     
there fluctuations in what happens in those two poles um yeah there's so let's see I'll     
10:27     
write that down and look into do it after the meeting that might be     
10:32     
something to introduce as well there they're all sorts of like sort of meta     
10:38     
properties to embryos that we could incorporate into this um and so just     
10:44     
basically once you could sell division once you get polarity established there're going to be a lot of things we can look at we consider build into the     
10:52     
model Etc yeah so this is a distinct from     
10:59     
normal left radius symmetry like placement of heart yeah that so like in     
11:06     
um in once you start to get like some of these left     
11:11     
right uh axes established then you get things that develop asymmetrically like the heart develops on one side of the     
11:19     
body or another and you know you don't see that as much in celegans but that's something that you know there is that     
11:27     
asymmetry um that's that's something it's I guess kind of an open question in     
11:33     
developmental biology as I recall um yeah so that's something we could     
11:39     
incorporate too yeah so let's go over the paper I     
11:44     
I've been working on revising it uh so basically this is the paper they're     
11:50     
talking about hypergraphs demonstrated an asmosis during dver     
11:56     
integration and I put my ho on here already so you know Soom wants to be on     
12:02     
it as well that's fine you know we're just kind of uh do I'm I'm just trying     
12:07     
to prepare a new version so that we can incorporate some of what mul did last     
12:12     
summer so um this is the abstract basically it's building these hyper     
12:18     
graphs so we have an earlier paper on embryo networks which is where we take     
12:25     
the cell tracking data that I talked about earlier and use those cells as the     
12:30     
nodes in the network and their edges which are like things like distance or they could be signaling or other types     
12:38     
of uh of things that link together nodes basically we have these hypergraphs     
12:43     
which are taking these nodes putting them into larger categories using what     
12:49     
we call hyper nodes to characterize groups of cells or groups of nodes and     
12:54     
then having these hyper edges which are collections of edges and you know we can do different things     
13:01     
with this kind of model so it's a higher level model Than A Memory and so this     
13:07     
paper kind of goes through a lot of what that is using this analog of celegans to     
13:13     
show kind of how uh you know as cells divide and differentiate you get these     
13:18     
different categories of cells and that those categories sort of break out over     
13:24     
development you get exchange of cells between them and it's kind of of establishes     
13:30     
this uh you can have like a directed graph or you can have an undirected     
13:35     
graph um or like a guess a semi-directed graph so that's we do there we also do     
13:43     
some something with a CYO bacterial species nosto pun uh     
13:48     
punctiform which is where we have this chain of cells that uh grows as it as     
13:55     
the cells divide and that's a different representation of the of that hypergraph     
14:01     
method and then we have this hyper degraph model which is Incorporated in the last part which is a computational     
14:08     
approach so there are two theoretical or there's a theoretical approach there are two examples and then there's a     
14:15     
computational approach which is the thing that um mahul worked on last     
14:22     
summer so this is setting up what we have here and we go to the first graph     
14:29     
this is a category a category theoretic representation of exchange between the     
14:36     
categories um and then we have this this is the one I want to show this is the     
14:42     
directed uh hypergraph where you have you start with a single cell you get     
14:48     
these uh you get this sort of undifferentiated network you start to     
14:53     
get things like a germline which is another hyper node and then you get this     
14:58     
uh embryo hyper node this connecto hyper node and you can see as the cells     
15:03     
increas in in their number through cell division and differentiation you get     
15:09     
these different hyper nodes that represent different classes of cell and     
15:15     
so you have embryo cells you have connectone cells germline cells in some     
15:21     
cases you could exchange between the two categories between two hyper nodes that     
15:27     
involve like changing uh cells changing their fate cells     
15:32     
changing their category and but it for the most part you get a continuation of the categories     
15:39     
as they grow so that's what this is all about and it's just like kind of a theoretical sketch of what that should     
15:45     
look like it's not using it's kind of tied Loosely to celan's development but     
15:52     
it's um we could easily build this with data that's available for SE     
15:59     
this is a spatial hyper graph so this is I guess you could call it a temporal hypergraph there's no spatial component     
16:06     
this is a spatial component where you have anterior posterior organization and     
16:12     
left right organization of these hyper nodes the hypernodes then are connected over time to new hypernodes so basically     
16:20     
this is 128 cell example where you have 128 cells in these four hyper nodes and     
16:28     
then and later you get these new hyper nodes that break apart you know so if     
16:35     
you go from six um you end up with uh a doubling     
16:41     
event which is 12 those are distributed in the following way you get two cells in one hyper node here eight cells in     
16:48     
another hyper node and two cells that contribute to this other hyper node which takes some another hyperon node     
16:54     
and combines things so you can combine cells into new categories you have doubling events which increase     
17:01     
the number of cells and those get distributed into new hyper nodes and so     
17:07     
that's what this is this is a 256 cell um example where you get more     
17:14     
categories and of different types you have the same three type actually you get four types in this but um you get uh     
17:23     
greater number of hypernotes again this is a theoretical example it doesn't it isn't really tied to the data so much     
17:30     
but it does give it's it's Loosely tied to the data just to give an idea of what's going on here okay and then this     
17:37     
is the example from nosto point for uh point for me where you get the uh this     
17:43     
chain of cells I think we talked about this in a meeting and again of worked up this example so we have this uh     
17:51     
restricted heteros cell differentiation along this chain of cells and the idea     
17:56     
is this chain of cells grows in length L and you can incorporate these cells into     
18:02     
hyper nodes and as you uh continue on in time this chain grows larger and you get     
18:08     
these hyper nodes that increase in size but also you can get other types of     
18:15     
hypernodes that Branch off I only show here these three examples so it's restricted heteros cell     
18:22     
differentiation over time and applying those this kind of method to this type of system it's it's a very it's a     
18:29     
simpler system than say something like celegans and in turn if you had     
18:35     
something like a mammal or something like that you would have an even more     
18:41     
complex uh hyper graph representation than this this is this is relatively     
18:46     
simple just to give an idea of what we're we're expecting okay so then I have some     
18:52     
methods in here uh actually this is taken from some data that is     
18:58     
incorporated in the paper these are some secondary data sets where you know these     
19:04     
are kind of actually these are pseudo data where I kind of analyze these     
19:09     
theoretical examples um and then we have uh actually     
19:16     
some other data in here where we actually uh you know it     
19:21     
provides some analysis I can't remember where those graphs are right now but anyways there there's a some sort of     
19:28     
pseudo dat showing what the data look like there's some actual data analysis     
19:33     
showing what category should look like in say like human cells and things like     
19:39     
that and then we have these graphs here these are graphs from Mah's work from     
19:44     
last summer so we actually Implement a hyper graph uh using an algorithm     
19:51     
clustering algorithm and so this is using the actual cion data set that     
19:57     
shows kind of how this was done this clustering algorithm where there's this     
20:02     
uh you know these clusters are defined for different cell types and we have     
20:08     
these graphs here um and so you know we have the regular lineage hyper graph     
20:15     
which is this first example a and then we have the spatial hyper graphs for B and C so the spatial     
20:23     
hypergraphs are using this clustering as the connectivity Criterion whereas a     
20:28     
lineage hyper graph uses common parents is the connectivity Criterion and so     
20:35     
this is what we're you know kind of trying to sort of replicate what we did in the theoretical examples but using an     
20:43     
algorithm on real data and so it's it's harder to interpret what these are actually and     
20:50     
that's something you know that I don't know how to approach in the paper and maybe mahul can help me with that like     
20:57     
how do you inter interpret this um but that's you know we get the these     
21:02     
different types of categories we get these different types of uh connectivity Criterion and we can build then our     
21:09     
graphs from that or you know we can plot them out in in more of a graphical form     
21:15     
so that's that's you know what we have now uh just kind of talking about future     
21:22     
directions and that's the paper so you know I think     
21:27     
we've got a nice uh paper I think yeah we need to have you know maybe some uh I     
21:35     
think we could use some analysis you know in different areas um I'm still working on it so I     
21:43     
was showing you work in progress there but that's um that we do have a preprint     
21:49     
of this on the archive but you know we're going to Pro we're going to be updating that soon and then probably     
21:56     
again so um any question about     
22:03     
that uh yeah badly so like some thoughts maybe like the point you mentioned     
22:09     
interpreting the hyper graph that in more detail I think what we can do is     
22:14     
focusing on particular hyper uh and removing the other uh kind of cells so     
22:21     
in different snapshots covering having the coverage of different kind of hypn     
22:26     
so that can provide much clearer uh interpretation or     
22:31     
understanding of these cells maybe yeah I think     
22:41     
so yeah and but yeah go go ahead yeah I had like some other also     
22:48     
some other questions also but I think I get the draft like like one more read uh     
22:54     
specifically about the diagrams like which had restricted hydr hety differenciation but to be more to have     
23:01     
more precise questions I'll go through that okay yeah I mean just just collect     
23:07     
your questions I think that's more useful to of get a list but yeah I think that's good uh yeah I think that's good     
23:14     
I think that'll come out uh you know I mean basically the structure is we have     
23:21     
sort of the theoretical aspect of it we have some kind of thinking about what     
23:27     
the data should look like and then we have the computational work at the end     
23:33     
and I want to kind of do that sort of organization organizational scheme     
23:39     
because it's kind of establishing the problem thinking about it in terms of     
23:44     
actual biology you know or at least kind of thinking about it in terms of the way we might see it in biology and then you     
23:52     
know what we might expect to be able to use it for some sort of spectral     
23:57     
analysis which is usually what hypergraphs can give you they give you this sort of spectrum of information     
24:05     
for uh different hyper nodes you know what are the constituents of the hypernodes and being able to plot that     
24:12     
out and then the the algorithmic work which is of course implementing it on     
24:17     
real data sets and you know having it in these kind of you know different using     
24:23     
different criteria to establish uh connectivity because you know connectiv     
24:29     
it's almost talked about is this theoretical thing you know it's like we know things are connected you know we     
24:36     
want things to be connected how do we establish that in addition to you know     
24:41     
how do we gain you know how do we collect this from data where the data     
24:47     
don't lend themselves to that type of organization necessarily we have to impose it through some methods so that's     
24:54     
that's um I think that's the kind of organization have and then of course you     
25:00     
know the I think the algorithmic part might be you know might benefit from some of these you know visualizations     
25:06     
where you're able to see kind of what what the algorithm is doing you know in     
25:12     
terms of the cells having those big visualizations but also the smaller you     
25:18     
know maybe taking one hyper node and looking at what's in it or you know     
25:24     
maybe looking at how you could build kind of you know actually something that's connected I don't know how what     
25:31     
kind of visualizations we could do but we'll have to think about     
25:37     
that because I took the images from like what you were doing last summer so I     
25:42     
mean I don't know but yeah even like those images were     
25:48     
like just came out of the blue like I wasn't thinking about producing those kind of images and yeah yeah probably     
25:56     
discover on the way well I think it's good to have those like in terms of like     
26:01     
understanding the basic types of connectivity but then you know having like a little bit more     
26:08     
detailed uh you know how do we get to you know how how are they interpretable     
26:13     
I guess in terms of biology is is a thing so     
26:18     
yeah so that's that's ongoing work     
26:23     
so dick uh put in the chat some things on um     
26:29     
this is on papers on fluctuating asymmetry so I don't know how many papers are here     
26:35     
out 10 12 papers actually and so there are a bunch of papers on this     
26:41     
topic um here's this paper by ucha at all deciphering the origin of     
26:48     
Developmental stability role of Interest cellular expression variability and evolutionary     
26:54     
conservation there's tesu at all Symmetry and fluctu ation of cell movements in neural     
27:00     
crust deriv facial mezzanine uh there's this other one a     
27:07     
high temperature and oxygen supplementation can mitigate the effects of hypoxia and developmental stability     
27:14     
of bilateral traits or incubation boiler breeder eggs uh there's one on zebra     
27:21     
fish uh there's one on Tillis orus the consequences of extremely low     
27:29     
frequency magnetic fields and the influence of ederse environmental conditions on the roach which is Rus     
27:37     
orus so there are a lot of different papers on that topic um basically     
27:43     
looking at how these asymmetries get established and developed and you know using these kind     
27:49     
of representations are useful for uncovering the origins of that     
27:57     
asymmetry so thank you dick for those references um we'll see you know if that     
28:04     
plays I know we had uh we might put that into the paper as well a section on     
28:11     
developmental asym or fluctuating asymmetries and things like that all     
28:16     
right so let's move on I have a paper I wanted to go over I think this was shared in the openworm slack so this is     
28:24     
um a paper on celan modeling so this is from nature computational     
28:31     
science this is by a Chinese group uh the title of this paper is an integrative datadriven model simulating     
28:39     
celan's brain body and environment interactions so this is uh I don't know     
28:46     
if I don't think we've covered this work I think we covered it may be actually something that we covered when it was a     
28:54     
pre-print and I'm not really sure um it's sounds familiar but I'm not well     
29:01     
well let's go over the paper to see what you know what they're doing so uh they're interested in not     
29:08     
only modeling the Worm but modeling the uh body and environmental     
29:14     
interactions so again they they state that the behavior of an organism is influenced by     
29:20     
the complex interplay between its brain body and environment and if you look at celegans     
29:26     
and its natural habitat you know it usually exists in this three-dimensional     
29:32     
habitat in soil so it moves around soil     
29:37     
up and down different Columns of soil across soil I mean you know we have     
29:42     
we're talking about an organism that's a few millimeters large and it's     
29:49     
navigating I don't know what the range for a single celegans is but where you     
29:55     
can usually find celegans as in an environment where there's rotting fruit or some other type     
30:02     
of uh you know where you get things that fall on the ground and start to rot in     
30:07     
the soil and they like that kind of environment because it provides them with a lot of bacterial     
30:13     
nutrients and so I've actually harvested Sea elegans from like a compost pile     
30:20     
which is interesting um and so you can do that and they exist in this     
30:26     
three-dimensional uh so Matrix so you know a lot of the things that the cybernetic group in open worm has done     
30:34     
is model this biophysical environment they model the worm moving through the     
30:39     
environment and this environment is usually some sort of material you know so they model it as like a material     
30:46     
Matrix where the worm moves through it so you know the Matrix might be of     
30:52     
different varying stiffness or varying um you know dens City and the worm moves     
30:59     
through that and you can actually model that process so the SE elegan environment is     
31:07     
going to be you know in a culture condition you know you might have it on a     
31:12     
two-dimensional uh plate where if there's a gel and it's moving around the gel and that's not really it's normal     
31:19     
mode of interaction It's usually the soil environment so it can vary and you     
31:25     
can have um you know but but to model this environment is something that     
31:32     
typically we often don't think about when we do a lot of computational studies on uh organisms but     
31:39     
anyways um existing data driven models focus on either the brain or the body     
31:44     
environment here we present ba iorm or borm an integrative uh data driven model     
31:52     
of celegans which consists of two submodels so we have two submodels we     
31:57     
have the brain model and the body environment model so we have these two sort of partitioned models of     
32:06     
cgans the brain model was built by a m multicompartment model uh approach with     
32:12     
realistic morphology connectome and neurop population Dynamics based on experimental data so this is kind of     
32:20     
what you know we usually do in we model uh seans we have these different     
32:25     
compartments we have the morphology we have the cells that you know this is the sort of thing that we're doing with our     
32:32     
hypergraphs we have the connectone which are the neural cells how they're connected functionally and how they     
32:39     
function and then neurop population Dynamics which are the Dynamics of those     
32:45     
cells in a connectone so what are the cells doing to produce     
32:51     
behavior and of course we can measure uh the     
32:56     
uh we can meure measure the activity of cells in the connecto and get these     
33:01     
neurop population DS okay so that's that's the brain model and we have these different compartments     
33:08     
in the brain model that contribute to the overall this brain model and then of     
33:14     
course we also have a body environment model which uses a lifelike body and a three-dimensional physical environment     
33:21     
so again we have this three-dimensional physical environment where it's some sort of material where their     
33:28     
interactions with not just on a flat surface but in this three-dimensional volume they're interacting with other     
33:35     
worms they're interacting with soil they're interacting with other things in the environment food they're orienting     
33:41     
themselves towards chemical signals and so forth uh through the closed loop     
33:47     
interaction between the two submodels remember we have the the sort of the brain and the body environment     
33:55     
submodels baa iorm reproduced realistic zigzag movement towards attractors     
34:01     
observed in Sea elegans so the you know sea elegans will make this I guess     
34:07     
zigzag movement towards different stimuli and that's what they're trying to get at so if you have like this food     
34:14     
source it's usually some sort of chemical source that they're trying to     
34:20     
get to they'll they have these uh stereotypical movements they'll use to     
34:27     
get to the source of the food and that goes through this environment so they're zigzagging through this volume of soil     
34:35     
so there's a a specific resistance to the soil it's a colloid so you know it's     
34:41     
it's moving in in a certain way it also has these stereotypical movements it's     
34:47     
executing which you can observe in culture and you want to reproduce that     
34:53     
you also want to reproduce sort of this environment where you can train it on     
34:59     
different uh you know chemical stimuli and reproduce these chemical gradients     
35:05     
that exist in nature so they're able to set this model up reproduce a lot of the     
35:12     
natural behaviors we expect and then leveraging this model we investigate the impact of neural system structure on     
35:19     
both neural activities and behaviors consequently VA AI worm can enhance our     
35:25     
understanding of how the brain controls the body to interact with its surrounding     
35:30     
environment so this is what they're trying to get at so this is where they talk about the different things that     
35:38     
exist for this they say currently there are specific data driven models that simulate either the brain such as the     
35:44     
mouse primary visual cortex the mouse strum the ratata sensory cortex or     
35:50     
exclusively the body and environment to reproduce animal behaviors such as neuromechanical models for dropa and     
35:58     
rodents then they talk about openworm where openworm developed data     
36:03     
driven models for both the nervous system this is the c302 model uh and the     
36:10     
body cybernetic we talked about cybernetic of C elegans which is a pioneering initiative in computational     
36:17     
biology however the integration of these models is open loop manying that they lack feedback from the environment so     
36:24     
again this is you know uh every time you build a biological     
36:29     
model often times you make decisions on how you represent things and sometimes     
36:34     
you you know it's easier not to include the environment explicitly and you know especially     
36:41     
providing feedback to the actual worm so in cybernetic it doesn't provide feedback to the worm it just kind of     
36:48     
models the worm as this uh agent that's moving through a     
36:53     
biophysically plausible environment and it's just kind of fix figuring out how     
36:59     
this biophysics work in this case they're actually including feedback the     
37:04     
same thing with a c302 you have this connectome you have the cells you have connections you have     
37:10     
the neural Dynamics and you have outputs but often times those are influenced by     
37:16     
feedback from the environment and so that's what they're they're trying to do     
37:21     
here so they they say that basically their work fills a hole in the litat OR     
37:28     
which is that no data driven model currently meets all requirements of an integrative data driven model although     
37:34     
such integration is urgently needed to progress our understanding of moral control mechanisms so yeah there's been a lot of     
37:41     
work on some of these so let's see what they have reference three reference four     
37:47     
five reference 6 S8 9 10 and so let's go     
37:53     
to those references 3 through 10     
38:01     
the     
38:08     
bottom all right here we go so they're talking about these papers 3 through     
38:14     
10 uh we have we start with uh this paper s     
38:21     
uh systematic integration of structural and functional data into multiscale models of mouse Prim visual cortex this     
38:29     
is a neuron from 2020 the second paper the micro circuits of strium and silico this is from pnas     
38:38     
in 2020 you notice that these papers aren't that old where a lot of The Works been     
38:44     
uh getting done this paper by macrum at all reconstruction and simulation of     
38:49     
neocortical micr circuitry that's um from cell uh this     
38:56     
paper neural Mech fly a neuromechanical model of AD adult roph melanogaster this     
39:03     
is a nature method so we're going from Mouse to gropa so we have this variety     
39:10     
of organisms and usually you'll use these organisms because they're model organisms we have data on them and we've     
39:18     
already kind of characterized their the system of Interest quite well um there's     
39:25     
this other paper deep neuro ology of a virtual rodent this is a conference paper then we have the openworm papers     
39:33     
we have this openworm paper that's sort of the main paper for that was published     
39:40     
in 2018 we cover all aspects of the openworm     
39:45     
foundations project portfolio we have this paper by por     
39:51     
gleon at all this is on c302 uh and then uh Andre panov and     
39:59     
Sergey kulan and those are the guys who develop cybernetic a software complex     
40:05     
based on the PCI algorithm init simulation problems and     
40:10     
biomechanics so there's this uh you know this this representative sampling of these     
40:17     
different whole organism simulations it's not just celegans but they're adding into this     
40:24     
newu I guess     
40:34     
okay so let's go to this this is an overview of their model uh they have a neural network which this is this this     
40:42     
uh this sort of body uh I guess it's just the brain subm     
40:49     
module this is where you have a neural network with neurons which are multicompartment models with real     
40:56     
electrophysiological features you have connections with your detailed synapses and GAP Junctions on youres and then     
41:03     
optimization so you basically have the neurons the connections and the     
41:08     
optimization and that's their neural model and then they have the body     
41:14     
environment model which is this uh down here where they model the body with     
41:19     
tetrahedrons and they add muscles to those tetrahedrons so they don't     
41:25     
actually model it at the cell level per se they're using these tetrahedrons to build a body and then they're adding     
41:32     
muscles to the body so this is an interesting way to approach this because you're not     
41:38     
really uh worried about the somatic cells so much you're interested in the     
41:44     
Integrity of the worm and the shape of the morphology so they use these tetrahedrons and then add muscles into     
41:52     
them um then you have the environment which are lighting effects visualization     
41:57     
tracking quantification uh this is not modeling it in a Swale Matrix this is modeling     
42:04     
it in this sort of environment that they've created for the you know so that     
42:10     
they have this uh these sources of feedback and then realtime simulation which is where     
42:17     
they have things in the environment that they were M to interact with and gain feedback from and hopefully you know it     
42:26     
incorporates it into their behavior so this is basically sensory AC or you have     
42:31     
muscle activation in the worm you get the worm interacting with its environment these blue dots which are     
42:38     
the sensory sources you get the sensory activation and it contributes back to     
42:43     
the muscle Activation so in other words the worm will move towards sources of     
42:49     
sensory stimulation and it will move its muscles as a result so it's this Clos Loop     
42:55     
feedback mechanism so that's basically what they're trying to     
43:01     
achieve uh this is a biophysically detailed neural network model of celegans this is where of course you get     
43:08     
the morphology the Ion channel models the electrophysiology the connectone the     
43:14     
synapse and GAP Junction model and then the network activity that's those are sort of the     
43:20     
basic uh data collection components then you get this component construction where you get multi compon     
43:27     
Department single neuron models and connections you get model construction which is where you have this     
43:32     
biophysically detailed neural network and then you have the model optimization where you take the Dynamics     
43:39     
into account and you see how this neural     
43:44     
network operates over time so basically you're looking at activity in the uh in     
43:51     
the neural network and the connectto whereas here you're just building the connectto from parts     
44:00     
and so this is a biomechanical body model so notice now we're moving body we     
44:06     
have these uh this body that basically is this set of     
44:13     
tetrahedron and we have these uh these serve as vertices for the sort of mesh     
44:19     
model that they're using so that's for the body the actuator of these muscles of the actuator is how this uh     
44:28     
this geometric model moves you have the solver which projects Dynamics into the     
44:33     
future so basically as the worm moves what should it look like and we project     
44:40     
that behavior outward and we need to use solvers to do that because it's     
44:46     
basically trying to solve uh for 96 muscles 96 degrees of freom so this is     
44:52     
pretty complex work and then we have these what we call the hydron Dynamics which are these sensory sources you get     
45:00     
thrust and drag of the body relative to the sensory sources in the environment we can characterize that using     
45:06     
hydrodynamic simulations so again you know this is something that um you know depends on the     
45:14     
substrate right you know you have you can use hydrodynamics models and they     
45:19     
characterize the substrate you know to varying degrees of success all we're     
45:24     
interested in is the general relationship between the body and the environment     
45:30     
um and so this is an internal view of the body model uh and then this is this     
45:37     
is the triangulated mesh and then you know using that as the     
45:45     
body okay so this is where they quantify the body's Behavior they have these     
45:52     
they're trying to calculate this trajectory of the body so the body is here and it's moving around in     
46:00     
space and you get these basically you get these Transformations so the body's     
46:05     
position back here is going to project to this point over here in the future     
46:10     
and as the worm moves and it kind of follows this trajectory this is this     
46:16     
line here is the trajectory of the target Body Center so the target body is moving along this path and you get this     
46:25     
transformation that you have to project which is basically when this part of the body moves along this trajectory it     
46:33     
moves to a different position so as the worm moves and undulates it has to predict the     
46:40     
different positions on the body and how they transform in space so this is you     
46:45     
know this is what they're trying to do in terms of predicting or projecting out     
46:50     
movement in the organism this example in B is where you get this uh you have this coordinate     
46:59     
system of the standard body head coordinate system of the target body head and you basically track this over     
47:06     
time so this is um you know a tracking     
47:11     
problem C shows how the worm is going of tracked in space so as the worm is     
47:17     
moving it's a series of tracking trajectories and that's basically how it     
47:22     
reconstructs its position over time so as it's moving it's it's following these     
47:28     
trajectories notice that there are multiple trajectories that that are tracking different parts of the body and     
47:34     
then that's being uh extrapolated out of the future and then you have steering angle     
47:40     
which is calculated from sort of the angle of movement in the future that     
47:46     
this that the worm uh experiences so this is all kind of tracking Motion in     
47:53     
the environment um this is relative position of 3D tracking points again we have     
48:01     
position X position Y and position Z in time and this is going from head to tail     
48:07     
so these different trajectories are from head to tail top to bottom or anterior     
48:12     
to posterior and it's um these are the tracking positions     
48:18     
here so we can get relative position and relative velocity so we have this     
48:23     
velocity component which is of course not just the position over time but how     
48:30     
it speeds up and slows down over time so this is this is impressive work     
48:36     
um they have this Clos looped interaction as we mentioned before where you have sensory activation which feeds     
48:42     
into the neural network model then it outputs muscle activation which then affects the body environment model which     
48:49     
then affects sensory Activation so it's this closed loop interaction and they show the s     
48:57     
and then they show experiments so they actually have worms in a two-dimensional gel in culture where they're looking at     
49:05     
the movement of the tail in this case they're tracking the tail and the tail does the zigzag as it's moving across     
49:13     
the gel and you have this these different parts of the worm that are being modeled in the simulation and this     
49:20     
is what you're observing in the experiment so it's able to reproduce what's going on in the experiment     
49:27     
uh quite well um you know of course it there's a lot more noise in the I guess     
49:35     
well I don't know what they're they're kind of extrapolating this line out but     
49:41     
um yeah it think it matches up somewhat so it's it's basically producing the     
49:46     
kind of output they want um and then they have these uh     
49:51     
different things where they have in the simulation uh they're measuring water neuron membrane potential so they have     
49:58     
the different motor neurons listed here you're able to look at the membrane potentials that simulate those and then     
50:04     
the muscle Cell Activation so in seans we have specific muscle cells and we can     
50:09     
look at their Activation so again one advantage using seans is we have specific cells that we can track by     
50:17     
nomenclature and then produce an activation for each one we know they're position in the worm so we can actually     
50:24     
uh you know account for this on a Cell by cell basis and be very explicit in     
50:30     
how these things happen so this looks like where they're simulating the     
50:36     
uh the connectto and they're actually doing some interesting things here where     
50:41     
they're removing different parts of the connectto and they're getting different types of     
50:47     
behaviors so this is where they have like sort of a control um     
50:54     
connectone and they have this uh sort of this connectivity map I'm not     
51:01     
sure exactly what this refers to but basically it produces an output of     
51:07     
locomotion so for the control it's producing this realistic type Locomotion you can see the uh the     
51:14     
trajectory here you can see the worm and in B what they're doing is they're removing     
51:20     
neurites they're getting a different so this is a connectivity Matrix here principal components for the connectto     
51:28     
in the control you see that the principal components Matrix has a certain sort of shape to it uh then when     
51:35     
we remove a neurite we remove neurites we get some change to this connectivity     
51:40     
Matrix and we get a change to the output behavior when we Shuffle locations which     
51:47     
is I guess shuffling uh cells in the connectone we get this uh we get this     
51:56     
connectiv Matrix that's a little bit different than what we get in the control but it still preserves some of     
52:01     
the structure of the control and we get this Behavior that's modified um you know it's just look at     
52:08     
looking at the trajectory it's much more compact uh shuffling the synapse weights     
52:15     
we get a much different connectivity Matrix and we get this Behavior where I don't know if it's maybe less robust I     
52:22     
guess it's kind of hard to interpret these behaviors but you see changes in Behavior Behavior as you do these different     
52:28     
manipulations you can shuffle The Gap Junction weights the Gap Junctions are the electrical connections between cells     
52:35     
we usually have like a chemical um connectone which is where we have     
52:41     
different um synapses that we account for or we have a gap Junction connectum     
52:47     
or we have different elect connections between cells or direct connections between cells that we have for so we     
52:54     
have two different types of connecto andell elegans in this case we have a     
52:59     
gap Junction um connecto and we Shuffle those Gap Junction weights so that we     
53:05     
change the connectone in in a you know in an appreciable way and we get this     
53:11     
output this a shifted output and behavior we can also remove synapses in     
53:17     
the chemical connecto and we get this change in the conductivity Matrix and we     
53:22     
get this changing Behavior we can also remove Gap Junction to get something similar so you see what they're doing     
53:29     
here is they're building this principal component Matrix of the connecto from     
53:35     
like the connect the data driven connects that we have available they     
53:41     
make changes to those connects and then they look at The Locomotion output and see if it's this control or if it's     
53:49     
changed by making these these manipulations and the reason we want to do this is because you know sometimes     
53:56     
you get mutant like you get mutant phenotypes where there's a mutation in the genome and those mutations sometimes     
54:04     
can affect how the worm moves and so you can get these uh mutations that affect     
54:10     
the connecto and its structure and it shows that those can affect movement     
54:16     
sometimes those mutations are not viable in other words if the worm can't move it's not a viable phenotype but     
54:23     
sometimes they are and they just Al it just Alters the movement patterns so that's why they're doing this this set     
54:29     
of experiments and but actually they have some mathematical modeling down below but I'm not going to get into     
54:36     
that um but yeah that's an interesting paper um and again they're kind of going     
54:42     
for this niche in the model whole organ as a modeling area which is to say that they have this     
54:51     
uh you know they're they're trying to look at this brain body environment set of interactions and it's not as easy as     
54:57     
You' think to get those interactions um you know we kind of talked I think we talked about I think     
55:03     
maybe a couple years ago now where we tried to characterize environment and what does it mean to have an     
55:10     
environmental effect on something especially in development or in Behavior     
55:15     
you know when when the environment changes or when the environment hasn't an effect what is     
55:22     
that what is the sort of signal that's being sent what is the effect you know     
55:27     
is it just something we say because it's you know uh nice to say it just gives us     
55:34     
a stand in for something we don't understand or what is it so I think they kind of get at some of these things that     
55:42     
are happening in the environment and how do incorporate them into a whole body model of behavior and of     
55:50     
movement so it looks like dick had to go um for another meeting um and M had to go so are there     
55:59     
any other comments looks like it's more     
56:04     
complicated than what I put together oh well I don't think you're trying to do     
56:09     
the same thing but no alth it does look like it could be a     
56:17     
10sr yeah you could probably build a 10 saity and as I've found out it's really     
56:25     
hard to connect them and actually that's why they do single cells a lot it's hard     
56:31     
to connect um 10 secrity models together     
56:37     
yeah they don't seem to cooperate with each other when you say 10 secrity models you mean more than a few cells so     
56:44     
if you wanted to yeah if you build a nice uh Integrity model it stands     
56:52     
upright you can manipulate it and then you want to connect it to another     
56:57     
one the exact same thing yeah them together well it doesn't work that     
57:04     
well to connect them together yeah which is interesting     
57:13     
yeah or a huge headache in my case yeah yeah I think that's true of     
57:19     
most modeling paradigms like where you try to connect things together like from     
57:26     
well you these multicompartment models which allow you to sort of model things as compartments but like a lot of times     
57:34     
getting the Dynamics to interact well like if I were to get to you know connect like two connectomes or     
57:40     
something that would be kind of hard to do or connect the effects between two connecton or two worms you know how     
57:47     
could I do that without uh blowing it up blowing the simulation up and yeah well     
57:57     
in Mye it was dog playy     
58:03     
toy I could only connect it with certain     
58:09     
um Junctions right     
58:15     
yeah yeah anyways yeah oh well if you read my paper you     
58:23     
can see that I can only use the simp simple ones on the outside and the more     
58:28     
complicated one in the middle that's that was one of the main     
58:34     
things about that thesis is this is how I managed to connect these 10 segres     
58:41     
together yeah I've been looking at your thesis and it's interesting I haven't gotten really deeply into it though okay     
58:49     
well the first part of it is is what is the physical model of cell using cyto     
58:57     
skeleton like what is it yeah and then the second part is to make something     
59:04     
that that works that follows that yeah I basically used the simplest     
59:12     
tties I could that's the simplest model I could come up with so     
59:19     
yeah so it's a minimalist type um t     
59:28     
model     
59:35     
yeah anyway I would like to make a more complicated model but     
59:42     
um yeah I tried to actually uh but then I     
59:48     
couldn't get the um my hexagons to line up yeah so I don't know what's up with     
59:56     
that oh     
1:00:02     
well are you yeah you're you're uh I guess you're GNA continue we talked     
1:00:07     
about this before you're G continue working on that yeah I'll continue working on that to make the seven self     
1:00:15     
um model I'll I'll try it sure um and I want to     
1:00:21     
investigate using other programs to make hand models but I'm not that interested     
1:00:28     
in in using um commands line code to     
1:00:34     
make a physical model just well I just don't want to but like     
1:00:41     
Linux um there's a program called     
1:00:47     
Moro um and but it's Linux so I'm anyway and     
1:00:55     
it won't work on this the computer that I'm on currently I got a new computer     
1:01:01     
I'm going to try things out yeah yeah so I'll keep keep working on     
1:01:07     
things that's good     
1:01:14     
yeah oh well yeah well thanks all right well if there's no more     
1:01:21     
uh no more questions or comments let's call it a day uh thanks for attending and see you     
1:01:28     
next week all right bye bye bye     
