     
Transcript     
0:00     
paper so it didn't get it to any journals but we're you know we might do another version of it too it's up to     
0:08     
you as I know that people have done like different comparisons like that it's not     
0:13     
just like you know this isn't the only paper on it but you know it's just like trying to get like uh I mean people     
0:20     
usually hit the same set of issues over and over and like getting some new issues in there would be good but all     
0:27     
right uh nice to see everyone here here well with soam Morgan dick     
0:35     
hello okay so today I wanted to talk about a couple of papers again uh so the     
0:42     
first paper today I'm going to talk about is this paper Gene regulatory patterning codes and early Sate     
0:48     
specification of the sea elegans embryo this is Allison Cole Tamar hash Hashim     
0:55     
shyu and itai and so the abstract reads pattern     
1:02     
formation originates during embryogenesis by a series of symmetry breaking steps throughout an expanding     
1:08     
lineage so this is where we have our lineage tree and the lineage tree     
1:14     
divides into so one cell divides into two Cells Two cells divide into four cells and you keep going now throughout     
1:23     
um the lineage tree you get first of all you get asymmetric divisions so you get     
1:29     
divisions where there's only one cell produced instead of two from from a division event the other example of a     
1:36     
symmetry breaking step is where cells will divide at different rates so if we     
1:42     
think about our hypothetical lineage tree we divide from one cell to two cells to four cells to eight cells to 16     
1:50     
cells and generally that's the way it works early on in Sean's development and     
1:57     
then later on in Sean's development you start to get not multiples of two but you get sort of     
2:03     
these rounds of division that are in between multiples of two so you might have two cells on the end of your 16     
2:11     
cell embryo on the uh anterior end they might undergo a division event which     
2:18     
gives you an U an 18 cell embryo and then the rest of the cells divide which     
2:25     
then gives you your 32 cemri so this is you know kind of this     
2:31     
the Symmetry breaking steps their divisions that occur sort of asymmetrically they're asymmetric     
2:38     
divisions themselves and those lead us in celegans because the cells are all     
2:44     
deterministic and the cell lineage tree is deterministic to the emergence of     
2:49     
different structures at different times so embryogenesis essentially is the     
2:55     
Symmetry breaking process as much as it is a pattern formation process     
3:01     
now they talk about dropa here the paper is about celegans but they talk about drop a little bit at the beginning so in     
3:08     
dropa classic work has been shown that the embryo segmentation is established by morphogens within a centium so a     
3:16     
centium is a single cell with multiple nuclei so muscle is a centium this is     
3:23     
multiple nuclei but a single cell and so you get this centium stage and drop     
3:29     
development where you get all these uh these nuclei within Ain a structure     
3:36     
and then that structure cellular rizes and you get to the next step step of     
3:42     
development so this is what they're kind of getting at here so they're talking     
3:50     
about within the centium you have the segmentation you have these morphogens which are these signaling molecules and     
3:57     
then the action of genes that set up these gradients that produce uh sort of     
4:04     
gradients within that centium and you see striping or you see     
4:09     
other types of gradual change across the anatomy of tropus so from anterior to     
4:17     
posterior end of that centium you see diff gradual differences in what's being     
4:25     
expressed so that's indrosophila and we'll see images of that later this classic model however does not     
4:31     
translate directly to species that lack a Cen tisue such as celegans where sufate is specified by cell autonomous     
4:38     
cell lineage programs and their inter signaling so a lot of the work on like     
4:45     
you know Hawk genes on Gap genes on segment polarity and that sort of thing     
4:50     
was done in ROP and that work androp applies to     
4:56     
miman systems but it does not apply to celegans which is a fellow     
5:02     
invertebrate and but it doesn't have the same so we're we're actually looking in C Elegance here we're thinking about a     
5:08     
different system altogether we're thinking actually about this kind of system where we have a deterministic     
5:15     
lineage tree so previous single cell AR seek     
5:20     
studies and celegans have analyzed cells from a mixed suspension of cells for many for many embryos the study weight     
5:27     
differentiation CH stages or individual early stage embryos to study early gene     
5:33     
expression in the embryo so what they're doing here is they're doing single cell or seek     
5:39     
they're looking at different cells they get a mixed suspension of cells and     
5:45     
they're going to sort of do this study where they look at the expression of cells in different sublineages in this     
5:51     
lineage tree and it it may tell us something about how the origins of this     
5:57     
asymmetry to study the intermediate stages of     
6:02     
early and late gastrulation 28 to 102 cell stages respectively missed by these approaches     
6:10     
here we determine the transcripts of the 1 to 102 cell stage to identify 119     
6:16     
embryonic cell States during a cell fate specification including equivalent group     
6:21     
cell identities so they're looking at these two stages of development early in L ASR     
6:28     
relation this is kind of you know getting out of this period of doubling this early period of doubling when you     
6:35     
get to about 16 cells you get these actually you get to 16 cells you     
6:41     
have these actually eight cell you have the founder cells and then at the 16 cell stage you have these sublineages     
6:48     
that emerge and so the sublineages are sometimes they're specialized for     
6:54     
different types of cells sometimes it's a mix of cells in them and sometimes it's the germ line so we'll see later     
7:01     
that you know there are these different sublineages that are already forming and so now you know we know kind     
7:08     
of what type of cell adult cell it's going to be and we'll see from the lineage tree that the developmental     
7:14     
cells have a nomenclature that matches this sort of these sort of sublineages     
7:19     
in the developmental lineage tree and so we start to get these sort     
7:26     
of asymmetric uh uh division events where we get the 20 28 cell stage get 102 cell stage so you can see that     
7:33     
gastation is producing or other processes as well are producing the Symmetry breaking     
7:40     
steps but then we also want to identify different types of cells so Sate     
7:45     
specification you know what what what what is the differentiation state is there like an intermediate state that     
7:53     
way can identify we find the gene expression     
7:58     
programs are modul according to the subcell lineages each establishing a set of Stripes by combinations of     
8:05     
transcription Factor gene expression across the anterior posterior axis so we     
8:11     
know of course that the uh the lineage tree is organized     
8:17     
anterior to posterior the left side of the tree is anterior the right side is posterior the right side of the tree has     
8:24     
the germ cells left side of the tree has the ab sublineages and everything in     
8:30     
between is defined and so instead of the sort of striping that we see in     
8:36     
dropa we just simply see things happening at the anterior end the posterior end and sort of in the middle     
8:44     
and so this is um kind of an interesting difference between these two types of     
8:50     
systems now we know that seelagan shares some of these genes like at least that     
8:56     
they share Gap par Rule and segment polarity genes and so we'll see if they play a role in     
9:03     
what's going on here in particular expression of the homeo demain G ex or genes estab establishes a comprehensive     
9:10     
lineage specific positioning system throughout the embryo beginning at the 28 cell stage so we actually get this     
9:15     
sort of uh positional information or this positional independence of these     
9:22     
cells and so we are already start to see that here and it involves some of the     
9:27     
homeo domain genes we see in the striping and indrop so moreover we find     
9:32     
that genes that segment the entire embryo andopa have orthologs and seans     
9:37     
that exhibit sublineage specific expression these results suggest that the seelan embryo is patterned by a jux     
9:43     
toos of distinct lineage specification genes regulatory programs each with a     
9:48     
unique encoding of cell location and fate so they talk start talking about developmental toolkit genes which is     
9:55     
sort of what we have in sort of determining the pattern of the organism how patterns are formed or how we go     
10:02     
from these undifferentiated cells to the sort of these patterns or these     
10:07     
differences within the embryo and so we can think of that as patterning we can think of that as heterogeneity we can     
10:14     
think of that as symmetry breaking so we can think about the seans endoderm and how that's specified by a Cascade of     
10:21     
transcription factors operating sequentially identifying the principles of Gene regulation has been a central     
10:27     
Endeavor in developmental biology in this paper they use single cell R seek     
10:33     
to identify the transcriptomes of each of the cells up to the 102 cell stage we     
10:38     
find that homeo demain genes are expressed in Stripes along the anterior posterior axis as early as the 28 cell     
10:45     
stage interestingly each founder cell lineage AB Ms c and e establishes its     
10:53     
own regionalization code and so AB of course includes a number of cell types     
10:59     
MSC and E are all very specific to specific cell types to organs so we're     
11:05     
going to see you know this this obviously it they claim that this     
11:10     
establishes a regionalization code so we'll see what that looks like these Drop Like stripe pattern suggested deep     
11:17     
homology and sulfate specification between the embryogenesis of non-segmented and segmented animals     
11:24     
despite differences in stici based and cell cleavage based modes of development so sention based are which you see in     
11:31     
drosophila cell cleavage based is like what you see in celegans so if we look at figure one we can see in     
11:39     
a the lineage tree and we can see that we start with uh two cells on each pole     
11:46     
AB which is anterior P1 which is posterior and then we go down to our different uh division events we go down     
11:54     
to the 102 cell stage and we go down even further but this kind of shows sort of that the end late gastation here and     
12:03     
we can see down here that we have these different um sublineages and then the     
12:09     
type of tissue that's formed from that or the cell types that form from these different sublineages     
12:16     
so in AB we have neural cells we have hypodermis cells we have     
12:23     
ferx and bl or blast cells and then MS s     
12:29     
we have fering cells and muscle cells e     
12:34     
we have allot cells C we have hypodermis and neural cells D we have     
12:41     
all muscle cells and then P we have germ cells so this is where we have the sort     
12:47     
of The Germ cells that are segregated out very early in the lineage tree and they're kept separate from the rest of     
12:54     
the cells so basically AB is very heterogeneous we have different cell types here     
13:00     
uh forming Ms e c and d are all more or less um you know specific to different     
13:07     
you know one or two different cell types and so we see all that happening at either at the middle of the anatomy or     
13:15     
towards the posterior end of the anatomy which is interesting because that gives us of course a place for those organs to     
13:23     
form but a lot of this sort of thing happens early on in gastr relation so     
13:29     
we're talking about again like from 28 to 102 in this part of the cell lineage     
13:36     
and of course these uh these symmetry breaking events are important for setting all this up now in this example     
13:43     
B we have a tne plot so we're taking tne one versus tne 2 and we're looking at     
13:49     
the different cell types in the different sublineages so we have the cell number this is     
13:57     
colorcoded by cell number so this is where we're going from like small numbers of cells here to larger numbers     
14:04     
of cells here it looks like larger number of cells are kind of on the outer edges of this and the smaller number of     
14:10     
cells are in the middle so we have of course the germ cells here the gut cells they form a     
14:17     
group muscle cells are out here a ferx is over here blast cells are up here and     
14:23     
the neural cells are here and so you can see that there is this sort of uh different     
14:30     
in this t- sneak plot of different cell types and looking at sort     
14:36     
of those um how what that looks like using dimensionality     
14:43     
reduction and so if we look at this a little bit more closely by sublineage     
14:48     
and by Gene we find that Ms expresses     
14:53     
ch51 here e expresses el7 here these are the mark workers that they're using for     
15:00     
these different lineages so we have c51 for Ms el7 for E P1 for C P1 for D and     
15:10     
nls2 for p and so you can see that you know these markers are sort of in the     
15:16     
neighborhood they're they're where we expect them to be more or less and then     
15:22     
for Our Fate we have different uh cell     
15:28     
Fates feren muscle hypodermis neuro and blast and then we have markers for these     
15:34     
so we have faor for ferin H hnd D1 for muscle elt1 for hypodermis ch58 for     
15:41     
neural and UNC 6 for blast and so they're all kind of where we expect them to be as well this is uh telling us the     
15:49     
level of gene expression so this is where these markers are highly expressed     
15:54     
on this T map versus where they're not which is gray and so if we look at their     
16:00     
results we found that the cells follow developmental trajectories according to their founder cell Origins each     
16:06     
identified by the expression of genes known to be expressed in a wage specific manner so these are uh these markers are     
16:14     
expressed above sort of the control level so when we have differential gene expression we look at some housekeeping     
16:21     
Gene and then we look at the expression of this marker above sort of background     
16:28     
or you know chance basically so the higher the level of expression the     
16:33     
greater the fold expression the more strongly that marker is expressed and     
16:39     
the better it is for using using it for a marker and if they're in cells that we know are that type of cell then that's     
16:45     
that means that it's a good marker to use as observed by other studies the founder of celline trajectories the     
16:52     
founder of cell linear trajectories are oriented according to cell Fates with similar cell Fates converging between     
16:58     
linear so for example the cells of the Ecto mesoderm producing sea lineage formed by     
17:04     
a biobed cloud with the muscle part adjacent to that of Ms and D lineages similarly Ms cells that will form the     
17:11     
ferx Orient towards the ferx producing portion of the ab lineage so figure two is where they try to infer the     
17:18     
transcriptome of these different cell lineages and so here I think D is a good     
17:23     
example of what this looks like in D we see that we kind of recapitulate that     
17:29     
uh the lineage tree we have different markers for each cell so ab and P1 we     
17:36     
use Mex 3 as a marker and they're both expressing Mex 3 at chance or at     
17:41     
background and everything that's dark blue is expressed at background everything that's a lighter shade     
17:47     
leading up to Red is expressed more highly so red is is high expression dark     
17:56     
blue is low expression and everything in between red and blue the lighter colors are moderate amounts of expression so we     
18:02     
see nothing in ab and P1 for X3 when we get down to the four cell stage we begin     
18:09     
to see a differential expression of SM 9 uh in ABA FL fh3 and EVP EMS has P10     
18:20     
and sm9 and then P2 has a ama1 so but     
18:25     
you can see what happens here you start to get different things Express differentially in different     
18:30     
lineages or sublineages so ABA and ABP despite originating from the same Mother     
18:37     
cell are expressing different markers differentially EMS and P2 descending     
18:44     
from P1 do the same thing this gets even more pronounced as we move down the when it's treated to the eight cell stage     
18:51     
where we have a bunch of genes here and we have differential expression now     
18:56     
that's I think a lot more clear where ABA and ab R basically um are not     
19:04     
expressing anything differentially in abpl and abpr you have hh26 and HMG 4     
19:12     
which are maybe expressed a little bit uh differentially but not very much but then when you get to Ms and E they start     
19:20     
expressing uh I think TB x35 and Ms and     
19:25     
end3 and E are expressed that a much higher level and then in the germ line     
19:31     
here P3 and then in C which is not the germ line um you get different like Mom     
19:37     
2 and c and then cy3 and P3 so you get these different uh types of differential     
19:45     
expression in different sublineages and this just becomes more pronounced as we move down especially when we start to     
19:50     
get to the 28 cell stage where we start to get like these high levels of differential     
19:57     
expression and different markers um and you know even in closely related lineages but in O you know     
20:05     
overall we have these groups of cells here um like ala and     
20:12     
AR that are uh much more similar than things like P and PRP in this other     
20:20     
sublineage so you see as you move away from each other in terms of it being part of the same sublineage you start to     
20:28     
express different things differentially so you see here uh msaa and     
20:34     
msap and Ms PPA and Ms PPP they both actually express the same things     
20:41     
differentially which is interesting and it's different than some of these other daughter cell comb uh combinations where     
20:50     
they're expressing other things differently so it's interesting that there are some cases where things maybe     
20:55     
get preserved or things aren't that much different and places where things are     
21:00     
sort of always unique and so what this might suggest is that you have a     
21:06     
specific cell type that's sort of Representative of this lineage we know     
21:12     
like from what we just said before that in these these lineages like Ms and E     
21:19     
that you're going to have like one or two cell types produced um as opposed to a mosaic of     
21:26     
cells in the ab lineages so this is kind of what they're getting     
21:31     
at here um and then you know as you go down it gets even more unique but you do     
21:37     
get patterns across sub lineages maybe this is where you start to coordinate uh     
21:42     
organ formation and things like that     
22:06     
so in figure three they start getting into how all this plays out spatio temporally because so far we've looked     
22:12     
at the temporal domain we've looked at the lineage tree and we've seen that there differences as the lineage tree     
22:18     
unfolds that's obviously in space but there's also the spatio temporal aspect     
22:23     
which is where we have these different lineages that will you know are going to be     
22:29     
sort of located in a certain location in space and then there is you know overlap     
22:36     
in some of these and so we want to see like these over because in in uh drop     
22:42     
when you get overlap of gradients they sort of form this is the place where they form Stripes so if you get two     
22:49     
gradients that overlap they form a sharp boundary and get Stripes so what happens in SE Elegance     
23:01     
well we'll look at two different lineage or we'll look at two different cells here um AB p o PPP and e a l a so these     
23:09     
are in different parts of the lineage tree so we actually have this little graph where it shows kind of these     
23:16     
different classes of uh genes that are found so for ab PPP we have 921     
23:24     
genes we have uh this blue area which are the homeobox     
23:30     
domain genes 103 genes we have transcription factors expressed in the     
23:36     
cell of Interest which is here and then gazing finger transcription factors 14 genes so we have the same thing for each     
23:43     
cell type and then here in this yellow we have this enrichment or actually it's the overlap between the blue and the     
23:50     
yellow it's this overlap area this overlap area is enrichment for homeobox     
23:56     
domain transcription factors and this is uh the case at a significant     
24:02     
P value and this is true for seven genes so seven genes are enriched for homeobox     
24:08     
domain transcription factors uh for eala we have both enrichment and depletion so     
24:15     
what we're going to have in some of these cells we're going to have things that are enriched we're going to have things that are upregulated and we     
24:21     
expect to see that because that's what's going to drive differentiation between cells we also have in     
24:29     
particularly is this depletion of cells or this depletion of genes so we have     
24:34     
things that are kind of shut down and things that are turned up and so this uh     
24:40     
in the in the case of e we have enrichment and depletion so overlap     
24:45     
between the red and the yellow that's enrichment for gazing finger transcription factors at a significant P     
24:52     
value that's five genes and then depletion for homeobox domain transcription factors rather than     
24:59     
enrichment is at a statistical significance of     
25:04     
0.01 and that's two genes so you see the logic of this test     
25:10     
we're looking for the overlap between transcription factors expressed in the cell of Interest all homeobox domain     
25:18     
genes and then if it's enriched depleted the same thing for G zinc fingers and     
25:23     
transcription factors expressed in the cellum interest and that's either enriched or deplete completed so if we     
25:29     
look down at this uh graph these graphs in B and C we see on the left homeo box     
25:35     
domain transcription factors and on the right uh Gata zinc finger transcription     
25:40     
factors and so we see enrichment in red and depletion in blue we can see that     
25:46     
for G zinc fingers almost everything is enriched in this uh in this part of the     
25:52     
lineage tree from the four cell to 102 cell stage and then we see for homeobox     
25:59     
domain transcription factors we see a lot of enrichment in the ab lineage and     
26:05     
their sub lineages but we also see a lot of depletion in the E C D and P sub     
26:14     
lineages so we actually see this with respect to these diff so homeobox domain     
26:21     
transcription factors of course were this area here and we can see that or     
26:27     
homeobox domain trans transcription factors they're both enriched in ab and     
26:32     
Ms sublineages and depleted in EC dnp sub lineages meaning that of course     
26:38     
there are going to be differences and differentiation here between these cells eventually but also that there's a     
26:44     
spatial segregation that towards the posterior end you get this depletion of     
26:49     
homeobox domain transcription factors we also see for G zinc finger     
26:55     
transcription factors no such spatial patterning although we see enrichment pretty solidly across the E of lineage     
27:02     
which means that that whatever cell types are going to come out of the es of lineage are going to be enriched for     
27:08     
ging finger transcription factors     
27:30     
and so in this graph in D we have the different stages we have the lineage for     
27:35     
different at different stages so this is 15 28 51 and 102 and then the dark blue     
27:42     
is AB and everything goes out to Ms E C D and P so you can see the sort of the color coding here so each one of these     
27:50     
columns here go down for each one of these classes of transcription factor and we want to see if it's either     
27:56     
depleted or enriched so for Helix Loop helix DNA binding     
28:02     
domain most of the time it's enriched although it doesn't it it stops being as     
28:08     
enriched as it is in the early stages later we see a little bit of depletion     
28:13     
in I believe it's the P lineage in the 28 cell stage which is interesting but     
28:19     
you know that we don't really know too much about that the homeobox domain     
28:24     
however has sort of patterns of depletion and enrichment as we we go along with a lot of depletion in the e     
28:31     
su lineage as we talked about in the last uh figure and so this just shows     
28:37     
that like this depletion is established actually somewhat early in     
28:45     
development the zinc finger c2h2 type is     
28:50     
slightly enriched across all times and across all suppin ages you see for zinc finger C4 type you     
28:58     
get a lot of depletion for Lian binding domains of nuclear hormone receptor we also get a lot of depletion and     
29:05     
enrichment a bzip transcription factor is almost always slightly enriched forut     
29:11     
domain is also almost always slightly enriched that a zinc finger is almost always slightly     
29:16     
enriched and so on and so forth so we can see for each of these categories of transcription factor that there these     
29:22     
different patterns in different sublineages at different stages of     
29:28     
development so we have the spatio temporal diversity here of these different um transcription Factor     
29:36     
families then we can see this broken out by sublineage so this is the MS sublineage the E sublineage the p     
29:44     
sublineage and this is true     
29:52     
for this is true for both uh cell stage and for subl     
30:17     
Lage finally if we take a look at our markers across the embryo so remember     
30:23     
the the lineage tree goes from anterior end to posterior end     
30:28     
you have these different cell stages the 28 cell stage the 51 cell stage and the 102     
30:35     
cell stage and you can actually look for spatial expression of these different     
30:42     
transcription or these different genes in different cells so actually what we see here is we see a lot of expression     
30:51     
across the embryo for different so actually within the ab lineage here we     
30:58     
see the gene expression matching the ab lineage almost exclusively through and     
31:05     
then we can look at the MS lineage and the C lineage and we can also get a     
31:15     
signal so what we have are these lineage specific stripes across the early embrio     
31:23     
throughout pastation just like what we see in dropa resembles dropa it's not     
31:29     
exactly like dropa because we have a different mode of division and     
31:35     
expression of these genes     
32:22     
okay now this is a fun paper I was I found this paper when uh just looking     
32:27     
through the literature uh this is an older paper um this is from 2017 and     
32:34     
this is from molecular biology and evolution title of this paper is multiple regulatory modules are required     
32:41     
for scale to feather conversion and it's by and it's actually a actually a     
32:47     
paleontological paper but it's also an evoo paper so it's evolution of development but they're looking back     
32:54     
into evolutionary history and looking at how we get these genetic regulatory     
33:00     
modules that control uh scale production and feather production and especially in     
33:06     
lineages where we go from scales to feathers or feathers to     
33:12     
scales so the abstract reads the origin of feathers is an important question in evoo studies with the eventual evolution     
33:20     
of veined feathers which are aerodynamic allowing feathered dinosaurs and early birds to fly inventure into new     
33:26     
ecological niches studying how feathers and scales are developmentally specified provides     
33:32     
insight into how a new organ May evolve we identified feather Associated genes     
33:37     
using genomic analyses the canidate genes were tested by expressing them in shiken and alligator scale forming     
33:44     
regions so this is where they're looking at these genes they're trying to find     
33:49     
things that are associated with feathers and then they want to express them in certain scale forming regions that will     
33:57     
allow us to do experiments where we can see their effect on the     
34:02     
phenotype ectopic expression of these genes induce intermediate morphotypes between scales and Feathers which reveal     
34:10     
several major morphogenetic events along this path so we're looking for the things     
34:17     
that form scales the things that form feathers and if we can form something in between those two and indeed we can so     
34:25     
there's several major morphogenetic events that we need to be you know sort of figure out how this process works so     
34:32     
the first is localized Zone growth formation the second is follicle     
34:38     
invagination the third is epithelial branching the fourth is feather keratin     
34:45     
differentiation and the fifth is dermal pill formation so we have this five-step     
34:51     
process and if it's disrupted in different ways we either get scales or feathers     
34:58     
in addition to molecules known to induce feathers on scales such as retinoic acid     
35:03     
and beta cadinin we identify novel skill feather converters socks 2 zi 1 remem 1 spry 2     
35:12     
socks 18 so there's some socks genes in here a zik gene and we know those are     
35:17     
important in developmental pattern formation and different cell differentiation so we get these scale to     
35:23     
feather converters and we get this induction of feathers on scale     
35:29     
these novel scale feather converters induce one or more regulatory modules guiding these morphogenetic events some     
35:36     
morphotypes resemble filamentous appendages found in feather dinosaur fossils or others exhibit     
35:42     
characteristics of modern Avan feathers we propose that these morphal regulatory modules were used to diversify     
35:49     
archosaurus scales and do initiate feather Evolution this goes back to     
35:55     
dinosaurs and how you know we can think about the common ancest of some of the     
36:01     
modern U lineages and clades that have feathers and have     
36:08     
scales the regulatory combination and hierarchical integration may have led to the     
36:14     
formation of extent feathered forms our study highlights the importance of integrating discoveries between     
36:20     
developmental biology and pale anology so of course in amniotes we have     
36:28     
different types of these inary appendages which are involves scales     
36:35     
feathers and hair so you'll see pictures of the feather follicles they look like     
36:40     
hair follicles and they are also involved in forming scales these appendages exhibit     
36:46     
region specific distributions on the body surface so for exist for instance     
36:51     
many birds have scutate in reticulate scales on their lower legs and feet and it feathers in most of the remaining     
36:57     
body parts crocodilians have different types of scales in their body surface so this you know this involves     
37:05     
avans this involves lizards and amphibians this involves snakes and this     
37:11     
also involves other amniotes as well the feather is a novel organ that     
37:17     
evolved from dinosaur uh integuments developmental biology studies and recent fossil fines reveal     
37:24     
that feathers evolve from a series of novel morphogenetic events which we talked about in the     
37:31     
abstract and they also talk about some of the early epidermis dermis recombination experiments where E9 Shake     
37:39     
dermis does not alter the fate of feather epidermis however at a later stage of development which is E12 shank     
37:46     
dermis can cause feather epidermis to form some scale-like skin appendages so these are some early     
37:52     
Studies by RS in 1963 that look to understand sort of the develop Nal     
37:58     
regulation of these different structures so this is an examp this is a philogyny     
38:06     
of amniotes so this is the stem amniotes here which is the common ancestor of all     
38:11     
amniotes we have mammals here avens and non- Aven reptiles so we have these     
38:19     
three different groups and so in mammals we have hair of course we have the hair     
38:25     
follicle we have bald stem C we have these other structures then if     
38:31     
we look over to avens we have feathers and scales we have feathers and again     
38:37     
they form this follicle they form this invagination into the skin just like hair so the epidermal invagination is     
38:45     
here in hair and here in feather so feather is wider and so it's a bigger invagination you have these stem cells     
38:53     
here and you have the supporting structures around it then you have scales and the scales are     
39:00     
like these sort of bends in the epidermis uh have different shapes so     
39:06     
you see that here you see that also in non-avian reptiles where you see maybe a little bit broader diversity of scale     
39:13     
types but you have these other scale types that are conserved I guess between avens and non-avian     
39:20     
reptiles so that's what we're talking about we're talking about hair we're talking about feather and scale development together in the same     
39:28     
group of organisms and then we have just exclusively scale production in non-avian reptiles but they all have     
39:35     
this sort of common origin in stem amniotes then we have this uh image B     
39:42     
here which is upregulated genes in the feather region so they have these different regions that they pick out in     
39:48     
this embryo one two three and four and they kind of zoom in on that     
39:55     
area they look at the different types of uh genes that are involved in different     
40:00     
form the formation of different things so they have this feather versus scale     
40:06     
feather versus scale E9 leg E9 wing and then E9 dorsal so one is dorsal this red     
40:12     
box this is feather versus scale uh the blue box two is leg the Box     
40:21     
three is wing and then box four is down here and they just mark this off and     
40:27     
they do this sort of vend diagram that shows that there are these uh genes sort     
40:33     
of that are expressed in common this leads us to this comparison where we     
40:39     
have between upregulated genes in the feather region not U regulated genes and     
40:44     
feather development between the uh this region here uh on the on the dorsal side of the     
40:52     
or embryo and the scutate and then we have this comparison     
40:58     
here we have 218 feather region independent genes 552 feather     
41:03     
development dependent genes and then 102 feather Associated genes so that's how     
41:09     
they get to feather Associated genes here's another figure that shows the expression of zi one and E9 embryo so     
41:16     
you have Zig one in these different areas uh you don't really see a lot of     
41:22     
expression a little bit down here towards this tail you see some sort of     
41:27     
patterning in different parts of the embryo but not very strongly as opposed to socks 18 where you see strong     
41:35     
patterning at the same developmental period in different parts of the embryo     
41:40     
we also see this analysis where we have feather at E7 and E9 and those genes are     
41:47     
upregulated uh compared to the scale genes which are not upregulated and this is their genomic position up here and     
41:54     
then this just shows their expression level we see this for socks 18 but in socks 18 we have feather uh related     
42:01     
expression here and then scale related Expressions we get some scale related expression for socks 18 but a lot more     
42:08     
feather related expression then we have these inc2 hybridizations on the right so this is where we have the control     
42:15     
which is here socks 2 gr 1 spry 2 Beta cenin so these are all the different     
42:22     
markers that they're using and then they show the thigh and the shank so this red box is the thigh this blue box is the     
42:28     
shank for each sample so the shank is down here the thigh is up here this is     
42:33     
the thigh this is the shank you don't see very much uh you don't really see any expression of anything in the     
42:39     
control obviously socks 2 you start to get some expression uh here you see this     
42:45     
pattern expression but you don't see anything in the shank really but you see it in the thigh and of course they have     
42:53     
an inset that shows the scales forming here you have gram one which forms this     
42:58     
different type of scale here and you see that in the thigh but not the shank spray two you get this scale     
43:06     
formation here where you have uh in the in the thigh that's a different pattern     
43:12     
and then in the shank you don't see anything finally beta cenin you see this pattern here uh where you have strong     
43:20     
expression in the thigh but also in the shank and you see what that looks like     
43:25     
in the inset on the shank pink in terms of morphology and so these red arrows is are feather Bud mezen kimes cells and so     
43:33     
for socks 2 grum One spriton beta Ken and you have these feather but mezanine     
43:40     
cells in this structure so finally what we end up with is the ability to form     
43:45     
intermediate phenotypes from molecular and chemical perturbations their data suggests that a successful formation of     
43:52     
feathers requires the integration and coordination of these molecular modules in this work our developmental biology     
43:59     
based approach revealed molecules that are involved in each regulatory module our studies provide groundwork towards     
44:05     
an understanding of how these regulatory modules are integrated at the molecular level for the generation of real     
44:11     
feathers is defined by the five Criterion and so the five Criterion are these five steps that they mentioned in     
44:17     
the     
44:26     
abstract     
44:56     
e     
45:25     
e     
45:55     
e     
46:25     
e     
46:55     
e     
47:25     
e     
47:55     
e     
48:25     
e     
48:55     
e     
49:25     
e     
49:55     
e     
50:25     
e     
50:55     
e     
51:25     
e     
51:55     
e     
52:25     
e     
52:55     
e     
53:25     
e     
53:55     
e     
54:25     
e     
54:55     
e     
55:25     
e     
55:55     
e     
56:25     
e     
56:55     
e     
57:25     
e e     
58:25     
Okay so so leth and I had a discussion in today's meeting and our discussion was about the difference between     
58:32     
artificial neural networks and biological neural networks and I had mentioned that we had published a paper     
58:37     
back in 2021 on the topic uh we being myself Krishna katyo and Jesse parent     
58:44     
and the name of this paper is connectionism complexity and living systems a comparison of artificial and     
58:50     
biological neural networks so this is on the archive um you can see here this is the     
58:56     
stub and in this paper we try to have a discussion on some of the major     
59:02     
differences between artificial neural networks and biological neural networks um nothing Earth shaking really we kind     
59:09     
of characterize the different uh you know the major differences between the two different     
59:17     
types of systems uh you know this is something that we did a presentation on in one of     
59:23     
the early neurom match conferences and then we decided to publish this so this     
59:28     
is the preprint um we make a number of comparisons between artificial and biological neural networks so we have     
59:36     
this part of the introduction where we talk about we don't really get into uh bio inspired models or more     
59:43     
exotic types of models we just focus on artificial neural networks different     
59:50     
types of things that artificial neural networks that can do which seem to be you know life like or     
59:57     
cognitive and so you know we have but we ask the question okay what are the     
1:00:02     
differences between models like that and biological neural networks knowing that artificial neural networks are not going     
1:00:09     
to give you the same result so maybe what are some of the reasons why that difference     
1:00:15     
exists and so we kind of get into some of the uh points here the first point is that there's a     
1:00:22     
sparse behavioral output and internal representation in artificial networks so     
1:00:28     
a sparse Behavior output means that it's doesn't provide a rich repertoire of     
1:00:35     
behaviors it provides maybe one stereotype type of behavior or multiple     
1:00:40     
stereotype types of behavior and those types of behavior are     
1:00:45     
generally the outcome of training the model so you have to train the model before you can get an     
1:00:51     
output in biological neural networks of course you have learning and you have development but some of     
1:00:59     
what a biological neural network does especially for stereotype types of     
1:01:04     
behaviors are innate they're born of internal to the     
1:01:10     
organism so you know we have organisms with innate behaviors we have organisms     
1:01:18     
with autonomous behaviors um and you know we we don't see that in artificial neural networks     
1:01:27     
they also have a difference in terms of representation so artificial neural     
1:01:33     
networks have sparse representations compared to Behavioral or compared to biological neural     
1:01:39     
networks and so we can think of this as a a difference in representational complexity so this is an example of the     
1:01:47     
difference in biological neural networks we have a number of different cell types     
1:01:52     
A number of different communication paths and and other types of support mechanisms that make for this biological     
1:02:00     
system uh and this is a you know a picture of a a couple of neurons     
1:02:06     
connected by dendrites and axons and synapses and so all of those things are     
1:02:11     
there plus we have support cells like glea and astrocytes which keep everything um as     
1:02:18     
as a homeostatic system in the inset we see this artificial neural network which is basically weights or computational     
1:02:25     
weight weights and a cell body that sums those weights and provides an output     
1:02:30     
function so we don't really have that same type of complexity here we don't have different levels of complexity like     
1:02:37     
biological networks we have a molecular uh level of complexity we have     
1:02:42     
phenotypic level of complexity we have a network level of complexity here we just     
1:02:48     
basically have the network level of complexity and some phenotypic complexity if you think that the cells     
1:02:56     
the the units or the connections are you know sort of stand on their own another     
1:03:02     
thing we talk about is heterogeneity and specifically that biological neural networks are     
1:03:08     
heterogeneous and so you know as in an artificial neural network basically you     
1:03:14     
have undifferentiated units and those units are connected by undifferentiated     
1:03:19     
weats and they combine in a combinatorial manner to give you the complexity that you see so you can Model     
1:03:27     
A dynamical system using a artificial neural network just fine but it doesn't offer the sort of     
1:03:36     
the functional heterogeneity of a biological neural network and so we we just assume that that functional     
1:03:43     
heterogenity plays some role in producing better     
1:03:49     
outputs you know if you want to think of it that way than artificial neural networks     
1:03:55     
so in biological neural networks we see great heterogeneity if we go back up to figure one we see that there different     
1:04:02     
cell types we have different types of uh structures we have different types of     
1:04:10     
relationships between cells different communication channels heterogeneity is the rule rather than the exception but     
1:04:17     
in artificial neural networks we tend to want to systematize everything and make     
1:04:22     
it homogeneous again we don't talk too much about     
1:04:28     
bioinspired systems and bioinspired systems we might think about this a little bit differently but in this case     
1:04:34     
we're just kind of drawing out this     
1:04:40     
distinction so another difference between biological neural networks and artificial neural networks is this     
1:04:46     
difference between something like a complex brain Network or a complex Network and a network that is sort of     
1:04:54     
organized either hierarchically or by layers and so in artificial networks we     
1:05:01     
have layers and in deep learning networks we have many layers and it's just a matter of adding     
1:05:07     
layers in a complex brain Network however we have we play off of this heterogeneity so we have different     
1:05:14     
processing centers but we also have a lot of interaction between say modules     
1:05:21     
or processing centers or regions and they follow the laws of a complex     
1:05:26     
Network so you see things you see hubs emerge which are centers of processing     
1:05:33     
that can emerge from other types of topological     
1:05:39     
organization use the other types of like sparse connectivity versus Rich     
1:05:44     
connectivity in different regions of the network so you think about the brain     
1:05:49     
some parts of the brain are you know act when when I have a task in front of me some parts of my brain are more active     
1:05:55     
some parts my brain are less active we don't have any of those distinctions in artificial neural     
1:06:05     
networks we also talk about robust function which is of course an offshoot of these other things we've been talking     
1:06:11     
about and we use the example of opponent processes opponent processes being uh     
1:06:16     
theory of function in Vision in mammals and so we talk about that a little bit     
1:06:22     
we also talk about uh fault tolerance and robot function more     
1:06:30     
generally and so this is an example here again of this type of heterogeneity     
1:06:36     
where we have many different types of neuronal morphologies they're all connected together they're connected in     
1:06:42     
ways that aren't sort of strictly in what unidirectional or even hierarchical     
1:06:48     
so then we kind of consider why should we emulate biological neural networks at all obviously we're giving sort of a     
1:06:54     
comparison between between artificial and biological neural networks and you know that that's fine if there's just a     
1:07:01     
simple comparison obviously they're different but perhaps we want to     
1:07:06     
actually not emulate biological neural networks because they might not be the     
1:07:12     
best way to do things we want to do in a computer and that argument certainly     
1:07:18     
exists but of course people have found biological networks attractive they've     
1:07:23     
found the at least the metaphor of biological inspiration or biological     
1:07:30     
sort of the goal biology emulating biology to be the thing to do so that's     
1:07:35     
that's where we're kind of going with this um we know we kind of think of the     
1:07:41     
things that we're trying to do with um you know that we're trying to do with     
1:07:47     
both types of networks so we think about this sort of trade-off between interactive and independent parallelism     
1:07:53     
how can both types of networks make manage that we also think of things like gold directedness as outputs and we     
1:08:00     
think about the information processing capabilities of biological systems versus the biological processing     
1:08:07     
capabilities of artificial systems and one of the I think we talk about this in     
1:08:13     
the paper there's some en energy consumption estimates so for biological     
1:08:18     
networks they can do lot more processing with a lot less energy and of course artificial neural networks are notorious     
1:08:26     
for gobbling up energy and doing Less in biological networks then we have the     
1:08:32     
section on artificial neural networks is a phenotype for data so this is where we     
1:08:38     
might want to process data using an artificial neural network but because it's not embodied we     
1:08:44     
might not have the kinds of you know the kind of power that we have in a     
1:08:49     
biological neural network so one thing to think about is what if we embodied an artificial neural Network we embodied in     
1:08:57     
our official neural network would that allow us to process the data better or you know maybe create a     
1:09:04     
context for the data Maybe create a situation where there's some spatiality or or spatial     
1:09:11     
heterogeneity to the data or at least the analysis it's something to think     
1:09:17     
about so this is an image here figure three where we have this cartoon organism where we embed     
1:09:23     
Ann and it's anatomically Salient has some sort of     
1:09:30     
anatomical uh realism to it and so that's kind of our thought experiment for the rest of the     
1:09:36     
paper and we so that's that's all we have for this um now this is something     
1:09:43     
that we're going to be adding to and I would like to think about this with while with you know what could we kind     
1:09:48     
of work on as a project here what could we kind of sort of follow up on I think he mentioned that he was interested in     
1:09:55     
the energetics question perhaps some of the other questions we can other people     
1:10:00     
can take on if they see fit uh if you're interested let us know     
