## Transcript     
0:00     
yeah so I guess I don't think anyone else is coming for a while yet so we'll     
0:05     
see all right all right so let's get started um     
0:11     
yeah have a couple things to talk about I actually run AC run across the new term for Susan vertex     
0:18     
biology which is where you have the tri Junctions and you know Junctions between     
0:25     
cells so that's it we'll cover some papers on that     
0:30     
oh wow yeah I don't know yeah who coined that term but it's a term now oh really     
0:37     
yeah all right all right so the first thing I want to talk about was this article on that I did for Darwin     
0:45     
Day a review of Carson isation from the biology to     
0:50     
computational models and so what Carson isation is is it's basically the idea     
0:56     
that you get a lot of phenotypes that's sort of default to crab phenotypes so this is uh an image of     
1:04     
different Marine invertebrates and the idea is that this crab     
1:10     
phenotype uh is convergent it you know exhibits convergent evolution over a     
1:15     
number of different uh clades so you see uh the crab things kind of default to     
1:22     
the crab phenotype and that's kind of this phenomena that they talk about in the     
1:28     
literature and what I did was I reviewed not only that phenomenon in the biology     
1:33     
literature or which you know they people have worked out the pH type kind of     
1:39     
defining what carcinization is but also talking about the genetics of     
1:45     
carcinization and some of the genetic mechanisms that are involved and then uh     
1:50     
what's interesting is they also have computational models of Caron uh     
1:56     
carcinization so they have these types of network models that were particularly interesting this is uh a     
2:03     
picture of these developmental Network topologies um from egg to crab phenotype     
2:11     
so they've done here is they've uh started with the egg which in a marine     
2:17     
invertebrate is this neg egg not uh     
2:22     
notus and then there's the zoa and then there's this uh megalopa and then     
2:28     
there's this crab phenotypes so you have these transitional phenotypes uh from     
2:34     
the egg through development and you see that this is based on the phenotype and you can see that there are these     
2:40     
different typologies that emerge over different life stages uh we also     
2:47     
have uh well this is the part about Gene regulatory networks and then that leads     
2:52     
into a discussion on epigenetic landscapes and how we can use epigenetic     
2:57     
Landscapes to understand the evolution of these uh     
3:03     
phenotypes leads to you know a few other computational models in the literature     
3:08     
where people are looking at this genotype to phenotype mapping and how that genotype to phenotype mapping is     
3:15     
convergent between many different genotypes to the same phenotype and so this is an interesting     
3:22     
article because like actually got through a lot of different references here uh both you know computational     
3:30     
references uh biological references and genomic     
3:36     
references just a blog post uh but yeah it's next thing I wanted to point out     
3:42     
was that uh we have our dvo learn repository so for those of you interested in Google     
3:50     
summer of code we have our dvo learn repository we have uh I think we talked     
3:56     
about this last time uh we have the dvo learn uh program which is something that we     
4:02     
worked on in Google summer of code from 2020 to I think two years ago so you     
4:10     
know I've been we've been kind of uh moving back and forth between D graph and dvo learn and sometimes people have     
4:17     
had to add to dvo learn to get their things in D graph to work so dvo learn     
4:24     
is a series of pre-trained models a series of deep learning models that allow people to to segment microscopy     
4:31     
images so if you have a microscopy image data set that you're interested in segmenting we can segment this with the     
4:39     
dvo learn uh program and include it in the same pipeline as the dvo graph     
4:46     
uh program and so this is sort of what was at one time called the stage one of     
4:52     
dor graph so if you see references to stage one in the documentation that's     
4:57     
what this refers to so so this is something it's been released on pipe we     
5:03     
have a number of different models here to work with those are also on uh     
5:08     
hugging face so we have yeah we so we have a bunch of different so we have our     
5:15     
links to the hugging face spaces we have the uh segment the cell membrane and     
5:21     
seigan embryo oh I'm sorry so we have the links to the hugging face spaces we     
5:27     
have segmenting the cell membrane and Cal an embryo we have the cell membrane segmenter for that segmenting the     
5:34     
nucleus and seans embryo we have the seans nucleus segmentor for that and then the cell lineage     
5:41     
population prediction we have the lineage population model for that so we     
5:47     
have a segmentor a membrane segmentor a nucleus segmentor and a lineage     
5:52     
population model and so those three U models will allow you to extract data     
6:00     
from an image file and produce an     
6:06     
outcome and so if you want to contribute to this you know that's open um open to     
6:13     
contributions and you may need to do that to do things with evog graph now the state-ofthe-art on this of     
6:21     
course is the segment anything model So Meta segment anything model um it's you     
6:27     
know been out for about two years uh we've kind of for we I think the way     
6:33     
this worked is we forked a biological version of the segment anything model     
6:39     
and we've kind of refined it for our own needs so what we're doing in a lot of the things that we've tested it with and     
6:45     
benchmarked it with we've done things at the celegans embryo um now we'd like to     
6:50     
move to other embryos but the problem is is that you know these are sometimes     
6:56     
different very different modes of development so if we train or pre-train a model on the celegans embryo you know     
7:04     
we'll get certain results and we have an an annotation of each cell but if we move to say like a dropa     
7:12     
embryo or some other embryo where we don't know the cellage or it's not an     
7:17     
exact cellage as we have in celegans then we run into some problems of     
7:23     
application and accuracy so you know we've been kind of focused on C     
7:30     
if people are interested in contributing to this package in a way so that we can     
7:35     
do other types other embryos other types of Developmental models you're welcome     
7:41     
to do that but that's not really the focus of the gso     
7:47     
project so that's dvo learn um and you can dive into that if you'd like we have     
7:54     
a repository for sellam which is the uh the biological instance of the segment     
8:00     
anything model this is our Fork here and we just you know if you want to just use     
8:06     
celam that's fine too we can um you know I think that's kind of uh sort of maybe less C elegans Focus     
8:17     
than I mean it's a more General model than DVA learn at this point but you     
8:22     
know this is where we're draw we kind of drawn from this Fork of celsian     
8:30     
and then of course we have our different uh models here nucleus segment or uh membrane segment are in     
8:38     
lineage models those are also on our GitHub but also on hugging face as well     
8:43     
so this is the D graph repository this is the focus of this year's Google     
8:48     
summer of code we have a number of different uh directories here we have     
8:56     
the two projects growing GNN and hyper Devo graph growing GNN is the one where     
9:03     
we've uh forked uh the neural developmental programs     
9:08     
repository and we've also forked some other repositories and topological data     
9:14     
analysis and we're building these uh embryos that grow in terms of the cells     
9:21     
and the cells that are visualized so this is what your growing graph looks like you have the nodes you have the     
9:27     
edges the vertices the edges they're labeled according to sort of the seigan     
9:33     
lineage tree and as the network grows in size it gets more dense and you can see     
9:40     
that it's plotted in this three-dimensional space that shifts over time so that's what we're trying to     
9:46     
produce we're trying to produce this model of the embryo as the cells divide     
9:53     
and this network Row in size and you know we can play with this network so that we only have the cells from     
10:00     
one uh one uh stage of development where we can have these graphs usually the way     
10:08     
that we plot them here is where all the cells subsequent or all the cells uh     
10:14     
prior to the current stage are plotted and that's just to show where those uh     
10:21     
centroids were in the embryo space so that's what this is so this is something     
10:27     
that we've been working on um this repository growing GNN should give you     
10:32     
some detail this is uh Pocky Balia who was the person who did this last year     
10:39     
and so we have some nice V uh documentation for this uh so we're basically combining temporal growth with     
10:46     
spatial learning and that's what we're doing here and this is again applied to     
10:52     
se alligan data so we can look at this for other embryos we can look at this for other systems but for right now kind     
10:59     
of specialized to celegans so there are some details to work out with respect to celegans data I think Pocky only got she     
11:07     
only got to maybe about the 128 cell stage they're actually uh     
11:14     
660 some cells at hatch so the worm grows the embryo grows until the egg     
11:22     
hatches and you have many so you have many more cells by the time the embryo     
11:28     
hatches then you do at the stage of which we have a representation here so     
11:33     
we need to work on that latter part of development which is harder because it has more cells and that may not be     
11:40     
possible because we don't have access to really high powered     
11:46     
Computing if people one one possibility is to improve the code so it'll work for     
11:54     
larger data sets more cheaply that's a possibility it's a we have a fork of the     
12:00     
original repository um you know another possibility is to uh try to visualize     
12:08     
some of the things that we see in early development so there are a lot of possibilities with uh the growing gnn's     
12:15     
part of this uh then we have the hyper Divo graph which is this     
12:21     
um where we're using hypergraphs to model cagan's development this is mahul     
12:27     
Aurora and I know a number of people already expressed interest in this this is where we build these hyper graphs     
12:34     
which are graphs within graphs so you have hyper nodes which contain regular     
12:41     
nodes you have hyper edges which contain regular edges and you can uh build these     
12:50     
multiscale representations so this is something that uh mahul has worked out the details     
12:56     
on and we've produced some nice     
13:03     
visualizations so these visualizations kind of look like this where you have     
13:09     
you know uh we have the cell nomenclature we have the cells represented by a hyper node and then     
13:17     
there's this radius uh by which you know we include all the cells in that radius it's kind     
13:23     
of hard to see the graph part of this but these uh these local overlaps and     
13:30     
these concentric circles are basically the edges and the hyper edges so we need     
13:36     
to work on the visualizations we need to work on how I mean Mah's worked a lot     
13:42     
done a lot of work on creating metrics graph metrics and     
13:48     
other types of metrics trying to get into some benchmarking and then the visualizations but I think the work that     
13:55     
can be done of course is making the you know I think writing you know working     
14:01     
with the code making it more efficient uh the visualizations if we can work on those some more and also     
14:09     
kind of benchmarking this against other I think M mentioned that we need some better benchmarks and so we you know we     
14:16     
can work on that that's something you can propose finally the thing I'd like to     
14:21     
talk about with respect to dor graph is that we have you if you explore     
14:29     
this repository you'll see that there are these different stages so you have stage     
14:34     
one stage two and stage three so this is kind of     
14:41     
old but it's still relevant stage one is kind of already been done but you'll     
14:50     
find a lot of things here with reference to uh image processing and a lot of     
14:55     
connections with dvo learn so that's why I talked about dvo learn so basically the tools of devarn the tools of segment     
15:02     
anything are stage one stage two is this     
15:07     
uh this visualization step creating the networks um you know this is something     
15:14     
that we did with topological data analysis there's a tool K mapper which     
15:20     
has been used in years past that we can um use as a visualization tool and then     
15:27     
stage three is kind of moving towards hyper graphs and just there's a TDA     
15:32     
folder here we haven't really I haven't updated this to reflect kind of how     
15:38     
hyperdog graph and the growing graph neural networks fits into stage three     
15:43     
but um that's basically it it's like that last stage at most complex     
15:49     
stage of the project so yeah this is basically applications of graph neural     
15:55     
networks to developmental biology and so there're a lot of ways people can contribute to that if we go to our slack     
16:02     
Channel I noticed that Morgan Huff has contributed this     
16:07     
um YouTube video hyperg grass rewriting for well-defined geometries so this is     
16:13     
very much in the spirit of what we were do what we're doing in dor graph uh take     
16:18     
a look at that we also have some readings and uh some videos on graph     
16:25     
neural networks in the slack openworm slack Channel so if you join the openworm foundation     
16:32     
slack and I know a lot of people who are coming through neurostars for Google     
16:38     
Sumer code they're encountering this link to the openworm slack and so if you     
16:44     
join that if you click on that and get an invitation um you will get access to     
16:50     
this slack and it has a lot of um you know of course our meetings and then     
16:56     
some of these other things I wanted to point out that Hussein Aether has Fork     
17:02     
two of the repositories in our dvo or dvo worm uh GitHub so we have an A D.A     
17:10     
worm GitHub that's separate from the D.A learn GitHub and the D.A worm GitHub is just kind of Legacy projects Legacy     
17:19     
repositories a fork to the digital Bassel area project which we haven't talked about in a while and that is a     
17:27     
digital trying to rep Digital Way represent uh A A genus of datom called     
17:35     
basaria which is this cool uh Colonial organism that moves in a very unique way     
17:43     
so if you're interested in that you know we also we that's not part of this year's Google summer of code but I am     
17:50     
glad to see that he forked that um so if you're interested in that let us know it's just kind of one of those projects     
17:57     
that hasn't been touched in a while um and then we have this other repository     
18:02     
theoretical types of embryo developmental networks that's kind of linked to the daph stuff it's a link to     
18:10     
some of the stuff I I showed in the blog post on uh embryo networks where you     
18:16     
take the centroids of cells in an embryo and you look at how they're connected by some Criterion either by distance or by     
18:25     
um some signaling Criterion or something else so congratulations to Hussein and organ     
18:33     
and uh anyone who's applying to Google summer code this     
18:38     
year so I think the descriptions are now live on neurostars so if you go to neuro     
18:44     
stars.org and you go to the gach uh     
18:49     
topic you'll find these projects uh or the graph neural networks project listed     
18:57     
and so go through that uh leave a comment if you'd like um or     
19:02     
attend one of the meetings live or raise a comment in the slack it should be     
19:09     
pretty good um we should have a pretty fun summer at least you know uh in terms     
19:15     
of thinking about these problems don't have a question I'm I got     
19:21     
my diploma in the mail and I'm graduated oh     
19:27     
graduations yeah I don't it's good     
19:33     
yeah     
19:40     
yeah I don't know I'm still working on things yeah I have a my thesis     
19:49     
presentation is going to uh be presented in APS um March meeting on the 19th in the     
19:58     
evening and they put me in the evening section section I don't know if they     
20:04     
knew what to do with me but anyways uh it's a 10minute     
20:10     
presentation okay well good luck with that is that just kind of a summary of your thesis then you're gonna like     
20:16     
condense that yeah yeah great so that and doing that I figure um     
20:27     
I'm Master's thesis one should write a paper about it and one should present it so that's my presentation and then     
20:35     
um I'm writing a     
20:43     
paper you're working on the paper now yeah um the direction Dr Zang wants     
20:52     
to go with it in the direction I want to go with it are two kind of diverging     
20:57     
problems so so um he he thinks he wanted     
21:03     
to um well I thought maybe we can make a     
21:09     
summary of of what type of morphology cells have like how they're connected     
21:16     
together but it's patterned and it's different for every tissue and that gets     
21:23     
into um another thesis perhaps yeah     
21:29     
so I I'm not sure I can do that part of it but maybe I just I should just do a     
21:36     
summary of my thesis and that's it     
21:41     
yeah well yeah there are a lot of kind of potential questions     
21:47     
there and of course like applying the 10 seg method you know people are going to     
21:53     
ask you about different questions and then you apply it to different questions     
21:58     
that risk of kind of using it as a hammer rubbery nail but you could probably find a good     
22:05     
set of things to work on now you can apply it where it     
22:12     
shouldn't be well yeah yeah um I can     
22:17     
take my model and make it um well what I call it's K Factor so     
22:23     
extreme that maybe it wouldn't ever exist in a cell and and then apply it to     
22:30     
uh well I did apply it to skin it kind of matches skin but um     
22:39     
really what it needs is a larger model like instead of in in uh one     
22:45     
micrometer two micrometers or three micrometers it needs to be     
22:51     
um like whatever skin is like how thick is that like 20 micrometers and need to     
22:58     
be a a larger model yeah it's still extreme because um it's     
23:08     
a contrast between um the bars and strings in the 10 segy     
23:13     
model so the elasticity has to be very different between the bars and     
23:21     
the strings in the model for it to have a different slope and     
23:26     
area of where it work and some of that just a little extreme I     
23:33     
don't think it's actually exists so you can make a model ites     
23:39     
doesn't actually exist in real life and apply it to um     
23:47     
tissue I don't know I don't know that's not wasn't the idea I wanted to make it     
23:53     
kind of a real life model like something that has actual     
23:59     
um foundation in reality that's what we hope for yeah     
24:07     
yeah oh yeah that's good and you know I was I was kind of wondering about the     
24:13     
computational power you might need for a larger model so is that something you think is tractable or do you think it's     
24:19     
something that like you would have to struggle with it's possible like my     
24:24     
model solves in seconds so you could probably get you can make     
24:31     
the lot larger and it would still work should be able to except it's a     
24:39     
tagr so it all has to fit together and     
24:45     
work and uh finite element analysis tends to not work with t sries because     
24:53     
it's not stable enough you'd have to change the programming for the finite     
25:00     
element um program code you'd have to change the     
25:06     
code yeah so I don't know how to do     
25:14     
that yeah it seems like a whole ordeal yeah I I haven't looked at it I     
25:22     
went to a conference on 10 segr and I have uh a lot of people's work work with     
25:28     
it and they were programming in Mathematica for one thing and and in mat     
25:36     
lab and python so I haven't looked at that     
25:44     
yet yeah good luck yeah I haven't seen any like Matha well     
25:53     
I haven't seen any python models for 10 SEC it doesn't mean that someone isn't trying Tri     
25:59     
it well they tried a bunch of things they were um did code in another     
26:06     
finite element analysis program it wasn't Anis it was something     
26:11     
else but people have programmed in something called Anis the finite element analysis     
26:19     
uh um no cul's finite element analysis and     
26:27     
and uh so's Anis and so's this other program they were     
26:35     
using uh dick said uh search free supercomput     
26:40     
access Yeah well yeah if I made     
26:46     
the model a thousand cells um large then     
26:52     
maybe I would need one but it's not um     
27:00     
with five or seven or 14 cells it's not gonna it'll just solve on a regular     
27:07     
computer yeah yeah yeah the way we usually do it here     
27:14     
is like with uh interactive notebooks like U different types of collab     
27:21     
notebooks and you get like computational power like for free from the from the     
27:29     
Cloud Server I mean there's a limit to what they offer for free and you have to start paying but that's usually though U     
27:36     
those kind of notebooks compile Python and maybe some other languages but specialized languages maybe not so that     
27:43     
doesn't help that's what I was asking about python because that's more of a standard where you can get like tool     
27:50     
boxes and things like that yeah it's really easy to convert from mat lab to     
27:58     
python they're very similar very similar     
28:03     
codes same structure yeah yeah I don't know if I want to get into     
28:10     
it or not I it's similar but not yeah you have to still have to convert it     
28:17     
yeah yeah time     
28:22     
consuming good Lu yeah yeah I have a Udi uh course on on     
28:30     
neural networks and um he's actually studying the brain and the signals from     
28:36     
the brain and his code he does it both in Python and and mat lab they're just     
28:44     
they're just very similar yeah yeah they     
28:49     
are yeah it's uh but definitely it's hard to get them to like     
28:56     
uh sometimes it's hard to convert things because python is a more formal structure or like especially if you get     
29:05     
deep into a program okay well I I don't know python     
29:11     
I I've learned too many programming languages in in my life so I'm     
29:17     
resisting yeah I have to know 10 why why do I want to know another one but I     
29:23     
guess python would be might be worthwhile yeah     
29:28     
yeah like I don't do anything with my computer programming yeah I'm I'm good     
29:34     
at it according to my marks and and what I do in class and then I go out in the     
29:39     
real world and they don't want me to do use it for anything so like is it useless it's just a hobby     
29:46     
like just just like a few other things computer programming is a hobby yeah     
29:53     
dick said you could use AI to convert it but I don't know well sometimes well     
29:58     
yeah apparently there there is some AI to convert that but     
30:05     
it it's so so similar at least in the code I've seen that you can just go Y     
30:12     
and just change that [Music] yeah okay well thanks     
30:21     
yeah I don't know all right let's move on to um the     
30:28     
protect biology I was talking okay so there are a couple of papers here I'm going to cover     
30:35     
um we have four of them here let's see so we'll start here um this is a paper uh by sugur     
30:46     
Otani vertex remodeling during epithel morphogenesis so this is kind of uh     
30:54     
where they talk about vertex remodeling so this is where the vert Es are moving and changing their topology with respect     
31:02     
to an epithelial sheet so um well we'll go through the abstract and then maybe     
31:09     
down through some of the details here so the abstract uh is here     
31:14     
epithelial cells adhere to each other by intracellular intercellular Junctions     
31:20     
that can be classified into bellular Junctions and     
31:26     
tricellular Contex s or vertices so you get these bicellular Junctions two cells     
31:33     
you these tricellular contacts which are end up being vertices in a     
31:39     
graph and so they talk about this actually down in this part of the paper here so we'll get to that     
31:45     
next epithelial morphogenesis involves cell rearrangement and requires     
31:51     
remodeling of bellular Junctions and vertices so morphogenesis in the     
31:56     
epithelial sheet so the cells move around they move their they change their shape they change     
32:02     
their contact which is important and then they remodel these Junctions and vertices as they do     
32:09     
so although we're understanding of how B how bicellular Junction mechanics Drive     
32:15     
epithelial morphogenesis has advanced so our understanding is is you know we're     
32:20     
getting to the point where we're giving it a name vertex biology the mechanisms underlying vertex remodeling during this     
32:27     
process have only received attention recently in this review We outline recent progress     
32:33     
and our understanding of how cells reorganized C adhesion and the cytoskeleton to trigger the displacement     
32:41     
and resolution of these vertices so there's cell adhesion there's     
32:46     
reorganization of the cytoskeleton as we know and then that triggers this     
32:51     
displacement and resolution of these vertices in the sheet we will also discuss how cells achieve the optimal     
32:58     
balance between the structural flexibility and stability of their vertices so there's this you know this     
33:05     
trade-off between being able to have these vertices that reconfigure and then having vertices     
33:11     
that are stable so if you have a t segr structure then you know we can maintain that 10 segr structure and it doesn't     
33:18     
move away from that configuration finally we introduce new modeling Frameworks designed to analyze mechanics     
33:25     
at cell vertices so they're actually building a formal model integration of lime Imaging and modeling techniques is     
33:32     
providing new insight into the active roles of cell vertices during epithelial     
33:38     
morphogenesis so this part here this paragraph is interesting right here um     
33:44     
cell cell Junctions between epithelial cells are classified into bicellular Junctions where two cells adhere to each     
33:52     
other so this is where we have this adhere U adhesion principle um and then     
33:58     
tricellular Junctions or tcjs and multicellular rosettes where three or more cells meet     
34:06     
so these are these Junctions that Susan's talked about where you have multiple cells meeting at one point the     
34:13     
term tcjs has historically been used to refer to tricellular uding Junctions so     
34:19     
this is where you have Tri Junctions but they olude one another where the cells are all kind of working against one     
34:27     
another they're including one another so that's a diff kind of a different Arrangement uh however recent     
34:33     
discoveries of the tricellular adherence Junctions or tjs and invertebrates raise     
34:40     
ambiguity in the term of tcjs so we have these including Junctions which are a     
34:46     
broader class of these tricol Junctions we also have these adherence Junctions     
34:52     
that exist in invertebrates and so there's this sort of um ambiguity in the term and so I     
35:00     
think you know as usual cross biology uh complexity is intervening in our     
35:06     
classification systems and you know people have to kind of rethink things a bit to get a better handle on what we're     
35:14     
dealing with in this review refer refer to tricellular contacts in general as     
35:21     
vertices and junctional specializations at vertices as tcjs so this is where this is their     
35:28     
definition tricellular contacts in as vertices and then junctional     
35:34     
specializations at vertices when distinct junctional structures can be re recognized by electron microscopy or     
35:41     
when the structure contains unique molecular components the vertices are referred to as tricellular tight     
35:48     
junctions so I think Susan also talked about tight junctions um and sort of the role of     
35:55     
that in epithelial sheets tricol or separate Junctions and depending on the ultra     
36:02     
structural feature and the molecular composition you can have either one or both actually it's tricellular septate     
36:10     
Junctions um tricellular contacts are also observed in maen kimal cells     
36:17     
although they likely lack distinct junctional specializations in ultr structural     
36:23     
levels um I I ran across an interesting um     
36:29     
note along the way uh somewhere where they said that there adherence Junctions     
36:35     
in between um there's adhering Junctions and tight junctions and they're kind of     
36:41     
mixed together and um when the cells are     
36:47     
hatching and like mamalian cells are hatching out of the zonal palua that the     
36:53     
adherence Junctions um don't exist like they dissolve     
36:59     
okay so that's an interesting thing because um that's part of how cancer me     
37:08     
metastasizes as well like they um don't go through a small opening and I wanted     
37:16     
to tell dick this but he's occupied anyway so yeah they um I don't I went     
37:23     
and lost the reference but but the adherent uh Junctions dissolve for so the cells     
37:30     
can um go through where the tissue can     
37:36     
move yeah to small space yeah yeah so so there these the     
37:44     
tri Junctions are important anyway yeah yeah all right     
37:51     
yeah thanks so the rest of the paper is uh this is a figure here     
37:57     
this shows the structure of a tricellular junction where you have some tissue deformation this supposed to be a     
38:04     
tissue sheet a you have the cell shape change where you get these Junctions you     
38:11     
see the white     
38:16     
dots and those are the Junctions here the other cells are coming converging on     
38:21     
this red cell the white dots are these Junctions so where they meet together     
38:27     
the EDG of the cells so if you think about it like um you know like a tiling     
38:32     
in mathematics where you have cells that are kind of tiled together and you have to see how they fit together without     
38:39     
huge gaps in between that's what these are and then the Junctions are just the the points at the vertices of the     
38:47     
network and so the salt shape can change of course it can be squeezed in     
38:53     
different ways and those Junction Arrangements can kind of move relative to one another and then these cell     
38:59     
arrangements are of course important in multicellular organisms where you have these different stable configuration so     
39:06     
here we have our cells represented as hexagons and then the Junctions here and     
39:12     
then this is where the cell kind of moved these two cells on     
39:18     
the uh vertical axis kind of move together     
39:24     
they form a Jun a different Junction and then there's this rotation of this sheet here     
39:30     
and that forms a different orientation of Junctions so there there are also um     
39:36     
five seven instead of hexagons that are sided there's also five seven     
39:43     
configurations and they're more fluid okay anyway they still have Tri     
39:50     
Junctions yeah all right uh then we have two     
39:56     
examples here in vertebrates and invertebrates so they mention that invertebrates you have these different     
40:03     
specialized types of adherence Junctions right so you have uh the well adherence     
40:10     
and then what was the other one um the SJ so we have the AJ and the SJ     
40:17     
and so we also have the TJ so the tight junctions theher Junctions and then SJ     
40:23     
and so these are all kind of specialized in these uh sheets so invertebrates the     
40:28     
mature epithelium has uh these different layers of adherence Junctions and then     
40:34     
SJ and then uh we have invertebrates we have the uh Tri Junctions here and then     
40:41     
the adherence Junctions below so we're looking from apical to Basil in the sheet you see that there are these     
40:48     
different types of configurations as one moves down the sheet and invertebrates     
40:54     
and invertebrates and this is mature epithelium so that Arrangement is very     
40:59     
different in invertebrates versus vertebrates so that's that's basically     
41:06     
the idea and then this just shows like a tri Junction and this is the actual     
41:12     
Junction that you see here in this micrograph it's like right here and then     
41:19     
that's this cartoon where you see the actin bundle kind of along the edge of     
41:24     
the junction so it follows the B B between the cells and then kind of skirts The Junction in different ways so     
41:31     
there these actin bundles that attach to the wall the membrane and to the     
41:36     
intersection of the of the cells and then you have the tight Junction here or the tri Junction here and so you can see     
41:44     
that acting plays a role in kind of configuring these     
41:50     
Junctions did we have anything on that Susan um I tried to make a model like     
41:56     
that with with um the acting bundles around the corner like that and it fell down or     
42:05     
something okay yeah it it didn't work but then um a standard T     
42:12     
segr um it's kind of in a star shape also never     
42:18     
works in a console here we go     
42:23     
again um yeah uh     
42:29     
yeah I have trouble with my video um anyway so again finite element     
42:38     
analysis does not work well with 10 segres and it need a     
42:43     
different boundary conditions or something because it doesn't fall down     
42:49     
it is merely changing configuration because you put little     
42:54     
finite forces on it until one of the strings becomes loose     
43:00     
for whatever reason in the configuration and then it will all of a sudden change     
43:07     
shape and it has not fallen down and broken it is merely change shape like T     
43:14     
segres do and so that has to be programmed into your finite element     
43:20     
analysis or is going to be very difficult to do T hyr in finite element     
43:26     
in analysis yeah so that's I've nearly     
43:32     
convinced myself that I should do some work in Moro which is out of TS or     
43:39     
Boston um where they've they've got um a     
43:44     
program but it's um I don't know it it's that line code     
43:51     
program um okay I I'm forgetting what it is um     
44:01     
yeah a programming language that I don't know all right so there are a couple     
44:06     
other papers here that's that paper I'll I'll send all these papers out after the     
44:13     
meeting this is another paper tricellular Junctions how to build     
44:18     
Junctions at the trickiest points of osis trick is TR and that's a     
44:27     
kind of an acronym so they're being cute here and uh so this is I guess their     
44:33     
model and then uh this is higashi and Miller um I think at University of     
44:40     
Michigan so in this case uh what they're looking at is they're looking at these tricellular contacts these are places     
44:47     
where three cells meet like we just saw in the last paper invertebrate epithelial cells and we talked about     
44:54     
that there's a difference between vertebrate and invertebrate aial cells specialized structures called     
45:00     
tricellular tight junctions or ttjs we saw that graph where we saw the role of     
45:06     
tight junctions of the sort of the apical part of the sheet um and tricellular adherence     
45:13     
Junctions which are lower down tjs have been identified ttjs are important for the     
45:20     
maintenance of barrier function and disruption of ttj proteins contributes     
45:25     
to familial deafness uh this is in in a phenotype so I guess this is in the ear where they're looking     
45:32     
at this um thas have recently been attracting the attention of mechanobiology because these sites are     
45:39     
hotpots for epithelial tension so this is where you have tension in the epithelial sheet and     
45:47     
that's kind of the place where that originates um uh and in the I guess in     
45:53     
the inner ear that would be pretty disruptive if you could um you know differences in epithelial tension     
46:01     
although in the molecular components regulation and function of ttjs and TS as well as of invertebrate     
46:07     
tricellular Junction so they're talking about uh vertebrae uh structures in this     
46:14     
first example and then the invertebrate tricellular Junctions um are also being     
46:20     
characterized as we saw in the figure many questions remain here we broadly     
46:25     
cover what is known about tricell cular Junctions propose a new model for tension     
46:30     
transmission at adher Junctions and discuss     
46:36     
keyop that's the way I made my model with the tri Junctions connecting the     
46:42     
cells together um bearing the mechanics the mechanical     
46:49     
load yeah maybe that's why I got interesting results see that that's in my thesis     
46:56     
that was diagram here here uh B I'm put that inis yeah I put that in my thesis     
47:04     
okay good so this yeah this is h a this is uh     
47:10     
a picture of kind of these Junctions using angul one one is the probe as the     
47:16     
stain we have Z1 which shows the membranes so we have the dots in angle     
47:23     
represented by angular one one Z1 represents the edges of the membrane you can see that the dots are just kind of     
47:29     
the intersections of these membrane edges and then merging that you see the the Junctions with in relation to the     
47:38     
edges of membranes and then this is Susan's favorite um where we have the     
47:44     
this is in 3D so we have the cells you know the tops of the cells but also the depth of the cells to see these     
47:51     
Junctions kind of how they sit at the you know they they sit if you see this image here kind of sits at the top of     
47:58     
the sheet but there's some depth here as well so the Junctions affect the depth     
48:03     
along these these axes so this is you know the the apical the basil axis     
48:09     
basically where these Junctions are acting so the cells can twist and bend underneath of course and we can have     
48:16     
these occlusions where cells will kind of move over another and so you have to     
48:22     
think about that when you think about this relationship and then we have the uh     
48:28     
tricellular Junction components we have tricellulin which is a     
48:33     
proteinin one two and three and those are all different proteins um we then have this this     
48:41     
section of the three dimensional tricellular Junction blown up we have     
48:47     
invertebrates invertebrates invertebrates so in in vertebrates and invertebrates of course we have     
48:53     
different structural components uh where we have PGA AG which are the     
49:01     
adherence Junctions the tricellular Junctions closer to the top there uh and     
49:06     
then showing that in vertebrates where we have ecad complexes and these TJ     
49:12     
strands and these are the like the actin that we saw in the last paper and then     
49:17     
this is SJ strands here and EAD complexes on top in the invertebrates so     
49:24     
they don't have actin in the same way that vertebrates do and it's just interesting how that they have the same     
49:31     
Junctions but the molecular component underneath is different and then this figure f and g     
49:38     
are where we have a model of tricellular     
49:44     
Junction organization in F so this just shows the angul and the tricellulin as     
49:50     
components of this geometry and then in G we have model of     
49:55     
uh tsj organization where we have anaconda and gliotactin is different molecules in     
50:04     
this organization so you can see that these different types of models kind of     
50:09     
propose different ways in which the structural Integrity is maintained of these     
50:16     
Junctions okay um and then this this figure is where     
50:23     
they kind of look at how tricellular Junctions are formed so they're formed     
50:28     
kind of from cell divisions in a you see a cell division where you get the Single     
50:34     
Cell triol Junctions the cell divides and you get these different structures     
50:41     
that occur at the boundary of the membrane and then you get these new Junctions that form so the cells are     
50:48     
always migrating but they're always div also always dividing and or they're dividing in some cases and so you get     
50:55     
these different opportunities for new Junctions to form for old Junctions to     
51:01     
sort of rearrange themselves and so that's and then we have an example from convergent extension here where a sheet     
51:08     
of epithelial cells stretches out along one axis and so convergent extension you     
51:14     
can see that like these different colored cells that are kind of arranged     
51:19     
in a parallel fashion then inter sort of interdigitate here during convergent     
51:25     
extension and they these Tri Junctions in different places so you can see     
51:30     
that's you know this is a component of morphogenesis where you get this rearrangement of the U of the epithelial     
51:39     
sheet and in a way that kind of reconfigures these tricol Junctions and     
51:45     
in a way that kind of changes the function of the tissue eventually okay so that's that paper um     
51:53     
next is this paper here     
51:58     
uh this is a figure um this is the one that we just saw in the last     
52:05     
paper and then this paper here which is the last example uh this is called tra this is     
52:12     
from the archive trajectory geometry suggests Sate decisions involve branches     
52:19     
rather than bifurcations so I'll just do the abstract on this and then we can talk about this and that'll be um the     
52:27     
you know I'll send these papers out and we can talk about them maybe next week as well so trajectory geometry is the name     
52:36     
of their program I don't want you to think that that's a typo um but they so     
52:42     
the abstract reads differentiation of multi- potential progenitor cells is a     
52:48     
key process in the development of any multicellular organism and often continues throughout its life it is     
52:54     
often assumed that a bip potential progenitor develops along a relatively straight trajectory until it reaches a     
53:01     
decision point where the trajectory bifurcates so we have these uh cells     
53:08     
that are differentiating and they can differentiate into different types of cell we have these progenitor cells and     
53:15     
then we have these cells at different Fates so a progenitor cell can differentiate into different     
53:21     
Fates and it can um you know this is something that we want to mon     
53:27     
alongside this sort of um you know these sheets these cellular sheets in their     
53:34     
configuration um it is often assumed that a bip potential progenitor develops along this relatively straight     
53:40     
trajectory at this point a one of two directions is chosen each Direction     
53:45     
representing the unfolding of a new transcriptomic program so we have these transcriptomic programs that turn on as     
53:53     
these uh different Fates are selected these different intermediate Fates are     
53:59     
selected however we lack a quantitative means for testing this model accordingly     
54:04     
we have developed the r package trajectory geometry applying this to publish data we find several examples     
54:11     
where rather than bifurcate Developmental Pathways Branch so this is     
54:16     
where I don't know what the difference is necessarily between bifurcation and branch that is the bip potential     
54:23     
progenitor develops a relatively straight traj trory leading to one of the potential Fates a second relatively     
54:30     
straight trajectory branches off from this towards the other potential fate in this sense only cells that Branch off to     
54:36     
follow the second trajectory make a quote unquote decision our methods give     
54:41     
precise descriptions of the genes and cellular Pathways involved in these trajectories we speculate that branching     
54:48     
may be the more more common behavior and may have advantages from a control theoretic Viewpoint so this is a little     
54:55     
bit different than talking about the sheet itself this is talking about how cells go you know from a progenitor cell     
55:02     
to a differentiated State and the multiple differentiated states that a lot of cells will undergo especially in     
55:10     
mammals um and so they're talking about these lineage decisions and the anic     
55:17     
nervous system and this is uh in this system uh We've uncovered a novel     
55:23     
configuration of differentiation trajectories so this is where we have this bifurcation model which is where uh cell     
55:32     
develops towards the chosen fate once a fate is chosen so a typical model of     
55:37     
sell fate decisions is as follows a multi- potential progenitor cell develops until it has reached the     
55:44     
decision Point here the cell chooses one of two possible fates Each of which requires the initiation of a new     
55:50     
developmental program so this is the bifurcation model then the uh     
55:56     
this is the the uh this other model here is this     
56:02     
um where you have this these trajectories in contrast to the     
56:08     
bifurcation model anic leog Genesis forms a default linear pathway of     
56:14     
progenitor maturation from which neurogenic trajectories Branch off during     
56:19     
embryogenesis a consequence of this branching configuration is that there are no identifiable points of commitment     
56:26     
along the gly gly ogenic trajectory and is only the cells which become neurons     
56:32     
that ever make a decision and initiate a new developmental program so from rather     
56:37     
than from a single Branch point there seems to be a region along the default trajectory where branching can take     
56:44     
place so this is kind of an example of this in figure one where they use     
56:50     
synthetic data to show these different vication and branching models so this is     
56:56     
where we have an A a simplified model of branching Behavior which is where we     
57:03     
have a cell that makes a decision to go into to uh differentiate into one of two     
57:09     
Sul Fates B is a simplified version of branching behavior is a model of Sate     
57:15     
decision and then C this is where bip potential cells proceed along a de Vault     
57:21     
Pathway to one of the potential outcomes so this is where they don't make it decision they just kind of move either     
57:28     
to the blue or to the green so there's in this a there's a decision being made at this Branch point in B you just have     
57:36     
this deterministic process I guess where they either follow one pathway or another and then in C we have branching     
57:44     
Behavior where we have this region rather than a single point which cells choose to leave the default pathway so     
57:52     
you have these four points here where you have uh you know decisions that are     
57:58     
made between the blue and the     
58:03     
green okay so it looks like uh we have new people here so uh     
58:11     
welcome uh so any questions about those     
58:16     
papers Susan just posted uh     
58:22     
reference it's an older reference but it was interesting because uh the um joining of the three cells or     
58:31     
two cells or whatever is usually depicted at the apex of the cell and     
58:39     
this um model shows or the um the paper here was showing that     
58:47     
on a stiff surface the uh join between the cells is actually at the Basils     
58:55     
surface or near the basal surface instead so I I thought that was interesting it's also something I could     
59:02     
try with my model yeah yeah that would be good yeah     
59:07     
so I think there's a lot of potential in like looking at the geometry looking at where different places in the     
59:14     
three-dimensional geometry where those things happen so it probably a huge difference I would     
59:20     
imagine well I I'd like to know more biology like more papers on where these     
59:28     
things are and their strength and for instance instance the strength of the     
59:34     
ACT bundles like what is its elasticity and they're just the     
59:40     
literature isn't there yeah you can see that like they have to use kind of a lot of diagrams     
59:47     
and a lot of synthetic data to get the the parameters aren't really     
59:52     
available that's true I think in a lot of biology like if you're trying to do computational biology you know something     
1:00:00     
like that you want to find a parameter value and you can't because it just doesn't exist someone did the     
1:00:07     
experiment um they almost describe um what you need but it's it's missing     
1:00:15     
like for my model I needed um a was attached by a point for instance and in     
1:00:23     
the biology papers I was getting uh attachment over an area so you go     
1:00:29     
throw your hands in the air and you go I don't     
1:00:35     
know the data is not available and it's just off by just I I think the     
1:00:42     
biologists need to look at some of the engineering terms that Engineers have used for     
1:00:50     
centuries to describe things that are physical yeah yeah just a comment     
1:00:57     
well yeah those measures already exist but yeah I think in biology there you know sometimes people aren't interested     
1:01:03     
in those questions maybe it's a different class of questions they're interested in like the you know that     
1:01:08     
well there's a con convergence in like mechanic biology which we saw in the     
1:01:14     
vertex papers but um you know a lot of times people are looking at different types of things like you know gene     
1:01:20     
expression or they're looking at uh just like an organism and different iation so     
1:01:26     
there're different questions you noticed in the differentiation paper you know they didn't use physical terms as much I     
1:01:33     
wanted to kind of connect it more but it was you know you can imagine though that like in the this final paper where you     
1:01:41     
get these cell decisions being made that this is all happening on an epithelial     
1:01:46     
sheet or on some cellular sheet they use the anic nervous system as an example so     
1:01:51     
that's when you're developing this threedimensional organ with multiple sheets and you have these different     
1:01:57     
cells that are differentiating on different ways so that's adding a lot more complexity there now dick said Nick     
1:02:05     
made a comment that these look like differentiation trees like in C especially which is true it's probably     
1:02:11     
kind of where they might be going with that but I don't think they make that connection     
1:02:18     
explicitly also I I did look in a little bit into the celan's uh     
1:02:24     
sential topic that we talked about last week and I did find some interesting things although I have't I'll have to     
1:02:30     
put together a report maybe for next week but yeah there there I don't know how much data we would have for that but     
1:02:38     
I think there are some interesting directions so you know it might be that     
1:02:43     
you have to put together a couple of data sets or get things from a couple of papers to get something together and uh     
1:02:52     
we may even have to do like some sort of pseudo data to see if that looks     
1:02:57     
reasonable H so I I'll do a report for next week I'll try to do it for next week and see if I can you know Enlighten     
1:03:04     
us a little bit more on that um     
1:03:10     
yeah I did a wide uh range of basement membrane     
1:03:16     
attachments for my thesis um just to see what worked because like I said um there     
1:03:23     
wasn't any data even though there was but it was in the wrong like I said it     
1:03:30     
was a dimension out um and I found that 10 Newtons per meter     
1:03:38     
was a bit soft for for most for my uh     
1:03:45     
construct and 100,000     
1:03:50     
um Newtons per meter was a bit strong for a spring attachments to the basement     
1:03:58     
membrane anyway that's that's just an observation yeah but it like it it was     
1:04:09     
uh annoying because there there are there are measurements of uh the     
1:04:16     
attachment to basement membranes but it's in um it's not a point attachment it's     
1:04:25     
um an area attachment yeah which is not possible in     
1:04:33     
in the program I was using yeah yeah that is a good point     
1:04:39     
that like intended that like you know kind of think of a tri Junction as a point but it's really an area is up down     
1:04:46     
the the axis and it's not going to be like straight it might be a straight axis     
1:04:53     
axial line but like it could Bend right like the edges of the cells could been     
1:04:59     
curv and yeah I didn't do scutoids or     
1:05:05     
anything um I just did the simplest ones like three different heights of cells     
1:05:11     
just straight up hexagons but anyways oh there yeah asking about um     
1:05:21     
daylight time yeah daylight I don't know if any uh anyone who joined joined at     
1:05:27     
the top of the hour have anything they want to present or add to the meeting     
1:05:32     
the thing is I was taking whatever meul have done last last week sorry last year     
1:05:40     
uh so basically he tried to like build some hyper grab and     
1:05:48     
overall grab so what what is the next uh uh like I thought about if we can uh     
1:05:56     
have a GNN or some GNN based model and that will take those uh like graphs     
1:06:04     
whatever is already built and uh process and maybe because it is a temporal data     
1:06:11     
if we can add some kind of temporal graph model uh maybe some discrete time     
1:06:16     
temporal graph model like evolve gcn or something like that uh or some updated     
1:06:22     
model is it uh that I I mean that uh area is it okay to have like see check     
1:06:31     
something like that yeah I think you know that's something we've been working on kind of if so if you look at the uh     
1:06:39     
the other project from last year the growing gnns um that is going to have like some     
1:06:47     
some materials for like a temporal graph model although it's not uh maybe not the     
1:06:53     
best one so I think let me go back to the repository here so this is the     
1:06:59     
growing GNN where we have this threedimensional space and the animation isn't running L     
1:07:07     
you know it starts to grow from like three cells to 10 cells to 20 and you get these connections okay and one of     
1:07:14     
the advantages of having like an explicitly temporal graph is you could     
1:07:19     
take out like each stage of nomenclature and just present that     
1:07:26     
and you know that you would have like different slices and so you could like summarize the previous States and then     
1:07:33     
have the current state as the graph and then that would be a good temporal model     
1:07:38     
because it would allow us to separate out the stages of each you know what you     
1:07:44     
would see like under a microscope but also have that representation of previous stages which is     
1:07:53     
also okay okay so is there any kind of uh prediction uh like prediction you're     
1:08:01     
trying to do or just representing a graph uh is fine or like what kind of     
1:08:10     
uh Direction you are thinking uh so I mean we for now we've been visualizing     
1:08:17     
things and we you know but eventually you know of course you want to predict things on graphs you want to have things     
1:08:23     
up in this case maybe cause it isn't like thing but maybe it is because why     
1:08:30     
are we building the graphs because we want to know sort of the interactions     
1:08:35     
between cells so we want to know that spatial relationship but moreover there's a lot     
1:08:41     
of things that happen in space so I cell you know the arrangement of cells in a     
1:08:47     
sheet which we were just talking about or signaling which is molecular signaling between cells using those uh     
1:08:56     
caliz the represent representative points so the farther they away they are     
1:09:01     
from each other the weaker the signal the closer they are the stronger the so there are a lot of reasons why we would     
1:09:08     
want to the graph and then we can answer questions so I mean you know you could     
1:09:13     
take like a graph and you could say I want to build a model of interaction you     
1:09:18     
know some signal molecules so each vert vertex has a signal that leaves the     
1:09:25     
vertex out into space and decays uniformly with space so the     
1:09:32     
further away the other centroids are the less effective they'll be the less     
1:09:38     
connected they'll be and then but you know you have that with the graft model but you also have some sort of signal     
1:09:46     
that transforms the cells in different ways so you might have like a state change from red to blue you say okay if     
1:09:54     
this if you know cells that are closer to the cell that's emitting the signal are R than the other     
1:10:01     
cells you can see these patterns more and that's kind of one of the things that we want to get out of these networ     
1:10:08     
it's these kind of relationships but also these mechanisms for pattern     
1:10:13     
formation and for other things okay yeah understood so yeah I     
1:10:20     
get the point here uh so one thing I was thinking like     
1:10:26     
uh like as with time it is changing and uh the embedding as you call Key     
1:10:34     
the uh like s change the color will change so based on some embedding change     
1:10:41     
the color will change so uh what if we will predict the next next state or     
1:10:47     
something like that he I already know till uh 158th time stamp what is     
1:10:54     
happening and I am trying to predict after 158th or one like at 160th time     
1:11:00     
what will happen and uh there will be some kind of semi-supervised learning or similar kind of thing and if we can     
1:11:07     
predict using some uh discrete time temporal graph models uh that can be one     
1:11:14     
area I was thinking yeah I think there's literature on the on the temp like on predictive     
1:11:22     
temporal grass because you're using I guess maybe a daisy model or     
1:11:28     
something like you know basically you have the state that you know what the state is going to be going forward or     
1:11:34     
you could even build pseudo data where you build a wining extrem and say these are where the sells to buy and then you     
1:11:42     
sort of train the model on that and you uh then try to predict what the next     
1:11:48     
state should be given a you know given like 12 what is the 150 two cell State look     
1:11:56     
like which involves some asymmetric divisions and the algorithm has to get     
1:12:01     
to that and it you know giv different outputs you can say oh that looks     
1:12:07     
plausible you know we can do something like that that would be good um because I think it's you know something that     
1:12:14     
sounds kind of trivial when I don't say that these cells have a function and     
1:12:20     
that eventually they need to be there for the adult to be functional um um so     
1:12:26     
yeah that that would be an interesting Direction um there are a lot of things at the temporal organization of these     
1:12:34     
graphs that you know kind ofate lineage tree and we haven't really     
1:12:40     
explored that too much okay that if that can be direction     
1:12:46     
for this G uh project yeah so     
1:12:53     
then go ahead okay okay so then I can like read the literature I know some     
1:13:00     
some literature on that and maybe I'll Explore More and uh try to connect and     
1:13:06     
uh maybe I'll talk about it more uh in next meeting or so is it fine yeah that     
1:13:13     
would be great yeah like to see presentation on it     
1:13:18     
yeah okay yeah thanks thanks thanks     
1:13:25     
okay some comments in the chat here uh actually it was a paper I don't know     
1:13:31     
what this paper is oh this is uh so this is a paper by hers at all on the archive     
1:13:39     
tracking by weekly supervised learning and graph optimization for whole embryo C elegans lineages this was mul posted     
1:13:47     
this in the chat um so this is actually where they're training a model all     
1:13:53     
nuclei of an embryo using dense fluoresence microscopy which     
1:13:58     
is also noisy so you have to get the nuclei segmented out of the image and     
1:14:05     
then uh you know use that as your training data and that's what we're doing with     
1:14:10     
d uh and then so this is their method for doing this with weekly supervised     
1:14:16     
learning and they kind of talk about the so I guess Mah did you want to talk about this a little bit or     
1:14:25     
uh yeah sure like I can tell like one or two lines so what Joy was asking was     
1:14:31     
like he was telling that he can he wanted to apply temporal graphs over CA Elegance data and you mentioned that     
1:14:38     
since we have like discrete time stamps available from like uh 0 to 180 something uh we can so yeah so this     
1:14:46     
paper is something similar but what it is exactly doing is uh so uh just give me a     
1:14:53     
second I'm just like seeing the paper right now okay uh yeah is doing tracking     
1:14:58     
of cell lineages uh over the uh like development of     
1:15:04     
C so I thought it is like quite similar to what Joy is like interested in uh     
1:15:09     
knowing and doing yeah yeah I think it is so yeah they're just trying to     
1:15:15     
because a lot of this segmentation is going to be prediction so it's like what this what cell should be in the next     
1:15:21     
frame you know if a cell divides and you have noise in the yes exactly then you     
1:15:28     
have to know what a centroid is and what noise is because you don't have the ground truth of this cell is actively     
1:15:35     
dividing sometimes you take images that are on either side of that and you can't     
1:15:40     
capture like the Dynamics directly in the microscop so you have to predict all     
1:15:46     
of that and what's most likely to be a centroid versus noise or something     
1:15:53     
else um yeah and then you know this is where we don't have the labels so this     
1:15:59     
is in seans we have nomenclature labels so that's a a good thing but the problem     
1:16:05     
is is that you know we sometimes we don't have those in the microscopy images we have to attach them later so     
1:16:13     
if we have a noisy image you know what are we attaching what label are we attaching to what and that's where we     
1:16:20     
can have problems and the same thing old true I think if you're simulating an embryo if if you're like simulating a     
1:16:26     
bunch of centroids you know is this like a plausible     
1:16:33     
outcome yeah yeah I get it okay get the challenges or     
1:16:41     
okay okay so yeah if if people are interested in presenting on that next week we can talk about that as well     
1:16:49     
um I don't know uh Leith or anth uh do you have any to add you anything to     
1:16:57     
update us on or de onu as     
1:17:09     
well oh yeah hey Bradley yeah yeah sorry I didn't get the thing about daylight     
1:17:16     
savings time being active sorry I got AIT late uh yes uh so I was actually uh     
1:17:23     
I talked to py about something and uh that came she came up with the she     
1:17:29     
suggested an idea to come up with a GUI for like uh visualizing how the uh graph     
1:17:35     
evolves in real time so I actually uh wanted to kind of explore that uh     
1:17:42     
particular aspect of uh like aspect and uh you know maybe try to make it a a     
1:17:49     
project for gck this year would that be a good direction to do uh to take it up and     
1:17:55     
yeah I think so that would be work um you know we we did some Explorations with uh the different topological data     
1:18:03     
analysis and we have that uh approach that we were taking uh so this     
1:18:09     
is uh back in the growing gnn's uh see if I can find it here     
1:18:16     
actually know it's back here I think it's in this stage two folder the stage     
1:18:22     
two directory this is K mapper we're doing some topological data     
1:18:27     
analysis but this is just a fork of a previous project basically we want to do     
1:18:33     
is create this visualization which is where you have     
1:18:38     
the cells by nomenclature and then you know arranging it in different ways so     
1:18:43     
like part of a graph or part of graph theory part of network theory is mapping things out in space so there are     
1:18:50     
different types of layouts you have Force layouts you have circular layouts some are more um sort of realistic to     
1:18:58     
the with respect to the embryo some are easier to to interpret with respect to     
1:19:05     
looking at it by eye so having a goey there which would allow us to build     
1:19:10     
these different topological um representations would be good now K mapper is uh there's like an     
1:19:21     
uh there there's a repository here and they have like a web interface and I can't bring it up right now but     
1:19:27     
basically the web interface is very limited in the different types of visualizations you can do I think this     
1:19:34     
Kepler mapper repository um you know allows you to to     
1:19:41     
modify it if you get into the code and you you know we have we can host it on our on our U website but you know you     
1:19:50     
can play around with the different types of visualization and you you know the it involves topological data     
1:19:56     
analysis which you know is of course another step and you can decide how how     
1:20:02     
deep how far down that road you want to go I would suggest probably just kind of a basic set of     
1:20:08     
representations but you know that's up to you so that I mean I would like to build     
1:20:13     
into what we have but also I think it's a good idea to pursue and I don't want     
1:20:20     
to constrain you too much on how you do that but I do want to have like you know I do want to kind of do the things that     
1:20:26     
we already have kind of expand those out so yeah okay thank you all right all     
1:20:33     
right so if there are no other comments or questions let's call it a meeting um any so I'll send out some     
1:20:43     
materials and if you have any questions let me know email or slack and see you     
1:20:48     
next week thank you uh thanks bye-bye thanks bye
