     
Transcript     
0:00     
hello hi     
0:06     
hello     
0:12     
hello so     
0:20     
um I didn't know if someone said that they might want to present on what they were working on if that's true uh you're     
0:28     
welcome to present today     
0:34     
yeah uh can I present today     
0:40     
yeah should I wait or share my screen and     
0:49     
start should I wait for some time uh no we can go ahead     
0:58     
okay okay i'm currently I'm student at     
1:05     
least today I'll talk about an introduction to temporal graph and uh     
1:11     
they are neural networks the unveiling time in     
1:18     
networks so yeah uh the outline of the top 15 at     
1:26     
first I talked about graphs and networks a whole and I introducing and then I     
1:32     
talked about temporal graphs how it captures the time and then uh I talked     
1:41     
about what how neural network can be designed for temporal maps and there are     
1:47     
two different kind of temporal map one is TDDG and another is CTD     
1:53     
that is discrete time procurement and continuous time and I'll uh uh talk     
2:00     
about for each one of them I'll talk about one model those are very popular     
2:06     
for CD region and ford then I'll talk about applications     
2:12     
how we can use it for de and client development uh I'll talk about my     
2:20     
thought on it Yeah     
2:27     
so so why uh networks we use networks to understand     
2:34     
relationship so graphs those are very powerful uh to represent relationship     
2:41     
between entities if there is some uh citation graph or social media graph or     
2:48     
it can be some road network we can use to understand the relationship so think     
2:56     
of some social network or biological interaction here the entities are nodes     
3:04     
that it has entity and relationship so in we use nodes as the entities and age     
3:13     
as the English sensory for uh for social media we use people as the mode and     
3:21     
friendship as age for citation network we use papers as the citation paper as     
3:29     
the notes and citations as the age like that     
3:35     
so at first I'll talk about the simplest of all that is static graph and uh I'll     
3:41     
talk about is how we use graph neural network     
3:47     
to like to work on those static structure so let's take a molecule here     
3:55     
uh so suppose we are taking nitrogen and here what we we we do is for this we     
4:04     
we can represent it it is a graph and for nitrogen we have different     
4:11     
uh what do you call it different uh neighbors and those neighbors are     
4:17     
different different different atom and we can use here for each     
4:26     
uh for uh for each layer we can we have     
4:32     
uh their their neighbors and this is neighbor of neighbors and these are the     
4:37     
neighbor of neighbor and each in each layer we use some kind of neural network     
4:42     
here we use some neural network and those are shared parameter network this is the uh gn overall gn structure and     
4:51     
here we do message passing so to update This nitrogen we pass the features of     
4:59     
the of its neighborhood uh as messages and we will update in that way the     
5:07     
overall features and it GNN can learn patterns     
5:12     
and make prediction uh based on the graphs structure as well as node vision     
5:20     
so instead of MLP or any other basic neural network we use GNA just to uh     
5:27     
like capture its graph structure so there are three different     
5:32     
kind of task that can be designed here one is note level task age level task     
5:40     
and graph level task over both all of those     
5:49     
sorry sorry can anyone explaining something     
5:58     
no I think you're fine go ahead okay so for for uh for all these kind of task we     
6:07     
have uh like uh we can we can have classification as     
6:14     
well as regression but uh in reality uh we have     
6:20     
like dynamic that is it is changing with time not always static graph can     
6:26     
represent all kind of networks suppose there is a road network so if you see     
6:32     
the road network it is constantly changing so how can we capture that when     
6:39     
network is evolving with time so here relationship might appear disappear or     
6:48     
uh strength might change so here what we can do we can capture only the     
6:55     
snapshot snapshot of of of the time for us sometime but if we just take a     
7:03     
snapshot and use GNN or some other kind of neural network the problem of that     
7:11     
will be we we can miss some intermediary uh intermediate uh all those temporal     
7:18     
information crucial information so the example can be for social interaction or     
7:24     
financial market fluctuation or biological system development here it is     
7:30     
changing all the time it can be daily hour hourly or in every     
7:38     
moment so now we understand why we need temporal graphs so now we'll uh take     
7:44     
detailed look into it so what uh it it can capture so it they capture how     
7:53     
relationship change over time so here uh for this graph at different time there     
8:01     
can be new age addition or new node that might come some new node has come some     
8:07     
new age might come or node update might happen or age update might happen so     
8:14     
here the key idea will be uh for different time step for time interval at different time interval note     
8:22     
and ages might change so it's a dynamic system so here we we have to understand     
8:29     
the temporal pattern as well so only capturing its spatial features will     
8:38     
not work here so we we need both temporal as well as spatial     
8:45     
feature if we see in terms of model complexity for static graph the models     
8:51     
are easier so if we have age weighted     
8:56     
uh where the age might change suppose uh for a network uh for a social media     
9:04     
network uh the the friend I have with whom I am talking every day the his     
9:11     
feature or her feature might be more related relatable to me but someone I     
9:18     
never talk to his or her teacher will not     
9:23     
not be that important so here the weight will come and model complexity will     
9:29     
increase with discrete time and the for continuous time it will be the     
9:35     
maximum when uh continuously with time the graphs is     
9:44     
changing so uh for how the data structure look for temporal graph so for     
9:51     
temporal graph uh as we see at time t1 there is a age between two graph two nodes and at     
10:00     
t2 there might be some more age that is coming or few more nodes coming at t3     
10:08     
there are more nodes in temporal age that can be represented as at source     
10:14     
node then uh target node when there is a time uh like age that is appearing and     
10:21     
its time stamp this is how we see temporal graph instead of static graph     
10:28     
we use temporal ages or events instead of only the     
10:33     
node we take we can have multiple interaction between the same node also     
10:41     
that we have to capture and each event represent an     
10:48     
interaction that uh or relationship that occurring on a specific time or some time     
10:54     
interval so there are as there are two kind of temporal graphs one is discrete     
10:59     
type temporal graphs that is the snapshot base suppose this is a graph we take snapshot at at different time and     
11:08     
as we see the structure is changing we we just take at different     
11:13     
time step at uh like at at a time interval uh suppose we are taking a time     
11:19     
interval daily hourly or some time interval t we we can uh see how it is     
11:26     
changing at different time we just take the snapshot so here node or age can appear or     
11:35     
disappear or it might be same as well so we are taking a discrete time that     
11:41     
that's why it is discrete time so example I have shown here one     
11:47     
example of DTD that is social media social network evolution so it's a very     
11:53     
small and simple graph I have shown so for 3 months suppose every month I'm     
12:00     
taking a snapshot so there are these are the friends A B C D and E so A is friend     
12:07     
with B B is friend with C C is friend with D and there is no friend of E and     
12:12     
at the month at month two I'm taking a snapshot so what happened here so so A     
12:19     
and B the friendship is there as as it is and C and between C and B also there     
12:25     
is a friendship uh that is there as it is there are new connection that has     
12:31     
been made between A and C and B and D and the friendship that some this probe     
12:39     
between B and C and here also it has changed so each month we have take a     
12:44     
snapshot of the static graph so together together they form a discrete time     
12:49     
temporal so now I'll talk about continuous time     
12:55     
tempor event that as it is it is happening so what is happening suppose     
13:01     
I'm taking a twitter graph so this person is tweeting at uh at this time     
13:08     
stamp so we'll take a snapshot then after that sometime after some time that     
13:13     
uh after some time there is another person who is tweeting at this time so     
13:19     
then again another person who tweeted like that that as uh like as time goes     
13:26     
there is uh some new ages that is coming up so based on this can we understand     
13:33     
what will be the future age if there will be any future age so here uh there     
13:38     
there are both kind of uh interaction between users and between users and the     
13:46     
content so this person can retweet this person's tweet it might also happen so     
13:52     
here the catch is no fixed time interval whenever there is some     
13:58     
uh event is happening we are taking a time st time stamp precisely at that     
14:05     
time and uh so here we get we are getting more fra representation or     
14:10     
temporal dynamics where it is useful so for financial transaction or online     
14:17     
interaction uh we can use these things so uh example is CTD in financial     
14:25     
transaction where in seconds or in milliseconds there are millions of     
14:30     
transaction that is happening so CTD can capture those exact timing of each     
14:36     
transaction where it will be useful so it will be useful for fraud detection or     
14:42     
analyzing market uh dynamics or recommendation system we can use so at     
14:48     
time t1 there is this graph at time t2 there is uh there is this both of     
14:55     
these is happening one new age that has come and one new node that appeared     
15:02     
and at t1 there is new node appear in t2 new age appear at t3 uh there is another     
15:09     
age appeared and this this was at first at t0 this was the     
15:15     
graphs for transaction So new mode that means for a financial transaction there     
15:21     
is new user that has come and suppose this is a this is a user and this is     
15:29     
a merchant so if this person has buy something at T2 so there is there will     
15:36     
be a connection and similarly there will be another connection here so this is CD     
15:41     
so between CT and DTG which one to choose so as I told it if CTD represent     
15:49     
more fine grain time timing at discrete time or dynameter it represents the     
15:56     
snapshot at different time so every 30 minutes here I have shown for a twitter     
16:02     
network every 30 minutes I'm taking a snapshot so so we can see here see dd is     
16:09     
more it is simpler easier to process and     
16:15     
uh good for data naturally aggregated into time step but the problem with it     
16:22     
will be loss of information in intermediate steps if there are new ages     
16:28     
or something is coming or again it go away then we we just     
16:35     
uh we cannot track that we just miss that information uh for CTD it will be more accurate it     
16:42     
captures fine grain temporal uh information uh but problem is it is more     
16:48     
complex to process so here it is it will be essential when there will be uh pre     
16:54     
precise event typing we need so it choice depends upon     
16:59     
the data and the application uh so these slides I'm showing because for uh the     
17:06     
our demograph which one we should uh choose for that uh     
17:14     
like I have shown this one so as I have shown already there are two kind of     
17:20     
temporal graph discrete time and uh continuous time so for them to     
17:27     
process using neural network what kind of we     
17:32     
some temporal gnome so here     
17:38     
uh I have shown both discrete time temporal graph and continuous time temporal graph so this is a time we have     
17:45     
taken a snapshot at different time stamp and here we we can use RNN uh     
17:52     
to understand the time uh portion and     
17:58     
this uh and use we can use GNN to understand its structure to exploit its     
18:05     
structure so uh here in uh the GNN as I already shown     
18:14     
we use message passing so here what we do we take messages from its neighbor     
18:21     
and then we aggregate and we update and and that will happen in a single     
18:28     
snapshot and then what we will do we'll take RNN and     
18:34     
uh the uh whatever the parameters or uh like embedding we'll find we'll get from     
18:43     
this uh snapshot we'll pass it through RNA And for RNN the hidden uh like hidden     
18:53     
uh hidden uh information will be this GCN parameters and for uh event stream     
19:02     
graph representation where whenever there is a new event that     
19:07     
is happening we are whenever there is a new age we we are using time dynamic how     
19:14     
we will process similarly We'll use temporal sample we will sample out of     
19:20     
this and then we use temporal aggregation and then we'll do temporal     
19:26     
propagation and based on the information updated information we will     
19:32     
do under the then we'll have some structure and we'll uh try to understand     
19:38     
uh what kind of task based on the task so like it can be     
19:45     
uh it can be both or like age level or not level task or graph level task     
19:52     
whatever that is needed so here we can process temporal evolution and     
19:58     
u here temporal gn are test of genns and they aim to learn both graph structure     
20:06     
as well as temporal dynamics     
20:11     
[Music] so for     
20:17     
uh as I have already shown this we have for we have for DTG and CTD we have     
20:27     
different kind of temporal graph neural network for for both both kind of uh     
20:34     
graphs and it aggregates information uh not only from neighbors but it across     
20:41     
time and uh the goal is to understand or predict the future state of the graph or     
20:49     
classify the temporal pattern so now I'll show an example     
20:57     
of temporal graph neural network one is evol GC so how it works     
21:04     
so this is at different time step so we have a GC as I already told so we have     
21:11     
different layers uh based on multihop information we take and and then we are     
21:18     
running RNN here so RNN is capturing RNN can be     
21:24     
G or LN RNN is capturing its temporal features and GCN is GCN is capturing its     
21:32     
spatial feature or the structural feature structure structure of the     
21:39     
graphs and it allows the GNA to adapt to the changing graph structure this is     
21:45     
evolve GCN so how it works at each time step we use     
21:54     
RNN to obtain the GCN weight based on the previous time step and     
22:01     
uh here what we are uh so these are the uh like gro that is taking the GCN's uh     
22:11     
previous what do you call it the uh previous information and uh the graph     
22:18     
convolution it is taking uh the graph structure and uh this process is     
22:24     
repeated for each time step and finally we get a node embedding     
22:30     
at time t so now I I'm showing uh example of a     
22:37     
CTD that is DGN it can we can show it as a     
22:43     
um encoder decoder kind of network so there is a CTD graph and we use TGN and     
22:52     
suppose we need uh based on the previous information we want to see if there will     
22:57     
be any age at T8 at T8 if there will be any age     
23:02     
between two and four so we run TGN and we get the embedding updated embedding     
23:08     
at T8 of this age uh G of that this     
23:17     
MLP or some other and then we try to predict the probability given at t= t a     
23:25     
time what is the probability there will be an age between 2 and 4 so uh it handles event in order of     
23:39     
occurrence events in continuous time using message passing and memory model     
23:44     
so uh here is the broader view of this what     
23:50     
are the key area or key here here uh they uh use the time encoding     
23:58     
so based on the time uh suppose uh there is a uh uh like at     
24:07     
t= 5 there are some uh     
24:14     
some there are like I want to see at t= 5 and t = 6 if there is any     
24:23     
uh age that exist between 1 and two and equal to six and if there is any age     
24:29     
that is exist that exist between two and three so what we we have before we have     
24:36     
raw messages on uh like one that is raw message of node     
24:44     
one and node two at time t1 t2 and raw message 2 and 3 at t3 t4 because there     
24:53     
are some interaction that happened before and we do message passing and based on the message     
25:00     
passing we updated the uh it's embedding at a different time     
25:07     
stamp and then we aggregate those and we at t = 3 we get some embedding based on     
25:16     
the aggregation so and we already have a memory we have a memory model where we     
25:21     
store the previous information of these nodes so based on the memory we have     
25:28     
this is the state state is what what was its information that was stored at t =     
25:36     
to 1 t1 t2 and t4 at that time we have     
25:42     
the memory now we are fetching the memory fetching from memory it takes and     
25:47     
we we we have the uh like based on     
25:53     
now if there is any um like event that is happening or if     
25:59     
there is any uh transaction that is happening based on that we will update at T5 what will     
26:06     
be its uh embedding at T5     
26:12     
for Z1 and Z2 that means for node one and node two and then we try to using a     
26:18     
decoder network we'll try to understand what will be the age probability between 1 and two and two and three If there     
26:25     
will be any age probability or if there will be any age that that will happen at     
26:31     
this time uh this is how and we based on the we'll calculate the loss uh based on     
26:38     
that we we can uh train this     
26:45     
network so uh so C elegance as we know it's a neato     
26:54     
worm a model organism in biology it its     
27:00     
development is complex and it's dynamic process so uh based on the previous work     
27:07     
uh the demo graph uh last time uh one person he already worked on this and uh     
27:15     
use different graph type to model its development so this is the visualization of uh of it so here nodes can be sales     
27:25     
ages can be sale celltoell contacts and time is development stage so this is     
27:32     
this is a visualization how it is changing this these are the application     
27:37     
that has been uh already explored so now what I want     
27:42     
to do is uh if we can use temporal gen to understand the development of of this     
27:50     
uh C elegant so we can represent C     
27:55     
elegance development as temporal graphs it can be DTD at different time step     
28:00     
we'll take a snapshot of its development and then we we can use temporal GN on     
28:07     
top of it so deorraph already have tools and data sets uh     
28:15     
now now my goal will be to predict the it can be prediction of cell fates that     
28:22     
that is how cell is changing with time cell features it can be uh like key     
28:30     
interaction sale interaction during its development we can identify or we can understand the dynamics of the     
28:37     
development process this we can do this can be uh the goal or the uh like     
28:45     
next so in this uh presentation I have covered temporal     
28:53     
graphs that is good for modeling dynamic system and understanding time involving     
28:59     
relationship where temporal graph neural network are important tools for learning     
29:05     
these temporal graphs and it can extend it extended already the capability of     
29:12     
static DNN that is not there that was not there and uh applications there are lot     
29:19     
of applications of it social network finance biology traffic and it is     
29:25     
working really well so uh in production like CTD they have used in Amazon uh or     
29:34     
Uber Eats or Pinterest they already have used this     
29:41     
temporal graph and recently uh for recommendation LinkedIn is also     
29:47     
using temporal graphs so for our this work the future direction will be we can     
29:55     
use temporal graph neural network to understand the development process and the lineage structure of C using     
30:04     
temporal graphs thank you and if you have any questions please feel free to     
30:09     
ask thank     
30:15     
you that was uh good good um so yeah we     
30:21     
have any questions so I I have a I guess basic     
30:29     
question about different types of networks so you proposed you have discrete time networks     
30:35     
and continuous time networks the discrete time networks are these kind of snapshots     
30:41     
and then the discrete the continuous networks are these times     
30:46     
so you're recording continuous process     
30:52     
um so I guess you know you asked what which one is better for C elements and I     
31:00     
would argue that maybe both of them yeah we can explore i think we can     
31:06     
explore both of them but initially I believe we can work with DTD discrete     
31:13     
time program because we already have information on different time stamp there are different time stamp already     
31:19     
there in the data sets so we can work on that and try to predict and uh maybe I I     
31:27     
uh my answer will be for different task will use different uh different uh     
31:33     
network uh so as standing here     
31:40     
uh so for predicting the cell phase we can use DTD     
31:46     
u for understanding the dynamics of development we can use CTD for the     
31:52     
dynamics of development process um so I mean the the thing is is that     
31:58     
you know obviously development is a continuous process but our data that     
32:03     
usual we use is discreet in the sense that it's like these snapshots and so a     
32:08     
lot of times we're trying to you know interpolate between them say this is the     
32:14     
lineage trade this time lineage trade that time but in reality it's a     
32:20     
continuous process so you could say have a cell division event that captured     
32:26     
midway through and still in a continuous process we always capture microscopy at     
32:32     
least like the way we analyze microscopy discrete snapshots you get     
32:39     
maybe a video of the process usually you know kind of extract discrete features     
32:46     
the future selection stage is discrete from there on it's but yeah I like the     
32:51     
idea of using time stamps to say there these these interactions that initiate     
32:57     
at a certain time so you know if you were thinking about cell division you'd have like this cell that we kind of     
33:05     
start to different everything would pull apart     
33:10     
and then you'd have like some point at which it's like two separate cells and     
33:16     
then it migrates you know where you can capture migration in the same way you'd have time stamps to represent those i     
33:24     
think you'd have to do some work to figure out how to represent them but I think that's an interesting kind of way     
33:30     
to do to do that continuous and it wouldn't even have to be truly continuous representation just     
33:37     
having something bradley I put a Bradley I put a similar     
33:44     
question in the chat okay so yeah we had a question from Dick would growth of an     
33:52     
embryo's differentiation tree be an example so that would mean that like you     
33:58     
know how would if you had a growing tree basically you had more cells they were     
34:05     
differentiating and then you had these sort of features like what we call differentiation waves which are these     
34:12     
signals that uh propagate from cell to cell and you know the change it affects     
34:19     
the cellular state and so you'd have this sort of you know it's not just     
34:24     
important to get like say a single cell state it's important to get the     
34:30     
interaction between the cells and like Yeah go ahead yeah one one     
34:35     
thing I noticed is that there seems to be a difference between a graph that     
34:42     
grows spontaneously so to speak and one that grows by uh environmental input by     
34:48     
something coming from the outside which is added to the graph     
34:57     
a differentiation tree presumably is mostly environmentally independent as it     
35:04     
grows uh but the graphs that Jadratha showed uh will seem to grow depending or     
35:12     
change their shape depending on external inputs to the graph     
35:22     
that's why I asked the     
35:29     
question J drop any comment on that external input versus intrinsic growth     
35:36     
of a graph i believe as I have shown here one once     
35:44     
it's like uh so if     
35:50     
uh so if there is any change of the age update if there is sometimes we don't     
35:58     
need uh to have new ages or something new uh interaction that is coming up but     
36:05     
what if the what the existing uh for in existing network the ages or the node     
36:13     
it's features that has been updated that uh you are mentioning so it's a in     
36:18     
intrinsic feature that is just updating so it can capture that also we just need     
36:24     
to know at which time stamp or where the update updation that has been happen if     
36:32     
we can find that uh we can capture in our network     
36:42     
Is it the answer you expected or something like     
36:57     
yeah i mean and and there there's like a difference in different types of processes so like in in the graphs that     
37:03     
you showed you know you're going to have growing graphs where you have nodes that     
37:09     
are added on like cell division and you're not representing old process of cell division you're just representing a     
37:16     
node that kind of comes into existence and then an edge that sort of describes     
37:21     
that new relationship and that's kind of you know it I mean you could make it different where you had like you know a     
37:29     
cell that kind of just pulls apart and then has different relationships to to     
37:34     
each node in terms of the length of the edge or the strength of the edge and so     
37:40     
you have like these different uh nodal relationships and you have these different edge     
37:46     
relationships and kind of one of the things that's interesting about our data set is it's rooted in space so you have     
37:53     
like these spatial locations of each node and sometimes those are very important so sometimes we can have like     
38:01     
you know depending on what we're interested in we have different spatial relationships so say we have like two     
38:09     
cells that are dividing and those cells are pulling apart or they're migrating     
38:14     
and they're migrating with respect to the rest of the network uh you could plot those out like just kind of as as     
38:22     
uh nodes and edges but it wouldn't really capture that process it would be like kind of it cells are migrating but     
38:29     
I can't I can't distinguish between like that process and any other process     
38:37     
like you know okay yeah so I don't know how to say that other than to say     
38:43     
there's kind of a spatial aspect to this as well for both the nodes and the edges     
38:48     
and it's kind of like the spatial spatial changes are also     
38:55     
okay understood so in that case there are already     
39:01     
geometric graph neural network that is available so what what it does is     
39:08     
suppose here uh so through GN also we can capture that we we we can just take     
39:16     
the as a feature we'll take the uh the spatial uh the coordinates 3D     
39:24     
coordinates the spatial coordinates and we can use use that uh for geometric     
39:31     
graph neural network what we can use Then other than the structure and the     
39:36     
known features in geometric uh graph neural network we can use the what do     
39:43     
you call it it's 3D coordinate it it will take that as a     
39:49     
input as well as uh it takes uh another kind of input uh like what is     
39:57     
its body order I mean uh the angle between angle between two or three nodes     
40:06     
so that also we can capture so if that is also important based on the     
40:12     
importance we'll see and we can uh add those as the features like and in that     
40:19     
case it's it's a uh it's a different process uh slightly different process     
40:25     
it's uh there are few kind of method few kind of     
40:31     
methods that are already available uh that exploit uh what do you call it     
40:39     
uh like equivariance or uh invariance uh     
40:46     
of the graph structure or the geometric structure yeah so those are like     
40:51     
equivariant uh neural graph neural network or or invariant graph neural     
40:57     
network so based on the uh application we can design     
41:03     
those okay uh Bradley got a question uh given that uh C elegance uh is mosaic     
41:12     
and seems to do given uh genotype seems to do the same thing each time yeah     
41:19     
couldn't could we represent the three-dimension simply by     
41:25     
making the tree three-dimensional and let it grow     
41:30     
maybe yeah i mean yeah it would be the same basically the same process over so     
41:38     
yeah you wouldn't you would Yeah yeah so I I think if we if we drew if we     
41:45     
actually drew the uh uh the differentiation     
41:52     
tree in three dimensions that might meet your     
41:57     
requirements to start representing a 3D graph for the growth     
42:03     
of the nematode right uh you know it's more complicated     
42:09     
for regulating embryos because uh you get whole patches of cells which are     
42:14     
supposedly similar okay but the the C elegance data     
42:21     
could perhaps be represented that way     
42:26     
yeah yeah we could you know that that would be a possibility so you'd have instead of having the slices like you     
42:34     
have with your temporal graph example you would have like a third dimension where just kind of go down and like     
42:40     
you'd have like the spatial aspect on the 2D plane and then the the temporal     
42:47     
dimension on the third dimension moving down so it's like you time space and I'm     
42:52     
thinking more I'm thinking more like a movie where you watch the tree grow and because it has a specific     
42:59     
structure uh in nematodes you might be able to capture     
43:05     
it oh okay so it it would be a movie a 3D movie     
43:13     
right okay so time is your fourth dimension there and you'd watch it grow just like     
43:20     
the one on the screen now you you can watch it grow over     
43:25     
time but you can imagine the nodes are represented by the actual locations of     
43:30     
the cells and are therefore three-dimensional     
43:37     
oh I see okay okay so you'd have 3D snapshots over     
43:44     
time which give you a movie basically of a moving thing that's growing     
43:54     
and it might fit in this framework too so uh maybe there's some things we could understand about it yeah so it would be     
44:01     
like treating those slices as a a flip book like you do with an animation     
44:07     
no no i'm saying skip that stage of the     
44:13     
slices and go straight for 3D so when a cell divides into two cells in a neatode     
44:20     
it has taken one node and made two nodes out of it or added two nodes to the uh     
44:27     
to the lineage tree so it's a lineage tree plotted in     
44:33     
three dimensions in other words oh okay     
44:39     
okay and now that would seem to be maybe difficult but     
44:47     
straightforward okay because you'd have to get the data on the 3D locations of     
44:52     
each cell and and what kind it is     
44:59     
right but then you could make a movie and     
45:04     
watch the whole neatode grow as a differentiation tree or a lineage     
45:12     
tree the ordinary lineage trees are all flattened in two dimensions what I'm     
45:18     
saying instead of those flat ones make them     
45:25     
three-dimensional okay and make a movie out of it and if that hasn't already been done it could be quite interesting     
45:37     
yeah sounds good all right well yeah uh     
45:43     
so go ahead yeah is it is it some kind of 3D to point cloud structure uh I mean     
45:51     
in 3D and then we'll see how it grows the SP I mean my point is how important     
45:58     
is the spatial like the location     
46:06     
embryo is very important yeah yeah we have you know fairly specific data and     
46:12     
you know we have the the relational data is     
46:18     
actually being a because in C elegance the lineage tree is such that you have     
46:24     
the spatial organization anterior to posterior and tail and then left so there's specific     
46:32     
relational things about different cells and they're always basically the same in     
46:39     
different organisms so you know we have a good number of nature so that's important to capture we have data on     
46:46     
kind of specific positions that's kind of uh you know that can be they have a     
46:52     
lot of technical abilities but basically space is really important at least in     
46:57     
coins to get that you know that sort of basic layout so you're you know     
47:03     
something happens anteriorly posterior capture     
47:10     
propagates through the tree you know however Also     
47:18     
you can also represent submotility by the nodes in the three-dimensional     
47:25     
uh lineage tree moving around a bit that might create new neighbors which     
47:33     
could represented by maybe edges of a different color because they're not     
47:38     
descendant edges yeah in that case in that case yeah as I     
47:46     
was telling geometric graphs can be like we can use that also as well and there     
47:54     
are a lot of research that has been done already and uh like I know a little bit     
47:59     
about that uh on geometric uh graphs and on top of it the geometric neural     
48:05     
network and there there we use like not     
48:11     
only the spatial structure but it's actual position of the spatial position     
48:18     
so do for drug discovery or material discovery or material property     
48:23     
prediction or molecular property prediction where these the actual cell     
48:29     
position or actual atom position is very important in that case and uh what is     
48:36     
the uh like angle between two atom that is also very important so they are people are using     
48:45     
but I'm not sure how if like since molecule is a static graph static     
48:52     
structure there we can use that but if there is any temporal     
48:58     
uh like temporal part is also available or not uh with 3D position that I'm not     
49:04     
sure that I have to check but uh if we want only uh if we are thinking about or     
49:12     
if you want only to exploit this uh the     
49:18     
location or the exact position in 3D that uh for that there are lot of good     
49:25     
models that already have so I think in terms of like a     
49:33     
project um you know I know we've thrown out a lot of ideas here but just to kind     
49:38     
of focus on like one main thing you want to get done and then you might try other     
49:44     
things uh I don't know how you want to structure the project i mean you know the when you write your proposal the the     
49:52     
timing is important you have to get kind of have like a deliverable at the end so     
49:58     
you know you might pick something that's kind of implementable and then experiment with some other things like     
50:04     
if you said you wanted to build continuous time     
50:09     
network with you know some features of I don't know how you want to structure it     
50:15     
but basically you could at least have one deliverable and then uh you know go     
50:21     
a certain distance on some of the other things and see how they work out now it's true for everyone's project you     
50:27     
know you don't want to uh just kind of propose that I do you know like 50 different things and then get nothing     
50:34     
which isn't deliverable you know like I know technically you could do that but the way I usually like to have people do     
50:41     
it is that you have one thing that you can you know put together that works     
50:46     
when you uh finish the project and then you know have things that we can follow up on later uh so so I we have our ebook     
50:56     
repository we've been kind of moving kind of in varying directions so like     
51:02     
you know each project kind of moves pivots away from the basic uh you know graph neural network idea in     
51:10     
different ways so yeah I would like kind of fix on maybe like thinking about     
51:16     
maybe a continuous graph representation and maybe thinking about a 3D component and then thinking about     
51:24     
kind of how you would put that together i mean first you have to implement it of course and make sure that you have a     
51:31     
good framework and then we can like get into some of the like you know some of     
51:37     
the biological details of it I guess um so that's that's that's what I would     
51:42     
suggest for the for the project itself and then you know we would you know keep     
51:49     
kind of thinking about it and getting specific on how to implement certain     
51:54     
things yeah like because this is like you know one of these things where it's been done in other things that are not     
52:01     
biological like you mentioned social media or finance or something like that and those are different types of systems     
52:08     
they have different properties so we don't worry about spatial location     
52:14     
very much sometimes but yeah the point is yeah but a a customer is a customer     
52:19     
no matter where they are yeah yeah you know okay uh but in the case of uh C     
52:26     
elegance embryos the cells have specific locations in space in addition to time     
52:34     
okay uh and they might also move around a bit now I have a suggestion for that     
52:40     
when you do computer graphics it's always on a flat screen okay because we don't have 3D     
52:47     
displays yet okay someday someday maybe we'll     
52:53     
have the uh Star Trek hologram but uh not yet okay so if you if you had a     
53:00     
three-dimensional differentiation tree at a given stage on the screen it would look it     
53:08     
would look like it was you'd have to flatten it to see it but there's an interesting optical illusion that works     
53:16     
very nicely if the representation on the screen is a representation of something     
53:22     
that's rotating then our eyes see it as     
53:27     
three-dimensional so what I'm suggesting is a very specific project make a 3D     
53:34     
growing differentiation or lineage tree of sea elegance and as it grows rotate     
53:40     
it also so that it looks three-dimensional     
53:47     
and that's that would seem to me to be a project that could work given the data     
53:54     
sets you've     
54:04     
Okay uh so it's a combination of making the     
54:10     
3D tree but also keep rotating it so it looks 3D     
54:20     
okay uh and other thing the first goal I mentioned was prediction predicting in     
54:26     
future time stamp uh of se sale fates uh     
54:32     
that is we already have the data the uh data set we already have and uh those     
54:39     
are like we can take uh snapshot based data we already have so at different     
54:45     
time t = 1 t = 3 till t = 180     
54:50     
from minus t = 19 or 20 to till = 18 we     
54:56     
have snapshots so what if uh we can take I     
55:01     
mean the simplest thing will be if we can predict uh so we will train uh the     
55:09     
data we have till t= 150 and then we'll try to predict what are the future cell     
55:17     
uh uh I mean features yeah but we already have that data what     
55:23     
I'm saying is for a given as a lineage tree develops     
55:29     
you know when a new node appears so you have the timing and you have the three-dimensional     
55:35     
position and so you can make this graph grow over time uh and and watch it uh     
55:43     
and if cells are interacting uh that can be represented by their     
55:50     
juxiposition by there being near neighbors of each other uh at least physically interacting     
56:00     
because you know if you have two cells near each other they're uh they can't get any closer than the size of the     
56:06     
cells in the sense okay so it's more like like like the uh     
56:14     
visualization kind of thing not any prediction task involved right right     
56:19     
right but I'm saying you can start with that and then if you did it for different you might be able to get some     
56:26     
uh of uh understanding I nematodes there are many many uh uh     
56:36     
mutations of nematodes that are known already I don't know how many have uh     
56:42     
have their own lineage trees but uh Bradley probably knows that the lineage     
56:48     
tree for different mutants are usually just removing cells or modifying     
56:53     
function I mean they don't really have their own there aren't mutants that have enough changes in the lineage tree to     
57:01     
make its own lineage tree it's usually just like cells will not divide or will not be viable so there's some mutants     
57:07     
where you have slightly fewer number of cells and then     
57:12     
so you could just ablate cells and that's interesting but you know it's like you could just put that into the     
57:19     
basic structure yeah you could compare you could have     
57:26     
two uh let's say a wild type and a mutant and have them both rotating and     
57:31     
watch the consequences to the differentiation trees as they develop     
57:38     
okay and you can have two rotating simultaneously on the screen     
57:50     
i think I think it's time to uh stop viewing embryogenesis uh as a     
57:56     
flat land thing oh yeah     
58:02     
okay so you guys have all the skills and all the data so uh I think it's just a matter of doing it     
58:13     
okay okay one one more thing I wanted to ask uh which which data I can use in the     
58:19     
in the infograph if we move my     
58:26     
uh we have data sets i have to pull up the data sets and I also need to still need to     
58:33     
get data sets or I still need to provide uh links to     
58:39     
the past G-S projects which I'll do i've not done that um but yeah we have some     
58:47     
data sets i'll make them available i'll put everything in the Slack channel if     
58:53     
everyone has access i don't know if Jo has access to this no actually I still     
58:59     
don't have that access well I think I have your email so I can     
59:05     
go and then also Okay Bradley if you can if there is data     
59:12     
on sensitial features maybe throw that in because Yeah okay yeah i don't Yeah I     
59:20     
don't know uh we we talked about that last time and I haven't followed up on     
59:26     
it but I don't think there are any good data sets for cential uh cells and     
59:32     
sele okay yeah I mean I wonder if like it could be uh extracted from the     
59:38     
lineage tree now because we do have the lineage tree you know we have like the interactive lineage tree oh yeah and and     
59:45     
you know which those are supposed to be sensitial uh well we have some yeah we have from the papers we have we know     
59:52     
kind of certain parts of the tree that are supposed to be sent so you could do like a you could build a database from     
59:58     
that I guess um like you could just say this cell and this cell go together this     
1:00:04     
cell and this cell go together so just identify the cells that are sential then we have the spatial information so we     
1:00:11     
could just say like these are the positions for Yes okay jrath did you get     
1:00:17     
that or uh you need an explanation yeah a little more explanation would be     
1:00:25     
better well stuff that's something I could work on like uh and you know we     
1:00:31     
can talk about that as we Okay the problem was that uh the sea elegance     
1:00:37     
lineage tree is usually usually represented as a large but simple binary     
1:00:44     
tree where you get branches at each uh from each cell uh but what actually     
1:00:50     
happens and some people say there are 44 cases of this some of the cells when they physically meet     
1:00:58     
uh fuse together and become large cells with multiple nuclei and that's what     
1:01:03     
substitial means in this case but nobody's ever represented that     
1:01:09     
in a graph as far as we can tell     
1:01:15     
okay I understood that is that a reasonable explanation     
1:01:22     
now I get it now I hear okay okay I mean there's an example in     
1:01:28     
your own body uh for example uh every muscle cell in your body is actually     
1:01:35     
sensitial and that it's created by uh the uh fusion     
1:01:41     
of what do you call them uh I forgot what you what you name them     
1:01:46     
but they're fusion of cells that meet and become one large cell and with     
1:01:52     
multiple nucle in it so uh it's it's a well-known phenomenon but usually     
1:02:02     
ignored complicates it complicates things because trees become     
1:02:07     
uh full graphs as a result     
1:02:13     
okay     
1:02:19     
uh so yeah that's very good thank you for that presentation it's very good     
1:02:27     
thank you thank you uh so I don't know yeah we don't I don't know if anyone else had anything they wanted to share     
1:02:34     
you're welcome again to present on something if you have it i know that uh Leith uh had some ideas i don't know     
1:02:42     
what he what the status of that is yes sir uh last time I was uh discussing     
1:02:48     
about uh like extracting the points from the movie so maybe I'll just present my     
1:02:54     
I mean like idea okay yeah so this I'll just give a brief     
1:03:00     
understanding of what uh I have understood and all so     
1:03:09     
um so is my slide visible yes yeah     
1:03:14     
so uh in the development of uh clans I     
1:03:20     
mean the embryogenesis so it starts from like a single new plant slowly develops     
1:03:25     
so then we have this uh you know what I initially wanted is in order to get a 3D     
1:03:32     
segmentation model which can actually identify the nucleus which uh and the     
1:03:37     
exact 3D point location of it so that from this we can build a 3D uh graph so     
1:03:45     
as I have read the the paper that uh you have written so I understood certain key     
1:03:52     
components in that see I have seen uh I mean like uh there are like two graphs that can be built on top of uh I mean     
1:03:59     
like while segregating the hyperraph so two hyperraphs can be built so first hyperraph could be a spatial so which     
1:04:06     
can be arranged by physical axis that is uh the it can distinguish uh suppose     
1:04:13     
here is a cell from here it's trying to grow and uh into like uh I mean like     
1:04:20     
number of cells and and then it becomes a full adult before that you know all     
1:04:25     
these cell has its own and each cell while the division happens there are there is like a full lineage tree data     
1:04:32     
so we have the 3D uh I mean like locations too so what we can do is first     
1:04:37     
we can build a spatial hypograph so which what it does is so it extracts the cell clusters which actually form     
1:04:45     
suppose like a tissues or we can see them uh I mean like yeah I mean like how     
1:04:51     
it forms through a a cluster of cells forming a tissues and whereas we have a linear hyperraph where it understands     
1:04:58     
the temporal like structure between like hyper nodes and its functional distinguished characters and thus we can     
1:05:05     
build the complete tree with it and then we can merge these two hyper graph in     
1:05:10     
order to do a full end to end simulation of how it the embryoenesis happens and     
1:05:16     
uh and can I interrupt a second     
1:05:23     
what we were talking about earlier is a third type of uh graph uh could you put     
1:05:29     
your put that slide back on yes okay you got a lineage hyperraph and what you     
1:05:35     
call a spatial hyperraph but you notice that it's arranged by the physical axes     
1:05:41     
anterior to posterior okay this third type which is a spatial graph hyperraph     
1:05:48     
which is arranged according to the actual locations of the cells being the nodes     
1:05:56     
in the locations in three dimensions yes and uh I'm wondering why you don't     
1:06:03     
consider that instead of the anterior posterior     
1:06:08     
uh I mean like uh I'll get to that actually so when we are developing this     
1:06:14     
linear hyperraph or temporal construction of the tree we can actually give uh this uh I mean like features     
1:06:22     
such as uh the three dimension location and in parallel we have to feed the uh I     
1:06:28     
mean like you know the lineage tree information I mean like you know how it has been split so when we combine these     
1:06:35     
two uh I mean like uh I mean like I my idea is to combine them both into like     
1:06:41     
the same itself but uh when I have distinguished between like you know what     
1:06:47     
gets into like hyper edge and hyper nodes so I have seen that hyper nodes models the cell category suppose we     
1:06:54     
categorize into like uh saying okay this is the mother cell and uh or uh and from     
1:07:00     
this you know these type of mutations happen so if I keep the data of the mutations and the 3D locations and uh I     
1:07:10     
I'm not really sure whether I have to keep a 3D location or keep the mutant information I mean like which it it has     
1:07:16     
mutated and the hyper edges we can uh you know uh I mean like model it as like     
1:07:23     
an uh anesmosis so where we can see that uh I mean like because hyper edge does     
1:07:28     
not give uh I mean like it do not have to be you know certain these three nodes     
1:07:34     
or these four nodes so it can have any you know a bunch of uh you know hyper     
1:07:39     
nodes it it forms a hyper edge during that it can bring some cross connections     
1:07:45     
so I am still figuring out how do I have to you know pipeline this both into a     
1:07:50     
scene so but what my idea is to first is to understand you know build a lineage     
1:07:57     
hyperraph so where it models the information and brings a linear lineage     
1:08:02     
hyperraph tree and spatial hyperraph tree too so where uh uh I mean like uh     
1:08:09     
we can merge these two into a single uh no hypograph neural network which can     
1:08:15     
Yes but I'm just saying that the spatial hyperraph would be more accurate in a     
1:08:21     
sense if you used not the anterior posterior location but the actual 3D     
1:08:27     
locations of the cells and use that as nodes yes i mean uh definitely I can do that     
1:08:33     
not just anterior posture we have like threedimensional coordinates so we can plug in that too that's I just mentioned     
1:08:39     
it as an example yeah now the the other thing the I like the idea of idea of     
1:08:45     
anastmosis the anastmosis can be of two types one is cells that are merely     
1:08:51     
neighbors being represented by a an edge maybe an edge of a different color yes     
1:08:57     
or cells that fuse which would be the sensitial cells yes yes because uh I     
1:09:04     
mean like uh there are uh uh suppose we we are uh supposeding this as a network     
1:09:10     
then uh it can be a mus a muscle based network or I mean like the neurons that     
1:09:17     
actually activates the muscle movement muscular movement and that actually gives like you know the information     
1:09:24     
about the food or the predator so these two which might not be uh physically     
1:09:29     
connected these can be connected crossconnected using the uh hyperd that was the intuition from that paper I got     
1:09:36     
it so I thought this this can be done by you know uh tweaking some parameters in     
1:09:42     
like designing the hyper edge yeah in displaying it I'd suggest just using     
1:09:48     
different colors for the edges as to whether they represent the original     
1:09:53     
uh descendant edges in the lineage tree or if they represent just positions of     
1:10:01     
cells or fusions of cells okay     
1:10:06     
okay and uh can you think of anything else uh Bradley oh yeah in this slide     
1:10:14     
you say for for anastmosis non-tree connections so the anastmoses are really     
1:10:20     
kind of like say you had two graph two subgraphs where the cells were located     
1:10:26     
in space and the idea is that either the cells migrate so that they form these     
1:10:31     
kind of clusters and then the anastmosis are kind yeah the weak I guess the weak     
1:10:37     
connectors yes yeah so they're kind of like I don't know they're not really non     
1:10:42     
tree they're kind of like uh I don't know how to describe them maybe weak connector vectors or uh you know uh     
1:10:51     
cross subgraph linking subgraph something like yeah I'm not sure that a     
1:10:57     
nerve cell attached to a muscle cell and activating the muscle cell is weak well     
1:11:04     
it's the terminology they use but uh but it's def but they're still     
1:11:10     
separate cells whereas if they are sensitial then uh     
1:11:15     
that would require different represent presentation but but I think you can get that perhaps     
1:11:23     
you can get these things merely by changing the color of the edge     
1:11:36     
uh sorry I didn't get it i'm sorry okay okay uh yeah     
1:11:47     
so I mean like uh we I mean like the data that is available in the DVO and uh     
1:11:54     
I think DVO graph has the uh the data both in the Mayor's repository and the     
1:12:00     
DVO graph repository i can take the temporal locations and uh and model a     
1:12:06     
hyperraph which can uh I mean like first I will try to model this linear     
1:12:11     
hyperraph and spatial hyperraph and see for a certain short movie where it can     
1:12:17     
you know merge I mean like it's developing from small cells to multiple cells so how the I mean the change that     
1:12:24     
look like and probably I can give some more insights or maybe I can     
1:12:30     
uh bring up a new I mean like a detailed direction of like what could be done from this i'm also interested in like     
1:12:37     
doing uh you know automate this process because this process takes a whole lot     
1:12:43     
of time for like you know to manually annotate from like each and every cell so if we can you know make this process     
1:12:49     
like quite automate I mean like uh that's that's already there in deol learn I think it has been by already     
1:12:57     
done by you know but we can bring more advanced methods where it bit which can     
1:13:02     
do it like more with more precision like uh 98 or 90 plus 95% plus accuracy so     
1:13:11     
that this automation could be more stronger enough to you know deploy it in the real     
1:13:17     
Yeah le if you just take these pictures uh in your second row the bottom row     
1:13:24     
suppose you draw lines between cells that descended from another cell     
1:13:32     
yes now I'm not sure exactly how you'd represent that     
1:13:38     
uh but uh because in a way it's a line in time between let's say the uh     
1:13:46     
leftmost picture and the next one okay in other words each sum of the     
1:13:53     
four cells on the on the left hand side gave rise to two new     
1:13:59     
cells and the line that goes from the     
1:14:04     
line between them or the edges between them that we there would be two edges would go from the left image to the     
1:14:11     
right image yes yes sir okay so it's not     
1:14:16     
it's not clear how you how you going to make that clear as a     
1:14:21     
display okay this is just a representation of the image i mean uh uh     
1:14:28     
I mean like when we actually play the I mean like the actual video then it starts from like two cells and then it     
1:14:35     
multiplicates into that so yeah but you're not what I'm saying is     
1:14:41     
the relationships between the cells are not shown here yes i mean like I Yes uh in this diagram     
1:14:48     
we do not have any relationships mentioned yes means the projection of     
1:14:53     
the 3D locations uh uh yes it's only displaying the 3D     
1:14:59     
location well even two dimensional display of the 3D locations yes i mean     
1:15:04     
like I can make a three-dimensional new like good display of like 3D next time so this was just I     
1:15:11     
brought it from that paper which I have yeah how do you do a 4 DB representation     
1:15:17     
with Yeah in this work what they have done is     
1:15:23     
uh as this is a 3D data they considered this each node and at each node they had     
1:15:30     
they embedded the information of the lineage tree so doing so they said that     
1:15:35     
it's a 4D because in a 3D structure that each uh cell type what they have and     
1:15:41     
they have the tree information too in that node so so that's how they say that this is a 4D model so my focus is not on     
1:15:49     
4D model as of now so my focus is on 3D itself but uh so I mean like     
1:15:57     
yeah for for us mere humans I think the four fourth dimension is going to have     
1:16:03     
to be time which means that the proper representation will be some kind of     
1:16:08     
movie uh so     
1:16:14     
uh I I get the point yeah I mean like during the fourth I mean like when we     
1:16:20     
mentioned the time is the fourth dimension uh I mean like while moving uh in in this scenario the they haven't     
1:16:28     
considered the time as a fourth dimension these people have considered that the information that persist inside     
1:16:34     
the cell nuclei is there one of the feature so     
1:16:40     
when it drops down this is a fourdimensional uh uh I mean like threedimensional     
1:16:46     
structure which has like threedimensional location and at each threedimensional location they had an extra feature called the lineage     
1:16:53     
information so thus it makes like four dimension for them so if you consider the time dimension too it'll become     
1:16:58     
fifth dimension probably yeah I mean one one could imagine for instance that uh     
1:17:04     
if we had enough colors that each cell type is represented by a single color     
1:17:11     
and when a cell divides its daughter cells are temporarily represented by a separate color so you can see where the     
1:17:18     
daughter cells are yes yes that that that can be done yeah that's that's an     
1:17:24     
example i mean it's not that's why I say it's not obvious how to represent this but I think some sort of either using     
1:17:32     
color or time as the fourth dimension might make this much easier to     
1:17:37     
understand yes I get get your point i mean like the the the way I mean like     
1:17:44     
the we model the I mean like suppose we model this the whole embryoenesis using some uh neural network like hypograph     
1:17:51     
and all so the way we visualize it actually makes more sense when we visualize based on like the intuitive     
1:17:59     
way that we have approached with so but uh I just bought this from this     
1:18:06     
paper so I cannot uh you know so next time I can bring a nice good     
1:18:12     
presentation out of that by from the data okay so maybe we're on our way to getting a proper fourdimensional lineage     
1:18:19     
tree yes     
1:18:25     
so yes sir I mean like this is what I'm thinking of probably I'll come with a more uh refined presentation or a     
1:18:32     
refined documentation of what could be done and uh how I could you know you     
1:18:38     
know I mean like keep a nice good checkpoints and detail like what could be the plan of action for this couple of     
1:18:45     
time and uh okay maybe you and Jay Adratha should communicate with each     
1:18:51     
other about this too yeah sure because he's he's been looking     
1:18:57     
at the temporal aspect and now the question in a sense is okay how do we represent all of this so that we can     
1:19:04     
make sense of it so so I mean like my one question is do     
1:19:11     
can I start with developing an automated segmentation model for like uh the cells     
1:19:16     
and then apply hypographs or uh shall I start from the I mean like uh the 3D     
1:19:23     
point locations which we already have no actually those are four fourdimensional yes sir because an     
1:19:30     
individual cell can also migrate within the embryo     
1:19:36     
yeah I mean like sorry I mean like four dimensional information i know I know it's it's hard to put all this together     
1:19:43     
yeah we so we have the data i would start from the extracted data that we have the numeric data uh we can talk we     
1:19:52     
can think about the uh features uh selection and capabilities later i mean     
1:19:58     
like you know we have as devo learned the history of that is like we've used a number of methods for that so you know     
1:20:05     
as the methods have improved and it takes you know it's a pretty fast generation time so we've uh worked with     
1:20:12     
pre-trained models we've worked with different types of convolutional neural networks we've then moved to the segment     
1:20:20     
anything model that was like two years ago so we have all this stuff in the evol package and you know if it's     
1:20:29     
something that you can think of a good you know improvement to segmenting     
1:20:34     
images then you know I'd work on it but I wouldn't like start there because we have data that numeric data we can just     
1:20:41     
use to see do with yeah because you know we would want to use the devolar and     
1:20:48     
stuff on like microscopy data but a lot of that We've had this pernicious     
1:20:53     
problem of trying to uh sort of segment membranes and only membranes and segment     
1:21:01     
uh you know nuclei and only nuclei getting those right every time and it's     
1:21:07     
very challenging it's not an easy problem even in coins so     
1:21:14     
maybe first yeah I mean I know the segment anything model was keeps     
1:21:20     
improving so you know I don't know what the current fork is i mean uh but but     
1:21:26     
yeah the idea is that you know we want to make it uh very very you know if we want to get     
1:21:32     
go in that direction we want to make some improvements that don't kind of reinvent the wheel just want to kind of     
1:21:39     
improve on the performance sure so I mean like uh siding out to the     
1:21:46     
point uh so after SAM there is a new version called SAM 2 which actually is     
1:21:52     
good for the you know the temporal version of the images I mean like videos     
1:22:00     
so okay Bradley I got a question is there enough     
1:22:08     
data to take a given wild type let's say     
1:22:14     
uh and look at variation in it in other words are there enough data     
1:22:20     
sets so we can say this is the location of a cell in this embryo at this time but it's slightly different in that in     
1:22:27     
another embryo at another at the same time well I think like the the data sets     
1:22:32     
that people have collected they collect them over a number of different embryos and they just average them together     
1:22:38     
because a lot of that variation is stochastic it's kind of intrinsic stochastic variation so there's going to     
1:22:45     
be some variation in the exact position but relatively they're usually in the     
1:22:50     
same position so like one you know cells are always anterior to other cells but     
1:22:56     
like their distance you know is going to vary by a little bit and it's right but     
1:23:03     
in other words can we quantitatively uh ascertain how good the claim is that     
1:23:10     
for a given genotype uh a mosaic organism ends up being the same     
1:23:18     
you mean like rel with their relative position well yeah yeah yeah i think that's look this this goes back to the     
1:23:26     
problem of fluctuating asymmetry in regulating embryos uh why are they     
1:23:34     
reasonably precisely bilaterally symmetric on the outside you know uh are there are there feedback     
1:23:42     
mechanisms that correct for errors     
1:23:47     
they and you could start to ask that question in nematodes because uh this     
1:23:52     
assumption is they're all the same but actually you're saying they're not uh yeah there actually are like uh mutant     
1:24:01     
phenotypes where you have like you know I said that sometimes cells are ablated     
1:24:08     
but sometimes the phenotype is sort of different that's like blobby versus like     
1:24:14     
you know there other different attributes to so that fluctuating asymmetry given that like I guess the     
1:24:22     
idea is that the environment has some effect or that there's some effect in the how the yeahition I'm asking     
1:24:30     
assuming the genotype is the same between two individuals can we start to un to get at the     
1:24:38     
question as to why they're so similar In other words are there feedback mechanisms that correct for minor errors     
1:24:48     
uh maybe i'm not sure i'm not sure either you know i mean because it's like     
1:24:54     
how would you say well something is really similar versus Yeah let me give you an experiment do you have any     
1:25:01     
children's cubic blocks not on me no not on you okay if you make     
1:25:07     
a if you make a tower of them eventually it falls over yes that's because there's     
1:25:13     
a small error in putting each block on but if you had a feedback mechanism to make them more accurately placed maybe     
1:25:19     
you could get it higher okay so it's it's a simple analogy but     
1:25:26     
uh how much deviation can you stand and do you correct for it or not     
1:25:33     
we think that question could be asked about nematodeses maybe um that sounds like something     
1:25:38     
you'd have to simulate like Yeah maybe try different thing like different distances and tolerances yeah until you     
1:25:46     
get an emo that falls apart right     
1:25:51     
okay but if you can get some kids blocks try the experiment you'll see exactly what I mean yeah yeah     
1:25:59     
okay uh because uh it always falls down eventually     
1:26:06     
but that's because kids don't make any corrections as they put them you know I if if you made a stack     
1:26:14     
and it's still stable you could then push it together and make it even     
1:26:19     
straighter so you could get even higher okay but is there a mechanism that does that     
1:26:25     
right okay so I think that's a fundamental question in embryogenesis which I've never seen     
1:26:32     
addressed     
1:26:38     
okay yeah that's all pretty good all right uh well I Does anyone     
1:26:46     
else have anything uh uh hi Bradley hello hello everyone uh     
1:26:53     
yeah so I wanted to like uh tell a few updates uh by the way like I have I I     
1:26:59     
missed that Joyy's presentation but I we have discussed it like already like much detail but I think it was like great     
1:27:06     
stuff which he's presenting uh so Bradley So and I are like exchanging a     
1:27:12     
few messages on LinkedIn for some time so uh I'm not sure like whether SOM has already discussed it uh but like so I     
1:27:19     
prepared a small like doc uh for like uh what might be possible for like his Gog     
1:27:27     
so I'll just share my screen okay yeah start camera where is     
1:27:33     
Yeah okay how do we share the screen here uh should be like an arrow like box     
1:27:39     
with an arrow yeah I think I got it share the     
1:27:45     
screen okay it's there yeah so like as much I understood uh So     
1:27:52     
is interested in more of like visualizing aspect of do graph uh and     
1:27:57     
hyperraph uh yes he did mention other like points which he want to focus on more of like research but that is what     
1:28:04     
primarily he was looking at so I just prepared a doc like which I think could     
1:28:10     
have helped so I already shared it with like suham but I just wanted to like put it in the meeting also so I mentioned     
1:28:17     
like few directions and like what are the tools which can be utilized so I started with like why do we need it and     
1:28:23     
like what hyperraph is like skipping these parts there are some interesting tools and libraries like which we have     
1:28:28     
already used before and uh like some new ones so the hyperetex like this is the one which I used uh last summer like     
1:28:36     
extensively so it's built on over networkX and this library extends traditional graphs methods uh for the     
1:28:43     
hyper graphs actually it provides like good visualization uh uh methods     
1:28:49     
including the dual hyperraphs so what that basically means is that we first     
1:28:54     
convert uh representations to bipartite and then uh dual nodes with hyper and     
1:29:00     
hyper with nodes something like that uh we discussed those points like in the     
1:29:05     
end of the like my project and then there is like one uh nice uh library     
1:29:11     
which I saw XGI so I'll just open that screen     
1:29:17     
uh I think this one yeah so this was also providing a few visualizations for     
1:29:23     
like hyper graphs this this this does not like focuses just on hyperraphs but     
1:29:29     
it has graphs hyperraph simplicial networks and so on so it is like much     
1:29:35     
more expensive than the hyper netex so I think this was like interesting if you     
1:29:40     
want to extend it in future uh then I was looking so like said like what are     
1:29:47     
the places that we can fill like what are the dots we want to connect so I was     
1:29:52     
thinking like if if if he plans to develop like a complete library uh for     
1:29:58     
visualizations uh like common to both C elegance and hyperraph then having more     
1:30:04     
control over the it like would be like really great so I was just seeing what are the other libraries one is B3.js so     
1:30:11     
this is used a lot by like companies more of like so these are to develop for     
1:30:17     
these are for developing web based uh interactive visualization it is JavaScript based so it needs a lot of     
1:30:25     
manual coding uh but the good point is that we can uh offer like what it offers     
1:30:31     
is it offers like really customized representation so I did open it somewhere yeah so it's like really good     
1:30:38     
but not specific to hyper graphs then I was seeing Few more so there is something called Cytoscope this is     
1:30:44     
actually a web- based application uh really good for uh networking let me     
1:30:50     
just open it so it also provides like different kinds of graphs including     
1:30:55     
hyper graphs and it is mostly web based and it has like really interesting like     
1:31:02     
uh functions uh like which can be used i think this was like really good for network and over this library can be     
1:31:10     
built for the hyper graph and clans and Jeffy is also like similar for that so     
1:31:16     
Jeffy doesn't support hyperraphs i was seeing one of the feature like when will they support hyperraph they this was     
1:31:21     
like a thread uh four years back they still don't have hyperraph feature i was     
1:31:26     
looking at the Jeffy libraries but I just like put it for the sake of putting then like the possible outcomes     
1:31:33     
what I was thinking is so like for at least the visualization part itself uh     
1:31:38     
like different libraries can be integrated so for example probably XGI or hyperetics     
1:31:45     
hypernet X one of them can be taken uh uh to make a more front end specific uh     
1:31:52     
interactive visualizations for that either D3 or Cytoscope can be used like     
1:31:58     
one of it with one of the hyperetex or XGI so this is for the implementations     
1:32:04     
perspective and then yeah uh this can be deployed on like either cytoscope or the     
1:32:09     
D3 then I was like seeing like few uh like research on like more of vis on on     
1:32:15     
vis on visualizations of graphs and hyperraphs uh starting with graph layout and design so like like we discussed     
1:32:22     
right a dual of a hyperraph and bipartite representation so this can     
1:32:28     
make hyperraph representation like little easier like the dual or the bipartite and uh there are some like     
1:32:35     
algorithms established layout algorithms which already exist so I was looking at     
1:32:40     
few uh then yeah so there was like one     
1:32:46     
article on y combinator founded by accident like few I think a few weeks back uh this is like uh they were also     
1:32:53     
proposing a hyperraph how we how we can represent it as bipartite graph and so     
1:32:59     
what they said was like the pair wise connectivity like when we have adjacency matrix we make into matrix when we make     
1:33:06     
a hyperraph so incidence matrix have nodes and uh edges as like connections     
1:33:12     
so this is how they represented as hyperraph and they just like discussing a few things I just put it there uh but     
1:33:19     
then there are like few research like uh papers uh and things I was like seeing     
1:33:25     
so I just put it uh and this is like first idea second idea is on top of this     
1:33:31     
uh so uh once we have on we have established a bipartite representation     
1:33:37     
positioning nodes effectively becomes crucial so there there are algorithms like force directed algorithms so the     
1:33:44     
second one it offers like a solution for simulating physical forces uh what I mean by this is like uh physical forces     
1:33:51     
attractive forces draw connected nodes together and repressive forces uh uh     
1:33:56     
push unconnected nodes apart so in context of hyper graphs like if we see this definition of four character     
1:34:02     
algorithms uh we can adapt this algorithms by assigning weights to hyper     
1:34:07     
edges based on the number of nodes they are connected to or the strength of their interactions with those nodes uh     
1:34:13     
second adjusting the simulating parameters to accommodate additionally addition complexity introduced by hyper     
1:34:19     
edges so like I like we can read more about this algorithm like in wiki page     
1:34:25     
and there are like some uh uh good presentation about it also so this was like presentation from n us and these     
1:34:32     
guys have like a paper also on on this hyperraph drawing by this force directed     
1:34:38     
algorithm so this was like really cool uh and this can be used for uh like     
1:34:44     
how like if the goal is to develop hyperraph visualization from the scratch i'm not sure whether that is a like     
1:34:51     
possibly a good goal but like I was seeing more on the literature how it can be     
1:34:57     
done uh then coming last uh so clustering and community detection so     
1:35:04     
this is basically like we had this problem right so when uh we made like a     
1:35:09     
very crowded hyperraph it was really not possible to understand what's happening so instead of that identifying clusters     
1:35:16     
or communities within like a big hyperraph can be useful for uncovering uh critical areas for example some     
1:35:22     
function modules or whatever in the biological or like CL network so this is where I thought like this can be useful     
1:35:29     
there was like a nature paper uh I was seeing uh uh yeah I did not really go     
1:35:37     
into detail of it uh so yeah so they were using this     
1:35:43     
information uh compression uh problem so basically trying to so so basically the     
1:35:49     
goal is uh to have better uh clustering and community detection and which further can be used to make more clear     
1:35:55     
hyper graphs using this compression uh of the hyperraph and then later all     
1:36:01     
this information can be made for the temporal analysis so this is like the     
1:36:07     
complete process I thought can be done for making this library few things yeah     
1:36:13     
but like definitely uh this is like we we can discuss more on it if this much     
1:36:19     
details is required just for the uh or the implementation is required for like     
1:36:25     
this uh for for our goal of visualization of the hyper graphs     
1:36:30     
okay sounds good uh Dick had a question in the chat     
1:36:35     
proposed project does the standard deviation and location of a cell and     
1:36:41     
embryogenesis of a mosaic embryo increase its graph distance from the egg     
1:36:46     
in the lineage tree yeah that's that's interesting i think we have the data for     
1:36:52     
sure i mean you know to get the actual var     
1:36:59     
between embryos if as I said if you if you build a stack of blocks and don't do anything but add one at a time the     
1:37:07     
deviation tends to increase and it falls over yeah yeah but in a real embryo that     
1:37:12     
doesn't seem to happen but that suggests the standard deviation in position     
1:37:19     
uh is corrected somehow and what I'm suggesting the way to detect whether there's a correction mechanism is to     
1:37:25     
measure stand deviation versus uh the number of nodes or the graph     
1:37:32     
distance from the egg okay so I I think we can the data     
1:37:39     
probably has that buried in it is to indicate whether or not there is a uh a     
1:37:45     
correction mechanism yeah doesn't tell you what the     
1:37:50     
correction mechanism is if it exists that's a later question     
1:37:58     
okay um so yeah uh so Mahul are you going to make that document more widely     
1:38:05     
available or what are your plans for it uh so I have shared it right now with uh     
1:38:13     
Sham uh like this like these were the points based on like what are what what     
1:38:19     
are the questions he's been asking uh to me I can share the repo if like uh if it     
1:38:24     
can help in anyway uh if it can help in maybe like anyone's proposal I think     
1:38:30     
then it will be good all right yeah you could put it in the     
1:38:36     
slack channel okay sure i just wanted to ask if we have some time     
1:38:45     
yeah i I Yeah I wanted to ask some something if we if we have some time     
1:38:52     
yeah so for my proposal uh so if I understand correctly it will be like     
1:38:58     
understanding 3D uh how uh lineage is happening on like     
1:39:06     
based on the time it's more of kind of like a 3D movie uh on time is it     
1:39:15     
or it's more about the visualization right     
1:39:20     
yeah yeah the 3D movie idea is more of a visualization     
1:39:26     
and uh the data will be shared uh on the slack later yeah yeah i'll share the     
1:39:31     
data sets we have some standard data sets we use and then uh you know we have     
1:39:36     
image data sets but um that would require to segment the data so maybe the     
1:39:45     
using the numeric data but yeah I'll share the data sets     
1:39:52     
okay so based on that then I'll write the proposal     
1:39:59     
thanks so yeah um GC proposals are due on the     
1:40:06     
7th we have I think another meeting before that but you know it's just     
1:40:12     
prepare for that deadline that we can have more discussions offline and then     
1:40:18     
if if uh so wants to present next time you can     
1:40:24     
um hello am I audible now okay cool um okay so I have a quick five     
1:40:32     
minutes presentation um can I just show it go ahead Yeah okay um so yeah Mayol     
1:40:40     
uh did cover one half of it so what I wanted to cover is so I've divided my     
1:40:47     
proposal into two parts the first one takes up the uh NDP project from last     
1:40:53     
year and so the idea is that the growing GNN that we have last year it's um to if     
1:41:01     
we can convert that into a hyperraph so instead of these pair-wise connections     
1:41:07     
at every time step we have these uh hyper edge connections so the idea is     
1:41:14     
that since these are hyper edges they should be able to model the structure     
1:41:19     
better than uh standard graphs for example if you see here in the third graph you can um even visually clearly     
1:41:27     
see that which cell like the lineage here is more clearly represented than the graphs on the left side     
1:41:34     
so um this was the first part of the project so once we have a hypograph ready we could use hypograph embeddings     
1:41:40     
and then um an NDP to um predict the     
1:41:46     
growth so another use case of this could be that if you want to study the effect of some modulations we could maybe make     
1:41:53     
small changes in the inputs and then see what the network predicts um so once we     
1:41:59     
have these hyperraphs u as Mel was talking one major issue is that the visualization part of it which he     
1:42:06     
covered so what I thought was cytoscape since it is a very widely used     
1:42:12     
biological tool so and it allows plugins um I could use existing libraries like     
1:42:19     
hypernetex and xi and tailor it for our uh needs specifically for this condition     
1:42:26     
and then we could use it as a plug again for cytoscape so that even uh wider like     
1:42:31     
we could cater to a wider audience in that case and then again I have a third part of the project which takes it up     
1:42:38     
like and converts the whole thing into a library but then I don't think we will have enough time to cover the so I have     
1:42:46     
this in the time permits section so yeah that's it thanks for your time um would appreciate your feedback     
1:42:52     
yeah that looks great um yeah thanks for showing knowing that um yeah I think     
1:42:59     
that that's uh pretty consistent we were talking about before so and I think that     
1:43:04     
sounds like a it should be doable within the summer so     
1:43:11     
um do you think the hyperraph presentation uh the hyperraph thing with an NDP could work better than growing     
1:43:18     
networks or should I look for other ways to enhance the model um I mean you can     
1:43:24     
use NDP you can use something else um like NDP we were having problems with     
1:43:31     
computational power so like you know simulating a larger graph larger I think     
1:43:36     
flame line like 48 nodes or something was hard and you might be there might be     
1:43:43     
ways to get around that you might use something different how that works it's up to you     
1:43:50     
I guess all right um well I think that's all for today um a lot of things that we     
1:43:58     
talked about today food for thought and all that so um yeah we'll follow up on some of this     
1:44:04     
next time that's all for this week uh see you next     
