## TRANSCRIPT    

0:00     
hi Bradley hello how are you I am good how are you how are you doing good so I actually asked like a     
0:08     
couple of friends to join uh today so like you have met lit right in the past     
0:13     
l b yeah I think he has joined before also he will be joining today and like     
0:18     
another friend of mine he he is like a graph researcher Jad Jad he we call him     
0:24     
Joy so he was interested in like graphs research and wanted to apply it in Neuroscience or like uh computational     
0:32     
biology that is why he joined would you like to introduce himself or yeah yeah thank you thanks yeah I'm I'm joyu you     
0:41     
can call me joy and U I'm doing M by research in Tri H only and uh uh now I'm     
0:52     
uh working on graph neural network both in static as well as uh like     
1:00     
nature of craft and I am working on two research Pap uh and is finished now and     
1:10     
uh uh like I I I I'll uh like I'm about to finish my Ms within few months     
1:20     
yeah that sounds good thank you for the introduction welcome yeah thanks thanks and I'm also     
1:28     
interested in New neuroscience and the application of GRA neural netork whatever it is applicable     
1:35     
yeah yeah it's it's a fun area I guess um you know we've been talking about a     
1:41     
lot of things here in that area so well yeah     
1:47     
welcome yeah thank you thanks you might be interested in joining our open worm     
1:54     
slack um I guess you know mul can probably get you set up there with uh     
2:00     
think he he's joined it before yeah yeah definitely I want to     
2:07     
join there okay welcome yeah we     
2:12     
have Jay mahul soan Morgan Susan and Leith     
2:19     
um so any updates uh anything new that's     
2:25     
happen hey um hi Bradley uh am I audible yes     
2:31     
yeah so I had this project proposal idea which I have been discussing with mul so     
2:37     
I wanted to discuss it with you so I was thinking if we could extend the hypergraph idea and if possible maybe we     
2:43     
could make a library out of it so let's say uh some researcher has a c elegant     
2:49     
data maybe a video data set of the cells so he could use the library to detect the individual neurons U maybe even     
2:56     
classify them and track them over time and if bu uh the library would also give     
3:02     
him like a dynamic hypergraph or maybe some kind of benchmarking based on his based on the DAT     
3:08     
desert okay yeah so that's kind of what our Deo learned program is supposed to     
3:16     
do we could we could use that and like we could Repose     
3:22     
it you know gra all that works we've had people work on that sort of thing in the     
3:28     
past or we've improved upon it with new algorithms because there's always stuff     
3:34     
coming out so yeah that would be a good idea working on that taking the hyper     
3:40     
graph libraries and extending it to some sort of image extraction and the original sort of pipeline for this was     
3:49     
to have those those first Parts integrated with     
3:54     
graph so like what will be the functionalities you would expect in such a library like something that's not     
4:02     
present and something you would want me to build on uh so we have like I said we have de which is the if you go to the EV     
4:11     
graph repository sort of the root of that organization EV and then it's EV     
4:19     
package and we've been working on a number of different ways algorithms for that I think the last uh point at which     
4:26     
we stopped we have the um uh segment     
4:33     
anything from meta and there's a biological version um and so you know we can use I     
4:40     
don't I don't want to spend a lot of time Reinventing the V but you know I want to be able to have that and that     
4:47     
that you know kind of coming out of what what's already there is actually pretty capable of like segmenting images and     
4:54     
segmented movies but what it doesn't really do is it's not well integrated into the     
5:00     
uh hyper stuff at all and that's kind of what we wanted to do originally by maybe     
5:08     
the year before or you know when we started de had gotten that     
5:14     
point so um integrating it with hypergraph will that be enough for a     
5:20     
summer project or is there something more I can do maybe well I think like     
5:25     
you know with with like d d learn I think maybe improving that somewhat and     
5:30     
I'm sure there are going to be things that we can improve upon specializing into graph Works um and then you know I     
5:39     
I don't know if you want to do how much work you want to do on looking you know     
5:45     
do developing tools for graphs You Know M when he was doing his     
5:50     
project last year proposed some measures and I don't think we got those     
5:56     
implemented and so that's you know one thing that's that     
6:02     
things um but yeah but there are other things like you know once you have the     
6:07     
data then how do you you represent it in a graph uh what what what can you like     
6:14     
kind of extract from images that you can represent there sort of a middle area there that's not well defined like if I     
6:21     
segment an image I might use like a bunch of points or I might seg regions     
6:28     
you know those going to serve as nodes or hyper nodes or what how am I going to represent that in the in the graph     
6:35     
structure so I mean that's that's something that you know     
6:41     
may it probably needs to be clarified technical side and on the conceptual     
6:49     
side okay cool um I'll go TR thank     
6:54     
you yeah Bradley can I uh can I share my like uh like in sort of like very     
7:01     
informal presentation so we have some base uh to discuss like overall as a group that's great yeah go ahead and     
7:08     
share your screen     
7:14     
yeah okay one sec so I really don't have like a formal     
7:21     
presentation but I just wanted to share like a few thoughts uh so this is our     
7:27     
dor graph uh Library module in that there is a sub module called     
7:32     
hyper dorra like this is where contributed and like these are like a     
7:37     
few scripts which I have contributed so I can just like give a quick walk through over the data uh few     
7:43     
visualizations and the metrics we saw like last Summers so this project     
7:49     
basically yeah yeah uh please feel free to add like thoughts comment Bradley please any comments like you feel are     
7:54     
necessary and if anyone else have any questions please uh interrupt me and feel free to ask     
8:00     
okay yeah so this project like basically the data which I was using was cleance     
8:05     
Data uh and the phenotype lineage data I'll show them like in in the Jupiter     
8:12     
notebooks so what I wanted to so what it focuses on is the spatial     
8:18     
characteristics uh of the cleance uh basically it it has gone through many     
8:23     
pre-processing steps and what we have is certain physical characteristics of every cell uh which includes the size of     
8:30     
it and the positions of it and then we have the development stages of the cell so at every time step how the cell is     
8:37     
evolving so every cell divides into like daughter cells couple of daughter cells so at every time point we have data     
8:44     
overall the data included like 780 unique cells so this is like how many cells cleans has the unique cells it     
8:51     
definitely has like more cells and the time uh and the time points included in     
8:57     
the data were from like- 16 19 19 okay so this was the raw data this is     
9:03     
the data which I'm describing right now uh it starts from like minor 16 which is like predivision stages it I think has a     
9:10     
couple of cells in there not much and then it evolves like the the lifespan of     
9:16     
that uh organism is like till 190 steps maybe it would be dying at some point     
9:22     
but at Max it was at 190 then as I told like there were like     
9:28     
a few physical characteristics one was the coordinates of those of the organism so every cell has like a 3D uh     
9:35     
coordinates given in the space and then there was cell size which were varying from 24 to 123 units this is for every     
9:44     
cell yeah and there were like other uh data so I was I think I obtained it from     
9:51     
some other source I don't remember the source exactly but what it provided was a cell lineage information so telling uh     
9:58     
who is the parent or who is a daughter of that particular cell telling the Mother Mother cell and the daughter cell     
10:05     
yeah so yeah so these are the data fields which I made in my like graph or     
10:11     
hyper graph uh it includes a cell name the time of observation of that particular cell the coordinate system     
10:18     
the size of it and the mother it belongs to then yeah so the data preparation was     
10:25     
like this maybe I can go through the code for this so this is about the read     
10:30     
file you if you want to like go very quickly over the details uh this is how it is uh we we went into the graph     
10:38     
creation of it so how we can create the graph so this was like the preliminary graph using which we wanted to create     
10:45     
the hyper graph and we had two formulations of the hyper graph one was the spatial and the other was the     
10:51     
lineage so spatial graph basically tells us uh the proximity of different edges     
10:57     
it is created based on the of proximity uh where the cells like how far is the     
11:03     
cell from the other cells and the second uh conceptual idea of hypergraph was     
11:09     
based on the lineage so creating a hypergraph based on uh their mother or     
11:14     
grandmother SS sort of like that we can increase how far or how deeper we want     
11:20     
to go in the lineage then there was a concept of hypergraph which like I was discussing with suam also in the meeting     
11:26     
of the weekend so here uh we want we can model the hypergraph as the time evolv     
11:32     
so it is like more like a temporal hyper graph yeah so these are like few uh     
11:39     
things where we can go through so here we have like I have like compiled the data this is the process data uh which I     
11:46     
was directly using as as hypergraph you can see the original Source also but you can find it at uh particular locations I     
11:53     
can point you out during the scripts and there are certain visualizations so these visualizations all almost all of     
11:59     
them are also present in the jupyter notebooks yeah so let's actually go     
12:06     
through these notebooks in little like detail so we can start with data exploration     
12:23     
part yeah yeah so this is how like initially     
12:28     
yeah so this is how I was like talking about the data this is how it initially looked like I think it is present at     
12:33     
this location this is like uh yeah temporary location uh yeah maybe you can     
12:39     
go back little bit or I can point you out later where exactly the data is this     
12:44     
is like a temporary location this is not where it is located it would be located right     
12:50     
now okay so this is how we were given the original data uh we had the cell names the corresponding time and their     
12:57     
characteristics as like we saw before and yeah so there are like certain     
13:04     
characteristics of it so the cells there are like size of the cell as I told right so it starts from 24 to     
13:11     
103 and then before that like I told about time also like what hour the time time stamp so it starts from 1 to 190 so     
13:18     
it is like contigous time there is no gaps in between and there are like certain uh prebirth time of that cell so     
13:25     
here only a couple of cells exist one cell exists so it is is only P1 cell and     
13:31     
it exists at different all these four time points and from P1 cell only     
13:37     
different uh the organism starts evolving okay so now coming to the     
13:44     
lineage data so the lineage data is like a simple uh uh look through table so for     
13:50     
every cell we have a mother cell associated with it it is given like this in a data     
13:55     
frame okay then I did some data preparation uh that data preparation includes uh so we had a coordinate     
14:01     
system right 3D coordinate system what I did was I normalized the values between zero and one uh so we can have like uh     
14:11     
these in the normalized format but now I feel that it should be standardized actually instead of normalized because     
14:18     
uh we can have newer distances uh right uh so like let's say we have another     
14:25     
organism which has different uh distances maybe it has some lower values or higher values in those cases     
14:31     
normalization won't be a good idea we should actually do standardization so this can be     
14:36     
changed and uh after this uh uh changing the XYZ coordinates normalizing them and     
14:44     
doing some processing over it uh I was focusing on the uh yeah and then I     
14:49     
combined the lineage data also just to make the property of it so this is how like the final data set would look like     
14:55     
after like some processing uh we have cell we have mother and when we have certain properties of it so I have     
15:02     
basically combined the two data points and here comes the graph creation part     
15:08     
so in a graph like we can create two kinds of graph as we like discussed one is based on the proximity of their     
15:14     
distances of every cell so here I will be talking about nodes and like I'll be     
15:19     
using nodes and cells like alternately then in the normal graph we will have nodes as cells as edges we will have     
15:27     
distance between the cells so it since it is a proximity waste gra and weight of the yeah it is it will be basically a     
15:34     
fully connected kind of graph we can do certain thresholding but uh if let's say     
15:39     
if it is a fully connected graph then the weight of every Edge will be the distance between those two particular     
15:46     
cells uh yeah and we can also add the node properties like which we saw now it     
15:53     
also includes the mother of the cell along with their position and the cell size     
15:59     
yeah so this is like first kind of graph the second kind of graph which we talked about is like the is using is utilizing     
16:06     
the information about cell lineage so here uh the node is nodes are basically     
16:12     
the cells and the edges represent the mother daughter relationships and here     
16:17     
we can have U I don't like we can have different definitions of weight between it but for now it is better to not     
16:24     
assign any part any any weights to the edges since it all the it all represents the mother daughter relationship maybe     
16:31     
it can be a directional graph because the inverse of the graph won't hold the     
16:36     
mother da relationship it will be the inverse of it but our Focus was majorly on hyper     
16:42     
graph so using these graphs we created the hyper graphs out of it so this is     
16:48     
basically the visualization of graphs here I was using M plot Li which really is not very good for visualization and     
16:56     
since we have XYZ plots uh it 3D plots I just plotted the 2D plot so this graph     
17:02     
is not giving the complete picture of the spatial hyper graph Al spatial graph also okay now using the two above graphs     
17:10     
what we what I created what what we created was the hyper graphs out of it so hyper graphs are basically the graphs     
17:17     
are the generalized version of graphs in graphs we have diic connections between two noes but in a hypergraph a single     
17:24     
edge or a single hyper Edge can connect to multiple nodes so this is like a simple definition of a hyper graph so     
17:31     
now creating from the graph spatial graph we can create the spal hyper graph     
17:37     
uh as I was mentioning about the threshold based method in which we can reduce like uh the the number of edges     
17:43     
if we don't have a threshold we will have all like fully connected graph here we can use a fully connected graph     
17:49     
directly and provide a certain threshold to the values and based on that we can     
17:56     
uh we can create the hyper graph so a certain uh Central node will be there     
18:01     
and we will only let's say consider all the nearby nodes which are let's say 50     
18:07     
distance away 50 is like an arbitrary value or you can change it you can change it as a hyper     
18:12     
parameter so if the threshold is let's say very high then the graph looks like     
18:18     
this so here almost all the nodes are connecting to each other so we can have     
18:24     
like with a reduced uh uh this distance proximity value let's say five then we     
18:30     
can see like some little more clear clusters but still it is very unclear     
18:35     
the idea was basically we can create the subgraphs of it uh focusing on certain     
18:40     
nodes at a time so we can understand uh uh every graph like every portion of the     
18:46     
graph keeping them separate so this is one method to create     
18:52     
uh the spatial hypergraph using the distance threshold the other is uh using     
18:58     
the string method so we can use K means or we can use DB scan uh the idea is     
19:04     
basically find the nearby nodes which can be clustered together or which can be treated as a single hyperedge so uh I     
19:12     
I like we tried with both but we went ahead with DV scan uh essentially like     
19:17     
for like following for few reasons of the K means right so K means doesn't work for certain uh uh certain kinds of     
19:25     
uh diagram so it doesn't work well for the ring kind of structures or sp sp or     
19:31     
spical kind of structures so we will see like what are the reasons yeah but DB     
19:37     
scan has like two hyper parameters so one is Epsilon Epsilon basically tells the maximum distance between two samples     
19:43     
so for us we treated it like the hyper parameter if two samples have like     
19:49     
certain distances if they come in like certain distances then we would treat them in a single hyper graph and then we     
19:58     
can Define minimum samples uh uh so what it says is that it the minimum number of     
20:04     
points required to form a dense region or a or hyper graph so let's say if a few nodes or the few cells are like very     
20:12     
further from each other uh and let's say there are only two nodes like that so     
20:19     
one way is that we can consider them as a hyper graph or we can consider them as outlines so it basically M number of     
20:27     
samples decides that if if Min samples is let's say five cells then those two     
20:32     
nodes won't be belonging to any hyper Edge or     
20:38     
cluster yeah so the few reasion which we chose like for which we chose DB scan in our use case was Al unlike K means uh     
20:45     
for DB scan we don't have to specify the number of clusters uh in in advance to tell that these many clusters are what     
20:52     
we want so this is where DB scan helps uh second is that it it removes the     
20:58     
problem of outlier so if there are let's say any outlier kind of uh cells which is which is let's say very far or if     
21:05     
let's say it falls very away from the lineage tree then uh uh then DV scan is     
21:13     
a better algorithm to uh to correctly classify them to the particular cluster     
21:19     
or the hyperedge the third is like we already saw like we already discussed     
21:25     
that DB scan takes care of the specific structure like spherical ones or the     
21:30     
ring ones where K means fails So based on that we created like     
21:35     
these structures and these are much more clear structures uh compared to the     
21:40     
threshold ones we can argue which one is better uh like both have their own places and by the way these graphs were     
21:47     
created using a library called hyper netics uh yeah so this is a library only     
21:54     
focusing on hyper graphs especially the metrics and the visualization part but I felt visualizations are not really good     
22:01     
for our use case because it's not focused on the cells so that is where uh soam and I were discussing how like what     
22:08     
met can be done in terms of visualization and so has some good ideas which we     
22:15     
discussed okay coming back now we saw like spal graph uh like how we can     
22:21     
create one but the two methods first is like distance threshold the second is the clustering so the second uh type of     
22:27     
hypergraph which we we was seeing was the lineage graph right it was created using the lineage data the parent the     
22:33     
the daughter cell data so uh again uh using the lineage     
22:40     
graph we can extrapolate the uh we can extrapolate it to make the hyper edges     
22:47     
and uh based on that we can have these kind of graphs so what basically tells     
22:52     
is that this uh ring uh is having a certain distance uh the hyper parameters     
23:02     
are okay yeah so like based on these uh     
23:10     
hyper Edge we can say that these uh cells are belonging to a closer family     
23:15     
so if yeah so if if they're belonging if they're like very far from each other the like in terms of like their uh     
23:23     
family relations then they won't be belonging to a single hyper Edge so this B basically gives the proximity in terms     
23:30     
of their lineages or relationships so that is the purpose of the lineage hyper     
23:36     
graph so these are two very conceptual ideas we can have some like uh     
23:41     
contradictions why do we need them but I think these are like good Concepts to capture the different kinds of     
23:48     
relationships we can form yeah so this is about it then we     
23:54     
were discussing few more ideas uh like one idea was since we were seeing this     
24:00     
uh uh since our data was temporal we thought that we wanted to see how this relation since we have created now two     
24:06     
concepts of like hyper graphs now we wanted to see how it is evolving temporarily so for that uh that idea     
24:15     
like uh is not complete uh we had some preliminary uh like visualizations of     
24:22     
it yeah so I'm using the same data again like some pre-processing steps uh in the     
24:28     
beginning and then this is how like we can make the hypergraph so I have imported the hypergraph library uh like     
24:35     
code from the previous ones so this is how it looks like so at     
24:42     
every time point we can see how the evoluation of the hyper uh edges are we     
24:48     
can see it in terms of uh uh clustering or the spatial hypergraph but also as a     
24:54     
lineage hyper graph and at every uh particular time point we can focus on so     
25:01     
like we had the idea before that yeah so this is for the uh lineage graph so     
25:07     
before that our problem was that the graphs were too clustered to understand everything right so sort of this cluster     
25:15     
but as we go in time I think it is much more cleaner and efficient to understand     
25:21     
and see how the relationship of every particular cell is evolving along with the interactions based on the pro     
25:28     
proximity but also all the lineage of the graph so this was the idea to see     
25:34     
much more clearer growth during the embryogenesis then uh we had some     
25:41     
thoughts and uh this is uh basically we wanted to see and put     
25:47     
the relationships and the interactions in numbers so we were focusing a little     
25:52     
bit on metrics     
25:59     
yeah so in the beginning everything is like the introduction part then     
26:07     
uh yeah then then coming to the few metrics we were thinking     
26:13     
about statistics there are some statistics on the     
26:18     
C give me a     
26:24     
minute okay yeah so above are some statistics uh we can see those uh but the some of the metrics which     
26:30     
which we wanted to see so these metrics are borrowed from like the hyper nettic library but also from different uh     
26:37     
sources I can find so in this particular domain I did not really find any concept much uh the conceptual idea of     
26:45     
hypergraph clearly defined uh and definitely not the metrics so these are the metrics which I borrowed from other     
26:52     
kind of other literature not from the cleans or the computation biology so of     
26:58     
the ideas I thought which we can borrow and use in the C data so starting with degree distribution the dist the the     
27:05     
degree distribution basically tells how the frequency of the nodes like how they occur we can Define it in different ways     
27:12     
uh uh like different context but in the context of hyper graphs the degree of node can be defined uh like this so uh     
27:21     
the number of hyper Edge or nodes participates in so for an example uh we can like understanding the     
27:28     
degree distribution can help us identify uh different connecting cells okay coming to First the     
27:34     
definition I am also seeing this after a long time so basically uh degree of that     
27:40     
particular node can tell how many hyper hyper hyperedges it belongs to so a     
27:47     
certain node can have a lot of let's say family members in terms of linear relationships so it can have like a lot     
27:53     
of uh hyperedges it belongs to uh at at the same point in the time so if it     
28:00     
belongs to let's say a lot of families then we can sell then we can tell that it it's it is connected to a lot of     
28:07     
hyper enges so it's like highly connected cell maybe in the lineage fashion or maybe in the proximity     
28:13     
fashion so we can tell the importance of those nodes in like different respects     
28:19     
so this is like the importance this is like its importance the C Elance data uh     
28:24     
we can help this degree distribution metric can help us identify how connected uh the certain cell are so I     
28:31     
was calling it as Hub so since Hub is like a central node so uh if if let's     
28:38     
say a certain node is highly connected or I'm like as call called as head it     
28:43     
can help us uh uh tell uh uh critical informations about its role for example     
28:50     
in signaling different Pathways or its role in the differentiation process or     
28:56     
maybe how integr it is in the complete structure so I implemented it like we     
29:03     
implemented it little bit and there are some visualizations of it uh I don't exactly remember these     
29:10     
metrics so I'll just continue uh describing all the metrics like we have     
29:19     
listed yeah so we can think maybe more about these graphs like what we can     
29:24     
represent or we can make something better out of it     
29:30     
okay so now coming to this hyper Ed size distribution so the hyperedge size distribution basically just like we were     
29:37     
seeing the frequency of the nodes here we can see the frequency of hyperedges based on their cardinality basically the     
29:45     
number of cells they are connected to if a particular hyper Edge is connected to 10 cells then it will have a cardinality     
29:52     
or let's say hyperedge size of 10 it is like how many nodes it uh has     
29:58     
so the importance can be it can be used to analyze uh like sizes of the hyper     
30:04     
graphs which can tells the which can tell the nature of the cell interactions so a small hypergraph can tell that uh a     
30:11     
small hypergraph let's say a size of two can say that it is like a peerwise interactions possibly it can represent     
30:17     
like cellto cell communication only but a large hypergraph can suggest like more     
30:22     
complex interactions or like it is a group interactions so it can be it can be like very important uh for a     
30:30     
coordinated uh coordinated development of the cell maybe lineage maybe in a     
30:35     
family or maybe in like structural formation so here was the implementation of it     
30:43     
uh yeah now coming to S adjacency Matrix now we know like what adjacency Matrix     
30:50     
looks like it is basically uh the node to node interactions if like nodes Bel if have if nodes have like direct     
30:57     
connection then we call that uh then we call that value as one if let's say we are not talking about weight s adjacency     
31:03     
Matrix basically talks that if two uh if two nodes belong to S adjacency Matrix     
31:09     
together s number of s number of hyper edges then we can call uh those two     
31:16     
nodes are connected so then AI J where I and J are two nodes which belongs to at     
31:22     
least s hyper graphs then we will say it as one otherwise it is not correct     
31:28     
so the importance of these can be like so this I will be using later in the definitions in more in other definitions     
31:36     
but the usage of this can be if let's say two cells are uh have a connection     
31:42     
that means they are frequently interacting through like multiple hyper edges so those interactions between     
31:48     
interactions between these two cells are like much stronger or like the pathways are much like denser and S can be tuned     
31:56     
also we can Define it as a value of two which is like a simple graph but we can     
32:02     
uh if we want to capture only the stronger uh uh uh connections or     
32:07     
Pathways then we can uh put S as a stronger value maybe s     
32:14     
10 and yeah so low connectivity would would mean that cell have no connection     
32:19     
so they have like limited roles uh uh together in any process or any     
32:25     
development pathway     
32:30     
yeah so these are like fully connected because the yeah uh NS equal to two it     
32:36     
will be fully it will be fully connected because in the proximity graph we have connected all the all the nodes to     
32:42     
everyone but as we filter down then we can have uh these kind of     
32:50     
structures so it is symmetric in this particular case but let's say we only focus on the lineage graph then it will     
32:56     
be completely asymmet because the relationships won't be it will be directional in one one way so in those     
33:03     
cases it will very asymmetric yeah so if you broke that down into like different     
33:10     
stages uh where certain cells you know because all those cells that you see     
33:15     
being compared sometimes they're alive at different time points so yeah you     
33:21     
know I mean I don't know if you like you probably have to do that for like a larger stage like where you have 64     
33:28     
cells or 128 cells but yeah let's see what that looks like but yeah that's really nice it's a nice     
33:36     
metric yeah yeah we can call it very similar to correlation also not not     
33:41     
exactly uh but     
33:46     
yeah okay maybe we can see this in more detail so this is only for like uh distance threshold we can see it later     
33:53     
for like the other kinds of graphs so this is just for the proximity     
33:59     
graph yeah yeah yeah so coming to the other uh things so we defined for the S     
34:06     
adjacency node graph right so this is it is exactly same but I made it more     
34:11     
general definition so it can be extended to uh okay this is just a visualization of it now what I wanted to extend it was     
34:18     
for this adjacency Matrix graph so just like we were seeing uh the uh like the     
34:25     
adjacency Matrix for the nodes now the definition is flipped now instead of node we are treating Edge as a node like     
34:33     
just flipping the definition of nodes and edges so yeah a matrix the The     
34:39     
Matrix represents connection what this adeny graph means is that a matrix that represent connection between hyper edges     
34:45     
instead of nodes on the basis of the same the S adjacent relationship right so it is like almost     
34:53     
same it is just we are flipping nodes and edges and S line graph can be defined as as like as like this the     
35:00     
graph derived from the hyper graph where each node represents like a hyper Edge since now we have reversed the     
35:05     
definition and edges represent the s s adjacent relationships between graphs uh     
35:12     
I am little dubious why I did it so there is one more definition like one     
35:17     
more thing which we tried uh the called the Dual hypergraph in which we were reversing the roles of nodes and edges I     
35:25     
don't I we had a very strong why of doing it but I cannot recall it right     
35:30     
now uh maybe we can think of if I think of it later then we will definitely I'll definitely     
35:36     
share but yeah this metric will be useful uh in in cases of uh this S adjac     
35:43     
symatrix and S line graph will be useful when we are doing uh this uh dual of the     
35:50     
hyper graph so a hypergraph the Dual of hyper graph is defined in which we interchange the roles of nodes and hyper     
36:00     
uh but anyways uh coming to the newer     
36:07     
metrics yeah so there were a few metrics I was finding some difficulty implementing them so I put them as like     
36:12     
future implementation those are like a few of them are like centrality measures uh there are some distance     
36:19     
metrics which can be useful and uh there were some spectral ones also yeah so     
36:24     
some spectral properties I think Joy will be like very much uh would have a good knowledge of     
36:30     
these so he's very good in like mathematics of these but we can like just see like very quickly about these     
36:37     
points so centrality measures basically are like degree centrality what centrality measures tells is the     
36:42     
importance of uh nodes in the hypergraph so uh like how much influence a node has     
36:49     
uh based on their position like how many nodes how many other nodes it connects to and how differently it connects to     
36:56     
nod so it can be uh described using these measures so degree centrality is basically how many hyperedge that     
37:03     
particular node participates in so that is one important that is one way to quantify its importance the other way is     
37:09     
between the centrality so how many uh that nodes falls on how many shortest Parts uh shortest Parts between like two     
37:18     
random noes so counting those number of counting the frequency uh of that path     
37:24     
in which that node is positioned in we can have the definition we can have this between the centrality then there is     
37:30     
closeness centrality in which like which says uh how closely a node is positioned     
37:37     
to all other nodes in the hyper graph and then there are like     
37:42     
hyper centrality which tells the importance based on like the neighborhoods of those uh of that     
37:48     
Central node yeah so the importance like in terms of C Elegance of is of that can be     
37:56     
like we can identify key cells that are playing like a pivotal role in the development both spatially and in the     
38:02     
lineage so if there are high cardinality we can assume that that cell has     
38:09     
numerous connections which can be acted so that so they can act as a hubs uh     
38:15     
which is useful for uh communication or maybe coordination things like that and     
38:21     
high between a centrality can be useful in our case uh in the case of Sals is     
38:27     
that cell that particular cell can Bridge uh different relationships or relation or lineages like it can be     
38:34     
crucial for integrating very diverse uh uh development Pathways uh like I had some thoughts     
38:41     
about it uh like why it can be useful probably I can describe in more detail these are like oneliner     
38:48     
informations uh then there is like closeness centrality so here uh like cells can     
38:54     
quickly influence like if if a cell has has high closer centrality then that cell can quickly influence or get     
39:01     
influenced by others like in like in very short time since it is like centrally     
39:06     
placed and it may be essential for like rapid developmental responses then we     
39:13     
have higher igon Vector centrality like a when a cell which is having high ion Vector centrality can be more     
39:19     
influential to the other cells uh maybe maybe structurally maybe maybe     
39:25     
structurally or maybe like in lineage I'm not sure but structurally yes like maybe to orchestrate like some complex     
39:32     
processes occurring in the organism     
39:39     
yeah then there are like uh certain connected components so I was seeing like different some components were not     
39:45     
being connected so I just like thought like there can be some good uh metrics for them also so there are like some     
39:52     
isolated clusters which basically indicates group of cells that are only act there only interacting among     
39:59     
themselves so forming like their own functional modules which are like quite independent from the full organism so     
40:06     
those can be acted as like isolated clusters and those do occur at like certain time points right so this can be     
40:15     
like important thing to see and at certain time points we can have like completely connected components     
40:21     
suggesting that uh at this point uh the cellular network is like highly integrated with a lot of of interactions     
40:28     
in between so I by isolated I just wanted to clarify by isolated leses I     
40:33     
don't mean that the cell uh like at a certain point the cell is not even     
40:38     
integrated to the organism what I mean to say is that it is not strongly enough     
40:43     
uh uh like connected to the connected to the organism and by that strong     
40:49     
definition uh so strongly can be defined using that s right so we have like few     
40:54     
definitions of s adjacency metrics like those so that strength or the strength     
40:59     
of the signal or the strength of the connection can be defined through that s and based on that we can uh we can     
41:05     
divide them into isolated cluster or a connected cluster or a multi component clusters at different points in time of     
41:12     
evolving of the graph yeah so the applications like I     
41:17     
have listed like we can detect like clusters of the cell that from like specific anatomical stres yeah so in     
41:24     
like in in spal kind of data detecting these clust of cells that are forming like specific anatomical structures     
41:31     
maybe let's say it's separate from the Clusters or maybe a connected component     
41:36     
forming like a certain anatomical structure and in the lineage graph we can identify uh their specific     
41:43     
development Pathways or like branches within the organisms gr yeah and there are like certain     
41:50     
distance metrics also which I thought can be useful so uh these are basically s distance s diameter so s distance is     
41:57     
basically the length of the shortest s between two we have already seen what s s is so s is basically uh it is like a     
42:05     
random walk which we have in the graph but uh in s walk we can only walk between two nodes only if two nodes     
42:12     
belong to at least s number of uh hyper edges so that is what S Walk is and the     
42:18     
longer we walk have the longer we can have uh this work the higher will be the     
42:24     
S distance value for that Point similarly we can have the S diameter     
42:30     
definition also so it it is defined as a maximum s distance across all the     
42:36     
pairs across all the pairs of nodes in that like time point the hyper     
42:41     
graph so the importance can be that uh like how we can what it can tell     
42:46     
basically is that let's say we have a shorter s distance it could tell that we can have we have more efficient commun     
42:53     
communication Pathways in between the organism at that point of time and let's say we have a longer s     
43:00     
distance it may reveal that uh there are some bottlenecks uh in the in the     
43:06     
connectivity of let's say that org of the organism at that particular time point of its like lifetime so it can be     
43:13     
highlighted potenti it can potentially highlight areas requiring like forther investigation yeah and then s s diameter     
43:21     
is like another metric to measure the same thing and and yeah so here I have     
43:29     
defined the centralities uh both in the cases of node and Edge uh uh and between     
43:38     
the like in the edge do so Edge one will will be used in the Dual hyper graphs we     
43:44     
can discuss those in like later and yeah then there are like     
43:50     
spectral properties so spectral properties uh which spectral properties basically utilizes igen values and     
43:57     
vectors uh associated with the hyper graph and which which can be obtained     
44:03     
using the llan Matrix which is essentially in the graph's case adjacency Matrix right so     
44:10     
the properties uh can provide some interesting uh findings into like hypergraphs connectivity how robustly it     
44:17     
is connected and their Dynamic behaviors like how it evolves     
44:22     
throughout yeah so like I was just mapping the importance of C Elegance data like I IG values can tell uh can     
44:30     
indicate the connectivity levels and the presence of clusters in the in in the cell uh I vectors can be used to detect     
44:38     
like Community detection uh like when the community is being formed between the cells um possibly identifying uh     
44:47     
like um modules of functional cells functional groups which can be obtained from the SE from the lineage data or     
44:54     
maybe maybe also the structural communities which can be uh obtained from the uh yeah proximity     
45:03     
graph yeah and then we have spectal Gap so the differences between consecutive ion vectors so these can basically     
45:09     
provide information about hypergraphs robustness like how robust it is if there are like certain uh differences in     
45:15     
the I values so these definitions are really     
45:22     
not clear and for every metric uh what we can do is like we can go really     
45:27     
really deep and there is not much literature existing applying these metrics on the hyper on the     
45:35     
Clans uh especially right in the any of these hypergraph metrics so we can pick     
45:41     
any of these and literally go very deep dive into it starting from the implementations of it uh so this part of     
45:49     
the project is still like really very incomplete and a lot of the things can be done uh both in this first is in the     
45:56     
in the metrics part and second is in the visualizations part uh which like so and I were discussing     
46:03     
before yeah uh so these are all the uh notebooks uh please feel free to go to     
46:09     
them in detail yeah yeah that's it okay from a second     
46:18     
that's that's great uh thank you for that M that was great um so I had a     
46:25     
couple of questions here I think I think uh Dick had some     
46:30     
questions and see okay thing is uh he has three     
46:39     
questions um number one size could approximate the cell was an ellipsoid     
46:46     
with three axes of different lengths so that would be like you could     
46:51     
that he's talking about there is going back to the source data and characterizing the size of the cell and     
46:57     
doing in doing that and you could you know start with some you know pseudo data where you could get better uh size     
47:06     
data and so you have the cell is an ellipsoid which is this you know     
47:12     
elongated sphere and then you have three axes of different lengths you can     
47:18     
measure X Y and Z or the the size in X Y and Z and then the size of the embryo     
47:25     
and the position number two each cell usually has two daughter cells requiring two     
47:32     
three-dimensional vectors to their future centers so this is where you have     
47:37     
like from the main cell and you're going you know the lineage tree shows a dividing and then those daughter cells     
47:44     
end up in a new position so you have the position of the Mother cell position of     
47:49     
the daughter cells they basically move where that cell divides and then those     
47:55     
cells move to those new position positions so each cell usually has two daughter     
48:00     
cells and then those vectors would calculate to the Future centers and then     
48:06     
three some cells Fus with others which should be named so that that's you know I don't know if that's a thing that     
48:14     
we're dealing with a lot in C elegans but lineage tree becomes a graph with     
48:20     
articulations okay so yeah you can have like different representations of this     
48:26     
if you uh you know want to think about like you know we we just kind of have a     
48:31     
basic sort of representation of the biology and then we're mapping it to the hyper graph and then we're doing things     
48:37     
with the hypergraph so a lot of the things that we're doing at the hypergraph are great and you need to do     
48:44     
those kind of calculations on that topology but the the source data from     
48:50     
the organism is the thing that we're also wanting to kind of model before we     
48:55     
plug it into the hypog graph so um yeah so dick says 44 tissues claim to be     
49:01     
sential and C elegants so yeah a lot of the muscle cells are sential where the muscle like you you'll have muscle cells     
49:08     
that are in the lineage tree those muscle cells or those tissues are are     
49:14     
sential meaning they have multiple uh nuclei in a single um uh tissue so this     
49:22     
is something you know we can model model um but this is like     
49:29     
pre uh you know before you put it into a matrix that like you're showing that     
49:35     
would be have um so that's that's that's you know that's something there and then     
49:40     
Susan says the slam meeting last week is worth watching so I don't know what the     
49:46     
slam meeting was about last week did you have uh more details on     
49:53     
that um I was just looking up uh the guy's name is um not going to pronounce     
50:02     
this um but it was about algae and their     
50:07     
mucus Trails oh wow and sequestering     
50:13     
carbon with um I don't know they were calling it snow in the     
50:19     
ocean and yeah just let me figure out their uh his name I'll post it but it     
50:29     
was it was last week and they haven't got it the recording up yet but they did     
50:34     
record it um yeah I just since since you were dealing with diatoms     
50:41     
that that that would be interesting um and they also noticed that some of     
50:49     
they like they there's um I think they're living organisms in     
50:56     
the ocean and they rise during the day and fall at     
51:01     
[Music] night yeah and that's part of their life     
51:08     
cycle and if they don't rise to a certain a certain amount then they     
51:13     
eventually end up on the bottom of the     
51:18     
ocean yeah yeah that's great yeah I haven't     
51:23     
caught up the slam meetings in a while okay well I just thought because you're     
51:29     
into diatoms and mucus that you might might like to see some of of what they     
51:35     
had to say about the ocean snow oh yeah that' be great     
51:42     
yeah yeah just let me find     
51:51     
this yeah uh I think uh so     
51:59     
there it is Dick said those are Dagu oh     
52:07     
okay yeah my yeah so these three points like which     
52:13     
has highlighted are like great uh I mean I really have not thought about it and I     
52:19     
feel with better visualizations we will be having uh better things with these     
52:24     
points we can do better Justice with these points and yeah I wanted to highlight a few interactions which I had     
52:31     
with like uh other participants but before that so all thanks to Bradley I     
52:36     
would say all possible so whatever progress we made it was all possible due to Bradley like the Bas on interactions     
52:43     
I had from him the feedbacks it was like really lovely working with him throughout he was very cooperative and     
52:50     
also like all the inputs were like really great and I feel I could have done a better job if I like if I could     
52:57     
have utilized uh him like more like Yeah by attending the meetings and all those     
53:03     
things but anyways uh so I had some interactions with soham and lit and I     
53:09     
was also talking to Joy so I'm not sure whether Joy is interested in gong he's just interested into research but with     
53:14     
soam I had a meeting uh this like weekend so we were discussing like uh on     
53:19     
what we can do so the major points which came up were like dorra visualizations extending the hypergraph library like     
53:26     
which we are already working on which really we did not start as a module but it can be started as a like module or     
53:32     
Library uh focusing more on the visualizations part at least like in soham's case we were thinking so I have     
53:39     
some like experience in developing a module so I work in energy trading domain like these days where I have like     
53:46     
developed uh along with my team developed like a strong uh uh like a     
53:52     
robust library to download data and also     
53:57     
uh process it like these were the two core functionalities so probably uh     
54:02     
using those ideas and like probably for this particular project uh we can     
54:07     
develop a module for it so other participants or other researchers will be like more drawn to this domain so     
54:15     
that is one and second with lit I was uh discussing like a few ideas lit can     
54:21     
definitely uh tell them in more details he's very interested in applying uh some some self supervised techniques on the     
54:28     
cleance data so he has been already seeing the data uh of uh like uh uh even     
54:37     
even before the pre-processing part so that is where he wants he thinks he can     
54:43     
contribute okay yeah yeah there's um we also have     
54:48     
other data sets for other organisms uh so this kind of data that we have for     
54:53     
seans is cell tracking data that's collected uh by by microscopy and and they have a     
55:00     
certain way that they do this so like they'll have you know a single observation and then they'll have     
55:06     
another observation they have these movies that are sort of combined in     
55:11     
these data sets so like a single data set will have like you know like several     
55:18     
hundred movies of these uh you know just observing the embryo over time and then     
55:24     
you know getting that's why they're not not uh you know uh clearly aligned you     
55:30     
know you don't have from like zero to whatever time you have like things that are prehatch and other things because     
55:37     
they kind of observe them at different points in time and just kind of stitch them together into a data set but     
55:44     
there's uh there's some data sets on zebra fish and I think uh some other data sets     
55:51     
on well there's some data I don't know if we have a good cell tracking data set like that on aidian or C squirts but     
55:59     
that's another organism and so you have other uh data sets that exist with the     
56:04     
same type of data we could use those data sets to do some of this work as     
56:10     
well especially if you want to do unsupervised in an angle you know something like that and it would be the     
56:17     
I mean it would basically be the same process so you you'd have similar variables then you could model it in a     
56:24     
similar way and say you know you have uh you don't now in zebra fish you don't     
56:29     
have the nomenclature uh in the same way but you do have like you know cell divisions and     
56:35     
then having that whole that basic process um so that that's something we     
56:41     
could think about as well um if we if we want to c c alans is not you know it's     
56:47     
not uh we can do more zebra fish     
56:54     
so yeah yeah even Le was like seeing some data from Nature's paper maybe L     
56:59     
can tell in this meeting the upcoming meetings yeah but these are great points     
57:05     
uh definitely the zebra fish the idea in which the process is similar but nomenclature since it's like     
57:10     
unsupervised task we can use S supervis techniques here yeah it is a yeah yeah     
57:16     
yeah um one more thing I'd like you to reshare your screen and I wanted to go back to the GitHub     
57:23     
repository so this is the stuff this is the package that I was talking about earlier so we worked this out over     
57:31     
a number of um Summers mainly and this has um you know this is for taking data     
57:39     
from microscopy images and segmenting images and then getting centroids and     
57:45     
regions defined and then building these kind of visualizations this has nothing     
57:50     
to do with graph neural networks or graphs it's just a way to get data from my cross     
57:56     
so like if we have a data set of cell tracking numbers we have numeric data sets for that and that doesn't really     
58:03     
involve this but if we had uh you know we want to take microscopy images and do     
58:08     
this and extract them from there we can do this at this software but the software also has a number of and and we     
58:16     
worked on this over a number of iterations so it has a number of     
58:21     
different tool kits in it and so you know there's the segment anything model     
58:28     
which allows you to use prompts or use like definitions of shapes you know so     
58:35     
you can use like verb you could label everything and use verbal prompts or or text prompts and or you can use things     
58:42     
like shape prompts or you know centroid prompts to get things segmented and so     
58:49     
you know this is a problem in um like looking at microscopy images is how do     
58:55     
you segments cells out of an image especially when there's a lot of noise     
59:00     
and so we can do this and so the cell tracking data usually only uses um so     
59:07     
the cell tracking data usually only uses centroids they Define centroids they     
59:12     
Define maybe coordinates and then that's what you have but in this case we can take shapes of cells at this point so we     
59:19     
segment out a cell and we can Define the shape and that's what dick was talking about with defining the the volume of a     
59:27     
cell you can get a centroid you can get the boundaries of a cell you can get the shape and then you can uh get more data     
59:35     
in terms of coordinates uh not just for the centroid but along the edges and the     
59:40     
size of the cell so if you build a network you have I think you know not just the distance which is which you get     
59:47     
from centroids being in a in a network or in a in a matrix and you analyze that     
59:53     
but then the shape parameters which you could also put in you build a network from um and I've not done that but it's     
1:00:01     
something that's um probably uh you know really interest well potentially really     
1:00:08     
interesting because you know they're they're different relationships you can kind of extract from it so that's that's     
1:00:16     
what so this is a tool we have available so if you're interested in working on um     
1:00:22     
you know things with microscopy or there are other tools in here that while you to do     
1:00:27     
segmentation so that that's something and know you know I kind of want to develop this because we've been     
1:00:32     
developing it and it includes a lot of state-of-the-art tools I mean we've     
1:00:38     
we've actually have things like Gans which aren't state-ofthe-art but things that we not followed up on as well so if     
1:00:45     
anyone has things that they want to if they want to say participate in G so they want to do something with graph     
1:00:51     
neural networks but also with image segmentation or or working with some of     
1:00:56     
the other machine learning models that exist out there and integrating those at the project this is a a great place to     
1:01:03     
kind of link that with what's going on in the D graph     
1:01:13     
repository yeah yeah this is like very comprehensive thanks for sharing yeah yeah     
1:01:20     
problem okay so uh I think that's all for today I don't know if uh we had any     
1:01:26     
other questions or     
1:01:33     
comments thank you m it was a good introduction I've been looking at youro     
1:01:39     
like this week been trying to understand few missing parts and it was so elegant     
1:01:45     
that you have explained it and uh yeah thanks also sir I've been looking at to     
1:01:52     
the things that you have actually shared so that's a kind of long paper that neural competition so I've been reading     
1:01:59     
it and uh know trying to understand and how these dots are connected and I'm     
1:02:05     
trying to map with some of the components that were actually the paper that you have actually written long back     
1:02:11     
regarding that biological neural networks and artificial neural networks and uh see how things are going on and     
1:02:20     
uh yeah so I'm still reading out some works so maybe I'll get with a good uh     
1:02:26     
overview by next week by reading all this and you know come up with some     
1:02:31     
solid uh you know stuff by reading and you know understanding it in     
1:02:37     
detail so that's that's from my S yeah that sounds great yeah I'm glad     
1:02:43     
you're following up on that yeah so you know last time we talked about the biological and artificial neural     
1:02:49     
networks distinction and some of the differences and there's of course a lot there it's it's not you know what we put     
1:02:56     
in that paper was just kind of like our initial take on it and I haven't thought about it in a while actually but there     
1:03:02     
are a lot of what what's interesting is there are a lot of um types of tests that you can do so you can do something     
1:03:09     
in um like a you can have a vision task where you have a vision data set and you     
1:03:16     
can have um like a neural network do some you know process it like a deep learning dat and then you can take like     
1:03:22     
some neuroimaging data someone looks at some images and then you can compare     
1:03:28     
like different parts of the brain with that Network and um you know those are     
1:03:33     
statistical tests and they kind of give you a something like a correlation     
1:03:39     
between the two types of networks but why it's doing what it's doing why they're why they're     
1:03:46     
different that's yes sir that's more interesting line of research that I wanted to you know see and how it works     
1:03:51     
and all yeah so I'll dive deep into these things and and understand what what I can you know do a little bit     
1:03:59     
more sounds good yes all right well that's it for today     
1:04:04     
uh thank you for attending see you next     
1:04:10     
week thanks M all right so uh I wanted to cover     
1:04:16     
another additional paper uh talking about something we actually talked about     
1:04:21     
last week which was looking at molecular data and cigan lineage tree and in the     
1:04:29     
paper we did last week we did some we talked about asymmetries and sort of     
1:04:35     
precursors of asymmetries and how gene expression sets up those asymmetries in     
1:04:40     
different sublineages this is a maybe a similar paper it's     
1:04:46     
from molecular systems biology and this is systems level quantification division     
1:04:51     
type reveals a common genetic architecture controlling asynchrony F so let's go over the     
1:04:59     
abstract first the abstract reads coordination of     
1:05:04     
cell division timing is crucial for proper Sate specification and tissue growth we have to coordinate cell     
1:05:11     
division timing and of course we talked about this last week where you get this timing of division events and sometimes     
1:05:19     
those are synchronized across the embryo sometimes they're not Sometimes some uh     
1:05:25     
sub uh trees or sometimes some sublineages uh divide separately or asynchronously     
1:05:33     
from other subs and then of course leads it sets up tissue formation and other     
1:05:40     
things and it can determine the growth rate because the more cells you have in a tissue the the more division events     
1:05:47     
you have over a shorter period of time the more cells you have under tissue so if you need to specify tissue a large     
1:05:56     
number of cells you just have a a quick several rounds of cell division and you     
1:06:02     
end up with those cells and so this can be either synchronous the rest of the     
1:06:08     
embryo or asynchronous specialized to sub however the differential regulation     
1:06:15     
of cell division timing across or within cell types during metazoan development     
1:06:21     
remains poorly understood so this is talking about how do you get these     
1:06:27     
asymmetries how you get them across in within cell types and they're talking about metazone development so that     
1:06:34     
includes all animals uh it includes invertebrates vertebrates and even into     
1:06:42     
mammals and uh you know so there was a lot of diversity there and it's you know     
1:06:48     
poorly understood in the sense that we don't know if there are any sort of uh overarching     
1:06:53     
principles so you know we May understand it in model systems but it's hard to see     
1:06:59     
how aside from like specialized uh things going on in in     
1:07:04     
different lineage trees you know if there are some overarching so to elucidate the systems     
1:07:11     
level genetic architecture coordinating cell division timing We performed a high content screening for genes his     
1:07:19     
depletion produced a significant reduction in the asynchrony of division     
1:07:24     
between sister cells and that is abbreviated as ads so this is where you have this asynchrony of     
1:07:31     
division between sister cells so in this case you would have two sister cells the     
1:07:36     
sister cells would then divide and those divisions would be     
1:07:41     
either at the same time or at different times so if they're at different times     
1:07:46     
then there's an asynchrony of division and so you know so for example if you have a mother cell that's born at 120     
1:07:54     
Minutes the two daughters are born one daughter then divides at 140 minutes and the     
1:08:02     
other daughter divides at 155 minutes that would be asynchrony and that asynchrony can     
1:08:08     
continue throughout that sublineage you could have some daughters that divide it one time some daughters     
1:08:15     
that divide it another time and so forth and you can see that you could have quite a astounding degree of temporal     
1:08:24     
complexity so we're interested in the significant reduction in ads compared to     
1:08:29     
that of wild type during cigan up Genesis so this is actually um a gene     
1:08:38     
EDS where you know we we want to compare that condition with the     
1:08:44     
Bon so we quantifi division timing using 3D timlapse imaging Follow by a     
1:08:50     
computerated lineage analysis total of 822 genes selected for curation based on     
1:08:57     
their conservation and Mills and development so they're going to uh do     
1:09:02     
some sort of study where they mutating modifying the expression of     
1:09:08     
these genes uh a total of 822 genes were     
1:09:13     
selected surprisingly we find that sell fate determinants are not only essential for establishing fat     
1:09:19     
symmetry but also are imperative for setting the ads regardless of cellular     
1:09:25     
cont text indicating a common genetic architecture used by both cellular     
1:09:31     
processes so this means that we have sufate determinance establishing fat symmetry     
1:09:39     
which is where you have differences the fate of cells but also this asynchrony     
1:09:45     
of division between sister cells so again we have different different types     
1:09:51     
of cells being produced but also differences in timing that may be or     
1:09:56     
precursors to that and again like I said you can have multiple rounds of division     
1:10:03     
to produce a lot of cells maybe for a specific tissue but then that also sets     
1:10:08     
up differentiation needs to say like muscle cells and ronal cells and so this     
1:10:15     
is I guess consistent with the idea that you're forming tissues you're farming     
1:10:20     
tissues you have to have both of these things in tend and you have to have     
1:10:26     
asymmetries in cell division but also asymmetries in differentiation within maybe a     
1:10:34     
sub so we have ads and we have sufate     
1:10:39     
determinants and the connection between these two indicates a common genetic architecture used by both Cal processes     
1:10:47     
the Fate determinants demonstrate either are coupled or separate regulation     
1:10:52     
between the two processes temp Coral coordination appears to facilitate cell migration     
1:10:59     
fate specification tissue our quantitative data set with cellular     
1:11:04     
resolution provides a resource for future analysis of the genetic control     
1:11:09     
of spatial and temporal coordination during medone development so again this is spatial because it's at some position     
1:11:17     
in the lineage tree it's temporal because it's at a certain depth in the lineage tree and then we can observe     
1:11:24     
these two different things go previous studies of metazone development have suggested an intertwined relationship     
1:11:31     
between cell division self fate specification so this is somewhat known in from previous studies so hypermorphic     
1:11:40     
alals of cdk1 which encodes one of the major c l dependent kinases or cdks so     
1:11:46     
there's a family of cdk genes C elements it drive cell cyer progression produce     
1:11:53     
extraintestinal cells and blast C at the expense of hypodermis and muscle cells     
1:11:59     
during Sean zenesis so blastam or C is one of founder cells one of the sublineages and     
1:12:06     
that produces the C line of cells with the c label on its noral cature and so     
1:12:14     
what you see here is that cell cycle progression increases its speed you can produce extra intestinal cells in that     
1:12:22     
sublineage and that comes at the expense of hyper and muscle cells uh which they show in in these two     
1:12:30     
papers here shyama and NAD all mutant and epistatic Analysis have     
1:12:37     
demonstrated that the proliferative fate of celegans germine stem cells this is in the germ Line This is     
1:12:44     
the part of the lineage tree that segregates out from the rest of the organism and produces germ cells this is     
1:12:52     
the P4 line sublineage and so these are special cells that are     
1:12:58     
going to lead to eggs intestines uh these stem cells depend on cdk2     
1:13:04     
cy1 so this is another uh genetic mechanism that involves proliferative     
1:13:10     
fate and so you produce a lot of cells in this uh dependent on this type of     
1:13:16     
activity these are two canonical components cell cycle control so you     
1:13:21     
have the cell cycle switch or the cell cycle     
1:13:26     
regulator that we've identified and that plays an important role in producing these stem cells and     
1:13:34     
the genetic analysis revealed that polycom repressive complex 2 controls     
1:13:39     
oite fate specification are regulating the activities of c and the cdk     
1:13:45     
inhibitor the Copo indrop so this is indrop we're looking at um oite fate     
1:13:53     
specification and how this is regular so we have another Cent dependent kinas in     
1:13:59     
drop at cin and then cdk which is a dependent kyes inhibitor so we have     
1:14:06     
deapo which is inhibiting uh these what are sort of     
1:14:12     
related to the cdk Gen and C kyom wide RNA interference screen is     
1:14:19     
shown that phosphorilation of the wind signaling receptor LPR 56     
1:14:25     
requires cdk l63 interop another link to the cdk genes Cy independent kyes genes so far     
1:14:34     
studies of metazone division timing are mainly focused on a specific tissue so a lot of these studies will focus on     
1:14:41     
germline or intestinal cells or whatever and so it's important to     
1:14:47     
remember because we don't have a lot of studies across different um you know     
1:14:53     
different types of tissues so that's important thing to remember but these are also important because they show     
1:14:59     
these common mechanisms across different cell typ for example a gain of function     
1:15:05     
mutation in CDC 241 which encodes a cell cycle prompting phos phosphotase or a loss of function     
1:15:13     
mutation and the Gat type transcription Factor N3 has produced an elevated Pace     
1:15:18     
in the c e lineage this is one of these uh founder cells for the E lineage with     
1:15:24     
the e label lature that exclusively develops into the intestine so Eline will develop in     
1:15:32     
intestinal cells recent work on the cell cycle regulator we 1.1 has demonstrated     
1:15:39     
that the extended cell cyc links in cells EA and EP so the E uh founder cell     
1:15:47     
divides into an anterior daughter cell and a posterior daughter cell and those     
1:15:53     
have extended cell site LS in both of those SS and but this is dependent on the P2 EMS sign so P2 EMS are other     
1:16:02     
founder cells in that same sublineage and there's a signal that comes Downstream from that or from the U     
1:16:10     
parent cells and it's influencing cell cycle EA and     
1:16:18     
EP however a reduction in the cell cyc Links of EA and EP caused by a mutation     
1:16:23     
in was not associated with their celf fate specification during C Genesis     
1:16:30     
indicating that regulation of celf division timing can also be uncoupled from fate     
1:16:36     
specification so you can uncouple cell division timing with fate specification through some     
1:16:42     
mutation and but you know they're they're normally in Wild type they're     
1:16:47     
in okay so importantly coordination of division timing is frequently manifested     
1:16:52     
as the asynchrony of division consist cell this is this ads so this is     
1:16:58     
something that of course is this construct that they're using here which is widely observed not only between the     
1:17:04     
sister cells giving rise to different Fates so EA and EP are going to give     
1:17:10     
rise to different Fates and it's dependent on these mechanisms but also between those developing into the same     
1:17:16     
fate so if EA and EP let's suppose that they were going to uh you know     
1:17:25     
differentia the same fate you could still have differences in timing and you     
1:17:30     
could produce more cells maybe anteriorly fewer cells posteriorly or a     
1:17:37     
lot of cells both anterior and posterior so you could have these different variations on division timing and the     
1:17:44     
production of specific cell type or the production of different cell     
1:17:51     
types this indicates that another dimension of Regulation coordinates division pace which is linked to her     
1:17:57     
independent of Fate specification during metazone development interestingly a Slowdown of     
1:18:03     
overall development rated seans embryo by altering the temperature or     
1:18:09     
introducing mutations resulting in a global decrease in division Pace but the     
1:18:14     
relative timings between cells well maintained so this is where you get this     
1:18:20     
uh change in division pace for mutations but the timings between cells were well     
1:18:28     
maintained suggesting that regulation of the overall development pace is     
1:18:33     
independent of the control over the asynchrony between individual cells so this is where you have a     
1:18:39     
difference between this asynchron and division so like let's say EA and EP     
1:18:45     
we'll use that example again if EA divides sooner than EP which introduces     
1:18:52     
a symmetry that's independent then this overall developmental Pace or the     
1:18:57     
regulation of the developmental pace so you could have an individual event     
1:19:03     
that's an asynchronous event then you could have a lot of Divisions in a ea     
1:19:08     
and fewer divisions in EP and those two events the number of Divisions in EA     
1:19:14     
Downstream from EA and its daughter cells and its daughter daughter cells and all of its uh descendants will be     
1:19:21     
different than the mechanism controlling the asynchron between e a and     
1:19:27     
EP so this is this construct ads they're kind of looking at these different types of uh genetic relationships looking at     
1:19:35     
it using     
1:19:43     
mutations okay so now figure one in this paper gives an overview of Sean sulfate     
1:19:48     
map and division asynchrony so this shows the uh worm from anterior end at     
1:19:55     
the left a posterior end at the right from head to tail and you see that you have these different regions that like     
1:20:02     
where you have differentiated tissue types neuron ferx muscle hypodermis     
1:20:07     
germine vulva intestine expiratory so you have these places where you need to produce a lot of cells maybe of a     
1:20:15     
certain type or you need to produce fewer cells of a certain type and it's consistent with these variations in the     
1:20:21     
tissue and so in B you see the lineage tree and the lineage tree of course unfolds on the left you have the     
1:20:29     
anterior end on the right you have the posterior end and you see that there are these variations     
1:20:36     
in it differentiated cells or differentiation events so for example     
1:20:41     
you have hypodermis in red here you have germline in blue here light blue uh you     
1:20:48     
have ference and dark blue and of course these cells can migrate around uh in     
1:20:55     
location this is where they're produced and so you see that like there is     
1:21:01     
this difference you have a difference in timing you see between e and abpl in     
1:21:08     
this in these two sublineages but also in terms of differentiation when it     
1:21:13     
differentiates and how many cells are we supposed to differentiate or in this lineage so you have hypodermis a neuron     
1:21:23     
this is uh this is where this fade occurs here and then you have intestine up here at e at     
1:21:31     
the founder cell so you see that there are differences in where these     
1:21:36     
differentiation events occur and the differences in timing of the division     
1:21:43     
events in C we see that we have the zygote we have these different ads     
1:21:49     
modules and we can have these different ads modules where there's this     
1:21:54     
asynchrony and division timing within cell types asynchrony and division timing between last and differentiated     
1:22:02     
cell so we have a blast cell so we have a a sublineage where one part of it or     
1:22:09     
one daughter becomes some differentiated cell type and another daughter retains its blast cell identity and so this can     
1:22:17     
continue on asynchronously within that sub Lage and then finally we have cell     
1:22:23     
division asyn synony between cell types so you can have one cell type and     
1:22:28     
another cell type which are both kind of you know you could have neuron and feric     
1:22:33     
for example in the same sublineage and that's that's another variation so this is for this ads     
1:22:40     
construct describing all these this different variation in asynchron and the production of syn so in figure three     
1:22:48     
what they're doing here is they're actually validating uh their experimental model     
1:22:53     
which is to observe an embryo but deplete the embryo     
1:22:58     
the gene cbp1 which is known to produce extra cell divisions you deplete the     
1:23:05     
expression of that Gene and then you look at its results in the in tree so     
1:23:11     
you see that the lineage tree is here we have this RNA which depletes     
1:23:17     
cvp1 you have this comparison between so the ri tree is going is dividing Upward     
1:23:24     
at the bottom and in a and then the wild tape is dividing     
1:23:30     
downward uh at the top so EMS while tape EMS RNA for cbp1 and then you can see in     
1:23:38     
the middle here where you can compare the tips of the tree so you can see that in the oil type you have these division     
1:23:46     
events where you have asymmetries and then you have different asymmetries in the r uh depleted model so you have where     
1:23:54     
you have these asynchronies looks like the divisions are a lot closer to the sign in the RNA     
1:24:01     
model and this particular part of the MS sublineage whereas in this part of the     
1:24:06     
MS sublineage you see much more variation in much earlier Division and uh so yeah you see this     
1:24:15     
where I don't even see the e subin in the ri it's like they're two different     
1:24:20     
Ms subes Ms and quotes So this is where you have large scale differences between     
1:24:28     
the wh tablee and this CBP one depleted embryo and then here you can see that B     
1:24:36     
and C so B and C show what this looks like in a three-dimensional model uh B is the     
1:24:44     
vile type c is the cvp1 depletion using R so you can see they've labeled these     
1:24:50     
by these different sublineages so you see that differences in Ms cells     
1:24:56     
differences in E cells and see them here you have many more e cells and C and A B     
1:25:05     
uh and so this shows and this I guess this C cell is where e is this msing     
1:25:12     
quotes and it shows up in this cell here so in this model here so that's and of     
1:25:19     
course it looks also like you get changes in how the cells position cells     
1:25:24     
so you have this group of ABP cells up here and the wild type that don't really exist up here and they're kind of over     
1:25:31     
moved over a little bit you have this large group of ABP cells down here     
1:25:37     
ventrally that don't exist in the same way in the O type so and then there's     
1:25:42     
some also some proliferation of ABA cells so there it looks like very interesting changes that occur with the     
1:25:49     
depletion of CBP so you can see actually in D you have     
1:25:55     
this uh graph where on the y axis you have cell cycle length     
1:26:00     
and minutes uh and then you have for each suin AG you have a comparison     
1:26:06     
between wild type and cvp1 depletion the rni and this is just cell cycle length     
1:26:13     
plotted out so you can see that the cell cycle length increases in the cbp1     
1:26:18     
depletion in AB Ms and c and then in D     
1:26:24     
and then in E that relationship is inverted so that the wild type average     
1:26:30     
is higher CBP and that's interesting because the cell cycle length in minutes is negative     
1:26:37     
in all these except for an e so something you really kind of wholesale     
1:26:43     
modifying the es sub lineage with this depletion this is where they have MS in     
1:26:49     
quotes So Ms in quotes I guess it's e but it's larger and it has a lot so this     
1:26:55     
depletion of cvp1 has much wider scale effects than in any other sublineage but     
1:27:01     
it affects other sub as well so we can actually see that in figure 4 a panel a     
1:27:06     
is divided into two parts this is a comparison between sublineage ABA and     
1:27:12     
sublineage ABP and we can see that there differences in the cell types so we have     
1:27:18     
differences in self division events and differences in differentiation and so simp of these     
1:27:24     
differences are driven by notch signaling they do some experiments where they introduce a second Notch at     
1:27:30     
Ada and the arrows kind of show where uh Notch signaling is induced so you have     
1:27:38     
some places where it's induced some places so this just shows like the effects of these perations on the     
1:27:45     
lineage so you can actually get reproducible results through this uh     
1:27:50     
perturbation so this chart here figure five shows all the significant results for     
1:27:58     
50% reduction in ads remember we said ads is this construct where you get this     
1:28:04     
you can have these interventions where completion of uh these factors and so     
1:28:11     
you end up with these different types of effects affecting different genes in     
1:28:17     
different sub ages so the question is are these significantly different or are     
1:28:22     
these just variations of different image so here where we have uh we have the     
1:28:28     
cell types we have the genes we have the function of the genes pathway here and     
1:28:34     
then we have the significance statistical significance of 0.01 this is a P value of red and then     
1:28:42     
black A P value of 0.05 in for these other     
1:28:48     
signific so we have these subline edges which have different Fates     
1:28:55     
uh number one is hyper ferx and we can see that a number of genes for a number of genes there's a significant     
1:29:01     
difference at the 0.01 level there's one at 0.05 level mom     
1:29:09     
to but we can see a lot of these uh have     
1:29:14     
not a lot of genes have significance may you see this that there's this uh     
1:29:20     
difference across the different sublineages so some genes for some genes     
1:29:26     
there's significance for subsumes over others so we'll take um one for example     
1:29:32     
which in almost every sublineage there's a significant difference in ads over     
1:29:38     
50% change in ads and so you know some of these are at     
1:29:43     
0.01 level some of these at 0.05 level in other cases like for for example Lin     
1:29:50     
23 you have uh in first second uh ninth and 10th eth and Ninth     
1:30:00     
categories you have a significant difference so this is where we have the sublineages     
1:30:06     
um that represent a different um different Fates so this is kind of     
1:30:12     
showing all this across 27 different sublineages with varying Fates and then     
1:30:21     
we can see these genes they're they their function so uh I I can see a     
1:30:26     
couple of examples here in Wind signaling genes or wind signaling related genes where a lot of these genes     
1:30:33     
are at a level of statistical significance across almost every subl a     
1:30:40     
pop one wm1 lit one     
1:30:46     
gsk3 D2 those are examples of ones where it's almost universally statistically     
1:30:52     
significant for this test you also have Notch targets so some of these sublineages are Notch Targets in     
1:30:59     
the cations that we saw in figure four and you can see that there are some     
1:31:04     
genes that are significant for that as well okay so let's move on to the discussion during the early stages of     
1:31:10     
metazone Genesis temporal coordination of cell division is crucial for proper     
1:31:15     
Sate specification and tissue growth and it has remained largely unknown how this coordination is genetically regulated     
1:31:22     
and devil that's resolution so they've looked at characterizing genetic control of     
1:31:29     
asynchrony of cell division um using a combination of RNA o technology and     
1:31:35     
automated lineaging we found that most of the genes involved in the temporal regulation are those that are commonly     
1:31:42     
required for Sate specification indicating that metazone     
1:31:48     
species use a common regulatory architecture for establishing both fat Symmetry and division asynchrony during     
1:31:55     
the proliferative stage of emesis and regulation of division     
1:32:01     
asynchrony during tissue growth May partially serve to determine the total rounds of cell division to control     
1:32:07     
tissue size whereas a regulation divisionary syn during fate specification May mostly serve to     
1:32:13     
coordinate cell vibration regulatory factors identify their screen provide an entry point for     
1:32:22     
establishing a mechanistic understand understanding of how these things are coordinated dur metone development it's     
1:32:29     
a figure seven shows their model of temporal coordination during development so you have the     
1:32:34     
zygo and they produce daughter cells and they express different variations on     
1:32:41     
this ads so you have three different modules you have two daughter cells with     
1:32:47     
the same fate that experience this sort of asynchron and division you also have     
1:32:54     
daughter cells one being say a differentiated Fate One retaining its blast fate it can also exhibit this uh     
1:33:02     
asynchron Division and then you can also have cells of different Fates da cells     
1:33:08     
of different Fates that show this asynchrony of division and then that asynchrony of     
1:33:14     
division can lead to self fate specification or further Sate specification or tissue growth when the     
1:33:20     
Fate is the same in the daughter cells and so more cells you produce the more     
1:33:26     
tissue you get in certain regions and we can look at that spatially in a 3D model     
1:33:33     
of the embryo we also have timing that affects tissue growth so this is where     
1:33:39     
we uh distinctions become important but also if we knock this down with RNA a     
1:33:47     
mimicking a m the effect of a mutation we can also get changes to tissue growth     
1:33:54     
so in the Fate specification side of things we have the same kind of thing but we have this with a little bit more     
1:34:01     
variety that we're going to have different cell types and the differential representation of different     
1:34:06     
cell types across space and across time and then of course we have we have to     
1:34:12     
plug cell migration into the equation where cells are moving around not just     
1:34:17     
in CIT tube but they're moving around the embryo and of course if you produce more cells uh they can move you know     
1:34:24     
more of them are sort of diffuse across the embryo they move to new places and     
1:34:29     
you get the kind of differences I think in figure two or three they the threedimensional     
1:34:34     
models and the uh mutational changes like to this dispersion of different     
1:34:41     
sublineages across the and so our cell determinance play a     
1:34:48     
role in this Sal migration plays a role these asymmetries in tissue growth versus sulfate specification     
1:34:54     
so this is their model for temporal coordination during Monona development and it leads to this sort of leak to     
1:35:14     
tissue so that's the paper um and thanks for paying attention hope you learn     
1:35:20     
something     
